[![EMBO | EMBL Symposia](https://doi.org/sda/501054/SNS25-01_Conferences_leaderboard_728x90px.gif)](https://doi.org/action/clickThrough?id=501054&url=https%3A%2F%2Fwww.embl.org%2Fabout%2Finfo%2Fcourse-and-conference-office%2Fevents%2FSNS25-01%2F%3Futm_source%3DEMBOPress%26utm_medium%3Dreferral%26utm_id%3DSNS25-01&loc=%2Fdoi%2Ffull%2F10.15252%2Fmsb.20188746&pubId=40559820&placeholderId=501022&productId=501022 "EMBO | EMBL Symposia")
[![logo](https://doi.org/pb-assets/embo-site/images/17444292-1553268207403.svg)](https://doi.org/journal/17444292)
[![embp press logo](https://doi.org/pb-assets/embo-site/images/EMBOpress_monochrome-1553268207193.svg)](https://doi.org/)
[ JOURNALS](https://doi.org/10.15252/msb.20188746)
  * [ ![The EMBO Journal](https://doi.org/10.15252/msb.20188746) ![The EMBO Journal](https://doi.org/pb-assets/embo-site/images/EMBOjournal-1553268207590.svg)The EMBO Journal ](https://doi.org/journal/14602075)
  * [ ![EMBO Reports](https://doi.org/10.15252/msb.20188746) ![The Reports](https://doi.org/pb-assets/embo-site/images/EMBOreports-1553268207573.svg)EMBO Reports ](https://doi.org/journal/14693178)
  * [ ![Molecular Systems Biology](https://doi.org/10.15252/msb.20188746) ![Molecular Systems Biology](https://doi.org/pb-assets/embo-site/images/EMBOmsb-1553268207403.svg)Molecular Systems Biology ](https://doi.org/journal/17444292)
  * [ ![EMBO Molecular Medicine](https://doi.org/10.15252/msb.20188746) ![EMBO Molecular Medicine](https://doi.org/pb-assets/embo-site/images/EMBOmolecularmedicine-1553268207420.svg)EMBO Molecular Medicine ](https://doi.org/journal/17574684)

[This Journal](https://doi.org/10.15252/msb.20188746)
  * [This Journal](https://doi.org/10.15252/msb.20188746#pane-1d947046-fbb2-4a34-8b7b-f39302c8c2330)
  * [Anywhere](https://doi.org/10.15252/msb.20188746#pane-1d947046-fbb2-4a34-8b7b-f39302c8c2331)

Search journal
Search
Search EMBO Press
Search
[Advanced search](https://doi.org/search/advanced) [Citation search](https://doi.org/search/advanced#citation)
  * [ Login ](https://doi.org/action/showLogin?uri=%2Fdoi%2Ffull%2F10.15252%2Fmsb.20188746)

[Close Drawer MenuOpen Drawer Menu](https://doi.org/10.15252/msb.20188746#embo-menu)[Menu](https://doi.org/10.15252/msb.20188746)
  *     * [This Journal](https://doi.org/10.15252/msb.20188746#pane-1d947046-fbb2-4a34-8b7b-f39302c8c2330)
    * [Anywhere](https://doi.org/10.15252/msb.20188746#pane-1d947046-fbb2-4a34-8b7b-f39302c8c2331)
Search journal
Search
Search EMBO Press
Search
[Advanced search](https://doi.org/search/advanced) [Citation search](https://doi.org/search/advanced#citation)
  * [JOURNALS](https://doi.org/10.15252/msb.20188746)
    *       * [ ![The EMBO Journal](https://doi.org/10.15252/msb.20188746) ![The EMBO Journal](https://doi.org/pb-assets/embo-site/images/EMBOjournal-1553268207590.svg)The EMBO Journal ](https://doi.org/journal/14602075)
      * [ ![EMBO Reports](https://doi.org/10.15252/msb.20188746) ![The Reports](https://doi.org/pb-assets/embo-site/images/EMBOreports-1553268207573.svg)EMBO Reports ](https://doi.org/journal/14693178)
      * [ ![Molecular Systems Biology](https://doi.org/10.15252/msb.20188746) ![Molecular Systems Biology](https://doi.org/pb-assets/embo-site/images/EMBOmsb-1553268207403.svg)Molecular Systems Biology ](https://doi.org/journal/17444292)
      * [ ![EMBO Molecular Medicine](https://doi.org/10.15252/msb.20188746) ![EMBO Molecular Medicine](https://doi.org/pb-assets/embo-site/images/EMBOmolecularmedicine-1553268207420.svg)EMBO Molecular Medicine ](https://doi.org/journal/17574684)
  * [ Login ](https://doi.org/action/showLogin?uri=%2Fdoi%2Ffull%2F10.15252%2Fmsb.20188746)

[Skip main navigation](https://doi.org/10.15252/msb.20188746#afterNav-j7a)
Close Drawer MenuOpen Drawer Menu[Menu](https://doi.org/10.15252/msb.20188746)
  * [Current Issue](https://doi.org/toc/17444292/current)
  * [About](https://doi.org/10.15252/msb.20188746)
    * [Aims & Scope](https://doi.org/page/journal/17444292/about)
    * [Editors & Board](https://doi.org/page/journal/17444292/editors)
    * [EMBO Press Policies](https://doi.org/policies)
    * [Open Access](https://doi.org/page/journal/17444292/about#openaccess)
    * [Licenses & Charges](https://doi.org/page/journal/17444292/authorguide#chargesguide)
    * [Journal Indicators](https://doi.org/page/journal/17444292/about#bibliometrics)
  * [Information](https://doi.org/10.15252/msb.20188746)
    * [For Authors](https://doi.org/page/journal/17444292/authorguide)
    * [For Reviewers](https://doi.org/page/journal/17444292/refereeguide)
    * [For Media](https://doi.org/media)
  * [Archive](https://doi.org/loi/17444292)
  * [Alerts](https://doi.org/action/doUpdateAlertSettings?action=addJournal&journalCode=17444292)
  * [Submit](https://doi.org/page/journal/17444292/submit)

Review
19 June 2019
Open access
# Current best practices in single‐cell RNA‐seq analysis: a tutorial
[Malte D Luecken](https://doi.org/10.15252/msb.20188746#msb188746-cr-0001) and [Fabian J Theis](https://doi.org/10.15252/msb.20188746#msb188746-cr-0002) [Author Information](https://doi.org/10.15252/msb.20188746#tab-contributors)
Mol Syst Biol
(2019)
15: e8746
<https://doi.org/10.15252/msb.20188746>
[](https://doi.org/action/addCitationAlert?doi=10.15252%2Fmsb.20188746 "Track Citations")[](https://doi.org/personalize/addFavoritePublication?doi=10.15252%2Fmsb.20188746 "Bookmark")
[](https://doi.org/10.15252/msb.20188746#tab-citations "Cite")
[](https://doi.org/10.15252/msb.20188746#core-collateral-share "Share")
Contents ![logo](https://doi.org/pb-assets/embo-site/images/17444292-1553268207403.svg)
[Vol. 15 | No. 6](https://doi.org/toc/17444292/15/6)
  * [Abstract](https://doi.org/10.15252/msb.20188746#abstract)
  * [Introduction](https://doi.org/10.15252/msb.20188746#sec-1)
  * [Pre‐processing and visualization](https://doi.org/10.15252/msb.20188746#sec-2)
  * [Analysis platforms](https://doi.org/10.15252/msb.20188746#sec-3)
  * [Conclusions and outlook](https://doi.org/10.15252/msb.20188746#sec-4)
  * [Author contributions](https://doi.org/10.15252/msb.20188746#sec-5)
  * [Acknowledgements](https://doi.org/10.15252/msb.20188746#acknowledgments)
  * [
  * [References](https://doi.org/10.15252/msb.20188746#bibliography)

  * [Information & Authors](https://doi.org/10.15252/msb.20188746#core-collateral-info "Information & Authors")
  * [Metrics & Citations](https://doi.org/10.15252/msb.20188746#core-collateral-metrics "Metrics & Citations")
  * [References](https://doi.org/10.15252/msb.20188746#core-collateral-references "References")
  * [Figures](https://doi.org/10.15252/msb.20188746#core-collateral-figures "Figures")
  * [Tables](https://doi.org/10.15252/msb.20188746#core-collateral-tables "Tables")
  * [Media](https://doi.org/10.15252/msb.20188746#core-collateral-media "Media")
  * [Share](https://doi.org/10.15252/msb.20188746#core-collateral-share "Share")

## Abstract
Single‐cell RNA‐seq has enabled gene expression to be studied at an unprecedented resolution. The promise of this technology is attracting a growing user base for single‐cell analysis methods. As more analysis tools are becoming available, it is becoming increasingly difficult to navigate this landscape and produce an up‐to‐date workflow to analyse one's data. Here, we detail the steps of a typical single‐cell RNA‐seq analysis, including pre‐processing (quality control, normalization, data correction, feature selection, and dimensionality reduction) and cell‐ and gene‐level downstream analysis. We formulate current best‐practice recommendations for these steps based on independent comparison studies. We have integrated these best‐practice recommendations into a workflow, which we apply to a public dataset to further illustrate how these steps work in practice. Our documented case study can be found at . This review will serve as a workflow tutorial for new entrants into the field, and help established users update their analysis pipelines.
## Introduction
In recent years, single‐cell RNA sequencing (scRNA‐seq) has significantly advanced our knowledge of biological systems. We have been able to both study the cellular heterogeneity of zebrafish, frogs and planaria (Briggs _et al_ , [2018](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0016); Plass _et al_ , [2018](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0102); Wagner _et al_ , [2018](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0138)) and discover previously obscured cellular populations (Montoro _et al_ , [2018](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0092); Plasschaert _et al_ , [2018](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0103)). The great potential of this technology has motivated computational biologists to develop a range of analysis tools (Rostom _et al_ , [2017](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0114)). Despite considerable effort being undertaken by the field to ensure the usability of individual tools, a barrier of entry for novices in single‐cell data analysis is the lack of standardization due to the relative immaturity of the field. In this paper, we present a tutorial for scRNA‐seq analysis and outline current best practices to lay a foundation for future analysis standardization.
The challenges to standardization include the growing number of analysis methods (385 tools as of 7 March 2019) and exploding dataset sizes (Angerer _et al_ , [2017](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0006); Zappia _et al_ , [2018](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0152)). We are continuously finding new ways to use the data at our disposal. For example, it has recently become possible to predict cell fates in differentiation (La Manno _et al_ , [2018](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0068)). While the continuous improvement of analysis tools is beneficial for generating new scientific insight, it complicates standardization.
Further challenges for standardization lie in technical aspects. Analysis tools for scRNA‐seq data are written in a variety of programming languages—most prominently R and Python (Zappia _et al_ , [2018](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0152)). Although cross‐environment support is growing (preprint: Scholz _et al_ , [2018](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0120)), the choice of programming language is often also a choice between analysis tools. Popular platforms such as Seurat (Butler _et al_ , [2018](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0020)), Scater (McCarthy _et al_ , [2017](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0087)), or Scanpy (Wolf _et al_ , [2018](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0146)) provide integrated environments to develop pipelines and contain large analysis toolboxes. However, out of necessity these platforms limit themselves to tools developed in their respective programming languages. By extension, language restrictions also hold true for currently available scRNA‐seq analysis tutorials, many of which revolve around the above platforms (R and bioconductor tools: and ; Lun _et al_ , [2016b](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0079); Seurat: ; Scanpy: ).
Considering the above‐mentioned challenges, instead of targeting a standardized analysis pipeline, we outline current best practices and common tools independent of programming language. We guide the reader through the various steps of a scRNA‐seq analysis pipeline (Fig [1](https://doi.org/10.15252/msb.20188746#msb188746-fig-0001)), present current best practices, and discuss analysis pitfalls and open questions. Where best practices cannot be determined due to novelty of the tools and lack of comparisons, we list popular available tools. The outlined steps start from read or count matrices and lead to potential analysis endpoints. Earlier pre‐processing steps are covered in Lun et al ([2016b](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0079)). A detailed case study that integrates the established current best practices is available on our github at: . Here, we have applied the current best practices in a practical example workflow to analyse a public dataset. The analysis workflow integrates R and Python tools in a Jupyter–Ipython notebook with rpy2. With the available documentation, it is readily adaptable as a workflow template.
[![image](https://doi.org/cms/10.15252/msb.20188746/asset/5bcc5719-49b9-4156-a754-76a2448e5392/assets/graphic/msb188746-fig-0001-m.png)](https://doi.org/10.15252/msb.20188746#msb188746-fig-0001)
Figure 1. Schematic of a typical single‐cell RNA‐seq analysis workflow
Raw sequencing data are processed and aligned to give count matrices, which represent the start of the workflow. The count data undergo pre‐processing and downstream analysis. Subplots are generated using the best‐practices workflow on intestinal epithelium data from Haber _et al_ ([2017](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0046)).
  * [Download figure](https://doi.org/cms/asset/5bcc5719-49b9-4156-a754-76a2448e5392/msb188746-fig-0001-m.png)
  * [Download PowerPoint](https://doi.org/action/downloadFigures?doi=10.15252%2Fmsb.20188746&id=msb188746-fig-0001)

[Open in viewer](https://doi.org/10.15252/msb.20188746#msb188746-fig-0001)
## Pre‐processing and visualization
Raw data generated by sequencing machines are processed to obtain matrices of molecular counts (count matrices) or, alternatively, read counts (read matrices), depending on whether unique molecular identifiers (UMIs) were incorporated in the single‐cell library construction protocol (see Box [1](https://doi.org/10.15252/msb.20188746#msb188746-blk-0001) for an overview of the experimental steps that precede the analysis). Raw data processing pipelines such as Cell Ranger (Zheng _et al_ , [2017](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0156)), indrops (Klein _et al_ , [2015](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0065)), SEQC (Azizi _et al_ , [2018](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0010)), or zUMIs (Parekh _et al_ , [2018](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0097)) take care of read quality control (QC), assigning reads to their cellular barcodes and mRNA molecules of origin (also called “demultiplexing”), genome alignment, and quantification. The resulting read or count matrices have the dimension number of barcodes x number of transcripts. The term “barcode” is used here instead of “cell” as all reads assigned to the same barcode may not correspond to reads from the same cell. A barcode may mistakenly tag multiple cells (doublet) or may not tag any cells (empty droplet/well).
### Box 1: Key elements of an experimental scRNA‐seq workflow
Generating single‐cell data from a biological sample requires multiple steps. Typical workflows incorporate single‐cell dissociation, single‐cell isolation, library construction, and sequencing. We give a brief overview of these stages here. A more detailed explanation and comparison of different protocols can be found in Ziegenhain _et al_ ([2017](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0159)); Macosko _et al_ ([2015](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0083)); Svensson _et al_ ([2017](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0126)).
Input material for a single‐cell experiment is typically obtained in the form of biological tissue samples. As a first step, a single‐cell suspension is generated in a process called _single‐cell dissociation_ in which the tissue is digested.
To profile the mRNA in each cell separately, cells must be isolated. _Single‐cell isolation_ is performed differently depending on the experimental protocol. While plate‐based techniques isolate cells into wells on a plate, droplet‐based methods rely on capturing each cell in its own microfluidic droplet. In both cases, errors can occur that lead to multiple cells being captured together (_doublets_ or _multiplets_), non‐viable cells being captured, or no cell being captured at all (_empty droplets/wells_). Empty droplets are especially common as droplet‐based methods rely on a low concentration flow of input cells to control doublet rates.
Each well or droplet contains the necessary chemicals to break down the cell membranes and perform _library construction_. Library construction is the process in which the intracellular mRNA is captured, reverse‐transcribed to cDNA molecules and amplified. As cells undergo this process in isolation, the mRNA from each cell can be labelled with a well‐ or droplet‐specific cellular barcode. Furthermore, many experimental protocols also label captured molecules with a _unique molecular identifier (UMI)_. Cellular cDNA is amplified before sequencing to increase its probability of being measured. UMIs allow us to distinguish between amplified copies of the same mRNA molecule and reads from separate mRNA molecules transcribed from the same gene.
After library construction, cellular cDNA libraries are labelled with cellular barcodes and, depending on the protocol, UMIs. These libraries are pooled together (_multiplexed_) for _sequencing_. Sequencing produces read data, which undergo quality control, grouping based on their assigned barcodes (_demultiplexing_) and alignment in read processing pipelines. For UMI‐based protocols, read data can be further demultiplexed to produce counts of captured mRNA molecules (_count data_).
While read and count data differ in the level of measurement noise, the processing steps in a typical analysis pipeline are the same. For simplicity, we will refer to the data as count matrices in this tutorial. Where results for read and count matrices differ, read matrices are specifically mentioned.
### Quality control
Before analysing the single‐cell gene expression data, we must ensure that all cellular barcode data correspond to viable cells. Cell QC is commonly performed based on three QC covariates: the number of counts per barcode (count depth), the number of genes per barcode, and the fraction of counts from mitochondrial genes per barcode (Ilicic _et al_ , [2016](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0056); Griffiths _et al_ , [2018](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0044)). The distributions of these QC covariates are examined for outlier peaks that are filtered out by thresholding (Fig [2](https://doi.org/10.15252/msb.20188746#msb188746-fig-0002)). These outlier barcodes can correspond to dying cells, cells whose membranes are broken, or doublets. For example, barcodes with a low count depth, few detected genes, and a high fraction of mitochondrial counts are indicative of cells whose cytoplasmic mRNA has leaked out through a broken membrane, and thus, only mRNA located in the mitochondria is still conserved (Fig [2](https://doi.org/10.15252/msb.20188746#msb188746-fig-0002)). In contrast, cells with unexpectedly high counts and a large number of detected genes may represent doublets. Thus, high‐count depth thresholds are commonly used to filter out potential doublets. Three recent doublet detection tools offer more elegant and potentially better solutions (DoubletDecon: preprint: DePasquale _et al_ , [2018](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0030); Scrublet: Wolock _et al_ , [2019](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0148); Doublet Finder: McGinnis _et al_ , [2018](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0089)).
[![image](https://doi.org/cms/10.15252/msb.20188746/asset/1bd082a4-169f-4878-9161-5cab859d983c/assets/graphic/msb188746-fig-0002-m.png)](https://doi.org/10.15252/msb.20188746#msb188746-fig-0002)
Figure 2. Plots of quality control metrics with filtering decisions for a mouse intestinal epithelium dataset from Haber _et al_ ([2017](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0046))
(A) Histograms of count depth per cell. The smaller histogram is zoomed‐in on count depths below 4,000. A threshold is applied here at 1,500 based on the peak detected at around 1,200 counts. (B) Histogram of the number of genes detected per cell. A small noise peak is visible at approx. 400 genes. These cells are filtered out using the depicted threshold (red line) at 700 genes. (C) Count depth distribution from high to low count depths. This visualization is related to the log–log plot shown in Cell Ranger outputs that is used to filter out empty droplets. It shows an “elbow” where count depths start to decrease rapidly around 1,500 counts. (D) Number of genes versus the count depth coloured by the fraction of mitochondrial reads. Mitochondrial read fractions are only high in particularly low count cells with few detected genes. These cells are filtered out by our count and gene number thresholds. Jointly visualizing the count and gene thresholds shows the joint filtering effect, indicating that a lower gene threshold may have sufficed. [Correction added on 5 July 2019, after first online publication: the x‐axis label in panel B was corrected from “Count depth” to “Number of genes”.]
  * [Download figure](https://doi.org/cms/asset/1bd082a4-169f-4878-9161-5cab859d983c/msb188746-fig-0002-m.png)
  * [Download PowerPoint](https://doi.org/action/downloadFigures?doi=10.15252%2Fmsb.20188746&id=msb188746-fig-0002)

[Open in viewer](https://doi.org/10.15252/msb.20188746#msb188746-fig-0002)
Considering any of these three QC covariates in isolation can lead to misinterpretation of cellular signals. For example, cells with a comparatively high fraction of mitochondrial counts may be involved in respiratory processes. Likewise, other QC covariates also have biological interpretations. Cells with low counts and/or genes may correspond to quiescent cell populations, and cells with high counts may be larger in size. Indeed, molecular counts can differ strongly between cells (see case study on project github). Thus, QC covariates should be considered jointly when univariate thresholding decisions are made (Fig [2](https://doi.org/10.15252/msb.20188746#msb188746-fig-0002)D), and these thresholds should be set as permissive as possible to avoid filtering out viable cell populations unintentionally. In future, filtering models that account for multivariate QC dependencies may provide more sensitive QC options.
Datasets that contain heterogeneous mixtures of cell types may exhibit multiple QC covariate peaks. For example, Fig [2](https://doi.org/10.15252/msb.20188746#msb188746-fig-0002)D shows two populations of cells with different QC distributions. If no previous filtering step was performed (note that Cell Ranger also performs cell QC), then only the lowest count depth and gene per barcode peak should be considered as non‐viable cells. A further thresholding guideline is the proportion of cells that are filtered out with the chosen threshold. For high‐count filtering, this proportion should not exceed the expected doublet rate.
In addition to checking the integrity of cells, QC steps must also be performed at the level of transcripts. Raw count matrices often include over 20,000 genes. This number can be drastically reduced by filtering out genes that are not expressed in more than a few cells and are thus not informative of the cellular heterogeneity. A guideline to setting this threshold is to use the minimum cell cluster size that is of interest and leaving some leeway for dropout effects. For example, filtering out genes expressed in fewer than 20 cells may make it difficult to detect cell clusters with fewer than 20 cells. For datasets with high dropout rates, this threshold may also complicate the detection of larger clusters. The choice of threshold should scale with the number of cells in the dataset and the intended downstream analysis.
Further QC can be performed on the count data directly. _Ambient gene expression_ refers to counts that do not originate from a barcoded cell, but from other lysed cells whose mRNA contaminated the cell suspension prior to library construction. These added ambient counts can distort downstream analysis such as marker gene identification or other differential expression tests especially when levels vary between samples. It is possible to correct for these effects in droplet‐based scRNA‐seq datasets due to the large numbers of empty droplets, which can be used to model ambient RNA expression profiles. The recently developed SoupX (preprint: Young & Behjati, [2018](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0150)) uses this approach to directly correct the count data. Pragmatic approaches that ignore strongly ambient genes in downstream analysis have also been used to tackle this problem (Angelidis _et al_ , [2019](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0005)).
Quality control is performed to ensure that the data quality is sufficient for downstream analysis. As “sufficient data quality” cannot be determined _a priori_ , it is judged based on downstream analysis performance (e.g., cluster annotation). Thus, it may be necessary to revisit quality control decisions multiple times when analysing the data. Often it is beneficial to start with permissive QC thresholds and investigate the effects of these thresholds before going back to perform more stringent QC. This approach is particularly relevant for datasets containing heterogeneous cell populations where cell types or states may be misinterpreted as low‐quality outlier cells. In low‐quality datasets, stringent QC thresholds may be necessary. The quality of a dataset can be determined by experimental QC metrics (see [Appendix Supplementary Text S2](https://doi.org/10.15252/msb.20188746#msb188746-sup-0001)). In this iterative QC optimization, one should be aware of _data peeking_. QC thresholds should not be adapted to improve the outcome of a statistical test. Instead, QC can be evaluated from the distribution of QC covariates in dataset visualizations and clustering.
#### Pitfalls & recommendations:
•
Perform QC by finding outlier peaks in the number of genes, the count depth and the fraction of mitochondrial reads. Consider these covariates jointly instead of separately.
•
Be as permissive of QC thresholding as possible, and revisit QC if downstream clustering cannot be interpreted.
•
If the distribution of QC covariates differ between samples, QC thresholds should be determined separately for each sample to account for sample quality differences as in Plasschaert _et al_ ([2018](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0103)).
### Normalization
Each count in a count matrix represents the successful capture, reverse transcription and sequencing of a molecule of cellular mRNA (Box [1](https://doi.org/10.15252/msb.20188746#msb188746-blk-0001)). Count depths for identical cells can differ due to the variability inherent in each of these steps. Thus, when gene expression is compared between cells based on count data, any difference may have arisen solely due to sampling effects. Normalization addresses this issue by e.g. scaling count data to obtain correct relative gene expression abundances between cells.
Many normalization methods exist for bulk gene expression (preprint: Pachter, [2011](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0096); Dillies _et al_ , [2013](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0032)). While some of these methods have been applied to scRNA‐seq analysis, sources of variation specific to single‐cell data such as _technical dropouts_ (zero counts due to sampling) have prompted the development of scRNA‐seq‐specific normalization methods (Lun _et al_ , [2016a](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0078); Vallejos _et al_ , [2017](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0133)).
The most commonly used normalization protocol is count depth scaling, also referred to as “counts per million” or CPM normalization. This protocol comes from bulk expression analysis and normalizes count data using a so‐called size factor proportional to the count depth per cell. Variations of this method scale the size factors with different factors of 10, or by the median count depth per cell in the dataset. CPM normalization assumes that all cells in the dataset initially contained an equal number of mRNA molecules and count depth differences arise only due to sampling. This assumption is shared with the downsampling protocol, which is the process of randomly sampling reads or counts from the data to leave all cells with a pre‐specified number of counts or fewer. While downsampling throws away data, it also increases technical dropout rates which CPM and other global scaling normalization methods do not. Thus, downsampling can deliver a more realistic representation of what cellular expression profiles would look like at similar count depths.
As single‐cell datasets typically consist of heterogeneous cell populations with varying sizes and molecule counts, more complex normalization methods are usually appropriate. For example, Weinreb _et al_ ([2018](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0144)) used a simple extension of CPM that excludes genes that account for at least 5% of the total counts in any cell, when calculating their size factors. This approach allows for molecular count variability in few highly expressed genes. More cellular heterogeneity is permitted by Scran's pooling‐based size factor estimation method (Lun et al, [2016a](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0078)). Here, size factors are estimated based on a linear regression over genes, after cells are pooled to avoid technical dropout effects. This method limits variability to fewer than 50% of genes being differentially expressed between cells, and is consistently a top‐performing normalization method in independent comparisons. Scran has been shown to perform better than other tested normalization methods for batch correction (Buttner _et al_ , [2019](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0021)) and differential expression analysis (preprint: Vieth _et al,_ [2019](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0500)). The method was also shown to give robust size factor estimates in a small‐scale comparison from the original authors (Vallejos _et al_ , [2017](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0133)).
CPM, high‐count filtering CPM, and scran use linear, global scaling to normalize count data. Non‐linear normalization methods, which can account for more complex unwanted variation, also exist (Cole _et al_ , [2019](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0028)). Many such methods involve the parametric modelling of count data. For example, Mayer _et al_ ([2018](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0086)) fit a negative binomial model to count data, using technical covariates such as the read depth and the number of counts per gene to fit the model parameters. The residuals of the model fit serve as a normalized quantification of gene expression. Such an approach can combine technical and biological data correction (e.g. batch correction or correction for cell cycle effects) with count depth normalization. Non‐linear normalization methods have been shown to outperform global scaling methods especially in situations with strong batch effects (Cole _et al_ , [2019](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0028)). Thus, non‐linear normalization methods are particularly relevant for plate‐based scRNA‐seq data, which tend to have batch effects between plates. Furthermore, plate‐based data can exhibit larger variations in count depths per cell than droplet‐based data (Svensson _et al_ , [2017](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0126)). While non‐linear normalization methods, or alternative approaches such as downsampling, appear better suited to these conditions, comparative studies are needed to confirm this hypothesis. In this tutorial, we prefer to separate the normalization and data correction (batch correction, noise correction, etc.) steps to emphasize different processing stages of the data (see “Stages of pre‐processed data” section). Thus, we focus on global scaling normalization methods.
We cannot expect that a single normalization method is appropriate for all types of scRNA‐seq data. For example, Vieth _et al_ ([2017](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0136)) showed that read and count data are best fit by different models. Indeed Cole _et al_ ([2019](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0028)) find that different normalization methods perform optimally for different datasets and argue that their _scone_ tool should be used to select the appropriate normalization method for a specific dataset. Furthermore, scRNA‐seq techniques can be divided into full‐length and 3′ enrichment methods (Svensson _et al_ , [2017](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0126); Ziegenhain _et al_ , [2017](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0159)). Data from full‐length protocols may benefit from normalization methods that take into account gene length (e.g. Patel _et al_ , [2014](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0099); Kowalczyk _et al_ , [2015](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0067); Soneson & Robinson, [2018](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0123)), while 3′ enrichment data do not. A commonly used normalization method for full‐length scRNA‐seq data is TPM normalization (Li _et al_ , [2009](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0071)), which comes from bulk RNA‐seq analysis.
In the same way that cellular count data can be normalized to make them comparable between cells, gene counts can be scaled to improve comparisons between genes. Gene normalization constitutes scaling gene counts to have zero mean and unit variance (z scores). This scaling has the effect that all genes are weighted equally for downstream analysis. There is currently no consensus on whether or not to perform normalization over genes. While the popular Seurat tutorials (Butler _et al_ , [2018](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0020)) generally apply gene scaling, the authors of the Slingshot method opt against scaling over genes in their tutorial (Street _et al_ , [2018](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0125)). The preference between the two choices revolves around whether all genes should be weighted equally for downstream analysis, or whether the magnitude of expression of a gene is an informative proxy for the importance of the gene. In order to retain as much biological information as possible from the data, we opt to refrain from scaling over genes in this tutorial.
After normalization, data matrices are typically log(x+1)‐transformed. This transformation has three important effects. Firstly, distances between log‐transformed expression values represent log fold changes, which are the canonical way to measure changes in expression. Secondly, log transformation mitigates (but does not remove) the mean–variance relationship in single‐cell data (Brennecke _et al_ , [2013](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0015)). Finally, log transformation reduces the skewness of the data to approximate the assumption of many downstream analysis tools that the data are normally distributed. While scRNA‐seq data are not in fact log‐normally distributed (Vieth _et al_ , [2017](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0136)), these three effects make the log transformation a crude, but useful tool. This usefulness is highlighted by downstream applications for differential expression testing (Finak _et al_ , [2015](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0037); Ritchie _et al_ , [2015](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0112)) or batch correction (Johnson _et al_ , [2006](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0058); Buttner _et al_ , [2019](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0021)) that use log transformation for these purposes. It should however be noted that log transformation of normalized data can introduce spurious differential expression effects into the data (preprint: Lun, [2018](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0081)). This effect is particularly pronounced when normalization size factor distributions differ strongly between tested groups.
#### Pitfalls & recommendations:
•
We recommend scran for normalization of non‐full‐length datasets. An alternative is to evaluate normalization approaches via _scone_ especially for plate‐based datasets. Full‐length scRNA‐seq protocols can be corrected for gene length using bulk methods.
•
There is no consensus on scaling genes to 0 mean and unit variance. We prefer not to scale gene expression.
•
Normalized data should be log(x+1)‐transformed for use with downstream analysis methods that assume data are normally distributed.
### Data correction and integration
Normalization as described above attempts to remove the effects of count sampling. However, normalized data may still contain unwanted variability. Data correction targets further technical and biological covariates such as batch, dropout, or cell cycle effects. These covariates are not always corrected for. Instead, the decision of which covariates to consider will depend on the intended downstream analysis. We propose to consider correction for biological and technical covariates separately as these are used for different purposes and present unique challenges.
#### Regressing out biological effects
While correcting for technical covariates may be crucial to uncovering the underlying biological signal, correction for biological covariates serves to single out particular biological signals of interest. The most common biological data correction is to remove the effects of the cell cycle on the transcriptome. This data correction can be performed by a simple linear regression against a cell cycle score as implemented in the Scanpy and Seurat platforms (Butler _et al_ , [2018](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0020); Wolf _et al_ , [2018](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0146)) or in specialized packages with more complex mixture models such as scLVM (Buettner _et al_ , [2015](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0018)) or f‐scLVM (Buettner _et al_ , [2017](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0019)). Lists of marker genes to compute cell cycle scores are obtained from the literature (Macosko _et al_ , [2015](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0083)). These methods can also be used to regress out other known biological effects such as mitochondrial gene expression, which is interpreted as an indication of cell stress.
Several aspects should be considered prior to correcting data for biological effects. Firstly, correcting for biological covariates is not always helpful to interpret scRNA‐seq data. While removing cell cycle effects can improve the inference of developmental trajectories (Buettner _et al_ , [2015](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0018); Vento‐Tormo _et al_ , [2018](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0135)), cell cycle signals can also be informative of the biology. For example, proliferating cell populations can be identified based on cell cycle scores (see case study on project github). Also, biological signals must be understood in context. Given that biological processes occur within the same organism, there exist dependencies between these processes. Thus, correcting for one process may unintentionally mask the signal of another. Finally, it has been argued that variation in cell size accounts for the transcriptomic effect generally attributed to the cell cycle (McDavid _et al_ , [2016](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0088)). Thus, correcting for cell size via normalization, or dedicated tools such as cgCorrect (Blasi _et al_ , [2017](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0013)), also partially corrects for cell cycle effects in scRNA‐seq data.
### Regressing out technical effects
The variants of regression models used to regress out biological covariates can also be applied to technical covariates. The most prominent technical covariates in single‐cell data are count depth and batch. Although normalization scales count data to render gene counts comparable between cells, a count depth effect often remains in the data. This count depth effect can be both a biological and a technical artefact. For example, cells may differ in size and therefore in mRNA molecule counts. Yet, technical count effects may remain after normalization as no scaling method can infer the expression values of genes that were not detected due to poor sampling. Regressing out count depth effects can improve the performance of trajectory inference algorithms, which rely on finding transitions between cells (see case study on project github). When correcting for multiple covariates (e.g. cell cycle and count depth), the regression should be performed over all covariates in a single step to account for dependence between covariates.
An alternative to regression‐based strategies for removing count effects is to use a more rigorous normalization procedure such as downsampling or non‐linear normalization methods (see “Normalization” section). These approaches may be particularly relevant for plate‐based scRNA‐seq datasets where the larger variation of count depths per cell can mask the heterogeneity between cells.
### Batch effects and data integration
Batch effects can occur when cells are handled in distinct groups. These groups can consist of cells on different chips, cells in different sequencing lanes or cells harvested at different time points. The differing environments experienced by the cells can have an effect on the measurement of the transcriptome or on the transcriptome itself. The resulting effects exist on multiple levels: between groups of cells in an experiment, between experiments performed in the same laboratory or between datasets from different laboratories. Here, we distinguish between the first and the last two scenarios. Correcting for batch effects between samples or cells in the same experiment is the classical scenario known as _batch correction_ from bulk RNA‐seq. We distinguish this from the integration of data from multiple experiments, which we call _data integration_. While batch effects are typically corrected using linear methods, non‐linear approaches are used for data integration.
A recent comparison of classical batch correction methods has revealed that ComBat (Johnson _et al_ , [2006](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0058)) performs well also for single‐cell experiments of low‐to‐medium complexity (Buttner _et al_ , [2019](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0021)). ComBat consists of a linear model of gene expression where the batch contribution is taken into account both in the mean and the variance of the data (Fig [3](https://doi.org/10.15252/msb.20188746#msb188746-fig-0003)). Irrespective of computational methods, the best method of batch correction is pre‐empting the effect and avoiding it altogether by clever experimental design (Hicks _et al_ , [2017](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0051)). Batch effects can be avoided by pooling cells across experimental conditions and samples. Using strategies such as cell tagging (preprint: Gehring _et al_ , [2018](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0042)), or via genetic variation (Kang _et al_ , [2018](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0060)), it is possible to demultiplex cells that were pooled in the experiment.
[![image](https://doi.org/cms/10.15252/msb.20188746/asset/0ed29c9f-28ff-40f8-a712-110306de12dc/assets/graphic/msb188746-fig-0003-m.png)](https://doi.org/10.15252/msb.20188746#msb188746-fig-0003)
Figure 3. UMAP visualization before and after batch correction
Cells are coloured by sample of origin. Separation of batches is clearly visible before batch correction and less visible afterwards. Batch correction was performed using ComBat on mouse intestinal epithelium data from Haber _et al_ ([2017](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0046)).
  * [Download figure](https://doi.org/cms/asset/0ed29c9f-28ff-40f8-a712-110306de12dc/msb188746-fig-0003-m.png)
  * [Download PowerPoint](https://doi.org/action/downloadFigures?doi=10.15252%2Fmsb.20188746&id=msb188746-fig-0003)

[Open in viewer](https://doi.org/10.15252/msb.20188746#msb188746-fig-0003)
In comparison with batch correction, the additional challenge that data integration methods face revolves around compositional differences between datasets. When estimating batch effects, ComBat uses all cells in a batch to fit batch parameters. This approach will confound the batch effect with biological differences between cell types or states that are not shared among datasets. Data integration methods such as Canonical Correlation Analysis (CCA; Butler _et al_ , [2018](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0020)), Mutual Nearest Neighbours (MNN; Haghverdi _et al_ , [2018](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0049)), Scanorama (preprint: Hie _et al_ , [2018](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0052)), RISC (preprint: Liu _et al_ , [2018](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0074)), scGen (preprint: Lotfollahi _et al_ , [2018](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0076)), LIGER (preprint: Welch _et al_ , [2018](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0145)), BBKNN (preprint: Park _et al_ , [2018](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0098)), and Harmony (preprint: Korsunsky _et al_ , [2018](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0066)) have been developed to overcome this issue. While data integration methods can also be applied to simple batch correction problems, we recommend to be wary of over‐correction given the increased degrees of freedom of non‐linear data integration approaches. For example, MNN was shown to be outperformed by ComBat in the simpler batch correction setting (Buttner _et al_ , [2019](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0021)). Further comparison studies between data integration and batch correction methods are needed to assess how generally these methods can be applied.
### Expression recovery
A further type of technical data correction is _expression recovery_ (also _denoising_ or _imputation_). Measurements of single‐cell transcriptomes contain various sources of noise (Grün _et al_ , [2014](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0045); Kharchenko _et al_ , [2014](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0061); Hicks _et al_ , [2017](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0051)). A particularly prominent aspect of this noise is dropout. Inferring dropout events, replacing these zeros with appropriate expression values, and reducing the noise in the dataset have been the target of several recent tools (MAGIC: van Dijk _et al_ , [2018](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0031); DCA: Eraslan _et al_ , [2019](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0035); scVI: Lopez _et al_ , [2018](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0075); SAVER: Huang _et al_ , [2018](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0054); scImpute: Li & Li, [2018](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0072)). Performing expression recovery has been shown to improve the estimation of gene–gene correlations (van Dijk _et al_ , [2018](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0031); Eraslan _et al_ , [2019](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0035)). Furthermore, this step can be integrated with normalization, batch correction and other downstream analysis as implemented in the scVI tool (Lopez _et al_ , [2018](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0075)). While most data correction methods take normalized data as input, some expression recovery methods are based on expected negative binomial noise distributions and therefore run on raw count data. When applying expression recovery, one should take into consideration that no method is perfect. Thus, any method may over‐ or under‐correct noise in the data. Indeed, false correlation signals have been reported as a result of expression recovery (Andrews & Hemberg, [2018](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0004)). Given the difficulty of assessing successful expression recovery in a practical application, this scenario represents a challenge to the user pondering whether or not to denoise their data. Furthermore, scalability to large datasets is still an issue for currently available expression recovery methods. There is currently no consensus on how denoised data should be used in the light of these considerations (see “Stages of processed data” section). A prudent approach would be to use expression recovery only for visual display of data rather than to generate hypotheses during exploratory data analysis. Thorough experimental validation is particularly important here.
#### Pitfalls & recommendations:
•
Regress out biological covariates only for trajectory inference and if other biological processes of interest are not masked by the regressed out biological covariate.
•
Regress out technical and biological covariates jointly rather than serially.
•
Plate‐based dataset pre‐processing may require regressing out counts, normalization via non‐linear normalization methods or downsampling.
•
We recommend performing batch correction via ComBat when cell type and state compositions between batches are consistent
•
Data integration and batch correction should be performed by different methods. Data integration tools may over‐correct simple batch effects.
•
Users should be cautious of signals found only after expression recovery. Exploratory analysis may be best performed without this step.
### Feature selection, dimensionality reduction and visualization
A human single‐cell RNA‐seq dataset can contain expression values for up to 25,000 genes. Many of these genes will not be informative for a given scRNA‐seq dataset, and many genes will mostly contain zero counts. Even after filtering out these zero count genes in the QC step, the feature space for a single‐cell dataset can have over 15,000 dimensions. To ease the computational burden on downstream analysis tools, reduce the noise in the data, and to visualize the data, one can use several approaches to reduce the dimensionality of the dataset.
#### Feature selection
The first step of reducing the dimensionality of scRNA‐seq datasets commonly is feature selection. In this step, the dataset is filtered to keep only genes that are “informative” of the variability in the data. Thus, highly variable genes (HVGs) are often used (Brennecke _et al_ , [2013](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0015)). Depending on the task and the complexity of the dataset, typically between 1,000 and 5,000 HVGs are selected for downstream analysis (see Fig [EV1](https://doi.org/10.15252/msb.20188746#msb188746-fig-0001ev) and [Dataset EV1](https://doi.org/10.15252/msb.20188746#msb188746-sup-0003)). Preliminary results from Klein _et al_ ([2015](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0065)) suggest that downstream analysis is robust to the exact choice of the number of HVGs. While varying the number of HVGs between 200 and 2,400, the authors reported similar low‐dimensional representations in the PCA space. Based on this result, we prefer to err on the side of higher numbers of HVGs.
[![image](https://doi.org/cms/10.15252/msb.20188746/asset/40988faf-3bbb-4dc7-ba72-751c5fb2f775/assets/graphic/msb188746-fig-0001ev-m.png)](https://doi.org/10.15252/msb.20188746#msb188746-fig-0001ev)
Figure EV1. The number of highly variable genes (HVGs) used for datasets of different sizes
The data were obtained by a brief manual survey of recent scRNA‐seq analysis papers. The plotted data, along with further information on scRNA‐seq technology, publication year, reference and the number of reads per cell, are available in [Dataset EV1](https://doi.org/10.15252/msb.20188746#msb188746-sup-0003).
  * [Download figure](https://doi.org/cms/asset/40988faf-3bbb-4dc7-ba72-751c5fb2f775/msb188746-fig-0001ev-m.png)
  * [Download PowerPoint](https://doi.org/action/downloadFigures?doi=10.15252%2Fmsb.20188746&id=msb188746-fig-0001ev)

Click here to expand this figure.
[Open in viewer](https://doi.org/10.15252/msb.20188746#msb188746-fig-0001ev)
A simple, yet popular, method of selecting HVGs is implemented in both Scanpy and Seurat. Here, genes are binned by their mean expression, and the genes with the highest variance‐to‐mean ratio are selected as HVGs in each bin. There exist different flavours of this algorithm that expect either count data (Seurat) or log‐transformed data (Cell Ranger). Optimally, HVGs should be selected after technical data correction to avoid selecting genes that are highly variable only due to, e.g., batch effects. Other methods for HVG selection are reviewed in Yip _et al_ ([2018](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0149)).
#### Dimensionality reduction
After feature selection, the dimensions of single‐cell expression matrices can be further reduced by dedicated dimensionality reduction algorithms. These algorithms embed the expression matrix into a low‐dimensional space, which is designed to capture the underlying structure in the data in as few dimensions as possible. This approach works as single‐cell RNA‐seq data are inherently low‐dimensional (Heimberg _et al_ , [2016](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0050)). In other words, the biological manifold on which cellular expression profiles lie can be sufficiently described by far fewer dimensions than the number of genes. Dimensionality reduction aims to find these dimensions.
There are two main objectives of dimensionality reduction methods: _visualization_ and _summarization_. Visualization is the attempt to optimally describe the dataset in two or three dimensions. These reduced dimensions are used as coordinates on a scatter plot to obtain a visual representation of the data. Summarization does not prescribe the number of output components. Instead, higher components become less important for describing the variability present in the data. Summarization techniques can be used to reduce the data to its essential components by finding the inherent dimensionality of the data, and are thus helpful for downstream analysis. While a 2‐dimensional visualization output should not be used to summarize a dataset, a summarization method can be used to visualize the data using the leading reduced components. However, a dedicated visualization technique will typically provide a better representation of the variability.
Reduced dimensions are generated through linear or non‐linear combinations of feature space dimensions (gene expression vectors). Especially in the non‐linear case, the interpretability of the reduced dimensions is sacrificed in this process. An example application of some commonly used dimensionality reduction methods is shown in Fig [4](https://doi.org/10.15252/msb.20188746#msb188746-fig-0004). With a growing list of methods to choose from, it is out of the scope of this tutorial to review these methods in detail. Rather, we briefly outline the practical considerations that may aid users in choosing between common dimensionality reduction methods. A more detailed review of dimensionality reduction for single‐cell analysis can be found in Moon _et al_ ([2018](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0093)).
[![image](https://doi.org/cms/10.15252/msb.20188746/asset/8e97ce5f-5e00-4b4c-981b-fe59d71dd30b/assets/graphic/msb188746-fig-0004-m.png)](https://doi.org/10.15252/msb.20188746#msb188746-fig-0004)
Figure 4. Common visualization methods for scRNA‐seq data
Mouse intestinal epithelium regions data from Haber _et al_ ([2017](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0046)) visualized on the first two components for: (A) PCA, (B) t‐SNE, (C) diffusion maps, (D) UMAP and (E) A force‐directed graph layout via ForceAtlas2. Cells are coloured by count depth. (F) Variance explained by the first 31 principal components (PCs). The “elbow” of this plot, which is used to select relevant PCs to analyse the dataset, lies between PCs 5 and 7.
  * [Download figure](https://doi.org/cms/asset/8e97ce5f-5e00-4b4c-981b-fe59d71dd30b/msb188746-fig-0004-m.png)
  * [Download PowerPoint](https://doi.org/action/downloadFigures?doi=10.15252%2Fmsb.20188746&id=msb188746-fig-0004)

[Open in viewer](https://doi.org/10.15252/msb.20188746#msb188746-fig-0004)
Two popular dimensionality reduction techniques that are principally summarization methods are principal component analysis (PCA; Pearson, [1901](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0101)) and diffusion maps (Coifman _et al_ , [2005](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0027)) which were popularized for single‐cell analysis by Haghverdi _et al_ ([2015](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0047)). Principal component analysis is a linear approach that generates reduced dimensions by maximizing the captured residual variance in each further dimension. While PCA does not capture the structure of the data in few dimensions as well as non‐linear methods, it is the basis of many currently available analysis tools for clustering or trajectory inference. Indeed, PCA is commonly used as a pre‐processing step for non‐linear dimensionality reduction methods. Typically, PCA summarizes a dataset via its top N principal components, where N can be determined by “elbow” heuristics (see Fig [4](https://doi.org/10.15252/msb.20188746#msb188746-fig-0004)F) or the permutation‐test‐based jackstraw method (Chung & Storey, [2015](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0024); Macosko _et al_ , [2015](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0083)). The simple linearity of PCA has the advantage that distances in reduced dimensional space have a consistent interpretation in all regions of this space. Thus, we can correlate quantities of interest with principal components to assess their importance. For example, principal components can be projected onto technical nuisance covariates to investigate the performance of QC, data correction and normalization steps (Buttner _et al_ , [2019](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0021)), or show the importance of genes in the dataset (Chung & Storey, [2015](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0024)). Diffusion maps are a non‐linear data summarization technique. As diffusion components emphasize transitions in the data, they are principally used when continuous processes such as differentiation are of interest. Typically, each diffusion component (i.e. diffusion map dimension) highlights the heterogeneity of a different cell population.
#### Visualization
For visualization purposes, it is standard practice to use non‐linear dimensionality reduction methods (Fig [4](https://doi.org/10.15252/msb.20188746#msb188746-fig-0004)). The most common dimensionality reduction method for scRNA‐seq visualization is the t‐distributed stochastic neighbour embedding (t‐SNE; van der Maaten & Hinton, [2008](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0082)). t‐SNE dimensions focus on capturing local similarity at the expense of global structure. Thus, these visualizations may exaggerate differences between cell populations and overlook potential connections between these populations. A further difficulty is the choice of its perplexity parameter, as t‐SNE graphs may show strongly different numbers of clusters depending on its value (Wattenberg _et al_ , [2016](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0140)). Common alternatives to t‐SNE are the Uniform Approximation and Projection method (UMAP; preprint: McInnes & Healy, [2018](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0090)) or graph‐based tools such as SPRING (Weinreb _et al_ , [2018](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0144)). UMAP and SPRING's force‐directed layout algorithm ForceAtlas2 arguably represent the best approximation of the underlying topology (Wolf _et al_ , [2019](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0147), Supplemental Note 4). What sets UMAP apart in this comparison is its speed and ability to scale to large numbers of cells (Becht _et al_ , [2018](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0011)). Thus, in the absence of particular biological questions, we regard UMAP as best practice for exploratory data visualization. Moreover, UMAP can also summarize data in more than two dimensions. While we are not aware of any applications of UMAP for data summarization, it may prove a suitable alternative to PCA.
An alternative to classical visualization on the cell level is partition‐based graph abstraction (PAGA; Wolf _et al_ , [2019](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0147)). This tool has been shown to adequately approximate the topology of the data while coarse‐graining the visualization using clusters. In combination with any of the above visualization methods, PAGA produces coarse‐grained visualizations, which can simplify the interpretation of single‐cell data especially with large numbers of cells.
##### Pitfalls & recommendations:
•
We recommend selecting between 1,000 and 5,000 highly variable genes depending on dataset complexity.
•
Feature selection methods that use gene expression means and variances cannot be used when gene expression values have been normalized to zero mean and unit variance, or when residuals from model fitting are used as normalized expression values. Thus, one must consider what pre‐processing to perform before selecting HVGs.
•
Dimensionality reduction methods should be considered separately for summarization and visualization.
•
We recommend UMAP for exploratory visualization; PCA for general purpose summarization; and diffusion maps as an alternative to PCA for trajectory inference summarization.
•
PAGA with UMAP is a suitable alternative to visualize particularly complex datasets.
### Stages of pre‐processed data
While we have outlined common pre‐processing steps in scRNA‐seq as a sequential pipeline above, downstream analyses often prefer to take different levels of pre‐processed data, and it is recommended to adapt pre‐processing depending on the downstream application. To clarify this situation to a new user, we delineated pre‐processing into five stages of data processing: (i) raw data, (ii) normalized data, (iii) corrected data, (iv) feature‐selected data, and (v) dimensionality‐reduced data. These stages of data processing are grouped into three pre‐processing layers: measured data, corrected data, and reduced data. Cell and gene QC should always be performed and is therefore omitted from this characterization. While the order of the layers represent the typical workflow in scRNA‐seq analysis, it is also possible to skip layers or have slight alterations in the order of processing stages. For example, data correction may not be necessary for single batch datasets. In Table [1](https://doi.org/10.15252/msb.20188746#msb188746-tbl-0001), we summarize the appropriate downstream applications for each layer of pre‐processed data.
Table 1. Stages of data processing and appropriate downstream applications
Pre‐processing layer| Stage of data processing| Appropriate applications  
---|---|---  
Measured| 1) Raw| Statistical testing (Differential expression: marker genes, genes over condition, genes over time)  
2) Normalized (+ log transformed)  
Corrected| 3.1) Corrected (technical correction)| Visual comparison of data (plotting)  
3.2) Corrected (biological correction)| Pre‐processing for trajectory inference  
Reduced| 4) Feature selected| Visualization, trajectory inference  
5) Dimensionality reduced (summarized)| Visualization, clustering, KNN graph inference, trajectory inference  
[Open in viewer](https://doi.org/10.15252/msb.20188746#msb188746-tbl-0001)
The stages of pre‐processing in Table [1](https://doi.org/10.15252/msb.20188746#msb188746-tbl-0001) are divided into three groups: measured data, corrected data and reduced data. We define _measured data_ as raw data and processed data that retain the structure of zeros. By scaling count data with a cell‐specific factor, global scaling normalization methods retain zero expression values even after log(x+1)‐transformation. In contrast, correcting data for unwanted variability replaces zero expression values. The _corrected data_ layer represents the “cleanest” version of the data, which is the closest approximation of the underlying biological signal. We call the final pre‐processing layer _reduced data_. This data layer emphasizes dominant aspects of the data, which can be described using a reduced set of features.
The aforementioned characteristics determine the suitability of the pre‐processed data for particular downstream applications. As the final pre‐processing stage, reduced data would be the natural candidate for a broadly applicable data layer. However, testing for differential expression is only biologically interpretable in gene space, which is not (fully) represented in reduced data. The strength of reduced data lies in the summarization of the biology and the reduction of noise, which can mask biological signals. Thus, reduced data are used for exploratory methods that require data summaries (visualization, neighbourhood graph inference, clustering) and for computationally complex downstream analysis tools (trajectory inference). Indeed, many trajectory inference methods incorporate dimensionality reduction in the tools themselves.
The expression profiles of individual genes can only be compared in gene space, which is captured in measured and corrected data. Comparison of expression profiles can be performed visually and statistically. We argue that visual and statistical comparison should be performed on different data layers. For visual inspection of gene expression, corrected data are most appropriate. Should raw data be presented for visual comparison, the user is required to inherently understand the biases in the data in order to interpret the results. Corrected data facilitate this interpretation. However, one should consider corrected data for technical and biological covariates separately here. While correction for biological covariates may increase the strength of a particular biological signal, it produces a less accurate representation of the underlying biology and will mask other signals that may be relevant. Thus, biologically corrected data are appropriate mainly for analysis tools that focus on particular biological processes such as trajectory inference methods.
Statistical comparison of gene expression is most appropriate on the measured data layer. No perfect data correction method exists for denoising, batch correction or correcting for other sources of variation. Thus, data correction methods inevitably over‐ or under‐correct the data and therefore alter the variance of at least some gene expression profiles in an unintended way. Statistical tests of gene expression rely on assessing the background variance as a null model for noise in the data. As data correction tends to reduce background variation (Fig [EV2](https://doi.org/10.15252/msb.20188746#msb188746-fig-0002ev)), genes whose background variation is over‐corrected by data correction methods will be more likely assessed as significantly differentially expressed. Furthermore, certain data correction methods (e.g. ComBat) interpret expression signals that do not conform to the experimental design as noise, which is subsequently removed from the data. In addition to an underestimation of the noise, this optimization of the experimental design signal can lead to an overestimation of the effect size. In the light of these considerations, using measured data as input, as opposed to corrected data, constitutes a more conservative approach to differential testing. With measured data, technical covariates can and should be taken into consideration in the differential testing model.
[![image](https://doi.org/cms/10.15252/msb.20188746/asset/d6585605-fb7f-4bea-8276-71309cfcd3fb/assets/graphic/msb188746-fig-0002ev-m.png)](https://doi.org/10.15252/msb.20188746#msb188746-fig-0002ev)
Figure EV2. Change in coefficient of variation (CoV) of gene expression data upon batch correction and denoising
Negative values represent a reduction in CoV upon data correction. The top row shows CoV changes upon ComBat batch correction for (A) mouse intestinal epithelium (mIE) and (B) mouse embryonic stem cell (mESC) data. The lower row depicts CoV changes upon DCA denoising for (C) mIE and (D) mESC data. mIE data were obtained from Haber _et al_ ([2017](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0046)) and mESC from Klein _et al_ ([2015](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0065)).
  * [Download figure](https://doi.org/cms/asset/d6585605-fb7f-4bea-8276-71309cfcd3fb/msb188746-fig-0002ev-m.png)
  * [Download PowerPoint](https://doi.org/action/downloadFigures?doi=10.15252%2Fmsb.20188746&id=msb188746-fig-0002ev)

Click here to expand this figure.
[Open in viewer](https://doi.org/10.15252/msb.20188746#msb188746-fig-0002ev)
The above view is supported by a recent comparison of scRNA‐seq differential expression methods, which uses only raw and normalized data as input (Soneson & Robinson, [2018](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0123)). The normalized data used in this study revolve only around global scaling methods. However, many currently available non‐linear normalization methods blur the line between normalization and data correction (see “Normalization” section). Such normalized data may no longer be appropriate as input for differential testing.
#### Pitfalls & recommendations:
•
Use measured data for statistical testing, corrected data for visual comparison of data and reduced data for other downstream analysis based on finding the underlying biological data manifold.
#### Downstream analysis
After pre‐processing, methods that we call _downstream analysis_ are used to extract biological insights and describe the underlying biological system. These descriptions are obtained by fitting interpretable models to the data. Examples of such models are groups of cells with similar gene expression profiles representing cell‐type clusters; small changes in gene expression between similar cells denoting continuous (differentiation) trajectories; or genes with correlated expression profiles indicating co‐regulation.
Downstream analysis can be divided into cell‐ and gene‐level approaches as shown in Fig [5](https://doi.org/10.15252/msb.20188746#msb188746-fig-0005). Cell‐level analysis typically focuses on the description of two structures: clusters and trajectories. These structures can in turn be analysed on the cell and the gene level leading to cluster analysis and trajectory analysis methods. Broadly, cluster analysis methods attempt to explain the heterogeneity in the data based on a categorization of cells into groups. In contrast, in trajectory analysis the data are regarded as a snapshot of a dynamic process. Trajectory analysis methods investigate this underlying process.
[![image](https://doi.org/cms/10.15252/msb.20188746/asset/5593cfbd-38ec-43e5-96e8-9beaa495c0c2/assets/graphic/msb188746-fig-0005-m.png)](https://doi.org/10.15252/msb.20188746#msb188746-fig-0005)
Figure 5. Overview of downstream analysis methods
Methods are divided into cell‐ and gene‐level analysis. Cell‐level analysis approaches are again subdivided into cluster and trajectory analysis branches, which include also gene‐level analysis methods. All methods with a blue background are gene‐level approaches.
  * [Download figure](https://doi.org/cms/asset/5593cfbd-38ec-43e5-96e8-9beaa495c0c2/msb188746-fig-0005-m.png)
  * [Download PowerPoint](https://doi.org/action/downloadFigures?doi=10.15252%2Fmsb.20188746&id=msb188746-fig-0005)

[Open in viewer](https://doi.org/10.15252/msb.20188746#msb188746-fig-0005)
Here, we describe cell‐ and gene‐level cluster and trajectory analysis tools before detailing gene‐level analyses that are performed independently of these cellular structures.
### Cluster analysis
#### Clustering
Organizing cells into clusters is typically the first intermediate result of any single‐cell analysis. Clusters allow us to infer the identity of member cells. Clusters are obtained by grouping cells based on the similarity of their gene expression profiles. Expression profile similarity is determined via distance metrics, which often take dimensionality‐reduced representations as input. A common example of similarity scoring is Euclidean distances that are calculated on the PC‐reduced expression space. Two approaches exist to generate cell clusters from these similarity scores: clustering algorithms and community detection methods.
Clustering is a classical unsupervised machine learning problem, based directly on a distance matrix. Cells are assigned to clusters by minimizing intracluster distances or finding dense regions in the reduced expression space. The popular _k_ ‐means clustering algorithm divides cells into _k_ clusters by determining cluster centroids and assigning cells to the nearest cluster centroid. Centroid positions are iteratively optimized (MacQueen, [1967](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0084)). This approach requires an input of the number of clusters expected, which is usually unknown and must be calibrated heuristically. Applications of k‐means to single‐cell data vary in the distance metrics used. Alternatives to standard Euclidean distances include cosine similarity (Haghverdi _et al_ , [2018](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0049)), correlation‐based distance metrics (Kim _et al_ , [2018](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0062)) or the SIMLR method, which learns a distance metric for each dataset using Gaussian kernels (Wang _et al_ , [2017](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0139)). A recent comparison has suggested that correlation‐based distances may outperform other distance metrics when used with k‐means or as the basis for Gaussian kernels (Kim _et al_ , [2018](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0062)).
Community detection methods are graph‐partitioning algorithms and thus rely on a graph representation of single‐cell data. This graph representation is obtained using a _K_ ‐Nearest Neighbour approach (KNN graph). Cells are represented as nodes in the graph. Each cell is connected to its _K_ most similar cells, which are typically obtained using Euclidean distances on the PC‐reduced expression space. Depending on the size of the dataset, _K_ is commonly set to be between 5 and 100 nearest neighbours. The resulting graph captures the underlying topology of the expression data (Wolf _et al_ , [2019](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0147)). Densely sampled regions of expression space are represented as densely connected regions of the graph. These dense regions are detected using community detection methods. Community detection is often faster than clustering as only neighbouring cell pairs have to be considered as belonging to the same cluster. This approach thus greatly reduces the search space for possible clusters.
After the pioneering PhenoGraph method (Levine _et al_ , [2015](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0070)), the standard approach to clustering single‐cell datasets has become multi‐resolution modularity optimization (Newman & Girvan, [2004](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0094); Reichardt & Bornholdt, [2006](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0109)) as implemented in the Louvain algorithm (Blondel _et al_ , [2008](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0014)) on single‐cell KNN graphs. This method is the default clustering method implemented in the Scanpy and Seurat single‐cell analysis platforms. It has been shown to outperform other clustering methods for single‐cell RNA‐seq data (Duò _et al_ , [2018](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0034); Freytag _et al_ , [2018](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0039)), and flow and mass cytometry data (Weber & Robinson, [2016](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0142)). Conceptually, the Louvain algorithm detects communities as groups of cells that have more links between them than expected from the number of links the cells have in total. The optimized modularity function includes a resolution parameter, which allows the user to determine the scale of the cluster partition. By subsetting the KNN graph, it is also possible to subcluster only particular clusters. Such subclustering can allow the user to identify cell states within cell‐type clusters (Wagner _et al_ , [2016](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0137)), but may also lead to patterns that arise only from noise in the data.
##### Pitfalls & recommendations:
•
We recommend clustering by Louvain community detection on a single‐cell KNN graph.
•
Clustering does not have to be performed at a single resolution. Subclustering particular cell clusters is a valid approach to focus on more detailed substructures in a dataset.
#### Cluster annotation
On a gene level, clustered data are analysed by finding the gene signatures of each cluster. These so‐called _marker genes_ characterize the cluster and are used to annotate it with a meaningful biological label. This label represents the identity of cells within the cluster. As any clustering algorithm will produce a partition of the data, the validity of the identified clusters can only be determined by successful annotation of the represented biology.
While it may be tempting to assume that the clusters detected in single‐cell data represent cell types, there are several axes of variation that determine cellular identity (Wagner _et al_ , [2016](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0137); Clevers _et al_ , [2017](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0025)). Firstly, it is not always clear what constitutes a cell type. For example, while “T cells” may be a satisfactory label of a cell type to some, others may look for T‐cell subtypes within a dataset and distinguish between CD4+ and CD8+ T cells (Wagner _et al_ , [2016](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0137); Clevers _et al_ , [2017](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0025)). Furthermore, cells of the same cell type in different states may be detected in separate clusters. For the above reasons, it is best to use the term “cell identities” rather than “cell types”. Before clustering and annotating clusters, the user must decide which level of annotation detail, and thus which cluster resolution, is of interest.
Identifying and annotating clusters relies on using external sources of information describing the expected expression profiles of individual cell identities. Thanks to recent and ongoing efforts such as the mouse brain atlas (Zeisel _et al_ , [2018](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0153)) or the Human Cell Atlas (Regev _et al_ , [2017](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0107)), reference databases are increasingly becoming available. These databases greatly facilitate cell identity annotation. In the absence of a relevant reference database, cell identities can be annotated by comparing data‐derived marker genes with marker genes from the literature (see case study on project github) or by directly visualizing the expression values of literature‐derived marker genes (Fig [6](https://doi.org/10.15252/msb.20188746#msb188746-fig-0006)B). It should be noted that the latter method constrains the user to the classical understanding of cell types derived from bulk expression studies, rather than cell identities. Furthermore, it has been shown that commonly used cell surface markers are limited in their ability to define cell identities (Tabula Muris Consortium _et al_ , [2018](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0127)).
[![image](https://doi.org/cms/10.15252/msb.20188746/asset/62f76211-6a38-42ed-b151-b38d246b897e/assets/graphic/msb188746-fig-0006-m.png)](https://doi.org/10.15252/msb.20188746#msb188746-fig-0006)
Figure 6. Cluster analysis results of mouse intestinal epithelium dataset from Haber _et al_ ([2017](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0046))
(A) Annotated cell‐identity clusters found by Louvain clustering visualized in a UMAP representation. (B) Cell‐identity marker expression to identify stem cells (Slc12a2), enterocytes (Arg2), goblet cells (Tff3) and Paneth cells (Defa24). Corrected expression levels are visualized from low expression (grey) to high expression (red). Marker genes may be expressed also in other cell‐identity populations as shown for goblet and Paneth cells. (C) Cell‐identity composition heat maps of proximal (upper) and distal (lower) intestinal epithelium regions. High relative cell density is shown as dark red.
  * [Download figure](https://doi.org/cms/asset/62f76211-6a38-42ed-b151-b38d246b897e/msb188746-fig-0006-m.png)
  * [Download PowerPoint](https://doi.org/action/downloadFigures?doi=10.15252%2Fmsb.20188746&id=msb188746-fig-0006)

[Open in viewer](https://doi.org/10.15252/msb.20188746#msb188746-fig-0006)
There are two ways to use reference database information to annotate clusters: using data‐derived marker genes or using full gene expression profiles. Marker gene sets can be found by applying differential expression (DE) testing between two groups: the cells in one cluster and all other cells in the dataset (see “[Differential expression testing](https://doi.org/10.15252/msb.20188746#sec-2-12-1)”). Typically, we focus on genes that are up‐regulated in the cluster of interest. As marker genes are expected to have strong differential expression effects, simple statistical tests such as the Wilcoxon rank‐sum test or the _t_ ‐test are often used to rank genes by their difference in expression between these two groups. The top‐ranked genes from the respective test statistic are regarded as marker genes. Clusters can be annotated by comparing marker genes from the dataset and marker genes from reference dataset via enrichment tests, the Jaccard index or other overlap statistics. Reference webtools such as (Zeisel _et al_ , [2018](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0153)) or (Saunders _et al_ , [2018](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0118)) allow users to visualize the expression of dataset marker genes in the reference dataset to facilitate cell‐identity annotation.
Two aspects should be noted when detecting marker genes. Firstly, the _P_ ‐values obtained for marker genes are based on the assumption that the obtained cell clusters represent the biological ground truth. If one considers that there is uncertainty in the cluster assignment, the relationship between cluster assignment and marker gene detection must be accounted for in the statistical test. This relationship arises as both clusters and marker genes are typically determined based on the same gene expression data. The null hypothesis implicit in DE tests is that genes have the same distribution of expression values between the two groups. Yet, as the two groups are defined by the output of a clustering method in marker gene detection, there are differences in their gene expression profiles by design. We thus find significant marker genes even when clustering random data generated by splatter (Zappia _et al,_ [2017](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0151)) (see [Appendix Supplementary Text S3](https://doi.org/10.15252/msb.20188746#msb188746-sup-0001)). To obtain an appropriate measure of significance in clustered data, one can use a permutation test to account for the clustering step. This test is elaborated on in [Appendix Supplementary Text S3](https://doi.org/10.15252/msb.20188746#msb188746-sup-0001). A recent differential expression tool also specifically addresses this issue (preprint: Zhang _et al_ , [2018](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0155)). With the current set‐up, the _P_ ‐values are often inflated, which can lead to an overestimation of the number of marker genes. However, the ranking of genes based on _P_ ‐values is unaffected. Assuming the clustering is biologically meaningful, the top‐ranked marker genes will still be the best marker gene candidates. In the first instance, we can loosely verify marker genes via visual inspection. We emphasize that inflated _P_ ‐values occur specifically when defining cell‐identity clusters via an unsupervised clustering approach. When instead determining cell‐identity clusters via the expression of individual genes, the _P_ ‐values can be interpreted as expected for all other genes. This univariate approach to cluster annotation, although common, is however not recommended outside specific cases (e.g. insulin in beta cells or haemoglobin in erythrocytes). Secondly, marker genes differentiate a cluster from others in the dataset and are thus dependent not only on the cell cluster, but also on the dataset composition. If the dataset composition does not accurately represent background gene expression, detected marker genes will be biased towards what is missing. This aspect must be considered especially when computing marker genes for datasets with low cellular diversity.
Recently, automated cluster annotation has become available. By directly comparing the gene expression profiles of annotated reference clusters to individual cells, tools such as scmap (Kiselev _et al_ , [2018b](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0064)) or Garnett (preprint: Pliner _et al_ , [2019](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0104)) can transfer annotations between the reference and the dataset. Thus, these methods can perform annotation and cluster assignment simultaneously, without the need for a data‐driven clustering step. As cell type and state compositions differ between experimental conditions (Segerstolpe _et al_ , [2016](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0121); Tanay & Regev, [2017](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0128)), clustering based on reference data should not replace the data‐driven approach.
The iteration of clustering, cluster annotation, re‐ or subclustering and re‐annotation can be time‐consuming. Automated cluster annotation methods offer a vast speedup of this process. However, there are benefits and limitations to automated and manual approaches that make it difficult to recommend one over the other. Increases in speed go together with compromises in flexibility. As mentioned above, reference atlases will not contain exactly the same cell identities as the dataset under investigation. Thus, one should not forgo marker gene calculation for manual annotation. Especially for large datasets that contain many clusters, the current best practice is a combination of both approaches. In the interest of speed, automated cell‐identity annotation can be used to coarsely label cells and identify where subclustering may be needed. Subsequently, marker genes should be calculated for the dataset clusters and compared to known marker gene sets from the reference dataset or literature. For smaller datasets and datasets that lack reference atlases, manual annotation will suffice.
##### Pitfalls & recommendations:
•
Do not use marker gene _P_ ‐values to validate a cell‐identity cluster, especially when the detected marker genes do not help to annotate the community. _P_ ‐values may be inflated.
•
Note that marker genes for the same cell‐identity cluster may differ between datasets purely due to dataset cell type and state compositions.
•
If relevant reference atlases exist, we recommend using automated cluster annotation combined with data‐derived marker‐gene‐based manual annotation to annotate clusters.
#### Compositional analysis
At the cell level, we can analyse clustered data in terms of its compositional structure. Compositional data analysis revolves around the proportions of cells that fall into each cell‐identity cluster. These proportions can change in response to disease. For example, salmonella infection has been shown to increase the proportion of enterocytes in the mouse intestinal epithelium (Haber _et al_ , [2017](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0046)).
Investigating compositional changes in single‐cell data requires sufficient cell numbers to robustly assess cell‐identity cluster proportions, and sufficient sample numbers to evaluate expected background variation in cell‐identity cluster compositions. As appropriate datasets have only recently become available, dedicated tools are yet to be developed. In the mouse study mentioned above, cell‐identity counts were modelled using a Poisson process, including the condition as a covariate, and the total number of cells detected as an offset. Here, a statistical test can be performed over the regression coefficient to assess whether a particular cell identity has significantly changed in frequency. However, tests over other cell identities in the same dataset are not independent of each other. If the proportion of one cell‐identity cluster changes, the proportions of all others must have changed as well. Thus, one cannot assess whether the overall composition has significantly changed using this model. In the absence of dedicated tools, visual comparison of compositional data can be informative of changes in compositions between samples (Fig [6](https://doi.org/10.15252/msb.20188746#msb188746-fig-0006)C). Future developments in this field will likely borrow from the mass cytometry (e.g. Tibshirani _et al_ , [2002](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0131); Arvaniti & Claassen, [2017](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0008); Lun _et al_ , [2017](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0080); Weber _et al_ , [2018](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0143)) or the microbiome literature (Gloor _et al_ , [2017](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0043)), where compositional data analysis has received more attention.
##### Pitfalls & recommendations:
•
Consider that statistical tests over changes in the proportion of a cell‐identity cluster between samples are dependent on one another.
### Trajectory analysis
#### Trajectory inference
Cellular diversity cannot sufficiently be described by a discrete classification system such as clustering. The biological processes that drive the development of the observed heterogeneity are continuous processes (Tanay & Regev, [2017](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0128)). Thus, in order to capture transitions between cell identities, branching differentiation processes, or gradual, unsynchronized changes in biological function, we require dynamic models of gene expression. This class of methods is known as trajectory inference (TI).
Trajectory inference methods interpret single‐cell data as a snapshot of a continuous process. This process is reconstructed by finding paths through cellular space that minimize transcriptional changes between neighbouring cells (Fig [7](https://doi.org/10.15252/msb.20188746#msb188746-fig-0007)A and B). The ordering of cells along these paths is described by a _pseudotime_ variable. While this variable is related to transcriptional distances from a root cell, it is often interpreted as a proxy for developmental time (Moignard _et al_ , [2015](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0091); Haghverdi _et al_ , [2016](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0048); Fischer _et al_ , [2018](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0038); Griffiths _et al_ , [2018](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0044)).
[![image](https://doi.org/cms/10.15252/msb.20188746/asset/13e5de7d-250b-4715-8ab8-5dcb3eea526d/assets/graphic/msb188746-fig-0007-m.png)](https://doi.org/10.15252/msb.20188746#msb188746-fig-0007)
Figure 7. Trajectory analysis and graph abstraction of mouse intestinal epithelium data from Haber _et al_ ([2017](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0046))
(A) Distal and proximal enterocyte differentiation trajectories inferred by Slingshot. The Distal lineage is shown coloured by pseudotime from red to blue. Other cells in the dataset are grey. (B) Slingshot trajectories over clusters in PCA space. Clusters are abbreviated as follows: EP—enterocyte progenitors; Imm. Ent.—immature enterocytes; Mat. Ent.—mature enterocytes; Prox.—proximal; Dist.—distal. (C) Density over pseudotime for the distal enterocyte trajectory from Fig [7](https://doi.org/10.15252/msb.20188746#msb188746-fig-0007)A. Colours represent the dominant cluster labels in each pseudotime bin. (D) Abstracted graph representation of the dataset projected onto a UMAP representation. Clusters are shown as coloured nodes. Clusters that appear in other trajectories are labelled for comparison. “TA” denotes transit amplifying cells. (E) Gene expression dynamics over pseudotime in a general enterocyte trajectory using the “GAM” R library.
  * [Download figure](https://doi.org/cms/asset/13e5de7d-250b-4715-8ab8-5dcb3eea526d/msb188746-fig-0007-m.png)
  * [Download PowerPoint](https://doi.org/action/downloadFigures?doi=10.15252%2Fmsb.20188746&id=msb188746-fig-0007)

[Open in viewer](https://doi.org/10.15252/msb.20188746#msb188746-fig-0007)
Since Monocle (Trapnell _et al_ , [2014](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0132)) and Wanderlust (Bendall _et al_ , [2014](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0012)) established the TI field, the number of available methods has exploded. Currently available TI methods differ in the complexity of the paths that are modelled. Models range from simple linear or bifurcating trajectories, to complex graphs, trees, or multifurcating trajectories. In a recent comprehensive comparison of TI methods (Saelens _et al_ , [2018](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0116)), it was concluded that no individual method performs optimally for all types of trajectories. Instead, TI methods should be selected based on the complexity of the expected trajectory. The comparison revealed that Slingshot (Street _et al_ , [2018](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0125)) outperformed other methods for simple trajectories that range from linear to bi‐ and multifurcating models. If more complex trajectories are expected, PAGA (Wolf _et al_ , [2019](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0147)) was recommended by the authors. If the exact trajectory model is known, one can alternatively use more specialized methods to improve performance (Saelens _et al_ , [2018](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0116)). Generally, any inferred trajectory should be confirmed with an alternative method to avoid method bias.
In a typical workflow, TI methods are applied to reduced data or to corrected data when there is an inbuilt dimensionality reduction step. As multiple biological processes are typically occurring simultaneously within cells, it may be useful to regress out the biological effects of other processes to isolate the expected trajectory. For example, T cells may be undergoing cell cycle transitions during maturation (Buettner _et al_ , [2015](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0018)). Furthermore, as several top‐performing TI methods rely on clustered data, TI is typically performed after clustering. Clusters in inferred trajectories may represent stable or metastable states (see “[Metastable states](https://doi.org/10.15252/msb.20188746#sec-2-10-3)”; Fig [7](https://doi.org/10.15252/msb.20188746#msb188746-fig-0007)B and C). Subsequently, RNA velocities can be overlayed onto the trajectory to add directionality (La Manno _et al_ , [2018](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0068)).
Inferred trajectories do not have to represent biological processes. In the first instance, these only denote transcriptional similarity. Few TI methods include an evaluation of uncertainty in their model (Griffiths _et al_ , [2018](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0044)). Thus, further information is needed to validate whether a biological process was indeed captured. This information can come in the form of perturbation experiments, inferred regulatory gene dynamics, and support from RNA velocity.
##### Pitfalls & recommendations:
•
We recommend using the Saelens _et al_ ([2018](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0116)) review as a guide.
•
Inferred trajectories do not have to represent a biological process. Further sources of evidence should be collected to interpret a trajectory.
#### Gene expression dynamics
One approach to garner support that an inferred trajectory is not the result of fitting transcriptional noise is to analyse the trajectory on the gene level. Genes that vary smoothly across pseudotime characterize the trajectory and can be used to identify the underlying biological process. Furthermore, this group of trajectory‐associated genes is expected to contain genes that regulate the modelled process. Regulator genes help us understand how and why biological processes are triggered and represent potential drug targets (Gashaw _et al_ , [2012](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0041)).
While early approaches to find trajectory‐associated genes involved DE testing between cell clusters along a trajectory (Haghverdi _et al_ , [2016](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0048); Alpert _et al_ , [2018](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0002)), we now detect genes that vary across a trajectory by regressing gene expression against pseudotime. In order to enforce smooth variation of expression along this covariate, pseudotime is smoothed by fitting a spline or via an additional local regression step (e.g. loess). Regression frameworks differ in their noise model assumptions and the class of function used to describe the expression as a function of pseudotime. Potential regulatory genes are obtained by performing model selection for the genes’ dependence on pseudotime. This DE test over pseudotime is confounded by the trajectory inference method in the same way that DE testing between clusters is confounded by the clustering method (see “[Cluster annotation](https://doi.org/10.15252/msb.20188746#sec-2-9-2)” section). Thus, _P_ ‐values obtained in this set‐up should not be regarded as an evaluation of significance.
Currently few dedicated gene temporal dynamics tools exist. BEAM is a tool integrated into the Monocle TI pipeline (Qiu _et al_ , [2017a](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0105)), which allows for detection of branch‐specific gene dynamics. Outside of this pipeline, users can opt for LineagePulse (), which considers dropout noise but is still in development, or write their own testing framework using the limma package (Ritchie _et al_ , [2015](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0112)) or standard R libraries. An example of this can be found in the online Slingshot tutorial (Street _et al_ , [2018](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0125)) and in Fig [7](https://doi.org/10.15252/msb.20188746#msb188746-fig-0007)E.
Given the few available tools, a best practice for investigating gene temporal dynamics cannot yet be determined. Exploratory investigation of gene dynamics is surely possible using all above methods. In future, Gaussian processes may provide a natural model to investigate gene temporal dynamics. Furthermore, testing for regulatory modules rather than individual genes would likely improve the signal‐to‐noise ratio and facilitate the biological interpretation.
#### Metastable states
Cell‐level analysis of trajectories investigates cellular densities across pseudotime. Assuming that cells were sampled in an unbiased manner, dense regions along a trajectory indicate preferred transcriptomic states. When interpreting the trajectory as a temporal process, these dense regions may represent metastable states in, for example, development (Haghverdi _et al_ , [2016](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0048)). We can find these metastable states by plotting histograms of the pseudotime coordinate (Fig [7](https://doi.org/10.15252/msb.20188746#msb188746-fig-0007)C).
### Cell‐level analysis unification
Clustering and trajectory inference represent two distinct views of single‐cell data. These two views can be reconciled in coarse‐grained graph representations. By representing single‐cell clusters as nodes, and trajectories between the clusters as edges, one can represent both the static and dynamic nature of the data. This unification was proposed by the partition‐based graph abstraction tool (PAGA; Fig [7](https://doi.org/10.15252/msb.20188746#msb188746-fig-0007)D; Wolf _et al_ , [2019](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0147)). Using a statistical model for cell cluster interactions, PAGA places an edge between cluster nodes whose cells are more similar than expected. PAGA has been favourably compared to other TI methods in a recent review (Saelens _et al_ , [2018](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0116)). It was the only reviewed method able to cope with disconnected topologies and complex graphs containing cycles. This feature makes PAGA a helpful tool to visualize the topology of the entire dataset also for exploratory analysis.
### Gene‐level analysis
While we have so far focused on gene‐level analysis methods that characterize cellular structures, gene‐level analysis of single‐cell data has a broader scope. Differential expression testing, gene set analyses and gene regulatory network inference directly investigate molecular signals in the data. Rather than describing the cellular heterogeneity, these approaches use this heterogeneity as context in which gene expression is to be understood.
#### Differential expression testing
A common question asked of expression data is whether any genes are differentially expressed between two experimental conditions. DE testing is a well‐documented problem that originates from bulk gene expression analysis (Scholtens & von Heydebreck, [2005](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0119)). An advantage over bulk differential testing is that we can account for cellular heterogeneity in the single‐cell setting by performing tests within cell‐identity clusters. This set‐up tells us how individual cell identities react transcriptionally under particular experimental conditions (Kang _et al_ , [2018](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0060)).
Although designed to answer the same question, bulk and single‐cell DE tools differ methodologically. While bulk methods were developed to accurately estimate gene variance from few samples, single‐cell data do not present this problem. On the other hand, single‐cell data contain unique technical noise artefacts such as dropout, and high cell‐to‐cell variability (Hicks _et al_ , [2017](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0051); Vallejos _et al_ , [2017](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0133)). These artefacts are taken into account in methods designed specifically for single‐cell data (Kharchenko _et al_ , [2014](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0061); Finak _et al_ , [2015](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0037)). Yet, a recent, large‐scale comparison study of DE analysis has suggested that bulk DE testing packages perform comparably to the best‐performing single‐cell tools (Soneson & Robinson, [2018](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0123)). Furthermore, when bulk tools are adapted to model single‐cell data via introducing gene weights into the tests, these tools have been suggested to outperform their single‐cell counterparts (Van den Berge _et al_ , [2018](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0134)). According to this comparison, the top‐performing DE analysis tools are DESeq2 (Love _et al_ , [2014](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0077)) and EdgeR (Robinson _et al_ , [2010](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0113)) in combination with weights estimated by ZINB‐wave (Risso _et al_ , [2018](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0111)). Independent comparison studies that include weighted bulk DE testing methods are required to confirm these results.
The improved performance of weighted bulk DE testing comes at the cost of computational efficiency. Given the trend of increasing cell numbers in single‐cell experiments, algorithm runtime is becoming an increasingly important consideration in method choice. Thus, the single‐cell tool MAST (Finak _et al_ , [2015](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0037)) represents a potent alternative to weighted bulk DE tools. MAST uses a hurdle model to account for dropout while modelling changes in gene expression dependent upon condition and technical covariates. It was the best‐performing single‐cell DE testing method in the aforementioned study (Soneson & Robinson, [2018](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0123)), and outperformed bulk and single‐cell methods in a small‐scale comparison on a single dataset (Vieth _et al_ , [2017](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0136)). While MAST has a 10‐fold to 100‐fold faster runtime than weighted bulk methods (Van den Berge _et al_ , [2018](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0134)), a further 10‐fold speedup can be achieved using limma–voom (Law _et al_ , [2014](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0069)). Although limma is a bulk DE testing method, limma–voom was shown to achieve comparable performance to MAST.
As uncorrected, measured data should be used for DE testing, accounting for confounding factors is crucial to robust estimation of differentially expressed genes. While DE testing tools typically allow the user the flexibility to incorporate confounders, users must be vigilant which variables are added to the model. For example, in most single‐cell experimental set‐ups the sample and condition covariates are confounded, since it is rarely possible to obtain a single sample under multiple conditions. If we incorporate both the sample and condition covariates into the model, the variability associated with these covariates can no longer unambiguously be assigned. Thus, when testing over condition, we cannot include the sample covariate in the model in the given form. When correcting for multiple categorical batch covariates, it becomes increasingly difficult to visually spot confounding groups of covariates. In this situation, it is helpful to test whether the model design matrix is full rank. Even when design matrices are not full rank, DE testing tools will often adapt the matrix and run without outputting a warning. This will not deliver the intended results.
In the scenario we describe here, the condition covariate is determined in the experimental set‐up. Thus, a DE test over this covariate (within the same cluster) is independent of the clustering procedure. This set‐up distinguishes DE testing over conditions and DE testing over clusters. Obtained _P_ ‐values for DE tests over conditions represent the expected measures of significance and must be corrected for multiple testing. To reduce the multiple testing burden, transcripts that may not be of interest can be excluded from the dataset. While pseudogenes or non‐coding RNAs can be informative (An _et al_ , [2017](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0003)), they are often ignored in the analysis.
##### Pitfalls & recommendations:
•
DE testing should not be performed on corrected data (denoised, batch corrected, etc.), but instead on measured data with technical covariates included in the model.
•
Users should not rely on DE testing tools to correct models with confounded covariates. Model specification should be performed carefully ensuring a full‐rank design matrix.
•
We recommend using MAST or limma for DE testing.
#### Gene set analysis
Gene‐level analysis methods often produce long lists of candidate genes that are difficult to interpret. For example, thousands of genes may be differentially expressed between treated and control cells. We can facilitate the interpretation of these results by grouping the genes into sets based on shared characteristics and testing whether these characteristics are overrepresented in the candidate gene list.
Gene set information can be found in curated label databases for various applications. To interpret DE results, we typically group genes based on involvement in common biological processes. Biological process labels are stored in databases such as MSigDB (Liberzon _et al_ , [2011](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0073)), the Gene Ontology (Ashburner _et al_ , [2000](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0009); The Gene Ontology Consortium, [2017](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0130)), or the pathway databases KEGG (Kanehisa _et al_ , [2017](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0059)) and Reactome (Fabregat _et al_ , [2018](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0036)). Enrichment of annotations on the gene list can be tested using a vast array of tools, which are reviewed and compared in Huang _et al_ ([2009](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0053)) and Tarca _et al_ ([2013](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0129)).
A recent development in the single‐cell analysis field is the use of paired gene labels to perform ligand–receptor analysis. Here, interaction between cell clusters is inferred from the expression of receptors and their cognate ligands. Ligand–receptor pair labels can be obtained from the recent CellPhoneDB (Vento‐Tormo _et al_ , [2018](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0135)) and used to interpret the highly expressed genes across clusters using statistical models (Zepp _et al_ , [2017](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0154); Zhou _et al_ , [2017](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0157); Cohen _et al_ , [2018](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0026); Vento‐Tormo _et al_ , [2018](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0135)).
#### Gene regulatory networks
Genes do not function independently. Instead, the expression level of a gene is determined by a complex interplay of regulatory interactions with other genes and small molecules. Uncovering these regulatory interactions is the goal of gene regulatory network (GRN) inference methods.
Gene regulatory network inference is performed based on measurements of gene co‐expression such as correlation, mutual information, or via regression models (Chen & Mar, [2018](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0023)). If two genes show a co‐expression signal even when all other genes are taken into account as potential confounders, these genes are said to have a causal regulatory relationship. Inferring gene regulatory relationships is related to the detection of trajectory‐associated regulatory genes. Indeed, several single‐cell GRN inference methods use trajectories with mechanistic differential equation models (Ocone _et al_ , [2015](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0095); Matsumoto _et al_ , [2017](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0085)).
While there exist GRN inference methods that were specifically developed for scRNA‐seq data (SCONE: Matsumoto _et al_ , [2017](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0085); PIDC: Chan _et al_ , [2017](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0022); SCENIC: Aibar _et al_ , [2017](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0001)), a recent comparison has shown both bulk and single‐cell methods to perform poorly on these data (Chen & Mar, [2018](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0023)). GRN inference methods may still offer valuable insights to identify causal regulators of biological processes, yet we recommend that these methods be used with care.
##### Pitfalls & recommendations:
•
Users should be wary of uncertainty in the inferred regulatory relationships. Modules of genes that are enriched for regulatory relationships will be more reliable than individual edges.
## Analysis platforms
Single‐cell analysis workflows are collations of independently developed tools. To facilitate the movement of data between these tools, single‐cell platforms have been developed around consistent data formats. These platforms provide a basis for the construction of analysis pipelines. Currently available platforms exist on the command line in R (McCarthy _et al_ , [2017](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0087); Butler _et al_ , [2018](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0020)) or Python (Wolf _et al_ , [2018](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0146)), and as local applications (Patel, [2018](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0100); preprint: Scholz _et al_ , [2018](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0120)) or Web servers (Gardeux _et al_ , [2017](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0040); Zhu _et al_ , [2017](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0158)) with graphical user interfaces (GUIs). An overview of platforms is available in Zhu _et al_ ([2017](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0158)) and Zappia _et al_ ([2018](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0152)).
Among command line platforms, Scater (McCarthy _et al_ , [2017](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0087)) and Seurat (Butler _et al_ , [2018](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0020)) easily interface with the large variety of analysis tools available via the R Bioconductor project (Huber _et al_ , [2015](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0055)). Scater has a particular strength in QC and pre‐processing, while Seurat is arguably the most popular and comprehensive platform, which includes a large array of tools and tutorials. A recent addition to this group is scanpy (Wolf _et al_ , [2018](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0146)), a growing Python‐based platform, which exhibits improved scaling to larger numbers of cells. It leverages the increasing number of tools written in Python, which is particularly popular for machine learning applications.
Graphical user interface platforms enable non‐expert users to build single‐cell analysis workflows. Users are often guided through prescribed workflows that facilitate the analysis, but also limit user flexibility. These platforms are especially useful for exploratory analysis. Platforms such as Granatum (Zhu _et al_ , [2017](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0158)) and ASAP (Gardeux _et al_ , [2017](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0040)) differ in the tools they integrate, with Granatum including the larger variety of methods. As Web servers, these two platforms are readily available, yet computational infrastructure will limit their ability to scale to large datasets. For example, ASAP was tested on a dataset of only 92 cells. Alternatives to the Web‐based GUI platforms are packages such as FASTGenomics (preprint: Scholz _et al_ , [2018](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0120)), iSEE (Rue‐Albrecht _et al_ , [2018](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0115)), IS‐CellR (Patel, [2018](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0100)), and Granatum run on a local server. These are platforms and GUI wrappers that can scale with the locally available computational power. In future, the ongoing development of the Human Cell Atlas portals () will lead to more powerful visual data exploration tools that scale to large cell numbers.
## Conclusions and outlook
We have reviewed the steps of a typical scRNA‐seq analysis workflow and implemented these in a case study tutorial (). The tutorial was designed to follow current best practices as determined by available method comparisons. While aggregating individual best‐practice tools does not guarantee an optimal pipeline, we hope that our workflow represents a current snapshot of the state of the art in the single‐cell analysis field. It thus provides a suitable entry point into this field for newcomers and contributes to the efforts of the Human Cell Atlas to establish best practices in scRNA‐seq analysis (preprint: Regev _et al_ , [2018](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0108)). It should be noted that available method comparisons necessarily lag behind the latest method developments. Thus, we have mentioned new developments that have not yet been independently evaluated where possible. With the future development of new and better tools, and further comparative studies, the individual tool recommendations presented here will require updates, yet the general considerations regarding the stages of data processing should remain the same.
Two avenues of development that are of particular interest due to their potential for disruption to analysis pipelines are deep learning workflows and single‐cell omic integration. Due to its flexibility to scale to large data, deep learning has revolutionized fields from computer vision to natural language processing, and is starting to have a strong impact in genomics (Webb, [2018](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0141)). First applications to scRNA‐seq are starting to emerge from dimensionality reduction to denoising (e.g. scVis: Ding _et al_ , [2018](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0033); scGen: preprint: Lotfollahi _et al_ , [2018](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0076); DCA: Eraslan _et al_ , [2019](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0035)). Recently, deep learning has been used to produce an embedded workflow that can fit the data, denoise it and perform downstream analysis such as clustering and differential expression within the framework of the model (scVI: Lopez _et al_ , [2018](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0075)). In this set‐up, it is possible to include noise and batch effect estimates into downstream statistical tests while preserving accurate estimates of variation in the data. Integrated modelling approaches such as this have the potential to replace current pipelines, which are often an agglomeration of individual tools.
As single‐cell omic technologies improve, the need for integrated omic analysis pipelines will grow (Tanay & Regev, [2017](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0128)). Future single‐cell platforms will have to be able to deal with different data sources such as DNA methylation (Smallwood _et al_ , [2014](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0122)), chromatin accessibility (Buenrostro _et al_ , [2015](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0017)), or protein abundance (Stoeckius _et al_ , [2017](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0124)), and include tools that integrate these modalities. For this set‐up, it will no longer be possible to use only a single read or count matrix, which we use as the starting point of our tutorial. However, platforms are already adapting to multi‐modal data structures for the integration of RNA velocity, which is calculated from unspliced and spliced read data (La Manno _et al_ , [2018](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0068)). Single‐cell multi‐omic integration can occur via consensus clustering approaches, multi‐omic factor analysis (Argelaguet _et al_ , [2018](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0007)), or multi‐omic gene regulatory network inference (Colomé‐Tatché & Theis, [2018](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0029)). Analysis workflows with these capabilities will be the next stage of development. We envisage that such multi‐omic analysis workflows will build upon the foundation we have laid for scRNA‐seq.
## Author contributions
MDL reviewed the literature and wrote the paper, and FJT supervised the work and critically reviewed the manuscript. Both authors read and approved the final paper.
## Acknowledgements
Special thanks to Leander Dony, who debugged, updated, and tested the case study to work with the latest methods. Furthermore, we would like to thank the many people who proofread the case study notebook and the manuscript and improved it with their comments and expertise. For this, we acknowledge the input of Maren Buttner, David Fischer, Alex Wolf, Lukas Simon, Luis Ospina‐Forero, Sophie Tritschler, Niklas Koehler, Goekcen Eraslan, Benjamin Schubert, Meromit Singer, Dana Pe'er, and Rahul Satija. Special thanks for this also to the anonymous reviewers of the manuscript and the editor, Thomas Lemberger, for their thorough, constructive, and extensive comments. The case study notebook was tested and improved by the early adopters Marius Lange, Hananeh Aliee, Subarna Palit and Lisa Thiergart. Volker Bergen and Alex Wolf also contributed to the workflow by making scanpy adaptations. The choice of dataset to optimally show all aspects of the analysis workflow was facilitated by the kind input from Adam Haber and Aviv Regev. This work was supported by the BMBF grant# 01IS18036A and grant# 01IS18053A, by the German Research Foundation (DFG) within the Collaborative Research Centre 1243, Subproject A17, by the Helmholtz Association (Incubator grant sparse2big, grant # ZT‐I‐0007) and by the Chan Zuckerberg Initiative DAF (advised fund of Silicon Valley Community Foundation, 182835).
## 
[Appendix](https://doi.org/doi/suppl/10.15252/msb.20188746/suppl_file/msb188746-sup-0001-appendix.pdf) (PDF document, 204.29 KB)
[Expanded View Figures PDF](https://doi.org/doi/suppl/10.15252/msb.20188746/suppl_file/msb188746-sup-0002-evfigs.pdf) (PDF document, 118.85 KB)
[Dataset EV1](https://doi.org/doi/suppl/10.15252/msb.20188746/suppl_file/msb188746-sup-0003-datasetev1.zip) (Zip archive, 27.52 KB)
## References
Aibar S, González‐Blas CB, Moerman T, Huynh‐Thu VA, Imrichova H, Hulselmans G, Rambow F, Marine J‐C, Geurts P, Aerts J _et al_ (2017) SCENIC: single‐cell regulatory network inference and clustering. _Nat Methods_ 14: 1083–1086
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0001)
[Crossref](https://doi.org/10.1038/nmeth.4463)
[CAS](https://doi.org/servlet/linkout?suffix=e_1_3_2_2_1&dbid=32&doi=10.15252%2Fmsb.20188746&key=1%3ACAS%3A528%3ADC%252BC2sXhs1aitL7P)
Alpert A, Moore LS, Dubovik T, Shen‐Orr SS (2018) Alignment of single‐cell trajectories to compare cellular expression dynamics. _Nat Methods_ 15: 267–270
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0002)
[Crossref](https://doi.org/10.1038/nmeth.4628)
An Y, Furber KL, Ji S (2017) Pseudogenes regulate parental gene expression via ceRNA network. _J Cell Mol Med_ 21: 185–192
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0003)
Andrews TS, Hemberg M (2018) False signals induced by single‐cell imputation. _F1000Res_ 7: 1740
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0004)
Angelidis I, Simon LM, Fernandez IE, Strunz M, Mayr CH, Greiffo FR, Tsitsiridis G, Graf E, Strom TM, Eickelberg O _et al_ (2019) An atlas of the aging lung mapped by single cell transcriptomics and deep tissue proteomics. _Nat Commun_ 10: 963
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0005)
Angerer P, Simon L, Tritschler S, Alexander Wolf F, Fischer D, Theis FJ (2017) Single cells make big data: new challenges and opportunities in transcriptomics. _Curr Opin Syst Biol_ 4: 85–91
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0006)
Argelaguet R, Velten B, Arnol D, Dietrich S, Zenz T, Marioni JC, Buettner F, Huber W, Stegle O (2018) Multi‐Omics Factor Analysis—a framework for unsupervised integration of multi‐omics data sets. _Mol Syst Biol_ 14: e8124
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0007)
[Crossref](https://doi.org/servlet/linkout?suffix=e_1_3_2_8_1&dbid=4&doi=10.15252%2Fmsb.20188746&key=10.15252%2Fmsb.20178124&site=embo-site)
Arvaniti E, Claassen M (2017) Sensitive detection of rare disease‐associated cell subsets via representation learning. _Nat Commun_ 8: 14825
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0008)
Ashburner M, Ball CA, Blake JA, Botstein D, Butler H, Cherry JM, Davis AP, Dolinski K, Dwight SS, Eppig JT _et al_ (2000) Gene ontology: tool for the unification of biology. The Gene Ontology Consortium. _Nat Genet_ 25: 25–29
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0009)
[Crossref](https://doi.org/10.1038/75556)
[CAS](https://doi.org/servlet/linkout?suffix=e_1_3_2_10_1&dbid=32&doi=10.15252%2Fmsb.20188746&key=1%3ACAS%3A528%3ADC%252BD3cXjtFSlsbc%253D)
Azizi E, Carr AJ, Plitas G, Cornish AE, Konopacki C, Prabhakaran S, Nainys J, Wu K, Kiseliovas V, Setty M _et al_ (2018) Single‐cell map of diverse immune phenotypes in the breast tumor microenvironment. _Cell_ 174: 1293–1308.e36
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0010)
Becht E, McInnes L, Healy J, Dutertre C‐A, Kwok IWH, Ng LG, Ginhoux F, Newell EW (2018) Dimensionality reduction for visualizing single‐cell data using UMAP. _Nat Biotechnol_ 37: 38–44
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0011)
[Crossref](https://doi.org/10.1038/nbt.4314)
Bendall SC, Davis KL, Amir E‐AD, Tadmor MD, Simonds EF, Chen TJ, Shenfeld DK, Nolan GP, Pe'er D (2014) Single‐cell trajectory detection uncovers progression and regulatory coordination in human B cell development. _Cell_ 157: 714–725
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0012)
[Crossref](https://doi.org/10.1016/j.cell.2014.04.005)
[CAS](https://doi.org/servlet/linkout?suffix=e_1_3_2_13_1&dbid=32&doi=10.15252%2Fmsb.20188746&key=1%3ACAS%3A528%3ADC%252BC2cXntFGgsr8%253D)
Blasi T, Buettner F, Strasser MK, Marr C, Theis FJ (2017) cgCorrect: a method to correct for confounding cell‐cell variation due to cell growth in single‐cell transcriptomics. _Phys Biol_ 14: 036001
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0013)
Blondel VD, Guillaume J‐L, Lambiotte R, Lefebvre E (2008) Fast unfolding of communities in large networks. _J Stat Mech: Theory Exp_ 2008: P10008
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0014)
[Crossref](https://doi.org/10.1088/1742-5468/2008/10/P10008)
Brennecke P, Anders S, Kim JK, Kołodziejczyk AA, Zhang X, Proserpio V, Baying B, Benes V, Teichmann SA, Marioni JC _et al_ (2013) Accounting for technical noise in single‐cell RNA‐seq experiments. _Nat Methods_ 10: 1093–1095
[Crossref](https://doi.org/10.1038/nmeth.2645)
[CAS](https://doi.org/servlet/linkout?suffix=e_1_3_2_16_1&dbid=32&doi=10.15252%2Fmsb.20188746&key=1%3ACAS%3A528%3ADC%252BC3sXhsVyqtb3L)
  * [a [...] in single‐cell data (Brennecke et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0015-1)
  * [b [...] (HVGs) are often used (Brennecke et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0015-2)

Briggs JA, Weinreb C, Wagner DE, Megason S, Peshkin L, Kirschner MW, Klein AM (2018) The dynamics of gene expression in vertebrate embryogenesis at single‐cell resolution. _Science_ 360: eaar5780
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0016)
Buenrostro JD, Wu B, Litzenburger UM, Ruff D, Gonzales ML, Snyder MP, Chang HY, Greenleaf WJ (2015) Single‐cell chromatin accessibility reveals principles of regulatory variation. _Nature_ 523: 486–490
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0017)
[Crossref](https://doi.org/10.1038/nature14590)
[CAS](https://doi.org/servlet/linkout?suffix=e_1_3_2_18_1&dbid=32&doi=10.15252%2Fmsb.20188746&key=1%3ACAS%3A528%3ADC%252BC2MXhtFyltLrN)
Buettner F, Natarajan KN, Casale FP, Proserpio V, Scialdone A, Theis FJ, Teichmann SA, Marioni JC, Stegle O (2015) Computational analysis of cell‐to‐cell heterogeneity in single‐cell RNA‐sequencing data reveals hidden subpopulations of cells. _Nat Biotechnol_ 33: 155–160
[Crossref](https://doi.org/10.1038/nbt.3102)
[CAS](https://doi.org/servlet/linkout?suffix=e_1_3_2_19_1&dbid=32&doi=10.15252%2Fmsb.20188746&key=1%3ACAS%3A528%3ADC%252BC2MXhtFKjsLs%253D)
  * [a [...] models such as scLVM (Buettner et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0018-1)
  * [b [...] developmental trajectories (Buettner et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0018-2)
  * [c [...] during maturation (Buettner et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0018-3)

Buettner F, Pratanwanich N, McCarthy DJ, Marioni JC, Stegle O (2017) f‐scLVM: scalable and versatile factor analysis for single‐cell RNA‐seq. _Genome Biol_ 18: 212
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0019)
Butler A, Hoffman P, Smibert P, Papalexi E, Satija R (2018) Integrating single‐cell transcriptomic data across different conditions, technologies, and species. _Nat Biotechnol_ 36: 411–420
[Crossref](https://doi.org/10.1038/nbt.4096)
[CAS](https://doi.org/servlet/linkout?suffix=e_1_3_2_21_1&dbid=32&doi=10.15252%2Fmsb.20188746&key=1%3ACAS%3A528%3ADC%252BC1cXmslKrtL0%253D)
  * [a [...] platforms such as Seurat (Butler et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0020-1)
  * [b [...] the popular Seurat tutorials (Butler et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0020-2)
  * [c [...] Scanpy and Seurat platforms (Butler et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0020-3)
  * [d [...] Correlation Analysis (CCA; Butler et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0020-4)
  * [e [...] ; Butler et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0020-5)
  * [f [...] ) and Seurat (Butler et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0020-6)

Buttner M, Miao Z, Wolf A, Teichmann SA, Theis FJ (2019) A test metric for assessing single‐cell RNA‐seq batch correction. _Nat Methods_ 16: 43–49
  * [a [...] methods for batch correction (Buttner et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0021-1)
  * [b [...] ; Buttner et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0021-2)
  * [c [...] of low‐to‐medium complexity (Buttner et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0021-3)
  * [d [...] batch correction setting (Buttner et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0021-4)
  * [e [...] and normalization steps (Buttner et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0021-5)

Chan TE, Stumpf MPH, Babtie AC (2017) Gene regulatory network inference from single‐cell data using multivariate information measures. _Cell Syst_ 5: 251–267.e3
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0022)
[Crossref](https://doi.org/10.1016/j.cels.2017.08.014)
Chen S, Mar JC (2018) Evaluating methods of inferring gene regulatory networks highlights their lack of performance for single cell gene expression data. _BMC Bioinformatics_ 19: 232
[Crossref](https://doi.org/10.1186/s12859-018-2217-z)
  * [a [...] or via regression models (Chen & Mar, ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0023-1)
  * [b [...] perform poorly on these data (Chen & Mar, ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0023-2)

Chung NC, Storey JD (2015) Statistical significance of variables driving systematic variation in high‐dimensional data. _Bioinformatics_ 31: 545–554
  * [a [...] jackstraw method (Chung & Storey, ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0024-1)
  * [b [...] of genes in the dataset (Chung & Storey, ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0024-2)

Clevers H, Rafelski S, Elowitz M, Klein AM, Shendure J, Trapnell C, Lein E, Lundberg E, Uhlén M, Martinez‐Arias A _et al_ (2017) What is your conceptual definition of ‘cell type’ in the context of a mature organism? _Cell Syst_ 4: 255–259
  * [a [...] ; Clevers et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0025-1)
  * [b [...] ; Clevers et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0025-2)

Cohen M, Giladi A, Gorki A‐D, Solodkin DG, Zada M, Hladik A, Miklosi A, Salame T‐M, Halpern KB, David E _et al_ (2018) Lung single‐cell signaling interaction map reveals basophil role in macrophage imprinting. _Cell_ 175: 1031–1044
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0026)
Coifman RR, Lafon S, Lee AB, Maggioni M, Nadler B, Warner F, Zucker SW (2005) Geometric diffusions as a tool for harmonic analysis and structure definition of data: diffusion maps. _Proc Natl Acad Sci USA_ 102: 7426–7431
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0027)
Cole MB, Risso D, Wagner A, DeTomaso D, Ngai J, Purdom E, Dudoit S, Yosef N (2019) Performance assessment and selection of normalization procedures for single‐cell RNA‐seq. _Cell Syst_ 8: 315–328
  * [a [...] unwanted variation, also exist (Cole et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0028-1)
  * [b [...] with strong batch effects (Cole et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0028-2)
  * [c [...] fit by different models. Indeed Cole et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0028-3)

Colomé‐Tatché M, Theis FJ (2018) Statistical single cell multi‐omics integration. _Curr Opin Syst Biol_ 7: 54–59
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0029)
DePasquale EAK, Schnell DJ, Valiente I, Blaxall BC, Grimes HL, Singh H, Salomonis N (2018) DoubletDecon: Cell‐State Aware Removal of Single‐Cell RNA‐Seq Doublets. _bioRxiv_ <https://doi.org/10.1101/364810> [PREPRINT]
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0030)
van Dijk D, Sharma R, Nainys J, Yim K, Kathail P, Carr AJ, Burdziak C, Moon KR, Chaffer CL, Pattabiraman D _et al_ (2018) Recovering gene interactions from single‐cell data using data diffusion. _Cell_ 174: 716–729.e27
[Crossref](https://doi.org/10.1016/j.cell.2018.05.061)
[CAS](https://doi.org/servlet/linkout?suffix=e_1_3_2_32_1&dbid=32&doi=10.15252%2Fmsb.20188746&key=1%3ACAS%3A528%3ADC%252BC1cXht1Cis7rE)
  * [a [...] several recent tools (MAGIC: van Dijk et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0031-1)
  * [b [...] of gene–gene correlations (van Dijk et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0031-2)

Dillies M‐A, Rau A, Aubert J, Hennequet‐Antier C, Jeanmougin M, Servant N, Keime C, Marot G, Castel D, Estelle J _et al_ (2013) A comprehensive evaluation of normalization methods for Illumina high‐throughput RNA sequencing data analysis. _Brief Bioinform_ 14: 671–683
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0032)
[Crossref](https://doi.org/10.1093/bib/bbs046)
Ding J, Condon A, Shah SP (2018) Interpretable dimensionality reduction of single cell transcriptome data with deep generative models. _Nat Commun_ 9: 2002
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0033)
[Crossref](https://doi.org/10.1038/s41467-018-04368-5)
Duò A, Robinson MD, Soneson C (2018) A systematic performance evaluation of clustering methods for single‐cell RNA‐seq data. _F1000Res_ 7: 1141.
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0034)
Eraslan G, Simon LM, Mircea M, Mueller NS, Theis FJ (2019) Single cell RNA‐seq denoising using a deep count autoencoder. _Nat Commun_ 10: 390
  * [a [...] ; DCA: Eraslan et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0035-1)
  * [b [...] ; Eraslan et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0035-2)
  * [c [...] ; DCA: Eraslan et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0035-3)

Fabregat A, Jupe S, Matthews L, Sidiropoulos K, Gillespie M, Garapati P, Haw R, Jassal B, Korninger F, May B _et al_ (2018) The reactome pathway knowledgebase. _Nucleic Acids Res_ 46: D649–D655
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0036)
[Crossref](https://doi.org/10.1093/nar/gkx1132)
Finak G, McDavid A, Yajima M, Deng J, Gersuk V, Shalek AK, Slichter CK, Miller HW, Juliana McElrath M, Prlic M _et al_ (2015) MAST: a flexible statistical framework for assessing transcriptional changes and characterizing heterogeneity in single‐cell RNA sequencing data. _Genome Biol_ 16: 278
[Crossref](https://doi.org/10.1186/s13059-015-0844-5)
  * [a [...] expression testing (Finak et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0037-1)
  * [b [...] ; Finak et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0037-2)
  * [c [...] the single‐cell tool MAST (Finak et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0037-3)

Fischer DS, Theis FJ, Yosef N (2018) Impulse model‐based differential expression analysis of time course sequencing data. _Nucleic Acids Res_ 46: e119
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0038)
Freytag S, Tian L, Lönnstedt I, Ng M, Bahlo M (2018) Comparison of clustering tools in R for medium‐sized 10x Genomics single‐cell RNA‐sequencing data. _F1000Res_ 7: 1297
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0039)
Gardeux V, David FPA, Shajkofci A, Schwalie PC, Deplancke B (2017) ASAP: a web‐based platform for the analysis and interactive visualization of single‐cell RNA‐seq data. _Bioinformatics_ 33: 3123–3125
  * [a [...] ) or Web servers (Gardeux et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0040-1)
  * [b [...] ) and ASAP (Gardeux et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0040-2)

Gashaw I, Ellinghaus P, Sommer A, Asadullah K (2012) What makes a good drug target?. _Drug Discov Today_ 17: S24–S30
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0041)
Gehring J, Park JH, Chen S, Thomson M, Pachter L (2018) Highly Multiplexed Single‐Cell RNA‐seq for Defining Cell Population and Transcriptional Spaces. _bioRxiv_ <https://doi.org/10.1101/315333> [PREPRINT]
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0042)
Gloor GB, Macklaim JM, Pawlowsky‐Glahn V, Egozcue JJ (2017) Microbiome datasets are compositional: and this is not optional. _Front Microbiol_ 8: 2224
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0043)
Griffiths JA, Scialdone A, Marioni JC (2018) Using single‐cell genomics to understand developmental processes and cell fate decisions. _Mol Syst Biol_ 14: e8046
[Crossref](https://doi.org/servlet/linkout?suffix=e_1_3_2_45_1&dbid=4&doi=10.15252%2Fmsb.20188746&key=10.15252%2Fmsb.20178046&site=embo-site)
  * [a [...] ; Griffiths et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0044-1)
  * [b [...] ; Griffiths et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0044-2)
  * [c [...] uncertainty in their model (Griffiths et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0044-3)

Grün D, Kester L, van Oudenaarden A (2014) Validation of noise models for single‐cell transcriptomics. _Nat Methods_ 11: 637
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0045)
[Crossref](https://doi.org/10.1038/nmeth.2930)
[CAS](https://doi.org/servlet/linkout?suffix=e_1_3_2_46_1&dbid=32&doi=10.15252%2Fmsb.20188746&key=1%3ACAS%3A528%3ADC%252BC2cXmsVKlsLc%253D)
Haber AL, Biton M, Rogel N, Herbst RH, Shekhar K, Smillie C, Burgin G, Delorey TM, Howitt MR, Katz Y _et al_ (2017) A single‐cell survey of the small intestinal epithelium. _Nature_ 551: 333–339
[Crossref](https://doi.org/10.1038/nature24489)
[CAS](https://doi.org/servlet/linkout?suffix=e_1_3_2_47_1&dbid=32&doi=10.15252%2Fmsb.20188746&key=1%3ACAS%3A528%3ADC%252BC2sXhslygtLjK)
  * [a [...] intestinal epithelium data from Haber et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0046-1)
  * [b [...] epithelium dataset from Haber et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0046-2)
  * [c [...] intestinal epithelium data from Haber et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0046-3)
  * [d [...] epithelium regions data from Haber et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0046-4)
  * [e [...] data were obtained from Haber et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0046-5)
  * [f [...] epithelium dataset from Haber et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0046-6)
  * [g [...] mouse intestinal epithelium (Haber et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0046-7)
  * [h [...] intestinal epithelium data from Haber et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0046-8)

Haghverdi L, Buettner F, Theis FJ (2015) Diffusion maps for high‐dimensional single‐cell analysis of differentiation data. _Bioinformatics_ 31: 2989–2998
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0047)
Haghverdi L, Büttner M, Wolf FA, Buettner F, Theis FJ (2016) Diffusion pseudotime robustly reconstructs lineage branching. _Nat Methods_ 13: 845–848
[Crossref](https://doi.org/10.1038/nmeth.3971)
  * [a [...] ; Haghverdi et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0048-1)
  * [b [...] along a trajectory (Haghverdi et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0048-2)
  * [c [...] for example, development (Haghverdi et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0048-3)

Haghverdi L, Lun ATL, Morgan MD, Marioni JC (2018) Batch effects in single‐cell RNA‐sequencing data are corrected by matching mutual nearest neighbors. _Nat Biotechnol_ 36: 421–427
[Crossref](https://doi.org/10.1038/nbt.4091)
  * [a [...] Nearest Neighbours (MNN; Haghverdi et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0049-1)
  * [b [...] include cosine similarity (Haghverdi et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0049-2)

Heimberg G, Bhatnagar R, El‐Samad H, Thomson M (2016) Low dimensionality in gene expression data enables the accurate extraction of transcriptional programs from shallow sequencing. _Cell Syst_ 2: 239–250
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0050)
Hicks SC, Townes FW, Teng M, Irizarry RA (2017) Missing data and technical variability in single‐cell RNA‐sequencing experiments. _Biostatistics_ 19: 562–578
  * [a [...] by clever experimental design (Hicks et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0051-1)
  * [b [...] ; Hicks et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0051-2)
  * [c [...] high cell‐to‐cell variability (Hicks et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0051-3)

Hie BL, Bryson B, Berger B (2018) Panoramic stitching of heterogeneous single‐cell transcriptomic data. _bioRxiv_ <https://doi.org/10.1101/371179> [PREPRINT]
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0052)
Huang DW, Sherman BT, Lempicki RA (2009) Bioinformatics enrichment tools: paths toward the comprehensive functional analysis of large gene lists. _Nucleic Acids Res_ 37: 1–13
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0053)
[Crossref](https://doi.org/10.1093/nar/gkn923)
Huang M, Wang J, Torre E, Dueck H, Shaffer S, Bonasio R, Murray JI, Raj A, Li M, Zhang NR (2018) SAVER: gene expression recovery for single‐cell RNA sequencing. _Nat Methods_ 15: 539–542
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0054)
Huber W, Carey VJ, Gentleman R, Anders S, Carlson M, Carvalho BS, Bravo HC, Davis S, Gatto L, Girke T _et al_ (2015) Orchestrating high‐throughput genomic analysis with Bioconductor. _Nat Methods_ 12: 115–121
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0055)
[Crossref](https://doi.org/10.1038/nmeth.3252)
[CAS](https://doi.org/servlet/linkout?suffix=e_1_3_2_56_1&dbid=32&doi=10.15252%2Fmsb.20188746&key=1%3ACAS%3A528%3ADC%252BC2MXjvVCrurg%253D)
Ilicic T, Kim JK, Kolodziejczyk AA, Bagger FO, McCarthy DJ, Marioni JC, Teichmann SA (2016) Classification of low quality cells from single‐cell RNA‐seq data. _Genome Biol_ 17: 29
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0056)
[Crossref](https://doi.org/10.1186/s13059-016-0888-1)
Johnson WE, Li C, Rabinovic A (2006) Adjusting batch effects in microarray expression data using empirical Bayes methods. _Biostatistics_ 8: 118–127
  * [a [...] ) or batch correction (Johnson et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0058-1)
  * [b [...] has revealed that ComBat (Johnson et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0058-2)

Kanehisa M, Furumichi M, Tanabe M, Sato Y, Morishima K (2017) KEGG: new perspectives on genomes, pathways, diseases and drugs. _Nucleic Acids Res_ 45: D353–D361
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0059)
[Crossref](https://doi.org/10.1093/nar/gkw1092)
[CAS](https://doi.org/servlet/linkout?suffix=e_1_3_2_59_1&dbid=32&doi=10.15252%2Fmsb.20188746&key=1%3ACAS%3A528%3ADC%252BC1cXhslWgsrw%253D)
Kang HM, Subramaniam M, Targ S, Nguyen M, Maliskova L, McCarthy E, Wan E, Wong S, Byrnes L, Lanata CM _et al_ (2018) Multiplexed droplet single‐cell RNA‐sequencing using natural genetic variation. _Nat Biotechnol_ 36: 89–94
[Crossref](https://doi.org/10.1038/nbt.4042)
  * [a [...] ), or via genetic variation (Kang et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0060-1)
  * [b [...] experimental conditions (Kang et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0060-2)

Kharchenko PV, Silberstein L, Scadden DT (2014) Bayesian approach to single‐cell differential expression analysis. _Nat Methods_ 11: 740–742
[Crossref](https://doi.org/10.1038/nmeth.2967)
[CAS](https://doi.org/servlet/linkout?suffix=e_1_3_2_61_1&dbid=32&doi=10.15252%2Fmsb.20188746&key=1%3ACAS%3A528%3ADC%252BC2cXotFCjs70%253D)
  * [a [...] ; Kharchenko et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0061-1)
  * [b [...] for single‐cell data (Kharchenko et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0061-2)

Kim T, Chen IR, Lin Y, Wang AY‐Y, Yang JYH, Yang P (2018) Impact of similarity metrics on single‐cell RNA‐seq data clustering. _Brief Bioinform_ <https://doi.org/10.1093/bib/bby076>
  * [a [...] distance metrics (Kim et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0062-1)
  * [b [...] the basis for Gaussian kernels (Kim et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0062-2)

Kiselev VY, Yiu A, Hemberg M (2018b) Scmap: projection of single‐cell RNA‐seq data across data sets. _Nat Methods_ 15: 359–362
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0064)
[Crossref](https://doi.org/10.1038/nmeth.4644)
Klein AM, Mazutis L, Akartuna I, Tallapragada N, Veres A, Li V, Peshkin L, Weitz DA, Kirschner MW (2015) Droplet barcoding for single‐cell transcriptomics applied to embryonic stem cells. _Cell_ 161: 1187–1201
[Crossref](https://doi.org/10.1016/j.cell.2015.04.044)
[CAS](https://doi.org/servlet/linkout?suffix=e_1_3_2_64_1&dbid=32&doi=10.15252%2Fmsb.20188746&key=1%3ACAS%3A528%3ADC%252BC2MXpt1SgtL0%253D)
  * [a [...] ), indrops (Klein et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0065-1)
  * [b [...] ). Preliminary results from Klein et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0065-2)
  * [c [...] from Klein et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0065-3)

Korsunsky I, Fan J, Slowikowski K, Zhang F, Wei K (2018) Fast, sensitive, and accurate integration of single cell data with Harmony. _bioRxiv_ <https://doi.org/10.1101/461954> [PREPRINT]
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0066)
Kowalczyk MS, Tirosh I, Heckl D, Rao TN, Dixit A, Haas BJ, Schneider RK, Wagers AJ, Ebert BL, Regev A (2015) Single‐cell RNA‐seq reveals changes in cell cycle and differentiation programs upon aging of hematopoietic stem cells. _Genome Res_ 25: 1860–1872
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0067)
[Crossref](https://doi.org/10.1101/gr.192237.115)
La Manno G, Soldatov R, Zeisel A, Braun E, Hochgerner H, Petukhov V, Lidschreiber K, Kastriti ME, Lönnerberg P, Furlan A _et al_ (2018) RNA velocity of single cells. _Nature_ 560: 494–498
[Crossref](https://doi.org/10.1038/s41586-018-0414-6)
[CAS](https://doi.org/servlet/linkout?suffix=e_1_3_2_67_1&dbid=32&doi=10.15252%2Fmsb.20188746&key=1%3ACAS%3A528%3ADC%252BC1cXhsVynsrrL)
  * [a [...] fates in differentiation (La Manno et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0068-1)
  * [b [...] to add directionality (La Manno et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0068-2)
  * [c [...] and spliced read data (La Manno et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0068-3)

Law CW, Chen Y, Shi W, Smyth GK (2014) voom: precision weights unlock linear model analysis tools for RNA‐seq read counts. _Genome Biol_ 15: R29
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0069)
[Crossref](https://doi.org/10.1186/gb-2014-15-2-r29)
Levine JH, Simonds EF, Bendall SC, Davis KL, Amir E‐AD, Tadmor MD, Litvin O, Fienberg HG, Jager A, Zunder ER _et al_ (2015) Data‐driven phenotypic dissection of AML reveals progenitor‐like cells that correlate with prognosis. _Cell_ 162: 184–197
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0070)
[Crossref](https://doi.org/10.1016/j.cell.2015.05.047)
[CAS](https://doi.org/servlet/linkout?suffix=e_1_3_2_69_1&dbid=32&doi=10.15252%2Fmsb.20188746&key=1%3ACAS%3A528%3ADC%252BC2MXhtV2it7jE)
Li B, Ruotti V, Stewart RM, Thomson JA, Dewey CN (2009) RNA‐Seq gene expression estimation with read mapping uncertainty. _Bioinformatics_ 26: 493–500
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0071)
Li WV, Li JJ (2018) An accurate and robust imputation method scImpute for single‐cell RNA‐seq data. _Nat Commun_ 9: 997
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0072)
Liberzon A, Subramanian A, Pinchback R, Thorvaldsdottir H, Tamayo P, Mesirov JP (2011) Molecular signatures database (MSigDB) 3.0. _Bioinformatics_ 27: 1739–1740
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0073)
[Crossref](https://doi.org/10.1093/bioinformatics/btr260)
[CAS](https://doi.org/servlet/linkout?suffix=e_1_3_2_72_1&dbid=32&doi=10.15252%2Fmsb.20188746&key=1%3ACAS%3A528%3ADC%252BC3MXntFGktLw%253D)
Liu Y, Wang T, Zheng D (2018) RISC: robust integration of single‐cell RNA‐seq datasets with different extents of cell cluster overlap. _bioRxiv_ <https://doi.org/10.1101/483297> [PREPRINT]
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0074)
Lopez R, Regier J, Cole MB, Jordan M, Yosef N (2018) Deep generative modeling fro single‐cell transcriptomics. _Nat Methods_ 15: 1053–1058
  * [a [...] ; scVI: Lopez et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0075-1)
  * [b [...] implemented in the scVI tool (Lopez et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0075-2)
  * [c [...] framework of the model (scVI: Lopez et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0075-3)

Lotfollahi M, Wolf FA, Theis FJ (2018) Generative modeling and latent space arithmetics predict single‐cell perturbation response across cell types, studies and species. _bioRxiv_ <https://doi.org/10.1101/478503> [PREPRINT]
  * [a [...] ), scGen (preprint: Lotfollahi et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0076-1)
  * [b [...] ; scGen: preprint: Lotfollahi et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0076-2)

Love MI, Huber W, Anders S (2014) Moderated estimation of fold change and dispersion for RNA‐seq data with DESeq2. _Genome Biol_ 15: 550
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0077)
[Crossref](https://doi.org/10.1186/s13059-014-0550-8)
Lun ATL, Bach K, Marioni JC (2016a) Pooling across cells to normalize single‐cell RNA sequencing data with many zero counts. _Genome Biol_ 17: 75
[Crossref](https://doi.org/10.1186/s13059-016-0947-7)
  * [a [...] normalization methods (Lun et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0078-1)
  * [b [...] size factor estimation method (Lun et al, ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0078-2)

Lun ATL, McCarthy DJ, Marioni JC (2016b) A step‐by‐step workflow for low‐level analysis of single‐cell RNA‐seq data. _F1000Res_ 5: 2122
  * [a [...] ; Lun et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0079-1)
  * [b [...] steps are covered in Lun et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0079-2)

Lun ATL, Richard AC, Marioni JC (2017) Testing for differential abundance in mass cytometry data. _Nat Methods_ 14: 707–709
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0080)
Lun ATL (2018) Overcoming systematic errors caused by log‐transformation of normalized single‐cell RNA sequencing data. _bioRxiv_ <https://doi.org/10.1101/404962> [PREPRINT]
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0081)
van der Maaten L, Hinton G (2008) Visualizing data using t‐SNE. _J Mach Learn Res_ 9: 2579–2605
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0082)
Macosko EZ, Basu A, Satija R, Nemesh J, Shekhar K, Goldman M, Tirosh I, Bialas AR, Kamitaki N, Martersteck EM _et al_ (2015) Highly parallel genome‐wide expression profiling of individual cells using nanoliter droplets. _Cell_ 161: 1202–1214
[Crossref](https://doi.org/10.1016/j.cell.2015.05.002)
[CAS](https://doi.org/servlet/linkout?suffix=e_1_3_2_82_1&dbid=32&doi=10.15252%2Fmsb.20188746&key=1%3ACAS%3A528%3ADC%252BC2MXpt1Sgt7o%253D)
  * [a [...] ); Macosko et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0083-1)
  * [b [...] obtained from the literature (Macosko et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0083-2)
  * [c [...] ; Macosko et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0083-3)

MacQueen J (1967) Some methods for classification and analysis of multivariate observations. In _Proceedings of the fifth Berkeley symposium on mathematical statistics and probability_ , pp 281–297. Oakland, CA.
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0084)
Matsumoto H, Kiryu H, Furusawa C, Ko MSH, Ko SBH, Gouda N, Hayashi T, Nikaido I (2017) SCODE: an efficient regulatory network inference algorithm from single‐cell RNA‐Seq during differentiation. _Bioinformatics_ 33: 2314–2321
[Crossref](https://doi.org/10.1093/bioinformatics/btx194)
  * [a [...] ; Matsumoto et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0085-1)
  * [b [...] for scRNA‐seq data (SCONE: Matsumoto et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0085-2)

Mayer C, Hafemeister C, Bandler RC, Machold R, Batista Brito R, Jaglin X, Allaway K, Butler A, Fishell G, Satija R (2018) Developmental diversification of cortical inhibitory interneurons. _Nature_ 555: 457–462
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0086)
[Crossref](https://doi.org/10.1038/nature25999)
McCarthy DJ, Campbell KR, Lun ATL, Wills QF (2017) Scater: pre‐processing, quality control, normalization and visualization of single‐cell RNA‐seq data in R. _Bioinformatics_ 33: 1179–1186
[Crossref](https://doi.org/10.1093/bioinformatics/btw777)
[CAS](https://doi.org/servlet/linkout?suffix=e_1_3_2_86_1&dbid=32&doi=10.15252%2Fmsb.20188746&key=1%3ACAS%3A528%3ADC%252BC1cXhvFagtL%2FP)
  * [a [...] ), Scater (McCarthy et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0087-1)
  * [b [...] on the command line in R (McCarthy et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0087-2)
  * [c [...] line platforms, Scater (McCarthy et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0087-3)

McDavid A, Finak G, Gottardo R (2016) The contribution of cell cycle to heterogeneity in single‐cell RNA‐seq data. _Nat Biotechnol_ 34: 591–593
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0088)
McGinnis CS, Murrow LM, Gartner ZJ (2018) DoubletFinder: doublet detection in single‐cell RNA sequencing data using artificial nearest neighbors. _Cell Syst_ 8: 329–337
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0089)
McInnes L, Healy J (2018) UMAP: uniform manifold approximation and projection for dimension reduction. _arXiv_ [PREPRINT]
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0090)
Moignard V, Woodhouse S, Haghverdi L, Lilly AJ, Tanaka Y, Wilkinson AC, Buettner F, Macaulay IC, Jawaid W, Diamanti E _et al_ (2015) Decoding the regulatory network of early blood development from single‐cell gene expression measurements. _Nat Biotechnol_ 33: 269–276
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0091)
Montoro DT, Haber AL, Biton M, Vinarsky V, Lin B, Birket SE, Yuan F, Chen S, Leung HM, Villoria J _et al_ (2018) A revised airway epithelial hierarchy includes CFTR‐expressing ionocytes. _Nature_ 560: 319–324
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0092)
Moon KR, Stanley JS, Burkhardt D, van Dijk D, Wolf G, Krishnaswamy S (2018) Manifold learning‐based methods for analyzing single‐cell RNA‐sequencing data. _Curr Opin Syst Biol_ 7: 36–46
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0093)
Newman MEJ, Girvan M (2004) Finding and evaluating community structure in networks. _Phys Rev E Stat Nonlin Soft Matter Phys_ 69: 026113
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0094)
Ocone A, Haghverdi L, Mueller NS, Theis FJ (2015) Reconstructing gene regulatory dynamics from high‐dimensional single‐cell snapshot data. _Bioinformatics_ 31: i89–i96
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0095)
Pachter L (2011) Models for transcript quantification from RNA‐Seq. _arXiv_ [PREPRINT]
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0096)
Parekh S, Ziegenhain C, Vieth B, Enard W, Hellmann I (2018) zUMIs ‐ A fast and flexible pipeline to process RNA sequencing data with UMIs. _Gigascience_ 7: giy059
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0097)
[Crossref](https://doi.org/10.1093/gigascience/giy059)
Park JE, Polanski K, Meyer K, Teichmann SA (2018) Fast batch alignment of single cell transcriptomes unifies multiple mouse cell atlases into an integrated landscape. _bioRxiv_ <https://doi.org/10.1101/397042> [PREPRINT]
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0098)
Patel AP, Tirosh I, Trombetta JJ, Shalek AK, Gillespie SM, Wakimoto H, Cahill DP, Nahed BV, Curry WT, Martuza RL _et al_ (2014) Single‐cell RNA‐seq highlights intratumoral heterogeneity in primary glioblastoma. _Science_ 344: 1396–1401
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0099)
[Crossref](https://doi.org/10.1126/science.1254257)
[CAS](https://doi.org/servlet/linkout?suffix=e_1_3_2_98_1&dbid=32&doi=10.15252%2Fmsb.20188746&key=1%3ACAS%3A528%3ADC%252BC2cXpslygsL4%253D)
Patel MV (2018) iS‐CellR: a user‐friendly tool for analyzing and visualizing single‐cell RNA sequencing data. _Bioinformatics_ 34: 4305–4306
  * [a [...] ), and as local applications (Patel, ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0100-1)
  * [b [...] ), IS‐CellR (Patel, ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0100-2)

Pearson K (1901) On Lines and Planes of Closest Fit to Systems of Points in Space. _Philos Mag_ 2: 559–572
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0101)
Plass M, Solana J, Alexander Wolf F, Ayoub S, Misios A, Glažar P, Obermayer B, Theis FJ, Kocks C, Rajewsky N (2018) Cell type atlas and lineage tree of a whole complex animal by single‐cell transcriptomics. _Science_ 360: eaaq1723
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0102)
[Crossref](https://doi.org/10.1126/science.aaq1723)
Plasschaert LW, Žilionis R, Choo‐Wing R, Savova V, Knehr J, Roma G, Klein AM, Jaffe AB (2018) A single‐cell atlas of the airway epithelium reveals the CFTR‐rich pulmonary ionocyte. _Nature_ 560: 377–381
  * [a [...] ; Plasschaert et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0103-1)
  * [b [...] quality differences as in Plasschaert et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0103-2)

Pliner HA, Shendure J, Trapnell C (2019) Supervised classification enables rapid annotation of cell atlases. _bioRxiv_ <https://doi.org/10.1101/538652> [PREPRINT]
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0104)
Qiu X, Hill A, Packer J, Lin D, Ma Y‐A, Trapnell C (2017a) Single‐cell mRNA quantification and differential analysis with census. _Nat Methods_ 14: 309–315
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0105)
[Crossref](https://doi.org/10.1038/nmeth.4150)
Regev A, Teichmann SA, Lander ES, Amit I, Benoist C, Birney E, Bodenmiller B, Campbell PJ, Carninci P, Clatworthy M _et al_ (2017) Science forum: the human cell atlas. _Elife_ 6: e27041
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0107)
Regev A, Teichmann S, Rozenblatt‐Rosen O, Stubbington M, Ardlie K, Amit I, Arlotta P, Bader G, Benoist C, Biton M _et al_ (2018) The human cell atlas white paper. _arXiv_ [PREPRINT]
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0108)
Reichardt J, Bornholdt S (2006) Statistical mechanics of community detection. _Phys. Rev. E Stat. Nonlin. Soft Matter Phys_ 74: 016110
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0109)
Risso D, Perraudeau F, Gribkova S, Dudoit S, Vert J‐P (2018) A general and flexible method for signal extraction from single‐cell RNA‐seq data. _Nat Commun_ 9: 284
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0111)
[Crossref](https://doi.org/10.1038/s41467-017-02554-5)
Ritchie ME, Phipson B, Wu D, Hu Y, Law CW, Shi W, Smyth GK (2015) limma powers differential expression analyses for RNA‐sequencing and microarray studies. _Nucleic Acids Res_ 43: e47
[Crossref](https://doi.org/10.1093/nar/gkv007)
  * [a [...] ; Ritchie et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0112-1)
  * [b [...] using the limma package (Ritchie et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0112-2)

Robinson MD, McCarthy DJ, Smyth GK (2010) edgeR: a bioconductor package for differential expression analysis of digital gene expression data. _Bioinformatics_ 26: 139–140
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0113)
[Crossref](https://doi.org/10.1093/bioinformatics/btp616)
[CAS](https://doi.org/servlet/linkout?suffix=e_1_3_2_110_1&dbid=32&doi=10.15252%2Fmsb.20188746&key=1%3ACAS%3A528%3ADC%252BD1MXhs1WlurvO)
Rostom R, Svensson V, Teichmann SA, Kar G (2017) Computational approaches for interpreting scRNA‐seq data. _FEBS Lett_ 591: 2213–2225
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0114)
Rue‐Albrecht K, Marini F, Soneson C, Lun ATL (2018) iSEE: interactive summarizedexperiment explorer. _F1000Res_ 7: 741
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0115)
Saelens W, Cannoodt R, Todorov H, Saeys Y (2018) A comparison of single‐cell trajectory inference methods. _Nat Biotechnol_ 37: 547–554
  * [a [...] comparison of TI methods (Saelens et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0116-1)
  * [b [...] to improve performance (Saelens et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0116-2)
  * [c [...] We recommend using the Saelens et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0116-3)
  * [d [...] methods in a recent review (Saelens et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0116-4)

Saunders A, Macosko EZ, Wysoker A, Goldman M, Krienen FM, de Rivera H, Bien E, Baum M, Bortolin L, Wang S _et al_ (2018) Molecular diversity and specializations among the cells of the adult mouse brain. _Cell_ 174: 1015–1030.e16
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0118)
[Crossref](https://doi.org/10.1016/j.cell.2018.07.028)
[CAS](https://doi.org/servlet/linkout?suffix=e_1_3_2_114_1&dbid=32&doi=10.15252%2Fmsb.20188746&key=1%3ACAS%3A528%3ADC%252BC1cXhsVyltrrK)
Scholtens D, von Heydebreck A (2005) Analysis of Differential Gene Expression Studies. In _Statistics for Biology and Health_ , pp 229–248.
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0119)
Scholz CJ, Biernat P, Becker M, Baßler K, Günther P, Balfer J, Dickten H, Flöer L, Heikamp K, Angerer P _et al_ (2018) FASTGenomics: An analytical ecosystem for single‐cell RNA sequencing data. _bioRxiv_ <https://doi.org/10.1101/272476> [PREPRINT]
  * [a [...] support is growing (preprint: Scholz et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0120-1)
  * [b [...] ; preprint: Scholz et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0120-2)
  * [c [...] as FASTGenomics (preprint: Scholz et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0120-3)

Segerstolpe Å, Palasantza A, Eliasson P, Andersson E‐M, Andréasson A‐C, Sun X, Picelli S, Sabirsh A, Clausen M, Bjursell MK _et al_ (2016) Single‐cell transcriptome profiling of human pancreatic islets in health and type 2 diabetes. _Cell Metab_ 24: 593–607
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0121)
[Crossref](https://doi.org/10.1016/j.cmet.2016.08.020)
[CAS](https://doi.org/servlet/linkout?suffix=e_1_3_2_117_1&dbid=32&doi=10.15252%2Fmsb.20188746&key=1%3ACAS%3A528%3ADC%252BC28XhsFGru7zN)
Smallwood SA, Lee HJ, Angermueller C, Krueger F, Saadeh H, Peat J, Andrews SR, Stegle O, Reik W, Kelsey G (2014) Single‐cell genome‐wide bisulfite sequencing for assessing epigenetic heterogeneity. _Nat Methods_ 11: 817–820
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0122)
Soneson C, Robinson MD (2018) Bias, robustness and scalability in single‐cell differential expression analysis. _Nat Methods_ 15: 255–261
[Crossref](https://doi.org/10.1038/nmeth.4612)
  * [a [...] ; Soneson & Robinson, ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0123-1)
  * [b [...] data as input (Soneson & Robinson, ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0123-2)
  * [c [...] single‐cell tools (Soneson & Robinson, ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0123-3)
  * [d [...] aforementioned study (Soneson & Robinson, ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0123-4)

Stoeckius M, Hafemeister C, Stephenson W, Houck‐Loomis B, Chattopadhyay PK, Swerdlow H, Satija R, Smibert P (2017) Simultaneous epitope and transcriptome measurement in single cells. _Nat Methods_ 14: 865–868
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0124)
[Crossref](https://doi.org/10.1038/nmeth.4380)
Street K, Risso D, Fletcher RB, Das D, Ngai J, Yosef N, Purdom E, Dudoit S (2018) Slingshot: cell lineage and pseudotime inference for single‐cell transcriptomics. _BMC Genom_ 19: 477
[Crossref](https://doi.org/10.1186/s12864-018-4772-0)
  * [a [...] over genes in their tutorial (Street et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0125-1)
  * [b [...] revealed that Slingshot (Street et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0125-2)
  * [c [...] the online Slingshot tutorial (Street et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0125-3)

Svensson V, Natarajan KN, Ly L‐H, Miragaia RJ, Labalette C, Macaulay IC, Cvejic A, Teichmann SA (2017) Power analysis of single‐cell RNA‐sequencing experiments. _Nat Methods_ 14: 381–387
  * [a [...] ); Svensson et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0126-1)
  * [b [...] than droplet‐based data (Svensson et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0126-2)
  * [c [...] and 3′ enrichment methods (Svensson et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0126-3)

Tabula Muris Consortium, Overall coordination, Logistical coordination, Organ collection and processing, Library preparation and sequencing, Computational data analysis, Cell type annotation, Writing group, Supplemental text writing group & Principal investigators (2018) Single‐cell transcriptomics of 20 mouse organs creates a Tabula Muris. _Nature_ 562: 367–372
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0127)
[Crossref](https://doi.org/10.1038/s41586-018-0590-4)
Tanay A, Regev A (2017) Scaling single‐cell genomics from phenomenology to mechanism. _Nature_ 541: 331–338
  * [a [...] ; Tanay & Regev, ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0128-1)
  * [b [...] are continuous processes (Tanay & Regev, ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0128-2)
  * [c [...] pipelines will grow (Tanay & Regev, ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0128-3)

Tarca AL, Bhatti G, Romero R (2013) A comparison of gene set analysis methods in terms of sensitivity, prioritization and specificity. _PLoS One_ 8: e79217
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0129)
The Gene Ontology Consortium (2017) Expansion of the gene ontology knowledgebase and resources. _Nucleic Acids Res_ 45: D331–D338
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0130)
Tibshirani R, Hastie T, Narasimhan B, Chu G (2002) Diagnosis of multiple cancer types by shrunken centroids of gene expression. _Proc Natl Acad Sci USA_ 99: 6567–6572
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0131)
[Crossref](https://doi.org/10.1073/pnas.082099299)
[CAS](https://doi.org/servlet/linkout?suffix=e_1_3_2_127_1&dbid=32&doi=10.15252%2Fmsb.20188746&key=1%3ACAS%3A528%3ADC%252BD38XjvFCktbg%253D)
Trapnell C, Cacchiarelli D, Grimsby J, Pokharel P, Li S, Morse M, Lennon NJ, Livak KJ, Mikkelsen TS, Rinn JL (2014) The dynamics and regulators of cell fate decisions are revealed by pseudotemporal ordering of single cells. _Nat Biotechnol_ 32: 381–386
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0132)
[Crossref](https://doi.org/10.1038/nbt.2859)
[CAS](https://doi.org/servlet/linkout?suffix=e_1_3_2_128_1&dbid=32&doi=10.15252%2Fmsb.20188746&key=1%3ACAS%3A528%3ADC%252BC2cXks12ku7c%253D)
Vallejos CA, Risso D, Scialdone A, Dudoit S, Marioni JC (2017) Normalizing single‐cell RNA sequencing data: challenges and opportunities. _Nat Methods_ 14: 565–571
  * [a [...] ; Vallejos et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0133-1)
  * [b [...] from the original authors (Vallejos et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0133-2)
  * [c [...] ; Vallejos et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0133-3)

Van den Berge K, Perraudeau F, Soneson C, Love MI, Risso D, Vert J‐P, Robinson MD, Dudoit S, Clement L (2018) Observation weights unlock bulk RNA‐seq tools for zero inflation and single‐cell applications. _Genome Biol_ 19: 24
  * [a [...] counterparts (Van den Berge et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0134-1)
  * [b [...] weighted bulk methods (Van den Berge et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0134-2)

Vento‐Tormo R, Efremova M, Botting RA, Turco MY, Vento‐Tormo M, Meyer KB, Park J, Stephenson E, Polański K, Goncalves A _et al_ (2018) Single‐cell reconstruction of the early maternal‐fetal interface in humans. _Nature_ 563: 347–353
[Crossref](https://doi.org/10.1038/s41586-018-0698-6)
  * [a [...] ; Vento‐Tormo et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0135-1)
  * [b [...] the recent CellPhoneDB (Vento‐Tormo et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0135-2)
  * [c [...] ; Vento‐Tormo et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0135-3)

Vieth B, Ziegenhain C, Parekh S, Enard W, Hellmann I (2017) powsimR: power analysis for bulk and single cell RNA‐seq experiments. _Bioinformatics_ 33: 3486–3488
  * [a [...] of scRNA‐seq data. For example, Vieth et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0136-1)
  * [b [...] fact log‐normally distributed (Vieth et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0136-2)
  * [c [...] comparison on a single dataset (Vieth et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0136-3)

Vieth B, Parekh S, Ziegenhain C, Enard W, Hellmann I (2019) A systematic evaluation of single cell RNA‐seq analysis pipelines: library preparation and normalisation methods have the biggest impact on the performance of scRNA‐seq studies. _bioRxiv_ <https://doi.org/10.1101/583013> [PREPRINT]
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0500)
Wagner A, Regev A, Yosef N (2016) Revealing the vectors of cellular identity with single‐cell genomics. _Nat Biotechnol_ 34: 1145–1160
  * [a [...] within cell‐type clusters (Wagner et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0137-1)
  * [b [...] determine cellular identity (Wagner et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0137-2)
  * [c [...] T cells (Wagner et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0137-3)

Wagner DE, Weinreb C, Collins ZM, Briggs JA, Megason SG, Klein AM (2018) Single‐cell mapping of gene expression landscapes and lineage in the zebrafish embryo. _Science_ 360: 981–987
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0138)
Wang B, Zhu J, Pierson E, Ramazzotti D, Batzoglou S (2017) Visualization and analysis of single‐cell RNA‐seq data by kernel‐based similarity learning. _Nat Methods_ 14: 414–416
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0139)
Wattenberg M, Viégas F, Johnson I (2016) How to use t‐SNE effectively. _Distill_. <https://doi.org/10.23915/distill.00002>
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0140)
Webb S (2018) Deep learning for biology. _Nature_ 554: 555–557
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0141)
Weber LM, Robinson MD (2016) Comparison of clustering methods for high‐dimensional single‐cell flow and mass cytometry data. _Cytometry A_ 89: 1084–1096
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0142)
Weber LM, Nowicka M, Soneson C & Robinson MD (2018) diffcyt: Differential discovery in high‐dimensional cytometry via high‐resolution clustering. _Commun Biol_ 2: 183
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0143)
Weinreb C, Wolock S, Klein AM (2018) SPRING: a kinetic interface for visualizing high dimensional single‐cell expression data. _Bioinformatics_ 34: 1246–1248
  * [a [...] appropriate. For example, Weinreb et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0144-1)
  * [b [...] tools such as SPRING (Weinreb et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0144-2)

Welch J, Kozareva V, Ferreira A, Vanderburg C (2018) Integrative inference of brain cell similarities and differences from single‐cell genomics. _bioRxiv_ <https://doi.org/10.1101/459891> [PREPRINT]
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0145)
Wolf FA, Angerer P, Theis FJ (2018) SCANPY: large‐scale single‐cell gene expression data analysis. _Genome Biol_ 19: 15
[Crossref](https://doi.org/10.1186/s13059-017-1382-0)
  * [a [...] ), or Scanpy (Wolf et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0146-1)
  * [b [...] ; Wolf et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0146-2)
  * [c [...] ) or Python (Wolf et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0146-3)
  * [d [...] to this group is scanpy (Wolf et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0146-4)

Wolf FA, Hamey F, Plass M, Solana J, Dahlin JS, Gottgens B, Rajewsky N, Simon L, Theis FJ (2019) Graph abstraction reconciles clustering with trajectory inference through a topology preserving map of single cells. _Genome Biol_ 20: 59
  * [a [...] of the underlying topology (Wolf et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0147-1)
  * [b [...] graph abstraction (PAGA; Wolf et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0147-2)
  * [c [...] topology of the expression data (Wolf et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0147-3)
  * [d [...] trajectories are expected, PAGA (Wolf et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0147-4)
  * [e [...] D; Wolf et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0147-5)

Wolock SL, Lopez R, Klein AM (2019) Scrublet: computational identification of cell doublets in single‐cell transcriptomic data. _Cell Syst_ 8: 281–291.
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0148)
[Crossref](https://doi.org/10.1016/j.cels.2018.11.005)
Yip SH, Sham PC, Wang J (2018) Evaluation of tools for highly variable gene discovery from single‐cell RNA‐seq data. _Brief Bioinform_ : bby011
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0149)
Young MD, Behjati S (2018) SoupX removes ambient RNA contamination from droplet based single cell RNA sequencing data. _bioRxiv_ <https://doi.org/10.1101/303727> [PREPRINT]
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0150)
Zappia L, Phipson B, Oshlack A (2017) Splatter: simulation of single‐cell RNA sequencing data. _Genome Biol_ 18: 174
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0151)
Zappia L, Phipson B, Oshlack A (2018) Exploring the single‐cell RNA‐seq analysis landscape with the scRNA‐tools database. _PLoS Comput Biol_ 14: e1006245
  * [a [...] ; Zappia et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0152-1)
  * [b [...] prominently R and Python (Zappia et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0152-2)
  * [c [...] ) and Zappia et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0152-3)

Zeisel A, Hochgerner H, Lonnerberg P, Johnsson A, Memic F, van der Zwan J, Haring M, Braun E, Borm L, La Manno G _et al_ (2018) Molecular architecture of the mouse nervous system. _Cell_ 174: 999–1014
[Crossref](https://doi.org/10.1016/j.cell.2018.06.021)
[CAS](https://doi.org/servlet/linkout?suffix=e_1_3_2_150_1&dbid=32&doi=10.15252%2Fmsb.20188746&key=1%3ACAS%3A528%3ADC%252BC1cXhsVyltrvN)
  * [a [...] such as the mouse brain atlas (Zeisel et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0153-1)
  * [b [...] (Zeisel et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0153-2)

Zepp JA, Zacharias WJ, Frank DB, Cavanaugh CA, Zhou S, Morley MP, Morrisey EE (2017) Distinct mesenchymal lineages and niches promote epithelial self‐renewal and myofibrogenesis in the lung. _Cell_ 170: 1134–1148.e10
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0154)
Zhang JM, Kamath GM, Tse DN (2018) Towards a post‐clustering test for differential expression. _bioRxiv_ <https://doi.org/10.1101/463265> [PREPRINT]
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0155)
Zheng GXY, Terry JM, Belgrader P, Ryvkin P, Bent ZW, Wilson R, Ziraldo SB, Wheeler TD, McDermott GP, Zhu J _et al_ (2017) Massively parallel digital transcriptional profiling of single cells. _Nat Commun_ 8: 14049
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0156)
[Crossref](https://doi.org/10.1038/ncomms14049)
[CAS](https://doi.org/servlet/linkout?suffix=e_1_3_2_153_1&dbid=32&doi=10.15252%2Fmsb.20188746&key=1%3ACAS%3A528%3ADC%252BC2sXht1WlsLo%253D)
Zhou JX, Taramelli R, Pedrini E, Knijnenburg T, Huang S (2017) Extracting intercellular signaling network of cancer tissues using ligand‐receptor expression patterns from whole‐tumor and single‐cell transcriptomes. _Sci Rep_ 7: 8815
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0157)
Zhu X, Wolfgruber TK, Tasato A, Arisdakessian C, Garmire DG, Garmire LX (2017) Granatum: a graphical single‐cell RNA‐Seq analysis pipeline for genomics scientists. _Genome Med_ 9: 108
  * [a [...] ; Zhu et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0158-1)
  * [b [...] of platforms is available in Zhu et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0158-2)
  * [c [...] Platforms such as Granatum (Zhu et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0158-3)

Ziegenhain C, Vieth B, Parekh S, Reinius B, Guillaumet‐Adkins A, Smets M, Leonhardt H, Heyn H, Hellmann I, Enard W (2017) Comparative analysis of single‐cell RNA sequencing methods. _Mol Cell_ 65: 631–643.e4
[Crossref](https://doi.org/10.1016/j.molcel.2017.01.023)
  * [a [...] protocols can be found in Ziegenhain et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0159-1)
  * [b [...] ; Ziegenhain et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0159-2)

Show all references
## Information & Authors
InformationAuthors
### Information
#### Published In
[![Molecular Systems Biology cover image](https://doi.org/cms/asset/6cc440d9-ef76-4b3c-b661-75f325b10d7a/msb.v15.6.cover.jpg)Read More](https://doi.org/toc/17444292/15/6)
About the cover
Molecular Systems Biology
Vol. 15 | No. 6June 2019[Table of contents](https://doi.org/toc/17444292/15/6)
#### Submission history
**Received** : 16 November 2018
**Revision received** : 15 March 2019
**Accepted** : 3 April 2019
**Published in issue** : June 2019
**Published online** : 19 June 2019
#### Permissions
Request permissions for this article.
#### Keywords
  1. [analysis pipeline development](https://doi.org/action/doSearch?AllField=analysis+pipeline+development&stemming=yes&publication=17444292)
  2. [computational biology](https://doi.org/action/doSearch?AllField=computational+biology&stemming=yes&publication=17444292)
  3. [data analysis tutorial](https://doi.org/action/doSearch?AllField=data+analysis+tutorial&stemming=yes&publication=17444292)
  4. [single‐cell RNA‐seq](https://doi.org/action/doSearch?AllField=single%E2%80%90cell+RNA%E2%80%90seq&stemming=yes&publication=17444292)

#### Notes
 _Mol Syst Biol_. (2019) 15: e8746
#### 
© 2019 The Authors. Published under the terms of the CC BY 4.0 license. This is an open access article under the terms of the License, which permits use, distribution and reproduction in any medium, provided the original work is properly cited.
### Authors
#### AffiliationsExpand All
##### Malte D Luecken
Institute of Computational Biology Helmholtz Zentrum München German Research Center for Environmental Health Neuherberg Germany
[View all articles by this author](https://doi.org/authored-by/Luecken/Malte+D)
##### Fabian J Theis
Institute of Computational Biology Helmholtz Zentrum München German Research Center for Environmental Health Neuherberg Germany
Department of Mathematics Technische Universität München Garching bei München Germany
Corresponding author. Tel: +498931872211; E‐mail: 
[View all articles by this author](https://doi.org/authored-by/Theis/Fabian+J)
#### 
The authors declare that they have no 
#### Research Funding
[Bundesministerium für Bildung und Forschung (BMBF)](http://dx.doi.org/10.13039/501100002347): 01IS18036A, 01IS18053A
[Deutsche Forschungsgemeinschaft (DFG)](http://dx.doi.org/10.13039/501100001659): 1243, Subproject A17
[Helmholtz Association](http://dx.doi.org/10.13039/501100001656): ZT‐I‐0007
[Silicon Valley Community Foundation (Chan Zuckerberg Initiative)](http://dx.doi.org/10.13039/100000923): 182835
## Metrics & Citations
MetricsCitations
### Metrics

[View all metrics](https://doi.org/doi/metrics/10.15252/msb.20188746 "Go to metrics page")
  * [Downloads](https://doi.org/10.15252/msb.20188746 "Downloads")
  * [Citations](https://doi.org/10.15252/msb.20188746 "Citations")

No data available.
02,0004,0006,0008,00010,000Jan 2020Jan 2021Jan 2022Jan 2023Jan 2024Jan 2025
428,854
1,170
  * [Total](https://doi.org/10.15252/msb.20188746)
  * [First 90 Days](https://doi.org/10.15252/msb.20188746)
  * [6 Months](https://doi.org/10.15252/msb.20188746)
  * [12 Months](https://doi.org/10.15252/msb.20188746)

Total number of downloads and citations
**4357** readers on Mendeley 
### Citations
#### Download Citations
If you have the appropriate software installed, you can download article citation data to the citation manager of your choice. Select your manager software from the list below and click Download. 
Format
RIS (ProCite, Reference Manager) EndNote BibTex Medlars RefWorks
Direct Import
#### Citing Literature
  * Yuying Suo, Yuanli Song, Yuqiu Wang, Qian Liu, Henry Rodriguez, Hu Zhou, Advancements in proteogenomics for preclinical targeted cancer therapy research, Biophysics Reports, 10.52601/bpr.2024.240053, **11** , 1, (56), (2025).
[Crossref](https://doi.org/10.52601/bpr.2024.240053)
  * Junfei Gu, Xinjie Zhang, Qunye Zhang, Zhe Wang, Single-Cell Landscape and a Macrophage Subset Enhancing Brown Adipocyte Function in Diabetes (Diabetes Metab J 2024;48:885-900), Diabetes & Metabolism Journal, 10.4093/dmj.2024.0785, **49** , 1, (162-164), (2025).
[Crossref](https://doi.org/10.4093/dmj.2024.0785)
  * Hussein Sabit, Borros Arneth, Timothy M. Pawlik, Shaimaa Abdel-Ghany, Aysha Ghazy, Rawan M. Abdelazeem, Amany Alqosaibi, Ibtesam S. Al-Dhuayan, Jawaher Almulhim, Noof A. Alrabiah, Ahmed Hashash, Leveraging Single-Cell Multi-Omics to Decode Tumor Microenvironment Diversity and Therapeutic Resistance, Pharmaceuticals, 10.3390/ph18010075, **18** , 1, (75), (2025).
[Crossref](https://doi.org/10.3390/ph18010075)
  * Andrea Henriques-Pons, Maria Clicia S. Castro, Vanessa S. Silva, Maiana O. C. Costa, Helena S. I. L. Silva, Maria Emilia M. T. Walter, Anna Cristina C. Carvalho, Alba C. M. A. Melo, Kary Ocaña, Marcelo T. dos Santos, Marisa F. Nicolas, Fabrício A. B. Silva, Pulmonary Myeloid Cells in Mild Cases of COVID-19 Upregulate the Intracellular Fc Receptor TRIM21 and Transcribe Proteasome-Associated Molecules, International Journal of Molecular Sciences, 10.3390/ijms26062769, **26** , 6, (2769), (2025).
[Crossref](https://doi.org/10.3390/ijms26062769)
  * Shaozhuo Xie, Siyu Hou, Jiajia Chen, Xin Qi, Integrative Single-Cell and Bulk RNA Sequencing Identifies a Macrophage-Related Prognostic Signature for Predicting Prognosis and Therapy Responses in Colorectal Cancer, International Journal of Molecular Sciences, 10.3390/ijms26020811, **26** , 2, (811), (2025).
[Crossref](https://doi.org/10.3390/ijms26020811)
  * Melanie R. Nielsen, Marie Skougaard, Clara Drachmann, Zara R. Stisen, Sisse B. Ditlev, Leon E. Jessen, Lars Erik Kristensen, Single-cell RNA sequencing highlights the influence of innate and adaptive immune response mechanisms in psoriatic arthritis, Frontiers in Immunology, 10.3389/fimmu.2024.1490051, **15** , (2025).
[Crossref](https://doi.org/10.3389/fimmu.2024.1490051)
  * Mikhail Arbatsky, Ekaterina Vasilyeva, Veronika Sysoeva, Ekaterina Semina, Valeri Saveliev, Kseniya Rubina, Seurat function argument values in scRNA-seq data analysis: potential pitfalls and refinements for biological interpretation, Frontiers in Bioinformatics, 10.3389/fbinf.2025.1519468, **5** , (2025).
[Crossref](https://doi.org/10.3389/fbinf.2025.1519468)
  * Zonghao Liu, Xiaofang Zhang, Tianru Ben, Mo Li, Yi Jin, Tianlu Wang, Yingqiu Song, Focal adhesion in the tumour metastasis: from molecular mechanisms to therapeutic targets, Biomarker Research, 10.1186/s40364-025-00745-7, **13** , 1, (2025).
[Crossref](https://doi.org/10.1186/s40364-025-00745-7)
  * Xiaoqi Liang, Marni Torkel, Yue Cao, Jean Yee Hwa Yang, Multi-task benchmarking of spatially resolved gene expression simulation models, Genome Biology, 10.1186/s13059-025-03505-w, **26** , 1, (2025).
[Crossref](https://doi.org/10.1186/s13059-025-03505-w)
  * Nghia Millard, Jonathan H. Chen, Mukta G. Palshikar, Karin Pelka, Maxwell Spurrell, Colles Price, Jiang He, Nir Hacohen, Soumya Raychaudhuri, Ilya Korsunsky, Batch correcting single-cell spatial transcriptomics count data with Crescendo improves visualization and detection of spatial gene patterns, Genome Biology, 10.1186/s13059-025-03479-9, **26** , 1, (2025).
[Crossref](https://doi.org/10.1186/s13059-025-03479-9)

## View Options
### View options
####  PDF
## Figures
[Open all in viewer](https://doi.org/10.15252/msb.20188746#figures)
[![image](https://doi.org/cms/10.15252/msb.20188746/asset/5bcc5719-49b9-4156-a754-76a2448e5392/assets/graphic/msb188746-fig-0001-m.png)](https://doi.org/10.15252/msb.20188746#msb188746-fig-0001)
Figure 1. Schematic of a typical single‐cell RNA‐seq analysis workflow
Raw sequencing data are processed and aligned to give count matrices, which represent the start of the workflow. The count data undergo pre‐processing and downstream analysis. Subplots are generated using the best‐practices workflow on intestinal epithelium data from Haber _et al_ ([2017](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0046)).
  * [Download figure](https://doi.org/cms/asset/5bcc5719-49b9-4156-a754-76a2448e5392/msb188746-fig-0001-m.png)
  * [Download PowerPoint](https://doi.org/action/downloadFigures?doi=10.15252%2Fmsb.20188746&id=msb188746-fig-0001)

[Go to Figure](https://doi.org/10.15252/msb.20188746#msb188746-fig-0001)[Open in Viewer](https://doi.org/10.15252/msb.20188746#msb188746-fig-0001)
[![image](https://doi.org/cms/10.15252/msb.20188746/asset/1bd082a4-169f-4878-9161-5cab859d983c/assets/graphic/msb188746-fig-0002-m.png)](https://doi.org/10.15252/msb.20188746#msb188746-fig-0002)
Figure 2. Plots of quality control metrics with filtering decisions for a mouse intestinal epithelium dataset from Haber _et al_ ([2017](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0046))
(A) Histograms of count depth per cell. The smaller histogram is zoomed‐in on count depths below 4,000. A threshold is applied here at 1,500 based on the peak detected at around 1,200 counts. (B) Histogram of the number of genes detected per cell. A small noise peak is visible at approx. 400 genes. These cells are filtered out using the depicted threshold (red line) at 700 genes. (C) Count depth distribution from high to low count depths. This visualization is related to the log–log plot shown in Cell Ranger outputs that is used to filter out empty droplets. It shows an “elbow” where count depths start to decrease rapidly around 1,500 counts. (D) Number of genes versus the count depth coloured by the fraction of mitochondrial reads. Mitochondrial read fractions are only high in particularly low count cells with few detected genes. These cells are filtered out by our count and gene number thresholds. Jointly visualizing the count and gene thresholds shows the joint filtering effect, indicating that a lower gene threshold may have sufficed. [Correction added on 5 July 2019, after first online publication: the x‐axis label in panel B was corrected from “Count depth” to “Number of genes”.]
  * [Download figure](https://doi.org/cms/asset/1bd082a4-169f-4878-9161-5cab859d983c/msb188746-fig-0002-m.png)
  * [Download PowerPoint](https://doi.org/action/downloadFigures?doi=10.15252%2Fmsb.20188746&id=msb188746-fig-0002)

[Go to Figure](https://doi.org/10.15252/msb.20188746#msb188746-fig-0002)[Open in Viewer](https://doi.org/10.15252/msb.20188746#msb188746-fig-0002)
[![image](https://doi.org/cms/10.15252/msb.20188746/asset/0ed29c9f-28ff-40f8-a712-110306de12dc/assets/graphic/msb188746-fig-0003-m.png)](https://doi.org/10.15252/msb.20188746#msb188746-fig-0003)
Figure 3. UMAP visualization before and after batch correction
Cells are coloured by sample of origin. Separation of batches is clearly visible before batch correction and less visible afterwards. Batch correction was performed using ComBat on mouse intestinal epithelium data from Haber _et al_ ([2017](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0046)).
  * [Download figure](https://doi.org/cms/asset/0ed29c9f-28ff-40f8-a712-110306de12dc/msb188746-fig-0003-m.png)
  * [Download PowerPoint](https://doi.org/action/downloadFigures?doi=10.15252%2Fmsb.20188746&id=msb188746-fig-0003)

[Go to Figure](https://doi.org/10.15252/msb.20188746#msb188746-fig-0003)[Open in Viewer](https://doi.org/10.15252/msb.20188746#msb188746-fig-0003)
[![image](https://doi.org/cms/10.15252/msb.20188746/asset/40988faf-3bbb-4dc7-ba72-751c5fb2f775/assets/graphic/msb188746-fig-0001ev-m.png)](https://doi.org/10.15252/msb.20188746#msb188746-fig-0001ev)
Figure EV1. The number of highly variable genes (HVGs) used for datasets of different sizes
The data were obtained by a brief manual survey of recent scRNA‐seq analysis papers. The plotted data, along with further information on scRNA‐seq technology, publication year, reference and the number of reads per cell, are available in [Dataset EV1](https://doi.org/10.15252/msb.20188746#msb188746-sup-0003).
  * [Download figure](https://doi.org/cms/asset/40988faf-3bbb-4dc7-ba72-751c5fb2f775/msb188746-fig-0001ev-m.png)
  * [Download PowerPoint](https://doi.org/action/downloadFigures?doi=10.15252%2Fmsb.20188746&id=msb188746-fig-0001ev)

[Go to Figure](https://doi.org/10.15252/msb.20188746#msb188746-fig-0001ev)[Open in Viewer](https://doi.org/10.15252/msb.20188746#msb188746-fig-0001ev)
[![image](https://doi.org/cms/10.15252/msb.20188746/asset/8e97ce5f-5e00-4b4c-981b-fe59d71dd30b/assets/graphic/msb188746-fig-0004-m.png)](https://doi.org/10.15252/msb.20188746#msb188746-fig-0004)
Figure 4. Common visualization methods for scRNA‐seq data
Mouse intestinal epithelium regions data from Haber _et al_ ([2017](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0046)) visualized on the first two components for: (A) PCA, (B) t‐SNE, (C) diffusion maps, (D) UMAP and (E) A force‐directed graph layout via ForceAtlas2. Cells are coloured by count depth. (F) Variance explained by the first 31 principal components (PCs). The “elbow” of this plot, which is used to select relevant PCs to analyse the dataset, lies between PCs 5 and 7.
  * [Download figure](https://doi.org/cms/asset/8e97ce5f-5e00-4b4c-981b-fe59d71dd30b/msb188746-fig-0004-m.png)
  * [Download PowerPoint](https://doi.org/action/downloadFigures?doi=10.15252%2Fmsb.20188746&id=msb188746-fig-0004)

[Go to Figure](https://doi.org/10.15252/msb.20188746#msb188746-fig-0004)[Open in Viewer](https://doi.org/10.15252/msb.20188746#msb188746-fig-0004)
[![image](https://doi.org/cms/10.15252/msb.20188746/asset/d6585605-fb7f-4bea-8276-71309cfcd3fb/assets/graphic/msb188746-fig-0002ev-m.png)](https://doi.org/10.15252/msb.20188746#msb188746-fig-0002ev)
Figure EV2. Change in coefficient of variation (CoV) of gene expression data upon batch correction and denoising
Negative values represent a reduction in CoV upon data correction. The top row shows CoV changes upon ComBat batch correction for (A) mouse intestinal epithelium (mIE) and (B) mouse embryonic stem cell (mESC) data. The lower row depicts CoV changes upon DCA denoising for (C) mIE and (D) mESC data. mIE data were obtained from Haber _et al_ ([2017](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0046)) and mESC from Klein _et al_ ([2015](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0065)).
  * [Download figure](https://doi.org/cms/asset/d6585605-fb7f-4bea-8276-71309cfcd3fb/msb188746-fig-0002ev-m.png)
  * [Download PowerPoint](https://doi.org/action/downloadFigures?doi=10.15252%2Fmsb.20188746&id=msb188746-fig-0002ev)

[Go to Figure](https://doi.org/10.15252/msb.20188746#msb188746-fig-0002ev)[Open in Viewer](https://doi.org/10.15252/msb.20188746#msb188746-fig-0002ev)
[![image](https://doi.org/cms/10.15252/msb.20188746/asset/5593cfbd-38ec-43e5-96e8-9beaa495c0c2/assets/graphic/msb188746-fig-0005-m.png)](https://doi.org/10.15252/msb.20188746#msb188746-fig-0005)
Figure 5. Overview of downstream analysis methods
Methods are divided into cell‐ and gene‐level analysis. Cell‐level analysis approaches are again subdivided into cluster and trajectory analysis branches, which include also gene‐level analysis methods. All methods with a blue background are gene‐level approaches.
  * [Download figure](https://doi.org/cms/asset/5593cfbd-38ec-43e5-96e8-9beaa495c0c2/msb188746-fig-0005-m.png)
  * [Download PowerPoint](https://doi.org/action/downloadFigures?doi=10.15252%2Fmsb.20188746&id=msb188746-fig-0005)

[Go to Figure](https://doi.org/10.15252/msb.20188746#msb188746-fig-0005)[Open in Viewer](https://doi.org/10.15252/msb.20188746#msb188746-fig-0005)
[![image](https://doi.org/cms/10.15252/msb.20188746/asset/62f76211-6a38-42ed-b151-b38d246b897e/assets/graphic/msb188746-fig-0006-m.png)](https://doi.org/10.15252/msb.20188746#msb188746-fig-0006)
Figure 6. Cluster analysis results of mouse intestinal epithelium dataset from Haber _et al_ ([2017](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0046))
(A) Annotated cell‐identity clusters found by Louvain clustering visualized in a UMAP representation. (B) Cell‐identity marker expression to identify stem cells (Slc12a2), enterocytes (Arg2), goblet cells (Tff3) and Paneth cells (Defa24). Corrected expression levels are visualized from low expression (grey) to high expression (red). Marker genes may be expressed also in other cell‐identity populations as shown for goblet and Paneth cells. (C) Cell‐identity composition heat maps of proximal (upper) and distal (lower) intestinal epithelium regions. High relative cell density is shown as dark red.
  * [Download figure](https://doi.org/cms/asset/62f76211-6a38-42ed-b151-b38d246b897e/msb188746-fig-0006-m.png)
  * [Download PowerPoint](https://doi.org/action/downloadFigures?doi=10.15252%2Fmsb.20188746&id=msb188746-fig-0006)

[Go to Figure](https://doi.org/10.15252/msb.20188746#msb188746-fig-0006)[Open in Viewer](https://doi.org/10.15252/msb.20188746#msb188746-fig-0006)
[![image](https://doi.org/cms/10.15252/msb.20188746/asset/13e5de7d-250b-4715-8ab8-5dcb3eea526d/assets/graphic/msb188746-fig-0007-m.png)](https://doi.org/10.15252/msb.20188746#msb188746-fig-0007)
Figure 7. Trajectory analysis and graph abstraction of mouse intestinal epithelium data from Haber _et al_ ([2017](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0046))
(A) Distal and proximal enterocyte differentiation trajectories inferred by Slingshot. The Distal lineage is shown coloured by pseudotime from red to blue. Other cells in the dataset are grey. (B) Slingshot trajectories over clusters in PCA space. Clusters are abbreviated as follows: EP—enterocyte progenitors; Imm. Ent.—immature enterocytes; Mat. Ent.—mature enterocytes; Prox.—proximal; Dist.—distal. (C) Density over pseudotime for the distal enterocyte trajectory from Fig [7](https://doi.org/10.15252/msb.20188746#msb188746-fig-0007)A. Colours represent the dominant cluster labels in each pseudotime bin. (D) Abstracted graph representation of the dataset projected onto a UMAP representation. Clusters are shown as coloured nodes. Clusters that appear in other trajectories are labelled for comparison. “TA” denotes transit amplifying cells. (E) Gene expression dynamics over pseudotime in a general enterocyte trajectory using the “GAM” R library.
  * [Download figure](https://doi.org/cms/asset/13e5de7d-250b-4715-8ab8-5dcb3eea526d/msb188746-fig-0007-m.png)
  * [Download PowerPoint](https://doi.org/action/downloadFigures?doi=10.15252%2Fmsb.20188746&id=msb188746-fig-0007)

[Go to Figure](https://doi.org/10.15252/msb.20188746#msb188746-fig-0007)[Open in Viewer](https://doi.org/10.15252/msb.20188746#msb188746-fig-0007)
## Tables
[Open all in viewer](https://doi.org/10.15252/msb.20188746#tables)
Table 1. Stages of data processing and appropriate downstream applications
[Go to Table](https://doi.org/10.15252/msb.20188746#msb188746-tbl-0001)[Open in Viewer](https://doi.org/10.15252/msb.20188746#msb188746-tbl-0001)
## Media
## Share
### Share
#### Copy the content Link
https://www.embopress.org/doi/full/10.15252/msb.20188746
Copy Link
Copied!
Copying failed.
#### Share on social media
[Facebook](https://doi.org/#facebook "Share on Facebook")[X (formerly Twitter)](https://doi.org/#twitter "Share on X \(formerly Twitter\)")[LinkedIn](https://doi.org/#linkedin "Share on LinkedIn")[Reddit](https://doi.org/#reddit "Share on Reddit")[WeChat](https://doi.org/#wechat "Share on WeChat")[email](https://doi.org/#email "Share on email")
## References
### References
Aibar S, González‐Blas CB, Moerman T, Huynh‐Thu VA, Imrichova H, Hulselmans G, Rambow F, Marine J‐C, Geurts P, Aerts J _et al_ (2017) SCENIC: single‐cell regulatory network inference and clustering. _Nat Methods_ 14: 1083–1086
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0001)
[Crossref](https://doi.org/10.1038/nmeth.4463)
[CAS](https://doi.org/servlet/linkout?suffix=e_1_3_2_2_1&dbid=32&doi=10.15252%2Fmsb.20188746&key=1%3ACAS%3A528%3ADC%252BC2sXhs1aitL7P)
Alpert A, Moore LS, Dubovik T, Shen‐Orr SS (2018) Alignment of single‐cell trajectories to compare cellular expression dynamics. _Nat Methods_ 15: 267–270
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0002)
[Crossref](https://doi.org/10.1038/nmeth.4628)
An Y, Furber KL, Ji S (2017) Pseudogenes regulate parental gene expression via ceRNA network. _J Cell Mol Med_ 21: 185–192
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0003)
Andrews TS, Hemberg M (2018) False signals induced by single‐cell imputation. _F1000Res_ 7: 1740
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0004)
Angelidis I, Simon LM, Fernandez IE, Strunz M, Mayr CH, Greiffo FR, Tsitsiridis G, Graf E, Strom TM, Eickelberg O _et al_ (2019) An atlas of the aging lung mapped by single cell transcriptomics and deep tissue proteomics. _Nat Commun_ 10: 963
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0005)
Angerer P, Simon L, Tritschler S, Alexander Wolf F, Fischer D, Theis FJ (2017) Single cells make big data: new challenges and opportunities in transcriptomics. _Curr Opin Syst Biol_ 4: 85–91
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0006)
Argelaguet R, Velten B, Arnol D, Dietrich S, Zenz T, Marioni JC, Buettner F, Huber W, Stegle O (2018) Multi‐Omics Factor Analysis—a framework for unsupervised integration of multi‐omics data sets. _Mol Syst Biol_ 14: e8124
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0007)
[Crossref](https://doi.org/servlet/linkout?suffix=e_1_3_2_8_1&dbid=4&doi=10.15252%2Fmsb.20188746&key=10.15252%2Fmsb.20178124&site=embo-site)
Arvaniti E, Claassen M (2017) Sensitive detection of rare disease‐associated cell subsets via representation learning. _Nat Commun_ 8: 14825
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0008)
Ashburner M, Ball CA, Blake JA, Botstein D, Butler H, Cherry JM, Davis AP, Dolinski K, Dwight SS, Eppig JT _et al_ (2000) Gene ontology: tool for the unification of biology. The Gene Ontology Consortium. _Nat Genet_ 25: 25–29
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0009)
[Crossref](https://doi.org/10.1038/75556)
[CAS](https://doi.org/servlet/linkout?suffix=e_1_3_2_10_1&dbid=32&doi=10.15252%2Fmsb.20188746&key=1%3ACAS%3A528%3ADC%252BD3cXjtFSlsbc%253D)
Azizi E, Carr AJ, Plitas G, Cornish AE, Konopacki C, Prabhakaran S, Nainys J, Wu K, Kiseliovas V, Setty M _et al_ (2018) Single‐cell map of diverse immune phenotypes in the breast tumor microenvironment. _Cell_ 174: 1293–1308.e36
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0010)
Becht E, McInnes L, Healy J, Dutertre C‐A, Kwok IWH, Ng LG, Ginhoux F, Newell EW (2018) Dimensionality reduction for visualizing single‐cell data using UMAP. _Nat Biotechnol_ 37: 38–44
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0011)
[Crossref](https://doi.org/10.1038/nbt.4314)
Bendall SC, Davis KL, Amir E‐AD, Tadmor MD, Simonds EF, Chen TJ, Shenfeld DK, Nolan GP, Pe'er D (2014) Single‐cell trajectory detection uncovers progression and regulatory coordination in human B cell development. _Cell_ 157: 714–725
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0012)
[Crossref](https://doi.org/10.1016/j.cell.2014.04.005)
[CAS](https://doi.org/servlet/linkout?suffix=e_1_3_2_13_1&dbid=32&doi=10.15252%2Fmsb.20188746&key=1%3ACAS%3A528%3ADC%252BC2cXntFGgsr8%253D)
Blasi T, Buettner F, Strasser MK, Marr C, Theis FJ (2017) cgCorrect: a method to correct for confounding cell‐cell variation due to cell growth in single‐cell transcriptomics. _Phys Biol_ 14: 036001
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0013)
Blondel VD, Guillaume J‐L, Lambiotte R, Lefebvre E (2008) Fast unfolding of communities in large networks. _J Stat Mech: Theory Exp_ 2008: P10008
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0014)
[Crossref](https://doi.org/10.1088/1742-5468/2008/10/P10008)
Brennecke P, Anders S, Kim JK, Kołodziejczyk AA, Zhang X, Proserpio V, Baying B, Benes V, Teichmann SA, Marioni JC _et al_ (2013) Accounting for technical noise in single‐cell RNA‐seq experiments. _Nat Methods_ 10: 1093–1095
[Crossref](https://doi.org/10.1038/nmeth.2645)
[CAS](https://doi.org/servlet/linkout?suffix=e_1_3_2_16_1&dbid=32&doi=10.15252%2Fmsb.20188746&key=1%3ACAS%3A528%3ADC%252BC3sXhsVyqtb3L)
  * [a [...] in single‐cell data (Brennecke et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0015-1)
  * [b [...] (HVGs) are often used (Brennecke et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0015-2)

Briggs JA, Weinreb C, Wagner DE, Megason S, Peshkin L, Kirschner MW, Klein AM (2018) The dynamics of gene expression in vertebrate embryogenesis at single‐cell resolution. _Science_ 360: eaar5780
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0016)
Buenrostro JD, Wu B, Litzenburger UM, Ruff D, Gonzales ML, Snyder MP, Chang HY, Greenleaf WJ (2015) Single‐cell chromatin accessibility reveals principles of regulatory variation. _Nature_ 523: 486–490
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0017)
[Crossref](https://doi.org/10.1038/nature14590)
[CAS](https://doi.org/servlet/linkout?suffix=e_1_3_2_18_1&dbid=32&doi=10.15252%2Fmsb.20188746&key=1%3ACAS%3A528%3ADC%252BC2MXhtFyltLrN)
Buettner F, Natarajan KN, Casale FP, Proserpio V, Scialdone A, Theis FJ, Teichmann SA, Marioni JC, Stegle O (2015) Computational analysis of cell‐to‐cell heterogeneity in single‐cell RNA‐sequencing data reveals hidden subpopulations of cells. _Nat Biotechnol_ 33: 155–160
[Crossref](https://doi.org/10.1038/nbt.3102)
[CAS](https://doi.org/servlet/linkout?suffix=e_1_3_2_19_1&dbid=32&doi=10.15252%2Fmsb.20188746&key=1%3ACAS%3A528%3ADC%252BC2MXhtFKjsLs%253D)
  * [a [...] models such as scLVM (Buettner et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0018-1)
  * [b [...] developmental trajectories (Buettner et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0018-2)
  * [c [...] during maturation (Buettner et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0018-3)

Buettner F, Pratanwanich N, McCarthy DJ, Marioni JC, Stegle O (2017) f‐scLVM: scalable and versatile factor analysis for single‐cell RNA‐seq. _Genome Biol_ 18: 212
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0019)
Butler A, Hoffman P, Smibert P, Papalexi E, Satija R (2018) Integrating single‐cell transcriptomic data across different conditions, technologies, and species. _Nat Biotechnol_ 36: 411–420
[Crossref](https://doi.org/10.1038/nbt.4096)
[CAS](https://doi.org/servlet/linkout?suffix=e_1_3_2_21_1&dbid=32&doi=10.15252%2Fmsb.20188746&key=1%3ACAS%3A528%3ADC%252BC1cXmslKrtL0%253D)
  * [a [...] platforms such as Seurat (Butler et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0020-1)
  * [b [...] the popular Seurat tutorials (Butler et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0020-2)
  * [c [...] Scanpy and Seurat platforms (Butler et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0020-3)
  * [d [...] Correlation Analysis (CCA; Butler et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0020-4)
  * [e [...] ; Butler et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0020-5)
  * [f [...] ) and Seurat (Butler et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0020-6)

Buttner M, Miao Z, Wolf A, Teichmann SA, Theis FJ (2019) A test metric for assessing single‐cell RNA‐seq batch correction. _Nat Methods_ 16: 43–49
  * [a [...] methods for batch correction (Buttner et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0021-1)
  * [b [...] ; Buttner et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0021-2)
  * [c [...] of low‐to‐medium complexity (Buttner et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0021-3)
  * [d [...] batch correction setting (Buttner et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0021-4)
  * [e [...] and normalization steps (Buttner et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0021-5)

Chan TE, Stumpf MPH, Babtie AC (2017) Gene regulatory network inference from single‐cell data using multivariate information measures. _Cell Syst_ 5: 251–267.e3
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0022)
[Crossref](https://doi.org/10.1016/j.cels.2017.08.014)
Chen S, Mar JC (2018) Evaluating methods of inferring gene regulatory networks highlights their lack of performance for single cell gene expression data. _BMC Bioinformatics_ 19: 232
[Crossref](https://doi.org/10.1186/s12859-018-2217-z)
  * [a [...] or via regression models (Chen & Mar, ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0023-1)
  * [b [...] perform poorly on these data (Chen & Mar, ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0023-2)

Chung NC, Storey JD (2015) Statistical significance of variables driving systematic variation in high‐dimensional data. _Bioinformatics_ 31: 545–554
  * [a [...] jackstraw method (Chung & Storey, ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0024-1)
  * [b [...] of genes in the dataset (Chung & Storey, ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0024-2)

Clevers H, Rafelski S, Elowitz M, Klein AM, Shendure J, Trapnell C, Lein E, Lundberg E, Uhlén M, Martinez‐Arias A _et al_ (2017) What is your conceptual definition of ‘cell type’ in the context of a mature organism? _Cell Syst_ 4: 255–259
  * [a [...] ; Clevers et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0025-1)
  * [b [...] ; Clevers et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0025-2)

Cohen M, Giladi A, Gorki A‐D, Solodkin DG, Zada M, Hladik A, Miklosi A, Salame T‐M, Halpern KB, David E _et al_ (2018) Lung single‐cell signaling interaction map reveals basophil role in macrophage imprinting. _Cell_ 175: 1031–1044
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0026)
Coifman RR, Lafon S, Lee AB, Maggioni M, Nadler B, Warner F, Zucker SW (2005) Geometric diffusions as a tool for harmonic analysis and structure definition of data: diffusion maps. _Proc Natl Acad Sci USA_ 102: 7426–7431
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0027)
Cole MB, Risso D, Wagner A, DeTomaso D, Ngai J, Purdom E, Dudoit S, Yosef N (2019) Performance assessment and selection of normalization procedures for single‐cell RNA‐seq. _Cell Syst_ 8: 315–328
  * [a [...] unwanted variation, also exist (Cole et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0028-1)
  * [b [...] with strong batch effects (Cole et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0028-2)
  * [c [...] fit by different models. Indeed Cole et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0028-3)

Colomé‐Tatché M, Theis FJ (2018) Statistical single cell multi‐omics integration. _Curr Opin Syst Biol_ 7: 54–59
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0029)
DePasquale EAK, Schnell DJ, Valiente I, Blaxall BC, Grimes HL, Singh H, Salomonis N (2018) DoubletDecon: Cell‐State Aware Removal of Single‐Cell RNA‐Seq Doublets. _bioRxiv_ <https://doi.org/10.1101/364810> [PREPRINT]
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0030)
van Dijk D, Sharma R, Nainys J, Yim K, Kathail P, Carr AJ, Burdziak C, Moon KR, Chaffer CL, Pattabiraman D _et al_ (2018) Recovering gene interactions from single‐cell data using data diffusion. _Cell_ 174: 716–729.e27
[Crossref](https://doi.org/10.1016/j.cell.2018.05.061)
[CAS](https://doi.org/servlet/linkout?suffix=e_1_3_2_32_1&dbid=32&doi=10.15252%2Fmsb.20188746&key=1%3ACAS%3A528%3ADC%252BC1cXht1Cis7rE)
  * [a [...] several recent tools (MAGIC: van Dijk et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0031-1)
  * [b [...] of gene–gene correlations (van Dijk et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0031-2)

Dillies M‐A, Rau A, Aubert J, Hennequet‐Antier C, Jeanmougin M, Servant N, Keime C, Marot G, Castel D, Estelle J _et al_ (2013) A comprehensive evaluation of normalization methods for Illumina high‐throughput RNA sequencing data analysis. _Brief Bioinform_ 14: 671–683
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0032)
[Crossref](https://doi.org/10.1093/bib/bbs046)
Ding J, Condon A, Shah SP (2018) Interpretable dimensionality reduction of single cell transcriptome data with deep generative models. _Nat Commun_ 9: 2002
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0033)
[Crossref](https://doi.org/10.1038/s41467-018-04368-5)
Duò A, Robinson MD, Soneson C (2018) A systematic performance evaluation of clustering methods for single‐cell RNA‐seq data. _F1000Res_ 7: 1141.
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0034)
Eraslan G, Simon LM, Mircea M, Mueller NS, Theis FJ (2019) Single cell RNA‐seq denoising using a deep count autoencoder. _Nat Commun_ 10: 390
  * [a [...] ; DCA: Eraslan et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0035-1)
  * [b [...] ; Eraslan et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0035-2)
  * [c [...] ; DCA: Eraslan et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0035-3)

Fabregat A, Jupe S, Matthews L, Sidiropoulos K, Gillespie M, Garapati P, Haw R, Jassal B, Korninger F, May B _et al_ (2018) The reactome pathway knowledgebase. _Nucleic Acids Res_ 46: D649–D655
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0036)
[Crossref](https://doi.org/10.1093/nar/gkx1132)
Finak G, McDavid A, Yajima M, Deng J, Gersuk V, Shalek AK, Slichter CK, Miller HW, Juliana McElrath M, Prlic M _et al_ (2015) MAST: a flexible statistical framework for assessing transcriptional changes and characterizing heterogeneity in single‐cell RNA sequencing data. _Genome Biol_ 16: 278
[Crossref](https://doi.org/10.1186/s13059-015-0844-5)
  * [a [...] expression testing (Finak et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0037-1)
  * [b [...] ; Finak et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0037-2)
  * [c [...] the single‐cell tool MAST (Finak et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0037-3)

Fischer DS, Theis FJ, Yosef N (2018) Impulse model‐based differential expression analysis of time course sequencing data. _Nucleic Acids Res_ 46: e119
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0038)
Freytag S, Tian L, Lönnstedt I, Ng M, Bahlo M (2018) Comparison of clustering tools in R for medium‐sized 10x Genomics single‐cell RNA‐sequencing data. _F1000Res_ 7: 1297
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0039)
Gardeux V, David FPA, Shajkofci A, Schwalie PC, Deplancke B (2017) ASAP: a web‐based platform for the analysis and interactive visualization of single‐cell RNA‐seq data. _Bioinformatics_ 33: 3123–3125
  * [a [...] ) or Web servers (Gardeux et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0040-1)
  * [b [...] ) and ASAP (Gardeux et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0040-2)

Gashaw I, Ellinghaus P, Sommer A, Asadullah K (2012) What makes a good drug target?. _Drug Discov Today_ 17: S24–S30
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0041)
Gehring J, Park JH, Chen S, Thomson M, Pachter L (2018) Highly Multiplexed Single‐Cell RNA‐seq for Defining Cell Population and Transcriptional Spaces. _bioRxiv_ <https://doi.org/10.1101/315333> [PREPRINT]
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0042)
Gloor GB, Macklaim JM, Pawlowsky‐Glahn V, Egozcue JJ (2017) Microbiome datasets are compositional: and this is not optional. _Front Microbiol_ 8: 2224
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0043)
Griffiths JA, Scialdone A, Marioni JC (2018) Using single‐cell genomics to understand developmental processes and cell fate decisions. _Mol Syst Biol_ 14: e8046
[Crossref](https://doi.org/servlet/linkout?suffix=e_1_3_2_45_1&dbid=4&doi=10.15252%2Fmsb.20188746&key=10.15252%2Fmsb.20178046&site=embo-site)
  * [a [...] ; Griffiths et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0044-1)
  * [b [...] ; Griffiths et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0044-2)
  * [c [...] uncertainty in their model (Griffiths et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0044-3)

Grün D, Kester L, van Oudenaarden A (2014) Validation of noise models for single‐cell transcriptomics. _Nat Methods_ 11: 637
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0045)
[Crossref](https://doi.org/10.1038/nmeth.2930)
[CAS](https://doi.org/servlet/linkout?suffix=e_1_3_2_46_1&dbid=32&doi=10.15252%2Fmsb.20188746&key=1%3ACAS%3A528%3ADC%252BC2cXmsVKlsLc%253D)
Haber AL, Biton M, Rogel N, Herbst RH, Shekhar K, Smillie C, Burgin G, Delorey TM, Howitt MR, Katz Y _et al_ (2017) A single‐cell survey of the small intestinal epithelium. _Nature_ 551: 333–339
[Crossref](https://doi.org/10.1038/nature24489)
[CAS](https://doi.org/servlet/linkout?suffix=e_1_3_2_47_1&dbid=32&doi=10.15252%2Fmsb.20188746&key=1%3ACAS%3A528%3ADC%252BC2sXhslygtLjK)
  * [a [...] intestinal epithelium data from Haber et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0046-1)
  * [b [...] epithelium dataset from Haber et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0046-2)
  * [c [...] intestinal epithelium data from Haber et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0046-3)
  * [d [...] epithelium regions data from Haber et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0046-4)
  * [e [...] data were obtained from Haber et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0046-5)
  * [f [...] epithelium dataset from Haber et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0046-6)
  * [g [...] mouse intestinal epithelium (Haber et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0046-7)
  * [h [...] intestinal epithelium data from Haber et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0046-8)

Haghverdi L, Buettner F, Theis FJ (2015) Diffusion maps for high‐dimensional single‐cell analysis of differentiation data. _Bioinformatics_ 31: 2989–2998
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0047)
Haghverdi L, Büttner M, Wolf FA, Buettner F, Theis FJ (2016) Diffusion pseudotime robustly reconstructs lineage branching. _Nat Methods_ 13: 845–848
[Crossref](https://doi.org/10.1038/nmeth.3971)
  * [a [...] ; Haghverdi et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0048-1)
  * [b [...] along a trajectory (Haghverdi et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0048-2)
  * [c [...] for example, development (Haghverdi et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0048-3)

Haghverdi L, Lun ATL, Morgan MD, Marioni JC (2018) Batch effects in single‐cell RNA‐sequencing data are corrected by matching mutual nearest neighbors. _Nat Biotechnol_ 36: 421–427
[Crossref](https://doi.org/10.1038/nbt.4091)
  * [a [...] Nearest Neighbours (MNN; Haghverdi et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0049-1)
  * [b [...] include cosine similarity (Haghverdi et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0049-2)

Heimberg G, Bhatnagar R, El‐Samad H, Thomson M (2016) Low dimensionality in gene expression data enables the accurate extraction of transcriptional programs from shallow sequencing. _Cell Syst_ 2: 239–250
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0050)
Hicks SC, Townes FW, Teng M, Irizarry RA (2017) Missing data and technical variability in single‐cell RNA‐sequencing experiments. _Biostatistics_ 19: 562–578
  * [a [...] by clever experimental design (Hicks et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0051-1)
  * [b [...] ; Hicks et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0051-2)
  * [c [...] high cell‐to‐cell variability (Hicks et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0051-3)

Hie BL, Bryson B, Berger B (2018) Panoramic stitching of heterogeneous single‐cell transcriptomic data. _bioRxiv_ <https://doi.org/10.1101/371179> [PREPRINT]
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0052)
Huang DW, Sherman BT, Lempicki RA (2009) Bioinformatics enrichment tools: paths toward the comprehensive functional analysis of large gene lists. _Nucleic Acids Res_ 37: 1–13
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0053)
[Crossref](https://doi.org/10.1093/nar/gkn923)
Huang M, Wang J, Torre E, Dueck H, Shaffer S, Bonasio R, Murray JI, Raj A, Li M, Zhang NR (2018) SAVER: gene expression recovery for single‐cell RNA sequencing. _Nat Methods_ 15: 539–542
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0054)
Huber W, Carey VJ, Gentleman R, Anders S, Carlson M, Carvalho BS, Bravo HC, Davis S, Gatto L, Girke T _et al_ (2015) Orchestrating high‐throughput genomic analysis with Bioconductor. _Nat Methods_ 12: 115–121
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0055)
[Crossref](https://doi.org/10.1038/nmeth.3252)
[CAS](https://doi.org/servlet/linkout?suffix=e_1_3_2_56_1&dbid=32&doi=10.15252%2Fmsb.20188746&key=1%3ACAS%3A528%3ADC%252BC2MXjvVCrurg%253D)
Ilicic T, Kim JK, Kolodziejczyk AA, Bagger FO, McCarthy DJ, Marioni JC, Teichmann SA (2016) Classification of low quality cells from single‐cell RNA‐seq data. _Genome Biol_ 17: 29
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0056)
[Crossref](https://doi.org/10.1186/s13059-016-0888-1)
Johnson WE, Li C, Rabinovic A (2006) Adjusting batch effects in microarray expression data using empirical Bayes methods. _Biostatistics_ 8: 118–127
  * [a [...] ) or batch correction (Johnson et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0058-1)
  * [b [...] has revealed that ComBat (Johnson et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0058-2)

Kanehisa M, Furumichi M, Tanabe M, Sato Y, Morishima K (2017) KEGG: new perspectives on genomes, pathways, diseases and drugs. _Nucleic Acids Res_ 45: D353–D361
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0059)
[Crossref](https://doi.org/10.1093/nar/gkw1092)
[CAS](https://doi.org/servlet/linkout?suffix=e_1_3_2_59_1&dbid=32&doi=10.15252%2Fmsb.20188746&key=1%3ACAS%3A528%3ADC%252BC1cXhslWgsrw%253D)
Kang HM, Subramaniam M, Targ S, Nguyen M, Maliskova L, McCarthy E, Wan E, Wong S, Byrnes L, Lanata CM _et al_ (2018) Multiplexed droplet single‐cell RNA‐sequencing using natural genetic variation. _Nat Biotechnol_ 36: 89–94
[Crossref](https://doi.org/10.1038/nbt.4042)
  * [a [...] ), or via genetic variation (Kang et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0060-1)
  * [b [...] experimental conditions (Kang et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0060-2)

Kharchenko PV, Silberstein L, Scadden DT (2014) Bayesian approach to single‐cell differential expression analysis. _Nat Methods_ 11: 740–742
[Crossref](https://doi.org/10.1038/nmeth.2967)
[CAS](https://doi.org/servlet/linkout?suffix=e_1_3_2_61_1&dbid=32&doi=10.15252%2Fmsb.20188746&key=1%3ACAS%3A528%3ADC%252BC2cXotFCjs70%253D)
  * [a [...] ; Kharchenko et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0061-1)
  * [b [...] for single‐cell data (Kharchenko et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0061-2)

Kim T, Chen IR, Lin Y, Wang AY‐Y, Yang JYH, Yang P (2018) Impact of similarity metrics on single‐cell RNA‐seq data clustering. _Brief Bioinform_ <https://doi.org/10.1093/bib/bby076>
  * [a [...] distance metrics (Kim et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0062-1)
  * [b [...] the basis for Gaussian kernels (Kim et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0062-2)

Kiselev VY, Yiu A, Hemberg M (2018b) Scmap: projection of single‐cell RNA‐seq data across data sets. _Nat Methods_ 15: 359–362
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0064)
[Crossref](https://doi.org/10.1038/nmeth.4644)
Klein AM, Mazutis L, Akartuna I, Tallapragada N, Veres A, Li V, Peshkin L, Weitz DA, Kirschner MW (2015) Droplet barcoding for single‐cell transcriptomics applied to embryonic stem cells. _Cell_ 161: 1187–1201
[Crossref](https://doi.org/10.1016/j.cell.2015.04.044)
[CAS](https://doi.org/servlet/linkout?suffix=e_1_3_2_64_1&dbid=32&doi=10.15252%2Fmsb.20188746&key=1%3ACAS%3A528%3ADC%252BC2MXpt1SgtL0%253D)
  * [a [...] ), indrops (Klein et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0065-1)
  * [b [...] ). Preliminary results from Klein et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0065-2)
  * [c [...] from Klein et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0065-3)

Korsunsky I, Fan J, Slowikowski K, Zhang F, Wei K (2018) Fast, sensitive, and accurate integration of single cell data with Harmony. _bioRxiv_ <https://doi.org/10.1101/461954> [PREPRINT]
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0066)
Kowalczyk MS, Tirosh I, Heckl D, Rao TN, Dixit A, Haas BJ, Schneider RK, Wagers AJ, Ebert BL, Regev A (2015) Single‐cell RNA‐seq reveals changes in cell cycle and differentiation programs upon aging of hematopoietic stem cells. _Genome Res_ 25: 1860–1872
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0067)
[Crossref](https://doi.org/10.1101/gr.192237.115)
La Manno G, Soldatov R, Zeisel A, Braun E, Hochgerner H, Petukhov V, Lidschreiber K, Kastriti ME, Lönnerberg P, Furlan A _et al_ (2018) RNA velocity of single cells. _Nature_ 560: 494–498
[Crossref](https://doi.org/10.1038/s41586-018-0414-6)
[CAS](https://doi.org/servlet/linkout?suffix=e_1_3_2_67_1&dbid=32&doi=10.15252%2Fmsb.20188746&key=1%3ACAS%3A528%3ADC%252BC1cXhsVynsrrL)
  * [a [...] fates in differentiation (La Manno et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0068-1)
  * [b [...] to add directionality (La Manno et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0068-2)
  * [c [...] and spliced read data (La Manno et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0068-3)

Law CW, Chen Y, Shi W, Smyth GK (2014) voom: precision weights unlock linear model analysis tools for RNA‐seq read counts. _Genome Biol_ 15: R29
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0069)
[Crossref](https://doi.org/10.1186/gb-2014-15-2-r29)
Levine JH, Simonds EF, Bendall SC, Davis KL, Amir E‐AD, Tadmor MD, Litvin O, Fienberg HG, Jager A, Zunder ER _et al_ (2015) Data‐driven phenotypic dissection of AML reveals progenitor‐like cells that correlate with prognosis. _Cell_ 162: 184–197
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0070)
[Crossref](https://doi.org/10.1016/j.cell.2015.05.047)
[CAS](https://doi.org/servlet/linkout?suffix=e_1_3_2_69_1&dbid=32&doi=10.15252%2Fmsb.20188746&key=1%3ACAS%3A528%3ADC%252BC2MXhtV2it7jE)
Li B, Ruotti V, Stewart RM, Thomson JA, Dewey CN (2009) RNA‐Seq gene expression estimation with read mapping uncertainty. _Bioinformatics_ 26: 493–500
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0071)
Li WV, Li JJ (2018) An accurate and robust imputation method scImpute for single‐cell RNA‐seq data. _Nat Commun_ 9: 997
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0072)
Liberzon A, Subramanian A, Pinchback R, Thorvaldsdottir H, Tamayo P, Mesirov JP (2011) Molecular signatures database (MSigDB) 3.0. _Bioinformatics_ 27: 1739–1740
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0073)
[Crossref](https://doi.org/10.1093/bioinformatics/btr260)
[CAS](https://doi.org/servlet/linkout?suffix=e_1_3_2_72_1&dbid=32&doi=10.15252%2Fmsb.20188746&key=1%3ACAS%3A528%3ADC%252BC3MXntFGktLw%253D)
Liu Y, Wang T, Zheng D (2018) RISC: robust integration of single‐cell RNA‐seq datasets with different extents of cell cluster overlap. _bioRxiv_ <https://doi.org/10.1101/483297> [PREPRINT]
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0074)
Lopez R, Regier J, Cole MB, Jordan M, Yosef N (2018) Deep generative modeling fro single‐cell transcriptomics. _Nat Methods_ 15: 1053–1058
  * [a [...] ; scVI: Lopez et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0075-1)
  * [b [...] implemented in the scVI tool (Lopez et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0075-2)
  * [c [...] framework of the model (scVI: Lopez et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0075-3)

Lotfollahi M, Wolf FA, Theis FJ (2018) Generative modeling and latent space arithmetics predict single‐cell perturbation response across cell types, studies and species. _bioRxiv_ <https://doi.org/10.1101/478503> [PREPRINT]
  * [a [...] ), scGen (preprint: Lotfollahi et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0076-1)
  * [b [...] ; scGen: preprint: Lotfollahi et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0076-2)

Love MI, Huber W, Anders S (2014) Moderated estimation of fold change and dispersion for RNA‐seq data with DESeq2. _Genome Biol_ 15: 550
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0077)
[Crossref](https://doi.org/10.1186/s13059-014-0550-8)
Lun ATL, Bach K, Marioni JC (2016a) Pooling across cells to normalize single‐cell RNA sequencing data with many zero counts. _Genome Biol_ 17: 75
[Crossref](https://doi.org/10.1186/s13059-016-0947-7)
  * [a [...] normalization methods (Lun et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0078-1)
  * [b [...] size factor estimation method (Lun et al, ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0078-2)

Lun ATL, McCarthy DJ, Marioni JC (2016b) A step‐by‐step workflow for low‐level analysis of single‐cell RNA‐seq data. _F1000Res_ 5: 2122
  * [a [...] ; Lun et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0079-1)
  * [b [...] steps are covered in Lun et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0079-2)

Lun ATL, Richard AC, Marioni JC (2017) Testing for differential abundance in mass cytometry data. _Nat Methods_ 14: 707–709
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0080)
Lun ATL (2018) Overcoming systematic errors caused by log‐transformation of normalized single‐cell RNA sequencing data. _bioRxiv_ <https://doi.org/10.1101/404962> [PREPRINT]
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0081)
van der Maaten L, Hinton G (2008) Visualizing data using t‐SNE. _J Mach Learn Res_ 9: 2579–2605
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0082)
Macosko EZ, Basu A, Satija R, Nemesh J, Shekhar K, Goldman M, Tirosh I, Bialas AR, Kamitaki N, Martersteck EM _et al_ (2015) Highly parallel genome‐wide expression profiling of individual cells using nanoliter droplets. _Cell_ 161: 1202–1214
[Crossref](https://doi.org/10.1016/j.cell.2015.05.002)
[CAS](https://doi.org/servlet/linkout?suffix=e_1_3_2_82_1&dbid=32&doi=10.15252%2Fmsb.20188746&key=1%3ACAS%3A528%3ADC%252BC2MXpt1Sgt7o%253D)
  * [a [...] ); Macosko et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0083-1)
  * [b [...] obtained from the literature (Macosko et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0083-2)
  * [c [...] ; Macosko et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0083-3)

MacQueen J (1967) Some methods for classification and analysis of multivariate observations. In _Proceedings of the fifth Berkeley symposium on mathematical statistics and probability_ , pp 281–297. Oakland, CA.
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0084)
Matsumoto H, Kiryu H, Furusawa C, Ko MSH, Ko SBH, Gouda N, Hayashi T, Nikaido I (2017) SCODE: an efficient regulatory network inference algorithm from single‐cell RNA‐Seq during differentiation. _Bioinformatics_ 33: 2314–2321
[Crossref](https://doi.org/10.1093/bioinformatics/btx194)
  * [a [...] ; Matsumoto et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0085-1)
  * [b [...] for scRNA‐seq data (SCONE: Matsumoto et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0085-2)

Mayer C, Hafemeister C, Bandler RC, Machold R, Batista Brito R, Jaglin X, Allaway K, Butler A, Fishell G, Satija R (2018) Developmental diversification of cortical inhibitory interneurons. _Nature_ 555: 457–462
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0086)
[Crossref](https://doi.org/10.1038/nature25999)
McCarthy DJ, Campbell KR, Lun ATL, Wills QF (2017) Scater: pre‐processing, quality control, normalization and visualization of single‐cell RNA‐seq data in R. _Bioinformatics_ 33: 1179–1186
[Crossref](https://doi.org/10.1093/bioinformatics/btw777)
[CAS](https://doi.org/servlet/linkout?suffix=e_1_3_2_86_1&dbid=32&doi=10.15252%2Fmsb.20188746&key=1%3ACAS%3A528%3ADC%252BC1cXhvFagtL%2FP)
  * [a [...] ), Scater (McCarthy et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0087-1)
  * [b [...] on the command line in R (McCarthy et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0087-2)
  * [c [...] line platforms, Scater (McCarthy et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0087-3)

McDavid A, Finak G, Gottardo R (2016) The contribution of cell cycle to heterogeneity in single‐cell RNA‐seq data. _Nat Biotechnol_ 34: 591–593
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0088)
McGinnis CS, Murrow LM, Gartner ZJ (2018) DoubletFinder: doublet detection in single‐cell RNA sequencing data using artificial nearest neighbors. _Cell Syst_ 8: 329–337
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0089)
McInnes L, Healy J (2018) UMAP: uniform manifold approximation and projection for dimension reduction. _arXiv_ [PREPRINT]
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0090)
Moignard V, Woodhouse S, Haghverdi L, Lilly AJ, Tanaka Y, Wilkinson AC, Buettner F, Macaulay IC, Jawaid W, Diamanti E _et al_ (2015) Decoding the regulatory network of early blood development from single‐cell gene expression measurements. _Nat Biotechnol_ 33: 269–276
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0091)
Montoro DT, Haber AL, Biton M, Vinarsky V, Lin B, Birket SE, Yuan F, Chen S, Leung HM, Villoria J _et al_ (2018) A revised airway epithelial hierarchy includes CFTR‐expressing ionocytes. _Nature_ 560: 319–324
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0092)
Moon KR, Stanley JS, Burkhardt D, van Dijk D, Wolf G, Krishnaswamy S (2018) Manifold learning‐based methods for analyzing single‐cell RNA‐sequencing data. _Curr Opin Syst Biol_ 7: 36–46
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0093)
Newman MEJ, Girvan M (2004) Finding and evaluating community structure in networks. _Phys Rev E Stat Nonlin Soft Matter Phys_ 69: 026113
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0094)
Ocone A, Haghverdi L, Mueller NS, Theis FJ (2015) Reconstructing gene regulatory dynamics from high‐dimensional single‐cell snapshot data. _Bioinformatics_ 31: i89–i96
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0095)
Pachter L (2011) Models for transcript quantification from RNA‐Seq. _arXiv_ [PREPRINT]
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0096)
Parekh S, Ziegenhain C, Vieth B, Enard W, Hellmann I (2018) zUMIs ‐ A fast and flexible pipeline to process RNA sequencing data with UMIs. _Gigascience_ 7: giy059
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0097)
[Crossref](https://doi.org/10.1093/gigascience/giy059)
Park JE, Polanski K, Meyer K, Teichmann SA (2018) Fast batch alignment of single cell transcriptomes unifies multiple mouse cell atlases into an integrated landscape. _bioRxiv_ <https://doi.org/10.1101/397042> [PREPRINT]
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0098)
Patel AP, Tirosh I, Trombetta JJ, Shalek AK, Gillespie SM, Wakimoto H, Cahill DP, Nahed BV, Curry WT, Martuza RL _et al_ (2014) Single‐cell RNA‐seq highlights intratumoral heterogeneity in primary glioblastoma. _Science_ 344: 1396–1401
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0099)
[Crossref](https://doi.org/10.1126/science.1254257)
[CAS](https://doi.org/servlet/linkout?suffix=e_1_3_2_98_1&dbid=32&doi=10.15252%2Fmsb.20188746&key=1%3ACAS%3A528%3ADC%252BC2cXpslygsL4%253D)
Patel MV (2018) iS‐CellR: a user‐friendly tool for analyzing and visualizing single‐cell RNA sequencing data. _Bioinformatics_ 34: 4305–4306
  * [a [...] ), and as local applications (Patel, ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0100-1)
  * [b [...] ), IS‐CellR (Patel, ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0100-2)

Pearson K (1901) On Lines and Planes of Closest Fit to Systems of Points in Space. _Philos Mag_ 2: 559–572
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0101)
Plass M, Solana J, Alexander Wolf F, Ayoub S, Misios A, Glažar P, Obermayer B, Theis FJ, Kocks C, Rajewsky N (2018) Cell type atlas and lineage tree of a whole complex animal by single‐cell transcriptomics. _Science_ 360: eaaq1723
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0102)
[Crossref](https://doi.org/10.1126/science.aaq1723)
Plasschaert LW, Žilionis R, Choo‐Wing R, Savova V, Knehr J, Roma G, Klein AM, Jaffe AB (2018) A single‐cell atlas of the airway epithelium reveals the CFTR‐rich pulmonary ionocyte. _Nature_ 560: 377–381
  * [a [...] ; Plasschaert et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0103-1)
  * [b [...] quality differences as in Plasschaert et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0103-2)

Pliner HA, Shendure J, Trapnell C (2019) Supervised classification enables rapid annotation of cell atlases. _bioRxiv_ <https://doi.org/10.1101/538652> [PREPRINT]
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0104)
Qiu X, Hill A, Packer J, Lin D, Ma Y‐A, Trapnell C (2017a) Single‐cell mRNA quantification and differential analysis with census. _Nat Methods_ 14: 309–315
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0105)
[Crossref](https://doi.org/10.1038/nmeth.4150)
Regev A, Teichmann SA, Lander ES, Amit I, Benoist C, Birney E, Bodenmiller B, Campbell PJ, Carninci P, Clatworthy M _et al_ (2017) Science forum: the human cell atlas. _Elife_ 6: e27041
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0107)
Regev A, Teichmann S, Rozenblatt‐Rosen O, Stubbington M, Ardlie K, Amit I, Arlotta P, Bader G, Benoist C, Biton M _et al_ (2018) The human cell atlas white paper. _arXiv_ [PREPRINT]
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0108)
Reichardt J, Bornholdt S (2006) Statistical mechanics of community detection. _Phys. Rev. E Stat. Nonlin. Soft Matter Phys_ 74: 016110
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0109)
Risso D, Perraudeau F, Gribkova S, Dudoit S, Vert J‐P (2018) A general and flexible method for signal extraction from single‐cell RNA‐seq data. _Nat Commun_ 9: 284
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0111)
[Crossref](https://doi.org/10.1038/s41467-017-02554-5)
Ritchie ME, Phipson B, Wu D, Hu Y, Law CW, Shi W, Smyth GK (2015) limma powers differential expression analyses for RNA‐sequencing and microarray studies. _Nucleic Acids Res_ 43: e47
[Crossref](https://doi.org/10.1093/nar/gkv007)
  * [a [...] ; Ritchie et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0112-1)
  * [b [...] using the limma package (Ritchie et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0112-2)

Robinson MD, McCarthy DJ, Smyth GK (2010) edgeR: a bioconductor package for differential expression analysis of digital gene expression data. _Bioinformatics_ 26: 139–140
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0113)
[Crossref](https://doi.org/10.1093/bioinformatics/btp616)
[CAS](https://doi.org/servlet/linkout?suffix=e_1_3_2_110_1&dbid=32&doi=10.15252%2Fmsb.20188746&key=1%3ACAS%3A528%3ADC%252BD1MXhs1WlurvO)
Rostom R, Svensson V, Teichmann SA, Kar G (2017) Computational approaches for interpreting scRNA‐seq data. _FEBS Lett_ 591: 2213–2225
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0114)
Rue‐Albrecht K, Marini F, Soneson C, Lun ATL (2018) iSEE: interactive summarizedexperiment explorer. _F1000Res_ 7: 741
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0115)
Saelens W, Cannoodt R, Todorov H, Saeys Y (2018) A comparison of single‐cell trajectory inference methods. _Nat Biotechnol_ 37: 547–554
  * [a [...] comparison of TI methods (Saelens et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0116-1)
  * [b [...] to improve performance (Saelens et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0116-2)
  * [c [...] We recommend using the Saelens et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0116-3)
  * [d [...] methods in a recent review (Saelens et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0116-4)

Saunders A, Macosko EZ, Wysoker A, Goldman M, Krienen FM, de Rivera H, Bien E, Baum M, Bortolin L, Wang S _et al_ (2018) Molecular diversity and specializations among the cells of the adult mouse brain. _Cell_ 174: 1015–1030.e16
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0118)
[Crossref](https://doi.org/10.1016/j.cell.2018.07.028)
[CAS](https://doi.org/servlet/linkout?suffix=e_1_3_2_114_1&dbid=32&doi=10.15252%2Fmsb.20188746&key=1%3ACAS%3A528%3ADC%252BC1cXhsVyltrrK)
Scholtens D, von Heydebreck A (2005) Analysis of Differential Gene Expression Studies. In _Statistics for Biology and Health_ , pp 229–248.
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0119)
Scholz CJ, Biernat P, Becker M, Baßler K, Günther P, Balfer J, Dickten H, Flöer L, Heikamp K, Angerer P _et al_ (2018) FASTGenomics: An analytical ecosystem for single‐cell RNA sequencing data. _bioRxiv_ <https://doi.org/10.1101/272476> [PREPRINT]
  * [a [...] support is growing (preprint: Scholz et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0120-1)
  * [b [...] ; preprint: Scholz et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0120-2)
  * [c [...] as FASTGenomics (preprint: Scholz et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0120-3)

Segerstolpe Å, Palasantza A, Eliasson P, Andersson E‐M, Andréasson A‐C, Sun X, Picelli S, Sabirsh A, Clausen M, Bjursell MK _et al_ (2016) Single‐cell transcriptome profiling of human pancreatic islets in health and type 2 diabetes. _Cell Metab_ 24: 593–607
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0121)
[Crossref](https://doi.org/10.1016/j.cmet.2016.08.020)
[CAS](https://doi.org/servlet/linkout?suffix=e_1_3_2_117_1&dbid=32&doi=10.15252%2Fmsb.20188746&key=1%3ACAS%3A528%3ADC%252BC28XhsFGru7zN)
Smallwood SA, Lee HJ, Angermueller C, Krueger F, Saadeh H, Peat J, Andrews SR, Stegle O, Reik W, Kelsey G (2014) Single‐cell genome‐wide bisulfite sequencing for assessing epigenetic heterogeneity. _Nat Methods_ 11: 817–820
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0122)
Soneson C, Robinson MD (2018) Bias, robustness and scalability in single‐cell differential expression analysis. _Nat Methods_ 15: 255–261
[Crossref](https://doi.org/10.1038/nmeth.4612)
  * [a [...] ; Soneson & Robinson, ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0123-1)
  * [b [...] data as input (Soneson & Robinson, ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0123-2)
  * [c [...] single‐cell tools (Soneson & Robinson, ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0123-3)
  * [d [...] aforementioned study (Soneson & Robinson, ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0123-4)

Stoeckius M, Hafemeister C, Stephenson W, Houck‐Loomis B, Chattopadhyay PK, Swerdlow H, Satija R, Smibert P (2017) Simultaneous epitope and transcriptome measurement in single cells. _Nat Methods_ 14: 865–868
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0124)
[Crossref](https://doi.org/10.1038/nmeth.4380)
Street K, Risso D, Fletcher RB, Das D, Ngai J, Yosef N, Purdom E, Dudoit S (2018) Slingshot: cell lineage and pseudotime inference for single‐cell transcriptomics. _BMC Genom_ 19: 477
[Crossref](https://doi.org/10.1186/s12864-018-4772-0)
  * [a [...] over genes in their tutorial (Street et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0125-1)
  * [b [...] revealed that Slingshot (Street et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0125-2)
  * [c [...] the online Slingshot tutorial (Street et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0125-3)

Svensson V, Natarajan KN, Ly L‐H, Miragaia RJ, Labalette C, Macaulay IC, Cvejic A, Teichmann SA (2017) Power analysis of single‐cell RNA‐sequencing experiments. _Nat Methods_ 14: 381–387
  * [a [...] ); Svensson et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0126-1)
  * [b [...] than droplet‐based data (Svensson et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0126-2)
  * [c [...] and 3′ enrichment methods (Svensson et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0126-3)

Tabula Muris Consortium, Overall coordination, Logistical coordination, Organ collection and processing, Library preparation and sequencing, Computational data analysis, Cell type annotation, Writing group, Supplemental text writing group & Principal investigators (2018) Single‐cell transcriptomics of 20 mouse organs creates a Tabula Muris. _Nature_ 562: 367–372
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0127)
[Crossref](https://doi.org/10.1038/s41586-018-0590-4)
Tanay A, Regev A (2017) Scaling single‐cell genomics from phenomenology to mechanism. _Nature_ 541: 331–338
  * [a [...] ; Tanay & Regev, ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0128-1)
  * [b [...] are continuous processes (Tanay & Regev, ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0128-2)
  * [c [...] pipelines will grow (Tanay & Regev, ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0128-3)

Tarca AL, Bhatti G, Romero R (2013) A comparison of gene set analysis methods in terms of sensitivity, prioritization and specificity. _PLoS One_ 8: e79217
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0129)
The Gene Ontology Consortium (2017) Expansion of the gene ontology knowledgebase and resources. _Nucleic Acids Res_ 45: D331–D338
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0130)
Tibshirani R, Hastie T, Narasimhan B, Chu G (2002) Diagnosis of multiple cancer types by shrunken centroids of gene expression. _Proc Natl Acad Sci USA_ 99: 6567–6572
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0131)
[Crossref](https://doi.org/10.1073/pnas.082099299)
[CAS](https://doi.org/servlet/linkout?suffix=e_1_3_2_127_1&dbid=32&doi=10.15252%2Fmsb.20188746&key=1%3ACAS%3A528%3ADC%252BD38XjvFCktbg%253D)
Trapnell C, Cacchiarelli D, Grimsby J, Pokharel P, Li S, Morse M, Lennon NJ, Livak KJ, Mikkelsen TS, Rinn JL (2014) The dynamics and regulators of cell fate decisions are revealed by pseudotemporal ordering of single cells. _Nat Biotechnol_ 32: 381–386
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0132)
[Crossref](https://doi.org/10.1038/nbt.2859)
[CAS](https://doi.org/servlet/linkout?suffix=e_1_3_2_128_1&dbid=32&doi=10.15252%2Fmsb.20188746&key=1%3ACAS%3A528%3ADC%252BC2cXks12ku7c%253D)
Vallejos CA, Risso D, Scialdone A, Dudoit S, Marioni JC (2017) Normalizing single‐cell RNA sequencing data: challenges and opportunities. _Nat Methods_ 14: 565–571
  * [a [...] ; Vallejos et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0133-1)
  * [b [...] from the original authors (Vallejos et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0133-2)
  * [c [...] ; Vallejos et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0133-3)

Van den Berge K, Perraudeau F, Soneson C, Love MI, Risso D, Vert J‐P, Robinson MD, Dudoit S, Clement L (2018) Observation weights unlock bulk RNA‐seq tools for zero inflation and single‐cell applications. _Genome Biol_ 19: 24
  * [a [...] counterparts (Van den Berge et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0134-1)
  * [b [...] weighted bulk methods (Van den Berge et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0134-2)

Vento‐Tormo R, Efremova M, Botting RA, Turco MY, Vento‐Tormo M, Meyer KB, Park J, Stephenson E, Polański K, Goncalves A _et al_ (2018) Single‐cell reconstruction of the early maternal‐fetal interface in humans. _Nature_ 563: 347–353
[Crossref](https://doi.org/10.1038/s41586-018-0698-6)
  * [a [...] ; Vento‐Tormo et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0135-1)
  * [b [...] the recent CellPhoneDB (Vento‐Tormo et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0135-2)
  * [c [...] ; Vento‐Tormo et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0135-3)

Vieth B, Ziegenhain C, Parekh S, Enard W, Hellmann I (2017) powsimR: power analysis for bulk and single cell RNA‐seq experiments. _Bioinformatics_ 33: 3486–3488
  * [a [...] of scRNA‐seq data. For example, Vieth et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0136-1)
  * [b [...] fact log‐normally distributed (Vieth et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0136-2)
  * [c [...] comparison on a single dataset (Vieth et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0136-3)

Vieth B, Parekh S, Ziegenhain C, Enard W, Hellmann I (2019) A systematic evaluation of single cell RNA‐seq analysis pipelines: library preparation and normalisation methods have the biggest impact on the performance of scRNA‐seq studies. _bioRxiv_ <https://doi.org/10.1101/583013> [PREPRINT]
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0500)
Wagner A, Regev A, Yosef N (2016) Revealing the vectors of cellular identity with single‐cell genomics. _Nat Biotechnol_ 34: 1145–1160
  * [a [...] within cell‐type clusters (Wagner et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0137-1)
  * [b [...] determine cellular identity (Wagner et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0137-2)
  * [c [...] T cells (Wagner et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0137-3)

Wagner DE, Weinreb C, Collins ZM, Briggs JA, Megason SG, Klein AM (2018) Single‐cell mapping of gene expression landscapes and lineage in the zebrafish embryo. _Science_ 360: 981–987
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0138)
Wang B, Zhu J, Pierson E, Ramazzotti D, Batzoglou S (2017) Visualization and analysis of single‐cell RNA‐seq data by kernel‐based similarity learning. _Nat Methods_ 14: 414–416
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0139)
Wattenberg M, Viégas F, Johnson I (2016) How to use t‐SNE effectively. _Distill_. <https://doi.org/10.23915/distill.00002>
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0140)
Webb S (2018) Deep learning for biology. _Nature_ 554: 555–557
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0141)
Weber LM, Robinson MD (2016) Comparison of clustering methods for high‐dimensional single‐cell flow and mass cytometry data. _Cytometry A_ 89: 1084–1096
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0142)
Weber LM, Nowicka M, Soneson C & Robinson MD (2018) diffcyt: Differential discovery in high‐dimensional cytometry via high‐resolution clustering. _Commun Biol_ 2: 183
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0143)
Weinreb C, Wolock S, Klein AM (2018) SPRING: a kinetic interface for visualizing high dimensional single‐cell expression data. _Bioinformatics_ 34: 1246–1248
  * [a [...] appropriate. For example, Weinreb et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0144-1)
  * [b [...] tools such as SPRING (Weinreb et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0144-2)

Welch J, Kozareva V, Ferreira A, Vanderburg C (2018) Integrative inference of brain cell similarities and differences from single‐cell genomics. _bioRxiv_ <https://doi.org/10.1101/459891> [PREPRINT]
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0145)
Wolf FA, Angerer P, Theis FJ (2018) SCANPY: large‐scale single‐cell gene expression data analysis. _Genome Biol_ 19: 15
[Crossref](https://doi.org/10.1186/s13059-017-1382-0)
  * [a [...] ), or Scanpy (Wolf et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0146-1)
  * [b [...] ; Wolf et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0146-2)
  * [c [...] ) or Python (Wolf et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0146-3)
  * [d [...] to this group is scanpy (Wolf et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0146-4)

Wolf FA, Hamey F, Plass M, Solana J, Dahlin JS, Gottgens B, Rajewsky N, Simon L, Theis FJ (2019) Graph abstraction reconciles clustering with trajectory inference through a topology preserving map of single cells. _Genome Biol_ 20: 59
  * [a [...] of the underlying topology (Wolf et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0147-1)
  * [b [...] graph abstraction (PAGA; Wolf et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0147-2)
  * [c [...] topology of the expression data (Wolf et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0147-3)
  * [d [...] trajectories are expected, PAGA (Wolf et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0147-4)
  * [e [...] D; Wolf et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0147-5)

Wolock SL, Lopez R, Klein AM (2019) Scrublet: computational identification of cell doublets in single‐cell transcriptomic data. _Cell Syst_ 8: 281–291.
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0148)
[Crossref](https://doi.org/10.1016/j.cels.2018.11.005)
Yip SH, Sham PC, Wang J (2018) Evaluation of tools for highly variable gene discovery from single‐cell RNA‐seq data. _Brief Bioinform_ : bby011
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0149)
Young MD, Behjati S (2018) SoupX removes ambient RNA contamination from droplet based single cell RNA sequencing data. _bioRxiv_ <https://doi.org/10.1101/303727> [PREPRINT]
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0150)
Zappia L, Phipson B, Oshlack A (2017) Splatter: simulation of single‐cell RNA sequencing data. _Genome Biol_ 18: 174
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0151)
Zappia L, Phipson B, Oshlack A (2018) Exploring the single‐cell RNA‐seq analysis landscape with the scRNA‐tools database. _PLoS Comput Biol_ 14: e1006245
  * [a [...] ; Zappia et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0152-1)
  * [b [...] prominently R and Python (Zappia et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0152-2)
  * [c [...] ) and Zappia et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0152-3)

Zeisel A, Hochgerner H, Lonnerberg P, Johnsson A, Memic F, van der Zwan J, Haring M, Braun E, Borm L, La Manno G _et al_ (2018) Molecular architecture of the mouse nervous system. _Cell_ 174: 999–1014
[Crossref](https://doi.org/10.1016/j.cell.2018.06.021)
[CAS](https://doi.org/servlet/linkout?suffix=e_1_3_2_150_1&dbid=32&doi=10.15252%2Fmsb.20188746&key=1%3ACAS%3A528%3ADC%252BC1cXhsVyltrvN)
  * [a [...] such as the mouse brain atlas (Zeisel et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0153-1)
  * [b [...] (Zeisel et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0153-2)

Zepp JA, Zacharias WJ, Frank DB, Cavanaugh CA, Zhou S, Morley MP, Morrisey EE (2017) Distinct mesenchymal lineages and niches promote epithelial self‐renewal and myofibrogenesis in the lung. _Cell_ 170: 1134–1148.e10
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0154)
Zhang JM, Kamath GM, Tse DN (2018) Towards a post‐clustering test for differential expression. _bioRxiv_ <https://doi.org/10.1101/463265> [PREPRINT]
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0155)
Zheng GXY, Terry JM, Belgrader P, Ryvkin P, Bent ZW, Wilson R, Ziraldo SB, Wheeler TD, McDermott GP, Zhu J _et al_ (2017) Massively parallel digital transcriptional profiling of single cells. _Nat Commun_ 8: 14049
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0156)
[Crossref](https://doi.org/10.1038/ncomms14049)
[CAS](https://doi.org/servlet/linkout?suffix=e_1_3_2_153_1&dbid=32&doi=10.15252%2Fmsb.20188746&key=1%3ACAS%3A528%3ADC%252BC2sXht1WlsLo%253D)
Zhou JX, Taramelli R, Pedrini E, Knijnenburg T, Huang S (2017) Extracting intercellular signaling network of cancer tissues using ligand‐receptor expression patterns from whole‐tumor and single‐cell transcriptomes. _Sci Rep_ 7: 8815
[Go to Citation](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0157)
Zhu X, Wolfgruber TK, Tasato A, Arisdakessian C, Garmire DG, Garmire LX (2017) Granatum: a graphical single‐cell RNA‐Seq analysis pipeline for genomics scientists. _Genome Med_ 9: 108
  * [a [...] ; Zhu et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0158-1)
  * [b [...] of platforms is available in Zhu et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0158-2)
  * [c [...] Platforms such as Granatum (Zhu et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0158-3)

Ziegenhain C, Vieth B, Parekh S, Reinius B, Guillaumet‐Adkins A, Smets M, Leonhardt H, Heyn H, Hellmann I, Enard W (2017) Comparative analysis of single‐cell RNA sequencing methods. _Mol Cell_ 65: 631–643.e4
[Crossref](https://doi.org/10.1016/j.molcel.2017.01.023)
  * [a [...] protocols can be found in Ziegenhain et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0159-1)
  * [b [...] ; Ziegenhain et al ](https://doi.org/10.15252/msb.20188746#body-ref-msb188746-cit-0159-2)

[View full text](https://doi.org/doi/full/10.15252/msb.20188746)|[Download PDF](https://doi.org/doi/pdf/10.15252/msb.20188746?download=true)
[Open in viewer](https://doi.org/10.15252/msb.20188746)
[](https://doi.org/10.15252/msb.20188746 "Close figure viewer")
Go to
Go to
Show all references
Request permissionsExpand All
Click here to collapse this figure.
Click here to expand this figure.
[Authors Info & Affiliations](https://doi.org/10.15252/msb.20188746#tab-contributors)
#### Recommended
  * [Molecular Systems Biology](https://doi.org/journal/17444292)
ReviewApril 2018
###### [Using single‐cell genomics to understand developmental processes and cell fate decisions](https://doi.org/doi/full/10.15252/msb.20178046)
    * [Jonathan A Griffiths](https://doi.org/authored-by/Griffiths/Jonathan+A "Jonathan A Griffiths")
    * [ […] ](https://doi.org/10.15252/msb.20188746)
    * [John C Marioni](https://doi.org/authored-by/Marioni/John+C "John C Marioni")
  * [The EMBO Journal](https://doi.org/journal/14602075)
ResourceMay 2021
###### [A single‐cell RNA expression atlas of normal, preneoplastic and tumorigenic states in the human breast](https://doi.org/doi/full/10.15252/embj.2020107333)
    * [Bhupinder Pal](https://doi.org/authored-by/Pal/Bhupinder "Bhupinder Pal")
    * [ […] ](https://doi.org/10.15252/msb.20188746)
    * [Jane E Visvader](https://doi.org/authored-by/Visvader/Jane+E "Jane E Visvader")
  * [Molecular Systems Biology](https://doi.org/journal/17444292)
MethodOctober 2019
###### [Combinatorial prediction of marker panels from single‐cell transcriptomic data](https://doi.org/doi/full/10.15252/msb.20199005)
    * [Conor Delaney](https://doi.org/authored-by/Delaney/Conor "Conor Delaney")
    * [ […] ](https://doi.org/10.15252/msb.20188746)
    * [Meromit Singer](https://doi.org/authored-by/Singer/Meromit "Meromit Singer")

Now Reading:
Current best practices in single‐cell RNA‐seq analysis: a tutorial
[](https://doi.org/action/addCitationAlert?doi=10.15252%2Fmsb.20188746 "Track Citations")
[](https://doi.org/personalize/addFavoritePublication?doi=10.15252%2Fmsb.20188746 "Bookmark")
[](https://doi.org/10.15252/msb.20188746#tab-citations "Cite")
Share
  * [](https://doi.org/#facebook "Share on Facebook")
  * [](https://doi.org/#twitter "Share on X \(formerly Twitter\)")
  * [](https://doi.org/#linkedin "Share on LinkedIn")
  * [](https://doi.org/#reddit "Share on Reddit")
  * [](https://doi.org/#wechat "Share on WeChat")
  * [](https://doi.org/#email "Share on email")

###### [NEXT ARTICLEDisentangling bacterial invasiveness from lethality in an experimental host‐pathogen systemNext](https://doi.org/doi/10.15252/msb.20188707)
FiguresTables[](https://doi.org/10.15252/msb.20188746 "Close figure viewer")
[![image](https://doi.org/cms/10.15252/msb.20188746/asset/5bcc5719-49b9-4156-a754-76a2448e5392/assets/graphic/msb188746-fig-0001-m.png)View figure](https://doi.org/10.15252/msb.20188746#fv-msb188746-fig-0001)
Figure 1
Figure 1. Schematic of a typical single‐cell RNA‐seq analysis workflow
Raw sequencing data are processed and aligned to give count matrices, which represent the start of the workflow. The count data undergo pre‐processing and downstream analysis. Subplots are generated using the best‐practices workflow on intestinal epithelium data from Haber _et al_ ([2017](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0046)).
  * [Download figure](https://doi.org/cms/asset/5bcc5719-49b9-4156-a754-76a2448e5392/msb188746-fig-0001-m.png)
  * [Download PowerPoint](https://doi.org/action/downloadFigures?doi=10.15252%2Fmsb.20188746&id=msb188746-fig-0001)

[![image](https://doi.org/cms/10.15252/msb.20188746/asset/1bd082a4-169f-4878-9161-5cab859d983c/assets/graphic/msb188746-fig-0002-m.png)View figure](https://doi.org/10.15252/msb.20188746#fv-msb188746-fig-0002)
Figure 2
Figure 2. Plots of quality control metrics with filtering decisions for a mouse intestinal epithelium dataset from Haber _et al_ ([2017](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0046))
(A) Histograms of count depth per cell. The smaller histogram is zoomed‐in on count depths below 4,000. A threshold is applied here at 1,500 based on the peak detected at around 1,200 counts. (B) Histogram of the number of genes detected per cell. A small noise peak is visible at approx. 400 genes. These cells are filtered out using the depicted threshold (red line) at 700 genes. (C) Count depth distribution from high to low count depths. This visualization is related to the log–log plot shown in Cell Ranger outputs that is used to filter out empty droplets. It shows an “elbow” where count depths start to decrease rapidly around 1,500 counts. (D) Number of genes versus the count depth coloured by the fraction of mitochondrial reads. Mitochondrial read fractions are only high in particularly low count cells with few detected genes. These cells are filtered out by our count and gene number thresholds. Jointly visualizing the count and gene thresholds shows the joint filtering effect, indicating that a lower gene threshold may have sufficed. [Correction added on 5 July 2019, after first online publication: the x‐axis label in panel B was corrected from “Count depth” to “Number of genes”.]
  * [Download figure](https://doi.org/cms/asset/1bd082a4-169f-4878-9161-5cab859d983c/msb188746-fig-0002-m.png)
  * [Download PowerPoint](https://doi.org/action/downloadFigures?doi=10.15252%2Fmsb.20188746&id=msb188746-fig-0002)

[![image](https://doi.org/cms/10.15252/msb.20188746/asset/0ed29c9f-28ff-40f8-a712-110306de12dc/assets/graphic/msb188746-fig-0003-m.png)View figure](https://doi.org/10.15252/msb.20188746#fv-msb188746-fig-0003)
Figure 3
Figure 3. UMAP visualization before and after batch correction
Cells are coloured by sample of origin. Separation of batches is clearly visible before batch correction and less visible afterwards. Batch correction was performed using ComBat on mouse intestinal epithelium data from Haber _et al_ ([2017](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0046)).
  * [Download figure](https://doi.org/cms/asset/0ed29c9f-28ff-40f8-a712-110306de12dc/msb188746-fig-0003-m.png)
  * [Download PowerPoint](https://doi.org/action/downloadFigures?doi=10.15252%2Fmsb.20188746&id=msb188746-fig-0003)

[![image](https://doi.org/cms/10.15252/msb.20188746/asset/40988faf-3bbb-4dc7-ba72-751c5fb2f775/assets/graphic/msb188746-fig-0001ev-m.png)View figure](https://doi.org/10.15252/msb.20188746#fv-msb188746-fig-0001ev)
Figure EV1
Figure EV1. The number of highly variable genes (HVGs) used for datasets of different sizes
The data were obtained by a brief manual survey of recent scRNA‐seq analysis papers. The plotted data, along with further information on scRNA‐seq technology, publication year, reference and the number of reads per cell, are available in [Dataset EV1](https://doi.org/10.15252/msb.20188746#msb188746-sup-0003).
  * [Download figure](https://doi.org/cms/asset/40988faf-3bbb-4dc7-ba72-751c5fb2f775/msb188746-fig-0001ev-m.png)
  * [Download PowerPoint](https://doi.org/action/downloadFigures?doi=10.15252%2Fmsb.20188746&id=msb188746-fig-0001ev)

[![image](https://doi.org/cms/10.15252/msb.20188746/asset/8e97ce5f-5e00-4b4c-981b-fe59d71dd30b/assets/graphic/msb188746-fig-0004-m.png)View figure](https://doi.org/10.15252/msb.20188746#fv-msb188746-fig-0004)
Figure 4
Figure 4. Common visualization methods for scRNA‐seq data
Mouse intestinal epithelium regions data from Haber _et al_ ([2017](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0046)) visualized on the first two components for: (A) PCA, (B) t‐SNE, (C) diffusion maps, (D) UMAP and (E) A force‐directed graph layout via ForceAtlas2. Cells are coloured by count depth. (F) Variance explained by the first 31 principal components (PCs). The “elbow” of this plot, which is used to select relevant PCs to analyse the dataset, lies between PCs 5 and 7.
  * [Download figure](https://doi.org/cms/asset/8e97ce5f-5e00-4b4c-981b-fe59d71dd30b/msb188746-fig-0004-m.png)
  * [Download PowerPoint](https://doi.org/action/downloadFigures?doi=10.15252%2Fmsb.20188746&id=msb188746-fig-0004)

[![image](https://doi.org/cms/10.15252/msb.20188746/asset/d6585605-fb7f-4bea-8276-71309cfcd3fb/assets/graphic/msb188746-fig-0002ev-m.png)View figure](https://doi.org/10.15252/msb.20188746#fv-msb188746-fig-0002ev)
Figure EV2
Figure EV2. Change in coefficient of variation (CoV) of gene expression data upon batch correction and denoising
Negative values represent a reduction in CoV upon data correction. The top row shows CoV changes upon ComBat batch correction for (A) mouse intestinal epithelium (mIE) and (B) mouse embryonic stem cell (mESC) data. The lower row depicts CoV changes upon DCA denoising for (C) mIE and (D) mESC data. mIE data were obtained from Haber _et al_ ([2017](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0046)) and mESC from Klein _et al_ ([2015](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0065)).
  * [Download figure](https://doi.org/cms/asset/d6585605-fb7f-4bea-8276-71309cfcd3fb/msb188746-fig-0002ev-m.png)
  * [Download PowerPoint](https://doi.org/action/downloadFigures?doi=10.15252%2Fmsb.20188746&id=msb188746-fig-0002ev)

[![image](https://doi.org/cms/10.15252/msb.20188746/asset/5593cfbd-38ec-43e5-96e8-9beaa495c0c2/assets/graphic/msb188746-fig-0005-m.png)View figure](https://doi.org/10.15252/msb.20188746#fv-msb188746-fig-0005)
Figure 5
Figure 5. Overview of downstream analysis methods
Methods are divided into cell‐ and gene‐level analysis. Cell‐level analysis approaches are again subdivided into cluster and trajectory analysis branches, which include also gene‐level analysis methods. All methods with a blue background are gene‐level approaches.
  * [Download figure](https://doi.org/cms/asset/5593cfbd-38ec-43e5-96e8-9beaa495c0c2/msb188746-fig-0005-m.png)
  * [Download PowerPoint](https://doi.org/action/downloadFigures?doi=10.15252%2Fmsb.20188746&id=msb188746-fig-0005)

[![image](https://doi.org/cms/10.15252/msb.20188746/asset/62f76211-6a38-42ed-b151-b38d246b897e/assets/graphic/msb188746-fig-0006-m.png)View figure](https://doi.org/10.15252/msb.20188746#fv-msb188746-fig-0006)
Figure 6
Figure 6. Cluster analysis results of mouse intestinal epithelium dataset from Haber _et al_ ([2017](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0046))
(A) Annotated cell‐identity clusters found by Louvain clustering visualized in a UMAP representation. (B) Cell‐identity marker expression to identify stem cells (Slc12a2), enterocytes (Arg2), goblet cells (Tff3) and Paneth cells (Defa24). Corrected expression levels are visualized from low expression (grey) to high expression (red). Marker genes may be expressed also in other cell‐identity populations as shown for goblet and Paneth cells. (C) Cell‐identity composition heat maps of proximal (upper) and distal (lower) intestinal epithelium regions. High relative cell density is shown as dark red.
  * [Download figure](https://doi.org/cms/asset/62f76211-6a38-42ed-b151-b38d246b897e/msb188746-fig-0006-m.png)
  * [Download PowerPoint](https://doi.org/action/downloadFigures?doi=10.15252%2Fmsb.20188746&id=msb188746-fig-0006)

[![image](https://doi.org/cms/10.15252/msb.20188746/asset/13e5de7d-250b-4715-8ab8-5dcb3eea526d/assets/graphic/msb188746-fig-0007-m.png)View figure](https://doi.org/10.15252/msb.20188746#fv-msb188746-fig-0007)
Figure 7
Figure 7. Trajectory analysis and graph abstraction of mouse intestinal epithelium data from Haber _et al_ ([2017](https://doi.org/10.15252/msb.20188746#core-msb188746-cit-0046))
(A) Distal and proximal enterocyte differentiation trajectories inferred by Slingshot. The Distal lineage is shown coloured by pseudotime from red to blue. Other cells in the dataset are grey. (B) Slingshot trajectories over clusters in PCA space. Clusters are abbreviated as follows: EP—enterocyte progenitors; Imm. Ent.—immature enterocytes; Mat. Ent.—mature enterocytes; Prox.—proximal; Dist.—distal. (C) Density over pseudotime for the distal enterocyte trajectory from Fig [7](https://doi.org/10.15252/msb.20188746#msb188746-fig-0007)A. Colours represent the dominant cluster labels in each pseudotime bin. (D) Abstracted graph representation of the dataset projected onto a UMAP representation. Clusters are shown as coloured nodes. Clusters that appear in other trajectories are labelled for comparison. “TA” denotes transit amplifying cells. (E) Gene expression dynamics over pseudotime in a general enterocyte trajectory using the “GAM” R library.
  * [Download figure](https://doi.org/cms/asset/13e5de7d-250b-4715-8ab8-5dcb3eea526d/msb188746-fig-0007-m.png)
  * [Download PowerPoint](https://doi.org/action/downloadFigures?doi=10.15252%2Fmsb.20188746&id=msb188746-fig-0007)

[](https://doi.org/10.15252/msb.20188746#fv-msb188746-tbl-0001)
Table 1
Table 1. Stages of data processing and appropriate downstream applications
Close modalClose modal
![large cover image](https://doi.org/cms/asset/6cc440d9-ef76-4b3c-b661-75f325b10d7a/msb.v15.6.cover.jpg)
[View large image](https://doi.org/cms/asset/6cc440d9-ef76-4b3c-b661-75f325b10d7a/msb.v15.6.cover.jpg)
Molecular Systems Biology
Volume 15,Issue 6,Jun 2019
This month’s cover highlights the article by Tommaso Biancalani and Jeff Gore. Mathematical modeling and quantitative analyses of survival curves of _C. elegans_ exposed to three pathogens allow distinguishing between pathogen invasiveness and lethality and reveal quantitative laws of host-pathogen dynamics. (Cover illustration by Sandra Krahl)
BACK TO TOPTOP
[![Molecular Systems Biology logo](https://doi.org/pb-assets/embo-site/images/17444292-1553268207403.svg)](https://doi.org/journal/17444292)
###### [Journal](https://doi.org/journal/17444292)
  * [Home](https://doi.org/journal/17444292 "Home")
  * [Current Issue](https://doi.org/toc/17444292/current "Current Issue")
  * [Archive](https://doi.org/loi/17444292 "Archive")
  * [Article Collections](https://doi.org/page/journal/17444292/focus "Article Collections")
  * [Subject Areas](https://doi.org/results/embo-topics "Subject Areas")
  * [EMBO Press](https://doi.org/ "EMBO Press")
  * Online ISSN 1744-4292

###### [For Authors](https://doi.org/10.15252/msb.20188746)
  * [Submit an article](https://doi.org/page/journal/17444292/submit "Submit an article")
  * [Author Guidelines](https://doi.org/page/journal/17444292/authorguide "Author Guidelines")
  * [Aims & Scope](https://doi.org/page/journal/17444292/about "Aims & Scope")
  * [EMBO Press Policies](https://doi.org/policies "EMBO Press Policies")
  * [Licenses & Charges](https://doi.org/page/journal/17444292/authorguide#chargesguide "Licenses & Charges")
  * [Privacy policy](https://doi.org/privacy-policy "Privacy policy")

###### [Information](https://doi.org/10.15252/msb.20188746)
  * [In the News](https://doi.org/news "In the News")
  * [Customer Service](https://doi.org/page/journal/17444292/contact "Customer Service")

###### [Stay Connected](https://doi.org/10.15252/msb.20188746)
  * [](https://doi.org/feed/17444292/most-recent "RSS")
  * [](https://doi.org/action/showAlertSettings?journalCode=17444292&action=addJournal "alerts")

###### [EMBO](https://doi.org/10.15252/msb.20188746)
[![embo press](https://doi.org/pb-assets/embo-site/images/EMBOpress_color-1553268207600.svg)](https://doi.org/ "embo press") [![embo journal](https://doi.org/pb-assets/embo-site/images/EMBOjournal-1553268207590.svg)](https://doi.org/journal/14602075 "embo journal") [![embo reports](https://doi.org/pb-assets/embo-site/images/EMBOreports-1553268207573.svg)](https://doi.org/journal/14693178 "embo reports") [![embo MM](https://doi.org/pb-assets/embo-site/images/EMBOmolecularmedicine-1553268207420.svg)](https://doi.org/journal/17574684 "embo MM") [![embo MSB](https://doi.org/pb-assets/embo-site/images/EMBOmsb-1553268207403.svg)](https://doi.org/journal/17444292 "embo MSB")
© 2025
Copy link
✓
Thanks for sharing!
Find any service
[More…](https://doi.org/10.15252/msb.20188746#addtoany "Show all")
  *[PDF]: Portable Document Format