## Your privacy, your choice
We use essential cookies to make sure the site can function. We also use optional cookies for advertising, personalisation of content, usage analysis, and social media.
By accepting optional cookies, you consent to the processing of your personal data - including transfers to third parties. Some third parties are outside of the European Economic Area, with varying standards of data protection.
See our [privacy policy](https://www.nature.com/info/privacy) for more information on the use of your personal data.
Manage preferences for further information and to change your choices.
Accept all cookies
[Skip to main content](https://www.nature.com/articles/npp2012112#content)
Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript.
Advertisement
[ ![Neuropsychopharmacology](https://media.springernature.com/full/nature-cms/uploads/product/npp/header-ae200e1a412a69d3b9efe884088d6545.svg) ](https://www.nature.com/npp)
  * [ View all journals ](https://www.nature.com/siteindex)
  * ## Search
Search articles by subject, keyword or author
Show results from All journals This journal
Search
[ Advanced search ](https://www.nature.com/search/advanced)
### Quick links
    * [Explore articles by subject](https://www.nature.com/subjects)
    * [Find a job](https://www.nature.com/naturecareers)
    * [Guide to authors](https://www.nature.com/authors/index.html)
    * [Editorial policies](https://www.nature.com/authors/editorial_policies/)
  * [Log in](https://idp.nature.com/auth/personal/springernature?redirect_uri=https://www.nature.com/articles/npp2012112)

  * ## Explore content
    * [ Research articles ](https://www.nature.com/npp/research-articles)
    * [ Reviews & Analysis ](https://www.nature.com/npp/reviews-and-analysis)
    * [ News & Comment ](https://www.nature.com/npp/news-and-comment)
    * [ Podcasts ](https://www.nature.com/npp/podcast)
    * [ Current issue ](https://www.nature.com/npp/current-issue)
    * [ Collections ](https://www.nature.com/npp/collections)
    * [Sign up for alerts ](https://www.nature.com/my-account/alerts/subscribe-journal?list-id=72)
    * [ RSS feed ](https://www.nature.com/npp.rss)
  * ## About the journal
    * [ Journal Information ](https://www.nature.com/npp/journal-information)
    * [ Open Access Fees and Funding ](https://www.nature.com/npp/open-access)
    * [ About the Editors ](https://www.nature.com/npp/editors)
    * [ Contact ](https://www.nature.com/npp/contact)
    * [ Press Releases ](https://www.nature.com/npp/press-releases)
    * [ About the Partner ](https://www.nature.com/npp/partner)
    * [ For Advertisers ](https://www.nature.com/npp/advertising)
    * [ Subscribe ](https://www.nature.com/npp/subscribe)
    * [ Announcements ](https://www.nature.com/npp/announcements)
    * [ NPP has a new YouTube channel! ](https://www.nature.com/npp/youtube)
  * ## Publish with us
    * [ For Authors & Referees ](https://www.nature.com/npp/authors-and-referees)
    * [ Language editing services ](https://authorservices.springernature.com/go/sn/?utm_source=For+Authors&utm_medium=Website_Nature&utm_campaign=Platform+Experimentation+2022&utm_id=PE2022)
    * [Submit manuscript ](https://mts-npp.nature.com/cgi-bin/main.plex)

  * [ Sign up for alerts ](https://idp.nature.com/auth/personal/springernature?redirect_uri=https%3A%2F%2Fwww.nature.com%2Fmy-account%2Falerts%2Fsubscribe-journal%3Flist-id%3D72%26journal-link%3Dhttps%253A%252F%252Fwww.nature.com%252Fnpp%252F)
  * [ RSS feed ](https://www.nature.com/npp.rss)

  1. [nature](https://www.nature.com/)
  2. [neuropsychopharmacology](https://www.nature.com/npp)
  3. [neuropsychopharmacology reviews](https://www.nature.com/npp/articles?type=neuropsychopharmacology-reviews)
  4. article

DNA Methylation and Its Basic Function 
[ Download PDF ](https://www.nature.com/articles/npp2012112.pdf)
[ Download PDF ](https://www.nature.com/articles/npp2012112.pdf)
  * Neuropsychopharmacology Reviews
  * Published: 11 July 2012

# DNA Methylation and Its Basic Function
  * [Lisa D Moore](https://www.nature.com/articles/npp2012112#auth-Lisa_D-Moore-Aff1)[1](https://www.nature.com/articles/npp2012112#Aff1), 
  * [Thuc Le](https://www.nature.com/articles/npp2012112#auth-Thuc-Le-Aff1)[1](https://www.nature.com/articles/npp2012112#Aff1) &
  * [Guoping Fan](https://www.nature.com/articles/npp2012112#auth-Guoping-Fan-Aff1)[1](https://www.nature.com/articles/npp2012112#Aff1)

[_Neuropsychopharmacology_](https://www.nature.com/npp) **volume 38**, pages 23–38 (2013)[Cite this article](https://www.nature.com/articles/npp2012112#citeas)
  * 549k Accesses
  * 2877 Citations
  * 625 Altmetric
  * [Metrics details](https://www.nature.com/articles/npp2012112/metrics)

## Abstract
In the mammalian genome, DNA methylation is an epigenetic mechanism involving the transfer of a methyl group onto the C5 position of the cytosine to form 5-methylcytosine. DNA methylation regulates gene expression by recruiting proteins involved in gene repression or by inhibiting the binding of transcription factor(s) to DNA. During development, the pattern of DNA methylation in the genome changes as a result of a dynamic process involving both _de novo_ DNA methylation and demethylation. As a consequence, differentiated cells develop a stable and unique DNA methylation pattern that regulates tissue-specific gene transcription. In this chapter, we will review the process of DNA methylation and demethylation in the nervous system. We will describe the DNA (de)methylation machinery and its association with other epigenetic mechanisms such as histone modifications and noncoding RNAs. Intriguingly, postmitotic neurons still express DNA methyltransferases and components involved in DNA demethylation. Moreover, neuronal activity can modulate their pattern of DNA methylation in response to physiological and environmental stimuli. The precise regulation of DNA methylation is essential for normal cognitive function. Indeed, when DNA methylation is altered as a result of developmental mutations or environmental risk factors, such as drug exposure and neural injury, mental impairment is a common side effect. The investigation into DNA methylation continues to show a rich and complex picture about epigenetic gene regulation in the central nervous system and provides possible therapeutic targets for the treatment of neuropsychiatric disorders.
### Similar content being viewed by others
![](https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41392-022-01078-9/MediaObjects/41392_2022_1078_Fig1_HTML.png)
###  [The role of histone modifications: from neurodevelopment to neurodiseases ](https://www.nature.com/articles/s41392-022-01078-9?fromPaywallRec=false)
Article Open access 06 July 2022
![](https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41467-021-21917-7/MediaObjects/41467_2021_21917_Fig1_HTML.png)
###  [Evolution of DNA methylation in the human brain ](https://www.nature.com/articles/s41467-021-21917-7?fromPaywallRec=false)
Article Open access 01 April 2021
![](https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41390-024-03674-7/MediaObjects/41390_2024_3674_Fig1_HTML.png)
###  [Fundamentals of DNA methylation in development ](https://www.nature.com/articles/s41390-024-03674-7?fromPaywallRec=false)
Article 10 December 2024
## INTRODUCTION
Genetics is the study of heritable changes in gene activity or function due to the direct alteration of the DNA sequence. Such alterations include point mutations, deletions, insertions, and translocation. In contrast, epigenetics is the study of heritable changes in gene activity or function that is not associated with any change of the DNA sequence itself. Although virtually all cells in an organism contain the same genetic information, not all genes are expressed simultaneously by all cell types. In a broader sense, epigenetic mechanisms mediate the diversified gene expression profiles in a variety of cells and tissues in multicellular organisms.
In this chapter, we would introduce a major epigenetic mechanism involving direct chemical modification to the DNA called DNA methylation. Historically, DNA methylation was discovered in mammals as early as DNA was identified as the genetic material ([Avery et al, 1944](https://www.nature.com/articles/npp2012112#ref-CR7 "Avery OT, Macleod CM, McCarty M \(1944\). Studies on the chemical nature of the substance inducing transformation of pneumococcal types: induction of transformation by a desoxyribonucleic acid fraction isolated from pneumococcus type Iii. J Exp Med 79: 137–158."); [McCarty and Avery, 1946](https://www.nature.com/articles/npp2012112#ref-CR132 "McCarty M, Avery OT \(1946\). Studies on the chemical nature of the substance inducing transformation of pneumococcal types: II. Effect of desoxyribonuclease on the biological activity of the transforming substance. J Exp Med 83: 89–96.")). In 1948, Rollin Hotchkiss first discovered modified cytosine in a preparation of calf thymus using paper chromatography. [Hotchkiss (1948)](https://www.nature.com/articles/npp2012112#ref-CR81 "Hotchkiss RD \(1948\). The quantitative separation of purines, pyrimidines, and nucleosides by paper chromatography. J Biol Chem 175: 315–332. Discovered 5-methylcytosine in the mammalian genome.") hypothesized that this fraction was 5-methylcytosine (5mC) because it separated from cytosine in a manner that was similar to the way that thymine (also known as methyluracil) separated from uracil, and he further suggested that this modified cytosine existed naturally in DNA. Although many researchers proposed that DNA methylation might regulate gene expression, it was not until the 1980s that several studies demonstrated that DNA methylation was involved in gene regulation and cell differentiation ([Holliday and Pugh, 1975](https://www.nature.com/articles/npp2012112#ref-CR80 "Holliday R, Pugh JE \(1975\). DNA modification mechanisms and gene activity during development. Science 187: 226–232."); [Compere and Palmiter, 1981](https://www.nature.com/articles/npp2012112#ref-CR34 "Compere SJ, Palmiter RD \(1981\). DNA methylation controls the inducibility of the mouse metallothionein-I gene lymphoid cells. Cell 25: 233–240.")). It is now well recognized that DNA methylation, in concert with other regulators, is a major epigenetic factor influencing gene activities.
DNA methylation is catalyzed by a family of DNA methyltransferases (Dnmts) that transfer a methyl group from _S_ -adenyl methionine (SAM) to the fifth carbon of a cytosine residue to form 5mC ([Figure 1](https://www.nature.com/articles/npp2012112#Fig1)). Dnmt3a and Dnmt3b can establish a new methylation pattern to unmodified DNA and are thus known as _de novo_ Dnmt ([Figure 1a](https://www.nature.com/articles/npp2012112#Fig1)). On the other hand, Dnmt1 functions during DNA replication to copy the DNA methylation pattern from the parental DNA strand onto the newly synthesized daughter strand ([Figure 1b](https://www.nature.com/articles/npp2012112#Fig1)). All three Dnmts are extensively involved in the development of an embryo. By the time cells reach terminal differentiation, Dnmt expression is much reduced. This would seem to suggest that the DNA methylation pattern in postmitotic cells is stable. However, postmitotic neurons in the mature mammalian brain still express substantial levels of Dnmts, raising the possibility that Dnmts and DNA methylation may play a novel role in the brain ([Goto et al, 1994](https://www.nature.com/articles/npp2012112#ref-CR65 "Goto K, Numata M, Komura JI, Ono T, Bestor TH, Kondo H \(1994\). Expression of DNA methyltransferase gene in mature and immature neurons as well as proliferating cells in mice. Differentiation 56: 39–44. Identified DNA methyltransferase mRNA in postmitotic neurons and suggested that DNA methylation served a unique function in neurons."); [Feng et al, 2005](https://www.nature.com/articles/npp2012112#ref-CR49 "Feng J, Chang H, Li E, Fan G \(2005\). Dynamic expression of de novo DNA methyltransferases Dnmt3a and Dnmt3b in the central nervous system. J Neurosci Res 79: 734–746.")).
**Figure 1**
[![figure 1](https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fnpp.2012.112/MediaObjects/41386_2013_Article_BFnpp2012112_Fig1_HTML.jpg)](https://www.nature.com/articles/npp2012112/figures/1)
DNA methylation pathways. A family of DNA methyltransferases (Dnmts) catalyzes the transfer of a methyl group from _S_ -adenyl methionine (SAM) to the fifth carbon of cytosine residue to form 5-methylcytosine (5mC). (a) Dnmt3a and Dnmt3b are the _de novo_ Dnmts and transfer methyl groups (red) onto naked DNA. (b) Dnmt1 is the maintenance Dnmt and maintains DNA methylation pattern during replication. When DNA undergoes semiconservative replication, the parental DNA stand retains the original DNA methylation pattern (gray). Dnmt1 associates at the replication foci and precisely replicates the original DNA methylation pattern by adding methyl groups (red) onto the newly formed daughter strand (blue).
[PowerPoint slide](https://www.nature.com/articles/npp2012112#MOESM1)
[Full size image](https://www.nature.com/articles/npp2012112/figures/1)
Neurons react to the environment through patterns of depolarization that both relay information and encode a response. In recent years, it has become increasingly apparent that following depolarization, alterations in gene expression are accompanied by modifications of the epigenetic landscape that include alterations in the pattern of DNA methylation ([Martinowich et al, 2003](https://www.nature.com/articles/npp2012112#ref-CR129 "Martinowich K, Hattori D, Wu H, Fouse S, He F, Hu Y et al \(2003\). DNA methylation-related chromatin remodeling in activity-dependent BDNF gene regulation. Science 302: 890–893. Demonstrated that methylation and MeCP2 binding to the BDNF promoter are regulated by neuronal activity."); [Guo et al, 2011a](https://www.nature.com/articles/npp2012112#ref-CR66 "Guo J, Ma DK, Mo H, Ball MP, Jang MH, Bonaguidi MA et al \(2011a\). Neuronal activity modifies the DNA methylation landscape in the adult brain. Nat Neurosci 14: 1345–1351. Investigated the extent to which the DNA methylation pattern is modifiable by neural activity in vivo.")). In order for the DNA methylation pattern to be altered, there must be both active DNA methylation and demethylation in the neuronal genome. However, no enzymes are known to cleave the methyl group directly from 5mC. As discussed below, the recent identification of 5-hydroxymethyl-cytosine (5hmC) in postmitotic neurons suggests that 5hmC serves as an intermediate in the DNA demethylation pathway. In this review, we will discuss the basic function of DNA methylation in epigenetic gene regulation, and further highlight its role in neural development and neurological disease.
## LOCATION OF DNA METHYLATION
Although the brain contains some of the highest levels of DNA methylation of any tissue in the body, 5mC only accounts for ∼1% of nucleic acids in the human genome ([Ehrlich et al, 1982](https://www.nature.com/articles/npp2012112#ref-CR45 "Ehrlich M, Gama-Sosa MA, Huang LH, Midgett RM, Kuo KC, McCune RA et al \(1982\). Amount and distribution of 5-methylcytosine in human DNA from different types of tissues of cells. Nucleic Acids Res 10: 2709–2721.")). The majority of DNA methylation occurs on cytosines that precede a guanine nucleotide or CpG sites. Overall, mammalian genomes are depleted of CpG sites that may result from the mutagenic potential of 5mC that can deaminate to thymine ([Coulondre et al, 1978](https://www.nature.com/articles/npp2012112#ref-CR36 "Coulondre C, Miller JH, Farabaugh PJ, Gilbert W \(1978\). Molecular basis of base substitution hotspots in Escherichia coli. Nature 274: 775–780."); [Bird, 1980](https://www.nature.com/articles/npp2012112#ref-CR15 "Bird AP \(1980\). DNA methylation and the frequency of CpG in animal DNA. Nucleic Acids Res 8: 1499–1504.")). The remaining CpG sites are spread out across the genome where they are heavily methylated with the exception of CpG islands ([Bird et al, 1985](https://www.nature.com/articles/npp2012112#ref-CR14 "Bird AP, Taggart M, Frommer M, Miller OJ, Macleod D \(1985\). A fraction of the mouse genome that is derived from islands of nonmethylated, CpG-rich DNA. Cell 40: 91–99. One of the pioneering studies that first identified CpG islands and described their sequence characteristics.")). Interestingly, there is evidence of non-CpG methylation in mouse and human embryonic stem cells, however these methylation are lost in mature tissues (Ramsahoye _et al_ , 2000; Lister _et al_ , 2009). More thorough analysis of the murine frontal cortex has recently revealed that although the majority of methylation occurs within CpG sites, there is a significant percentage of methylated non-CpG sites ([Xie et al, 2012](https://www.nature.com/articles/npp2012112#ref-CR202 "Xie W, Barr CL, Kim A, Yue F, Lee AY, Eubanks J et al \(2012\). Base-resolution analyses of sequence and parent-of-origin dependent DNA methylation in the mouse genome. Cell 148: 816–831. Recent article that identified significant levels of non-CpG DNA methylation in the murine frontal cortex.")). Because of its recent discovery, the role of non-CpG methylation is still unclear.
DNA methylation is essential for silencing retroviral elements, regulating tissue-specific gene expression, genomic imprinting, and X chromosome inactivation. Importantly, DNA methylation in different genomic regions may exert different influences on gene activities based on the underlying genetic sequence. In the following sections, we will further elaborate upon the role of DNA methylation in different genomic regions.
### Intergenic Regions
Approximately 45% of the mammalian genome consists of transposable and viral elements that are silenced by bulk methylation ([Schulz et al, 2006](https://www.nature.com/articles/npp2012112#ref-CR172 "Schulz WA, Steinhoff C, Florl AR \(2006\). Methylation of endogenous human retroelements in health and disease. Curr Top Microbiol Immunol 310: 211–250.")). The vast majority of these elements are inactivated by DNA methylation or by mutations acquired over time as the result of the deamination of 5mC ([Walsh et al, 1998](https://www.nature.com/articles/npp2012112#ref-CR192 "Walsh CP, Chaillet JR, Bestor TH \(1998\). Transcription of IAP endogenous retroviruses is constrained by cytosine methylation. Nat Genet 20: 116–117.")). If expressed, these elements are potentially harmful as their replication and insertion can lead to gene disruption and DNA mutation ([Michaud et al, 1994](https://www.nature.com/articles/npp2012112#ref-CR136 "Michaud EJ, van Vugt MJ, Bultman SJ, Sweet HO, Davisson MT, Woychik RP \(1994\). Differential expression of a new dominant agouti allele \(Aiapy\) is correlated with methylation state and is influenced by parental lineage. Genes Dev 8: 1463–1472."); [Wu et al, 1997](https://www.nature.com/articles/npp2012112#ref-CR199 "Wu M, Rinchik EM, Wilkinson E, Johnson DK \(1997\). Inherited somatic mosaicism caused by an intracisternal A particle insertion in the mouse tyrosinase gene. Proc Natl Acad Sci USA 94: 890–894."); [Kuster et al, 1997](https://www.nature.com/articles/npp2012112#ref-CR105 "Kuster JE, Guarnieri MH, Ault JG, Flaherty L, Swiatek PJ \(1997\). IAP insertion in the murine LamB3 gene results in junctional epidermolysis bullosa. Mamm Genome 8: 673–681."); [Gwynn et al, 1998](https://www.nature.com/articles/npp2012112#ref-CR70 "Gwynn B, Lueders K, Sands MS, Birkenmeier EH \(1998\). Intracisternal A-particle element transposition into the murine beta-glucuronidase gene correlates with loss of enzyme activity: a new model for beta-glucuronidase deficiency in the C3H mouse. Mol Cell Biol 18: 6474–6481."); [Ukai et al, 2003](https://www.nature.com/articles/npp2012112#ref-CR189 "Ukai H, Ishii-Oba H, Ukai-Tadenuma M, Ogiu T, Tsuji H \(2003\). Formation of an active form of the interleukin-2/15 receptor beta-chain by insertion of the intracisternal A particle in a radiation-induced mouse thymic lymphoma and its role in tumorigenesis. Mol Carcinog 37: 110–119.")). The intracisternal A particle (IAP) is one of most aggressive retroviruses in the mouse genome ([Walsh et al, 1998](https://www.nature.com/articles/npp2012112#ref-CR192 "Walsh CP, Chaillet JR, Bestor TH \(1998\). Transcription of IAP endogenous retroviruses is constrained by cytosine methylation. Nat Genet 20: 116–117.")). IAP is heavily methylated throughout life in gametogenesis, development, and adulthood ([Walsh et al, 1998](https://www.nature.com/articles/npp2012112#ref-CR192 "Walsh CP, Chaillet JR, Bestor TH \(1998\). Transcription of IAP endogenous retroviruses is constrained by cytosine methylation. Nat Genet 20: 116–117."); [Gaudet et al, 2004](https://www.nature.com/articles/npp2012112#ref-CR57 "Gaudet F, Rideout III WM, Meissner A, Dausman J, Leonhardt H, Jaenisch R \(2004\). Dnmt1 expression in pre- and postimplantation embryogenesis and the maintenance of IAP silencing. Mol Cell Biol 24: 1640–1648.")). Even within the embryo when the rest of the genome is relatively hypomethylated, Dnmt1 maintains the repression of IAP elements ([Gaudet et al, 2004](https://www.nature.com/articles/npp2012112#ref-CR57 "Gaudet F, Rideout III WM, Meissner A, Dausman J, Leonhardt H, Jaenisch R \(2004\). Dnmt1 expression in pre- and postimplantation embryogenesis and the maintenance of IAP silencing. Mol Cell Biol 24: 1640–1648.")). When Dnmt1 is depleted by genetic mutations, leading to extensive hypomethylation, IAP elements are expressed ([Walsh et al, 1998](https://www.nature.com/articles/npp2012112#ref-CR192 "Walsh CP, Chaillet JR, Bestor TH \(1998\). Transcription of IAP endogenous retroviruses is constrained by cytosine methylation. Nat Genet 20: 116–117."); [Hutnick et al, 2010](https://www.nature.com/articles/npp2012112#ref-CR85 "Hutnick LK, Huang X, Loo TC, Ma Z, Fan G \(2010\). Repression of retrotransposal elements in mouse embryonic stem cells is primarily mediated by a DNA methylation-independent mechanism. J Biol Chem 285: 21082–21091.")). This demonstrates that within intergenic regions, one of the main roles of DNA methylation is to repress the expression of potentially harmful genetic elements.
### CpG Islands
CpG islands are stretches of DNA roughly 1000 base pairs long that have a higher CpG density than the rest of the genome but often are not methylated ([Bird et al, 1985](https://www.nature.com/articles/npp2012112#ref-CR14 "Bird AP, Taggart M, Frommer M, Miller OJ, Macleod D \(1985\). A fraction of the mouse genome that is derived from islands of nonmethylated, CpG-rich DNA. Cell 40: 91–99. One of the pioneering studies that first identified CpG islands and described their sequence characteristics.")). The majority of gene promoters, roughly 70%, reside within CpG islands ([Saxonov et al, 2006](https://www.nature.com/articles/npp2012112#ref-CR171 "Saxonov S, Berg P, Brutlag DL \(2006\). A genome-wide analysis of CpG dinucleotides in the human genome distinguishes two distinct classes of promoters. Proc Natl Acad Sci USA 103: 1412–1417.")). In particular, the promoters for housekeeping genes are often imbedded in CpG islands ([Gardiner-Garden and Frommer, 1987](https://www.nature.com/articles/npp2012112#ref-CR56 "Gardiner-Garden M, Frommer M \(1987\). CpG islands in vertebrate genomes. J Mol Biol 196: 261–282.")). CpG islands, especially those associated with promoters, are highly conserved between mice and humans ([Illingworth et al, 2010](https://www.nature.com/articles/npp2012112#ref-CR86 "Illingworth RS, Gruenewald-Schneider U, Webb S, Kerr AR, James KD, Turner DJ et al \(2010\). Orphan CpG islands identify numerous conserved promoters in the mammalian genome. PLoS Genet 6: e1001134.")). The location and preservation of CpG islands throughout evolution implies that these regions possess a functional importance.
It appears that CpG islands have been evolutionarily conserved to promote gene expression by regulating the chromatin structure and transcription factor binding. DNA is regularly wrapped around histone proteins forming small, packaged sections called nucleosomes. The more tightly associated with histone proteins the DNA is, the less permissive it is for gene expression. One of the common features of CpG islands is that they contain less nucleosomes than other stretches of DNA ([Tazi and Bird, 1990](https://www.nature.com/articles/npp2012112#ref-CR184 "Tazi J, Bird A \(1990\). Alternative chromatin structure at CpG islands. Cell 60: 909–920."); [Ramirez-Carrozzi et al, 2009](https://www.nature.com/articles/npp2012112#ref-CR165 "Ramirez-Carrozzi VR, Braas D, Bhatt DM, Cheng CS, Hong C, Doty KR et al \(2009\). A unifying model for the selective regulation of inducible transcription by CpG islands and nucleosome remodeling. Cell 138: 114–128."); [Choi, 2010](https://www.nature.com/articles/npp2012112#ref-CR31 "Choi JK \(2010\). Contrasting chromatin organization of CpG islands and exons in the human genome. Genome Biol 11: R70.")). The few nucleosomes with which CpG islands are associated often contain histones with modifications involved in enhancing gene expression ([Tazi and Bird, 1990](https://www.nature.com/articles/npp2012112#ref-CR184 "Tazi J, Bird A \(1990\). Alternative chromatin structure at CpG islands. Cell 60: 909–920."); [Mikkelsen et al, 2007](https://www.nature.com/articles/npp2012112#ref-CR137 "Mikkelsen TS, Ku M, Jaffe DB, Issac B, Lieberman E, Giannoukos G et al \(2007\). Genome-wide maps of chromatin state in pluripotent and lineage-committed cells. Nature 448: 553–560.")). Although ∼50% of CpG islands contain known transcription start sites, CpG islands are often devoid of common promoter elements such as TATA boxes ([Carninci et al, 2006](https://www.nature.com/articles/npp2012112#ref-CR24 "Carninci P, Sandelin A, Lenhard B, Katayama S, Shimokawa K, Ponjavic J et al \(2006\). Genome-wide analysis of mammalian promoter architecture and evolution. Nat Genet 38: 626–635.")). As many transcription factor binding sites are GC rich, CpG islands are likely to enhance binding to transcriptional start sites. Despite their lack of common promoter elements, CpG islands enhance the accessibility of DNA and promote transcription factor binding.
The methylation of CpG islands results in stable silencing of gene expression ([Mohn et al, 2008](https://www.nature.com/articles/npp2012112#ref-CR142 "Mohn F, Weber M, Rebhan M, Roloff TC, Richter J, Stadler MB et al \(2008\). Lineage-specific polycomb targets and de novo DNA methylation define restriction and potential of neuronal progenitors. Mol Cell 30: 755–766.")). During gametogenesis and early embryonic development, CpG islands undergo differential methylation ([Wutz et al, 1997](https://www.nature.com/articles/npp2012112#ref-CR200 "Wutz A, Smrzka OW, Schweifer N, Schellander K, Wagner EF, Barlow DP \(1997\). Imprinted expression of the Igf2r gene depends on an intronic CpG island. Nature 389: 745–749."); [Caspary et al, 1998](https://www.nature.com/articles/npp2012112#ref-CR25 "Caspary T, Cleary MA, Baker CC, Guan XJ, Tilghman SM \(1998\). Multiple mechanisms regulate imprinting of the mouse distal chromosome 7 gene cluster. Mol Cell Biol 18: 3466–3474."); [Zwart et al, 2001](https://www.nature.com/articles/npp2012112#ref-CR210 "Zwart R, Sleutels F, Wutz A, Schinkel AH, Barlow DP \(2001\). Bidirectional action of the Igf2r imprint control element on upstream and downstream imprinted genes. Genes Dev 15: 2361–2366."); [Kantor et al, 2004](https://www.nature.com/articles/npp2012112#ref-CR98 "Kantor B, Kaufman Y, Makedonski K, Razin A, Shemer R \(2004\). Establishing the epigenetic status of the Prader-Willi/Angelman imprinting center in the gametes and embryo. Hum Mol Genet 13: 2767–2779.")). The ability of methylation to regulate gene expression through CpG islands is particularly important for establishing imprinting ([Wutz et al, 1997](https://www.nature.com/articles/npp2012112#ref-CR200 "Wutz A, Smrzka OW, Schweifer N, Schellander K, Wagner EF, Barlow DP \(1997\). Imprinted expression of the Igf2r gene depends on an intronic CpG island. Nature 389: 745–749."); [Caspary et al, 1998](https://www.nature.com/articles/npp2012112#ref-CR25 "Caspary T, Cleary MA, Baker CC, Guan XJ, Tilghman SM \(1998\). Multiple mechanisms regulate imprinting of the mouse distal chromosome 7 gene cluster. Mol Cell Biol 18: 3466–3474."); [Zwart et al, 2001](https://www.nature.com/articles/npp2012112#ref-CR210 "Zwart R, Sleutels F, Wutz A, Schinkel AH, Barlow DP \(2001\). Bidirectional action of the Igf2r imprint control element on upstream and downstream imprinted genes. Genes Dev 15: 2361–2366."); [Choi et al, 2005](https://www.nature.com/articles/npp2012112#ref-CR30 "Choi JD, Underkoffler LA, Wood AJ, Collins JN, Williams PT, Golden JA et al \(2005\). A novel variant of Inpp5f is imprinted in brain, and its expression is correlated with differential methylation of an internal CpG island. Mol Cell Biol 25: 5514–5522.")). Imprinted genes are expressed from only one of the two inherited parental chromosomes and their expression is determined by the parent of inheritance. Beyond imprinted genes, DNA methylation of CpG islands regulates gene expression during development and differentiation ([Shen et al, 2007](https://www.nature.com/articles/npp2012112#ref-CR175 "Shen L, Kondo Y, Guo Y, Zhang J, Zhang L, Ahmed S et al \(2007\). Genome-wide profiling of DNA methylation reveals a class of normally methylated CpG island promoters. PLoS Genet 3: 2023–2036."); [Weber et al, 2007](https://www.nature.com/articles/npp2012112#ref-CR194 "Weber M, Hellmann I, Stadler MB, Ramos L, Paabo S, Rebhan M et al \(2007\). Distribution, silencing potential and evolutionary impact of promoter DNA methylation in the human genome. Nat Genet 39: 457–466."); [Fouse et al, 2008](https://www.nature.com/articles/npp2012112#ref-CR53 "Fouse SD, Shen Y, Pellegrini M, Cole S, Meissner A, Van Neste L et al \(2008\). Promoter CpG methylation contributes to ES cell gene regulation in parallel with Oct4/Nanog, PcG complex, and histone H3 K4/K27 trimethylation. Cell Stem Cell 2: 160–169."); [Mohn et al, 2008](https://www.nature.com/articles/npp2012112#ref-CR142 "Mohn F, Weber M, Rebhan M, Roloff TC, Richter J, Stadler MB et al \(2008\). Lineage-specific polycomb targets and de novo DNA methylation define restriction and potential of neuronal progenitors. Mol Cell 30: 755–766."); [Meissner et al, 2008](https://www.nature.com/articles/npp2012112#ref-CR135 "Meissner A, Mikkelsen TS, Gu H, Wernig M, Hanna J, Sivachenko A et al \(2008\). Genome-scale DNA methylation maps of pluripotent and differentiated cells. Nature 454: 766–770.")). As CpG islands are associated with the control of gene expression, it would be expected that CpG islands might display tissue-specific patterns of DNA methylation. Although CpG islands in intragenic and gene body regions can have tissue-specific patterns of methylation, CpG islands associated with transcription start sites rarely show tissue-specific methylation patterns ([Rakyan et al, 2004](https://www.nature.com/articles/npp2012112#ref-CR164 "Rakyan VK, Hildmann T, Novik KL, Lewin J, Tost J, Cox AV et al \(2004\). DNA methylation profiling of the human major histocompatibility complex: a pilot study for the human epigenome project. PLoS Biol 2: e405."); [Eckhardt et al, 2006](https://www.nature.com/articles/npp2012112#ref-CR44 "Eckhardt F, Lewin J, Cortese R, Rakyan VK, Attwood J, Burger M et al \(2006\). DNA methylation profiling of human chromosomes 6, 20 and 22. Nat Genet 38: 1378–1385."); [Meissner et al, 2008](https://www.nature.com/articles/npp2012112#ref-CR135 "Meissner A, Mikkelsen TS, Gu H, Wernig M, Hanna J, Sivachenko A et al \(2008\). Genome-scale DNA methylation maps of pluripotent and differentiated cells. Nature 454: 766–770."); [Illingworth et al, 2010](https://www.nature.com/articles/npp2012112#ref-CR86 "Illingworth RS, Gruenewald-Schneider U, Webb S, Kerr AR, James KD, Turner DJ et al \(2010\). Orphan CpG islands identify numerous conserved promoters in the mammalian genome. PLoS Genet 6: e1001134."); [Maunakea et al, 2010](https://www.nature.com/articles/npp2012112#ref-CR130 "Maunakea AK, Nagarajan RP, Bilenky M, Ballinger TJ, D’Souza C, Fouse SD et al \(2010\). Conserved role of intragenic DNA methylation in regulating alternative promoters. Nature 466: 253–257.")). Instead, regions called CpG island shores, located as far as 2 kb from CpG islands, have highly conserved patterns of tissue-specific methylation ([Irizarry et al, 2009](https://www.nature.com/articles/npp2012112#ref-CR89 "Irizarry RA, Ladd-Acosta C, Wen B, Wu Z, Montano C, Onyango P et al \(2009\). The human colon cancer methylome shows similar hypo- and hypermethylation at conserved tissue-specific CpG island shores. Nat Genet 41: 178–186.")). Like CpG islands, the methylation of CpG shores is highly correlated with reduced gene expression ([Irizarry et al, 2009](https://www.nature.com/articles/npp2012112#ref-CR89 "Irizarry RA, Ladd-Acosta C, Wen B, Wu Z, Montano C, Onyango P et al \(2009\). The human colon cancer methylome shows similar hypo- and hypermethylation at conserved tissue-specific CpG island shores. Nat Genet 41: 178–186.")).
The role of CpG islands in regulating gene expression is still being uncovered. Methylation of CpG islands can impair transcription factor binding, recruit repressive methyl-binding proteins, and stably silence gene expression. However, CpG islands, especially those associated with gene promoters, are rarely methylated. Further studies are needed to determine to what degree DNA methylation of CpG islands regulates gene expression.
### Gene Body
As the majority of CpG sites within the mammalian genome are methylated, the genes themselves must also contain methylation. The gene body is considered the region of the gene past the first exon because methylation of the first exon, like promoter methylation, leads to gene silencing ([Brenet et al, 2011](https://www.nature.com/articles/npp2012112#ref-CR20 "Brenet F, Moh M, Funk P, Feierstein E, Viale AJ, Socci ND et al \(2011\). DNA methylation of the first exon is tightly linked to transcriptional silencing. PLoS One 6: e14524.")). Evidence suggests that DNA methylation of the gene body is associated with a higher level of gene expression in dividing cells ([Hellman and Chess, 2007](https://www.nature.com/articles/npp2012112#ref-CR76 "Hellman A, Chess A \(2007\). Gene body-specific methylation on the active X chromosome. Science 315: 1141–1143."); [Ball et al, 2009](https://www.nature.com/articles/npp2012112#ref-CR8 "Ball MP, Li JB, Gao Y, Lee JH, LeProust EM, Park IH et al \(2009\). Targeted and genome-scale strategies reveal gene-body methylation signatures in human cells. Nat Biotechnol 27: 361–368."); [Aran et al, 2011](https://www.nature.com/articles/npp2012112#ref-CR5 "Aran D, Toperoff G, Rosenberg M, Hellman A \(2011\). Replication timing-related and gene body-specific methylation of active human genes. Hum Mol Genet 20: 670–680.")). However, in slowly dividing and nondividing cells such as the brain, gene body methylation is not associated with increased gene expression ([Aran et al, 2011](https://www.nature.com/articles/npp2012112#ref-CR5 "Aran D, Toperoff G, Rosenberg M, Hellman A \(2011\). Replication timing-related and gene body-specific methylation of active human genes. Hum Mol Genet 20: 670–680."); [Guo et al, 2011a](https://www.nature.com/articles/npp2012112#ref-CR66 "Guo J, Ma DK, Mo H, Ball MP, Jang MH, Bonaguidi MA et al \(2011a\). Neuronal activity modifies the DNA methylation landscape in the adult brain. Nat Neurosci 14: 1345–1351. Investigated the extent to which the DNA methylation pattern is modifiable by neural activity in vivo."), [2011b](https://www.nature.com/articles/npp2012112#ref-CR67 "Guo J, Su Y, Zhong C, Ming GL, Song H \(2011b\). Hydroxylation of 5-methylcytosine by TET1 promotes active DNA demethylation in the adult brain. Cell 145: 423–434. Also discovered few gene regions, including BDNF, that exhibit demethylation and remethylation cycle in postmitotic neurons."); [Xie et al, 2012](https://www.nature.com/articles/npp2012112#ref-CR202 "Xie W, Barr CL, Kim A, Yue F, Lee AY, Eubanks J et al \(2012\). Base-resolution analyses of sequence and parent-of-origin dependent DNA methylation in the mouse genome. Cell 148: 816–831. Recent article that identified significant levels of non-CpG DNA methylation in the murine frontal cortex.")). Furthermore, in the murine frontal cortex, methylation of non-CpG sites within gene bodies is negatively correlated with gene expression ([Xie et al, 2012](https://www.nature.com/articles/npp2012112#ref-CR202 "Xie W, Barr CL, Kim A, Yue F, Lee AY, Eubanks J et al \(2012\). Base-resolution analyses of sequence and parent-of-origin dependent DNA methylation in the mouse genome. Cell 148: 816–831. Recent article that identified significant levels of non-CpG DNA methylation in the murine frontal cortex.")). How DNA methylation of the gene body contributes to gene regulation is still unclear.
## BASIC MECHANISM OF DNA METHYLATION
The enzymes that establish, recognize, and remove DNA methylation are broken into three classes: writers, erasers, and readers. Writers are the enzymes that catalyze the addition of methyl groups onto cytosine residues. Erasers modify and remove the methyl group. Readers recognize and bind to methyl groups to ultimately influence gene expression. Thanks to the many years of research devoted to understanding how the epigenetic landscape is erased and reshaped during embryonic development, many of the proteins and mechanisms involved in DNA methylation have already been identified.
### Writing DNA Methylation: the Dnmts
Three members of the Dnmt family directly catalyze the addition of methyl groups onto DNA: Dnmt1, Dnmt3a, and Dnmt3b. Although these enzymes share a similar structure with a large N-terminal regulatory domain and a C-terminal catalytic domain, they have unique functions and expression patterns ([Yen et al, 1992](https://www.nature.com/articles/npp2012112#ref-CR203 "Yen RW, Vertino PM, Nelkin BD, Yu JJ, el-Deiry W, Cumaraswamy A et al \(1992\). Isolation and characterization of the cDNA encoding human DNA methyltransferase. Nucleic Acids Res 20: 2287–2291."); [Xie et al, 1999](https://www.nature.com/articles/npp2012112#ref-CR201 "Xie S, Wang Z, Okano M, Nogami M, Li Y, He WW et al \(1999\). Cloning, expression and chromosome locations of the human DNMT3 gene family. Gene 236: 87–95.")). Probably the best studied Dnmt, especially in the nervous system, is Dnmt1, which is highly expressed in mammalian tissues including the brain ([Goto et al, 1994](https://www.nature.com/articles/npp2012112#ref-CR65 "Goto K, Numata M, Komura JI, Ono T, Bestor TH, Kondo H \(1994\). Expression of DNA methyltransferase gene in mature and immature neurons as well as proliferating cells in mice. Differentiation 56: 39–44. Identified DNA methyltransferase mRNA in postmitotic neurons and suggested that DNA methylation served a unique function in neurons.")). Unlike the other Dnmts, Dnmt1 preferentially methylates hemimethylated DNA ([Pradhan et al, 1999](https://www.nature.com/articles/npp2012112#ref-CR160 "Pradhan S, Bacolla A, Wells RD, Roberts RJ \(1999\). Recombinant human DNA \(cytosine-5\) methyltransferase. I. Expression, purification, and comparison of de novo and maintenance methylation. J Biol Chem 274: 33002–33010."); [Ramsahoye et al, 2000](https://www.nature.com/articles/npp2012112#ref-CR166 "Ramsahoye BH, Biniszkiewicz D, Lyko F, Clark V, Bird AP, Jaenisch R \(2000\). Non-CpG methylation is prevalent in embryonic stem cells and may be mediated by DNA methyltransferase 3a. Proc Natl Acad Sci USA 97: 5237–5242. First study to demonstrate the presence of non-CG methylation in mammalian cells.")). During DNA replication, Dnmt1 localizes to the replication fork where newly synthesized hemimethylated DNA is formed ([Leonhardt et al, 1992](https://www.nature.com/articles/npp2012112#ref-CR113 "Leonhardt H, Page AW, Weier HU, Bestor TH \(1992\). A targeting sequence directs DNA methyltransferase to sites of DNA replication in mammalian nuclei. Cell 71: 865–873.")). Dnmt1 binds to the newly synthesized DNA and methylates it to precisely mimic the original methylation pattern present before DNA replication ([Hermann et al, 2004](https://www.nature.com/articles/npp2012112#ref-CR79 "Hermann A, Goyal R, Jeltsch A \(2004\). The Dnmt1 DNA-\(cytosine-C5\)-methyltransferase methylates DNA processively with high preference for hemimethylated target sites. J Biol Chem 279: 48350–48359.")) ([Figure 1b](https://www.nature.com/articles/npp2012112#Fig1)). Additionally, Dnmt1 also has the ability to repair DNA methylation ([Mortusewicz et al, 2005](https://www.nature.com/articles/npp2012112#ref-CR145 "Mortusewicz O, Schermelleh L, Walter J, Cardoso MC, Leonhardt H \(2005\). Recruitment of DNA methyltransferase I to DNA repair sites. Proc Natl Acad Sci USA 102: 8905–8909.")). For this reason, Dnmt1 is called the maintenance Dnmt because it maintains the original pattern of DNA methylation in a cell lineage. Knockout of _Dnmt1_ in mice results in embryonic lethality between E8.0 and E10.5 ([Li et al, 1992](https://www.nature.com/articles/npp2012112#ref-CR117 "Li E, Bestor TH, Jaenisch R \(1992\). Targeted mutation of the DNA methyltransferase gene results in embryonic lethality. Cell 69: 915–926. Developed the Dnmt1 knockout mice.")). At this time, knockout embryos exhibit a two-thirds loss of DNA methylation, in addition to numerous apoptotic cells in a variety of developing tissues including the brain. Interestingly, mouse embryonic stem cells lacking _Dnmt1_ remain viable ([Chen et al, 1998](https://www.nature.com/articles/npp2012112#ref-CR29 "Chen RZ, Pettersson U, Beard C, Jackson-Grusby L, Jaenisch R \(1998\). DNA hypomethylation leads to elevated mutation rates. Nature 395: 89–93.")). However, _in vitro_ differentiation results in massive cell death, recapitulating the phenotype observed in knockout embryos ([Jackson-Grusby et al, 2001](https://www.nature.com/articles/npp2012112#ref-CR92 "Jackson-Grusby L, Beard C, Possemato R, Tudor M, Fambrough D, Csankovszki G et al \(2001\). Loss of genomic methylation causes p53-dependent apoptosis and epigenetic deregulation. Nat Genet 27: 31–39.")). These findings firmly establish that Dnmt1 plays a critical role in cellular differentiation as well as in dividing cells.
Dnmt3a and Dnmt3b are extremely similar in structure and function. Unlike Dnmt1, both Dnmt3a and Dnmt3b when overexpressed are capable of methylating both native and synthetic DNA with no preference for hemimethylated DNA ([Okano et al, 1999](https://www.nature.com/articles/npp2012112#ref-CR153 "Okano M, Bell DW, Haber DA, Li E \(1999\). DNA methyltransferases Dnmt3a and Dnmt3b are essential for de novo methylation and mammalian development. Cell 99: 247–257. Demonstrated the function of Dnmt3 enzymes by studying both Dnmt3a and Dnmt3b knockout mice.")). For this reason, Dnmt3a and Dnmt3b are referred to as _de novo_ Dnmt because they can introduce methylation into naked DNA ([Figure 1a](https://www.nature.com/articles/npp2012112#Fig1)). What primarily distinguishes Dnmt3a from Dnmt3b is its gene expression pattern. Although _Dnmt3a_ is expressed relatively ubiquitously, _Dnmt3b_ is poorly expressed by the majority of differentiated tissues with the exception of the thyroid, testes, and bone marrow ([Xie et al, 1999](https://www.nature.com/articles/npp2012112#ref-CR201 "Xie S, Wang Z, Okano M, Nogami M, Li Y, He WW et al \(1999\). Cloning, expression and chromosome locations of the human DNMT3 gene family. Gene 236: 87–95.")). Similar to _Dnmt1_ , the knockout of _Dnmt3b_ in mice is embryonic lethal ([Okano et al, 1999](https://www.nature.com/articles/npp2012112#ref-CR153 "Okano M, Bell DW, Haber DA, Li E \(1999\). DNA methyltransferases Dnmt3a and Dnmt3b are essential for de novo methylation and mammalian development. Cell 99: 247–257. Demonstrated the function of Dnmt3 enzymes by studying both Dnmt3a and Dnmt3b knockout mice.")). On the other hand, _Dnmt3a_ knockout mice are runted but survive to ∼4 weeks after birth ([Okano et al, 1999](https://www.nature.com/articles/npp2012112#ref-CR153 "Okano M, Bell DW, Haber DA, Li E \(1999\). DNA methyltransferases Dnmt3a and Dnmt3b are essential for de novo methylation and mammalian development. Cell 99: 247–257. Demonstrated the function of Dnmt3 enzymes by studying both Dnmt3a and Dnmt3b knockout mice.")). From these results it appears that _Dnmt3b_ is required during early development, whereas _Dnmt3a_ is required for normal cellular differentiation.
A final member of the Dnmt family is Dnmt3L, a protein that lacks the catalytic domain present in other Dnmt enzymes ([Aapola et al, 2000](https://www.nature.com/articles/npp2012112#ref-CR1 "Aapola U, Kawasaki K, Scott HS, Ollila J, Vihinen M, Heino M et al \(2000\). Isolation and initial characterization of a novel zinc finger gene, DNMT3L, on 21q22.3, related to the cytosine-5-methyltransferase 3 gene family. Genomics 65: 293–298. Identified Dnmt3L and profiled its expression."); [Hata et al, 2002](https://www.nature.com/articles/npp2012112#ref-CR74 "Hata K, Okano M, Lei H, Li E \(2002\). Dnmt3L cooperates with the Dnmt3 family of de novo DNA methyltransferases to establish maternal imprints in mice. Development 129: 1983–1993.")). Dnmt3L is mainly expressed in early development and is restricted to the germ cells and thymus in adulthood ([Aapola et al, 2000](https://www.nature.com/articles/npp2012112#ref-CR1 "Aapola U, Kawasaki K, Scott HS, Ollila J, Vihinen M, Heino M et al \(2000\). Isolation and initial characterization of a novel zinc finger gene, DNMT3L, on 21q22.3, related to the cytosine-5-methyltransferase 3 gene family. Genomics 65: 293–298. Identified Dnmt3L and profiled its expression."), [2001](https://www.nature.com/articles/npp2012112#ref-CR2 "Aapola U, Lyle R, Krohn K, Antonarakis SE, Peterson P \(2001\). Isolation and initial characterization of the mouse Dnmt3l gene. Cytogenet Cell Genet 92: 122–126.")). Although Dnmt3L has no catalytic function of its own, it associates with the Dnmt3a and Dnmt3b and stimulates their methyltransferase activity ([Hata et al, 2002](https://www.nature.com/articles/npp2012112#ref-CR74 "Hata K, Okano M, Lei H, Li E \(2002\). Dnmt3L cooperates with the Dnmt3 family of de novo DNA methyltransferases to establish maternal imprints in mice. Development 129: 1983–1993."); [Suetake et al, 2004](https://www.nature.com/articles/npp2012112#ref-CR179 "Suetake I, Shinozaki F, Miyagawa J, Takeshima H, Tajima S \(2004\). DNMT3L stimulates the DNA methylation activity of Dnmt3a and Dnmt3b through a direct interaction. J Biol Chem 279: 27816–27823."); [Jia et al, 2007](https://www.nature.com/articles/npp2012112#ref-CR93 "Jia D, Jurkowska RZ, Zhang X, Jeltsch A, Cheng X \(2007\). Structure of Dnmt3a bound to Dnmt3L suggests a model for de novo DNA methylation. Nature 449: 248–251.")). Consistent with its presence in early development and in germ cells, in mice, Dnmt3L is required for establishing both maternal and paternal genomic imprinting, for methylating retrotransposons, and for compaction of the X chromosome ([Bourc’his et al, 2001](https://www.nature.com/articles/npp2012112#ref-CR18 "Bourc’his D, Xu GL, Lin CS, Bollman B, Bestor TH \(2001\). Dnmt3L and the establishment of maternal genomic imprints. Science 294: 2536–2539. Demonstrated the importance of Dnmt3L in the establishment of genomic imprints."); [Hata et al, 2002](https://www.nature.com/articles/npp2012112#ref-CR74 "Hata K, Okano M, Lei H, Li E \(2002\). Dnmt3L cooperates with the Dnmt3 family of de novo DNA methyltransferases to establish maternal imprints in mice. Development 129: 1983–1993."); [Kaneda et al, 2004](https://www.nature.com/articles/npp2012112#ref-CR97 "Kaneda M, Okano M, Hata K, Sado T, Tsujimoto N, Li E et al \(2004\). Essential role for de novo DNA methyltransferase Dnmt3a in paternal and maternal imprinting. Nature 429: 900–903."); [Bourc’his and Bestor, 2004](https://www.nature.com/articles/npp2012112#ref-CR17 "Bourc’his D, Bestor TH \(2004\). Meiotic catastrophe and retrotransposon reactivation in male germ cells lacking Dnmt3L. Nature 431: 96–99."); [Webster et al, 2005](https://www.nature.com/articles/npp2012112#ref-CR195 "Webster KE, O’Bryan MK, Fletcher S, Crewther PE, Aapola U, Craig J et al \(2005\). Meiotic and epigenetic defects in Dnmt3L-knockout mouse spermatogenesis. Proc Natl Acad Sci USA 102: 4068–4073."); [La Salle et al, 2007](https://www.nature.com/articles/npp2012112#ref-CR106 "La Salle S, Oakes CC, Neaga OR, Bourc’his D, Bestor TH, Trasler JM \(2007\). Loss of spermatogonia and wide-spread DNA methylation defects in newborn male mice deficient in DNMT3L. BMC Dev Biol 7: 104."); [Zamudio et al, 2011](https://www.nature.com/articles/npp2012112#ref-CR205 "Zamudio NM, Scott HS, Wolski K, Lo CY, Law C, Leong D et al \(2011\). DNMT3L is a regulator of X chromosome compaction and post-meiotic gene transcription. PLoS One 6: e18276.")). Although Dnmt3L is expressed in the developing brain, Dnmt3L is downregulated during neuronal differentiation and is not observed in the brain postnatally ([Lee et al, 2006](https://www.nature.com/articles/npp2012112#ref-CR112 "Lee MS, Jun DH, Hwang CI, Park SS, Kang JJ, Park HS et al \(2006\). Selection of neural differentiation-specific genes by comparing profiles of random differentiation. Stem Cells 24: 1946–1955."); [Kovacheva et al, 2007](https://www.nature.com/articles/npp2012112#ref-CR103 "Kovacheva VP, Mellott TJ, Davison JM, Wagner N, Lopez-Coviella I, Schnitzler AC et al \(2007\). Gestational choline deficiency causes global and Igf2 gene DNA hypermethylation by up-regulation of Dnmt1 expression. J Biol Chem 282: 31777–31788.")).
### Writing DNA Methylation: Targeting _De Novo_ DNA Methylation
(How the _de novo_ Dnmts target specific genetic regions is still unclear). However, several mechanisms have been proposed. Dnmt3a and Dnmt3b can bind to DNA via a conserved PWWP domain ([Ge et al, 2004](https://www.nature.com/articles/npp2012112#ref-CR58 "Ge YZ, Pu MT, Gowher H, Wu HP, Ding JP, Jeltsch A et al \(2004\). Chromatin targeting of de novo DNA methyltransferases by the PWWP domain. J Biol Chem 279: 25447–25454.")); however, it is unclear how Dnmt3a and Dnmt3b target specific DNA sequences. One hypothesis suggests that RNA interference (RNAi) mechanisms target Dnmts to silence specific sequences of DNA ([Morris et al, 2004](https://www.nature.com/articles/npp2012112#ref-CR144 "Morris KV, Chan SW, Jacobsen SE, Looney DJ \(2004\). Small interfering RNA-induced transcriptional gene silencing in human cells. Science 305: 1289–1292.")). Although RNAi is clearly involved in DNA methylation in plant cells, the existing evidence is still very weak for a role of RNAi in DNA methylation in mammalian cells. The other theory is that transcription factors regulate _de novo_ DNA methylation. Transcription factors can regulate DNA methylation by binding to specific DNA sequence to either recruit Dnmts for methylation or protect from DNA methylation. In some cases Dnmts bind to transcription factors or components of repressor complexes to target methylation to DNA ([Brenner et al, 2005](https://www.nature.com/articles/npp2012112#ref-CR21 "Brenner C, Deplus R, Didelot C, Loriot A, Vire E, De Smet C et al \(2005\). Myc represses transcription through recruitment of DNA methyltransferase corepressor. EMBO J 24: 336–346.")). In other cases, regardless of whether the gene is expressed, the transcription factor binding can help protect CpG sites from _de novo_ methylation ([Straussman et al, 2009](https://www.nature.com/articles/npp2012112#ref-CR177 "Straussman R, Nejman D, Roberts D, Steinfeld I, Blum B, Benvenisty N et al \(2009\). Developmental programming of CpG island methylation profiles in the human genome. Nat Struct Mol Biol 16: 564–571."); [Gebhard et al, 2010](https://www.nature.com/articles/npp2012112#ref-CR59 "Gebhard C, Benner C, Ehrich M, Schwarzfischer L, Schilling E, Klug M et al \(2010\). General transcription factor binding at CpG islands in normal cells correlates with resistance to de novo DNA methylation in cancer cells. Cancer Res 70: 1398–1407."); [Lienert et al, 2011](https://www.nature.com/articles/npp2012112#ref-CR119 "Lienert F, Wirbelauer C, Som I, Dean A, Mohn F, Schubeler D \(2011\). Identification of genetic elements that autonomously determine DNA methylation states. Nat Genet 43: 1091–1097. Demonstrated that transcription factor binding regulates DNA methylation by inserting 50 different DNA elements into the same chromosome region.")). CpG islands appear to primarily be protected from methylation by transcription factor binding ([Brandeis et al, 1994](https://www.nature.com/articles/npp2012112#ref-CR19 "Brandeis M, Frank D, Keshet I, Siegfried Z, Mendelsohn M, Nemes A et al \(1994\). Sp1 elements protect a CpG island from de novo methylation. Nature 371: 435–438."); [Macleod et al, 1994](https://www.nature.com/articles/npp2012112#ref-CR127 "Macleod D, Charlton J, Mullins J, Bird AP \(1994\). Sp1 sites in the mouse aprt gene promoter are required to prevent methylation of the CpG island. Genes Dev 8: 2282–2292."); [Straussman et al, 2009](https://www.nature.com/articles/npp2012112#ref-CR177 "Straussman R, Nejman D, Roberts D, Steinfeld I, Blum B, Benvenisty N et al \(2009\). Developmental programming of CpG island methylation profiles in the human genome. Nat Struct Mol Biol 16: 564–571."); [Gebhard et al, 2010](https://www.nature.com/articles/npp2012112#ref-CR59 "Gebhard C, Benner C, Ehrich M, Schwarzfischer L, Schilling E, Klug M et al \(2010\). General transcription factor binding at CpG islands in normal cells correlates with resistance to de novo DNA methylation in cancer cells. Cancer Res 70: 1398–1407.")). When transcription factor binding sites are mutated, CpG islands are unable to retain their unmethylated state ([Brandeis et al, 1994](https://www.nature.com/articles/npp2012112#ref-CR19 "Brandeis M, Frank D, Keshet I, Siegfried Z, Mendelsohn M, Nemes A et al \(1994\). Sp1 elements protect a CpG island from de novo methylation. Nature 371: 435–438."); [Macleod et al, 1994](https://www.nature.com/articles/npp2012112#ref-CR127 "Macleod D, Charlton J, Mullins J, Bird AP \(1994\). Sp1 sites in the mouse aprt gene promoter are required to prevent methylation of the CpG island. Genes Dev 8: 2282–2292.")). Similarly, as differentiation induces the downregulation of transcription factors that bind to specific gene promoters, the now-exposed CpG sites can be targeted for DNA methylation ([Lienert et al, 2011](https://www.nature.com/articles/npp2012112#ref-CR119 "Lienert F, Wirbelauer C, Som I, Dean A, Mohn F, Schubeler D \(2011\). Identification of genetic elements that autonomously determine DNA methylation states. Nat Genet 43: 1091–1097. Demonstrated that transcription factor binding regulates DNA methylation by inserting 50 different DNA elements into the same chromosome region.")). These studies describe two mechanisms that likely function together to establish _de novo_ DNA methylation. Dnmt3a and Dnmt3b can either be recruited to promoters by specific transcription factors or the _de novo_ Dnmt may simply methylate all CpG sites across the genome that are not protected by a bound transcription factor.
### Erasing DNA Methylation
DNA demethylation is characterized as either passive or active. Passive DNA demethylation occurs in dividing cells. As Dnmt1 actively maintains DNA methylation during cell replication, its inhibition or dysfunction allows newly incorporated cytosine to remain unmethylated and consequently reduces the overall methylation level following each cell division. Active DNA demethylation can occur in both dividing and nondividing cells but the process requires enzymatic reactions to process the 5mC in order to revert it back to a naked cytosine ([Mayer et al, 2000](https://www.nature.com/articles/npp2012112#ref-CR131 "Mayer W, Niveleau A, Walter J, Fundele R, Haaf T \(2000\). Demethylation of the zygotic paternal genome. Nature 403: 501–502."); [Oswald et al, 2000](https://www.nature.com/articles/npp2012112#ref-CR155 "Oswald J, Engemann S, Lane N, Mayer W, Olek A, Fundele R et al \(2000\). Active demethylation of the paternal genome in the mouse zygote. Curr Biol 10: 475–478."); [Paroush et al, 1990](https://www.nature.com/articles/npp2012112#ref-CR156 "Paroush Z, Keshet I, Yisraeli J, Cedar H \(1990\). Dynamics of demethylation and activation of the alpha-actin gene in myoblasts. Cell 63: 1229–1237."); [Zhang et al, 2007](https://www.nature.com/articles/npp2012112#ref-CR206 "Zhang F, Pomerantz JH, Sen G, Palermo AT, Blau HM \(2007\). Active tissue-specific DNA demethylation conferred by somatic cell nuclei in stable heterokaryons. Proc Natl Acad Sci USA 104: 4395–4400.")). As of yet, there is no known mechanism in mammalian cells that can cleave the strong covalent carbon-to-carbon bond that connects cytosine to a methyl group. Instead, demethylation occurs through a series of chemical reactions that further modify 5mC, by deamination and/or oxidation reactions to a product that is recognized by the base excision repair (BER) pathway to replace the modified base with naked cytosine. Although it is generally agreed upon that the BER pathway is the final step in DNA demethylation, the specific enzymes and the chemical intermediates that are formed during DNA demethylation are still debated ([Bhutani et al, 2011](https://www.nature.com/articles/npp2012112#ref-CR13 "Bhutani N, Burns DM, Blau HM \(2011\). DNA demethylation dynamics. Cell 146: 866–872.")).
Several mechanisms of active DNA demethylation have been proposed ([Figure 2](https://www.nature.com/articles/npp2012112#Fig2)). 5mC can be chemically modified at two sites, the amine group and the methyl group. Deamination of the amine to a carbonyl group by AID/APOBEC (activation-induced cytidine deaminase/apolipoprotein B mRNA-editing enzyme complex) effectively converts 5mC into thymine, thus creating a G/T mismatch and inducing the BER pathway to correct the base. Overexpression of AID/APOBEC promotes DNA demethylation in zebrafish ([Rai et al, 2008](https://www.nature.com/articles/npp2012112#ref-CR163 "Rai K, Huggins IJ, James SR, Karpf AR, Jones DA, Cairns BR \(2008\). DNA demethylation in zebrafish involves the coupling of a deaminase, a glycosylase, and gadd45. Cell 135: 1201–1212.")), whereas knockdown or knockout inhibits the DNA demethylation of various genes necessary for cellular reprogramming and development ([Bhutani et al, 2010](https://www.nature.com/articles/npp2012112#ref-CR12 "Bhutani N, Brady JJ, Damian M, Sacco A, Corbel SY, Blau HM \(2010\). Reprogramming towards pluripotency requires AID-dependent DNA demethylation. Nature 463: 1042–1047."); [Muramatsu et al, 2000](https://www.nature.com/articles/npp2012112#ref-CR146 "Muramatsu M, Kinoshita K, Fagarasan S, Yamada S, Shinkai Y, Honjo T \(2000\). Class switch recombination and hypermutation require activation-induced cytidine deaminase \(AID\), a potential RNA editing enzyme. Cell 102: 553–563."); [Popp et al, 2010](https://www.nature.com/articles/npp2012112#ref-CR159 "Popp C, Dean W, Feng S, Cokus SJ, Andrews S, Pellegrini M et al \(2010\). Genome-wide erasure of DNA methylation in mouse primordial germ cells is affected by AID deficiency. Nature 463: 1101–1105.")). Unlike the various _Dnmt_ knockout mice, knockout _AID_ mice are viable and fertile. If global DNA demethylation is as critical as DNA methylation in early development, then the knockout _AID_ mice study raises the possibility that multiple mechanisms for active DNA demethylation exist and can compensate for one another.
**Figure 2**
[![figure 2](https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fnpp.2012.112/MediaObjects/41386_2013_Article_BFnpp2012112_Fig2_HTML.jpg)](https://www.nature.com/articles/npp2012112/figures/2)
Active DNA demethylation pathways. 5-Methylcytosine (5mC) can be chemically modified at two sites: the amine group and the methyl group. The amine group of 5mC can be deaminated (green) by AID/APOBEC, converting 5mC into thymine (Thy). The methyl group of 5mC can be modified by the addition of a hydroxyl group mediated by Tet enzymes to generate 5-hydroxymethyl-cytosine (5hmC). 5hmC can also be chemically modified at two sites: the amine group and the hydroxymethyl group. AID/APOBEC can deaminate (green) 5hmC to produce 5-hydroxymethyl-uracil (5hmU). In another chemical pathway for 5hmC is that Tet can further oxidize (yellow) 5hmC to form 5-formyl-cytosine (5fC) and then 5-carboxy-cytosine (5caC). Eventually, the products of each pathway—Thy, 5hmU, 5fC, and 5caC—are recognized and cleaved off to replace with a naked cytosine mediated by TDG and/or SMUG1, both components of the base excision repair pathway (red).
[PowerPoint slide](https://www.nature.com/articles/npp2012112#MOESM2)
[Full size image](https://www.nature.com/articles/npp2012112/figures/2)
In line with the multiple mechanisms hypothesis, another active DNA demethylation mechanism is found to be mediated by the ten–eleven translocation (Tet) enzymes Tet1, Tet2, and Tet3. Tet enzymes add a hydroxyl group onto the methyl group of 5mC to form 5hmC ([Tahiliani et al, 2009](https://www.nature.com/articles/npp2012112#ref-CR181 "Tahiliani M, Koh KP, Shen Y, Pastor WA, Bandukwala H, Brudno Y et al \(2009\). Conversion of 5-methylcytosine to 5-hydroxymethylcytosine in mammalian DNA by MLL partner TET1. Science 324: 930–935. The first study to demonstrate the role of Tet enzymes in conversion of 5-methylcytosine to 5-hydroxymethylcytosine, thus leading to recent studies of the DNA demethylation mechanism in the mammalian system."); [Ito et al, 2010](https://www.nature.com/articles/npp2012112#ref-CR90 "Ito S, D’Alessio AC, Taranova OV, Hong K, Sowers LC, Zhang Y \(2010\). Role of Tet proteins in 5mC to 5hmC conversion, ES-cell self-renewal and inner cell mass specification. Nature 466: 1129–1133.")). The developed brain contains significant 5hmC levels in multiple regions, ranging from 0.3 to 0.7%, which is approximately tenfold lower than the average abundance of 5mC ([Kriaucionis and Heintz, 2009](https://www.nature.com/articles/npp2012112#ref-CR104 "Kriaucionis S, Heintz N \(2009\). The nuclear DNA base 5-hydroxymethylcytosine is present in Purkinje neurons and the brain. Science 324: 929–930. This is first paper describing that hydroxymethylcytosine is enriched in postmitotic CNS neurons."); [Globisch et al, 2010](https://www.nature.com/articles/npp2012112#ref-CR63 "Globisch D, Munzel M, Muller M, Michalakis S, Wagner M, Koch S et al \(2010\). Tissue distribution of 5-hydroxymethylcytosine and search for active demethylation intermediates. PLoS One 5: e15367.")). Once 5hmC is formed, two separate mechanisms can convert 5hmC back into cytosine in mammals. In the first, iterative oxidation by Tet enzymes continues to oxidize 5hmC first to 5-formyl-cytosine and then to 5-carboxy-cytosine ([Ito et al, 2011](https://www.nature.com/articles/npp2012112#ref-CR91 "Ito S, Shen L, Dai Q, Wu SC, Collins LB, Swenberg JA et al \(2011\). Tet proteins can convert 5-methylcytosine to 5-formylcytosine and 5-carboxylcytosine. Science 333: 1300–1303.")). In the second, 5hmC is deaminated by AID/APOBEC to form 5-hydroxymethyl-uracil ([Guo et al, 2011b](https://www.nature.com/articles/npp2012112#ref-CR67 "Guo J, Su Y, Zhong C, Ming GL, Song H \(2011b\). Hydroxylation of 5-methylcytosine by TET1 promotes active DNA demethylation in the adult brain. Cell 145: 423–434. Also discovered few gene regions, including BDNF, that exhibit demethylation and remethylation cycle in postmitotic neurons.")). Consistent with the role of Tet in converting 5mC into 5hmC _in vivo_ , _Tet1_ knockout mouse embryonic stem cells have reduced levels of 5hmC that is accompanied by a subtle increase in 5mC at a global level ([Dawlaty et al, 2011](https://www.nature.com/articles/npp2012112#ref-CR40 "Dawlaty MM, Ganz K, Powell BE, Hu YC, Markoulaki S, Cheng AW et al \(2011\). Tet1 is dispensable for maintaining pluripotency and its loss is compatible with embryonic and postnatal development. Cell Stem Cell 9: 166–175.")).
Whether 5hmC functions only as an intermediate in DNA demethylation is still unclear. Like methylation, 5hmC may regulate gene expression. In support of this theory, the conversion of 5mC to 5hmC impairs the binding of the repressive methyl-binding protein MeCP2 ([Valinluck et al, 2004](https://www.nature.com/articles/npp2012112#ref-CR190 "Valinluck V, Tsai HH, Rogstad DK, Burdzy A, Bird A, Sowers LC \(2004\). Oxidative damage to methyl-CpG sequences inhibits the binding of the methyl-CpG binding domain \(MBD\) of methyl-CpG binding protein 2 \(MeCP2\). Nucleic Acids Res 32: 4100–4108.")). But what is clear at this time is that 5hmC is found _in vivo_ in mammalian tissue and may play an important role in regulating DNA demethylation and gene expression.
In all the mentioned mechanisms of active DNA demethylation, the BER pathway uses thymine DNA glycosylase (TDG) to cleave off the modified residue—thymine, 5-hydroxymethyl-uracil, 5-formyl-cytosine, and 5-carboxy-cytosine—and replace it with a naked cytosine ([Cortellino et al, 2011](https://www.nature.com/articles/npp2012112#ref-CR35 "Cortellino S, Xu J, Sannai M, Moore R, Caretti E, Cigliano A et al \(2011\). Thymine DNA glycosylase is essential for active DNA demethylation by linked deamination-base excision repair. Cell 146: 67–79."); [He et al, 2011](https://www.nature.com/articles/npp2012112#ref-CR75 "He YF, Li BZ, Li Z, Liu P, Wang Y, Tang Q et al \(2011\). Tet-mediated formation of 5-carboxylcytosine and its excision by TDG in mammalian DNA. Science 333: 1303–1307.")). TDG is essential for DNA demethylation and is required for normal development. Knockout or inactivation of _TDG_ leads to embryonic lethality in mice. Moreover, these mutant embryos exhibit hypermethylation, particularly in imprinting genes such as _Igf2_ and _H19_ , suggesting that active demethylation by TDG protects imprinted genes from spontaneous _de novo_ methylation ([Cortellino et al, 2011](https://www.nature.com/articles/npp2012112#ref-CR35 "Cortellino S, Xu J, Sannai M, Moore R, Caretti E, Cigliano A et al \(2011\). Thymine DNA glycosylase is essential for active DNA demethylation by linked deamination-base excision repair. Cell 146: 67–79.")). Single-strand-selective monofunctional uracil-DNA glycosylase 1 (SMUG1), another BER enzyme from the same uracil DNA glycosylase family as TDG, is also found to be involved in DNA demethylation ([Cortellino et al, 2011](https://www.nature.com/articles/npp2012112#ref-CR35 "Cortellino S, Xu J, Sannai M, Moore R, Caretti E, Cigliano A et al \(2011\). Thymine DNA glycosylase is essential for active DNA demethylation by linked deamination-base excision repair. Cell 146: 67–79."); [Guo et al, 2011a](https://www.nature.com/articles/npp2012112#ref-CR66 "Guo J, Ma DK, Mo H, Ball MP, Jang MH, Bonaguidi MA et al \(2011a\). Neuronal activity modifies the DNA methylation landscape in the adult brain. Nat Neurosci 14: 1345–1351. Investigated the extent to which the DNA methylation pattern is modifiable by neural activity in vivo."), [2011b](https://www.nature.com/articles/npp2012112#ref-CR67 "Guo J, Su Y, Zhong C, Ming GL, Song H \(2011b\). Hydroxylation of 5-methylcytosine by TET1 promotes active DNA demethylation in the adult brain. Cell 145: 423–434. Also discovered few gene regions, including BDNF, that exhibit demethylation and remethylation cycle in postmitotic neurons.")). In summary, active DNA demethylation arises from multiple pathways involving multiple enzymes and this complexity has likely contributed to much of the current scientific debate.
### Reading DNA Methylation
Whereas DNA methylation may itself reduce gene expression by impairing the binding of transcriptional activators, a second class of proteins with a high affinity for 5mC inhibits transcription factor binding. DNA methylation is recognized by three separate families of proteins: the MBD proteins, the UHRF proteins, and the zinc-finger proteins. Of these families, the MBD was the first to be identified. MBD proteins contain a conserved methyl-CpG-binding domain (MBD) that confers a higher affinity for single methylated CpG sites ([Nan et al, 1993](https://www.nature.com/articles/npp2012112#ref-CR148 "Nan X, Meehan RR, Bird A \(1993\). Dissection of the methyl-CpG binding domain from the chromosomal protein MeCP2. Nucleic Acids Res 21: 4886–4892.")). This family includes MeCP2, the first identified methyl-binding protein, along with MBD1, MBD2, MBD3, and MBD4 ([Meehan et al, 1989](https://www.nature.com/articles/npp2012112#ref-CR134 "Meehan RR, Lewis JD, McKay S, Kleiner EL, Bird AP \(1989\). Identification of a mammalian protein that binds specifically to DNA containing methylated CpGs. Cell 58: 499–507."); [Lewis et al, 1992](https://www.nature.com/articles/npp2012112#ref-CR115 "Lewis JD, Meehan RR, Henzel WJ, Maurer-Fogy I, Jeppesen P, Klein F et al \(1992\). Purification, sequence, and cellular localization of a novel chromosomal protein that binds to methylated DNA. Cell 69: 905–914."); [Hendrich and Bird, 1998](https://www.nature.com/articles/npp2012112#ref-CR77 "Hendrich B, Bird A \(1998\). Identification and characterization of a family of mammalian methyl-CpG binding proteins. Mol Cell Biol 18: 6538–6547.")). MBDs are more highly expressed in the brain than in any other tissue, and many MBDs are important for normal neuronal development and function ([Amir et al, 1999](https://www.nature.com/articles/npp2012112#ref-CR4 "Amir RE, Van den Veyver IB, Wan M, Tran CQ, Francke U, Zoghbi HY \(1999\). Rett syndrome is caused by mutations in X-linked MECP2, encoding methyl-CpG-binding protein 2. Nat Genet 23: 185–188. Discovered MECP2 mutation as the cause of Rett Syndrome and paved way for further study of MECP2 protein role in the nervous system.")). Of the MBD family, MBD3 and MBD4 are unusual. For example, MBD3 is incapable of directly binding to DNA due to a mutation in its MBD domain ([Hendrich and Bird, 1998](https://www.nature.com/articles/npp2012112#ref-CR77 "Hendrich B, Bird A \(1998\). Identification and characterization of a family of mammalian methyl-CpG binding proteins. Mol Cell Biol 18: 6538–6547.")). Although MBD4 binds to DNA normally, it preferentially recognizes when guanine is mismatched with a thymine, uracil, or 5-fluorouracil and associates with proteins involved in DNA mismatch repair ([Bellacosa et al, 1999](https://www.nature.com/articles/npp2012112#ref-CR9 "Bellacosa A, Cicchillitti L, Schepis F, Riccio A, Yeung AT, Matsumoto Y et al \(1999\). MED1, a novel human methyl-CpG-binding endonuclease, interacts with DNA mismatch repair protein MLH1. Proc Natl Acad Sci USA 96: 3969–3974."); [Hendrich et al, 1999](https://www.nature.com/articles/npp2012112#ref-CR78 "Hendrich B, Hardeland U, Ng HH, Jiricny J, Bird A \(1999\). The thymine glycosylase MBD4 can bind to the product of deamination at methylated CpG sites. Nature 401: 301–304."); [Petronzelli et al, 2000](https://www.nature.com/articles/npp2012112#ref-CR158 "Petronzelli F, Riccio A, Markham GD, Seeholzer SH, Stoerker J, Genuardi M et al \(2000\). Biphasic kinetics of the human DNA repair protein MED1 \(MBD4\), a mismatch-specific DNA N-glycosylase. J Biol Chem 275: 32422–32429."); [Millar et al, 2002](https://www.nature.com/articles/npp2012112#ref-CR139 "Millar CB, Guy J, Sansom OJ, Selfridge J, MacDougall E, Hendrich B et al \(2002\). Enhanced CpG mutability and tumorigenesis in MBD4-deficient mice. Science 297: 403–405."); [Wong et al, 2002](https://www.nature.com/articles/npp2012112#ref-CR197 "Wong E, Yang K, Kuraguchi M, Werling U, Avdievich E, Fan K et al \(2002\). Mbd4 inactivation increases Cright-arrowT transition mutations and promotes gastrointestinal tumor formation. Proc Natl Acad Sci USA 99: 14937–14942.")). The remaining members of the MBD family have the ability to directly bind to methylated DNA and contain a transcriptional repression domain (TRD) that allows MBD proteins to bind to a variety of repressor complexes ([Nan et al, 1998](https://www.nature.com/articles/npp2012112#ref-CR149 "Nan X, Ng HH, Johnson CA, Laherty CD, Turner BM, Eisenman RN et al \(1998\). Transcriptional repression by the methyl-CpG-binding protein MeCP2 involves a histone deacetylase complex. Nature 393: 386–389."); [Ng et al, 1999](https://www.nature.com/articles/npp2012112#ref-CR151 "Ng HH, Zhang Y, Hendrich B, Johnson CA, Turner BM, Erdjument-Bromage H et al \(1999\). MBD2 is a transcriptional repressor belonging to the MeCP1 histone deacetylase complex. Nat Genet 23: 58–61."); [Sarraf and Stancheva, 2004](https://www.nature.com/articles/npp2012112#ref-CR168 "Sarraf SA, Stancheva I \(2004\). Methyl-CpG binding protein MBD1 couples histone H3 methylation at lysine 9 by SETDB1 to DNA replication and chromatin assembly. Mol Cell 15: 595–605.")). In addition to its role as a transcriptional repressor, MeCP2 appears to have a unique role in the maintenance of DNA methylation. MeCP2 binds to Dnmt1 via its TRD and can recruit Dnmt1 to hemimethylated DNA to perform maintenance methylation ([Kimura and Shiota, 2003](https://www.nature.com/articles/npp2012112#ref-CR100 "Kimura H, Shiota K \(2003\). Methyl-CpG-binding protein, MeCP2, is a target molecule for maintenance DNA methyltransferase, Dnmt1. J Biol Chem 278: 4806–4812.")). Although MBDs are the best studied class of methyl-binding proteins, they are not the only one.
The UHRF (ubiquitin-like, containing PHD and RING finger domain) proteins, including UHRF1 and UHRF2, are multidomain proteins that flip out and bind methylated cytosines via a SET- and RING-associated DNA-binding domain ([Hashimoto et al, 2008](https://www.nature.com/articles/npp2012112#ref-CR72 "Hashimoto H, Horton JR, Zhang X, Bostick M, Jacobsen SE, Cheng X \(2008\). The SRA domain of UHRF1 flips 5-methylcytosine out of the DNA helix. Nature 455: 826–829."), [2009](https://www.nature.com/articles/npp2012112#ref-CR73 "Hashimoto H, Horton JR, Zhang X, Cheng X \(2009\). UHRF1, a modular multi-domain protein, regulates replication-coupled crosstalk between DNA methylation and histone modifications. Epigenetics 4: 8–14.")). Unlike most methyl-binding proteins, the primary function of UHRF proteins is not to bind to DNA and repress transcription. The UHRF protein family first binds to Dnmt1 and then targets it to hemimethylated DNA in order to maintain DNA methylation, especially during DNA replication ([Sharif et al, 2007](https://www.nature.com/articles/npp2012112#ref-CR174 "Sharif J, Muto M, Takebayashi S, Suetake I, Iwamatsu A, Endo TA et al \(2007\). The SRA protein Np95 mediates epigenetic inheritance by recruiting Dnmt1 to methylated DNA. Nature 450: 908–912."); [Bostick et al, 2007](https://www.nature.com/articles/npp2012112#ref-CR16 "Bostick M, Kim JK, Esteve PO, Clark A, Pradhan S, Jacobsen SE \(2007\). UHRF1 plays a role in maintaining DNA methylation in mammalian cells. Science 317: 1760–1764."); [Achour et al, 2008](https://www.nature.com/articles/npp2012112#ref-CR3 "Achour M, Jacq X, Ronde P, Alhosin M, Charlot C, Chataigneau T et al \(2008\). The interaction of the SRA domain of ICBP90 with a novel domain of DNMT1 is involved in the regulation of VEGF gene expression. Oncogene 27: 2187–2197.")). UHRF1 appears to interact so closely with Dnmt1 that its deletion, like the deletion of Dnmt1, leads to embryonic lethality ([Muto et al, 2002](https://www.nature.com/articles/npp2012112#ref-CR147 "Muto M, Kanari Y, Kubo E, Takabe T, Kurihara T, Fujimori A et al \(2002\). Targeted disruption of Np95 gene renders murine embryonic stem cells hypersensitive to DNA damaging agents and DNA replication blocks. J Biol Chem 277: 34549–34555.")).
The last family of methyl-binding proteins binds to methylated DNA by a zinc-finger domain and is composed of Kaiso, ZBTB4, and ZBTB38 ([Prokhortchouk et al, 2001](https://www.nature.com/articles/npp2012112#ref-CR161 "Prokhortchouk A, Hendrich B, Jorgensen H, Ruzov A, Wilm M, Georgiev G et al \(2001\). The p120 catenin partner Kaiso is a DNA methylation-dependent transcriptional repressor. Genes Dev 15: 1613–1618."); [Filion et al, 2006](https://www.nature.com/articles/npp2012112#ref-CR52 "Filion GJ, Zhenilo S, Salozhin S, Yamada D, Prokhortchouk E, Defossez PA \(2006\). A family of human zinc finger proteins that bind methylated DNA and repress transcription. Mol Cell Biol 26: 169–181.")). Although ZBTB4 and ZBTB38 have distinct tissue expression patterns, both are highly expressed in the brain and can bind to a single methylated CpG. The zinc-finger domain proteins are unusual. Despite their ability to recognize methylcytosine, both Kaiso and ZBTB4 have preferential binding for a sequence motif lacking a methylcytosine ([Daniel et al, 2002](https://www.nature.com/articles/npp2012112#ref-CR39 "Daniel JM, Spring CM, Crawford HC, Reynolds AB, Baig A \(2002\). The p120\(ctn\)-binding partner Kaiso is a bi-modal DNA-binding protein that recognizes both a sequence-specific consensus and methylated CpG dinucleotides. Nucleic Acids Res 30: 2911–2919."); [Sasai et al, 2010](https://www.nature.com/articles/npp2012112#ref-CR169 "Sasai N, Nakao M, Defossez PA \(2010\). Sequence-specific recognition of methylated DNA by human zinc-finger proteins. Nucleic Acids Res 38: 5015–5022.")). Unlike other methyl-binding proteins, Kaiso preferentially binds to two consecutively methylated CpG sites ([Daniel et al, 2002](https://www.nature.com/articles/npp2012112#ref-CR39 "Daniel JM, Spring CM, Crawford HC, Reynolds AB, Baig A \(2002\). The p120\(ctn\)-binding partner Kaiso is a bi-modal DNA-binding protein that recognizes both a sequence-specific consensus and methylated CpG dinucleotides. Nucleic Acids Res 30: 2911–2919.")). Yet despite their differences, zinc-finger domain proteins, similar to the MBD family, repress transcription in a DNA methylation-dependent manner ([Prokhortchouk et al, 2001](https://www.nature.com/articles/npp2012112#ref-CR161 "Prokhortchouk A, Hendrich B, Jorgensen H, Ruzov A, Wilm M, Georgiev G et al \(2001\). The p120 catenin partner Kaiso is a DNA methylation-dependent transcriptional repressor. Genes Dev 15: 1613–1618."); [Yoon et al, 2003](https://www.nature.com/articles/npp2012112#ref-CR204 "Yoon HG, Chan DW, Reynolds AB, Qin J, Wong J \(2003\). N-CoR mediates DNA methylation-dependent repression through a methyl CpG binding protein Kaiso. Mol Cell 12: 723–734."); [Filion et al, 2006](https://www.nature.com/articles/npp2012112#ref-CR52 "Filion GJ, Zhenilo S, Salozhin S, Yamada D, Prokhortchouk E, Defossez PA \(2006\). A family of human zinc finger proteins that bind methylated DNA and repress transcription. Mol Cell Biol 26: 169–181."); [Lopes et al, 2008](https://www.nature.com/articles/npp2012112#ref-CR122 "Lopes EC, Valls E, Figueroa ME, Mazur A, Meng FG, Chiosis G et al \(2008\). Kaiso contributes to DNA methylation-dependent silencing of tumor suppressor genes in colon cancer cell lines. Cancer Res 68: 7258–7263.")).
### Crosstalk of DNA Methylation and Other Epigenetic Mechanisms
DNA methylation works with histone modifications and microRNA (miRNA) to regulate transcription ([Figure 3](https://www.nature.com/articles/npp2012112#Fig3)). In eukaryotes, DNA is associated with histone proteins that help to package the long strands of DNA into the small nuclear compartment. Chemical modifications that include methylation, acetylation, ubiquitination, and phosphorylation are added to three specific amino acids on the N-terminal histone tails. These modifications influence not only how DNA strands are packaged but also their transcriptional activity. Histone modifications that loosen DNA association with histones generally provide a permissive environment for transcription, whereas histone modifications that tightly package DNA and histones repress gene expression. Dnmts directly interact with enzymes that regulate histone modifications typically involved in gene repression ([Figure 3](https://www.nature.com/articles/npp2012112#Fig3)). Both Dnmt1 and Dnmt3a are known to bind to the histone methyltransferase SUV39H1 that restricts gene expression by methylation on H3K9 ([Fuks et al, 2003](https://www.nature.com/articles/npp2012112#ref-CR55 "Fuks F, Hurd PJ, Deplus R, Kouzarides T \(2003\). The DNA methyltransferases associate with HP1 and the SUV39H1 histone methyltransferase. Nucleic Acids Res 31: 2305–2312.")). Furthermore, Dnmt1 and Dnmt3b can both bind to histone deacetylases that remove acetylation from histones to make DNA pack more tightly and restrict access for transcription ([Fuks et al, 2000](https://www.nature.com/articles/npp2012112#ref-CR54 "Fuks F, Burgers WA, Brehm A, Hughes-Davies L, Kouzarides T \(2000\). DNA methyltransferase Dnmt1 associates with histone deacetylase activity. Nat Genet 24: 88–91. Demonstrates physical connection between the two epigenetic mechanisms: DNA methylation and histone modification."); [Geiman et al, 2004](https://www.nature.com/articles/npp2012112#ref-CR60 "Geiman TM, Sankpal UT, Robertson AK, Zhao Y, Zhao Y, Robertson KD \(2004\). DNMT3B interacts with hSNF2H chromatin remodeling enzyme, HDACs 1 and 2, and components of the histone methylation system. Biochem Biophys Res Commun 318: 544–555.")). In general, Dnmts cooperate with histone-modifying enzymes involved in adding and/or stripping histone markers in order to impose a repressive state on a gene region.
**Figure 3**
[![figure 3](https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fnpp.2012.112/MediaObjects/41386_2013_Article_BFnpp2012112_Fig3_HTML.jpg)](https://www.nature.com/articles/npp2012112/figures/3)
Epigenetic crosstalk. Transcription is ultimately regulated by the interaction of multiple epigenetic mechanisms that cooperate to activate or silence gene expression. Methylation is regulated by proteins such as Dnmt and Tet (purple) that are involved in the active addition or chemical modification (such as hydroxymethylation in red) of DNA methylation. To suppress gene expression, Dnmts target CpG sites and actively methylate DNA. For some Dnmts, their catalytic activity is enhanced by association with histone tails and with Dnmt3L. DNA methylation is recognized by methyl-binding proteins such as MBDs (yellow) that along with Dnmts recruit enzymes that modify the histone tails (orange) including histone deacetylases (HDACs), which remove acetylation (red), and histone methyltransferases (HMTs), which methylate histones (green) and in conjunction with DNA methylation serve to further repress gene expression. In regions of DNA with activate transcription, Tet removes DNA methylation, and histone tails in this region often contain H3K4me3 that inhibits Dnmt binding to unmethylated CpG sites and maintains a permissive environment for transcription.
[PowerPoint slide](https://www.nature.com/articles/npp2012112#MOESM3)
[Full size image](https://www.nature.com/articles/npp2012112/figures/3)
Histone modifications can also influence the DNA methylation pattern ([Figure 3](https://www.nature.com/articles/npp2012112#Fig3)). Dnmt3L binds to H3 histone tails and recruits Dnmt3a and Dnmt3b to methylate DNA ([Ooi et al, 2007](https://www.nature.com/articles/npp2012112#ref-CR154 "Ooi SK, Qiu C, Bernstein E, Li K, Jia D, Yang Z et al \(2007\). DNMT3L connects unmethylated lysine 4 of histone H3 to de novo methylation of DNA. Nature 448: 714–717.")). The direct binding of Dnmt3a to the H3 histone tail, sometimes facilitated by H3K36 trimethylation, a repressive histone mark, also stimulates its methyltransferase activity ([Dhayalan et al, 2010](https://www.nature.com/articles/npp2012112#ref-CR43 "Dhayalan A, Rajavelu A, Rathert P, Tamas R, Jurkowska RZ, Ragozin S et al \(2010\). The Dnmt3a PWWP domain reads histone 3 lysine 36 trimethylation and guides DNA methylation. J Biol Chem 285: 26114–26120."); [Li et al, 2011a](https://www.nature.com/articles/npp2012112#ref-CR116 "Li BZ, Huang Z, Cui QY, Song XH, Du L, Jeltsch A et al \(2011a\). Histone tails regulate DNA methylation by allosterically activating de novo methyltransferase. Cell Res 21: 1172–1181.")). However, the presence of the active histone modification H3K4 trimethylation (H3K4me3) impairs the binding of Dnmt3a, Dnmt3b, and Dnmt3L to H3 histone tails and prevents methylation ([Ooi et al, 2007](https://www.nature.com/articles/npp2012112#ref-CR154 "Ooi SK, Qiu C, Bernstein E, Li K, Jia D, Yang Z et al \(2007\). DNMT3L connects unmethylated lysine 4 of histone H3 to de novo methylation of DNA. Nature 448: 714–717."); [Zhang et al, 2010](https://www.nature.com/articles/npp2012112#ref-CR207 "Zhang Y, Jurkowska R, Soeroes S, Rajavelu A, Dhayalan A, Bock I et al \(2010\). Chromatin methylation activity of Dnmt3a and Dnmt3a/3L is guided by interaction of the ADD domain with the histone H3 tail. Nucleic Acids Res 38: 4246–4253.")). CpG islands contain particularly high levels of H3K4me3 ([Mikkelsen et al, 2007](https://www.nature.com/articles/npp2012112#ref-CR137 "Mikkelsen TS, Ku M, Jaffe DB, Issac B, Lieberman E, Giannoukos G et al \(2007\). Genome-wide maps of chromatin state in pluripotent and lineage-committed cells. Nature 448: 553–560.")). Cfp1 is a component of the H3K4 methyltransferase complex that targets unmethylated CpG sites often found at murine CpG islands and may play a role in maintaining their hypomethylation ([Lee and Skalnik, 2005](https://www.nature.com/articles/npp2012112#ref-CR111 "Lee JH, Skalnik DG \(2005\). CpG-binding protein \(CXXC finger protein 1\) is a component of the mammalian Set1 histone H3-Lys4 methyltransferase complex, the analogue of the yeast Set1/COMPASS complex. J Biol Chem 280: 41725–41731."); [Thomson et al, 2010](https://www.nature.com/articles/npp2012112#ref-CR187 "Thomson JP, Skene PJ, Selfridge J, Clouaire T, Guy J, Webb S et al \(2010\). CpG islands influence chromatin structure via the CpG-binding protein Cfp1. Nature 464: 1082–1086.")). Little is known regarding how the DNA demethylation machinery interacts with histone modifications, yet there is still evidence to suggest that they cooperate. For instance, elevated histone acetylation can trigger DNA demethylation ([Cervoni and Szyf, 2001](https://www.nature.com/articles/npp2012112#ref-CR26 "Cervoni N, Szyf M \(2001\). Demethylase activity is directed by histone acetylation. J Biol Chem 276: 40778–40787."); [D’Alessio et al, 2007](https://www.nature.com/articles/npp2012112#ref-CR38 "D’Alessio AC, Weaver IC, Szyf M \(2007\). Acetylation-induced transcription is required for active DNA demethylation in methylation-silenced genes. Mol Cell Biol 27: 7462–7474. An example of coordinated actions from both histone modification and DNA demethylation in regulating gene regulation.")). Tet1 contains a DNA-binding motif similar to Ctf1, suggesting that both proteins target similar sites, in this case CpG islands, to maintain DNA demethylation ([Tahiliani et al, 2009](https://www.nature.com/articles/npp2012112#ref-CR181 "Tahiliani M, Koh KP, Shen Y, Pastor WA, Bandukwala H, Brudno Y et al \(2009\). Conversion of 5-methylcytosine to 5-hydroxymethylcytosine in mammalian DNA by MLL partner TET1. Science 324: 930–935. The first study to demonstrate the role of Tet enzymes in conversion of 5-methylcytosine to 5-hydroxymethylcytosine, thus leading to recent studies of the DNA demethylation mechanism in the mammalian system.")). Although a direct connection between the two has yet to been shown, Tet1 does indeed localize to CpG islands and its depletion results in an increase in methylation within those CpG islands in mouse embryonic stem cell studies ([Ficz et al, 2011](https://www.nature.com/articles/npp2012112#ref-CR51 "Ficz G, Branco MR, Seisenberger S, Santos F, Krueger F, Hore TA et al \(2011\). Dynamic regulation of 5-hydroxymethylcytosine in mouse ES cells and during differentiation. Nature 473: 398–402."); [Wu and Zhang, 2011](https://www.nature.com/articles/npp2012112#ref-CR198 "Wu H, Zhang Y \(2011\). Tet1 and 5-hydroxymethylation: a genome-wide view in mouse embryonic stem cells. Cell Cycle 10: 2428–2436.")). Future studies are needed to further probe the interaction of Tet with histone modifications.
Methyl-binding proteins serve as the strongest link between DNA methylation and histone modification. Both the MBDs and the UHRF proteins interact with methylated DNA and histones to enhance gene repression ([Figure 3](https://www.nature.com/articles/npp2012112#Fig3)) ([Nan et al, 1998](https://www.nature.com/articles/npp2012112#ref-CR149 "Nan X, Ng HH, Johnson CA, Laherty CD, Turner BM, Eisenman RN et al \(1998\). Transcriptional repression by the methyl-CpG-binding protein MeCP2 involves a histone deacetylase complex. Nature 393: 386–389."); [Ng et al, 1999](https://www.nature.com/articles/npp2012112#ref-CR151 "Ng HH, Zhang Y, Hendrich B, Johnson CA, Turner BM, Erdjument-Bromage H et al \(1999\). MBD2 is a transcriptional repressor belonging to the MeCP1 histone deacetylase complex. Nat Genet 23: 58–61."); [Sarraf and Stancheva, 2004](https://www.nature.com/articles/npp2012112#ref-CR168 "Sarraf SA, Stancheva I \(2004\). Methyl-CpG binding protein MBD1 couples histone H3 methylation at lysine 9 by SETDB1 to DNA replication and chromatin assembly. Mol Cell 15: 595–605."); [Citterio et al, 2004](https://www.nature.com/articles/npp2012112#ref-CR32 "Citterio E, Papait R, Nicassio F, Vecchi M, Gomiero P, Mantovani R et al \(2004\). Np95 is a histone-binding protein endowed with ubiquitin ligase activity. Mol Cell Biol 24: 2526–2535."); [Karagianni et al, 2008](https://www.nature.com/articles/npp2012112#ref-CR99 "Karagianni P, Amazit L, Qin J, Wong J \(2008\). ICBP90, a novel methyl K9 H3 binding protein linking protein ubiquitination with heterochromatin formation. Mol Cell Biol 28: 705–717.")). MeCP2 recruits histone deacetylases to remove active histone modifications and repress gene transcription ([Jones et al, 1998](https://www.nature.com/articles/npp2012112#ref-CR95 "Jones PL, Veenstra GJ, Wade PA, Vermaak D, Kass SU, Landsberger N et al \(1998\). Methylated DNA and MeCP2 recruit histone deacetylase to repress transcription. Nat Genet 19: 187–191."); [Nan et al, 1998](https://www.nature.com/articles/npp2012112#ref-CR149 "Nan X, Ng HH, Johnson CA, Laherty CD, Turner BM, Eisenman RN et al \(1998\). Transcriptional repression by the methyl-CpG-binding protein MeCP2 involves a histone deacetylase complex. Nature 393: 386–389."); [Fuks et al, 2003](https://www.nature.com/articles/npp2012112#ref-CR55 "Fuks F, Hurd PJ, Deplus R, Kouzarides T \(2003\). The DNA methyltransferases associate with HP1 and the SUV39H1 histone methyltransferase. Nucleic Acids Res 31: 2305–2312.")). Furthermore, MeCP2 enhances the repressive chromatin state by recruiting histone methyltransferases that add repressive H3K9 methylation ([Fuks et al, 2003](https://www.nature.com/articles/npp2012112#ref-CR55 "Fuks F, Hurd PJ, Deplus R, Kouzarides T \(2003\). The DNA methyltransferases associate with HP1 and the SUV39H1 histone methyltransferase. Nucleic Acids Res 31: 2305–2312.")). Overall, DNA methylation and histone modifications work closely together to regulate gene expression.
Recently, miRNAs have emerged as another important epigenetic mechanism that influences gene expression. Precursor miRNA forms a double-stranded RNA connected by a hairpin loop. Once transported into the cytoplasm precursor, miRNA is processed by Dicer to generate a 22–23 nucleotide miRNA:miRNA* duplex. Mature miRNAs associate with a miRNA-induced silencing complex (miRISC) that can bind to their target mRNA and repress gene expression by inhibiting translation or inducing RNA degradation ([Berezikov, 2011](https://www.nature.com/articles/npp2012112#ref-CR11 "Berezikov E \(2011\). Evolution of microRNA diversity and regulation in animals. Nat Rev Genet 12: 846–860.")). Like other sequences within the genome, DNA methylation can regulate the expression of miRNAs ([Han et al, 2007](https://www.nature.com/articles/npp2012112#ref-CR71 "Han L, Witmer PD, Casey E, Valle D, Sukumar S \(2007\). DNA methylation regulates microRNA expression. Cancer Biol Ther 6: 1284–1288."); [Lujambio et al, 2008](https://www.nature.com/articles/npp2012112#ref-CR125 "Lujambio A, Calin GA, Villanueva A, Ropero S, Sanchez-Cespedes M, Blanco D et al \(2008\). A microRNA DNA methylation signature for human cancer metastasis. Proc Natl Acad Sci USA 105: 13556–13561.")). The loss of both _Dnmt1_ and _Dnmt3b_ in a colon cancer cell line revealed that ∼10% of detected miRNAs are regulated by DNA methylation ([Han et al, 2007](https://www.nature.com/articles/npp2012112#ref-CR71 "Han L, Witmer PD, Casey E, Valle D, Sukumar S \(2007\). DNA methylation regulates microRNA expression. Cancer Biol Ther 6: 1284–1288.")). When Dnmts are inhibited, cancer cells reactivate some miRNAs that are initially silenced by hypermethylation of their CpG islands ([Lujambio et al, 2008](https://www.nature.com/articles/npp2012112#ref-CR125 "Lujambio A, Calin GA, Villanueva A, Ropero S, Sanchez-Cespedes M, Blanco D et al \(2008\). A microRNA DNA methylation signature for human cancer metastasis. Proc Natl Acad Sci USA 105: 13556–13561.")). Understood together, these studies demonstrate that DNA methylation regulates miRNA expression.
Conversely, miRNAs can also regulate histone modifications and Dnmt expression and, in so doing, regulate DNA methylation ([Benetti et al, 2008](https://www.nature.com/articles/npp2012112#ref-CR10 "Benetti R, Gonzalo S, Jaco I, Munoz P, Gonzalez S, Schoeftner S et al \(2008\). A mammalian microRNA cluster controls DNA methylation and telomere recombination via Rbl2-dependent regulation of DNA methyltransferases. Nat Struct Mol Biol 15: 268–279."); [Sinkkonen et al, 2008](https://www.nature.com/articles/npp2012112#ref-CR176 "Sinkkonen L, Hugenschmidt T, Berninger P, Gaidatzis D, Mohn F, Artus-Revel CG et al \(2008\). MicroRNAs control de novo DNA methylation through regulation of transcriptional repressors in mouse embryonic stem cells. Nat Struct Mol Biol 15: 259–267.")). Knockout of _Dicer_ in mouse embryonic stem cells results in depletion of miRNAs, one of which is miRNA-290, which indirectly regulates _Dnmt3a_ and _Dnmt3b_ expression ([Benetti et al, 2008](https://www.nature.com/articles/npp2012112#ref-CR10 "Benetti R, Gonzalo S, Jaco I, Munoz P, Gonzalez S, Schoeftner S et al \(2008\). A mammalian microRNA cluster controls DNA methylation and telomere recombination via Rbl2-dependent regulation of DNA methyltransferases. Nat Struct Mol Biol 15: 268–279."); [Sinkkonen et al, 2008](https://www.nature.com/articles/npp2012112#ref-CR176 "Sinkkonen L, Hugenschmidt T, Berninger P, Gaidatzis D, Mohn F, Artus-Revel CG et al \(2008\). MicroRNAs control de novo DNA methylation through regulation of transcriptional repressors in mouse embryonic stem cells. Nat Struct Mol Biol 15: 259–267.")). This leads to a loss of DNA methylation and an increase in repressive histone methylation at H3K9. These studies provided evidence of a bidirectional influence between miRNA and DNA methylation.
## DNA METHYLATION IN THE BRAIN
### DNA Methylation in the Developing CNS
The precise temporal regulation of _de novo_ methylation and demethylation is particularly important for the differentiation and maturation of the mammalian central nervous system (CNS). Multipotent neural progenitor cells (NPCs) sequentially undergo neurogenesis and astrogliogenesis ([Qian et al, 2000](https://www.nature.com/articles/npp2012112#ref-CR162 "Qian X, Shen Q, Goderie SK, He W, Capela A, Davis AA et al \(2000\). Timing of CNS cell generation: a programmed sequence of neuron and glial cell production from isolated murine cortical stem cells. Neuron 28: 69–80."); [Sauvageot and Stiles, 2002](https://www.nature.com/articles/npp2012112#ref-CR170 "Sauvageot CM, Stiles CD \(2002\). Molecular mechanisms controlling cortical gliogenesis. Curr Opin Neurobiol 12: 244–249.")). In particular, the differentiation switch of NPCs from neurogenesis to astrogliogenesis coincides with DNA methylation and demethylation events on the _glial fibrillary acidic protein_ (_Gfap_) gene promoter region ([Teter et al, 1994](https://www.nature.com/articles/npp2012112#ref-CR185 "Teter B, Osterburg HH, Anderson CP, Finch CE \(1994\). Methylation of the rat glial fibrillary acidic protein gene shows tissue-specific domains. J Neurosci Res 39: 680–693.")). Early in neurogenesis at E11.5, DNA methylation of the _Gfap_ promoter represses its expression ([Teter et al, 1996](https://www.nature.com/articles/npp2012112#ref-CR186 "Teter B, Rozovsky I, Krohn K, Anderson C, Osterburg H, Finch C \(1996\). Methylation of the glial fibrillary acidic protein gene shows novel biphasic changes during brain development. Glia 17: 195–205."); [Takizawa et al, 2001](https://www.nature.com/articles/npp2012112#ref-CR182 "Takizawa T, Nakashima K, Namihira M, Ochiai W, Uemura A, Yanagisawa M et al \(2001\). DNA methylation is a critical cell-intrinsic determinant of astrocyte differentiation in the fetal brain. Dev Cell 1: 749–758.")). The continual expression of _Dnmt1_ in NPCs has been found to be important for the maintenance of the methylation pattern on the _Gfap_ promoter through subsequent cell divisions ([Fan et al, 2005](https://www.nature.com/articles/npp2012112#ref-CR47 "Fan G, Martinowich K, Chin MH, He F, Fouse SD, Hutnick L et al \(2005\). DNA methylation controls the timing of astrogliogenesis through regulation of JAK-STAT signaling. Development 132: 3345–3356.")). Interestingly, neurogenesis from E11.5 to E14.5 is the only time during neural development that _Dnmt3b_ is strongly expressed before declining to nearly undetectable levels in the CNS ([Fan et al, 2005](https://www.nature.com/articles/npp2012112#ref-CR47 "Fan G, Martinowich K, Chin MH, He F, Fouse SD, Hutnick L et al \(2005\). DNA methylation controls the timing of astrogliogenesis through regulation of JAK-STAT signaling. Development 132: 3345–3356.")). At E14.5, the _Gfap_ promoter undergoes DNA demethylation to coincide with the differentiation of the astrocytic lineage ([Teter et al, 1996](https://www.nature.com/articles/npp2012112#ref-CR186 "Teter B, Rozovsky I, Krohn K, Anderson C, Osterburg H, Finch C \(1996\). Methylation of the glial fibrillary acidic protein gene shows novel biphasic changes during brain development. Glia 17: 195–205.")). As development further progresses, the decline of _Dnmt3b_ and the peak expression of _Dnmt3a_ at 3 weeks postnatally coincide with remethylation and reduced transcription of the _Gfap_ promoter ([Fan et al, 2005](https://www.nature.com/articles/npp2012112#ref-CR47 "Fan G, Martinowich K, Chin MH, He F, Fouse SD, Hutnick L et al \(2005\). DNA methylation controls the timing of astrogliogenesis through regulation of JAK-STAT signaling. Development 132: 3345–3356."); [Nguyen et al, 2007](https://www.nature.com/articles/npp2012112#ref-CR152 "Nguyen S, Meletis K, Fu D, Jhaveri S, Jaenisch R \(2007\). Ablation of de novo DNA methyltransferase Dnmt3a in the nervous system leads to neuromuscular defects and shortened lifespan. Dev Dyn 236: 1663–1676. Demonstrate that knockout of Dnmt3a in neural tissue during development does not affect development of cortical neurons but results in hypomethylation of the Gfap promoter and improper motor neuron development.")). The coordinated expression of _Dnmts_ and their ability to regulate the methylation pattern of the _Gfap_ promoter organize and regulate neuronal development.
The importance of these coordinated events is highlighted by conditional knockout models of Dnmts during neural development ([Fan et al, 2001](https://www.nature.com/articles/npp2012112#ref-CR46 "Fan G, Beard C, Chen RZ, Csankovszki G, Sun Y, Siniaia M et al \(2001\). DNA hypomethylation perturbs the function and survival of CNS neurons in postnatal animals. J Neurosci 21: 788–797. One of the pioneering works in investigating the role of Dnmts in the CNS."), [2005](https://www.nature.com/articles/npp2012112#ref-CR47 "Fan G, Martinowich K, Chin MH, He F, Fouse SD, Hutnick L et al \(2005\). DNA methylation controls the timing of astrogliogenesis through regulation of JAK-STAT signaling. Development 132: 3345–3356."); [Golshani et al, 2005](https://www.nature.com/articles/npp2012112#ref-CR64 "Golshani P, Hutnick L, Schweizer F, Fan G \(2005\). Conditional Dnmt1 deletion in dorsal forebrain disrupts development of somatosensory barrel cortex and thalamocortical long-term potentiation. Thalamus Relat Syst 3: 227–233."); [Nguyen et al, 2007](https://www.nature.com/articles/npp2012112#ref-CR152 "Nguyen S, Meletis K, Fu D, Jhaveri S, Jaenisch R \(2007\). Ablation of de novo DNA methyltransferase Dnmt3a in the nervous system leads to neuromuscular defects and shortened lifespan. Dev Dyn 236: 1663–1676. Demonstrate that knockout of Dnmt3a in neural tissue during development does not affect development of cortical neurons but results in hypomethylation of the Gfap promoter and improper motor neuron development."); [Hutnick et al, 2009](https://www.nature.com/articles/npp2012112#ref-CR84 "Hutnick LK, Golshani P, Namihira M, Xue Z, Matynia A, Yang XW et al \(2009\). DNA hypomethylation restricted to the murine forebrain induces cortical degeneration and impairs postnatal neuronal maturation. Hum Mol Genet 18: 2875–2888."); [Feng et al, 2010](https://www.nature.com/articles/npp2012112#ref-CR50 "Feng J, Zhou Y, Campbell SL, Le T, Li E, Sweatt JD et al \(2010\). Dnmt1 and Dnmt3a maintain DNA methylation and regulate synaptic function in adult forebrain neurons. Nat Neurosci 13: 423–430. Demonstrated the importance of continual activity of Dnmts in postmitotic neurons and that the Dnmts exhibit overlapping roles of each other.")). Conditional knockout of _Dnmt1_ between E8.5 and E13.5, a time period that coincides with neurogenesis, leads to hypomethylation of differentiating neurons and demethylation of the _Gfap_ promoter in neural precursor cells, thus accelerating astrogliosis ([Fan et al, 2001](https://www.nature.com/articles/npp2012112#ref-CR46 "Fan G, Beard C, Chen RZ, Csankovszki G, Sun Y, Siniaia M et al \(2001\). DNA hypomethylation perturbs the function and survival of CNS neurons in postnatal animals. J Neurosci 21: 788–797. One of the pioneering works in investigating the role of Dnmts in the CNS."), [2005](https://www.nature.com/articles/npp2012112#ref-CR47 "Fan G, Martinowich K, Chin MH, He F, Fouse SD, Hutnick L et al \(2005\). DNA methylation controls the timing of astrogliogenesis through regulation of JAK-STAT signaling. Development 132: 3345–3356.")). Hypomethylated neurons are characterized by multiple maturation defects including dendritic arborization and impaired neuronal excitability ([Fan et al, 2001](https://www.nature.com/articles/npp2012112#ref-CR46 "Fan G, Beard C, Chen RZ, Csankovszki G, Sun Y, Siniaia M et al \(2001\). DNA hypomethylation perturbs the function and survival of CNS neurons in postnatal animals. J Neurosci 21: 788–797. One of the pioneering works in investigating the role of Dnmts in the CNS."); [Golshani et al, 2005](https://www.nature.com/articles/npp2012112#ref-CR64 "Golshani P, Hutnick L, Schweizer F, Fan G \(2005\). Conditional Dnmt1 deletion in dorsal forebrain disrupts development of somatosensory barrel cortex and thalamocortical long-term potentiation. Thalamus Relat Syst 3: 227–233."); [Hutnick et al, 2009](https://www.nature.com/articles/npp2012112#ref-CR84 "Hutnick LK, Golshani P, Namihira M, Xue Z, Matynia A, Yang XW et al \(2009\). DNA hypomethylation restricted to the murine forebrain induces cortical degeneration and impairs postnatal neuronal maturation. Hum Mol Genet 18: 2875–2888.")). These results are consistent with the conclusion that Dnmt1 has a critical role in neuronal differentiation and in maintaining the methylation of the _Gfap_ promoter. Furthermore, these results suggest that DNA methylation is essential for neuronal maturation. If _Dnmt3a_ is knocked out instead in the neural precursor cells, the majority of cortical neurons develop normally ([Nguyen et al, 2007](https://www.nature.com/articles/npp2012112#ref-CR152 "Nguyen S, Meletis K, Fu D, Jhaveri S, Jaenisch R \(2007\). Ablation of de novo DNA methyltransferase Dnmt3a in the nervous system leads to neuromuscular defects and shortened lifespan. Dev Dyn 236: 1663–1676. Demonstrate that knockout of Dnmt3a in neural tissue during development does not affect development of cortical neurons but results in hypomethylation of the Gfap promoter and improper motor neuron development.")). In this case, the _Gfap_ promoter that is normally remethylated in postnatal astrocytes remains hypomethylated in ∼50% of cortical tissue, likely corresponding to the glial population ([Nguyen et al, 2007](https://www.nature.com/articles/npp2012112#ref-CR152 "Nguyen S, Meletis K, Fu D, Jhaveri S, Jaenisch R \(2007\). Ablation of de novo DNA methyltransferase Dnmt3a in the nervous system leads to neuromuscular defects and shortened lifespan. Dev Dyn 236: 1663–1676. Demonstrate that knockout of Dnmt3a in neural tissue during development does not affect development of cortical neurons but results in hypomethylation of the Gfap promoter and improper motor neuron development.")). This is consistent with the idea that Dnmt3a is not required during differentiation or maturation by the majority of cortical neurons. Together, the knockout data confirm that precise regulation of DNA methylation is essential for differentiation and maturation of the CNS.
Like Dnmts, methyl-binding proteins are expressed in embryonic stem cells and in neural precursors, but unlike Dnmts their expression has little effect on neuronal or glial differentiation ([Kishi and Macklis, 2004](https://www.nature.com/articles/npp2012112#ref-CR101 "Kishi N, Macklis JD \(2004\). MECP2 is progressively expressed in post-migratory neurons and is involved in neuronal maturation rather than cell fate decisions. Mol Cell Neurosci 27: 306–321."); [Martin Caballero et al, 2009](https://www.nature.com/articles/npp2012112#ref-CR128 "Martin Caballero I, Hansen J, Leaford D, Pollard S, Hendrich BD \(2009\). The methyl-CpG binding proteins Mecp2, Mbd2 and Kaiso are dispensable for mouse embryogenesis, but play a redundant function in neural differentiation. PLoS One 4: e4315.")). Of the MBD proteins, MeCP2 is the best studied in the CNS because its mutation results in Rett Syndrome, one of the most common forms of mental retardation in females ([Amir et al, 1999](https://www.nature.com/articles/npp2012112#ref-CR4 "Amir RE, Van den Veyver IB, Wan M, Tran CQ, Francke U, Zoghbi HY \(1999\). Rett syndrome is caused by mutations in X-linked MECP2, encoding methyl-CpG-binding protein 2. Nat Genet 23: 185–188. Discovered MECP2 mutation as the cause of Rett Syndrome and paved way for further study of MECP2 protein role in the nervous system.")). During development, _MeCP2_ is first expressed in the brainstem and thalamus, the most ancient regions of the brain, followed by a rostral progression of expression ([LaSalle et al, 2001](https://www.nature.com/articles/npp2012112#ref-CR109 "LaSalle JM, Goldstine J, Balmer D, Greco CM \(2001\). Quantitative localization of heterogeneous methyl-CpG-binding protein 2 \(MeCP2\) expression phenotypes in normal and Rett syndrome brain by laser scanning cytometry. Hum Mol Genet 10: 1729–1740."); [Shahbazian et al, 2002](https://www.nature.com/articles/npp2012112#ref-CR173 "Shahbazian MD, Antalffy B, Armstrong DL, Zoghbi HY \(2002\). Insight into Rett syndrome: MeCP2 levels display tissue- and cell-specific differences and correlate with neuronal maturation. Hum Mol Genet 11: 115–124.")). Like other methyl-binding proteins, MeCP2 associates with a variety of transcriptional repressors, including Dnmt1, and influences gene expression ([Nan et al, 1998](https://www.nature.com/articles/npp2012112#ref-CR149 "Nan X, Ng HH, Johnson CA, Laherty CD, Turner BM, Eisenman RN et al \(1998\). Transcriptional repression by the methyl-CpG-binding protein MeCP2 involves a histone deacetylase complex. Nature 393: 386–389."); [Ng et al, 1999](https://www.nature.com/articles/npp2012112#ref-CR151 "Ng HH, Zhang Y, Hendrich B, Johnson CA, Turner BM, Erdjument-Bromage H et al \(1999\). MBD2 is a transcriptional repressor belonging to the MeCP1 histone deacetylase complex. Nat Genet 23: 58–61."); [Kimura and Shiota, 2003](https://www.nature.com/articles/npp2012112#ref-CR100 "Kimura H, Shiota K \(2003\). Methyl-CpG-binding protein, MeCP2, is a target molecule for maintenance DNA methyltransferase, Dnmt1. J Biol Chem 278: 4806–4812."); [Sarraf and Stancheva, 2004](https://www.nature.com/articles/npp2012112#ref-CR168 "Sarraf SA, Stancheva I \(2004\). Methyl-CpG binding protein MBD1 couples histone H3 methylation at lysine 9 by SETDB1 to DNA replication and chromatin assembly. Mol Cell 15: 595–605.")). Neural activity leads to phosphorylation of MeCP2, altering its ability to bind gene promoters and silence gene expression ([Zhou et al, 2006](https://www.nature.com/articles/npp2012112#ref-CR209 "Zhou Z, Hong EJ, Cohen S, Zhao WN, Ho HY, Schmidt L et al \(2006\). Brain-specific phosphorylation of MeCP2 regulates activity-dependent Bdnf transcription, dendritic growth, and spine maturation. Neuron 52: 255–269."); [Tao et al, 2009](https://www.nature.com/articles/npp2012112#ref-CR183 "Tao J, Hu K, Chang Q, Wu H, Sherman NE, Martinowich K et al \(2009\). Phosphorylation of MeCP2 at Serine 80 regulates its chromatin association and neurological function. Proc Natl Acad Sci USA 106: 4882–4887.")). MeCP2 is required for normal neuronal maturation and its loss or the loss of its ability to be phosphorylated results in aberrant dendritic arborization, synaptic function, and plasticity ([Chen et al, 2001](https://www.nature.com/articles/npp2012112#ref-CR28 "Chen RZ, Akbarian S, Tudor M, Jaenisch R \(2001\). Deficiency of methyl-CpG binding protein-2 in CNS neurons results in a Rett-like phenotype in mice. Nat Genet 27: 327–331."); [Moretti et al, 2006](https://www.nature.com/articles/npp2012112#ref-CR143 "Moretti P, Levenson JM, Battaglia F, Atkinson R, Teague R, Antalffy B et al \(2006\). Learning and memory and synaptic plasticity are impaired in a mouse model of Rett syndrome. J Neurosci 26: 319–327."); [Asaka et al, 2006](https://www.nature.com/articles/npp2012112#ref-CR6 "Asaka Y, Jugloff DG, Zhang L, Eubanks JH, Fitzsimonds RM \(2006\). Hippocampal synaptic plasticity is impaired in the Mecp2-null mouse model of Rett syndrome. Neurobiol Dis 21: 217–227."); [Nelson et al, 2006](https://www.nature.com/articles/npp2012112#ref-CR150 "Nelson ED, Kavalali ET, Monteggia LM \(2006\). MeCP2-dependent transcriptional repression regulates excitatory neurotransmission. Curr Biol 16: 710–716."); [Cohen et al, 2011](https://www.nature.com/articles/npp2012112#ref-CR33 "Cohen S, Gabel HW, Hemberg M, Hutchinson AN, Sadacca LA, Ebert DH et al \(2011\). Genome-wide activity-dependent MeCP2 phosphorylation regulates nervous system development and function. Neuron 72: 72–85."); [Li et al, 2011b](https://www.nature.com/articles/npp2012112#ref-CR118 "Li H, Zhong X, Chau KF, Williams EC, Chang Q \(2011b\). Loss of activity-induced phosphorylation of MeCP2 enhances synaptogenesis, LTP and spatial memory. Nat Neurosci 14: 1001–1008.")).
At the conclusion of neural development, the brain primarily consists of postmitotic neurons and glial cells with little proliferation potential. Although the expression of Dnmts normally declines in terminally differentiated cells, the brain appears to be an exception. Both _Dnmt1_ and _Dnmt3a_ are expressed by postmitotic neurons, whereas _Dnmt3b_ expression is either low or virtually undetectable ([Goto et al, 1994](https://www.nature.com/articles/npp2012112#ref-CR65 "Goto K, Numata M, Komura JI, Ono T, Bestor TH, Kondo H \(1994\). Expression of DNA methyltransferase gene in mature and immature neurons as well as proliferating cells in mice. Differentiation 56: 39–44. Identified DNA methyltransferase mRNA in postmitotic neurons and suggested that DNA methylation served a unique function in neurons."); [Inano et al, 2000](https://www.nature.com/articles/npp2012112#ref-CR88 "Inano K, Suetake I, Ueda T, Miyake Y, Nakamura M, Okada M et al \(2000\). Maintenance-type DNA methyltransferase is highly expressed in post-mitotic neurons and localized in the cytoplasmic compartment. J Biochem 128: 315–321."); [Fan et al, 2005](https://www.nature.com/articles/npp2012112#ref-CR47 "Fan G, Martinowich K, Chin MH, He F, Fouse SD, Hutnick L et al \(2005\). DNA methylation controls the timing of astrogliogenesis through regulation of JAK-STAT signaling. Development 132: 3345–3356.")). This surprising discovery led researchers to investigate the role of active DNA methylation in postmitotic neurons of the adult brain.
### DNA Methylation in the Adult Brain
To date, the majority of the DNA methylation mechanism has been characterized in embryonic stem cells. Although this _in vitro_ model may predict the function of DNA methylation in a dividing cell, embryonic stem cells are an inadequate model for studying DNA methylation in a postmitotic cell. The fact that Dnmts are required for normal neuronal differentiation and maturation hinders the study of DNA methylation solely in postmitotic neurons. Despite these limitations, two models have emerged to study DNA methylation in postmitotic neurons.
The first model utilizes pharmacological inhibitors such as 5-aza-2′-deoxycytadine, zebularine, or RG108 injected into the brain to impair Dnmt activity ([Levenson et al, 2006](https://www.nature.com/articles/npp2012112#ref-CR114 "Levenson JM, Roth TL, Lubin FD, Miller CA, Huang IC, Desai P et al \(2006\). Evidence that DNA \(cytosine-5\) methyltransferase regulates synaptic plasticity in the hippocampus. J Biol Chem 281: 15763–15773."); [Miller and Sweatt, 2007](https://www.nature.com/articles/npp2012112#ref-CR141 "Miller CA, Sweatt JD \(2007\). Covalent modification of DNA regulates memory formation. Neuron 53: 857–869."); [Miller et al, 2010](https://www.nature.com/articles/npp2012112#ref-CR140 "Miller CA, Gavin CF, White JA, Parrish RR, Honasoge A, Yancey CR et al \(2010\). Cortical DNA methylation maintains remote memory. Nat Neurosci 13: 664–666.")). Although pharmalogical inhibition of Dnmts has the advantage of inhibiting DNA methylation at the designated time point, the mechanism of action of Dnmt inhibitors in postmitotic neurons is still debated. Nucleoside inhibitors such as 5-aza-2′-deoxycytadine and zebularine must be first incorporated into DNA in order to inhibit Dnmts ([Creusot et al, 1982](https://www.nature.com/articles/npp2012112#ref-CR37 "Creusot F, Acs G, Christman JK \(1982\). Inhibition of DNA methyltransferase and induction of Friend erythroleukemia cell differentiation by 5-azacytidine and 5-aza-2′-deoxycytidine. J Biol Chem 257: 2041–2048.")). Once incorporated into the DNA, the nucleoside inhibitors trap Dnmts on the chromosome, which can lead to substantial cellular toxicity ([Jüttermann et al, 1994](https://www.nature.com/articles/npp2012112#ref-CR96 "Jüttermann R, Li E, Jaenisch R \(1994\). Toxicity of 5-aza-2′-deoxycytidine to mammalian cells is mediated primarily by covalent trapping of DNA methyltransferase rather than DNA demethylation. Proc Natl Acad Sci USA 91: 11797–11801."); [Zhou et al, 2002](https://www.nature.com/articles/npp2012112#ref-CR208 "Zhou L, Cheng X, Connolly BA, Dickman MJ, Hurd PJ, Hornby DP \(2002\). Zebularine: a novel DNA methylation inhibitor that forms a covalent complex with DNA methyltransferases. J Mol Biol 321: 591–599.")). Although it is possible that the BER pathway may be the source of nucleoside incorporation, it is still unclear how nucleoside inhibitors are incorporated into the DNA of a nondividing cell, like a postmitotic neuron. Because of the concerns surrounding the use of nucleoside inhibitors, new-generation Dnmt inhibitors, which include RG108, have been developed. After the discovery of the structure of the catalytic domain of Dnmt1, RG108 was identified in an _in silico_ screen as a small molecule that could inhibit Dnmt1 without being incorporated into the DNA ([Brueckner et al, 2005](https://www.nature.com/articles/npp2012112#ref-CR22 "Brueckner B, Garcia Boy R, Siedlecki P, Musch T, Kliem HC, Zielenkiewicz P et al \(2005\). Epigenetic reactivation of tumor suppressor genes by a novel small-molecule inhibitor of human DNA methyltransferases. Cancer Res 65: 6305–6311."); [Stresemann et al, 2006](https://www.nature.com/articles/npp2012112#ref-CR178 "Stresemann C, Brueckner B, Musch T, Stopper H, Lyko F \(2006\). Functional diversity of DNA methyltransferase inhibitors in human cancer cell lines. Cancer Res 66: 2794–2800.")). RG108 has emerged as a promising Dnmt1 inhibitor with less cytotoxicity. In cell-free assays, RG108 is capable of impairing Dnmts with catalytic domains similar to Dnmt1. Although Dnmt3a and Dnmt3b share a highly conserved catalytic domain to Dnmt1, it is still unclear whether RG108 inhibits these methyltransferases ([Brueckner et al, 2005](https://www.nature.com/articles/npp2012112#ref-CR22 "Brueckner B, Garcia Boy R, Siedlecki P, Musch T, Kliem HC, Zielenkiewicz P et al \(2005\). Epigenetic reactivation of tumor suppressor genes by a novel small-molecule inhibitor of human DNA methyltransferases. Cancer Res 65: 6305–6311.")).
To alleviate concerns of off-target effects and incomplete Dnmt inhibition, a second experimental model has emerged. In this model, _Dnmts_ are conditionally knocked out using the cre/loxP system, with cre expressed by a brain-specific promoter ([Fan et al, 2001](https://www.nature.com/articles/npp2012112#ref-CR46 "Fan G, Beard C, Chen RZ, Csankovszki G, Sun Y, Siniaia M et al \(2001\). DNA hypomethylation perturbs the function and survival of CNS neurons in postnatal animals. J Neurosci 21: 788–797. One of the pioneering works in investigating the role of Dnmts in the CNS."); [Golshani et al, 2005](https://www.nature.com/articles/npp2012112#ref-CR64 "Golshani P, Hutnick L, Schweizer F, Fan G \(2005\). Conditional Dnmt1 deletion in dorsal forebrain disrupts development of somatosensory barrel cortex and thalamocortical long-term potentiation. Thalamus Relat Syst 3: 227–233."); [Nguyen et al, 2007](https://www.nature.com/articles/npp2012112#ref-CR152 "Nguyen S, Meletis K, Fu D, Jhaveri S, Jaenisch R \(2007\). Ablation of de novo DNA methyltransferase Dnmt3a in the nervous system leads to neuromuscular defects and shortened lifespan. Dev Dyn 236: 1663–1676. Demonstrate that knockout of Dnmt3a in neural tissue during development does not affect development of cortical neurons but results in hypomethylation of the Gfap promoter and improper motor neuron development."); [Hutnick et al, 2009](https://www.nature.com/articles/npp2012112#ref-CR84 "Hutnick LK, Golshani P, Namihira M, Xue Z, Matynia A, Yang XW et al \(2009\). DNA hypomethylation restricted to the murine forebrain induces cortical degeneration and impairs postnatal neuronal maturation. Hum Mol Genet 18: 2875–2888."); [Feng et al, 2010](https://www.nature.com/articles/npp2012112#ref-CR50 "Feng J, Zhou Y, Campbell SL, Le T, Li E, Sweatt JD et al \(2010\). Dnmt1 and Dnmt3a maintain DNA methylation and regulate synaptic function in adult forebrain neurons. Nat Neurosci 13: 423–430. Demonstrated the importance of continual activity of Dnmts in postmitotic neurons and that the Dnmts exhibit overlapping roles of each other.")). Unlike pharmacological inhibition, this second method allows investigators to study the role of a specific Dnmt in subpopulations of neurons. Dnmts are required for normal neuronal differentiation. Therefore, to study the role of Dnmts in the adult brain, cre must be expressed by postmitotic brain-specific promoters such as _CamKIIα_ ([Fan et al, 2001](https://www.nature.com/articles/npp2012112#ref-CR46 "Fan G, Beard C, Chen RZ, Csankovszki G, Sun Y, Siniaia M et al \(2001\). DNA hypomethylation perturbs the function and survival of CNS neurons in postnatal animals. J Neurosci 21: 788–797. One of the pioneering works in investigating the role of Dnmts in the CNS."); [Golshani et al, 2005](https://www.nature.com/articles/npp2012112#ref-CR64 "Golshani P, Hutnick L, Schweizer F, Fan G \(2005\). Conditional Dnmt1 deletion in dorsal forebrain disrupts development of somatosensory barrel cortex and thalamocortical long-term potentiation. Thalamus Relat Syst 3: 227–233."); [Nguyen et al, 2007](https://www.nature.com/articles/npp2012112#ref-CR152 "Nguyen S, Meletis K, Fu D, Jhaveri S, Jaenisch R \(2007\). Ablation of de novo DNA methyltransferase Dnmt3a in the nervous system leads to neuromuscular defects and shortened lifespan. Dev Dyn 236: 1663–1676. Demonstrate that knockout of Dnmt3a in neural tissue during development does not affect development of cortical neurons but results in hypomethylation of the Gfap promoter and improper motor neuron development."); [Hutnick et al, 2009](https://www.nature.com/articles/npp2012112#ref-CR84 "Hutnick LK, Golshani P, Namihira M, Xue Z, Matynia A, Yang XW et al \(2009\). DNA hypomethylation restricted to the murine forebrain induces cortical degeneration and impairs postnatal neuronal maturation. Hum Mol Genet 18: 2875–2888."); [Feng et al, 2010](https://www.nature.com/articles/npp2012112#ref-CR50 "Feng J, Zhou Y, Campbell SL, Le T, Li E, Sweatt JD et al \(2010\). Dnmt1 and Dnmt3a maintain DNA methylation and regulate synaptic function in adult forebrain neurons. Nat Neurosci 13: 423–430. Demonstrated the importance of continual activity of Dnmts in postmitotic neurons and that the Dnmts exhibit overlapping roles of each other.")).
In postmitotic neurons, Dnmt1 and Dnmt3a appear to have overlapping roles. Neither knockout of _Dnmt1_ nor of _Dnmt3a_ in forebrain postmitotic neurons leads to any observable change in DNA methylation, gene expression, synaptic plasticity, or behavior ([Feng et al, 2010](https://www.nature.com/articles/npp2012112#ref-CR50 "Feng J, Zhou Y, Campbell SL, Le T, Li E, Sweatt JD et al \(2010\). Dnmt1 and Dnmt3a maintain DNA methylation and regulate synaptic function in adult forebrain neurons. Nat Neurosci 13: 423–430. Demonstrated the importance of continual activity of Dnmts in postmitotic neurons and that the Dnmts exhibit overlapping roles of each other.")). However, double knockouts have reduced DNA methylation that leads to deficits in synaptic plasticity in addition to learning and memory. Although research has yet to tease apart the role of Dnmt1 _vs_ Dnmt3a in postmitotic neurons, DNA methylation has repeatedly been shown to play a role in learning and memory in the adult brain.
When neuronal activity is inhibited during fear conditioning, not only is memory formation prevented but so are changes in DNA methylation ([Lubin et al, 2008](https://www.nature.com/articles/npp2012112#ref-CR123 "Lubin FD, Roth TL, Sweatt JD \(2008\). Epigenetic regulation of BDNF gene transcription in the consolidation of fear memory. J Neurosci 28: 10576–10586.")). Early studies demonstrated that _in vitro_ neuronal activity regulated the expression of _Bdnf_ in an activity-dependent manner ([Martinowich et al, 2003](https://www.nature.com/articles/npp2012112#ref-CR129 "Martinowich K, Hattori D, Wu H, Fouse S, He F, Hu Y et al \(2003\). DNA methylation-related chromatin remodeling in activity-dependent BDNF gene regulation. Science 302: 890–893. Demonstrated that methylation and MeCP2 binding to the BDNF promoter are regulated by neuronal activity.")). Neuronal depolarization demethylates the _Bdnf_ promoter, releasing the MeCP2 repressor complex from the promoter and increasing _Bdnf_ expression ([Martinowich et al, 2003](https://www.nature.com/articles/npp2012112#ref-CR129 "Martinowich K, Hattori D, Wu H, Fouse S, He F, Hu Y et al \(2003\). DNA methylation-related chromatin remodeling in activity-dependent BDNF gene regulation. Science 302: 890–893. Demonstrated that methylation and MeCP2 binding to the BDNF promoter are regulated by neuronal activity.")). Persistent activity in neurons, as occurs during electroconvulsive stimulation or exercise, leads to active DNA methylation and demethylation across several genes within the brain. However, alterations in DNA methylation do not always correlate with the alterations in gene expression observed after heightened activity ([Guo et al, 2011a](https://www.nature.com/articles/npp2012112#ref-CR66 "Guo J, Ma DK, Mo H, Ball MP, Jang MH, Bonaguidi MA et al \(2011a\). Neuronal activity modifies the DNA methylation landscape in the adult brain. Nat Neurosci 14: 1345–1351. Investigated the extent to which the DNA methylation pattern is modifiable by neural activity in vivo.")). Hence, although both DNA methylation and demethylation are altered by neuronal activity, DNA methylation functions alongside other regulatory proteins and epigenetic mechanisms that determine gene expression.
Another class of proteins that work with DNA methylation to regulate gene expression in the CNS is the class of methyl-binding proteins. Methyl-binding proteins are continually expressed in the adult CNS and often act as repressors that recognize and bind to methylated cytosines ([Nan et al, 1998](https://www.nature.com/articles/npp2012112#ref-CR149 "Nan X, Ng HH, Johnson CA, Laherty CD, Turner BM, Eisenman RN et al \(1998\). Transcriptional repression by the methyl-CpG-binding protein MeCP2 involves a histone deacetylase complex. Nature 393: 386–389."); [Ng et al, 1999](https://www.nature.com/articles/npp2012112#ref-CR151 "Ng HH, Zhang Y, Hendrich B, Johnson CA, Turner BM, Erdjument-Bromage H et al \(1999\). MBD2 is a transcriptional repressor belonging to the MeCP1 histone deacetylase complex. Nat Genet 23: 58–61."); [Sarraf and Stancheva, 2004](https://www.nature.com/articles/npp2012112#ref-CR168 "Sarraf SA, Stancheva I \(2004\). Methyl-CpG binding protein MBD1 couples histone H3 methylation at lysine 9 by SETDB1 to DNA replication and chromatin assembly. Mol Cell 15: 595–605.")). Hence, when methylation is removed as a result of neuronal activity, it is not surprising that MBDs are often released from promoters ([Martinowich et al, 2003](https://www.nature.com/articles/npp2012112#ref-CR129 "Martinowich K, Hattori D, Wu H, Fouse S, He F, Hu Y et al \(2003\). DNA methylation-related chromatin remodeling in activity-dependent BDNF gene regulation. Science 302: 890–893. Demonstrated that methylation and MeCP2 binding to the BDNF promoter are regulated by neuronal activity.")). However, the role of methyl-binding proteins is not this simple. Some MBDs like MeCP2 undergo posttranslational modifications that alter their ability to bind to DNA ([Zhou et al, 2006](https://www.nature.com/articles/npp2012112#ref-CR209 "Zhou Z, Hong EJ, Cohen S, Zhao WN, Ho HY, Schmidt L et al \(2006\). Brain-specific phosphorylation of MeCP2 regulates activity-dependent Bdnf transcription, dendritic growth, and spine maturation. Neuron 52: 255–269."); [Tao et al, 2009](https://www.nature.com/articles/npp2012112#ref-CR183 "Tao J, Hu K, Chang Q, Wu H, Sherman NE, Martinowich K et al \(2009\). Phosphorylation of MeCP2 at Serine 80 regulates its chromatin association and neurological function. Proc Natl Acad Sci USA 106: 4882–4887.")). Phosphorylation of MeCP2 is induced by neuronal activity and results in altered gene expression. When phosphorylation of MeCP2 is inhibited, synapse formation, synaptic plasticity, and learning and memory behavior are all affected ([Cohen et al, 2011](https://www.nature.com/articles/npp2012112#ref-CR33 "Cohen S, Gabel HW, Hemberg M, Hutchinson AN, Sadacca LA, Ebert DH et al \(2011\). Genome-wide activity-dependent MeCP2 phosphorylation regulates nervous system development and function. Neuron 72: 72–85."); [Li et al, 2011a](https://www.nature.com/articles/npp2012112#ref-CR116 "Li BZ, Huang Z, Cui QY, Song XH, Du L, Jeltsch A et al \(2011a\). Histone tails regulate DNA methylation by allosterically activating de novo methyltransferase. Cell Res 21: 1172–1181."), [2011b](https://www.nature.com/articles/npp2012112#ref-CR118 "Li H, Zhong X, Chau KF, Williams EC, Chang Q \(2011b\). Loss of activity-induced phosphorylation of MeCP2 enhances synaptogenesis, LTP and spatial memory. Nat Neurosci 14: 1001–1008.")). As phosphorylation is normally a short-term modification, activity-dependent phosphorylation may temporarily release MeCP2 from promoters, making the gene sequence accessible for demethylation. On the other hand, DNA methylation and demethylation may be responsible for long-term changes in gene expression that regulate synaptic plasticity as well as learning and memory.
### DNA Methylation in the Etiology of Neurological and Psychiatric Disorders
The pattern of DNA methylation established during development can be modulated by neural activity in order to encode learning and memory. When the mechanisms that establish and recognize the DNA methylation pattern are dysfunctional, problems with learning and memory frequently result. One of the most common forms of mental retardation, Rett Syndrome, is frequently caused by a mutation to the methyl-binding protein MeCP2 ([Amir et al, 1999](https://www.nature.com/articles/npp2012112#ref-CR4 "Amir RE, Van den Veyver IB, Wan M, Tran CQ, Francke U, Zoghbi HY \(1999\). Rett syndrome is caused by mutations in X-linked MECP2, encoding methyl-CpG-binding protein 2. Nat Genet 23: 185–188. Discovered MECP2 mutation as the cause of Rett Syndrome and paved way for further study of MECP2 protein role in the nervous system.")). The onset of symptoms at 6–18 months of age coincides with a time in early development when sensory experience is driving dendritic pruning and shaping connections in the brain ([Samaco and Neul, 2011](https://www.nature.com/articles/npp2012112#ref-CR167 "Samaco RC, Neul JL \(2011\). Complexities of Rett syndrome and MeCP2. J Neurosci 31: 7951–7959.")). Although _MeCP2_ is expressed by the majority of cells, it is particularly important for normal neuronal function. In mice, loss of _MeCP2_ in neurons is sufficient to recapitulate the majority of Rett symptoms ([Chen et al, 2001](https://www.nature.com/articles/npp2012112#ref-CR28 "Chen RZ, Akbarian S, Tudor M, Jaenisch R \(2001\). Deficiency of methyl-CpG binding protein-2 in CNS neurons results in a Rett-like phenotype in mice. Nat Genet 27: 327–331."); [Guy et al, 2001](https://www.nature.com/articles/npp2012112#ref-CR69 "Guy J, Hendrich B, Holmes M, Martin JE, Bird A \(2001\). A mouse Mecp2-null mutation causes neurological symptoms that mimic Rett syndrome. Nat Genet 27: 322–326.")). The phenotype of _MeCP2_ mutant mice can be reversed by restoration of the _MeCP2_ gene in postmitotic neurons ([Luikenhuis et al, 2004](https://www.nature.com/articles/npp2012112#ref-CR124 "Luikenhuis S, Giacometti E, Beard CF, Jaenisch R \(2004\). Expression of MeCP2 in postmitotic neurons rescues Rett syndrome in mice. Proc Natl Acad Sci USA 101: 6033–6038."); [Giacometti et al, 2007](https://www.nature.com/articles/npp2012112#ref-CR62 "Giacometti E, Luikenhuis S, Beard C, Jaenisch R \(2007\). Partial rescue of MeCP2 deficiency by postnatal activation of MeCP2. Proc Natl Acad Sci USA 104: 1931–1936."); [Guy et al, 2007](https://www.nature.com/articles/npp2012112#ref-CR68 "Guy J, Gan J, Selfridge J, Cobb S, Bird A \(2007\). Reversal of neurological defects in a mouse model of Rett syndrome. Science 315: 1143–1147.")). As previously mentioned, MeCP2 is regulated by neuronal activity and in turn regulates the expression of _BDNF_ , which has enhanced expression following depolarization ([Martinowich et al, 2003](https://www.nature.com/articles/npp2012112#ref-CR129 "Martinowich K, Hattori D, Wu H, Fouse S, He F, Hu Y et al \(2003\). DNA methylation-related chromatin remodeling in activity-dependent BDNF gene regulation. Science 302: 890–893. Demonstrated that methylation and MeCP2 binding to the BDNF promoter are regulated by neuronal activity.")). The overexpression of _BDNF_ in postmitotic neurons of MeCP2 mutant mice ameliorates their phenotype, suggesting that MeCP2 is critical for regulating the expression of genes like _BDNF_ that are regulated by neuronal activity and essential for normal cognitive function ([Chang et al, 2006](https://www.nature.com/articles/npp2012112#ref-CR27 "Chang Q, Khare G, Dani V, Nelson S, Jaenisch R \(2006\). The disease progression of Mecp2 mutant mice is affected by the level of BDNF expression. Neuron 49: 341–348.")). The role of MeCP2 in Rett Syndrome will be further discussed in later chapters.
One extremely rare neurodegenerative disease illustrates the importance of proper DNMT activity in the adult brain. Patients with hereditary sensory and autonomic neuropathy type 1 (HSAN1) develop dementia and hearing loss in adulthood that result from an autosomal-dominant mutation in the N-terminal regulatory domain of _DNMT1_ ([Klein et al, 2011](https://www.nature.com/articles/npp2012112#ref-CR102 "Klein CJ, Botuyan MV, Wu Y, Ward CJ, Nicholson GA, Hammans S et al \(2011\). Mutations in DNMT1 cause hereditary sensory neuropathy with dementia and hearing loss. Nat Genet 43: 595–600.")). This mutation results in misfolding, impaired nuclear localization, and early degradation of DNMT1. However, the mutation does not affect the targeting of DNMT1 to the replication foci during cellular replication, but the DNMT1 association with heterochromatin beyond S phase is disrupted. This association may affect the maintenance of DNA methylation within these regions. Although there is only a modest 8% reduction of global DNA methylation level, neurodegeneration does result. The involvement of DNMT1 in the pathogenic mechanism of HSAN1 supports the necessity of DNMT1 in the adult brain.
Improper methylation of a single gene or a single allele can have drastic consequences within the brain. Fragile X Syndrome is caused by abnormal methylation of a trinucleotide repeat expansion in the _FMR1_ gene on the X chromosome and is a common form of mental retardation ([Verkerk et al, 1991](https://www.nature.com/articles/npp2012112#ref-CR191 "Verkerk AJ, Pieretti M, Sutcliffe JS, Fu YH, Kuhl DP, Pizzuti A et al \(1991\). Identification of a gene \(FMR-1\) containing a CGG repeat coincident with a breakpoint cluster region exhibiting length variation in fragile X syndrome. Cell 65: 905–914."); [Turner et al, 1996](https://www.nature.com/articles/npp2012112#ref-CR188 "Turner G, Webb T, Wake S, Robinson H \(1996\). Prevalence of fragile X syndrome. Am J Med Genet 64: 196–197.")). The hypermethylation in the repeat expansion of _FMR1_ results in transcriptional silencing ([Devys et al, 1993](https://www.nature.com/articles/npp2012112#ref-CR42 "Devys D, Lutz Y, Rouyer N, Bellocq JP, Mandel JL \(1993\). The FMR-1 protein is cytoplasmic, most abundant in neurons and appears normal in carriers of a fragile X premutation. Nat Genet 4: 335–340.")). Translation of the _FMR1_ gene is regulated by neuronal activity ([Weiler et al, 1997](https://www.nature.com/articles/npp2012112#ref-CR196 "Weiler IJ, Irwin SA, Klintsova AY, Spencer CM, Brazelton AD, Miyashiro K et al \(1997\). Fragile X mental retardation protein is translated near synapses in response to neurotransmitter activation. Proc Natl Acad Sci USA 94: 5395–5400.")) and its protein product, FMRP, is involved in protein synthesis at the synapses following depolarization (for review, see [Fatemi and Folsom, 2011](https://www.nature.com/articles/npp2012112#ref-CR48 "Fatemi SH, Folsom TD \(2011\). The role of fragile X mental retardation protein in major mental disorders. Neuropharmacology 60: 1221–1226.")). Similarly, improper methylation of a single imprinted allele, found in some disorders such as Prader–Willi Syndrome and Angelman Syndrome, can cause significant mental impairments (for review, see [Buiting, 2010](https://www.nature.com/articles/npp2012112#ref-CR23 "Buiting K \(2010\). Prader-Willi syndrome and Angelman syndrome. Am J Med Genet C Semin Med Genet 154C: 365–376.")). As incorrect expression or loss of function of a single gene can have a dramatic effect in the brain, it is important to understand the mechanism of how DNA methylation affects gene expression.
DNA methylation can also be altered by repeated modulation of the microenvironment of the brain. In the case of recurrent seizures this microenvironment is repeatedly subject to unusual, synchronized neuronal activity. One way to mimic this unusual neural activity is by electric convulsive stimulation, which was found to result in genome-wide changes in the DNA methylation pattern ([Ma et al, 2009](https://www.nature.com/articles/npp2012112#ref-CR126 "Ma DK, Jang MH, Guo JU, Kitabatake Y, Chang ML, Pow-Anpongkul N et al \(2009\). Neuronal activity-induced Gadd45b promotes epigenetic DNA demethylation and adult neurogenesis. Science 323: 1074–1077."); [Guo et al, 2011a](https://www.nature.com/articles/npp2012112#ref-CR66 "Guo J, Ma DK, Mo H, Ball MP, Jang MH, Bonaguidi MA et al \(2011a\). Neuronal activity modifies the DNA methylation landscape in the adult brain. Nat Neurosci 14: 1345–1351. Investigated the extent to which the DNA methylation pattern is modifiable by neural activity in vivo.")). Similarly, repeated drug usage modulates neuronal function as in the case of cocaine. Cocaine usage modulates _Dnmt3a_ expression within the nucleus accumbens and enhances spine formation ([LaPlant et al, 2010](https://www.nature.com/articles/npp2012112#ref-CR108 "LaPlant Q, Vialou V, Covington III HE, Dumitriu D, Feng J, Warren BL et al \(2010\). Dnmt3a regulates emotional behavior and spine plasticity in the nucleus accumbens. Nat Neurosci 13: 1137–1143.")). Also, repeated cocaine usage increases _MeCP2_ that, in turn, increases _Bdnf_ expression ([Im et al, 2010](https://www.nature.com/articles/npp2012112#ref-CR87 "Im HI, Hollander JA, Bali P, Kenny PJ \(2010\). MeCP2 controls BDNF expression and cocaine intake through homeostatic interactions with microRNA-212. Nat Neurosci 13: 1120–1127.")). Sometimes, drug exposure, like neural activity, can add posttranslational modifications to components of the methylation machinery such as MeCP2 ([Deng et al, 2010](https://www.nature.com/articles/npp2012112#ref-CR41 "Deng JV, Rodriguiz RM, Hutchinson AN, Kim IH, Wetsel WC, West AE \(2010\). MeCP2 in the nucleus accumbens contributes to neural and behavioral responses to psychostimulants. Nat Neurosci 13: 1128–1136."); [Hutchinson et al, 2012](https://www.nature.com/articles/npp2012112#ref-CR83 "Hutchinson AN, Deng JV, Aryal DK, Wetsel WC, West AE \(2012\). Differential regulation of MeCP2 phosphorylation in the CNS by dopamine and serotonin. Neuropsychopharmacology 37: 321–337.")).
Although DNA methylation is clearly altered in the above disorders stemming from mutations, inappropriate methylation, or repeated modulation of the microenvironment, the role of DNA methylation in most psychiatric disorders is less clear. Yet, there is mounting evidence that altered patterns of DNA methylation are associated with many psychiatric disorders. For example, early-life stress in the form of maternal neglect was sufficient to alter DNA methylation in the brain of a rodent model ([Weaver et al, 2004](https://www.nature.com/articles/npp2012112#ref-CR193 "Weaver IC, Cervoni N, Champagne FA, D’Alessio AC, Sharma S, Seckl JR et al \(2004\). Epigenetic programming by maternal behavior. Nat Neurosci 7: 847–854. One of the first studies to demonstrate the association of methylation changes in the brain with neural behavior.")). Maternal neglect increased methylation within the promoter of the glucocorticoid receptor, thus reducing its expression. Surprisingly, this alteration in the DNA methylation pattern was retained into adulthood, leading to a heightened stress response. Similarly, in humans, childhood abuse results in increased methylation of the promoter for the glucocorticoid receptor and a decrease in its expression, recapitulating the rodent model ([McGowan et al, 2009](https://www.nature.com/articles/npp2012112#ref-CR133 "McGowan PO, Sasaki A, D’Alessio AC, Dymov S, Labonte B, Szyf M et al \(2009\). Epigenetic regulation of the glucocorticoid receptor in human brain associates with childhood abuse. Nat Neurosci 12: 342–348. Observed childhood abuse in suicidal victims associated with higher DNA methylation level in the glucocorticoid receptor promoter in the hippocampus.")). Furthermore, altered patterns of DNA methylation are observed in psychiatric patients diagnosed with schizophrenia and bipolar disorder ([Mill et al, 2008](https://www.nature.com/articles/npp2012112#ref-CR138 "Mill J, Tang T, Kaminsky Z, Khare T, Yazdanpanah S, Bouchard L et al \(2008\). Epigenomic profiling reveals DNA-methylation changes associated with major psychosis. Am J Hum Genet 82: 696–711.")).
## FUTURE DIRECTIONS AND CLINICAL IMPLICATIONS
DNA methylation varies not only between tissues but also between brain regions, between gray matter and white matter, and possibly even between cells ([Ladd-Acosta et al, 2007](https://www.nature.com/articles/npp2012112#ref-CR107 "Ladd-Acosta C, Pevsner J, Sabunciyan S, Yolken RH, Webster MJ, Dinkins T et al \(2007\). DNA methylation signatures within the human brain. Am J Hum Genet 81: 1304–1315."); [Ghosh et al, 2010](https://www.nature.com/articles/npp2012112#ref-CR61 "Ghosh S, Yates AJ, Frühwald MC, Miecznikowski JC, Plass C, Smiraglia D \(2010\). Tissue specific DNA methylation of CpG islands in normal human adult somatic tissues distinguishes neural from non-neural tissues. Epigenetics 5: 527–538.")). Although current technology limits our ability to distinguish cell-specific methylation patterns, the advent of next-generation DNA sequencing has provided powerful tools to examine the genome-wide DNA methylation pattern with single-nucleotide resolution ([Meissner et al, 2008](https://www.nature.com/articles/npp2012112#ref-CR135 "Meissner A, Mikkelsen TS, Gu H, Wernig M, Hanna J, Sivachenko A et al \(2008\). Genome-scale DNA methylation maps of pluripotent and differentiated cells. Nature 454: 766–770."); [Lister et al, 2009](https://www.nature.com/articles/npp2012112#ref-CR120 "Lister R, Pelizzola M, Dowen RH, Hawkins RD, Hon G, Tonti-Filippini J et al \(2009\). Human DNA methylomes at base resolution show widespread epigenomic differences. Nature 462: 315–322."); [Popp et al, 2010](https://www.nature.com/articles/npp2012112#ref-CR159 "Popp C, Dean W, Feng S, Cokus SJ, Andrews S, Pellegrini M et al \(2010\). Genome-wide erasure of DNA methylation in mouse primordial germ cells is affected by AID deficiency. Nature 463: 1101–1105.")). As technology improves, the cost of performing sequencing analysis will decline, thus making the technology more accessible. Recent technical developments have allowed for genome-wide DNA methylation analysis to be performed even with a sample amount as low as 150 ng ([Popp et al, 2010](https://www.nature.com/articles/npp2012112#ref-CR159 "Popp C, Dean W, Feng S, Cokus SJ, Andrews S, Pellegrini M et al \(2010\). Genome-wide erasure of DNA methylation in mouse primordial germ cells is affected by AID deficiency. Nature 463: 1101–1105.")). Aberrant DNA methylation patterns are observed in a wide variety of psychiatric and neurological illnesses. With declining costs and the ability to perform genome-wide methylation analysis on limited tissue quantities, it will soon be possible to map-out genome-wide DNA methylation patterns from distinct brain regions from patients with neurological and psychiatric disorders. The analysis of neural tissue from psychiatric patients will lead to new insights into the etiology of psychiatric illness and open up new avenues of drug discovery and targeted therapies.
Although current protocols enable scientists to precisely quantify DNA methylation at single-nucleotide resolution using progressively smaller tissue quantities, many of the most commonly used methods for profiling and quantification of DNA methylation, such as bisulfite sequencing and methylation-sensitive enzyme-based assays, are unable to distinguish between 5hmC and 5mC ([Tahiliani et al, 2009](https://www.nature.com/articles/npp2012112#ref-CR181 "Tahiliani M, Koh KP, Shen Y, Pastor WA, Bandukwala H, Brudno Y et al \(2009\). Conversion of 5-methylcytosine to 5-hydroxymethylcytosine in mammalian DNA by MLL partner TET1. Science 324: 930–935. The first study to demonstrate the role of Tet enzymes in conversion of 5-methylcytosine to 5-hydroxymethylcytosine, thus leading to recent studies of the DNA demethylation mechanism in the mammalian system."); [Huang et al, 2010](https://www.nature.com/articles/npp2012112#ref-CR82 "Huang Y, Pastor WA, Shen Y, Tahiliani M, Liu DR, Rao A \(2010\). The behaviour of 5-hydroxymethylcytosine in bisulfite sequencing. PLoS One 5: e8888.")). A few protocols are capable of distinguishing 5hmC from 5mC in the genome: CpG end-labeling followed by thin-layer chromatography ([Tahiliani et al, 2009](https://www.nature.com/articles/npp2012112#ref-CR181 "Tahiliani M, Koh KP, Shen Y, Pastor WA, Bandukwala H, Brudno Y et al \(2009\). Conversion of 5-methylcytosine to 5-hydroxymethylcytosine in mammalian DNA by MLL partner TET1. Science 324: 930–935. The first study to demonstrate the role of Tet enzymes in conversion of 5-methylcytosine to 5-hydroxymethylcytosine, thus leading to recent studies of the DNA demethylation mechanism in the mammalian system.")) and high-performance liquid chromatography (HPLC) with either UV detection ([Liutkeviciute et al, 2009](https://www.nature.com/articles/npp2012112#ref-CR121 "Liutkeviciute Z, Lukinavicius G, Masevicius V, Daujotyte D, Klimasauskas S \(2009\). Cytosine-5-methyltransferases add aldehydes to DNA. Nat Chem Biol 5: 400–402.")) or tandem mass spectrometry ([Globisch et al, 2010](https://www.nature.com/articles/npp2012112#ref-CR63 "Globisch D, Munzel M, Muller M, Michalakis S, Wagner M, Koch S et al \(2010\). Tissue distribution of 5-hydroxymethylcytosine and search for active demethylation intermediates. PLoS One 5: e15367."); [Le et al, 2011](https://www.nature.com/articles/npp2012112#ref-CR110 "Le T, Kim KP, Fan G, Faull KF \(2011\). A sensitive mass spectrometry method for simultaneous quantification of DNA methylation and hydroxymethylation levels in biological samples. Anal Biochem 412: 203–209.")). Hydroxymethylated DNA can be enriched using antibodies that bind specifically to 5hmC or by biotinylation of modified 5hmC and precipitated sequences can be identified using microarray chips or by DNA sequencing ([Szwagierczak et al, 2010](https://www.nature.com/articles/npp2012112#ref-CR180 "Szwagierczak A, Bultmann S, Schmidt CS, Spada F, Leonhardt H \(2010\). Sensitive enzymatic quantification of 5-hydroxymethylcytosine in genomic DNA. Nucleic Acids Res 38: e181."); [Ficz et al, 2011](https://www.nature.com/articles/npp2012112#ref-CR51 "Ficz G, Branco MR, Seisenberger S, Santos F, Krueger F, Hore TA et al \(2011\). Dynamic regulation of 5-hydroxymethylcytosine in mouse ES cells and during differentiation. Nature 473: 398–402."); [Jin et al, 2011](https://www.nature.com/articles/npp2012112#ref-CR94 "Jin SG, Wu X, Li AX, Pfeifer GP \(2011\). Genomic mapping of 5-hydroxymethylcytosine in the human brain. Nucleic Acids Res 39: 5015–5024."); [Pastor et al, 2011](https://www.nature.com/articles/npp2012112#ref-CR157 "Pastor WA, Pape UJ, Huang Y, Henderson HR, Lister R, Ko M et al \(2011\). Genome-wide mapping of 5-hydroxymethylcytosine in embryonic stem cells. Nature 473: 394–397."); [Wu and Zhang, 2011](https://www.nature.com/articles/npp2012112#ref-CR198 "Wu H, Zhang Y \(2011\). Tet1 and 5-hydroxymethylation: a genome-wide view in mouse embryonic stem cells. Cell Cycle 10: 2428–2436.")). Although these methods can quantify 5hmC and identify DNA sequences with which it is associated, single base-pair resolution has not been attained. In order to clarify the genomic distribution and the epigenetic role of 5hmC in the brain, a locus-specific method of identifying 5hmC will need to be developed.
As other high-throughput techniques, including RNA and chromatin immunoprecipitation (ChIP) sequencing, become more accessible to researchers, there is a growing need to integrate high-throughput data. Currently, DNA methylation, histone modification, and miRNA are studied in relative isolation. In order to fully understand how gene expression is regulated within the nervous system, future research must consider the epigenome as a whole. By dissecting the biological mechanisms that mediate crosstalk among these biological mechanisms and integrating high-throughput data, we can begin to study the epigenome as a whole. Finally, for a complete understanding of how the epigenome regulates gene expression, future research will have to uncover the biological mechanisms that mediate activity-dependent changes in the epigenomic landscape of the mammalian brain.
## References
  * Aapola U, Kawasaki K, Scott HS, Ollila J, Vihinen M, Heino M _et al_ (2000). Isolation and initial characterization of a novel zinc finger gene, DNMT3L, on 21q22.3, related to the cytosine-5-methyltransferase 3 gene family. _Genomics_ **65** : 293–298. **Identified Dnmt3L and profiled its expression**.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD3cXktFarsrc%3D)
  * Aapola U, Lyle R, Krohn K, Antonarakis SE, Peterson P (2001). Isolation and initial characterization of the mouse Dnmt3l gene. _Cytogenet Cell Genet_ **92** : 122–126.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD3MXjs12gtr4%3D)
  * Achour M, Jacq X, Ronde P, Alhosin M, Charlot C, Chataigneau T _et al_ (2008). The interaction of the SRA domain of ICBP90 with a novel domain of DNMT1 is involved in the regulation of VEGF gene expression. _Oncogene_ **27** : 2187–2197.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1cXktFCnsrg%3D)
  * Amir RE, Van den Veyver IB, Wan M, Tran CQ, Francke U, Zoghbi HY (1999). Rett syndrome is caused by mutations in X-linked MECP2, encoding methyl-CpG-binding protein 2. _Nat Genet_ **23** : 185–188. **Discovered MECP2 mutation as the cause of Rett Syndrome and paved way for further study of MECP2 protein role in the nervous system**.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK1MXmtlOhtbo%3D)
  * Aran D, Toperoff G, Rosenberg M, Hellman A (2011). Replication timing-related and gene body-specific methylation of active human genes. _Hum Mol Genet_ **20** : 670–680.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXpsFygtw%3D%3D)
  * Asaka Y, Jugloff DG, Zhang L, Eubanks JH, Fitzsimonds RM (2006). Hippocampal synaptic plasticity is impaired in the Mecp2-null mouse model of Rett syndrome. _Neurobiol Dis_ **21** : 217–227.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2MXhtlCkt7fP)
  * Avery OT, Macleod CM, McCarty M (1944). Studies on the chemical nature of the substance inducing transformation of pneumococcal types: induction of transformation by a desoxyribonucleic acid fraction isolated from pneumococcus type Iii. _J Exp Med_ **79** : 137–158.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaH2cXhtValsQ%3D%3D)
  * Ball MP, Li JB, Gao Y, Lee JH, LeProust EM, Park IH _et al_ (2009). Targeted and genome-scale strategies reveal gene-body methylation signatures in human cells. _Nat Biotechnol_ **27** : 361–368.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1MXjslart7o%3D)
  * Bellacosa A, Cicchillitti L, Schepis F, Riccio A, Yeung AT, Matsumoto Y _et al_ (1999). MED1, a novel human methyl-CpG-binding endonuclease, interacts with DNA mismatch repair protein MLH1. _Proc Natl Acad Sci USA_ **96** : 3969–3974.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK1MXjslChsbg%3D)
  * Benetti R, Gonzalo S, Jaco I, Munoz P, Gonzalez S, Schoeftner S _et al_ (2008). A mammalian microRNA cluster controls DNA methylation and telomere recombination via Rbl2-dependent regulation of DNA methyltransferases. _Nat Struct Mol Biol_ **15** : 268–279.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1cXislKmurw%3D)
  * Berezikov E (2011). Evolution of microRNA diversity and regulation in animals. _Nat Rev Genet_ **12** : 846–860.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXhsVykt7vL)
  * Bhutani N, Brady JJ, Damian M, Sacco A, Corbel SY, Blau HM (2010). Reprogramming towards pluripotency requires AID-dependent DNA demethylation. _Nature_ **463** : 1042–1047.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXitl2htb8%3D)
  * Bhutani N, Burns DM, Blau HM (2011). DNA demethylation dynamics. _Cell_ **146** : 866–872.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXhtF2ht7fK)
  * Bird AP, Taggart M, Frommer M, Miller OJ, Macleod D (1985). A fraction of the mouse genome that is derived from islands of nonmethylated, CpG-rich DNA. _Cell_ **40** : 91–99. **One of the pioneering studies that first identified CpG islands and described their sequence characteristics**.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaL2MXhtVOgt7o%3D)
  * Bird AP (1980). DNA methylation and the frequency of CpG in animal DNA. _Nucleic Acids Res_ **8** : 1499–1504.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaL3cXktVantb8%3D)
  * Bostick M, Kim JK, Esteve PO, Clark A, Pradhan S, Jacobsen SE (2007). UHRF1 plays a role in maintaining DNA methylation in mammalian cells. _Science_ **317** : 1760–1764.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2sXhtVGmsLbN)
  * Bourc’his D, Bestor TH (2004). Meiotic catastrophe and retrotransposon reactivation in male germ cells lacking Dnmt3L. _Nature_ **431** : 96–99.
  * Bourc’his D, Xu GL, Lin CS, Bollman B, Bestor TH (2001). Dnmt3L and the establishment of maternal genomic imprints. _Science_ **294** : 2536–2539. **Demonstrated the importance of Dnmt3L in the establishment of genomic imprints**.
  * Brandeis M, Frank D, Keshet I, Siegfried Z, Mendelsohn M, Nemes A _et al_ (1994). Sp1 elements protect a CpG island from de novo methylation. _Nature_ **371** : 435–438.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK2cXmsVOqtrc%3D)
  * Brenet F, Moh M, Funk P, Feierstein E, Viale AJ, Socci ND _et al_ (2011). DNA methylation of the first exon is tightly linked to transcriptional silencing. _PLoS One_ **6** : e14524.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXht1OhtLw%3D)
  * Brenner C, Deplus R, Didelot C, Loriot A, Vire E, De Smet C _et al_ (2005). Myc represses transcription through recruitment of DNA methyltransferase corepressor. _EMBO J_ **24** : 336–346.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2MXmslOnsA%3D%3D)
  * Brueckner B, Garcia Boy R, Siedlecki P, Musch T, Kliem HC, Zielenkiewicz P _et al_ (2005). Epigenetic reactivation of tumor suppressor genes by a novel small-molecule inhibitor of human DNA methyltransferases. _Cancer Res_ **65** : 6305–6311.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2MXmt1egsb8%3D)
  * Buiting K (2010). Prader-Willi syndrome and Angelman syndrome. _Am J Med Genet C Semin Med Genet_ **154C** : 365–376.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXhtFKqsr7L)
  * Carninci P, Sandelin A, Lenhard B, Katayama S, Shimokawa K, Ponjavic J _et al_ (2006). Genome-wide analysis of mammalian promoter architecture and evolution. _Nat Genet_ **38** : 626–635.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD28XltVOhtbo%3D)
  * Caspary T, Cleary MA, Baker CC, Guan XJ, Tilghman SM (1998). Multiple mechanisms regulate imprinting of the mouse distal chromosome 7 gene cluster. _Mol Cell Biol_ **18** : 3466–3474.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK1cXjsFSrs7o%3D)
  * Cervoni N, Szyf M (2001). Demethylase activity is directed by histone acetylation. _J Biol Chem_ **276** : 40778–40787.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD3MXosVKhurg%3D)
  * Chang Q, Khare G, Dani V, Nelson S, Jaenisch R (2006). The disease progression of Mecp2 mutant mice is affected by the level of BDNF expression. _Neuron_ **49** : 341–348.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD28Xhs1eiurw%3D)
  * Chen RZ, Akbarian S, Tudor M, Jaenisch R (2001). Deficiency of methyl-CpG binding protein-2 in CNS neurons results in a Rett-like phenotype in mice. _Nat Genet_ **27** : 327–331.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD3MXhslOisbo%3D)
  * Chen RZ, Pettersson U, Beard C, Jackson-Grusby L, Jaenisch R (1998). DNA hypomethylation leads to elevated mutation rates. _Nature_ **395** : 89–93.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK1cXmtVWjsr8%3D)
  * Choi JD, Underkoffler LA, Wood AJ, Collins JN, Williams PT, Golden JA _et al_ (2005). A novel variant of Inpp5f is imprinted in brain, and its expression is correlated with differential methylation of an internal CpG island. _Mol Cell Biol_ **25** : 5514–5522.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2MXlslCjtr0%3D)
  * Choi JK (2010). Contrasting chromatin organization of CpG islands and exons in the human genome. _Genome Biol_ **11** : R70.
  * Citterio E, Papait R, Nicassio F, Vecchi M, Gomiero P, Mantovani R _et al_ (2004). Np95 is a histone-binding protein endowed with ubiquitin ligase activity. _Mol Cell Biol_ **24** : 2526–2535.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2cXitVWqt7Y%3D)
  * Cohen S, Gabel HW, Hemberg M, Hutchinson AN, Sadacca LA, Ebert DH _et al_ (2011). Genome-wide activity-dependent MeCP2 phosphorylation regulates nervous system development and function. _Neuron_ **72** : 72–85.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXht12kt7bJ)
  * Compere SJ, Palmiter RD (1981). DNA methylation controls the inducibility of the mouse metallothionein-I gene lymphoid cells. _Cell_ **25** : 233–240.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaL3MXkvFCmurw%3D)
  * Cortellino S, Xu J, Sannai M, Moore R, Caretti E, Cigliano A _et al_ (2011). Thymine DNA glycosylase is essential for active DNA demethylation by linked deamination-base excision repair. _Cell_ **146** : 67–79.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXoslCks74%3D)
  * Coulondre C, Miller JH, Farabaugh PJ, Gilbert W (1978). Molecular basis of base substitution hotspots in Escherichia coli. _Nature_ **274** : 775–780.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaE1MXpvFylsA%3D%3D)
  * Creusot F, Acs G, Christman JK (1982). Inhibition of DNA methyltransferase and induction of Friend erythroleukemia cell differentiation by 5-azacytidine and 5-aza-2′-deoxycytidine. _J Biol Chem_ **257** : 2041–2048.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaL38XhtlGntLw%3D)
  * D’Alessio AC, Weaver IC, Szyf M (2007). Acetylation-induced transcription is required for active DNA demethylation in methylation-silenced genes. _Mol Cell Biol_ **27** : 7462–7474. **An example of coordinated actions from both histone modification and DNA demethylation in regulating gene regulation**.
  * Daniel JM, Spring CM, Crawford HC, Reynolds AB, Baig A (2002). The p120(ctn)-binding partner Kaiso is a bi-modal DNA-binding protein that recognizes both a sequence-specific consensus and methylated CpG dinucleotides. _Nucleic Acids Res_ **30** : 2911–2919.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD38XlsFCqsLs%3D)
  * Dawlaty MM, Ganz K, Powell BE, Hu YC, Markoulaki S, Cheng AW _et al_ (2011). Tet1 is dispensable for maintaining pluripotency and its loss is compatible with embryonic and postnatal development. _Cell Stem Cell_ **9** : 166–175.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXpvFGit74%3D)
  * Deng JV, Rodriguiz RM, Hutchinson AN, Kim IH, Wetsel WC, West AE (2010). MeCP2 in the nucleus accumbens contributes to neural and behavioral responses to psychostimulants. _Nat Neurosci_ **13** : 1128–1136.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXhtVais7jP)
  * Devys D, Lutz Y, Rouyer N, Bellocq JP, Mandel JL (1993). The FMR-1 protein is cytoplasmic, most abundant in neurons and appears normal in carriers of a fragile X premutation. _Nat Genet_ **4** : 335–340.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK3sXlsFOlsL8%3D)
  * Dhayalan A, Rajavelu A, Rathert P, Tamas R, Jurkowska RZ, Ragozin S _et al_ (2010). The Dnmt3a PWWP domain reads histone 3 lysine 36 trimethylation and guides DNA methylation. _J Biol Chem_ **285** : 26114–26120.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXhtVahtbvM)
  * Eckhardt F, Lewin J, Cortese R, Rakyan VK, Attwood J, Burger M _et al_ (2006). DNA methylation profiling of human chromosomes 6, 20 and 22. _Nat Genet_ **38** : 1378–1385.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD28Xht1CntrbK)
  * Ehrlich M, Gama-Sosa MA, Huang LH, Midgett RM, Kuo KC, McCune RA _et al_ (1982). Amount and distribution of 5-methylcytosine in human DNA from different types of tissues of cells. _Nucleic Acids Res_ **10** : 2709–2721.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaL38XitFCrur8%3D)
  * Fan G, Beard C, Chen RZ, Csankovszki G, Sun Y, Siniaia M _et al_ (2001). DNA hypomethylation perturbs the function and survival of CNS neurons in postnatal animals. _J Neurosci_ **21** : 788–797. **One of the pioneering works in investigating the role of Dnmts in the CNS**.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD3MXhtF2qurg%3D)
  * Fan G, Martinowich K, Chin MH, He F, Fouse SD, Hutnick L _et al_ (2005). DNA methylation controls the timing of astrogliogenesis through regulation of JAK-STAT signaling. _Development_ **132** : 3345–3356.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2MXpvFOju7c%3D)
  * Fatemi SH, Folsom TD (2011). The role of fragile X mental retardation protein in major mental disorders. _Neuropharmacology_ **60** : 1221–1226.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXkvVKitbo%3D)
  * Feng J, Chang H, Li E, Fan G (2005). Dynamic expression of de novo DNA methyltransferases Dnmt3a and Dnmt3b in the central nervous system. _J Neurosci Res_ **79** : 734–746.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2MXitFygsLg%3D)
  * Feng J, Zhou Y, Campbell SL, Le T, Li E, Sweatt JD _et al_ (2010). Dnmt1 and Dnmt3a maintain DNA methylation and regulate synaptic function in adult forebrain neurons. _Nat Neurosci_ **13** : 423–430. **Demonstrated the importance of continual activity of Dnmts in postmitotic neurons and that the Dnmts exhibit overlapping roles of each other**.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXjtFynur8%3D)
  * Ficz G, Branco MR, Seisenberger S, Santos F, Krueger F, Hore TA _et al_ (2011). Dynamic regulation of 5-hydroxymethylcytosine in mouse ES cells and during differentiation. _Nature_ **473** : 398–402.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXktFahs7s%3D)
  * Filion GJ, Zhenilo S, Salozhin S, Yamada D, Prokhortchouk E, Defossez PA (2006). A family of human zinc finger proteins that bind methylated DNA and repress transcription. _Mol Cell Biol_ **26** : 169–181.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD28Xjslaqsbg%3D)
  * Fouse SD, Shen Y, Pellegrini M, Cole S, Meissner A, Van Neste L _et al_ (2008). Promoter CpG methylation contributes to ES cell gene regulation in parallel with Oct4/Nanog, PcG complex, and histone H3 K4/K27 trimethylation. _Cell Stem Cell_ **2** : 160–169.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1cXitlygtb0%3D)
  * Fuks F, Burgers WA, Brehm A, Hughes-Davies L, Kouzarides T (2000). DNA methyltransferase Dnmt1 associates with histone deacetylase activity. _Nat Genet_ **24** : 88–91. **Demonstrates physical connection between the two epigenetic mechanisms: DNA methylation and histone modification**.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD3cXks1eltg%3D%3D)
  * Fuks F, Hurd PJ, Deplus R, Kouzarides T (2003). The DNA methyltransferases associate with HP1 and the SUV39H1 histone methyltransferase. _Nucleic Acids Res_ **31** : 2305–2312.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD3sXjt1Ogt7o%3D)
  * Gardiner-Garden M, Frommer M (1987). CpG islands in vertebrate genomes. _J Mol Biol_ **196** : 261–282.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaL2sXlt1ygsrc%3D)
  * Gaudet F, Rideout III WM, Meissner A, Dausman J, Leonhardt H, Jaenisch R (2004). Dnmt1 expression in pre- and postimplantation embryogenesis and the maintenance of IAP silencing. _Mol Cell Biol_ **24** : 1640–1648.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2cXhtFOjtrw%3D)
  * Ge YZ, Pu MT, Gowher H, Wu HP, Ding JP, Jeltsch A _et al_ (2004). Chromatin targeting of de novo DNA methyltransferases by the PWWP domain. _J Biol Chem_ **279** : 25447–25454.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2cXksFygsLY%3D)
  * Gebhard C, Benner C, Ehrich M, Schwarzfischer L, Schilling E, Klug M _et al_ (2010). General transcription factor binding at CpG islands in normal cells correlates with resistance to de novo DNA methylation in cancer cells. _Cancer Res_ **70** : 1398–1407.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXhvFaks7Y%3D)
  * Geiman TM, Sankpal UT, Robertson AK, Zhao Y, Zhao Y, Robertson KD (2004). DNMT3B interacts with hSNF2H chromatin remodeling enzyme, HDACs 1 and 2, and components of the histone methylation system. _Biochem Biophys Res Commun_ **318** : 544–555.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2cXjs1CjsLs%3D)
  * Ghosh S, Yates AJ, Frühwald MC, Miecznikowski JC, Plass C, Smiraglia D (2010). Tissue specific DNA methylation of CpG islands in normal human adult somatic tissues distinguishes neural from non-neural tissues. _Epigenetics_ **5** : 527–538.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXht1Clt7jI)
  * Giacometti E, Luikenhuis S, Beard C, Jaenisch R (2007). Partial rescue of MeCP2 deficiency by postnatal activation of MeCP2. _Proc Natl Acad Sci USA_ **104** : 1931–1936.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2sXitVejs7Y%3D)
  * Globisch D, Munzel M, Muller M, Michalakis S, Wagner M, Koch S _et al_ (2010). Tissue distribution of 5-hydroxymethylcytosine and search for active demethylation intermediates. _PLoS One_ **5** : e15367.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXjvVSluw%3D%3D)
  * Golshani P, Hutnick L, Schweizer F, Fan G (2005). Conditional Dnmt1 deletion in dorsal forebrain disrupts development of somatosensory barrel cortex and thalamocortical long-term potentiation. _Thalamus Relat Syst_ **3** : 227–233.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2sXmtl2gsr0%3D)
  * Goto K, Numata M, Komura JI, Ono T, Bestor TH, Kondo H (1994). Expression of DNA methyltransferase gene in mature and immature neurons as well as proliferating cells in mice. _Differentiation_ **56** : 39–44. **Identified DNA methyltransferase mRNA in postmitotic neurons and suggested that DNA methylation served a unique function in neurons**.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK2cXksFCnu70%3D)
  * Guo J, Ma DK, Mo H, Ball MP, Jang MH, Bonaguidi MA _et al_ (2011a). Neuronal activity modifies the DNA methylation landscape in the adult brain. _Nat Neurosci_ **14** : 1345–1351. **Investigated the extent to which the DNA methylation pattern is modifiable by neural activity** **_in vivo_**.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXhtV2gt7jJ)
  * Guo J, Su Y, Zhong C, Ming GL, Song H (2011b). Hydroxylation of 5-methylcytosine by TET1 promotes active DNA demethylation in the adult brain. _Cell_ **145** : 423–434. **Also discovered few gene regions, including BDNF, that exhibit demethylation and remethylation cycle in postmitotic neurons**.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXlsFShsrs%3D)
  * Guy J, Gan J, Selfridge J, Cobb S, Bird A (2007). Reversal of neurological defects in a mouse model of Rett syndrome. _Science_ **315** : 1143–1147.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2sXhvFWrs7s%3D)
  * Guy J, Hendrich B, Holmes M, Martin JE, Bird A (2001). A mouse Mecp2-null mutation causes neurological symptoms that mimic Rett syndrome. _Nat Genet_ **27** : 322–326.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD3MXhslOisLc%3D)
  * Gwynn B, Lueders K, Sands MS, Birkenmeier EH (1998). Intracisternal A-particle element transposition into the murine beta-glucuronidase gene correlates with loss of enzyme activity: a new model for beta-glucuronidase deficiency in the C3H mouse. _Mol Cell Biol_ **18** : 6474–6481.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK1cXmvF2nsr4%3D)
  * Han L, Witmer PD, Casey E, Valle D, Sukumar S (2007). DNA methylation regulates microRNA expression. _Cancer Biol Ther_ **6** : 1284–1288.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1cXkvVahtw%3D%3D)
  * Hashimoto H, Horton JR, Zhang X, Bostick M, Jacobsen SE, Cheng X (2008). The SRA domain of UHRF1 flips 5-methylcytosine out of the DNA helix. _Nature_ **455** : 826–829.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1cXht1WktLnE)
  * Hashimoto H, Horton JR, Zhang X, Cheng X (2009). UHRF1, a modular multi-domain protein, regulates replication-coupled crosstalk between DNA methylation and histone modifications. _Epigenetics_ **4** : 8–14.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1MXovF2jt70%3D)
  * Hata K, Okano M, Lei H, Li E (2002). Dnmt3L cooperates with the Dnmt3 family of de novo DNA methyltransferases to establish maternal imprints in mice. _Development_ **129** : 1983–1993.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD38XjslSls70%3D)
  * He YF, Li BZ, Li Z, Liu P, Wang Y, Tang Q _et al_ (2011). Tet-mediated formation of 5-carboxylcytosine and its excision by TDG in mammalian DNA. _Science_ **333** : 1303–1307.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXhtV2jt7nO)
  * Hellman A, Chess A (2007). Gene body-specific methylation on the active X chromosome. _Science_ **315** : 1141–1143.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2sXhvFWrs7o%3D)
  * Hendrich B, Bird A (1998). Identification and characterization of a family of mammalian methyl-CpG binding proteins. _Mol Cell Biol_ **18** : 6538–6547.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK1cXmvF2nsrg%3D)
  * Hendrich B, Hardeland U, Ng HH, Jiricny J, Bird A (1999). The thymine glycosylase MBD4 can bind to the product of deamination at methylated CpG sites. _Nature_ **401** : 301–304.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK1MXmt1yrsrs%3D)
  * Hermann A, Goyal R, Jeltsch A (2004). The Dnmt1 DNA-(cytosine-C5)-methyltransferase methylates DNA processively with high preference for hemimethylated target sites. _J Biol Chem_ **279** : 48350–48359.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2cXptl2nu78%3D)
  * Holliday R, Pugh JE (1975). DNA modification mechanisms and gene activity during development. _Science_ **187** : 226–232.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaE2MXht1Sitrw%3D)
  * Hotchkiss RD (1948). The quantitative separation of purines, pyrimidines, and nucleosides by paper chromatography. _J Biol Chem_ **175** : 315–332. **Discovered 5-methylcytosine in the mammalian genome**.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaH1cXkt1Sqtg%3D%3D)
  * Huang Y, Pastor WA, Shen Y, Tahiliani M, Liu DR, Rao A (2010). The behaviour of 5-hydroxymethylcytosine in bisulfite sequencing. _PLoS One_ **5** : e8888.
  * Hutchinson AN, Deng JV, Aryal DK, Wetsel WC, West AE (2012). Differential regulation of MeCP2 phosphorylation in the CNS by dopamine and serotonin. _Neuropsychopharmacology_ **37** : 321–337.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXhs1eju73F)
  * Hutnick LK, Golshani P, Namihira M, Xue Z, Matynia A, Yang XW _et al_ (2009). DNA hypomethylation restricted to the murine forebrain induces cortical degeneration and impairs postnatal neuronal maturation. _Hum Mol Genet_ **18** : 2875–2888.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1MXosVans7g%3D)
  * Hutnick LK, Huang X, Loo TC, Ma Z, Fan G (2010). Repression of retrotransposal elements in mouse embryonic stem cells is primarily mediated by a DNA methylation-independent mechanism. _J Biol Chem_ **285** : 21082–21091.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXnvFSiu7o%3D)
  * Illingworth RS, Gruenewald-Schneider U, Webb S, Kerr AR, James KD, Turner DJ _et al_ (2010). Orphan CpG islands identify numerous conserved promoters in the mammalian genome. _PLoS Genet_ **6** : e1001134.
  * Im HI, Hollander JA, Bali P, Kenny PJ (2010). MeCP2 controls BDNF expression and cocaine intake through homeostatic interactions with microRNA-212. _Nat Neurosci_ **13** : 1120–1127.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXhtVaisLzJ)
  * Inano K, Suetake I, Ueda T, Miyake Y, Nakamura M, Okada M _et al_ (2000). Maintenance-type DNA methyltransferase is highly expressed in post-mitotic neurons and localized in the cytoplasmic compartment. _J Biochem_ **128** : 315–321.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD3cXlvVeksbs%3D)
  * Irizarry RA, Ladd-Acosta C, Wen B, Wu Z, Montano C, Onyango P _et al_ (2009). The human colon cancer methylome shows similar hypo- and hypermethylation at conserved tissue-specific CpG island shores. _Nat Genet_ **41** : 178–186.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1MXntFGrtA%3D%3D)
  * Ito S, D’Alessio AC, Taranova OV, Hong K, Sowers LC, Zhang Y (2010). Role of Tet proteins in 5mC to 5hmC conversion, ES-cell self-renewal and inner cell mass specification. _Nature_ **466** : 1129–1133.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXovFCntro%3D)
  * Ito S, Shen L, Dai Q, Wu SC, Collins LB, Swenberg JA _et al_ (2011). Tet proteins can convert 5-methylcytosine to 5-formylcytosine and 5-carboxylcytosine. _Science_ **333** : 1300–1303.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXhtV2jt7nN)
  * Jackson-Grusby L, Beard C, Possemato R, Tudor M, Fambrough D, Csankovszki G _et al_ (2001). Loss of genomic methylation causes p53-dependent apoptosis and epigenetic deregulation. _Nat Genet_ **27** : 31–39.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD3MXis1yksw%3D%3D)
  * Jia D, Jurkowska RZ, Zhang X, Jeltsch A, Cheng X (2007). Structure of Dnmt3a bound to Dnmt3L suggests a model for de novo DNA methylation. _Nature_ **449** : 248–251.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2sXhtVagtbrF)
  * Jin SG, Wu X, Li AX, Pfeifer GP (2011). Genomic mapping of 5-hydroxymethylcytosine in the human brain. _Nucleic Acids Res_ **39** : 5015–5024.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXoslOhu7g%3D)
  * Jones PL, Veenstra GJ, Wade PA, Vermaak D, Kass SU, Landsberger N _et al_ (1998). Methylated DNA and MeCP2 recruit histone deacetylase to repress transcription. _Nat Genet_ **19** : 187–191.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK1cXls1Cqsbg%3D)
  * Jüttermann R, Li E, Jaenisch R (1994). Toxicity of 5-aza-2′-deoxycytidine to mammalian cells is mediated primarily by covalent trapping of DNA methyltransferase rather than DNA demethylation. _Proc Natl Acad Sci USA_ **91** : 11797–11801.
  * Kaneda M, Okano M, Hata K, Sado T, Tsujimoto N, Li E _et al_ (2004). Essential role for de novo DNA methyltransferase Dnmt3a in paternal and maternal imprinting. _Nature_ **429** : 900–903.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2cXltVKltL8%3D)
  * Kantor B, Kaufman Y, Makedonski K, Razin A, Shemer R (2004). Establishing the epigenetic status of the Prader-Willi/Angelman imprinting center in the gametes and embryo. _Hum Mol Genet_ **13** : 2767–2779.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2cXhtVSkt7jF)
  * Karagianni P, Amazit L, Qin J, Wong J (2008). ICBP90, a novel methyl K9 H3 binding protein linking protein ubiquitination with heterochromatin formation. _Mol Cell Biol_ **28** : 705–717.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1cXkslGmug%3D%3D)
  * Kimura H, Shiota K (2003). Methyl-CpG-binding protein, MeCP2, is a target molecule for maintenance DNA methyltransferase, Dnmt1. _J Biol Chem_ **278** : 4806–4812.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD3sXhtVKlsL4%3D)
  * Kishi N, Macklis JD (2004). MECP2 is progressively expressed in post-migratory neurons and is involved in neuronal maturation rather than cell fate decisions. _Mol Cell Neurosci_ **27** : 306–321.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2cXpt1Wjs70%3D)
  * Klein CJ, Botuyan MV, Wu Y, Ward CJ, Nicholson GA, Hammans S _et al_ (2011). Mutations in DNMT1 cause hereditary sensory neuropathy with dementia and hearing loss. _Nat Genet_ **43** : 595–600.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXlsVGlurs%3D)
  * Kovacheva VP, Mellott TJ, Davison JM, Wagner N, Lopez-Coviella I, Schnitzler AC _et al_ (2007). Gestational choline deficiency causes global and Igf2 gene DNA hypermethylation by up-regulation of Dnmt1 expression. _J Biol Chem_ **282** : 31777–31788.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2sXhtFyku77L)
  * Kriaucionis S, Heintz N (2009). The nuclear DNA base 5-hydroxymethylcytosine is present in Purkinje neurons and the brain. _Science_ **324** : 929–930. **This is first paper describing that hydroxymethylcytosine is enriched in postmitotic CNS neurons**.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1MXlslWnurk%3D)
  * Kuster JE, Guarnieri MH, Ault JG, Flaherty L, Swiatek PJ (1997). IAP insertion in the murine LamB3 gene results in junctional epidermolysis bullosa. _Mamm Genome_ **8** : 673–681.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK2sXls1Gqsbw%3D)
  * La Salle S, Oakes CC, Neaga OR, Bourc’his D, Bestor TH, Trasler JM (2007). Loss of spermatogonia and wide-spread DNA methylation defects in newborn male mice deficient in DNMT3L. _BMC Dev Biol_ **7** : 104.
  * Ladd-Acosta C, Pevsner J, Sabunciyan S, Yolken RH, Webster MJ, Dinkins T _et al_ (2007). DNA methylation signatures within the human brain. _Am J Hum Genet_ **81** : 1304–1315.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2sXhtl2ju7%2FI)
  * LaPlant Q, Vialou V, Covington III HE, Dumitriu D, Feng J, Warren BL _et al_ (2010). Dnmt3a regulates emotional behavior and spine plasticity in the nucleus accumbens. _Nat Neurosci_ **13** : 1137–1143.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXhtVGgtLfJ)
  * LaSalle JM, Goldstine J, Balmer D, Greco CM (2001). Quantitative localization of heterogeneous methyl-CpG-binding protein 2 (MeCP2) expression phenotypes in normal and Rett syndrome brain by laser scanning cytometry. _Hum Mol Genet_ **10** : 1729–1740.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD3MXntFCitLo%3D)
  * Le T, Kim KP, Fan G, Faull KF (2011). A sensitive mass spectrometry method for simultaneous quantification of DNA methylation and hydroxymethylation levels in biological samples. _Anal Biochem_ **412** : 203–209.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXkt1yqurg%3D)
  * Lee JH, Skalnik DG (2005). CpG-binding protein (CXXC finger protein 1) is a component of the mammalian Set1 histone H3-Lys4 methyltransferase complex, the analogue of the yeast Set1/COMPASS complex. _J Biol Chem_ **280** : 41725–41731.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2MXhtlShsbbM)
  * Lee MS, Jun DH, Hwang CI, Park SS, Kang JJ, Park HS _et al_ (2006). Selection of neural differentiation-specific genes by comparing profiles of random differentiation. _Stem Cells_ **24** : 1946–1955.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD28XhtFKiu77J)
  * Leonhardt H, Page AW, Weier HU, Bestor TH (1992). A targeting sequence directs DNA methyltransferase to sites of DNA replication in mammalian nuclei. _Cell_ **71** : 865–873.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK3sXjtl2hsg%3D%3D)
  * Levenson JM, Roth TL, Lubin FD, Miller CA, Huang IC, Desai P _et al_ (2006). Evidence that DNA (cytosine-5) methyltransferase regulates synaptic plasticity in the hippocampus. _J Biol Chem_ **281** : 15763–15773.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD28Xlt1Ckt7c%3D)
  * Lewis JD, Meehan RR, Henzel WJ, Maurer-Fogy I, Jeppesen P, Klein F _et al_ (1992). Purification, sequence, and cellular localization of a novel chromosomal protein that binds to methylated DNA. _Cell_ **69** : 905–914.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK3sXitVOrtrg%3D)
  * Li BZ, Huang Z, Cui QY, Song XH, Du L, Jeltsch A _et al_ (2011a). Histone tails regulate DNA methylation by allosterically activating de novo methyltransferase. _Cell Res_ **21** : 1172–1181.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXps1yrur4%3D)
  * Li E, Bestor TH, Jaenisch R (1992). Targeted mutation of the DNA methyltransferase gene results in embryonic lethality. _Cell_ **69** : 915–926. **Developed the Dnmt1 knockout mice**.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK38XksVGgsr0%3D)
  * Li H, Zhong X, Chau KF, Williams EC, Chang Q (2011b). Loss of activity-induced phosphorylation of MeCP2 enhances synaptogenesis, LTP and spatial memory. _Nat Neurosci_ **14** : 1001–1008.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXovFyhu7s%3D)
  * Lienert F, Wirbelauer C, Som I, Dean A, Mohn F, Schubeler D (2011). Identification of genetic elements that autonomously determine DNA methylation states. _Nat Genet_ **43** : 1091–1097. **Demonstrated that transcription factor binding regulates DNA methylation by inserting 50 different DNA elements into the same chromosome region**.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXht1Crt7zE)
  * Lister R, Pelizzola M, Dowen RH, Hawkins RD, Hon G, Tonti-Filippini J _et al_ (2009). Human DNA methylomes at base resolution show widespread epigenomic differences. _Nature_ **462** : 315–322.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1MXht1Kiu73E)
  * Liutkeviciute Z, Lukinavicius G, Masevicius V, Daujotyte D, Klimasauskas S (2009). Cytosine-5-methyltransferases add aldehydes to DNA. _Nat Chem Biol_ **5** : 400–402.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1MXls1Cmtro%3D)
  * Lopes EC, Valls E, Figueroa ME, Mazur A, Meng FG, Chiosis G _et al_ (2008). Kaiso contributes to DNA methylation-dependent silencing of tumor suppressor genes in colon cancer cell lines. _Cancer Res_ **68** : 7258–7263.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1cXhtFWltLbO)
  * Lubin FD, Roth TL, Sweatt JD (2008). Epigenetic regulation of BDNF gene transcription in the consolidation of fear memory. _J Neurosci_ **28** : 10576–10586.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1cXht1ymt7vI)
  * Luikenhuis S, Giacometti E, Beard CF, Jaenisch R (2004). Expression of MeCP2 in postmitotic neurons rescues Rett syndrome in mice. _Proc Natl Acad Sci USA_ **101** : 6033–6038.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2cXjsFKnsbs%3D)
  * Lujambio A, Calin GA, Villanueva A, Ropero S, Sanchez-Cespedes M, Blanco D _et al_ (2008). A microRNA DNA methylation signature for human cancer metastasis. _Proc Natl Acad Sci USA_ **105** : 13556–13561.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1cXhtFeqs7bF)
  * Ma DK, Jang MH, Guo JU, Kitabatake Y, Chang ML, Pow-Anpongkul N _et al_ (2009). Neuronal activity-induced Gadd45b promotes epigenetic DNA demethylation and adult neurogenesis. _Science_ **323** : 1074–1077.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1MXitVyntrg%3D)
  * Macleod D, Charlton J, Mullins J, Bird AP (1994). Sp1 sites in the mouse aprt gene promoter are required to prevent methylation of the CpG island. _Genes Dev_ **8** : 2282–2292.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK2MXhtF2gsbs%3D)
  * Martin Caballero I, Hansen J, Leaford D, Pollard S, Hendrich BD (2009). The methyl-CpG binding proteins Mecp2, Mbd2 and Kaiso are dispensable for mouse embryogenesis, but play a redundant function in neural differentiation. _PLoS One_ **4** : e4315.
  * Martinowich K, Hattori D, Wu H, Fouse S, He F, Hu Y _et al_ (2003). DNA methylation-related chromatin remodeling in activity-dependent BDNF gene regulation. _Science_ **302** : 890–893. **Demonstrated that methylation and MeCP2 binding to the BDNF promoter are regulated by neuronal activity**.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD3sXos1Cmt74%3D)
  * Maunakea AK, Nagarajan RP, Bilenky M, Ballinger TJ, D’Souza C, Fouse SD _et al_ (2010). Conserved role of intragenic DNA methylation in regulating alternative promoters. _Nature_ **466** : 253–257.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXosVKiu70%3D)
  * Mayer W, Niveleau A, Walter J, Fundele R, Haaf T (2000). Demethylation of the zygotic paternal genome. _Nature_ **403** : 501–502.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD3cXht1ShsbY%3D)
  * McCarty M, Avery OT (1946). Studies on the chemical nature of the substance inducing transformation of pneumococcal types: II. Effect of desoxyribonuclease on the biological activity of the transforming substance. _J Exp Med_ **83** : 89–96.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaH28XhtFGrtg%3D%3D)
  * McGowan PO, Sasaki A, D’Alessio AC, Dymov S, Labonte B, Szyf M _et al_ (2009). Epigenetic regulation of the glucocorticoid receptor in human brain associates with childhood abuse. _Nat Neurosci_ **12** : 342–348. **Observed childhood abuse in suicidal victims associated with higher DNA methylation level in the glucocorticoid receptor promoter in the hippocampus**.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1MXitFKlurc%3D)
  * Meehan RR, Lewis JD, McKay S, Kleiner EL, Bird AP (1989). Identification of a mammalian protein that binds specifically to DNA containing methylated CpGs. _Cell_ **58** : 499–507.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaL1MXlsVOrsrs%3D)
  * Meissner A, Mikkelsen TS, Gu H, Wernig M, Hanna J, Sivachenko A _et al_ (2008). Genome-scale DNA methylation maps of pluripotent and differentiated cells. _Nature_ **454** : 766–770.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1cXps1Sms7w%3D)
  * Michaud EJ, van Vugt MJ, Bultman SJ, Sweet HO, Davisson MT, Woychik RP (1994). Differential expression of a new dominant agouti allele (Aiapy) is correlated with methylation state and is influenced by parental lineage. _Genes Dev_ **8** : 1463–1472.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK2cXlsFSgtL4%3D)
  * Mikkelsen TS, Ku M, Jaffe DB, Issac B, Lieberman E, Giannoukos G _et al_ (2007). Genome-wide maps of chromatin state in pluripotent and lineage-committed cells. _Nature_ **448** : 553–560.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2sXosVSrtrc%3D)
  * Mill J, Tang T, Kaminsky Z, Khare T, Yazdanpanah S, Bouchard L _et al_ (2008). Epigenomic profiling reveals DNA-methylation changes associated with major psychosis. _Am J Hum Genet_ **82** : 696–711.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1cXksVGrtbY%3D)
  * Millar CB, Guy J, Sansom OJ, Selfridge J, MacDougall E, Hendrich B _et al_ (2002). Enhanced CpG mutability and tumorigenesis in MBD4-deficient mice. _Science_ **297** : 403–405.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD38Xls1els78%3D)
  * Miller CA, Gavin CF, White JA, Parrish RR, Honasoge A, Yancey CR _et al_ (2010). Cortical DNA methylation maintains remote memory. _Nat Neurosci_ **13** : 664–666.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXmsVWmsrs%3D)
  * Miller CA, Sweatt JD (2007). Covalent modification of DNA regulates memory formation. _Neuron_ **53** : 857–869.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2sXktFCmtLg%3D)
  * Mohn F, Weber M, Rebhan M, Roloff TC, Richter J, Stadler MB _et al_ (2008). Lineage-specific polycomb targets and de novo DNA methylation define restriction and potential of neuronal progenitors. _Mol Cell_ **30** : 755–766.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1cXnvFWnt7g%3D)
  * Moretti P, Levenson JM, Battaglia F, Atkinson R, Teague R, Antalffy B _et al_ (2006). Learning and memory and synaptic plasticity are impaired in a mouse model of Rett syndrome. _J Neurosci_ **26** : 319–327.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD28XmtV2ltw%3D%3D)
  * Morris KV, Chan SW, Jacobsen SE, Looney DJ (2004). Small interfering RNA-induced transcriptional gene silencing in human cells. _Science_ **305** : 1289–1292.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2cXmvFCit78%3D)
  * Mortusewicz O, Schermelleh L, Walter J, Cardoso MC, Leonhardt H (2005). Recruitment of DNA methyltransferase I to DNA repair sites. _Proc Natl Acad Sci USA_ **102** : 8905–8909.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2MXlvF2qurY%3D)
  * Muramatsu M, Kinoshita K, Fagarasan S, Yamada S, Shinkai Y, Honjo T (2000). Class switch recombination and hypermutation require activation-induced cytidine deaminase (AID), a potential RNA editing enzyme. _Cell_ **102** : 553–563.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD3cXmsFWit7g%3D)
  * Muto M, Kanari Y, Kubo E, Takabe T, Kurihara T, Fujimori A _et al_ (2002). Targeted disruption of Np95 gene renders murine embryonic stem cells hypersensitive to DNA damaging agents and DNA replication blocks. _J Biol Chem_ **277** : 34549–34555.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD38Xnt1Wisr0%3D)
  * Nan X, Meehan RR, Bird A (1993). Dissection of the methyl-CpG binding domain from the chromosomal protein MeCP2. _Nucleic Acids Res_ **21** : 4886–4892.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK2cXjtFar)
  * Nan X, Ng HH, Johnson CA, Laherty CD, Turner BM, Eisenman RN _et al_ (1998). Transcriptional repression by the methyl-CpG-binding protein MeCP2 involves a histone deacetylase complex. _Nature_ **393** : 386–389.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK1cXjsFKgurc%3D)
  * Nelson ED, Kavalali ET, Monteggia LM (2006). MeCP2-dependent transcriptional repression regulates excitatory neurotransmission. _Curr Biol_ **16** : 710–716.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD28XjtFGiurg%3D)
  * Ng HH, Zhang Y, Hendrich B, Johnson CA, Turner BM, Erdjument-Bromage H _et al_ (1999). MBD2 is a transcriptional repressor belonging to the MeCP1 histone deacetylase complex. _Nat Genet_ **23** : 58–61.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK1MXlvFWhsb8%3D)
  * Nguyen S, Meletis K, Fu D, Jhaveri S, Jaenisch R (2007). Ablation of de novo DNA methyltransferase Dnmt3a in the nervous system leads to neuromuscular defects and shortened lifespan. _Dev Dyn_ **236** : 1663–1676. **Demonstrate that knockout of Dnmt3a in neural tissue during development does not affect development of cortical neurons but results in hypomethylation of the Gfap promoter and improper motor neuron development**.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2sXnvFOht7c%3D)
  * Okano M, Bell DW, Haber DA, Li E (1999). DNA methyltransferases Dnmt3a and Dnmt3b are essential for de novo methylation and mammalian development. _Cell_ **99** : 247–257. **Demonstrated the function of Dnmt3 enzymes by studying both Dnmt3a and Dnmt3b knockout mice**.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK1MXnt1Gqsrc%3D)
  * Ooi SK, Qiu C, Bernstein E, Li K, Jia D, Yang Z _et al_ (2007). DNMT3L connects unmethylated lysine 4 of histone H3 to de novo methylation of DNA. _Nature_ **448** : 714–717.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2sXos1yksL8%3D)
  * Oswald J, Engemann S, Lane N, Mayer W, Olek A, Fundele R _et al_ (2000). Active demethylation of the paternal genome in the mouse zygote. _Curr Biol_ **10** : 475–478.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD3cXislyltLo%3D)
  * Paroush Z, Keshet I, Yisraeli J, Cedar H (1990). Dynamics of demethylation and activation of the alpha-actin gene in myoblasts. _Cell_ **63** : 1229–1237.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK3MXls1Omsw%3D%3D)
  * Pastor WA, Pape UJ, Huang Y, Henderson HR, Lister R, Ko M _et al_ (2011). Genome-wide mapping of 5-hydroxymethylcytosine in embryonic stem cells. _Nature_ **473** : 394–397.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXlvVahu7c%3D)
  * Petronzelli F, Riccio A, Markham GD, Seeholzer SH, Stoerker J, Genuardi M _et al_ (2000). Biphasic kinetics of the human DNA repair protein MED1 (MBD4), a mismatch-specific DNA N-glycosylase. _J Biol Chem_ **275** : 32422–32429.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD3cXnslOisbs%3D)
  * Popp C, Dean W, Feng S, Cokus SJ, Andrews S, Pellegrini M _et al_ (2010). Genome-wide erasure of DNA methylation in mouse primordial germ cells is affected by AID deficiency. _Nature_ **463** : 1101–1105.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXitl2gs7Y%3D)
  * Pradhan S, Bacolla A, Wells RD, Roberts RJ (1999). Recombinant human DNA (cytosine-5) methyltransferase. I. Expression, purification, and comparison of de novo and maintenance methylation. _J Biol Chem_ **274** : 33002–33010.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK1MXntlygurg%3D)
  * Prokhortchouk A, Hendrich B, Jorgensen H, Ruzov A, Wilm M, Georgiev G _et al_ (2001). The p120 catenin partner Kaiso is a DNA methylation-dependent transcriptional repressor. _Genes Dev_ **15** : 1613–1618.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD3MXltFKqt70%3D)
  * Qian X, Shen Q, Goderie SK, He W, Capela A, Davis AA _et al_ (2000). Timing of CNS cell generation: a programmed sequence of neuron and glial cell production from isolated murine cortical stem cells. _Neuron_ **28** : 69–80.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD3cXnvVGlt7o%3D)
  * Rai K, Huggins IJ, James SR, Karpf AR, Jones DA, Cairns BR (2008). DNA demethylation in zebrafish involves the coupling of a deaminase, a glycosylase, and gadd45. _Cell_ **135** : 1201–1212.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1MXisFyjtw%3D%3D)
  * Rakyan VK, Hildmann T, Novik KL, Lewin J, Tost J, Cox AV _et al_ (2004). DNA methylation profiling of the human major histocompatibility complex: a pilot study for the human epigenome project. _PLoS Biol_ **2** : e405.
  * Ramirez-Carrozzi VR, Braas D, Bhatt DM, Cheng CS, Hong C, Doty KR _et al_ (2009). A unifying model for the selective regulation of inducible transcription by CpG islands and nucleosome remodeling. _Cell_ **138** : 114–128.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1MXps1ygurg%3D)
  * Ramsahoye BH, Biniszkiewicz D, Lyko F, Clark V, Bird AP, Jaenisch R (2000). Non-CpG methylation is prevalent in embryonic stem cells and may be mediated by DNA methyltransferase 3a. _Proc Natl Acad Sci USA_ **97** : 5237–5242. **First study to demonstrate the presence of non-CG methylation in mammalian cells**.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD3cXjsVWntLs%3D)
  * Samaco RC, Neul JL (2011). Complexities of Rett syndrome and MeCP2. _J Neurosci_ **31** : 7951–7959.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXnsFykt7g%3D)
  * Sarraf SA, Stancheva I (2004). Methyl-CpG binding protein MBD1 couples histone H3 methylation at lysine 9 by SETDB1 to DNA replication and chromatin assembly. _Mol Cell_ **15** : 595–605.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2cXns1ynu70%3D)
  * Sasai N, Nakao M, Defossez PA (2010). Sequence-specific recognition of methylated DNA by human zinc-finger proteins. _Nucleic Acids Res_ **38** : 5015–5022.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXhtV2qsLnE)
  * Sauvageot CM, Stiles CD (2002). Molecular mechanisms controlling cortical gliogenesis. _Curr Opin Neurobiol_ **12** : 244–249.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD38XktFygsLo%3D)
  * Saxonov S, Berg P, Brutlag DL (2006). A genome-wide analysis of CpG dinucleotides in the human genome distinguishes two distinct classes of promoters. _Proc Natl Acad Sci USA_ **103** : 1412–1417.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD28Xhs1Gnsrs%3D)
  * Schulz WA, Steinhoff C, Florl AR (2006). Methylation of endogenous human retroelements in health and disease. _Curr Top Microbiol Immunol_ **310** : 211–250.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD28XpslGqsro%3D)
  * Shahbazian MD, Antalffy B, Armstrong DL, Zoghbi HY (2002). Insight into Rett syndrome: MeCP2 levels display tissue- and cell-specific differences and correlate with neuronal maturation. _Hum Mol Genet_ **11** : 115–124.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD38XhtlKmsrY%3D)
  * Sharif J, Muto M, Takebayashi S, Suetake I, Iwamatsu A, Endo TA _et al_ (2007). The SRA protein Np95 mediates epigenetic inheritance by recruiting Dnmt1 to methylated DNA. _Nature_ **450** : 908–912.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2sXhtl2gsb3L)
  * Shen L, Kondo Y, Guo Y, Zhang J, Zhang L, Ahmed S _et al_ (2007). Genome-wide profiling of DNA methylation reveals a class of normally methylated CpG island promoters. _PLoS Genet_ **3** : 2023–2036.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2sXhsVOnsr%2FK)
  * Sinkkonen L, Hugenschmidt T, Berninger P, Gaidatzis D, Mohn F, Artus-Revel CG _et al_ (2008). MicroRNAs control de novo DNA methylation through regulation of transcriptional repressors in mouse embryonic stem cells. _Nat Struct Mol Biol_ **15** : 259–267.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1cXislKmtbk%3D)
  * Straussman R, Nejman D, Roberts D, Steinfeld I, Blum B, Benvenisty N _et al_ (2009). Developmental programming of CpG island methylation profiles in the human genome. _Nat Struct Mol Biol_ **16** : 564–571.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1MXks12kurs%3D)
  * Stresemann C, Brueckner B, Musch T, Stopper H, Lyko F (2006). Functional diversity of DNA methyltransferase inhibitors in human cancer cell lines. _Cancer Res_ **66** : 2794–2800.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD28XhvV2gtrY%3D)
  * Suetake I, Shinozaki F, Miyagawa J, Takeshima H, Tajima S (2004). DNMT3L stimulates the DNA methylation activity of Dnmt3a and Dnmt3b through a direct interaction. _J Biol Chem_ **279** : 27816–27823.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2cXkvFGmsr8%3D)
  * Szwagierczak A, Bultmann S, Schmidt CS, Spada F, Leonhardt H (2010). Sensitive enzymatic quantification of 5-hydroxymethylcytosine in genomic DNA. _Nucleic Acids Res_ **38** : e181.
  * Tahiliani M, Koh KP, Shen Y, Pastor WA, Bandukwala H, Brudno Y _et al_ (2009). Conversion of 5-methylcytosine to 5-hydroxymethylcytosine in mammalian DNA by MLL partner TET1. _Science_ **324** : 930–935. **The first study to demonstrate the role of Tet enzymes in conversion of 5-methylcytosine to 5-hydroxymethylcytosine, thus leading to recent studies of the DNA demethylation mechanism in the mammalian system**.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1MXlslWnurY%3D)
  * Takizawa T, Nakashima K, Namihira M, Ochiai W, Uemura A, Yanagisawa M _et al_ (2001). DNA methylation is a critical cell-intrinsic determinant of astrocyte differentiation in the fetal brain. _Dev Cell_ **1** : 749–758.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD3MXptFOntbg%3D)
  * Tao J, Hu K, Chang Q, Wu H, Sherman NE, Martinowich K _et al_ (2009). Phosphorylation of MeCP2 at Serine 80 regulates its chromatin association and neurological function. _Proc Natl Acad Sci USA_ **106** : 4882–4887.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1MXktFOjs7k%3D)
  * Tazi J, Bird A (1990). Alternative chromatin structure at CpG islands. _Cell_ **60** : 909–920.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK3cXitlWmurw%3D)
  * Teter B, Osterburg HH, Anderson CP, Finch CE (1994). Methylation of the rat glial fibrillary acidic protein gene shows tissue-specific domains. _J Neurosci Res_ **39** : 680–693.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK2MXislahtLc%3D)
  * Teter B, Rozovsky I, Krohn K, Anderson C, Osterburg H, Finch C (1996). Methylation of the glial fibrillary acidic protein gene shows novel biphasic changes during brain development. _Glia_ **17** : 195–205.
[CAS](https://www.nature.com/articles/cas-redirect/1:STN:280:DyaK28vjslCrtw%3D%3D)
  * Thomson JP, Skene PJ, Selfridge J, Clouaire T, Guy J, Webb S _et al_ (2010). CpG islands influence chromatin structure via the CpG-binding protein Cfp1. _Nature_ **464** : 1082–1086.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXkslartbc%3D)
  * Turner G, Webb T, Wake S, Robinson H (1996). Prevalence of fragile X syndrome. _Am J Med Genet_ **64** : 196–197.
[CAS](https://www.nature.com/articles/cas-redirect/1:STN:280:DyaK28vitlyktw%3D%3D)
  * Ukai H, Ishii-Oba H, Ukai-Tadenuma M, Ogiu T, Tsuji H (2003). Formation of an active form of the interleukin-2/15 receptor beta-chain by insertion of the intracisternal A particle in a radiation-induced mouse thymic lymphoma and its role in tumorigenesis. _Mol Carcinog_ **37** : 110–119.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD3sXksFSitr8%3D)
  * Valinluck V, Tsai HH, Rogstad DK, Burdzy A, Bird A, Sowers LC (2004). Oxidative damage to methyl-CpG sequences inhibits the binding of the methyl-CpG binding domain (MBD) of methyl-CpG binding protein 2 (MeCP2). _Nucleic Acids Res_ **32** : 4100–4108.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2cXntFagu78%3D)
  * Verkerk AJ, Pieretti M, Sutcliffe JS, Fu YH, Kuhl DP, Pizzuti A _et al_ (1991). Identification of a gene (FMR-1) containing a CGG repeat coincident with a breakpoint cluster region exhibiting length variation in fragile X syndrome. _Cell_ **65** : 905–914.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK38Xht1ahur4%3D)
  * Walsh CP, Chaillet JR, Bestor TH (1998). Transcription of IAP endogenous retroviruses is constrained by cytosine methylation. _Nat Genet_ **20** : 116–117.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK1cXms1amsLg%3D)
  * Weaver IC, Cervoni N, Champagne FA, D’Alessio AC, Sharma S, Seckl JR _et al_ (2004). Epigenetic programming by maternal behavior. _Nat Neurosci_ **7** : 847–854. **One of the first studies to demonstrate the association of methylation changes in the brain with neural behavior**.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2cXmtVamt7c%3D)
  * Weber M, Hellmann I, Stadler MB, Ramos L, Paabo S, Rebhan M _et al_ (2007). Distribution, silencing potential and evolutionary impact of promoter DNA methylation in the human genome. _Nat Genet_ **39** : 457–466.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2sXjsV2hsrk%3D)
  * Webster KE, O’Bryan MK, Fletcher S, Crewther PE, Aapola U, Craig J _et al_ (2005). Meiotic and epigenetic defects in Dnmt3L-knockout mouse spermatogenesis. _Proc Natl Acad Sci USA_ **102** : 4068–4073.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2MXis12jtr8%3D)
  * Weiler IJ, Irwin SA, Klintsova AY, Spencer CM, Brazelton AD, Miyashiro K _et al_ (1997). Fragile X mental retardation protein is translated near synapses in response to neurotransmitter activation. _Proc Natl Acad Sci USA_ **94** : 5395–5400.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK2sXjtleis7k%3D)
  * Wong E, Yang K, Kuraguchi M, Werling U, Avdievich E, Fan K _et al_ (2002). Mbd4 inactivation increases Cright-arrowT transition mutations and promotes gastrointestinal tumor formation. _Proc Natl Acad Sci USA_ **99** : 14937–14942.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD38Xpt1yrtLY%3D)
  * Wu H, Zhang Y (2011). Tet1 and 5-hydroxymethylation: a genome-wide view in mouse embryonic stem cells. _Cell Cycle_ **10** : 2428–2436.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38XktlaqsQ%3D%3D)
  * Wu M, Rinchik EM, Wilkinson E, Johnson DK (1997). Inherited somatic mosaicism caused by an intracisternal A particle insertion in the mouse tyrosinase gene. _Proc Natl Acad Sci USA_ **94** : 890–894.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK2sXhtVOksrY%3D)
  * Wutz A, Smrzka OW, Schweifer N, Schellander K, Wagner EF, Barlow DP (1997). Imprinted expression of the Igf2r gene depends on an intronic CpG island. _Nature_ **389** : 745–749.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK2sXmvVSns7c%3D)
  * Xie S, Wang Z, Okano M, Nogami M, Li Y, He WW _et al_ (1999). Cloning, expression and chromosome locations of the human DNMT3 gene family. _Gene_ **236** : 87–95.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK1MXls1Oru7k%3D)
  * Xie W, Barr CL, Kim A, Yue F, Lee AY, Eubanks J _et al_ (2012). Base-resolution analyses of sequence and parent-of-origin dependent DNA methylation in the mouse genome. _Cell_ **148** : 816–831. **Recent article that identified significant levels of non-CpG DNA methylation in the murine frontal cortex**.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38XjtV2qsLg%3D)
  * Yen RW, Vertino PM, Nelkin BD, Yu JJ, el-Deiry W, Cumaraswamy A _et al_ (1992). Isolation and characterization of the cDNA encoding human DNA methyltransferase. _Nucleic Acids Res_ **20** : 2287–2291.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK3sXit12jsrw%3D)
  * Yoon HG, Chan DW, Reynolds AB, Qin J, Wong J (2003). N-CoR mediates DNA methylation-dependent repression through a methyl CpG binding protein Kaiso. _Mol Cell_ **12** : 723–734.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD3sXnvFGrt7o%3D)
  * Zamudio NM, Scott HS, Wolski K, Lo CY, Law C, Leong D _et al_ (2011). DNMT3L is a regulator of X chromosome compaction and post-meiotic gene transcription. _PLoS One_ **6** : e18276.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXkvVeqtr4%3D)
  * Zhang F, Pomerantz JH, Sen G, Palermo AT, Blau HM (2007). Active tissue-specific DNA demethylation conferred by somatic cell nuclei in stable heterokaryons. _Proc Natl Acad Sci USA_ **104** : 4395–4400.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2sXjsFGnsLk%3D)
  * Zhang Y, Jurkowska R, Soeroes S, Rajavelu A, Dhayalan A, Bock I _et al_ (2010). Chromatin methylation activity of Dnmt3a and Dnmt3a/3L is guided by interaction of the ADD domain with the histone H3 tail. _Nucleic Acids Res_ **38** : 4246–4253.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXpsFyisL8%3D)
  * Zhou L, Cheng X, Connolly BA, Dickman MJ, Hurd PJ, Hornby DP (2002). Zebularine: a novel DNA methylation inhibitor that forms a covalent complex with DNA methyltransferases. _J Mol Biol_ **321** : 591–599.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD38Xms1Wmtbs%3D)
  * Zhou Z, Hong EJ, Cohen S, Zhao WN, Ho HY, Schmidt L _et al_ (2006). Brain-specific phosphorylation of MeCP2 regulates activity-dependent Bdnf transcription, dendritic growth, and spine maturation. _Neuron_ **52** : 255–269.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD28Xht1WhtbjM)
  * Zwart R, Sleutels F, Wutz A, Schinkel AH, Barlow DP (2001). Bidirectional action of the Igf2r imprint control element on upstream and downstream imprinted genes. _Genes Dev_ **15** : 2361–2366.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD3MXnsV2hs7g%3D)

## Acknowledgements
We thank Kevin Huang, Jonathan Moore, and members of our laboratory for critical reading of the manuscript and constructive suggestions. This work was funded by NIH grants.
## Author information
### Authors and Affiliations
  1. Interdepartmental Program in Neuroscience and Department of Human Genetics, David Geffen School of Medicine, University of California, Los Angeles, Los Angeles, CA, USA
Lisa D Moore, Thuc Le & Guoping Fan

Authors
  1. Lisa D Moore
[View author publications](https://www.nature.com/search?author=Lisa%20D%20Moore)
You can also search for this author in
  2. Thuc Le
[View author publications](https://www.nature.com/search?author=Thuc%20Le)
You can also search for this author in
  3. Guoping Fan
[View author publications](https://www.nature.com/search?author=Guoping%20Fan)
You can also search for this author in

### Corresponding author
Correspondence to .
## Ethics declarations
### Competing interests
The authors declare no 
## PowerPoint slides
## Rights and permissions
## 
### Cite this article
Moore, L., Le, T. & Fan, G. DNA Methylation and Its Basic Function. _Neuropsychopharmacol_ **38** , 23–38 (2013). https://doi.org/10.1038/npp.2012.112
  * Received: 06 March 2012
  * Revised: 07 May 2012
  * Accepted: 08 May 2012
  * Published: 11 July 2012
  * Issue Date: January 2013
  * DOI: https://doi.org/10.1038/npp.2012.112

### 
Anyone you share the following link with will be able to read this content:
Get shareable link
Sorry, a shareable link is not currently available for this article.
Copy to clipboard
Provided by the Springer Nature SharedIt content-sharing initiative 
### Keywords
  * [epigenetics](https://www.nature.com/search?query=epigenetics&facet-discipline="Medicine%20%26%20Public%20Health")
  * [gene regulation](https://www.nature.com/search?query=gene%20regulation&facet-discipline="Medicine%20%26%20Public%20Health")
  * [neuron](https://www.nature.com/search?query=neuron&facet-discipline="Medicine%20%26%20Public%20Health")
  * [synaptic plasticity](https://www.nature.com/search?query=synaptic%20plasticity&facet-discipline="Medicine%20%26%20Public%20Health")
  * [demethylation](https://www.nature.com/search?query=demethylation&facet-discipline="Medicine%20%26%20Public%20Health")

### Subjects
  * [Epigenetics in the nervous system](https://www.nature.com/subjects/epigenetics-in-the-nervous-system)
  * [Target identification](https://www.nature.com/subjects/target-identification)

## This article is cited by
  *     * Chengzhou Wu
    * Xichen Mou
    * Hongmei Zhang
_BMC Bioinformatics_ (2024)
  *     * Ailing Wang
    * Wei-sha Zheng
    * Shi Zhang
_Journal of Translational Medicine_ (2024)
  *     * Jianhua Jia
    * Rufeng Lei
    * Xin Wei
_BMC Genomics_ (2024)
  *     * Laura A. Wortinger
    * Anne-Kristin Stavrum
    * Stephanie Le Hellard
_Translational Psychiatry_ (2024)
  *     * Xindan Zhang
    * Zhenggang Zhao
    * Fanghong Li
_Cell Death & Disease_ (2024)

[ Download PDF ](https://www.nature.com/articles/npp2012112.pdf)
  * Sections
  * Figures
  * References

  * [Abstract](https://www.nature.com/articles/npp2012112#Abs1)
  * [INTRODUCTION](https://www.nature.com/articles/npp2012112#Sec1)
  * [LOCATION OF DNA METHYLATION](https://www.nature.com/articles/npp2012112#Sec2)
  * [BASIC MECHANISM OF DNA METHYLATION](https://www.nature.com/articles/npp2012112#Sec6)
  * [DNA METHYLATION IN THE BRAIN](https://www.nature.com/articles/npp2012112#Sec12)
  * [FUTURE DIRECTIONS AND CLINICAL IMPLICATIONS](https://www.nature.com/articles/npp2012112#Sec16)
  * [References](https://www.nature.com/articles/npp2012112#Bib1)
  * [Acknowledgements](https://www.nature.com/articles/npp2012112#Ack1)
  * [Author information](https://www.nature.com/articles/npp2012112#author-information)
  * [Ethics declarations](https://www.nature.com/articles/npp2012112#ethics)
  * [PowerPoint slides](https://www.nature.com/articles/npp2012112#Sec17)
  * [Rights and permissions](https://www.nature.com/articles/npp2012112#rightslink)
  * [
  * [This article is cited by](https://www.nature.com/articles/npp2012112#further-reading)

Advertisement
  * **Figure 1**
[View in article](https://www.nature.com/articles/npp2012112#Fig1)[Full size image](https://www.nature.com/articles/npp2012112/figures/1)
  * **Figure 2**
[View in article](https://www.nature.com/articles/npp2012112#Fig2)[Full size image](https://www.nature.com/articles/npp2012112/figures/2)
  * **Figure 3**
[View in article](https://www.nature.com/articles/npp2012112#Fig3)[Full size image](https://www.nature.com/articles/npp2012112/figures/3)

  1. Aapola U, Kawasaki K, Scott HS, Ollila J, Vihinen M, Heino M _et al_ (2000). Isolation and initial characterization of a novel zinc finger gene, DNMT3L, on 21q22.3, related to the cytosine-5-methyltransferase 3 gene family. _Genomics_ **65** : 293–298. **Identified Dnmt3L and profiled its expression**.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD3cXktFarsrc%3D)
  2. Aapola U, Lyle R, Krohn K, Antonarakis SE, Peterson P (2001). Isolation and initial characterization of the mouse Dnmt3l gene. _Cytogenet Cell Genet_ **92** : 122–126.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD3MXjs12gtr4%3D)
  3. Achour M, Jacq X, Ronde P, Alhosin M, Charlot C, Chataigneau T _et al_ (2008). The interaction of the SRA domain of ICBP90 with a novel domain of DNMT1 is involved in the regulation of VEGF gene expression. _Oncogene_ **27** : 2187–2197.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1cXktFCnsrg%3D)
  4. Amir RE, Van den Veyver IB, Wan M, Tran CQ, Francke U, Zoghbi HY (1999). Rett syndrome is caused by mutations in X-linked MECP2, encoding methyl-CpG-binding protein 2. _Nat Genet_ **23** : 185–188. **Discovered MECP2 mutation as the cause of Rett Syndrome and paved way for further study of MECP2 protein role in the nervous system**.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK1MXmtlOhtbo%3D)
  5. Aran D, Toperoff G, Rosenberg M, Hellman A (2011). Replication timing-related and gene body-specific methylation of active human genes. _Hum Mol Genet_ **20** : 670–680.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXpsFygtw%3D%3D)
  6. Asaka Y, Jugloff DG, Zhang L, Eubanks JH, Fitzsimonds RM (2006). Hippocampal synaptic plasticity is impaired in the Mecp2-null mouse model of Rett syndrome. _Neurobiol Dis_ **21** : 217–227.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2MXhtlCkt7fP)
  7. Avery OT, Macleod CM, McCarty M (1944). Studies on the chemical nature of the substance inducing transformation of pneumococcal types: induction of transformation by a desoxyribonucleic acid fraction isolated from pneumococcus type Iii. _J Exp Med_ **79** : 137–158.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaH2cXhtValsQ%3D%3D)
  8. Ball MP, Li JB, Gao Y, Lee JH, LeProust EM, Park IH _et al_ (2009). Targeted and genome-scale strategies reveal gene-body methylation signatures in human cells. _Nat Biotechnol_ **27** : 361–368.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1MXjslart7o%3D)
  9. Bellacosa A, Cicchillitti L, Schepis F, Riccio A, Yeung AT, Matsumoto Y _et al_ (1999). MED1, a novel human methyl-CpG-binding endonuclease, interacts with DNA mismatch repair protein MLH1. _Proc Natl Acad Sci USA_ **96** : 3969–3974.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK1MXjslChsbg%3D)
  10. Benetti R, Gonzalo S, Jaco I, Munoz P, Gonzalez S, Schoeftner S _et al_ (2008). A mammalian microRNA cluster controls DNA methylation and telomere recombination via Rbl2-dependent regulation of DNA methyltransferases. _Nat Struct Mol Biol_ **15** : 268–279.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1cXislKmurw%3D)
  11. Berezikov E (2011). Evolution of microRNA diversity and regulation in animals. _Nat Rev Genet_ **12** : 846–860.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXhsVykt7vL)
  12. Bhutani N, Brady JJ, Damian M, Sacco A, Corbel SY, Blau HM (2010). Reprogramming towards pluripotency requires AID-dependent DNA demethylation. _Nature_ **463** : 1042–1047.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXitl2htb8%3D)
  13. Bhutani N, Burns DM, Blau HM (2011). DNA demethylation dynamics. _Cell_ **146** : 866–872.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXhtF2ht7fK)
  14. Bird AP, Taggart M, Frommer M, Miller OJ, Macleod D (1985). A fraction of the mouse genome that is derived from islands of nonmethylated, CpG-rich DNA. _Cell_ **40** : 91–99. **One of the pioneering studies that first identified CpG islands and described their sequence characteristics**.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaL2MXhtVOgt7o%3D)
  15. Bird AP (1980). DNA methylation and the frequency of CpG in animal DNA. _Nucleic Acids Res_ **8** : 1499–1504.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaL3cXktVantb8%3D)
  16. Bostick M, Kim JK, Esteve PO, Clark A, Pradhan S, Jacobsen SE (2007). UHRF1 plays a role in maintaining DNA methylation in mammalian cells. _Science_ **317** : 1760–1764.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2sXhtVGmsLbN)
  17. Bourc’his D, Bestor TH (2004). Meiotic catastrophe and retrotransposon reactivation in male germ cells lacking Dnmt3L. _Nature_ **431** : 96–99.
  18. Bourc’his D, Xu GL, Lin CS, Bollman B, Bestor TH (2001). Dnmt3L and the establishment of maternal genomic imprints. _Science_ **294** : 2536–2539. **Demonstrated the importance of Dnmt3L in the establishment of genomic imprints**.
  19. Brandeis M, Frank D, Keshet I, Siegfried Z, Mendelsohn M, Nemes A _et al_ (1994). Sp1 elements protect a CpG island from de novo methylation. _Nature_ **371** : 435–438.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK2cXmsVOqtrc%3D)
  20. Brenet F, Moh M, Funk P, Feierstein E, Viale AJ, Socci ND _et al_ (2011). DNA methylation of the first exon is tightly linked to transcriptional silencing. _PLoS One_ **6** : e14524.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXht1OhtLw%3D)
  21. Brenner C, Deplus R, Didelot C, Loriot A, Vire E, De Smet C _et al_ (2005). Myc represses transcription through recruitment of DNA methyltransferase corepressor. _EMBO J_ **24** : 336–346.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2MXmslOnsA%3D%3D)
  22. Brueckner B, Garcia Boy R, Siedlecki P, Musch T, Kliem HC, Zielenkiewicz P _et al_ (2005). Epigenetic reactivation of tumor suppressor genes by a novel small-molecule inhibitor of human DNA methyltransferases. _Cancer Res_ **65** : 6305–6311.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2MXmt1egsb8%3D)
  23. Buiting K (2010). Prader-Willi syndrome and Angelman syndrome. _Am J Med Genet C Semin Med Genet_ **154C** : 365–376.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXhtFKqsr7L)
  24. Carninci P, Sandelin A, Lenhard B, Katayama S, Shimokawa K, Ponjavic J _et al_ (2006). Genome-wide analysis of mammalian promoter architecture and evolution. _Nat Genet_ **38** : 626–635.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD28XltVOhtbo%3D)
  25. Caspary T, Cleary MA, Baker CC, Guan XJ, Tilghman SM (1998). Multiple mechanisms regulate imprinting of the mouse distal chromosome 7 gene cluster. _Mol Cell Biol_ **18** : 3466–3474.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK1cXjsFSrs7o%3D)
  26. Cervoni N, Szyf M (2001). Demethylase activity is directed by histone acetylation. _J Biol Chem_ **276** : 40778–40787.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD3MXosVKhurg%3D)
  27. Chang Q, Khare G, Dani V, Nelson S, Jaenisch R (2006). The disease progression of Mecp2 mutant mice is affected by the level of BDNF expression. _Neuron_ **49** : 341–348.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD28Xhs1eiurw%3D)
  28. Chen RZ, Akbarian S, Tudor M, Jaenisch R (2001). Deficiency of methyl-CpG binding protein-2 in CNS neurons results in a Rett-like phenotype in mice. _Nat Genet_ **27** : 327–331.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD3MXhslOisbo%3D)
  29. Chen RZ, Pettersson U, Beard C, Jackson-Grusby L, Jaenisch R (1998). DNA hypomethylation leads to elevated mutation rates. _Nature_ **395** : 89–93.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK1cXmtVWjsr8%3D)
  30. Choi JD, Underkoffler LA, Wood AJ, Collins JN, Williams PT, Golden JA _et al_ (2005). A novel variant of Inpp5f is imprinted in brain, and its expression is correlated with differential methylation of an internal CpG island. _Mol Cell Biol_ **25** : 5514–5522.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2MXlslCjtr0%3D)
  31. Choi JK (2010). Contrasting chromatin organization of CpG islands and exons in the human genome. _Genome Biol_ **11** : R70.
  32. Citterio E, Papait R, Nicassio F, Vecchi M, Gomiero P, Mantovani R _et al_ (2004). Np95 is a histone-binding protein endowed with ubiquitin ligase activity. _Mol Cell Biol_ **24** : 2526–2535.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2cXitVWqt7Y%3D)
  33. Cohen S, Gabel HW, Hemberg M, Hutchinson AN, Sadacca LA, Ebert DH _et al_ (2011). Genome-wide activity-dependent MeCP2 phosphorylation regulates nervous system development and function. _Neuron_ **72** : 72–85.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXht12kt7bJ)
  34. Compere SJ, Palmiter RD (1981). DNA methylation controls the inducibility of the mouse metallothionein-I gene lymphoid cells. _Cell_ **25** : 233–240.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaL3MXkvFCmurw%3D)
  35. Cortellino S, Xu J, Sannai M, Moore R, Caretti E, Cigliano A _et al_ (2011). Thymine DNA glycosylase is essential for active DNA demethylation by linked deamination-base excision repair. _Cell_ **146** : 67–79.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXoslCks74%3D)
  36. Coulondre C, Miller JH, Farabaugh PJ, Gilbert W (1978). Molecular basis of base substitution hotspots in Escherichia coli. _Nature_ **274** : 775–780.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaE1MXpvFylsA%3D%3D)
  37. Creusot F, Acs G, Christman JK (1982). Inhibition of DNA methyltransferase and induction of Friend erythroleukemia cell differentiation by 5-azacytidine and 5-aza-2′-deoxycytidine. _J Biol Chem_ **257** : 2041–2048.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaL38XhtlGntLw%3D)
  38. D’Alessio AC, Weaver IC, Szyf M (2007). Acetylation-induced transcription is required for active DNA demethylation in methylation-silenced genes. _Mol Cell Biol_ **27** : 7462–7474. **An example of coordinated actions from both histone modification and DNA demethylation in regulating gene regulation**.
  39. Daniel JM, Spring CM, Crawford HC, Reynolds AB, Baig A (2002). The p120(ctn)-binding partner Kaiso is a bi-modal DNA-binding protein that recognizes both a sequence-specific consensus and methylated CpG dinucleotides. _Nucleic Acids Res_ **30** : 2911–2919.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD38XlsFCqsLs%3D)
  40. Dawlaty MM, Ganz K, Powell BE, Hu YC, Markoulaki S, Cheng AW _et al_ (2011). Tet1 is dispensable for maintaining pluripotency and its loss is compatible with embryonic and postnatal development. _Cell Stem Cell_ **9** : 166–175.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXpvFGit74%3D)
  41. Deng JV, Rodriguiz RM, Hutchinson AN, Kim IH, Wetsel WC, West AE (2010). MeCP2 in the nucleus accumbens contributes to neural and behavioral responses to psychostimulants. _Nat Neurosci_ **13** : 1128–1136.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXhtVais7jP)
  42. Devys D, Lutz Y, Rouyer N, Bellocq JP, Mandel JL (1993). The FMR-1 protein is cytoplasmic, most abundant in neurons and appears normal in carriers of a fragile X premutation. _Nat Genet_ **4** : 335–340.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK3sXlsFOlsL8%3D)
  43. Dhayalan A, Rajavelu A, Rathert P, Tamas R, Jurkowska RZ, Ragozin S _et al_ (2010). The Dnmt3a PWWP domain reads histone 3 lysine 36 trimethylation and guides DNA methylation. _J Biol Chem_ **285** : 26114–26120.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXhtVahtbvM)
  44. Eckhardt F, Lewin J, Cortese R, Rakyan VK, Attwood J, Burger M _et al_ (2006). DNA methylation profiling of human chromosomes 6, 20 and 22. _Nat Genet_ **38** : 1378–1385.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD28Xht1CntrbK)
  45. Ehrlich M, Gama-Sosa MA, Huang LH, Midgett RM, Kuo KC, McCune RA _et al_ (1982). Amount and distribution of 5-methylcytosine in human DNA from different types of tissues of cells. _Nucleic Acids Res_ **10** : 2709–2721.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaL38XitFCrur8%3D)
  46. Fan G, Beard C, Chen RZ, Csankovszki G, Sun Y, Siniaia M _et al_ (2001). DNA hypomethylation perturbs the function and survival of CNS neurons in postnatal animals. _J Neurosci_ **21** : 788–797. **One of the pioneering works in investigating the role of Dnmts in the CNS**.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD3MXhtF2qurg%3D)
  47. Fan G, Martinowich K, Chin MH, He F, Fouse SD, Hutnick L _et al_ (2005). DNA methylation controls the timing of astrogliogenesis through regulation of JAK-STAT signaling. _Development_ **132** : 3345–3356.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2MXpvFOju7c%3D)
  48. Fatemi SH, Folsom TD (2011). The role of fragile X mental retardation protein in major mental disorders. _Neuropharmacology_ **60** : 1221–1226.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXkvVKitbo%3D)
  49. Feng J, Chang H, Li E, Fan G (2005). Dynamic expression of de novo DNA methyltransferases Dnmt3a and Dnmt3b in the central nervous system. _J Neurosci Res_ **79** : 734–746.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2MXitFygsLg%3D)
  50. Feng J, Zhou Y, Campbell SL, Le T, Li E, Sweatt JD _et al_ (2010). Dnmt1 and Dnmt3a maintain DNA methylation and regulate synaptic function in adult forebrain neurons. _Nat Neurosci_ **13** : 423–430. **Demonstrated the importance of continual activity of Dnmts in postmitotic neurons and that the Dnmts exhibit overlapping roles of each other**.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXjtFynur8%3D)
  51. Ficz G, Branco MR, Seisenberger S, Santos F, Krueger F, Hore TA _et al_ (2011). Dynamic regulation of 5-hydroxymethylcytosine in mouse ES cells and during differentiation. _Nature_ **473** : 398–402.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXktFahs7s%3D)
  52. Filion GJ, Zhenilo S, Salozhin S, Yamada D, Prokhortchouk E, Defossez PA (2006). A family of human zinc finger proteins that bind methylated DNA and repress transcription. _Mol Cell Biol_ **26** : 169–181.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD28Xjslaqsbg%3D)
  53. Fouse SD, Shen Y, Pellegrini M, Cole S, Meissner A, Van Neste L _et al_ (2008). Promoter CpG methylation contributes to ES cell gene regulation in parallel with Oct4/Nanog, PcG complex, and histone H3 K4/K27 trimethylation. _Cell Stem Cell_ **2** : 160–169.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1cXitlygtb0%3D)
  54. Fuks F, Burgers WA, Brehm A, Hughes-Davies L, Kouzarides T (2000). DNA methyltransferase Dnmt1 associates with histone deacetylase activity. _Nat Genet_ **24** : 88–91. **Demonstrates physical connection between the two epigenetic mechanisms: DNA methylation and histone modification**.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD3cXks1eltg%3D%3D)
  55. Fuks F, Hurd PJ, Deplus R, Kouzarides T (2003). The DNA methyltransferases associate with HP1 and the SUV39H1 histone methyltransferase. _Nucleic Acids Res_ **31** : 2305–2312.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD3sXjt1Ogt7o%3D)
  56. Gardiner-Garden M, Frommer M (1987). CpG islands in vertebrate genomes. _J Mol Biol_ **196** : 261–282.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaL2sXlt1ygsrc%3D)
  57. Gaudet F, Rideout III WM, Meissner A, Dausman J, Leonhardt H, Jaenisch R (2004). Dnmt1 expression in pre- and postimplantation embryogenesis and the maintenance of IAP silencing. _Mol Cell Biol_ **24** : 1640–1648.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2cXhtFOjtrw%3D)
  58. Ge YZ, Pu MT, Gowher H, Wu HP, Ding JP, Jeltsch A _et al_ (2004). Chromatin targeting of de novo DNA methyltransferases by the PWWP domain. _J Biol Chem_ **279** : 25447–25454.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2cXksFygsLY%3D)
  59. Gebhard C, Benner C, Ehrich M, Schwarzfischer L, Schilling E, Klug M _et al_ (2010). General transcription factor binding at CpG islands in normal cells correlates with resistance to de novo DNA methylation in cancer cells. _Cancer Res_ **70** : 1398–1407.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXhvFaks7Y%3D)
  60. Geiman TM, Sankpal UT, Robertson AK, Zhao Y, Zhao Y, Robertson KD (2004). DNMT3B interacts with hSNF2H chromatin remodeling enzyme, HDACs 1 and 2, and components of the histone methylation system. _Biochem Biophys Res Commun_ **318** : 544–555.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2cXjs1CjsLs%3D)
  61. Ghosh S, Yates AJ, Frühwald MC, Miecznikowski JC, Plass C, Smiraglia D (2010). Tissue specific DNA methylation of CpG islands in normal human adult somatic tissues distinguishes neural from non-neural tissues. _Epigenetics_ **5** : 527–538.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXht1Clt7jI)
  62. Giacometti E, Luikenhuis S, Beard C, Jaenisch R (2007). Partial rescue of MeCP2 deficiency by postnatal activation of MeCP2. _Proc Natl Acad Sci USA_ **104** : 1931–1936.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2sXitVejs7Y%3D)
  63. Globisch D, Munzel M, Muller M, Michalakis S, Wagner M, Koch S _et al_ (2010). Tissue distribution of 5-hydroxymethylcytosine and search for active demethylation intermediates. _PLoS One_ **5** : e15367.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXjvVSluw%3D%3D)
  64. Golshani P, Hutnick L, Schweizer F, Fan G (2005). Conditional Dnmt1 deletion in dorsal forebrain disrupts development of somatosensory barrel cortex and thalamocortical long-term potentiation. _Thalamus Relat Syst_ **3** : 227–233.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2sXmtl2gsr0%3D)
  65. Goto K, Numata M, Komura JI, Ono T, Bestor TH, Kondo H (1994). Expression of DNA methyltransferase gene in mature and immature neurons as well as proliferating cells in mice. _Differentiation_ **56** : 39–44. **Identified DNA methyltransferase mRNA in postmitotic neurons and suggested that DNA methylation served a unique function in neurons**.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK2cXksFCnu70%3D)
  66. Guo J, Ma DK, Mo H, Ball MP, Jang MH, Bonaguidi MA _et al_ (2011a). Neuronal activity modifies the DNA methylation landscape in the adult brain. _Nat Neurosci_ **14** : 1345–1351. **Investigated the extent to which the DNA methylation pattern is modifiable by neural activity** **_in vivo_**.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXhtV2gt7jJ)
  67. Guo J, Su Y, Zhong C, Ming GL, Song H (2011b). Hydroxylation of 5-methylcytosine by TET1 promotes active DNA demethylation in the adult brain. _Cell_ **145** : 423–434. **Also discovered few gene regions, including BDNF, that exhibit demethylation and remethylation cycle in postmitotic neurons**.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXlsFShsrs%3D)
  68. Guy J, Gan J, Selfridge J, Cobb S, Bird A (2007). Reversal of neurological defects in a mouse model of Rett syndrome. _Science_ **315** : 1143–1147.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2sXhvFWrs7s%3D)
  69. Guy J, Hendrich B, Holmes M, Martin JE, Bird A (2001). A mouse Mecp2-null mutation causes neurological symptoms that mimic Rett syndrome. _Nat Genet_ **27** : 322–326.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD3MXhslOisLc%3D)
  70. Gwynn B, Lueders K, Sands MS, Birkenmeier EH (1998). Intracisternal A-particle element transposition into the murine beta-glucuronidase gene correlates with loss of enzyme activity: a new model for beta-glucuronidase deficiency in the C3H mouse. _Mol Cell Biol_ **18** : 6474–6481.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK1cXmvF2nsr4%3D)
  71. Han L, Witmer PD, Casey E, Valle D, Sukumar S (2007). DNA methylation regulates microRNA expression. _Cancer Biol Ther_ **6** : 1284–1288.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1cXkvVahtw%3D%3D)
  72. Hashimoto H, Horton JR, Zhang X, Bostick M, Jacobsen SE, Cheng X (2008). The SRA domain of UHRF1 flips 5-methylcytosine out of the DNA helix. _Nature_ **455** : 826–829.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1cXht1WktLnE)
  73. Hashimoto H, Horton JR, Zhang X, Cheng X (2009). UHRF1, a modular multi-domain protein, regulates replication-coupled crosstalk between DNA methylation and histone modifications. _Epigenetics_ **4** : 8–14.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1MXovF2jt70%3D)
  74. Hata K, Okano M, Lei H, Li E (2002). Dnmt3L cooperates with the Dnmt3 family of de novo DNA methyltransferases to establish maternal imprints in mice. _Development_ **129** : 1983–1993.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD38XjslSls70%3D)
  75. He YF, Li BZ, Li Z, Liu P, Wang Y, Tang Q _et al_ (2011). Tet-mediated formation of 5-carboxylcytosine and its excision by TDG in mammalian DNA. _Science_ **333** : 1303–1307.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXhtV2jt7nO)
  76. Hellman A, Chess A (2007). Gene body-specific methylation on the active X chromosome. _Science_ **315** : 1141–1143.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2sXhvFWrs7o%3D)
  77. Hendrich B, Bird A (1998). Identification and characterization of a family of mammalian methyl-CpG binding proteins. _Mol Cell Biol_ **18** : 6538–6547.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK1cXmvF2nsrg%3D)
  78. Hendrich B, Hardeland U, Ng HH, Jiricny J, Bird A (1999). The thymine glycosylase MBD4 can bind to the product of deamination at methylated CpG sites. _Nature_ **401** : 301–304.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK1MXmt1yrsrs%3D)
  79. Hermann A, Goyal R, Jeltsch A (2004). The Dnmt1 DNA-(cytosine-C5)-methyltransferase methylates DNA processively with high preference for hemimethylated target sites. _J Biol Chem_ **279** : 48350–48359.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2cXptl2nu78%3D)
  80. Holliday R, Pugh JE (1975). DNA modification mechanisms and gene activity during development. _Science_ **187** : 226–232.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaE2MXht1Sitrw%3D)
  81. Hotchkiss RD (1948). The quantitative separation of purines, pyrimidines, and nucleosides by paper chromatography. _J Biol Chem_ **175** : 315–332. **Discovered 5-methylcytosine in the mammalian genome**.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaH1cXkt1Sqtg%3D%3D)
  82. Huang Y, Pastor WA, Shen Y, Tahiliani M, Liu DR, Rao A (2010). The behaviour of 5-hydroxymethylcytosine in bisulfite sequencing. _PLoS One_ **5** : e8888.
  83. Hutchinson AN, Deng JV, Aryal DK, Wetsel WC, West AE (2012). Differential regulation of MeCP2 phosphorylation in the CNS by dopamine and serotonin. _Neuropsychopharmacology_ **37** : 321–337.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXhs1eju73F)
  84. Hutnick LK, Golshani P, Namihira M, Xue Z, Matynia A, Yang XW _et al_ (2009). DNA hypomethylation restricted to the murine forebrain induces cortical degeneration and impairs postnatal neuronal maturation. _Hum Mol Genet_ **18** : 2875–2888.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1MXosVans7g%3D)
  85. Hutnick LK, Huang X, Loo TC, Ma Z, Fan G (2010). Repression of retrotransposal elements in mouse embryonic stem cells is primarily mediated by a DNA methylation-independent mechanism. _J Biol Chem_ **285** : 21082–21091.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXnvFSiu7o%3D)
  86. Illingworth RS, Gruenewald-Schneider U, Webb S, Kerr AR, James KD, Turner DJ _et al_ (2010). Orphan CpG islands identify numerous conserved promoters in the mammalian genome. _PLoS Genet_ **6** : e1001134.
  87. Im HI, Hollander JA, Bali P, Kenny PJ (2010). MeCP2 controls BDNF expression and cocaine intake through homeostatic interactions with microRNA-212. _Nat Neurosci_ **13** : 1120–1127.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXhtVaisLzJ)
  88. Inano K, Suetake I, Ueda T, Miyake Y, Nakamura M, Okada M _et al_ (2000). Maintenance-type DNA methyltransferase is highly expressed in post-mitotic neurons and localized in the cytoplasmic compartment. _J Biochem_ **128** : 315–321.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD3cXlvVeksbs%3D)
  89. Irizarry RA, Ladd-Acosta C, Wen B, Wu Z, Montano C, Onyango P _et al_ (2009). The human colon cancer methylome shows similar hypo- and hypermethylation at conserved tissue-specific CpG island shores. _Nat Genet_ **41** : 178–186.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1MXntFGrtA%3D%3D)
  90. Ito S, D’Alessio AC, Taranova OV, Hong K, Sowers LC, Zhang Y (2010). Role of Tet proteins in 5mC to 5hmC conversion, ES-cell self-renewal and inner cell mass specification. _Nature_ **466** : 1129–1133.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXovFCntro%3D)
  91. Ito S, Shen L, Dai Q, Wu SC, Collins LB, Swenberg JA _et al_ (2011). Tet proteins can convert 5-methylcytosine to 5-formylcytosine and 5-carboxylcytosine. _Science_ **333** : 1300–1303.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXhtV2jt7nN)
  92. Jackson-Grusby L, Beard C, Possemato R, Tudor M, Fambrough D, Csankovszki G _et al_ (2001). Loss of genomic methylation causes p53-dependent apoptosis and epigenetic deregulation. _Nat Genet_ **27** : 31–39.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD3MXis1yksw%3D%3D)
  93. Jia D, Jurkowska RZ, Zhang X, Jeltsch A, Cheng X (2007). Structure of Dnmt3a bound to Dnmt3L suggests a model for de novo DNA methylation. _Nature_ **449** : 248–251.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2sXhtVagtbrF)
  94. Jin SG, Wu X, Li AX, Pfeifer GP (2011). Genomic mapping of 5-hydroxymethylcytosine in the human brain. _Nucleic Acids Res_ **39** : 5015–5024.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXoslOhu7g%3D)
  95. Jones PL, Veenstra GJ, Wade PA, Vermaak D, Kass SU, Landsberger N _et al_ (1998). Methylated DNA and MeCP2 recruit histone deacetylase to repress transcription. _Nat Genet_ **19** : 187–191.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK1cXls1Cqsbg%3D)
  96. Jüttermann R, Li E, Jaenisch R (1994). Toxicity of 5-aza-2′-deoxycytidine to mammalian cells is mediated primarily by covalent trapping of DNA methyltransferase rather than DNA demethylation. _Proc Natl Acad Sci USA_ **91** : 11797–11801.
  97. Kaneda M, Okano M, Hata K, Sado T, Tsujimoto N, Li E _et al_ (2004). Essential role for de novo DNA methyltransferase Dnmt3a in paternal and maternal imprinting. _Nature_ **429** : 900–903.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2cXltVKltL8%3D)
  98. Kantor B, Kaufman Y, Makedonski K, Razin A, Shemer R (2004). Establishing the epigenetic status of the Prader-Willi/Angelman imprinting center in the gametes and embryo. _Hum Mol Genet_ **13** : 2767–2779.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2cXhtVSkt7jF)
  99. Karagianni P, Amazit L, Qin J, Wong J (2008). ICBP90, a novel methyl K9 H3 binding protein linking protein ubiquitination with heterochromatin formation. _Mol Cell Biol_ **28** : 705–717.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1cXkslGmug%3D%3D)
  100. Kimura H, Shiota K (2003). Methyl-CpG-binding protein, MeCP2, is a target molecule for maintenance DNA methyltransferase, Dnmt1. _J Biol Chem_ **278** : 4806–4812.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD3sXhtVKlsL4%3D)
  101. Kishi N, Macklis JD (2004). MECP2 is progressively expressed in post-migratory neurons and is involved in neuronal maturation rather than cell fate decisions. _Mol Cell Neurosci_ **27** : 306–321.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2cXpt1Wjs70%3D)
  102. Klein CJ, Botuyan MV, Wu Y, Ward CJ, Nicholson GA, Hammans S _et al_ (2011). Mutations in DNMT1 cause hereditary sensory neuropathy with dementia and hearing loss. _Nat Genet_ **43** : 595–600.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXlsVGlurs%3D)
  103. Kovacheva VP, Mellott TJ, Davison JM, Wagner N, Lopez-Coviella I, Schnitzler AC _et al_ (2007). Gestational choline deficiency causes global and Igf2 gene DNA hypermethylation by up-regulation of Dnmt1 expression. _J Biol Chem_ **282** : 31777–31788.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2sXhtFyku77L)
  104. Kriaucionis S, Heintz N (2009). The nuclear DNA base 5-hydroxymethylcytosine is present in Purkinje neurons and the brain. _Science_ **324** : 929–930. **This is first paper describing that hydroxymethylcytosine is enriched in postmitotic CNS neurons**.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1MXlslWnurk%3D)
  105. Kuster JE, Guarnieri MH, Ault JG, Flaherty L, Swiatek PJ (1997). IAP insertion in the murine LamB3 gene results in junctional epidermolysis bullosa. _Mamm Genome_ **8** : 673–681.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK2sXls1Gqsbw%3D)
  106. La Salle S, Oakes CC, Neaga OR, Bourc’his D, Bestor TH, Trasler JM (2007). Loss of spermatogonia and wide-spread DNA methylation defects in newborn male mice deficient in DNMT3L. _BMC Dev Biol_ **7** : 104.
  107. Ladd-Acosta C, Pevsner J, Sabunciyan S, Yolken RH, Webster MJ, Dinkins T _et al_ (2007). DNA methylation signatures within the human brain. _Am J Hum Genet_ **81** : 1304–1315.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2sXhtl2ju7%2FI)
  108. LaPlant Q, Vialou V, Covington III HE, Dumitriu D, Feng J, Warren BL _et al_ (2010). Dnmt3a regulates emotional behavior and spine plasticity in the nucleus accumbens. _Nat Neurosci_ **13** : 1137–1143.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXhtVGgtLfJ)
  109. LaSalle JM, Goldstine J, Balmer D, Greco CM (2001). Quantitative localization of heterogeneous methyl-CpG-binding protein 2 (MeCP2) expression phenotypes in normal and Rett syndrome brain by laser scanning cytometry. _Hum Mol Genet_ **10** : 1729–1740.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD3MXntFCitLo%3D)
  110. Le T, Kim KP, Fan G, Faull KF (2011). A sensitive mass spectrometry method for simultaneous quantification of DNA methylation and hydroxymethylation levels in biological samples. _Anal Biochem_ **412** : 203–209.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXkt1yqurg%3D)
  111. Lee JH, Skalnik DG (2005). CpG-binding protein (CXXC finger protein 1) is a component of the mammalian Set1 histone H3-Lys4 methyltransferase complex, the analogue of the yeast Set1/COMPASS complex. _J Biol Chem_ **280** : 41725–41731.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2MXhtlShsbbM)
  112. Lee MS, Jun DH, Hwang CI, Park SS, Kang JJ, Park HS _et al_ (2006). Selection of neural differentiation-specific genes by comparing profiles of random differentiation. _Stem Cells_ **24** : 1946–1955.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD28XhtFKiu77J)
  113. Leonhardt H, Page AW, Weier HU, Bestor TH (1992). A targeting sequence directs DNA methyltransferase to sites of DNA replication in mammalian nuclei. _Cell_ **71** : 865–873.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK3sXjtl2hsg%3D%3D)
  114. Levenson JM, Roth TL, Lubin FD, Miller CA, Huang IC, Desai P _et al_ (2006). Evidence that DNA (cytosine-5) methyltransferase regulates synaptic plasticity in the hippocampus. _J Biol Chem_ **281** : 15763–15773.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD28Xlt1Ckt7c%3D)
  115. Lewis JD, Meehan RR, Henzel WJ, Maurer-Fogy I, Jeppesen P, Klein F _et al_ (1992). Purification, sequence, and cellular localization of a novel chromosomal protein that binds to methylated DNA. _Cell_ **69** : 905–914.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK3sXitVOrtrg%3D)
  116. Li BZ, Huang Z, Cui QY, Song XH, Du L, Jeltsch A _et al_ (2011a). Histone tails regulate DNA methylation by allosterically activating de novo methyltransferase. _Cell Res_ **21** : 1172–1181.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXps1yrur4%3D)
  117. Li E, Bestor TH, Jaenisch R (1992). Targeted mutation of the DNA methyltransferase gene results in embryonic lethality. _Cell_ **69** : 915–926. **Developed the Dnmt1 knockout mice**.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK38XksVGgsr0%3D)
  118. Li H, Zhong X, Chau KF, Williams EC, Chang Q (2011b). Loss of activity-induced phosphorylation of MeCP2 enhances synaptogenesis, LTP and spatial memory. _Nat Neurosci_ **14** : 1001–1008.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXovFyhu7s%3D)
  119. Lienert F, Wirbelauer C, Som I, Dean A, Mohn F, Schubeler D (2011). Identification of genetic elements that autonomously determine DNA methylation states. _Nat Genet_ **43** : 1091–1097. **Demonstrated that transcription factor binding regulates DNA methylation by inserting 50 different DNA elements into the same chromosome region**.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXht1Crt7zE)
  120. Lister R, Pelizzola M, Dowen RH, Hawkins RD, Hon G, Tonti-Filippini J _et al_ (2009). Human DNA methylomes at base resolution show widespread epigenomic differences. _Nature_ **462** : 315–322.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1MXht1Kiu73E)
  121. Liutkeviciute Z, Lukinavicius G, Masevicius V, Daujotyte D, Klimasauskas S (2009). Cytosine-5-methyltransferases add aldehydes to DNA. _Nat Chem Biol_ **5** : 400–402.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1MXls1Cmtro%3D)
  122. Lopes EC, Valls E, Figueroa ME, Mazur A, Meng FG, Chiosis G _et al_ (2008). Kaiso contributes to DNA methylation-dependent silencing of tumor suppressor genes in colon cancer cell lines. _Cancer Res_ **68** : 7258–7263.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1cXhtFWltLbO)
  123. Lubin FD, Roth TL, Sweatt JD (2008). Epigenetic regulation of BDNF gene transcription in the consolidation of fear memory. _J Neurosci_ **28** : 10576–10586.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1cXht1ymt7vI)
  124. Luikenhuis S, Giacometti E, Beard CF, Jaenisch R (2004). Expression of MeCP2 in postmitotic neurons rescues Rett syndrome in mice. _Proc Natl Acad Sci USA_ **101** : 6033–6038.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2cXjsFKnsbs%3D)
  125. Lujambio A, Calin GA, Villanueva A, Ropero S, Sanchez-Cespedes M, Blanco D _et al_ (2008). A microRNA DNA methylation signature for human cancer metastasis. _Proc Natl Acad Sci USA_ **105** : 13556–13561.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1cXhtFeqs7bF)
  126. Ma DK, Jang MH, Guo JU, Kitabatake Y, Chang ML, Pow-Anpongkul N _et al_ (2009). Neuronal activity-induced Gadd45b promotes epigenetic DNA demethylation and adult neurogenesis. _Science_ **323** : 1074–1077.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1MXitVyntrg%3D)
  127. Macleod D, Charlton J, Mullins J, Bird AP (1994). Sp1 sites in the mouse aprt gene promoter are required to prevent methylation of the CpG island. _Genes Dev_ **8** : 2282–2292.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK2MXhtF2gsbs%3D)
  128. Martin Caballero I, Hansen J, Leaford D, Pollard S, Hendrich BD (2009). The methyl-CpG binding proteins Mecp2, Mbd2 and Kaiso are dispensable for mouse embryogenesis, but play a redundant function in neural differentiation. _PLoS One_ **4** : e4315.
  129. Martinowich K, Hattori D, Wu H, Fouse S, He F, Hu Y _et al_ (2003). DNA methylation-related chromatin remodeling in activity-dependent BDNF gene regulation. _Science_ **302** : 890–893. **Demonstrated that methylation and MeCP2 binding to the BDNF promoter are regulated by neuronal activity**.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD3sXos1Cmt74%3D)
  130. Maunakea AK, Nagarajan RP, Bilenky M, Ballinger TJ, D’Souza C, Fouse SD _et al_ (2010). Conserved role of intragenic DNA methylation in regulating alternative promoters. _Nature_ **466** : 253–257.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXosVKiu70%3D)
  131. Mayer W, Niveleau A, Walter J, Fundele R, Haaf T (2000). Demethylation of the zygotic paternal genome. _Nature_ **403** : 501–502.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD3cXht1ShsbY%3D)
  132. McCarty M, Avery OT (1946). Studies on the chemical nature of the substance inducing transformation of pneumococcal types: II. Effect of desoxyribonuclease on the biological activity of the transforming substance. _J Exp Med_ **83** : 89–96.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaH28XhtFGrtg%3D%3D)
  133. McGowan PO, Sasaki A, D’Alessio AC, Dymov S, Labonte B, Szyf M _et al_ (2009). Epigenetic regulation of the glucocorticoid receptor in human brain associates with childhood abuse. _Nat Neurosci_ **12** : 342–348. **Observed childhood abuse in suicidal victims associated with higher DNA methylation level in the glucocorticoid receptor promoter in the hippocampus**.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1MXitFKlurc%3D)
  134. Meehan RR, Lewis JD, McKay S, Kleiner EL, Bird AP (1989). Identification of a mammalian protein that binds specifically to DNA containing methylated CpGs. _Cell_ **58** : 499–507.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaL1MXlsVOrsrs%3D)
  135. Meissner A, Mikkelsen TS, Gu H, Wernig M, Hanna J, Sivachenko A _et al_ (2008). Genome-scale DNA methylation maps of pluripotent and differentiated cells. _Nature_ **454** : 766–770.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1cXps1Sms7w%3D)
  136. Michaud EJ, van Vugt MJ, Bultman SJ, Sweet HO, Davisson MT, Woychik RP (1994). Differential expression of a new dominant agouti allele (Aiapy) is correlated with methylation state and is influenced by parental lineage. _Genes Dev_ **8** : 1463–1472.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK2cXlsFSgtL4%3D)
  137. Mikkelsen TS, Ku M, Jaffe DB, Issac B, Lieberman E, Giannoukos G _et al_ (2007). Genome-wide maps of chromatin state in pluripotent and lineage-committed cells. _Nature_ **448** : 553–560.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2sXosVSrtrc%3D)
  138. Mill J, Tang T, Kaminsky Z, Khare T, Yazdanpanah S, Bouchard L _et al_ (2008). Epigenomic profiling reveals DNA-methylation changes associated with major psychosis. _Am J Hum Genet_ **82** : 696–711.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1cXksVGrtbY%3D)
  139. Millar CB, Guy J, Sansom OJ, Selfridge J, MacDougall E, Hendrich B _et al_ (2002). Enhanced CpG mutability and tumorigenesis in MBD4-deficient mice. _Science_ **297** : 403–405.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD38Xls1els78%3D)
  140. Miller CA, Gavin CF, White JA, Parrish RR, Honasoge A, Yancey CR _et al_ (2010). Cortical DNA methylation maintains remote memory. _Nat Neurosci_ **13** : 664–666.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXmsVWmsrs%3D)
  141. Miller CA, Sweatt JD (2007). Covalent modification of DNA regulates memory formation. _Neuron_ **53** : 857–869.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2sXktFCmtLg%3D)
  142. Mohn F, Weber M, Rebhan M, Roloff TC, Richter J, Stadler MB _et al_ (2008). Lineage-specific polycomb targets and de novo DNA methylation define restriction and potential of neuronal progenitors. _Mol Cell_ **30** : 755–766.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1cXnvFWnt7g%3D)
  143. Moretti P, Levenson JM, Battaglia F, Atkinson R, Teague R, Antalffy B _et al_ (2006). Learning and memory and synaptic plasticity are impaired in a mouse model of Rett syndrome. _J Neurosci_ **26** : 319–327.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD28XmtV2ltw%3D%3D)
  144. Morris KV, Chan SW, Jacobsen SE, Looney DJ (2004). Small interfering RNA-induced transcriptional gene silencing in human cells. _Science_ **305** : 1289–1292.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2cXmvFCit78%3D)
  145. Mortusewicz O, Schermelleh L, Walter J, Cardoso MC, Leonhardt H (2005). Recruitment of DNA methyltransferase I to DNA repair sites. _Proc Natl Acad Sci USA_ **102** : 8905–8909.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2MXlvF2qurY%3D)
  146. Muramatsu M, Kinoshita K, Fagarasan S, Yamada S, Shinkai Y, Honjo T (2000). Class switch recombination and hypermutation require activation-induced cytidine deaminase (AID), a potential RNA editing enzyme. _Cell_ **102** : 553–563.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD3cXmsFWit7g%3D)
  147. Muto M, Kanari Y, Kubo E, Takabe T, Kurihara T, Fujimori A _et al_ (2002). Targeted disruption of Np95 gene renders murine embryonic stem cells hypersensitive to DNA damaging agents and DNA replication blocks. _J Biol Chem_ **277** : 34549–34555.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD38Xnt1Wisr0%3D)
  148. Nan X, Meehan RR, Bird A (1993). Dissection of the methyl-CpG binding domain from the chromosomal protein MeCP2. _Nucleic Acids Res_ **21** : 4886–4892.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK2cXjtFar)
  149. Nan X, Ng HH, Johnson CA, Laherty CD, Turner BM, Eisenman RN _et al_ (1998). Transcriptional repression by the methyl-CpG-binding protein MeCP2 involves a histone deacetylase complex. _Nature_ **393** : 386–389.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK1cXjsFKgurc%3D)
  150. Nelson ED, Kavalali ET, Monteggia LM (2006). MeCP2-dependent transcriptional repression regulates excitatory neurotransmission. _Curr Biol_ **16** : 710–716.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD28XjtFGiurg%3D)
  151. Ng HH, Zhang Y, Hendrich B, Johnson CA, Turner BM, Erdjument-Bromage H _et al_ (1999). MBD2 is a transcriptional repressor belonging to the MeCP1 histone deacetylase complex. _Nat Genet_ **23** : 58–61.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK1MXlvFWhsb8%3D)
  152. Nguyen S, Meletis K, Fu D, Jhaveri S, Jaenisch R (2007). Ablation of de novo DNA methyltransferase Dnmt3a in the nervous system leads to neuromuscular defects and shortened lifespan. _Dev Dyn_ **236** : 1663–1676. **Demonstrate that knockout of Dnmt3a in neural tissue during development does not affect development of cortical neurons but results in hypomethylation of the Gfap promoter and improper motor neuron development**.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2sXnvFOht7c%3D)
  153. Okano M, Bell DW, Haber DA, Li E (1999). DNA methyltransferases Dnmt3a and Dnmt3b are essential for de novo methylation and mammalian development. _Cell_ **99** : 247–257. **Demonstrated the function of Dnmt3 enzymes by studying both Dnmt3a and Dnmt3b knockout mice**.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK1MXnt1Gqsrc%3D)
  154. Ooi SK, Qiu C, Bernstein E, Li K, Jia D, Yang Z _et al_ (2007). DNMT3L connects unmethylated lysine 4 of histone H3 to de novo methylation of DNA. _Nature_ **448** : 714–717.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2sXos1yksL8%3D)
  155. Oswald J, Engemann S, Lane N, Mayer W, Olek A, Fundele R _et al_ (2000). Active demethylation of the paternal genome in the mouse zygote. _Curr Biol_ **10** : 475–478.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD3cXislyltLo%3D)
  156. Paroush Z, Keshet I, Yisraeli J, Cedar H (1990). Dynamics of demethylation and activation of the alpha-actin gene in myoblasts. _Cell_ **63** : 1229–1237.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK3MXls1Omsw%3D%3D)
  157. Pastor WA, Pape UJ, Huang Y, Henderson HR, Lister R, Ko M _et al_ (2011). Genome-wide mapping of 5-hydroxymethylcytosine in embryonic stem cells. _Nature_ **473** : 394–397.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXlvVahu7c%3D)
  158. Petronzelli F, Riccio A, Markham GD, Seeholzer SH, Stoerker J, Genuardi M _et al_ (2000). Biphasic kinetics of the human DNA repair protein MED1 (MBD4), a mismatch-specific DNA N-glycosylase. _J Biol Chem_ **275** : 32422–32429.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD3cXnslOisbs%3D)
  159. Popp C, Dean W, Feng S, Cokus SJ, Andrews S, Pellegrini M _et al_ (2010). Genome-wide erasure of DNA methylation in mouse primordial germ cells is affected by AID deficiency. _Nature_ **463** : 1101–1105.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXitl2gs7Y%3D)
  160. Pradhan S, Bacolla A, Wells RD, Roberts RJ (1999). Recombinant human DNA (cytosine-5) methyltransferase. I. Expression, purification, and comparison of de novo and maintenance methylation. _J Biol Chem_ **274** : 33002–33010.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK1MXntlygurg%3D)
  161. Prokhortchouk A, Hendrich B, Jorgensen H, Ruzov A, Wilm M, Georgiev G _et al_ (2001). The p120 catenin partner Kaiso is a DNA methylation-dependent transcriptional repressor. _Genes Dev_ **15** : 1613–1618.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD3MXltFKqt70%3D)
  162. Qian X, Shen Q, Goderie SK, He W, Capela A, Davis AA _et al_ (2000). Timing of CNS cell generation: a programmed sequence of neuron and glial cell production from isolated murine cortical stem cells. _Neuron_ **28** : 69–80.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD3cXnvVGlt7o%3D)
  163. Rai K, Huggins IJ, James SR, Karpf AR, Jones DA, Cairns BR (2008). DNA demethylation in zebrafish involves the coupling of a deaminase, a glycosylase, and gadd45. _Cell_ **135** : 1201–1212.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1MXisFyjtw%3D%3D)
  164. Rakyan VK, Hildmann T, Novik KL, Lewin J, Tost J, Cox AV _et al_ (2004). DNA methylation profiling of the human major histocompatibility complex: a pilot study for the human epigenome project. _PLoS Biol_ **2** : e405.
  165. Ramirez-Carrozzi VR, Braas D, Bhatt DM, Cheng CS, Hong C, Doty KR _et al_ (2009). A unifying model for the selective regulation of inducible transcription by CpG islands and nucleosome remodeling. _Cell_ **138** : 114–128.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1MXps1ygurg%3D)
  166. Ramsahoye BH, Biniszkiewicz D, Lyko F, Clark V, Bird AP, Jaenisch R (2000). Non-CpG methylation is prevalent in embryonic stem cells and may be mediated by DNA methyltransferase 3a. _Proc Natl Acad Sci USA_ **97** : 5237–5242. **First study to demonstrate the presence of non-CG methylation in mammalian cells**.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD3cXjsVWntLs%3D)
  167. Samaco RC, Neul JL (2011). Complexities of Rett syndrome and MeCP2. _J Neurosci_ **31** : 7951–7959.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXnsFykt7g%3D)
  168. Sarraf SA, Stancheva I (2004). Methyl-CpG binding protein MBD1 couples histone H3 methylation at lysine 9 by SETDB1 to DNA replication and chromatin assembly. _Mol Cell_ **15** : 595–605.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2cXns1ynu70%3D)
  169. Sasai N, Nakao M, Defossez PA (2010). Sequence-specific recognition of methylated DNA by human zinc-finger proteins. _Nucleic Acids Res_ **38** : 5015–5022.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXhtV2qsLnE)
  170. Sauvageot CM, Stiles CD (2002). Molecular mechanisms controlling cortical gliogenesis. _Curr Opin Neurobiol_ **12** : 244–249.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD38XktFygsLo%3D)
  171. Saxonov S, Berg P, Brutlag DL (2006). A genome-wide analysis of CpG dinucleotides in the human genome distinguishes two distinct classes of promoters. _Proc Natl Acad Sci USA_ **103** : 1412–1417.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD28Xhs1Gnsrs%3D)
  172. Schulz WA, Steinhoff C, Florl AR (2006). Methylation of endogenous human retroelements in health and disease. _Curr Top Microbiol Immunol_ **310** : 211–250.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD28XpslGqsro%3D)
  173. Shahbazian MD, Antalffy B, Armstrong DL, Zoghbi HY (2002). Insight into Rett syndrome: MeCP2 levels display tissue- and cell-specific differences and correlate with neuronal maturation. _Hum Mol Genet_ **11** : 115–124.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD38XhtlKmsrY%3D)
  174. Sharif J, Muto M, Takebayashi S, Suetake I, Iwamatsu A, Endo TA _et al_ (2007). The SRA protein Np95 mediates epigenetic inheritance by recruiting Dnmt1 to methylated DNA. _Nature_ **450** : 908–912.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2sXhtl2gsb3L)
  175. Shen L, Kondo Y, Guo Y, Zhang J, Zhang L, Ahmed S _et al_ (2007). Genome-wide profiling of DNA methylation reveals a class of normally methylated CpG island promoters. _PLoS Genet_ **3** : 2023–2036.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2sXhsVOnsr%2FK)
  176. Sinkkonen L, Hugenschmidt T, Berninger P, Gaidatzis D, Mohn F, Artus-Revel CG _et al_ (2008). MicroRNAs control de novo DNA methylation through regulation of transcriptional repressors in mouse embryonic stem cells. _Nat Struct Mol Biol_ **15** : 259–267.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1cXislKmtbk%3D)
  177. Straussman R, Nejman D, Roberts D, Steinfeld I, Blum B, Benvenisty N _et al_ (2009). Developmental programming of CpG island methylation profiles in the human genome. _Nat Struct Mol Biol_ **16** : 564–571.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1MXks12kurs%3D)
  178. Stresemann C, Brueckner B, Musch T, Stopper H, Lyko F (2006). Functional diversity of DNA methyltransferase inhibitors in human cancer cell lines. _Cancer Res_ **66** : 2794–2800.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD28XhvV2gtrY%3D)
  179. Suetake I, Shinozaki F, Miyagawa J, Takeshima H, Tajima S (2004). DNMT3L stimulates the DNA methylation activity of Dnmt3a and Dnmt3b through a direct interaction. _J Biol Chem_ **279** : 27816–27823.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2cXkvFGmsr8%3D)
  180. Szwagierczak A, Bultmann S, Schmidt CS, Spada F, Leonhardt H (2010). Sensitive enzymatic quantification of 5-hydroxymethylcytosine in genomic DNA. _Nucleic Acids Res_ **38** : e181.
  181. Tahiliani M, Koh KP, Shen Y, Pastor WA, Bandukwala H, Brudno Y _et al_ (2009). Conversion of 5-methylcytosine to 5-hydroxymethylcytosine in mammalian DNA by MLL partner TET1. _Science_ **324** : 930–935. **The first study to demonstrate the role of Tet enzymes in conversion of 5-methylcytosine to 5-hydroxymethylcytosine, thus leading to recent studies of the DNA demethylation mechanism in the mammalian system**.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1MXlslWnurY%3D)
  182. Takizawa T, Nakashima K, Namihira M, Ochiai W, Uemura A, Yanagisawa M _et al_ (2001). DNA methylation is a critical cell-intrinsic determinant of astrocyte differentiation in the fetal brain. _Dev Cell_ **1** : 749–758.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD3MXptFOntbg%3D)
  183. Tao J, Hu K, Chang Q, Wu H, Sherman NE, Martinowich K _et al_ (2009). Phosphorylation of MeCP2 at Serine 80 regulates its chromatin association and neurological function. _Proc Natl Acad Sci USA_ **106** : 4882–4887.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1MXktFOjs7k%3D)
  184. Tazi J, Bird A (1990). Alternative chromatin structure at CpG islands. _Cell_ **60** : 909–920.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK3cXitlWmurw%3D)
  185. Teter B, Osterburg HH, Anderson CP, Finch CE (1994). Methylation of the rat glial fibrillary acidic protein gene shows tissue-specific domains. _J Neurosci Res_ **39** : 680–693.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK2MXislahtLc%3D)
  186. Teter B, Rozovsky I, Krohn K, Anderson C, Osterburg H, Finch C (1996). Methylation of the glial fibrillary acidic protein gene shows novel biphasic changes during brain development. _Glia_ **17** : 195–205.
[CAS](https://www.nature.com/articles/cas-redirect/1:STN:280:DyaK28vjslCrtw%3D%3D)
  187. Thomson JP, Skene PJ, Selfridge J, Clouaire T, Guy J, Webb S _et al_ (2010). CpG islands influence chromatin structure via the CpG-binding protein Cfp1. _Nature_ **464** : 1082–1086.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXkslartbc%3D)
  188. Turner G, Webb T, Wake S, Robinson H (1996). Prevalence of fragile X syndrome. _Am J Med Genet_ **64** : 196–197.
[CAS](https://www.nature.com/articles/cas-redirect/1:STN:280:DyaK28vitlyktw%3D%3D)
  189. Ukai H, Ishii-Oba H, Ukai-Tadenuma M, Ogiu T, Tsuji H (2003). Formation of an active form of the interleukin-2/15 receptor beta-chain by insertion of the intracisternal A particle in a radiation-induced mouse thymic lymphoma and its role in tumorigenesis. _Mol Carcinog_ **37** : 110–119.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD3sXksFSitr8%3D)
  190. Valinluck V, Tsai HH, Rogstad DK, Burdzy A, Bird A, Sowers LC (2004). Oxidative damage to methyl-CpG sequences inhibits the binding of the methyl-CpG binding domain (MBD) of methyl-CpG binding protein 2 (MeCP2). _Nucleic Acids Res_ **32** : 4100–4108.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2cXntFagu78%3D)
  191. Verkerk AJ, Pieretti M, Sutcliffe JS, Fu YH, Kuhl DP, Pizzuti A _et al_ (1991). Identification of a gene (FMR-1) containing a CGG repeat coincident with a breakpoint cluster region exhibiting length variation in fragile X syndrome. _Cell_ **65** : 905–914.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK38Xht1ahur4%3D)
  192. Walsh CP, Chaillet JR, Bestor TH (1998). Transcription of IAP endogenous retroviruses is constrained by cytosine methylation. _Nat Genet_ **20** : 116–117.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK1cXms1amsLg%3D)
  193. Weaver IC, Cervoni N, Champagne FA, D’Alessio AC, Sharma S, Seckl JR _et al_ (2004). Epigenetic programming by maternal behavior. _Nat Neurosci_ **7** : 847–854. **One of the first studies to demonstrate the association of methylation changes in the brain with neural behavior**.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2cXmtVamt7c%3D)
  194. Weber M, Hellmann I, Stadler MB, Ramos L, Paabo S, Rebhan M _et al_ (2007). Distribution, silencing potential and evolutionary impact of promoter DNA methylation in the human genome. _Nat Genet_ **39** : 457–466.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2sXjsV2hsrk%3D)
  195. Webster KE, O’Bryan MK, Fletcher S, Crewther PE, Aapola U, Craig J _et al_ (2005). Meiotic and epigenetic defects in Dnmt3L-knockout mouse spermatogenesis. _Proc Natl Acad Sci USA_ **102** : 4068–4073.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2MXis12jtr8%3D)
  196. Weiler IJ, Irwin SA, Klintsova AY, Spencer CM, Brazelton AD, Miyashiro K _et al_ (1997). Fragile X mental retardation protein is translated near synapses in response to neurotransmitter activation. _Proc Natl Acad Sci USA_ **94** : 5395–5400.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK2sXjtleis7k%3D)
  197. Wong E, Yang K, Kuraguchi M, Werling U, Avdievich E, Fan K _et al_ (2002). Mbd4 inactivation increases Cright-arrowT transition mutations and promotes gastrointestinal tumor formation. _Proc Natl Acad Sci USA_ **99** : 14937–14942.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD38Xpt1yrtLY%3D)
  198. Wu H, Zhang Y (2011). Tet1 and 5-hydroxymethylation: a genome-wide view in mouse embryonic stem cells. _Cell Cycle_ **10** : 2428–2436.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38XktlaqsQ%3D%3D)
  199. Wu M, Rinchik EM, Wilkinson E, Johnson DK (1997). Inherited somatic mosaicism caused by an intracisternal A particle insertion in the mouse tyrosinase gene. _Proc Natl Acad Sci USA_ **94** : 890–894.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK2sXhtVOksrY%3D)
  200. Wutz A, Smrzka OW, Schweifer N, Schellander K, Wagner EF, Barlow DP (1997). Imprinted expression of the Igf2r gene depends on an intronic CpG island. _Nature_ **389** : 745–749.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK2sXmvVSns7c%3D)
  201. Xie S, Wang Z, Okano M, Nogami M, Li Y, He WW _et al_ (1999). Cloning, expression and chromosome locations of the human DNMT3 gene family. _Gene_ **236** : 87–95.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK1MXls1Oru7k%3D)
  202. Xie W, Barr CL, Kim A, Yue F, Lee AY, Eubanks J _et al_ (2012). Base-resolution analyses of sequence and parent-of-origin dependent DNA methylation in the mouse genome. _Cell_ **148** : 816–831. **Recent article that identified significant levels of non-CpG DNA methylation in the murine frontal cortex**.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38XjtV2qsLg%3D)
  203. Yen RW, Vertino PM, Nelkin BD, Yu JJ, el-Deiry W, Cumaraswamy A _et al_ (1992). Isolation and characterization of the cDNA encoding human DNA methyltransferase. _Nucleic Acids Res_ **20** : 2287–2291.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK3sXit12jsrw%3D)
  204. Yoon HG, Chan DW, Reynolds AB, Qin J, Wong J (2003). N-CoR mediates DNA methylation-dependent repression through a methyl CpG binding protein Kaiso. _Mol Cell_ **12** : 723–734.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD3sXnvFGrt7o%3D)
  205. Zamudio NM, Scott HS, Wolski K, Lo CY, Law C, Leong D _et al_ (2011). DNMT3L is a regulator of X chromosome compaction and post-meiotic gene transcription. _PLoS One_ **6** : e18276.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXkvVeqtr4%3D)
  206. Zhang F, Pomerantz JH, Sen G, Palermo AT, Blau HM (2007). Active tissue-specific DNA demethylation conferred by somatic cell nuclei in stable heterokaryons. _Proc Natl Acad Sci USA_ **104** : 4395–4400.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2sXjsFGnsLk%3D)
  207. Zhang Y, Jurkowska R, Soeroes S, Rajavelu A, Dhayalan A, Bock I _et al_ (2010). Chromatin methylation activity of Dnmt3a and Dnmt3a/3L is guided by interaction of the ADD domain with the histone H3 tail. _Nucleic Acids Res_ **38** : 4246–4253.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXpsFyisL8%3D)
  208. Zhou L, Cheng X, Connolly BA, Dickman MJ, Hurd PJ, Hornby DP (2002). Zebularine: a novel DNA methylation inhibitor that forms a covalent complex with DNA methyltransferases. _J Mol Biol_ **321** : 591–599.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD38Xms1Wmtbs%3D)
  209. Zhou Z, Hong EJ, Cohen S, Zhao WN, Ho HY, Schmidt L _et al_ (2006). Brain-specific phosphorylation of MeCP2 regulates activity-dependent Bdnf transcription, dendritic growth, and spine maturation. _Neuron_ **52** : 255–269.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD28Xht1WhtbjM)
  210. Zwart R, Sleutels F, Wutz A, Schinkel AH, Barlow DP (2001). Bidirectional action of the Igf2r imprint control element on upstream and downstream imprinted genes. _Genes Dev_ **15** : 2361–2366.
[CAS](https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD3MXnsV2hs7g%3D)

Neuropsychopharmacology (_Neuropsychopharmacol._)  ISSN 1740-634X (online)  ISSN 0893-133X (print) 
## nature.com sitemap
### About Nature Portfolio
  * [About us](https://www.nature.com/npg_/company_info/index.html)
  * [Press releases](https://www.nature.com/npg_/press_room/press_releases.html)
  * [Press office](https://press.nature.com/)
  * [Contact us](https://support.nature.com/support/home)

### Discover content
  * [Journals A-Z](https://www.nature.com/siteindex)
  * [Articles by subject](https://www.nature.com/subjects)

### Publishing policies
  * [Nature portfolio policies](https://www.nature.com/authors/editorial_policies)
  * [Open access](https://www.nature.com/nature-research/open-access)

### Author & Researcher services
  * [Reprints & permissions](https://www.nature.com/reprints)
  * [Research data](https://www.springernature.com/gp/authors/research-data)
  * [Language editing](https://authorservices.springernature.com/language-editing/)
  * [Scientific editing](https://authorservices.springernature.com/scientific-editing/)
  * [Nature Masterclasses](https://masterclasses.nature.com/)
  * [Research Solutions](https://solutions.springernature.com/)

### Libraries & institutions
  * [Librarian service & tools](https://www.springernature.com/gp/librarians/tools-services)
  * [Librarian portal](https://www.springernature.com/gp/librarians/manage-your-account/librarianportal)
  * [Open research](https://www.nature.com/openresearch/about-open-access/information-for-institutions)
  * [Recommend to library](https://www.springernature.com/gp/librarians/recommend-to-your-library)

### Advertising & partnerships
  * [Advertising](https://partnerships.nature.com/product/digital-advertising/)
  * [Partnerships & Services](https://partnerships.nature.com/)
  * [Media kits](https://partnerships.nature.com/media-kits/)
  * [Branded content](https://partnerships.nature.com/product/branded-content-native-advertising/)

### Professional development
  * [Nature Careers](https://www.nature.com/naturecareers/)
  * [Nature Conferences](https://conferences.nature.com)

### Regional websites
  * [Nature Africa](https://www.nature.com/natafrica)
  * [Nature India](https://www.nature.com/nindia)
  * [Nature Italy](https://www.nature.com/natitaly)
  * [Nature Middle East](https://www.nature.com/nmiddleeast)

  * [Privacy Policy](https://www.nature.com/info/privacy)
  * [Use of cookies](https://www.nature.com/info/cookies)
  * Your privacy choices/Manage cookies 
  * [Legal notice](https://www.nature.com/info/legal-notice)
  * [Accessibility statement](https://www.nature.com/info/accessibility-statement)
  * [Terms & Conditions](https://www.nature.com/info/terms-and-conditions)
  * [Your US state privacy rights](https://www.springernature.com/ccpa)

[ ![Springer Nature](https://www.nature.com/static/images/logos/sn-logo-white-ea63208b81.svg) ](https://www.springernature.com/)
© 2025 Springer Nature Limited
  *[DOI]: Digital Object Identifier
  *[ISSN]: International Standard Serial Number