Typesetting math: 100%
[Skip to main content](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#screen-reader-main-content)[Skip to article](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#screen-reader-main-title)
[![Elsevier logo](https://sdfestaticassets-us-east-1.sciencedirectassets.com/shared-assets/24/images/elsevier-non-solus-new-grey.svg)ScienceDirect](https://www.sciencedirect.com/)
  * [Journals & Books](https://www.sciencedirect.com/browse/journals-and-books)

  * [](https://www.sciencedirect.com/search "Search")

[](https://www.sciencedirect.com/user/login?targetURL=%2Fscience%2Farticle%2Fpii%2FS0092867424004082&from=globalheader "My account")
  * [View **PDF**](https://www.sciencedirect.com/science/article/pii/S0092867424004082/pdfft?md5=6491378b815074cbaf82ac7f4ad419cf&pid=1-s2.0-S0092867424004082-main.pdf)
  * Download full issue

Search ScienceDirect
## Outline
  1. [Highlights](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#abs0020 "Highlights")
  2. [Summary](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#abs0010 "Summary")
  3. [Graphical abstract](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#abs0015 "Graphical abstract")
  4. **Beta****Powered by GenAI**[ Questions answered in this article](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#ai-components-toc-id "Questions answered in this article")
  5. [Keywords](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#kwrds0010 "Keywords")
  6. [Introduction](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#sec1 "Introduction")
  7. [Results](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#sec2 "Results")
  8. [Discussion](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#sec3 "Discussion")
  9. [STAR★Methods](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#sec4 "STAR★Methods")
  10. [Acknowledgments](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#ack0010 "Acknowledgments")
  11. [Supplemental information](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#app2 "Supplemental information")
  12. [References](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#cebib0010 "References")

Show full outline
## [Cited by (40)](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#section-cited-by)
## Figures (19)
  1. [![Unlabelled figure](https://ars.els-cdn.com/content/image/1-s2.0-S0092867424004082-fx1.sml)](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#undfig1)
  2. [![FigureS1. Characteristics of our updated scRNA-seq human bone marrow atlas, related to…](https://ars.els-cdn.com/content/image/1-s2.0-S0092867424004082-figs1.sml)](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#figs1)
  3. [![Figure1. A single-cell transcriptomic atlas of hematopoietic and non-hematopoietic cells…](https://ars.els-cdn.com/content/image/1-s2.0-S0092867424004082-gr1.sml)](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#fig1)
  4. [![FigureS2. Comparison of inflammatory cytokine RNA expression and signature scores across…](https://ars.els-cdn.com/content/image/1-s2.0-S0092867424004082-figs2.sml)](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#figs2)
  5. [![Figure2. Defining the non-hematopoietic cellular composition of human bone marrow](https://ars.els-cdn.com/content/image/1-s2.0-S0092867424004082-gr2.sml)](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#fig2)
  6. [![FigureS3. Non-hematopoietic cell subsets quality control, selected expression profiles,…](https://ars.els-cdn.com/content/image/1-s2.0-S0092867424004082-figs3.sml)](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#figs3)

Show 13 more figures
## Tables (1)
  1. [Table](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#undtbl1)

## Extras (10)
  1. Download all 
  2. [Table S1. Table summarizing the known clinical information for NBM/AML/NSM samples, related to Figures 1, 2, 3, 4, 5, 6, and 7](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#mmc1 "Table S1. Table summarizing the known clinical information for NBM/AML/NSM samples, related to Figures 1, 2, 3, 4, 5, 6, and 7")
  3. [Table S2. List of differentially expressed genes for all annotated cell types, related to Figures 1 and 2](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#mmc2 "Table S2. List of differentially expressed genes for all annotated cell types, related to Figures 1 and 2")
  4. [Table S3. Results of the CellChat and CODEX co-localization enrichment analyses, related to Figures 3, 4, and 5](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#mmc3 "Table S3. Results of the CellChat and CODEX co-localization enrichment analyses, related to Figures 3, 4, and 5")
  5. [Table S4. Summary of the cell types detected using CODEX multiplexed imaging and the markers used to annotate the different cell types, related to Figure 4](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#mmc4 "Table S4. Summary of the cell types detected using CODEX multiplexed imaging and the markers used to annotate the different cell types, related to Figure 4")
  6. [Table S5. List of antibodies used in the study including the vendors, clone information, and other relevant information about antibodies to allow for replication of our panel, related to Figures 4 and 7](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#mmc5 "Table S5. List of antibodies used in the study including the vendors, clone information, and other relevant information about antibodies to allow for replication of our panel, related to Figures 4 and 7")
  7. [Table S6. Imaging parameters including the exposure times and cycle/channel assignment for each antibody in the panel, related to Figures 4 and 7](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#mmc6 "Table S6. Imaging parameters including the exposure times and cycle/channel assignment for each antibody in the panel, related to Figures 4 and 7")

Show all extras
[![Cell](https://ars.els-cdn.com/content/image/B00928674.svg)Cell](https://www.sciencedirect.com/journal/cell "Go to Cell on ScienceDirect")
[Volume 187, Issue 12](https://www.sciencedirect.com/journal/cell/vol/187/issue/12 "Go to table of contents for this volume/issue"), 6 June 2024, Pages 3120-3140.e29
[![Journal home page for Cell](https://ars.els-cdn.com/content/image/Dcellpress.gif)](https://www.sciencedirect.com/journal/cell/vol/187/issue/12)
# 
Resource
Mapping the cellular biogeography of human bone marrow niches using single-cell transcriptomics and proteomic imaging
Author links open overlay panelShovik Bandyopadhyay 1 2, Michael P. Duffy 3 14, Kyung Jin Ahn 4 14, Jonathan H. Sussman 2 8, Minxing Pang 5, David Smith 6, Gwendolyn Duncan 7, Iris Zhang 9, Jeffrey Huang 7, Yulieh Lin 3, Barbara Xiong 2 8, Tamjid Imtiaz 7, Chia-Hui Chen 4, Anusha Thadi 4, Changya Chen 4, Jason Xu 2 8, Melissa Reichart 3, Zachary Martinez 4, Caroline Diorio 4 13, Chider Chen 10…Kai Tan 4 6 13 15
Show more
Outline
Add to Mendeley
Share
Cite
Under a Creative Commons 
Open access
## Highlights
  * •
scRNA-seq and CODEX reveal the composition and spatial architecture of human bone marrow
  * •
Human mesenchymal stromal cells are transcriptionally and functionally heterogeneous
  * •
Adult HSPCs exhibit a peri-adipocytic spatial localization
  * •
CODEX reference mapping to our healthy atlas reveals MSC expansion in AML

## Summary
Non-hematopoietic cells are essential contributors to [hematopoiesis](https://www.sciencedirect.com/topics/immunology-and-microbiology/hematopoiesis "Learn more about hematopoiesis from ScienceDirect's AI-generated Topic Pages"). However, heterogeneity and spatial organization of these cells in human bone marrow remain largely uncharacterized. We used single-cell [RNA sequencing](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/rna-sequence "Learn more about RNA sequencing from ScienceDirect's AI-generated Topic Pages") (scRNA-seq) to profile 29,325 non-hematopoietic cells and discovered nine transcriptionally distinct subtypes. We simultaneously profiled 53,417 [hematopoietic cells](https://www.sciencedirect.com/topics/immunology-and-microbiology/hematopoietic-cell "Learn more about hematopoietic cells from ScienceDirect's AI-generated Topic Pages") and predicted their interactions with non-hematopoietic subsets. We employed co-detection by indexing (CODEX) to spatially profile over 1.2 million cells. We integrated scRNA-seq and CODEX data to link predicted [cellular signaling](https://www.sciencedirect.com/topics/immunology-and-microbiology/signal-transduction "Learn more about cellular signaling from ScienceDirect's AI-generated Topic Pages") with spatial proximity. Our analysis revealed a hyperoxygenated arterio-endosteal neighborhood for early [myelopoiesis](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/myelopoiesis "Learn more about myelopoiesis from ScienceDirect's AI-generated Topic Pages"), and an adipocytic localization for early hematopoietic stem and progenitor cells (HSPCs). We used our CODEX atlas to annotate new images and uncovered [mesenchymal stromal cell](https://www.sciencedirect.com/topics/immunology-and-microbiology/mesenchymal-stem-cell "Learn more about mesenchymal stromal cell from ScienceDirect's AI-generated Topic Pages") (MSC) expansion and spatial neighborhoods co-enriched for leukemic blasts and MSCs in [acute myeloid leukemia](https://www.sciencedirect.com/topics/medicine-and-dentistry/acute-myeloid-leukemia "Learn more about acute myeloid leukemia from ScienceDirect's AI-generated Topic Pages") (AML) patient samples. This spatially resolved, multiomic atlas of human bone marrow provides a reference for investigation of [cellular interactions](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/cell-cell-interaction "Learn more about cellular interactions from ScienceDirect's AI-generated Topic Pages") that drive [hematopoiesis](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/hematopoiesis "Learn more about hematopoiesis from ScienceDirect's AI-generated Topic Pages").
## Graphical abstract
![](https://ars.els-cdn.com/content/image/1-s2.0-S0092867424004082-fx1.jpg)
## Questions answered in this article
**Beta****Powered by GenAI**
 _This is generative AI content and the quality may vary. Learn more._
  1. What are the key findings regarding the spatial organization of hematopoietic and non-hematopoietic cell types in healthy human bone marrow?
  2. Which specific Mesenchymal Stromal Cell (MSC) subsets were found to be transcriptionally specialized for hematopoiesis?
  3. What challenges exist in defining the non-hematopoietic cells of the human bone marrow microenvironment?
  4. What hematopoietic populations were identified in the analysis?
  5. What is the significance of the spatial relationship between Hematopoietic Stem And Progenitor Cells (HSPCs) and adipocytes in the context of aging and acute leukemia?

  * [Previous article in issue](https://www.sciencedirect.com/science/article/pii/S0092867424004112)
  * [Next article in issue](https://www.sciencedirect.com/science/article/pii/S0092867424004070)

## Keywords
bone marrow
microenvironment
neighborhood
hematopoiesis
mesenchymal
spatial omics
signaling
single-cell
leukemia
CODEX
## Introduction
The bone marrow is a complex organ that houses diverse cells from the hematopoietic, mesenchymal, endothelial, [vascular smooth muscle](https://www.sciencedirect.com/topics/immunology-and-microbiology/vascular-smooth-muscle "Learn more about vascular smooth muscle from ScienceDirect's AI-generated Topic Pages"), and neural [lineages](https://www.sciencedirect.com/topics/immunology-and-microbiology/lineages "Learn more about lineages from ScienceDirect's AI-generated Topic Pages").[1](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib1) Relatively rare non-hematopoietic cells are known to make critical contributions to [hematopoiesis](https://www.sciencedirect.com/topics/immunology-and-microbiology/hematopoiesis "Learn more about hematopoiesis from ScienceDirect's AI-generated Topic Pages").[2](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib2) Many non-hematopoietic cell types, including [endothelial cells](https://www.sciencedirect.com/topics/medicine-and-dentistry/endothelial-cell "Learn more about endothelial cells from ScienceDirect's AI-generated Topic Pages") (ECs), [mesenchymal stromal cells](https://www.sciencedirect.com/topics/immunology-and-microbiology/mesenchymal-stem-cell "Learn more about mesenchymal stromal cells from ScienceDirect's AI-generated Topic Pages") (MSCs), and [osteoblasts](https://www.sciencedirect.com/topics/medicine-and-dentistry/osteoblast "Learn more about osteoblasts from ScienceDirect's AI-generated Topic Pages") have been proposed to be critical constituents of the bone marrow niche.[3](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib3),[4](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib4),[5](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib5),[6](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib6) Recent technological advances employing single-cell [RNA sequencing](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/rna-sequence "Learn more about RNA sequencing from ScienceDirect's AI-generated Topic Pages") (scRNA-seq) revealed the existence of multiple subpopulations of these non-hematopoietic marrow elements in mice.[7](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib7),[8](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib8),[9](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib9),[10](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib10)
Despite extensive work studying human [hematopoietic cells](https://www.sciencedirect.com/topics/immunology-and-microbiology/hematopoietic-cell "Learn more about hematopoietic cells from ScienceDirect's AI-generated Topic Pages"), there remains a relative paucity of analogous studies defining the non-hematopoietic cells that make up the human [bone marrow microenvironment](https://www.sciencedirect.com/topics/immunology-and-microbiology/bone-marrow-microenvironment "Learn more about bone marrow microenvironment from ScienceDirect's AI-generated Topic Pages").[11](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib11),[12](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib12),[13](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib13) Defining the single-cell composition of bone marrow niches has been limited by challenges in isolating sufficient viable non-hematopoietic cells, which make up <0.5% of the marrow cellularity.[14](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib14),[15](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib15) Moreover, specific cell isolation methods may greatly influence the diversity of cell types captured, as shown in mice.[9](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib9) [Bone marrow aspiration](https://www.sciencedirect.com/topics/medicine-and-dentistry/bone-marrow-biopsy "Learn more about Bone marrow aspiration from ScienceDirect's AI-generated Topic Pages"), analogous to the flushing of mouse bones, fails to capture cells that are tightly adhered to the bone [surface](https://www.sciencedirect.com/topics/immunology-and-microbiology/surface-property "Learn more about surface from ScienceDirect's AI-generated Topic Pages") and may bias the bone marrow MSC composition toward adipocytic, rather than osteogenic cells. Much of human MSC profiling has been performed on such aspirate samples,[13](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib13),[16](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib16),[17](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib17),[18](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib18) which have indeed yielded mostly adipogenic MSCs. Other human MSC profiling has been performed following a period of _ex vivo_ expansion[19](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib19),[20](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib20) that may alter the transcriptional profile of MSCs from their _in vivo_ homeostatic state. Furthermore, most aforementioned analyses lack spatial information of these niche cells, and it remains unknown whether there are human MSC subtypes with distinct spatial organization.
Here, we overcame these limitations by performing [transcriptomic](https://www.sciencedirect.com/topics/medicine-and-dentistry/transcriptomics "Learn more about transcriptomic from ScienceDirect's AI-generated Topic Pages") profiling of single cells from enzymatically digested fresh human bone marrow tissue without the need for expansion. We employed scRNA-seq to comprehensively characterize the [transcriptomic](https://www.sciencedirect.com/topics/immunology-and-microbiology/transcriptomics "Learn more about transcriptomic from ScienceDirect's AI-generated Topic Pages") profiles of both hematopoietic and non-hematopoietic cell types, predict patterns of intercellular communication, and identify the mediating signaling molecules. In parallel, we used co-detection by indexing (CODEX)[21](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib21) with a 53-antibody panel to systematically and simultaneously characterize all major [bone marrow cell](https://www.sciencedirect.com/topics/immunology-and-microbiology/bone-marrow-cell "Learn more about bone marrow cell from ScienceDirect's AI-generated Topic Pages") types and microenvironmental structures. We uncovered significant non-hematopoietic diversity in the human bone marrow with discrete spatial phenotypes and quantitatively defined how niche elements are organized. Furthermore, we demonstrated the utility of our CODEX atlas in [acute myeloid leukemia](https://www.sciencedirect.com/topics/medicine-and-dentistry/acute-myeloid-leukemia "Learn more about acute myeloid leukemia from ScienceDirect's AI-generated Topic Pages") (AML) patients, where we uncovered enriched MSC-leukemic blast spatial interactions. Collectively, this work represents a comprehensive, spatially resolved, multiomic human bone marrow atlas that will serve as an important reference for future studies of the human bone marrow niche.
## Results
### A comprehensive scRNA-seq atlas of the human bone marrow
To uncover the cellular composition of the human bone marrow, we developed an experimental pipeline for enzymatic release of cells from [femoral head](https://www.sciencedirect.com/topics/immunology-and-microbiology/femoral-head "Learn more about femoral head from ScienceDirect's AI-generated Topic Pages") tissue ([Figure S1](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#figs1), [Figure S10](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#figs10), [Figure S11](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#figs11)A and S1B). We confirmed that areas of interest had normal trabecular structure by performing micro-computed tomography analysis (trabecular bone volume [BV] fraction: 0.261 ± 0.098 [mean ± SD], _n_ = 6, [Figure S1](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#figs1), [Figure S10](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#figs10), [Figure S11](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#figs11)A). Next, we devised an enrichment strategy that would capture [hematopoietic cells](https://www.sciencedirect.com/topics/immunology-and-microbiology/hematopoietic-cell "Learn more about hematopoietic cells from ScienceDirect's AI-generated Topic Pages"), rare hematopoietic stem and progenitor cells (HSPCs), and non-hematopoietic microenvironmental cells ([Figure 1](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#fig1)A; [STAR Methods](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#sec4)). The three populations were pooled in a ratio favoring representation of rare HSPCs and non-hematopoietic cells and then subjected to scRNA-seq ([Figure 1](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#fig1)A).
![](https://ars.els-cdn.com/content/image/1-s2.0-S0092867424004082-figs1.jpg)
Figure S1. Characteristics of our updated scRNA-seq human bone marrow atlas, related to [Figures 1](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#fig1) and [2](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#fig2)
(A) A representative image of femoral head samples received as surgical waste from [total hip arthroplasty](https://www.sciencedirect.com/topics/medicine-and-dentistry/total-hip-arthroplasty "Learn more about total hip arthroplasty from ScienceDirect's AI-generated Topic Pages") surgeries prior to enzymatic digestion, showing that there is ample visible red marrow in many of the samples. The area immediately next to the areas collected for sequencing and histology analysis was subjected to microCT analysis at a 6-μm isotropic voxel size to show the normal trabecular bone structure (circle). A representative example is shown with a representative bone volume/total volume (BV/TV) ratio.
(B) Diagram showing femoral head enzymatic digestion and cell isolation protocol.
(C) Violin plots showing the final atlas distribution of unique expressed genes, sequencing depth (UMI counts, y axis cut at 99th percentile), and percentage of reads mapping to the [mitochondrial genome](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/mitochondrial-genome "Learn more about mitochondrial genome from ScienceDirect's AI-generated Topic Pages") after filtering out low quality cells.
(D) Normalized gene expression is projected onto UMAPs to visualize expression of key marker genes highlighting hematopoietic and non-hematopoietic cell types in the data— _CXCL12_ , stromal; _NCAM1_ , osteolineage; _CDH5_ , endothelial; _[PTPRC](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/protein-tyrosine-phosphatase-receptor-type-c "Learn more about PTPRC from ScienceDirect's AI-generated Topic Pages")_ , hematopoietic; _MZB1_ , plasma cell; _CSF3R_ , granulocyte.
(E) Cell-type frequencies stratified by age of donors.
(F) Comparison of [stromal cell](https://www.sciencedirect.com/topics/medicine-and-dentistry/stromal-cell "Learn more about stromal cell from ScienceDirect's AI-generated Topic Pages") content in the Azimuth bone marrow atlas compared with our study. The circled regions represent the non-hematopoietic bone marrow fractions captured in the published data compared with our atlas designed to capture non-hematopoietic cells.
![](https://ars.els-cdn.com/content/image/1-s2.0-S0092867424004082-gr1.jpg)
Figure 1. A single-cell transcriptomic atlas of hematopoietic and non-hematopoietic cells of human bone marrow
(A) Schematic for the scRNA-seq workflow. Magnetic-activated cell sorting (MACS) separation of hematopoietic, stem/progenitor, and mesenchymal fractions was performed and then pooled into one scRNA-seq reaction per patient.
(B) Uniform manifold approximation and projection (UMAP) representation of 82,742 single-cell [transcriptomes](https://www.sciencedirect.com/topics/medicine-and-dentistry/transcriptome "Learn more about transcriptomes from ScienceDirect's AI-generated Topic Pages") from bone marrow of 12 individuals. AEC, arterial endothelial cell; SEC, sinusoidal endothelial cell; [VSMC](https://www.sciencedirect.com/topics/immunology-and-microbiology/vascular-smooth-muscle-cell "Learn more about VSMC from ScienceDirect's AI-generated Topic Pages"), vascular smooth muscle cell; Ba, [basophil](https://www.sciencedirect.com/topics/medicine-and-dentistry/basophil-cell "Learn more about basophil from ScienceDirect's AI-generated Topic Pages"); Eo, [eosinophil](https://www.sciencedirect.com/topics/medicine-and-dentistry/eosinophil "Learn more about eosinophil from ScienceDirect's AI-generated Topic Pages"); Ma, mast cell; RBC, red [blood cell](https://www.sciencedirect.com/topics/immunology-and-microbiology/hemocyte "Learn more about blood cell from ScienceDirect's AI-generated Topic Pages"); pDC, [plasmacytoid dendritic cell](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/plasmacytoid-dendritic-cell "Learn more about plasmacytoid dendritic cell from ScienceDirect's AI-generated Topic Pages"); CLP, common lymphoid progenitor; MEP, [megakaryocyte](https://www.sciencedirect.com/topics/medicine-and-dentistry/megakaryocyte "Learn more about megakaryocyte from ScienceDirect's AI-generated Topic Pages") erythroid progenitor; GMP, granulocyte monocyte progenitor; MPP, multipotent progenitor; HSPC, hematopoietic stem and progenitor cell; HSC, hematopoietic stem cell; Meg/E, megakaryocyte/erythroid; MSC, mesenchymal stromal cell.
(C) Bar plots showing the cell counts for each lineage captured (left) and the [cell lineage](https://www.sciencedirect.com/topics/medicine-and-dentistry/cell-lineage "Learn more about cell lineage from ScienceDirect's AI-generated Topic Pages") proportions per sample (right).
(D) Heatmap showing normalized gene expression scaled by row (gene) of top differentially expressed and key [cell lineage](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/cell-lineage "Learn more about cell lineage from ScienceDirect's AI-generated Topic Pages") marker genes. EC, endothelial cell; M, vascular smooth muscle. Genes are color-coded to match the lineage of the cell type in which they are differentially expressed.
See also [Figures S1](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#figs1), [S2](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#figs2), and [S3](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#figs3).
We collected bone marrow from fresh femoral heads from 16 individuals between 52 and 80 years of age who underwent hip replacement surgery, performing scRNA-seq on 12 of them and CODEX on another 12, with 8 overlapping samples ([Table S1](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#mmc1)). In total, we sequenced 53,417 hematopoietic cells and 29,325 non-hematopoietic cells ([Figures 1](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#fig1)B and 1C). We captured rich, high-quality transcriptional information in each cell [lineage](https://www.sciencedirect.com/topics/immunology-and-microbiology/lineages "Learn more about lineage from ScienceDirect's AI-generated Topic Pages"), with per-cell medians of 16,903 unique molecular identifiers (UMIs), 3,117 unique genes, and ∼3% mitochondrial reads detected after quality control filtering ([Figure S1](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#figs1), [Figure S10](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#figs10), [Figure S11](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#figs11)C).
We first analyzed the hematopoietic populations. As expected, we identified tri-lineage [hematopoiesis](https://www.sciencedirect.com/topics/immunology-and-microbiology/hematopoiesis "Learn more about hematopoiesis from ScienceDirect's AI-generated Topic Pages") and HSPC subsets ([Figure 1](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#fig1)D; [Table S2](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#mmc2)). Additionally, we improved upon other hematopoiesis-focused bone marrow scRNA-seq references[11](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib11),[12](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib12) by capturing the full [granulocyte](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/granulocyte "Learn more about granulocyte from ScienceDirect's AI-generated Topic Pages") differentiation trajectory by virtue of our sample processing not including density-based isolation of mononuclear cells—a standard clinical [tissue banking](https://www.sciencedirect.com/topics/medicine-and-dentistry/tissue-banking "Learn more about tissue banking from ScienceDirect's AI-generated Topic Pages") process which results in [granulocyte](https://www.sciencedirect.com/topics/medicine-and-dentistry/granulocyte "Learn more about granulocyte from ScienceDirect's AI-generated Topic Pages") loss due to their multi-lobated nuclei (_CSF3R_ ; [Figures 1](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#fig1)B and [Figure S1](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#figs1), [Figure S10](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#figs10), [Figure S11](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#figs11)D). Numerous plasma cells were also captured as a byproduct of the CD45 depletion procedure (_MZB1_ ; [Figures 1](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#fig1)B and [Figure S1](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#figs1), [Figure S10](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#figs10), [Figure S11](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#figs11)D). Age did not skew the frequencies of captured cell types ([Figure S1](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#figs1), [Figure S10](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#figs10), [Figure S11](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#figs11)E).
Next, we analyzed the non-hematopoietic [cell compartment](https://www.sciencedirect.com/topics/medicine-and-dentistry/cell-compartment "Learn more about cell compartment from ScienceDirect's AI-generated Topic Pages"). We identified three major cell lineages—vascular smooth muscle cells (VSMCs) (_[ACTA2](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/acta2 "Learn more about ACTA2 from ScienceDirect's AI-generated Topic Pages")_ , _[RGS5](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/rgs5 "Learn more about RGS5 from ScienceDirect's AI-generated Topic Pages")_ , and _TAGLN_), [ECs](https://www.sciencedirect.com/topics/medicine-and-dentistry/endothelial-cell "Learn more about ECs from ScienceDirect's AI-generated Topic Pages") (_CDH5_ , _PECAM1_ , and [VWF](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/von-willebrand-factor "Learn more about VWF from ScienceDirect's AI-generated Topic Pages")), and mesenchymal cells (_CXCL12_ and _[PDGFRA](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/platelet-derived-growth-factor-receptor-alpha "Learn more about PDGFRA from ScienceDirect's AI-generated Topic Pages")_) including [MSCs](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/stromal-cell "Learn more about MSCs from ScienceDirect's AI-generated Topic Pages") and [osteoblasts](https://www.sciencedirect.com/topics/medicine-and-dentistry/osteoblast "Learn more about osteoblasts from ScienceDirect's AI-generated Topic Pages") ([Figures 1](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#fig1)B–1D and [Figure S1](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#figs1), [Figure S10](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#figs10), [Figure S11](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#figs11)D; [Table S2](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#mmc2)). Our analysis revealed far greater cellular diversity within these cells in bone marrow than existing scRNA-seq references ([Figure S1](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#figs1), [Figure S10](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#figs10), [Figure S11](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#figs11)F). Notably, contrary to earlier findings that characterize [MSCs](https://www.sciencedirect.com/topics/immunology-and-microbiology/mesenchymal-stem-cell "Learn more about MSCs from ScienceDirect's AI-generated Topic Pages") as relatively homogeneous with an adipocytic transcriptional profile,[13](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib13),[16](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib16),[17](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib17),[18](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib18) we found these enzymatically released cells to be highly heterogeneous with numerous clusters identified ([Figures 1](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#fig1)B and [Figure S1](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#figs1), [Figure S10](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#figs10), [Figure S11](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#figs11)F).
Importantly, we did not observe an [inflammatory response](https://www.sciencedirect.com/topics/immunology-and-microbiology/inflammation-response "Learn more about inflammatory response from ScienceDirect's AI-generated Topic Pages") signature, except in cell types which express [inflammatory cytokines](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/inflammatory-cytokine "Learn more about inflammatory cytokines from ScienceDirect's AI-generated Topic Pages") as part of their identity (mature [neutrophils](https://www.sciencedirect.com/topics/immunology-and-microbiology/neutrophil-granulocyte "Learn more about neutrophils from ScienceDirect's AI-generated Topic Pages") and [monocytes](https://www.sciencedirect.com/topics/immunology-and-microbiology/monocyte "Learn more about monocytes from ScienceDirect's AI-generated Topic Pages"), e.g., _CXCL8_ and [interleukin](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/interleukin "Learn more about interleukin from ScienceDirect's AI-generated Topic Pages") _IL-1B_), supporting the notion that these samples represent healthy aged bone marrow ([Figures S2](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#figs2)A and S2B). Moreover, compared to existing healthy donor aspirate-derived bone marrows, our samples did not express more [inflammatory cytokines](https://www.sciencedirect.com/topics/immunology-and-microbiology/inflammatory-cytokine "Learn more about inflammatory cytokines from ScienceDirect's AI-generated Topic Pages") ([Figure S2](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#figs2)C). To further ensure that our samples were not inflamed, we compared our data to published scRNA-seq data from cultured human [MSCs](https://www.sciencedirect.com/topics/medicine-and-dentistry/mesenchymal-stem-cell "Learn more about MSCs from ScienceDirect's AI-generated Topic Pages") treated with [tumor necrosis factor alpha](https://www.sciencedirect.com/topics/immunology-and-microbiology/tumor-necrosis-factor-alpha "Learn more about tumor necrosis factor alpha from ScienceDirect's AI-generated Topic Pages") (TNF-α) and interferon (IFN) γ. We found that our mesenchymal cells had significantly less of an inflamed signature than known inflamed MSCs and were comparable to healthy aspirate MSCs from Li et al. ([Figure S2](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#figs2)D).[17](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib17)
![](https://ars.els-cdn.com/content/image/1-s2.0-S0092867424004082-figs2.jpg)
Figure S2. Comparison of inflammatory cytokine RNA expression and signature scores across multiple bone marrow scRNA-seq datasets, related to [Figures 1](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#fig1) and [2](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#fig2)
(A) UMAP showing hallmark inflammatory response AUCell scores from 300 randomly sampled cells per cell type in our atlas.
(B) Violin plot showing hallmark inflammatory response AUCell scores for all cell types in our atlas.
(C) Normalized RNA expression levels of key inflammatory cytokines across multiple datasets, the pediatric atlas from Chen et al. (CC_Peds), the adult atlas from the [Human Cell](https://www.sciencedirect.com/topics/immunology-and-microbiology/human-cell "Learn more about Human Cell from ScienceDirect's AI-generated Topic Pages") Atlas bone marrow data (HCA_Adult), bone marrow aspirate MSCs from Li et al. (Li_MSCs), and our femoral head-derived atlas (SB_EntireAtlas).
(D) Hallmark inflammatory response AUCell scores from 5,000 cells randomly sampled from each dataset, where control and inflamed MSCs are from GSE115149 in which P3–P6 cultured MSCs were treated with 24 h treatment of TNF-α and IFNγ or control. Li_MSCs are from Li et al. and SB_MSCs are the mesenchymal cells derived from our femoral head-derived atlas. _p_ values were calculated using a two-sided Welch’s two-sample test for each sample versus inflamed MSCs. ∗∗∗ _p_ < 2.2E−16.
### Non-hematopoietic cell subset analysis demonstrates MSC and EC transcriptional diversity
We discerned significant heterogeneity among mesenchymal cells, identifying osteolineage (_NCAM1_ , _SPP1_ , and _BGLAP_), adipo-lineage (_APOE_ , _LPL_ , _PPARG_ , and [CEBPA](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/ccaat-enhancer-binding-protein-alpha "Learn more about CEBPA from ScienceDirect's AI-generated Topic Pages")), and fibroblastic cells (_[PDPN](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/painful-diabetic-peripheral-neuropathy "Learn more about PDPN from ScienceDirect's AI-generated Topic Pages")_ , _CSPG4_ , _DCN_ , and _DPT_) ([Figures 2](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#fig2)A, 2B, [S3](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#figs3)A, and S3B). Osteolineage cells were divided into _IBSP/BGLAP_ -high osteoblasts and _IBSP/BGLAP_ -low Osteo-MSCs ([Figure 2](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#fig2)B). Despite not capturing adipocytes likely due to sample processing loss, we found a cell cluster with the highest _CXCL12_ level and high expression of [adipose](https://www.sciencedirect.com/topics/immunology-and-microbiology/adipose-tissue "Learn more about adipose from ScienceDirect's AI-generated Topic Pages") lineage genes such as _[CEBPA](https://www.sciencedirect.com/topics/immunology-and-microbiology/ccaat-enhancer-binding-protein-alpha "Learn more about CEBPA from ScienceDirect's AI-generated Topic Pages")_ , _PPARG_ , _APOE_ , and _LPL_ , which we labeled as Adipo-MSCs ([Figures 2](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#fig2)B and [S3](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#figs3)B). These cells resemble reported marrow adipogenic lineage precursors (MALPs)[8](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib8) and Adipo-Cxcl12-abundant-reticular (Adipo-CAR) cells[9](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib9) in mice. We also found a population of _THY1_ + MSCs, which expressed adipocytic genes strongly, but also a unique expression profile including _THY1_ and _LBP_ , but lower _LPL_ expression compared with Adipo-MSCs, which we termed THY1+ MSCs ([Figures 2](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#fig2)B and [S3](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#figs3)B). THY1+ MSCs did not have a clear murine counterpart.[22](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib24) Fibroblast-like cells that expressed _CXCL12_ , highly expressed _DPT_ and [NT5E](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/nt5e "Learn more about NT5E from ScienceDirect's AI-generated Topic Pages"), and very specifically expressed _PDPN_ and _[HAS1](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/has1 "Learn more about HAS1 from ScienceDirect's AI-generated Topic Pages")_ were also observed, which we termed Fibro-MSCs ([Figure 2](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#fig2)B). We also identified an _APOD_ + _GSN_ hi population of cells, which we were unable to correlate to previously known cell types in humans or mice, and so we named them APOD+ MSCs as this gene was highly and specifically expressed ([Figures 2](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#fig2)A and 2B).
![](https://ars.els-cdn.com/content/image/1-s2.0-S0092867424004082-gr2.jpg)
Figure 2. Defining the non-hematopoietic cellular composition of human bone marrow
(A) UMAP of 19,257 mesenchymal cells from 12 individuals showing different mesenchymal subsets, with RNAlo MSCs being excluded due to their quality control (QC) profile.
(B) Dot plot showing normalized expression of key mesenchymal marker genes in MSC subsets.
(C) Dot plot showing the normalized expression of literature-derived marker genes for human MSCs including [NT5E](https://www.sciencedirect.com/topics/immunology-and-microbiology/nt5e "Learn more about NT5E from ScienceDirect's AI-generated Topic Pages") (CD73), _THY1_ (CD90), and _[ENG](https://www.sciencedirect.com/topics/medicine-and-dentistry/electronystagmography "Learn more about ENG from ScienceDirect's AI-generated Topic Pages")_ (CD105). _NGFR_ (CD271)[23](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib22) and [MCAM](https://www.sciencedirect.com/topics/medicine-and-dentistry/cd146-antigen "Learn more about MCAM from ScienceDirect's AI-generated Topic Pages") (CD146)[24](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib23) have also been described as canonical MSC markers.
(D) CytoTRACE analysis projected onto the MSC UMAP showing the predicted differentiation score.
(E) Boxplots showing the relative fibroblast colony-forming potential of sorted MSC subtypes. Each data point is one sample. _p_ values were computed using Welch’s two-sample two-sided test.
(F) Line plot showing the population doublings over the course of cell culturing for eight passages, where all cells were passaged every 7 days. _p_ values were computed as Fibro-MSC versus other MSCs by two-way ANOVA. ∗ _p_ < 0.05; ∗∗ _p_ < 0.01; ∗∗∗ _p_ < 0.001.
(G) UMAP showing 3,874 endothelial cells from the 12 individuals. SEC, sinusoidal endothelial cell; AEC, arterial endothelial cell.
(H) Dot plot showing normalized expression of selected pan-endothelial and differentially expressed genes by adjusted _p_ value between endothelial subsets.
(I) Reference mapping of MSC subsets from human bone marrow between this study and published studies. Left, counts of mapped MSCs in each dataset; right, percentages of mapped MSCs in each dataset. Datasets are color-coded based on the experimental protocol.
See also [Figures S1](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#figs1), [S2](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#figs2), [S3](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#figs3), and [S4](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#figs4).
The Fibro-MSC population aligned closely with the previously described early mesenchymal progenitor population[8](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib8),[9](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib9) in mice defined by _CPSG4_ , [CD34](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/cd34 "Learn more about CD34 from ScienceDirect's AI-generated Topic Pages"), and _DPT_ expression and the human skeletal stem cell population[25](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib25) defined by _PDPN_ , _CD164_ , and _NT5E_ (CD73) expression and lack of [lineage marker](https://www.sciencedirect.com/topics/medicine-and-dentistry/lineage-markers "Learn more about lineage marker from ScienceDirect's AI-generated Topic Pages") expression ([Figures 2](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#fig2)B and [S3](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#figs3)B). Fibro-MSCs also expressed mesenchymal [stem cell markers](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/stem-cell-marker "Learn more about stem cell markers from ScienceDirect's AI-generated Topic Pages") defined by the International Society for Cellular Therapy[26](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib26) (ISCT), namely _NT5E_ (CD73), _THY1_ (CD90), and _[ENG](https://www.sciencedirect.com/topics/medicine-and-dentistry/electronystagmography "Learn more about ENG from ScienceDirect's AI-generated Topic Pages")_ (CD105) ([Figure 2](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#fig2)C). It is important to note that expression of these three genes was highly variable among MSC subsets, and this bias should be considered when using them to isolate MSCs. In particular, inclusion of [CD90](https://www.sciencedirect.com/topics/immunology-and-microbiology/cd90 "Learn more about CD90 from ScienceDirect's AI-generated Topic Pages") (_THY1_) as a sorting marker may bias the isolated MSC population toward Fibro- and THY1+ MSCs. _NGFR_ (_CD271_) was more consistently expressed among subsets, providing rationale for its continued use[23](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib22) as an unbiased MSC marker ([Figure 2](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#fig2)C). [MCAM](https://www.sciencedirect.com/topics/immunology-and-microbiology/cd146-antigen "Learn more about MCAM from ScienceDirect's AI-generated Topic Pages") (_[CD146](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/cd146 "Learn more about CD146 from ScienceDirect's AI-generated Topic Pages")_), however, was not highly expressed. _NES_ , a MSC marker in mice,[27](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib27) was not detected in human MSCs but was expressed in sinusoidal endothelial cells (SECs), arterial endothelial cells (AECs), and VSMCs ([Figure S3](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#figs3)B). It should be noted that there was considerable variability in mesenchymal cell frequency between samples, which could reflect dynamic cell states such as activation or technical factors including sampling differences ([Figure S3](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#figs3)C).
![](https://ars.els-cdn.com/content/image/1-s2.0-S0092867424004082-figs3.jpg)
Figure S3. Non-hematopoietic cell subsets quality control, selected expression profiles, validation, and frequencies, related to [Figures 1](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#fig1), [2](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#fig2), and [3](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#fig3)
(A) Violin plots showing the non-hematopoietic [cell distribution](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/subcellular-localization "Learn more about cell distribution from ScienceDirect's AI-generated Topic Pages") of uniquely expressed genes, sequencing depth (UMI counts), and percentage of reads mapped to the [mitochondrial genome](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/mitochondrial-genome "Learn more about mitochondrial genome from ScienceDirect's AI-generated Topic Pages"). RNAlo MSCs were removed from subsequent analysis based on these metrics.
(B) Dot plot showing gene expression of additional canonical marker genes in bone marrow non-hematopoietic cells.
(C) Stacked bar plot showing the per-sample frequency of MSC (top) and endothelial (bottom) subpopulations. Samples were ordered by age.
(D) Gating strategy from a representative sort used to sort MSC subpopulations in [Figures 2](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#fig2)E, 2F, and [3](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#fig3)C.
(E) Violin plot showing normalized expression of genes coding for markers used in cell sorting to distinguish MSC subsets.
(F) Day 28 micrographs of adipogenic, osteogenic and chondrogenic differentiation of Fibro-MSCs. [Adipogenic differentiation](https://www.sciencedirect.com/topics/immunology-and-microbiology/adipogenesis "Learn more about Adipogenic differentiation from ScienceDirect's AI-generated Topic Pages") was assessed by oil red O staining, osteogenic by [Alizarin](https://www.sciencedirect.com/topics/medicine-and-dentistry/alizarin "Learn more about Alizarin from ScienceDirect's AI-generated Topic Pages") red staining, and chondrogenic by [Alcian blue](https://www.sciencedirect.com/topics/medicine-and-dentistry/alcian-blue "Learn more about Alcian blue from ScienceDirect's AI-generated Topic Pages") staining.
(G) Day 14 [qPCR](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/real-time-polymerase-chain-reaction "Learn more about qPCR from ScienceDirect's AI-generated Topic Pages") comparing RNA expression following differentiation of cultured Fibro-MSCs using adipogenic or osteogenic medium. Control samples were taken from day 0 of the differentiation.
Using CytoTRACE pseudotime analysis,[28](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib28) we determined that Fibro-MSCs were the most primitive among MSC subsets, consistent with their [transcriptomic](https://www.sciencedirect.com/topics/immunology-and-microbiology/transcriptomics "Learn more about transcriptomic from ScienceDirect's AI-generated Topic Pages") similarity to reported mesenchymal progenitors and the ISCT definition ([Figures 2](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#fig2)C and 2D). THY1+ MSCs also had a high CytoTRACE score, suggesting that they may also be more primitive. To validate that Fibro- and THY1+ MSCs were more stem/progenitor-like, we sorted four [MSC subpopulations](https://www.sciencedirect.com/topics/immunology-and-microbiology/cell-subpopulation "Learn more about MSC subpopulations from ScienceDirect's AI-generated Topic Pages") (Adipo-MSCs, THY1+ MSCs, Osteo-MSCs/osteoblasts, and Fibro-MSCs) based on markers derived from our scRNA-seq data ([Figures S3](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#figs3)D and S3E) and performed fibroblast colony-forming unit (CFU-F) assays. Sorted Podoplanin+ Fibro-MSCs exhibited the highest CFU-F forming capacity, followed by THY1+ MSCs, with Osteo-MSCs and Adipo-MSCs showing the least ([Figure 2](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#fig2)E). Cultured Fibro- and THY1+ MSCs maintained proliferation at high passages while Adipo-MSCs and Osteo-MSCs slowed progressively ([Figure 2](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#fig2)F). Fibro-MSCs could differentiate into osteoblasts, adipocytes, and [chondrocytes](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/chondrocyte "Learn more about chondrocytes from ScienceDirect's AI-generated Topic Pages") after being cultured in the appropriate differentiation medium ([Figures S3](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#figs3)F and S3G). Notably, these experiments using flow-sorted cells also confirmed the protein-level divergence of these MSC subsets and identified [Podoplanin](https://www.sciencedirect.com/topics/medicine-and-dentistry/podoplanin "Learn more about Podoplanin from ScienceDirect's AI-generated Topic Pages") as a robust marker for mesenchymal stem cells. Overall, our findings underscore Fibro-MSCs’ alignment with the human mesenchymal stem cell criteria defined by the ISCT and resemblance to reported early mesenchymal progenitors in both human and mouse.[8](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib8),[25](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib25)
Our data also captured two major classes of bone marrow ECs—AECs and SECs ([Figure 2](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#fig2)G). AECs showed higher expression of _CXCL12_ , [ICAM2](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/icam2 "Learn more about ICAM2 from ScienceDirect's AI-generated Topic Pages"), and _PODXL_ , but lower levels of venous genes such as _EPHB4_ and _NR2F2_ than SECs, which expressed more genes associated with endothelial-hematopoietic [cell interactions](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/cell-cell-interaction "Learn more about cell interactions from ScienceDirect's AI-generated Topic Pages") (e.g., _ACKR1_ and _SELE_) ([Figure 2](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#fig2)H). We did not find cells associated with type H vessels that connect [arterioles](https://www.sciencedirect.com/topics/medicine-and-dentistry/arteriole "Learn more about arterioles from ScienceDirect's AI-generated Topic Pages") to [sinusoids](https://www.sciencedirect.com/topics/medicine-and-dentistry/sinusoid "Learn more about sinusoids from ScienceDirect's AI-generated Topic Pages"), possibly due to their low frequency and specific association with the [growth plate](https://www.sciencedirect.com/topics/medicine-and-dentistry/epiphyseal-plate "Learn more about growth plate from ScienceDirect's AI-generated Topic Pages") of young individuals.[29](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib29) Additionally, we did not capture a cluster of _PROX1_ + _PDPN_ + _LYVE1_ + lymphatic ECs but did observe rare _LYVE1_ -expressing cells ([Figure S4](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#figs4)A), which could be related to very rare bone marrow lymphatic progenitors previously described.[30](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib30),[31](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib31)
![](https://ars.els-cdn.com/content/image/1-s2.0-S0092867424004082-figs4.jpg)
Figure S4. Signaling families comprising intercellular communication modules, related to [Figure 3](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#fig3)
(A) UMAP plots of endothelial cell subsets with overlaid expression of  _PDPN_ , _LYVE1_ , and  _PROX1_ , which are markers of lymphatic endothelial cells.
(B) Stacked bar plots of Jardine et al. data RPCA-mapped to our atlas split by sample (left) and by Jardine et al.’s original cluster annotations (right).
(C) Metascape [gene ontology](https://www.sciencedirect.com/topics/immunology-and-microbiology/gene-ontology "Learn more about gene ontology from ScienceDirect's AI-generated Topic Pages") results for cell types, showing the top ten for both adult (our NBM atlas) and fetal (Jardine et al.) Fibro-MSCs.
(D) Representative flow cytometry plots corresponding to [Figure 3](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#fig3)C, pre-gated for lineage-negative cells.
(E) UMAP plot of the entire scRNA-seq atlas with overlaid normalized  _CSF3_ expression across all cell types in the atlas.
(F) Heatmap showing the contribution of each signaling family to the NMF-derived cellular communication modules from [Figure 3](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#fig3). Contribution scores were calculated as previously described in Jin et al.
To further emphasize the importance of capturing the heterogeneity of non-hematopoietic cells, we mapped multiple published scRNA-seq datasets of [healthy human](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/normal-human "Learn more about healthy human from ScienceDirect's AI-generated Topic Pages") MSCs[13](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib13),[16](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib16),[17](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib17),[18](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib18),[32](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib32),[33](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib33) to our atlas. We observed a dominance of Adipo-MSCs in aspirate-derived samples ([Figure 2](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#fig2)I). Our atlas captured 24.4-fold more non-Adipo-MSC subsets than all mapped adult datasets combined ([Figure 2](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#fig2)I). Fibro-MSCs were essentially not captured at all in other adult datasets. Only 76 THY1+ MSCs were captured across all other datasets combined, compared with 4,178 in our dataset ([Figure 2](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#fig2)I). Since Jardine et al. was the only other non-aspirate-derived dataset (crushed fetal femur), we were interested to compare fetal cell states with adult cell states. Most fetal MSC populations corresponded well with adult MSC populations, with mapping of labels from Jardine et al. to corresponding labels in our atlas—“fibroblasts” to Fibro-MSCs, “Adipo-CAR” to Adipo and THY1+ MSCs, and “osteogenic populations” to Osteo-MSCs/osteoblasts/APOD+ MSCs ([Figure S4](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#figs4)B). We next examined the differences between fetal and adult Fibro-MSCs, finding that fetal Fibro-MSCs had a radial glial-like signature perhaps indicative of [neural crest](https://www.sciencedirect.com/topics/medicine-and-dentistry/neural-crest "Learn more about neural crest from ScienceDirect's AI-generated Topic Pages") origin[34](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib34) and that adult Fibro-MSCs more closely resemble prototypical fibroblasts ([Figure S4](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#figs4)C). In sum, we found that there is previously unrepresented and functionally consequential diversity within human bone marrow MSCs that cannot be captured efficiently from aspirate samples and demonstrated the utility of our atlas as a healthy reference to characterize similarities and differences to MSCs in other scRNA-seq datasets.
### MSCs, ECs, and osteolineage cells cooperate to produce diverse hematopoietic support factors
We next asked whether [MSC heterogeneity](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/cell-heterogeneity "Learn more about MSC heterogeneity from ScienceDirect's AI-generated Topic Pages") translated to differential transcript levels of support factors for [hematopoiesis](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/hematopoiesis "Learn more about hematopoiesis from ScienceDirect's AI-generated Topic Pages"). Starting with hematopoietic factors, which have been widely studied in non-hematopoietic cell subsets[2](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib2),[35](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib35),[36](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib36) ([Figure 3](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#fig3)A), we found different MSC types specialized in producing specific support factors; for example, _CXCL12_ was highest in Adipo- and THY1+ MSCs, whereas [IGF1](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/insulin-like-growth-factor-1 "Learn more about IGF1 from ScienceDirect's AI-generated Topic Pages") was predominantly expressed by Fibro-MSCs.
![](https://ars.els-cdn.com/content/image/1-s2.0-S0092867424004082-gr3.jpg)
Figure 3. Cell-cell communication analysis reveals diverse signaling patterns between hematopoietic and non-hematopoietic cell types
(A) Dot plot showing the normalized expression of manually curated support factors for hematopoiesis in mesenchymal and endothelial populations.
(B) Chord diagrams showing significant interactions between source mesenchymal/endothelial cell types and target HSPC cell types predicted by CellChat. Thickness of the line corresponds to the strength of the predicted interaction.
(C) Co-culture of freshly isolated CD34+ HSPCs with sorted and cultured MSC subsets. Error bar represents mean ± SD of samples from 3 (monoculture) or 4 (co-culture) independent experiments. _p_ values were computed using Welch’s two-sample one-sided test.
(D) CellChat network analysis with cell types scored based on their outgoing and incoming contributions to the network, where strength is defined by number of significant outgoing interactions and the count refers to the number of cells in each group.
(E and F) Outgoing (ligand enriched, E) and incoming (receptor enriched, F) significant signaling predictions were scored for each annotated signaling family, and the strength of signaling (CellChat interaction probability) for each cell type was row-scaled and plotted by pathway. The barplot on the right-hand side of each panel shows the total signaling strength across all cell groups.
(G) Non-negative matrix factorization (NMF) to identify modules of outgoing signaling and the contribution of each cell type to the pattern.
See also [Figures S3](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#figs3), [S4](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#figs4), and [S8](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#figs8).
We next employed CellChat[37](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib37) to systematically interrogate cellular communications in the bone marrow based on cognate ligand and receptor co-expression between two cell types, as demonstrated by _CXCL12_ and _SELE_ interactions of MSCs and SECs with different HSPC subsets, respectively ([Figure 3](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#fig3)B). To validate the predicted interactions between HSPCs and MSC subsets, we performed co-culture of freshly sorted MSC subsets and freshly isolated CD34+ cells. We found that sorted Adipo-, Osteo-, and THY1+ MSCs significantly preserved primitive Lin− CD34+ CD38− cells, along with a strong trend from Fibro-MSCs, as opposed to monoculture where these cells were scantly preserved ([Figures 3](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#fig3)C and [S4](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#figs4)D). We also found that globally, mesenchymal lineage cells had the most outgoing signaling contributions (i.e., significant interactions in which they expressed the ligand) ([Figure 3](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#fig3)D; [Table S3](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#mmc3)).
Next, we interrogated support factor expression and predicted communication of non-hematopoietic cells with hematopoietic cells focusing on HSPC maintenance, [myelopoiesis](https://www.sciencedirect.com/topics/medicine-and-dentistry/myelopoiesis "Learn more about myelopoiesis from ScienceDirect's AI-generated Topic Pages"), and [lymphopoiesis](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/lymphopoiesis "Learn more about lymphopoiesis from ScienceDirect's AI-generated Topic Pages"). We grouped support factors based on their shared pathway annotation described in the CellChat database[37](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib37) (e.g., ANGPT family contains multiple [angiopoietin](https://www.sciencedirect.com/topics/medicine-and-dentistry/angiopoietin "Learn more about angiopoietin from ScienceDirect's AI-generated Topic Pages") genes). Several cell types have been reported to be important for HSPC maintenance, including osteoblasts[5](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib5),[38](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib38),[39](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib39) and perivascular MSCs.[3](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib3),[4](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib4) Our data suggest that both osteoblasts and MSCs may support human HSPCs, but most of the canonical support factors are produced by MSC subsets, especially Adipo- and THY1+ MSCs. For instance, Adipo- and THY1+ MSCs produced the highest levels of canonical HSPC niche factors such as _CXCL12_[ 40](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib40) (CXCL family) and _KITLG_[ 41](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib41) (KIT family), which were broadly received by hematopoietic cells without strong specificity for HSPCs ([Figures 3](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#fig3)A, 3E, and 3F). Osteolineage cells expressed _TGFB1_ and [CDH2](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/cdh2 "Learn more about CDH2 from ScienceDirect's AI-generated Topic Pages") more than Adipo- or THY1+ MSCs ([Figure 3](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#fig3)A), which have been shown to support hematopoietic stem cell (HSC) quiescence in mice.[5](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib5),[42](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib42) HSCs also had significant predicted incoming signaling from E-selectin provided by SECs ([Figures 3](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#fig3)A and 3F).
We next focused on factors known to be important for [myelopoiesis](https://www.sciencedirect.com/topics/immunology-and-microbiology/myelopoiesis "Learn more about myelopoiesis from ScienceDirect's AI-generated Topic Pages"). _CSF1_ was expressed by multiple MSC subsets, with the highest expression in Adipo-MSCs ([Figure 3](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#fig3)A). _CSF2_ and _CSF3_ , however, were not detected at high levels, although a subset of SECs (5.8%) did have detectable expression of _CSF3_ (n.s., [Figure S4](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#figs4)E). This result may reflect the inherent challenge of detecting transient cytokine gene expression.[43](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib43) While most hematopoietic support factors were provided by mesenchymal cells, we also noted that CD4+ T cells, which are not typically thought of as critical niche cells, were major producers of _FLT3LG_ in the bone marrow ([Figure 3](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#fig3)E; [Table S3](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#mmc3)). HSCs, granulocyte monocyte progenitors (GMPs), and common lymphoid progenitors (CLPs) were predicted to receive _FLT3LG_ , which is known to be critical for both [myelopoiesis](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/myelopoiesis "Learn more about myelopoiesis from ScienceDirect's AI-generated Topic Pages")[44](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib44) and [lymphopoiesis](https://www.sciencedirect.com/topics/medicine-and-dentistry/lymphopoiesis "Learn more about lymphopoiesis from ScienceDirect's AI-generated Topic Pages")[45](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib45) ([Figure 3](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#fig3)F).
We then studied niche contributions to [lymphopoiesis](https://www.sciencedirect.com/topics/immunology-and-microbiology/lymphopoiesis "Learn more about lymphopoiesis from ScienceDirect's AI-generated Topic Pages") by interrogating factors such as _IL-7_ and Notch ligands.[46](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib46),[47](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib47) _IL-7_ was primarily produced by Adipo- and THY1+ MSCs ([Figure 3](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#fig3)A). The concept that the vascular niche provides the majority of [Notch signaling](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/notch-signaling-pathway "Learn more about Notch signaling from ScienceDirect's AI-generated Topic Pages") in the bone marrow has been suggested in mice,[7](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib7) while in humans, CD146+ MSCs[48](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib48) have been proposed to be critical but the vascular contribution has not been clearly delineated. Collectively, ECs and VSMCs are indeed the major contributors of _DLL1,__[DLL4](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/dll4 "Learn more about DLL4 from ScienceDirect's AI-generated Topic Pages")_ , [JAG1](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/jag1 "Learn more about JAG1 from ScienceDirect's AI-generated Topic Pages"), and _[JAG2](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/jag2 "Learn more about JAG2 from ScienceDirect's AI-generated Topic Pages")_ in the human bone marrow, with [arterioles](https://www.sciencedirect.com/topics/immunology-and-microbiology/arteriole "Learn more about arterioles from ScienceDirect's AI-generated Topic Pages") (AECs and VSMCs) expressing the most _JAG1_ ([Figure 3](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#fig3)A). We also noted that [Notch signaling](https://www.sciencedirect.com/topics/medicine-and-dentistry/notch-signaling "Learn more about Notch signaling from ScienceDirect's AI-generated Topic Pages") was predicted as outgoing from mesenchymal cells ([Figure 3](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#fig3)E) but was weaker than endothelial-driven [Notch signaling](https://www.sciencedirect.com/topics/immunology-and-microbiology/notch-signaling "Learn more about Notch signaling from ScienceDirect's AI-generated Topic Pages") and largely based on _JAG1_ expression. Interestingly, VSMCs also received significant incoming Notch signaling through _NOTCH3_ , possibly received from AECs ([Figure 3](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#fig3)F; [Table S3](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#mmc3)). Notably, all of these non-hematopoietic cell types, as well as HSPCs, were predicted to receive Notch signals to some extent, raising the possibility of crosstalk and coordinated regulation between niche elements. It should be noted that, surprisingly, osteoblasts did not express high levels of lymphoid-associated factors ([Figure 3](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#fig3)A).
Using non-negative matrix factorization (NMF),[37](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib37) we identified higher-level communication modules in the bone marrow. We found that different non-hematopoietic subsets contributed to different modules, which we annotated based on the cell types involved ([Figures 3](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#fig3)G and [S4](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#figs4)F). The endosteal module, characterized by bone-specific [gene families](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/multigene-family "Learn more about gene families from ScienceDirect's AI-generated Topic Pages") like NGF, ncWNT, and [osteopontin](https://www.sciencedirect.com/topics/immunology-and-microbiology/osteopontin "Learn more about osteopontin from ScienceDirect's AI-generated Topic Pages"), differed significantly from the MSC module which expressed families such as KIT, FGF, THY1. _CXCL12_ hi MSC subsets such as Adipo- and THY1+ MSCs contributed unique signals mainly to the MSC module whereas Osteo- or Fibro-MSCs were involved in both the endosteal and MSC modules ([Figure 3](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#fig3)G). Furthermore, consistent with pathway-level CellChat analysis, VSMCs, and to a lesser extent, AECs, contributed to both endosteal and vascular signaling modules.
In sum, our analysis highlights the complex communication potential of human bone marrow mesenchymal and ECs and suggests specialization of these cells in supporting different aspects of [hematopoiesis](https://www.sciencedirect.com/topics/medicine-and-dentistry/hematopoiesis "Learn more about hematopoiesis from ScienceDirect's AI-generated Topic Pages"). Particularly, Adipo- and THY1+ MSCs emerged as the most transcriptionally specialized to support hematopoiesis through their high expression of key niche factors such as _CXCL12_ , _KITLG_ , _CSF1_ , and _IL-7_.
### CODEX multiplexed imaging reveals the anatomy of the human bone marrow niche _in situ_
We next sought to define the single-cell [anatomy](https://www.sciencedirect.com/topics/medicine-and-dentistry/anatomy "Learn more about anatomy from ScienceDirect's AI-generated Topic Pages") of the [bone marrow cell](https://www.sciencedirect.com/topics/immunology-and-microbiology/bone-marrow-cell "Learn more about bone marrow cell from ScienceDirect's AI-generated Topic Pages") types that we defined in our scRNA-seq atlas. We employed whole-slide CODEX multiplexed imaging of 12 specimens, 8 of which were included in our [transcriptomic](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/transcriptomics "Learn more about transcriptomic from ScienceDirect's AI-generated Topic Pages") atlas ([Figure 4](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#fig4)A). Guided by our scRNA-seq data, we designed and validated a 54-plex panel (53 antibodies + DAPI) ([Figures 4](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#fig4)A, 4B, and [S5](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#figs5); [Tables S4](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#mmc4), [S5](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#mmc5), and [S6](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#mmc6); [Data S1](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#mmc10)). For instance, in addition to canonical HSPC markers such as CD34 and [CD38](https://www.sciencedirect.com/topics/immunology-and-microbiology/cd38 "Learn more about CD38 from ScienceDirect's AI-generated Topic Pages"), we included markers derived from our scRNA-seq data such as _SPINK2_ , which was expressed most highly in primitive HSCs[49](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib49),[50](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib50) ([Table S2](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#mmc2)). This increased the resolution to identify primitive HSPCs as the dynamic range of the CODEX assay is lower than that of flow cytometry where CD34 and [CD38](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/cd38 "Learn more about CD38 from ScienceDirect's AI-generated Topic Pages") are the gold-standard. Whole-slide CODEX images were acquired and subsequently segmented using Mesmer[51](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib51) to identify single cells. Cells were then annotated by unsupervised clustering, and in rare circumstances refinement by manual gating ([STAR Methods](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#sec4)). Importantly, after each round of clustering, cell labels were overlaid on the fluorescent image and clusters were iteratively visually inspected and corrected if necessary ([Figures S5](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#figs5) and [S6](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#figs6)A; [Data S1](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#mmc10)). We computationally annotated 803,132 cells (91.6% of segmented objects) ([Figures 4](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#fig4)C and [S6](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#figs6)B) covering 32 cell types across 12 samples, including rare cells such as immunophenotypic HSCs (Lin− CD34+ CD38− CD45RA− CD90+) and [Schwann cells](https://www.sciencedirect.com/topics/medicine-and-dentistry/schwann-cell "Learn more about Schwann cells from ScienceDirect's AI-generated Topic Pages") (PLP1+ CD271+) ([Figures 4](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#fig4)C and [S6](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#figs6)B; [Table S4](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#mmc4); [Data S1](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#mmc10)). The overall cell type distribution was similar between individuals ([Figure S6](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#figs6)C; [Table S1](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#mmc1)). The protein-level phenotyping also revealed certain interesting patterns, such as [CD163](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/cd163 "Learn more about CD163 from ScienceDirect's AI-generated Topic Pages") positivity of all observed macrophages and different clusters of GATA1+/− [megakaryocytes](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/megakaryocyte "Learn more about megakaryocytes from ScienceDirect's AI-generated Topic Pages") ([Figure S6](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#figs6)D; [Data S1](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#mmc10)).
![](https://ars.els-cdn.com/content/image/1-s2.0-S0092867424004082-gr4.jpg)
Figure 4. 54-plex CODEX imaging reveals the spatial cellular topography of human bone marrow
(A) Schematic depicting the CODEX experimental and computational workflow leading to cell type identification.
(B) 53-antibody CODEX panel split by target cell population.
(C) Heatmap showing average centered log ratio (CLR)-normalized expression per cell type scaled by protein marker and hierarchically clustered by protein expression (left). Boxes highlight coordinated marker expression. UMAP showing the 803,131 single cells in the CODEX atlas from 12 individuals colored by cell type (right). Sh, [Schwann cells](https://www.sciencedirect.com/topics/immunology-and-microbiology/schwann-cell "Learn more about Schwann cells from ScienceDirect's AI-generated Topic Pages").
(D) CODEX images showing manually identified examples of Fibro-MSC, osteolineage cells (Osteo-MSC [CXCL12+ CD56+], osteoblast [CXCL12lo CD56+]), Adipo-MSCs (FOXC1+ CXCL12+), and THY1+ MSCs (CD90+ CXCL12+ FOXC1+).
(E) Violin plot showing the nearest distance of annotated cells from different MSC subtypes (_n_ = 64 Fibro-MSC, _n_ = 422 Osteo-MSC, _n_ = 5, 110 Adipo-MSC, and _n_ = 4,108 THY1+ MSC) to the manually annotated bone contours. Distances were clipped to the 99th percentile for each cell type for improved visualization. _p_ values were computed using two-sided Wilcoxon rank-sum test.
(F) CODEX image (left) is paired with the cell phenotype map (CPM, right), showing the segmented cells colored by cell annotation.
See also [Figures S5](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#figs5) and [S6](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#figs6).
![](https://ars.els-cdn.com/content/image/1-s2.0-S0092867424004082-figs5.jpg)
Figure S5. Grayscale images of all CODEX markers, related to [Figure 4](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#fig4)
All 54 markers (53 antibodies + DAPI) used in the primary panel of normal bone marrow are shown. [MCT](https://www.sciencedirect.com/topics/medicine-and-dentistry/tryptase "Learn more about MCT from ScienceDirect's AI-generated Topic Pages") is shown in red to help distinguish autofluorescent/sticky cells compared with true staining.
![](https://ars.els-cdn.com/content/image/1-s2.0-S0092867424004082-figs6.jpg)
Figure S6. CODEX cell typing validation and cell phenotype maps, related to [Figure 4](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#fig4)
(A) Flow chart showing the cell-type annotation workflow.
(B) CODEX images with selected markers and corresponding cell phenotype maps (CPMs) showing appropriate co-labeling of certain markers with registered H&E images. Gray cell masks refer to all other segmented cells in the final analysis (imaging artifacts and mast cell tryptase [MCT]+ autofluorescent/sticky cells removed). MCT is used as a marker of [autofluorescence](https://www.sciencedirect.com/topics/medicine-and-dentistry/autofluorescence "Learn more about autofluorescence from ScienceDirect's AI-generated Topic Pages") in these images ([STAR Methods](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#sec4)). Further details on marker combinations used to define each cell type can be found in [Table S4](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#mmc4). Registered H&E images of the exact imaged region are also provided. Comprehensive CODEX CPMs and pertinent positive/negative markers as well as associated scRNA-seq profiles can be found in the supplemental cell-type annotation guide ([Data S1](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#mmc10)).
(C) Stacked bar plot showing cell-type frequency distribution across samples.
(D) CODEX image of macrophages showing that CD163 better labels cytoplasmic projections, but bone marrow macrophages are positive for both [CD68](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/cd68 "Learn more about CD68 from ScienceDirect's AI-generated Topic Pages") and CD163. MCT is included to label autofluorescent/sticky cells.
(E) Violin plots showing concordance of marker gene and protein expression for MSC subsets.
(F) Table showing most correlated cell types annotated based on orthologous expression in scRNA-seq and CODEX data.
(G) Pearson correlations of marker gene (scRNA-seq) and protein (CODEX) profiles for a given cell-type pairs. Cell labels were harmonized between scRNA-seq and CODEX cell-type labels to the extent possible—cells without a clear match were not included in the analysis. Osteo- and Fibro-MSCs were manually annotated and also included in this analysis in addition to the rest of the clustering-derived cell types.
We subcategorized the identified non-hematopoietic cells into AECs, SECs, VSMCs, endosteal cells, Adipo-MSCs, and THY1+ MSCs. Notably, despite some tissue processing artifacts, such as frequent loss of [bone tissue](https://www.sciencedirect.com/topics/medicine-and-dentistry/bone-tissue "Learn more about bone tissue from ScienceDirect's AI-generated Topic Pages"), we were able to identify para-trabecular cells with non-hematopoietic expression profiles that we termed “endosteal.” Neither Osteo-MSCs nor Fibro-MSCs were initially detected computationally, suggesting that these cells were tied more closely to bone and were thus lost during tissue processing. Upon closer manual inspection, we indeed detected rare instances of these cells ([Figure 4](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#fig4)D), with Osteo-MSCs and osteoblasts being detected on the hematopoietic bone interface, and Podoplanin+ Fibro-MSCs being detected largely in the middle of the bone region ([Figures 4](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#fig4)D, 4E, and [S6](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#figs6)E). Additionally, 53% of the 422 manually annotated Osteo-MSCs overlapped with clustered endosteal cells, showing that the endosteal population is enriched for osteolineage cells.
Furthermore, the [protein expression](https://www.sciencedirect.com/topics/immunology-and-microbiology/protein-expression "Learn more about protein expression from ScienceDirect's AI-generated Topic Pages") profiles of the CODEX cell types mapped well to our scRNA-seq data, with the matching cell types scoring very highly ([Figures S6](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#figs6)E–S6G). The final cell type annotations were used to visualize cellular topology in “cell phenotype maps” (CPMs, [Figure 4](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#fig4)F). Collectively, this annotated reference serves as a comprehensive map of the cellular topology of hematopoietic cells in the context of their microenvironment and demonstrates the complementarity of these two atlases to understand the cellular anatomy of human bone marrow.
### EMPs and GMPs localize to a relatively hyperoxygenated arterio-endosteal niche
While multiple niches have been proposed to organize hematopoiesis,[2](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib2),[5](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib5),[35](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib35),[36](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib36) we sought to utilize an unbiased, statistically rigorous approach to answer the question of how hematopoietic and mesenchymal cells organize into cellular neighborhoods (CNs). Indeed, even with interactive visualization of the CODEX images, we noticed discrete patterns of cellular organization, such as peri-endosteal localization of MPOhi early myeloid progenitors (EMPs), and erythroblastic islands composed of macrophages and erythroid precursors ([Figure S7](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#figs7)A).
![](https://ars.els-cdn.com/content/image/1-s2.0-S0092867424004082-figs7.jpg)
Figure S7. CODEX reveals a relatively hyperoxygenated peri-arteriolar/peri-endosteal niche, related to [Figure 5](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#fig5)
(A) CODEX images and associated cell phenotype maps (CPMs) for manually identified peri-endosteal early myeloid cells (top) and erythroblastic islands (bottom).
(B) Boxplot showing each neighborhood ranked by its proximity to manually annotated bone in each sample, and the normalized rank was computed for each comparison, such that a value of 0 means most proximal and 1 means least proximal. ∗ _p_ < 0.05.
(C) CODEX images shown with selected markers and paired neighborhood masks demonstrating the spatial organization of the tissue with respect to both hematopoietic and non-hematopoietic cell types. Neighborhoods with the same annotation (e.g., erythroid 1 [CN13] and erythroid 2 [CN15]) were combined for visualization purposes.
(D) Stacked bar plot of sample frequencies within each neighborhood organized by age. Median linear R value of age effect on neighborhood composition was 0.051 (not significant).
(E) Violin plots showing CLR-normalized HIF1A expression across all cell types.
We therefore performed unsupervised neighborhood analysis as previously described[52](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib52) and identified 15 CNs that we manually annotated based on their enrichment of cell types using hypergeometric tests ([Figure 5](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#fig5)A). Given the challenge of bone detachment during the CODEX protocol, we first confirmed that neighborhoods enriched for endosteal cells were indeed close to manually annotated bone locations ([STAR Methods](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#sec4); [Figure S7](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#figs7)B). We found expected neighborhoods such as the “erythroid” neighborhood (CN13 and CN15) consistent with erythroblastic islands, as well as novel neighborhoods such as one neighborhood of peri-arteriolar [lymphoid cells](https://www.sciencedirect.com/topics/medicine-and-dentistry/lymphoid-cell "Learn more about lymphoid cells from ScienceDirect's AI-generated Topic Pages") ([Figures 5](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#fig5)A and [S7](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#figs7)C). We also found multiple mixed-lineage neighborhoods (CN7 and CN8) with all three hematopoietic lineages (erythroid/myeloid/lymphoid). HSPCs, including more primitive SPINK2+ HSPCs, were found most in mixed-lineage neighborhoods (CN7 and CN8), and SPINK2+ HSPCs, to a lesser degree in myeloid/lymphoid (CN5) and early myeloid/arteriolar (CN4) neighborhoods. Lymphoid progenitors such as CLPs and immature B cells were found predominantly in mixed lineage (CN7 and CN8), plasma cell/arteriolar (CN3), and peri-arteriolar lymphoid (CN2) neighborhoods. Two neighborhoods (CN4 and CN6), enriched for GMP/myeloblasts and early myeloid progenitors (EMPs), were strongly localized along the endosteal [surface](https://www.sciencedirect.com/topics/immunology-and-microbiology/surface-property "Learn more about surface from ScienceDirect's AI-generated Topic Pages") and around arterioles ([Figures 5](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#fig5)A–5C). This was consistent with the endosteal signaling module that we identified using CellChat which had contribution from AECs and VSMCs ([Figure 3](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#fig3)G). We therefore termed them early myeloid/arteriolar (CN4) and early myeloid/endosteal (CN6), based on the enrichment of AECs and endosteal cells, respectively. Notably, the peri-arteriolar lymphoid neighborhood was not enriched for endosteal cells, unlike the early myeloid, vascular/endosteal/MSC, or plasma cell/arteriolar neighborhoods ([Figures 5](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#fig5)A, [S7](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#figs7)B, and S7C). Finally, the vascular/endosteal/MSC-enriched neighborhood was highly enriched for [Schwann cells](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/schwann-cell "Learn more about Schwann cells from ScienceDirect's AI-generated Topic Pages"), suggesting that this neighborhood is innervated. All of the arteriolar and mixed lineage neighborhoods (CN1–4 and CN7–8) were enriched for Adipo- and THY1+ MSCs ([Figure 5](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#fig5)A). The neighborhoods were not overly biased towards one sample or by age ([Figure S7](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#figs7)D). Importantly, this analysis also shows that the concept of the peri-arteriolar and endosteal niches holds up to unbiased statistical measurement of spatial association, and that in line with recent studies using mouse models,[53](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib53) adult human HSPCs do not preferentially occupy an endosteal or peri-arteriolar niche.
![](https://ars.els-cdn.com/content/image/1-s2.0-S0092867424004082-gr5.jpg)
Figure 5. Neighborhood analysis reveals an oxygen-rich arteriolar/endosteal niche for GMP/early [myeloid progenitor cells](https://www.sciencedirect.com/topics/medicine-and-dentistry/myeloid-progenitor-cell "Learn more about myeloid progenitor cells from ScienceDirect's AI-generated Topic Pages")
(A) Heatmap showing enrichment of the cell types present in neighborhoods. Neighborhood names were assigned based on the enriched cell types. _p_ values for enrichment were computed using a hypergeometric test and adjusted for multiple hypothesis testing using the Benjamini-Hochberg method. ∗ _p_ < 0.05.
(B) Cell phenotype masks colored by neighborhood membership were plotted alongside ASMA fluorescent signal (left only).
(C) HIF1A staining pattern across neighborhoods. Top, CODEX image showing the MPO+ early myeloid progenitors not expressing HIF1A, in contrast to mature myeloid elements. Bottom, neighborhood masks which share a legend with (B).
(D) Boxplots showing the normalized HIF1A expression levels in early myeloid progenitors split by neighborhood membership. Data were clipped between 0.1 and 0.95 for improved visualization only. _p_ values were computed using Welch’s one-sided two-sample test comparing cells from each neighborhood with cells in all other neighborhoods and adjusted using the Benjamini-Hochberg method. ∗ _p_ < 0.05; ∗∗∗ _p_ < 0.001.
(E) Violin plots showing [hypoxia](https://www.sciencedirect.com/topics/medicine-and-dentistry/hypoxia "Learn more about hypoxia from ScienceDirect's AI-generated Topic Pages") signature scores computed using AUCell and our scRNA-seq data. _p_ values were comparing AUCell hypoxia scores from each cell type with all other cell types as in (D).
(F) Bubble plot showing correlation and CODEX-CellChat effect size (CCES) between CellChat interaction prediction strength and max CODEX spatial neighborhood co-localization strength of two cell types.
See also [Figures S7](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#figs7) and [S8](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#figs8).
The importance of the endosteal niche has been reported in both healthy and [malignant hematopoiesis](https://www.sciencedirect.com/topics/medicine-and-dentistry/malignant-hematopoiesis "Learn more about malignant hematopoiesis from ScienceDirect's AI-generated Topic Pages").[5](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib5),[54](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib54) We therefore sought to further investigate interactions within this niche using both our CODEX and scRNA-seq atlases. First, we found that arteriolar cells were near the [trabecular bone](https://www.sciencedirect.com/topics/immunology-and-microbiology/trabecular-bone "Learn more about trabecular bone from ScienceDirect's AI-generated Topic Pages") more frequently than random expectation ([Figures 5](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#fig5)B and [S7](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#figs7)B), suggesting that the endosteal niche may actually combine with the arteriolar niche. Second, we found that [HIF1A](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/hif1a "Learn more about HIF1A from ScienceDirect's AI-generated Topic Pages") levels were low in EMPs and GMPs, suggesting that these cells are not experiencing [hypoxia](https://www.sciencedirect.com/topics/medicine-and-dentistry/hypoxia "Learn more about hypoxia from ScienceDirect's AI-generated Topic Pages") ([Figure 5](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#fig5)C), correlated with their localization near oxygen-supplying vessels. In contrast, more mature [myeloid cells](https://www.sciencedirect.com/topics/medicine-and-dentistry/myeloid-cell "Learn more about myeloid cells from ScienceDirect's AI-generated Topic Pages") had high levels of HIF1A ([Figure S7](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#figs7)E). Furthermore, EMPs in non-arteriolar neighborhoods had higher levels of HIF1A, suggesting that the spatial patterns of hypoxia are not due to intrinsic cell type differences ([Figure 5](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#fig5)D). Likewise, we found that EMPs have the lowest hypoxia signature score in our scRNA-seq atlas ([Figure 5](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#fig5)E), lending support to our CODEX findings. Given that the bone marrow is broadly a hypoxic environment, this finding suggests that a relatively oxygenated niche is associated with early myelopoiesis. Overall, our analysis suggests that the peri-arteriolar and endosteal niches are not, as is often discussed in the literature,[2](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib2),[36](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib36) two discrete niches and they may cooperate in some way important for early myelopoiesis.
We next aimed to integrate our findings based on our scRNA-seq and CODEX data. First, we sought to validate specific expression patterns and interactions from our scRNA-seq/CellChat predictions using an additional CODEX panel targeting some of the ligand-receptor pairs ([Table S5](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#mmc5)). We found several instances of CODEX data matching the RNA/CellChat predictions. Specifically, we found that indeed CXCL12 expression was a key discriminating factor between AECs and SECs ([Figure S8](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#figs8)A). We also found that Notch ligands JAG1 and DLL4 were expressed in both VSMCs and AECs. Finding DLL4 expression in VSMCs was surprising but has been previously reported.[55](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib55),[56](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib56) We also observed [FGF2](https://www.sciencedirect.com/topics/medicine-and-dentistry/fibroblast-growth-factor-2 "Learn more about FGF2 from ScienceDirect's AI-generated Topic Pages") expression in AECs and Angipopoietin-1 in MSC projections ([Figures 3](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#fig3)A and [S8](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#figs8)A; [Table S3](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#mmc3)). Next, we aimed to interrogate whether CellChat-predicted interactions were recapitulated by spatial proximity in a dose-dependent fashion. We annotated cell types in the new CODEX images by reference mapping to our normal bone marrow (NBM) CODEX atlas and analyzed four predicted interactions involving mesenchymal and ECs. We confirmed both well-established (VSMC-NOTCH3 and VSMC/AEC-JAG1; MSC-IL-7 and CLP-IL-7R) and previously underappreciated interactions (THY1+ MSC-COL1A1 and monocyte-CD44, erythroid/megakaryocyte-angiopoietin-1, and SEC-TIE2) ([Figure S8](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#figs8)B). We found that CellChat-predicted interactions could be recapitulated by spatial proximity and that there was a dose-dependent proximity of the target cell based on the level of [receptor expression](https://www.sciencedirect.com/topics/medicine-and-dentistry/receptor-expression "Learn more about receptor expression from ScienceDirect's AI-generated Topic Pages") ([Figures S8](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#figs8)B and S8C).
![](https://ars.els-cdn.com/content/image/1-s2.0-S0092867424004082-figs8.jpg)
Figure S8. CODEX analysis of predicted ligand-receptor interactions, related to [Figures 3](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#fig3) and [5](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#fig5)
(A) Selected CODEX images highlighting factors predicted to be expressed by the cell types of interest. Representative cell type or structure examples in the image are marked.
(B) CellChat ligand-receptor predictions (top) where the line thickness is proportional to the CellChat predicted interaction probability and associated CODEX image (bottom) with markers showing the cell types of interest and the ligand/receptor pairs—from left to right (1) CD14+ monocytes and THY1(CD90)/FOXC1+ THY1+ MSCs; (2) AECs and VSMCs can be identified from typical arteriole morphology and ASMA staining was confirmed but not shown here for visualization purposes; (3) SECs and [erythroid cells](https://www.sciencedirect.com/topics/immunology-and-microbiology/erythroid-cell "Learn more about erythroid cells from ScienceDirect's AI-generated Topic Pages"). GYPC stains [erythroid cells](https://www.sciencedirect.com/topics/medicine-and-dentistry/erythroid-cell "Learn more about erythroid cells from ScienceDirect's AI-generated Topic Pages") and ANGPT1+ megakaryocytes had a very distinct morphology; (4) TDT+ IL-7RA+ CLPs and IL-7+ MSCs. MSCs exhibit stromal morphology. CXCL12 was checked to confirm but not shown here for visualization purposes.
(C) CODEX images were segmented, and reciprocal principal component analysis (RPCA) mapped to the original atlas. Target cell types were analyzed for their distance to the nearest instance of the source cell type. Target cells (expressing the receptor) were split by their quartile of receptor expression (0%–25%, 25%–75%, and 75%–100%) and a one-sided Wilcoxon rank-sum test was used to compute _p_ values between the groups. AF-autofluorescence, mono-monocyte, Mk-megakaryocyte.
(D) Schematic of calculation of neighborhood co-localization score (NCLS), CellChat-CODEX Effect Size (CCES), and example of how this could be utilized to prioritize specific interactions. Note that in the example table the different L/R pairs would need to be from different cell types to have different CCES values.
(E) Representative CODEX image of [type 1 collagen](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/type-i-collagen "Learn more about type 1 collagen from ScienceDirect's AI-generated Topic Pages") encasing arterioles, which are marked. The blank area near the bottom is bone.
While it is not possible to test every CellChat prediction, a useful resource could be one enabling prioritization of the CellChat predictions by the respective neighborhood co-localization for a given cell type pair. To this end, we devised a unified CellChat-CODEX effect size (CCES) score for whether a given cell type pair is both co-localized and has a high weight in the CellChat-predicted signaling network ([STAR Methods](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#sec4); [Figures 5](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#fig5)F and [S8](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#figs8)D; [Table S8](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#mmc8)). Osteoblast-VSMC was one of the highest scoring interactions by this metric, which we found to co-localize and form an arterio-endosteal niche associated with lower hypoxia signature, demonstrating the utility of this metric in prioritizing important interactions ([Table S8](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#mmc8)). Arterioles were found to be encased in [type 1 collagen](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/type-i-collagen "Learn more about type 1 collagen from ScienceDirect's AI-generated Topic Pages") ([Figure S8](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#figs8)E), which has the highest expression in osteoblast cells ([Figure 2](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#fig2)B), suggesting one possible mechanism for the observed arterio-endosteal relationship.
### Imaging-based microanatomical structural analysis reveals peri-adipocytic localization of HSPCs
We next studied six types of key microenvironmental structures that are not well captured by conventional cell segmentation methods due to their complex shapes and/or sizes, including adipocytes, arterioles, bone, macrophages and their projections, sinusoids, and CXCL12+ [stroma](https://www.sciencedirect.com/topics/medicine-and-dentistry/stroma "Learn more about stroma from ScienceDirect's AI-generated Topic Pages"). We first employed manual annotation and thresholding to confidently annotate these structures ([Figures S9](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#figs9)A and S9B). Next, using point pattern analysis ([STAR Methods](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#sec4)), we analyzed the distance between cell types or neighborhoods to each of these newly annotated structures.[57](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib57) We used permutation tests to calculate the significance of identified proximity taking into account of the differential abundance of the identified structures ([Figure S9](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#figs9)C). We found that of all the measured structures, the arterioles were closest to bone ([Figures 6](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#fig6)A and 6B, _p_ = 4.2E−6). This supports the finding from neighborhood analysis that the endosteal niche was arteriole rich ([Figure 6](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#fig6)B). In general, we found that the central niche, as defined by not being statistically closer to bone, was characterized by sinusoids, adipocytes, stroma, and macrophages, although adipocytes and sinusoids were further away from bone than stroma or macrophages ([Figures 6](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#fig6)A and 6B). Additionally, structural analysis confirmed that the previously mentioned erythroid niche corresponds to erythroblastic islands, as this niche was very proximal to macrophages, which were not captured in the neighborhood analysis due to their irregular shape and size ([Figure S6](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#figs6)D). The early myeloid neighborhoods were close to both arterioles and bone, providing further evidence of the arterio-endosteal EMP niche ([Figure 6](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#fig6)C).
![](https://ars.els-cdn.com/content/image/1-s2.0-S0092867424004082-figs9.jpg)
Figure S9. Structural analysis annotations and result summary, related to [Figure 6](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#fig6)
(A) Paired CODEX images and overlaid masks with representative examples of the structural masks compared with the fluorescent images. The sinusoid and adipocyte annotations are shown in the same region of interest.
(B) Table showing annotation methods for each structure. Manual annotation and thresholding both were performed in QuPath, with thresholds determined on a per-image basis across the entire image. Sinusoids and adipocytes were enumerated in specific ROIs, such that an entire ROI was annotated for structural analysis. Number of annotated structures is listed for each structure type.
(C) Chord diagram showing only significant interactions (_p_ < 0.05, Stouffer’s method for meta-analysis of permutation test _p_ values across all 12 samples), chord thickness is proportional to the negative log10 of the median normalized rank + 1E−12.
![](https://ars.els-cdn.com/content/image/1-s2.0-S0092867424004082-gr6.jpg)
Figure 6. Comprehensive structural analysis of bone marrow microenvironment uncovers adipocytic localization of lineage-unspecified HSPCs
(A) Heatmap showing the normalized rank proximity from each microenvironmental structure to each structure, using a Poisson point process model (_n_ = 12). Each structure was ranked by its proximity to each other structure in each sample, and the rank was normalized such that a value of 0 (red) means most proximal and 1 (blue) means least proximal.
(B) Data-driven illustration of structural proximity in normal bone marrow. Edge lengths were scaled to the median normalized rank proximity of each structure, and the physical median distance across 12 samples was labeled above each line. The only significant _p_ value is highlighted, which was computed using Stouffer’s method to aggregate per-sample permutation test _p_ values.
(C and D) Heatmap showing the normalized rank proximity of each neighborhood (C) and cell type to each type of microenvironmental structure (D). Data are from all 12 samples, except when a cell type or neighborhood was too rare to rank in a sample (<10 in at least half of samples, indicated by gray color or excluded from heatmap).
(E) Boxplot showing the normalized rank proximity of SPINK2+ HSPCs (solid boxes) and HSPCs (dashed boxes) to each microenvironmental structure. _p_ values were computed by Stouffer’s method as in (B) comparing observed versus random distance of each HSPC subset to each structure. ∗ _p_ < 0.05; ∗∗∗ _p_ < 0.001, _n_ = 12.
(F) Representative CODEX images and associated cell phenotype maps from two samples showing the peri-adipocytic localization of both HSPCs and SPINK2+ HSPCs.
(G) Cartoon showing a proposed model of human [myelopoiesis](https://www.sciencedirect.com/topics/medicine-and-dentistry/myelopoiesis "Learn more about myelopoiesis from ScienceDirect's AI-generated Topic Pages") spatial [dynamics](https://www.sciencedirect.com/topics/immunology-and-microbiology/dynamics "Learn more about dynamics from ScienceDirect's AI-generated Topic Pages").
See also [Figure S9](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#figs9).
Our results also validated our cell type annotations, as cell types co-localized with expected structures such as Adipo-MSCs-stroma (_p_ = 3.8E−133), macrophages-macrophages (_p_ = 5.3E−156), endosteal-bone (_p_ = 6.8E−147), and SEC-sinusoids (_p_ = 9.8E−179) ([Figure 6](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#fig6)D). Interestingly, we found that all MSCs were close to the arterio-endosteal niche, but Adipo-MSCs were found near sinusoids while THY1+ MSCs were not ([Figure 6](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#fig6)D). This suggests that Adipo-MSCs may be analogous to previously described HSC-supportive peri-sinusoidal MSCs.[41](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib41),[58](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib58)
We also discovered spatial patterns not apparent in our cell segmentation-based analysis. Both Lin− SPINK2+ and SPINK2− CD34+ HSPCs were found to be more proximal to adipocytes than expected by chance (_p_ = 1.77E-6 and 0.0198 respectively, [Figures 6](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#fig6)D and 6E). Manual inspection of CODEX images revealed that both types of HSPCs were in frequent contact with adipocytes ([Figure 6](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#fig6)F; [Table S7](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#mmc7)). HSPCs, but not more primitive SPINK2+ HSPCs, were also found to be close to macrophages (_p_ = 5.23E−11 and n.s., respectively, [Figure 6](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#fig6)E). Interestingly, GMP/myeloblast cells as well as EMPs were also close to adipocytes, although these cells were significantly closer to bone while HSPCs were not ([Table S7](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#mmc7)). Mature myeloid cells, by contrast, were closest to sinusoids and not arterioles or bone, suggesting that maturation occurs in a central, peri-sinusoidal niche (_p_ = 6.71E−68, [Figure 6](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#fig6)D). Collectively, these data suggest a spatial restriction of myeloid development, where the earliest myeloid progenitors are specified in an adipocytic niche from HSPCs, migrate to a relatively hyperoxygenated arterio-endosteal early myeloid niche, and finally migrate to and mature near sinusoids that they may utilize to egress from the marrow into the bloodstream ([Figure 6](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#fig6)G).
### Unsupervised reference mapping using healthy bone marrow CODEX atlas reveals AML stromal expansion and novel AML-MSC-enriched neighborhoods
Understanding the spatial organization of the [bone marrow microenvironment](https://www.sciencedirect.com/topics/immunology-and-microbiology/bone-marrow-microenvironment "Learn more about bone marrow microenvironment from ScienceDirect's AI-generated Topic Pages") has significant implications for healthy hematopoiesis as well as disease states. For instance, [AML](https://www.sciencedirect.com/topics/medicine-and-dentistry/acute-myeloid-leukemia "Learn more about AML from ScienceDirect's AI-generated Topic Pages") has been described previously to interact with the endosteal niche to achieve chemoresistance.[54](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib54) We applied our bone marrow CODEX panel to explore tumor evolution and microenvironmental changes using our healthy atlas as a reference. We profiled three diagnostic (Dx) and two paired post-therapy (PostTx) AML patient samples from [iliac crest](https://www.sciencedirect.com/topics/immunology-and-microbiology/iliac-crest "Learn more about iliac crest from ScienceDirect's AI-generated Topic Pages") biopsies treated with [venetoclax](https://www.sciencedirect.com/topics/medicine-and-dentistry/venetoclax "Learn more about venetoclax from ScienceDirect's AI-generated Topic Pages") plus a [hypomethylating agent](https://www.sciencedirect.com/topics/medicine-and-dentistry/hypomethylating-agent "Learn more about hypomethylating agent from ScienceDirect's AI-generated Topic Pages") (Ven/HMA) ([Table S1](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#mmc1)). As controls, we also included three negative lymphoma staging bone marrow biopsies (NSM) from the [iliac crest](https://www.sciencedirect.com/topics/medicine-and-dentistry/iliac-crest "Learn more about iliac crest from ScienceDirect's AI-generated Topic Pages") of 38- to 62-year-old patients. We selected AML patients with a detected [NPM1](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/npm1 "Learn more about NPM1 from ScienceDirect's AI-generated Topic Pages") mutation (_NPM1c_ W288∗fs) due to availability of a mutant-specific antibody[59](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib59),[60](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib60) so that we could identify putative leukemic blasts. We employed reciprocal [principal component analysis](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/principal-component-analysis "Learn more about principal component analysis from ScienceDirect's AI-generated Topic Pages") (RPCA) reference mapping to classify cells in the AML and NSM samples to their closest counterparts in our healthy atlas ([Figure 7](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#fig7)A; [STAR Methods](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#sec4)). As expected, we found a significant increase in the fraction of myeloid cells, excluding mature myeloid cells, in Dx AML compared with NSM (47.6% versus 23.1% across samples; _p_ < 2.2E−16 by proportion test) ([Figure 7](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#fig7)B). This demonstrates the utility of our atlas to rapidly annotate hundreds of thousands of cells in both healthy and diseased contexts. Projecting these cell labels onto the segmentation masks revealed grossly appreciable architectural changes in Dx and PostTx AML samples compared with NSM samples, such as a near total loss of adipocytes in the leukemic setting, and incomplete hematopoietic recovery 30 days PostTx in the presence of residual blasts ([Figure 7](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#fig7)C). A more detailed look at the cellular landscape of AML also revealed marked stromal expansion, with 2- to 3-fold higher relative frequency of Adipo- and THY1+ MSCs in AML compared with NSM samples ([Figure 7](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#fig7)D). Next, we aimed to identify leukemic blasts using a classifier based on mutant NPM1 and CD141 staining ([Figures S10](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#figs10)A and S10B; [STAR Methods](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#sec4)). Our classified blast percentages correlated with the _NPM1c_ variant allele frequencies (R = 0.62; [Figure S10](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#figs10)C). Using this approach, we identified leukemic populations including a GATA1+ NPM1c+ double positive population in the PostTx sample of AML3 and individual _NPM1_ mutant PostTx cells ([Figure 7](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#fig7)E). We observed a ∼2-fold increase in [GATA1](https://www.sciencedirect.com/topics/medicine-and-dentistry/transcription-factor-gata-1 "Learn more about GATA1 from ScienceDirect's AI-generated Topic Pages") expression in _NPM1_ mutant blasts across both PostTx samples, suggesting that these residual cells exhibited lineage plasticity ([Figure 7](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#fig7)F).
![](https://ars.els-cdn.com/content/image/1-s2.0-S0092867424004082-gr7.jpg)
Figure 7. Unsupervised single-cell mapping of AML reveals stromal expansion and MSC-enriched AML-specific neighborhoods
(A) Schematic showing our unsupervised label transfer computational approach.
(B) Bar plot showing the myeloid cell type frequencies in each sample, excluding mature myeloid cells.
(C) Cell phenotype maps (CPMs) shown for representative sample images. Masks are colored by cell type and share a legend with (B). Dx, diagnostic; PostTx, post-therapy.
(D) Bar plot showing per sample frequency of MSCs in AML versus NSM. MSC frequency was calculated as the proportion of cells in each sample which were annotated as Adipo- or THY1+ MSCs. The distribution of MSC subtype frequencies in AML samples (AML_Dx and PostTx combined) was compared with that of NSM samples using Welch’s two-sample two-sided test.
(E) CODEX image showing clustering of rare residual _NPM1_ mutant blasts, which also stained positive for GATA1, juxtaposed with the CPM highlighting the segmented blasts.
(F) Violin plot showing GATA1 protein levels in the NPM1 mutant blasts from diagnosis and post-therapy samples. _p_ value was computed using Welch’s two-sample two-sided test.
(G) Heatmap showing the neighborhood cell enrichments (AML: _n_ = 175,365 cells, NSM: _n_ = 152,198 cells). _p_ value for enrichment was computed using hypergeometric test and adjusted for multiple hypothesis testing using the Benjamini-Hochberg method. ∗ _p_ < 0.05.
See also [Figures S10](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#figs10) and [S11](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#figs11).
![](https://ars.els-cdn.com/content/image/1-s2.0-S0092867424004082-figs10.jpg)
Figure S10. CODEX enables classification of _NPM1_ mutant blasts and comparison with negative staging marrow from iliac crest and femoral head samples, related to [Figure 7](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#fig7)
(A) Violin plot of CODEX CLR-normalized protein expression showing the non-specific expression of the mutant-specific NPM1C antibody in negative staging marrow samples.
(B) CODEX images showing that CD141, but not early myeloid or erythroid markers (MPO and GATA1), almost completely co-stains with mutant-specific NPM1C in a representative negative staging marrow sample. Exclusion of these CD141+ cells enables the use of the antibody to identify true [mutant cells](https://www.sciencedirect.com/topics/medicine-and-dentistry/cell-mutant "Learn more about mutant cells from ScienceDirect's AI-generated Topic Pages").
(C) Table showing the detected NPM1c mutant percentage in each sample compared with the clinical variant allele frequency for NPM1c. MRD, minimal residual disease.
(D) Heatmap with relative enrichment values for each cell type in the neighborhoods identified using CODEX. Neighborhoods were manually annotated based on the cell types that are enriched. _p_ value for enrichment was computed using the hypergeometric test and adjusted for multiple hypothesis testing using the Benjamini-Hochberg method. ∗ _p_ < 0.05. EB-erythroblast, NC monocyte-non-classical monocyte, early myeloid-early myeloid progenitor, int myeloid-intermediate myeloid.
(E) Plot showing the Pearson correlation of each neighborhood in the femoral head NBM atlas presented in [Figure 5](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#fig5) to the matching iliac crest NSM neighborhoods based on the relative enrichment of each cell type in each neighborhood, the R value is plotted on the y axis.
(F) Representative CODEX images of HSPCs contacting adipocytes (labeled as A) in adult and pediatric samples.
(G) Representative images of manually identified MSC subsets in iliac crest samples, demonstrating existence of these subsets across multiple anatomic sites.
We next performed neighborhood analyses on AML Dx (_n_ = 3), PostTx (_n_ = 2), and NSM (_n_ = 3) samples separately and identified 15 CNs in each ([Figures 7](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#fig7)G and [S10](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#figs10)D). We first analyzed the composition of the negative staging marrow samples. These samples were nearly identical in neighborhood composition to the [femoral head](https://www.sciencedirect.com/topics/medicine-and-dentistry/femoral-head "Learn more about femoral head from ScienceDirect's AI-generated Topic Pages") samples used to generate our atlas, including early myeloid/arterial/endosteal neighborhoods (NSM-early myeloid/arteriolar, NSM-early myeloid/endosteal, R = 0.83 and 0.74, respectively). The correlation of the cell type enrichment profiles across neighborhoods was generally high, which confirmed the generalizability of the femoral head data to bone marrow specimens from iliac crest (median neighborhood R = 0.71; [Figures S10](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#figs10)D and S10E). We also observed CD34+ cells contacting adipocytes in both adult and [pediatric](https://www.sciencedirect.com/topics/medicine-and-dentistry/pediatrics "Learn more about pediatric from ScienceDirect's AI-generated Topic Pages") NSM samples ([Figure S10](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#figs10)F). Lastly, we observed all four MSC subsets in the NSM samples, suggesting that these MSC subsets exist across multiple anatomic sites ([Figure S10](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#figs10)G). In general, this analysis lends credence to the use of NSM samples as controls, as well as the generalizability of femoral head data.
In the Dx AML specimens, we found four blast-enriched neighborhoods—Dx-CN3/CN5/CN9/CN12 ([Figures 7](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#fig7)G and [S11](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#figs11)A), two of which were enriched for MSC populations. Notably, THY1+ MSCs were significantly enriched exclusively in these blast-enriched neighborhoods ([Figure 7](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#fig7)G). No neighborhoods identified were exclusive to just one sample ([Figure S11](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#figs11)B). We observed neighborhoods similar to healthy samples as well, such as a plasma cell-AEC neighborhood (Dx-CN8) and lymphoid neighborhoods (Dx-CN6/CN14/CN15) ([Figures 5](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#fig5)A and [7](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#fig7)G). Dx-CN9, a blast-enriched neighborhood, was also enriched for AECs, endosteal cells, and Adipo- and THY1+ MSCs, very similar to the healthy EMP niche. Interestingly, we found that both PostTx blast-enriched neighborhoods (PostTx-CN4/CN14) resembled Dx-CN9 and NSM-early myeloid/arteriolar/endosteal neighborhoods, the arterio-endosteal early myeloid niche ([Figure S11](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#figs11)C). Notably, PostTx-CN4 and Dx-CN9/CN12, but not PostTx-CN14, were statistically closer to bone as determined by point pattern analysis ([Figure S11](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#figs11)D). We also observed that recovering Post-Tx GMP/EMP neighborhoods did not contain AECs/endosteal cells (PostTx-CN8), which could reflect a difference between steady-state and emergency myelopoiesis, as previously reported[61](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib61) ([Figure 7](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#fig7)G). In general, these data hint at conserved spatial patterns marking hematopoietic recovery.
![](https://ars.els-cdn.com/content/image/1-s2.0-S0092867424004082-figs11.jpg)
Figure S11. CODEX enables neighborhood analysis of _NPM1_ mutant blasts, related to [Figure 7](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#fig7)
(A) CODEX fluorescent image of a diagnostic AML sample showing MSCs and _NPM1_ mutant blasts. White arrows point to FOXC1+ CD271+ MSCs.
(B) Barplots showing the relative sample composition of each neighborhood, separated by time point.
(C) CODEX images of post-therapy NPM1 mutant blasts in the peri-arteriolar (left, arteriole border marked by ASMA) and peri-endosteal neighborhoods (right, endosteal border marked by CD56), with the contours of annotated blasts colored in purple.
(D) Boxplots of the normalized rank of the bone proximity computed as in [Figure 6](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#fig6), where we calculated normalized rank proximity using the Poisson point pattern model from the neighborhoods to manually annotated bone. Data are from all 5 AML samples split by time point. Each structure was ranked by its proximity to each other structure in each sample, and the rank was normalized such that a value of 0 means most proximal and 1 means least proximal. _p_ values were calculated by a permutation test for each sample and aggregated using Stouffer’s method, diagnostic neighborhoods enriched for endosteal cells: Dx-CN02, _p_ = 1, Dx-CN03, _p_ = 0.8, Dx-CN09, _p_ = 0.019, Dx-CN12, _p_ = 2.3E−24. In the post-therapy neighborhoods enriched for endosteal cells: PostTx-CN04, _p_ = 2.9E−11, PostTx-CN14, _p_ = 1.
(E–G) HIF1A, BCL2, and complex IV CODEX CLR expression levels in _NPM1_ mutant blasts. A two-sided Welch’s two-sample test was performed between all post-therapy samples against all diagnostic samples for each protein.
Lastly, we examined the intrinsic properties of _NPM1_ mutant blasts. Both Dx and PostTx _NPM1_ mutant blasts had low HIF1A levels consistent with GMPs and early myeloid cells ([Figure S11](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#figs11)E). Comparing paired samples, PostTx cells had slightly decreased BCL2 and increased mitochondrial complex IV expression after therapy (_p_ < 2.2E−16 for both, [Figures S11](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#figs11)F and S11G). This could reflect decreased reliance of these PostTx cells on the BCL2 pathway and a compensatory increase in mitochondrial mass to circumvent venetoclax-mediated suppression of BCL2 and [oxidative phosphorylation](https://www.sciencedirect.com/topics/immunology-and-microbiology/oxidative-phosphorylation "Learn more about oxidative phosphorylation from ScienceDirect's AI-generated Topic Pages").[62](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib62),[63](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib63) These data provide _in vivo_ evidence of AML-MSC co-localization and suggest that the arterio-endosteal early myeloid neighborhood that we identified may play a role in therapy resistance, although more samples will need to be studied to draw more general conclusions.
## Discussion
We report the most comprehensive [healthy human](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/normal-human "Learn more about healthy human from ScienceDirect's AI-generated Topic Pages") bone marrow reference to date, covering both hematopoietic and non-hematopoietic cell types and their spatial organization. We found that the bone marrow microenvironment contains at least six mesenchymal subsets, two endothelial cell subsets, and one [vascular smooth muscle](https://www.sciencedirect.com/topics/immunology-and-microbiology/vascular-smooth-muscle "Learn more about vascular smooth muscle from ScienceDirect's AI-generated Topic Pages") population. Our data clearly demonstrate the limitations of bone marrow aspirates as a sample source, where Adipo-MSCs are heavily favored and subsets like Fibro-MSCs are not captured at all.[13](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib13),[16](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib16),[17](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib17),[18](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib18) We identified markers to sort [MSC subpopulations](https://www.sciencedirect.com/topics/medicine-and-dentistry/cell-subpopulation "Learn more about MSC subpopulations from ScienceDirect's AI-generated Topic Pages") and pinpointed the rare Podoplanin+ Fibro-MSCs as the closest to the ISCT-defined mesenchymal stem cell. By contrast, Adipo- and THY1+ MSCs were found to be the most interactive with hematopoietic cells and not part of the endosteal signaling module. Adipo-, THY1+, Osteo-, Fibro-, and APOD+ MSCs also had significantly different profiles of support factor expression. These findings collectively demonstrate the utility of our atlas for understanding differential functions of MSC subsets.
We elucidated the spatial organization of the human bone marrow on a systematic, single-cell level. While recent studies have assessed human bone marrow organization using 7-color [immunofluorescence](https://www.sciencedirect.com/topics/medicine-and-dentistry/immunofluorescence "Learn more about immunofluorescence from ScienceDirect's AI-generated Topic Pages"),[64](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib64) our work more deeply dissected the [cellular differentiation](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/cellular-differentiation "Learn more about cellular differentiation from ScienceDirect's AI-generated Topic Pages") continua of both hematopoietic and non-hematopoietic lineages by virtue of our 53-antibody panel. Our healthy atlas captured the spatial relationships of 32 cell types with respect to each other and to many manually annotated, difficult to segment cell types/structures. This atlas links mesenchymal heterogeneity on the transcriptional level to discrete spatial phenotypes _in situ_ , for instance in the case of more central Adipo-/THY1+ versus endosteal Osteo-/Fibro-MSCs. Our spatial findings were generalizable across anatomic sites as well, as our observations in [femoral head](https://www.sciencedirect.com/topics/immunology-and-microbiology/femoral-head "Learn more about femoral head from ScienceDirect's AI-generated Topic Pages") bone marrow were closely recapitulated in negative lymphoma staging marrows from [iliac crest](https://www.sciencedirect.com/topics/immunology-and-microbiology/iliac-crest "Learn more about iliac crest from ScienceDirect's AI-generated Topic Pages"). Our data will serve as a powerful tool for uncovering bone marrow niche biology in health and disease by enabling rapid reference mapping and label transfer to study how these otherwise conserved spatial patterns become dysregulated or co-opted, as we show in the AML vignette.
Our findings also highlight the importance of measuring all bone marrow cell types and microanatomical structures simultaneously. For instance, we extend prior findings[65](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib65) of an HSPC-adipocyte spatial relationship by showing that the relationship is strongest for more primitive[49](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib49),[50](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib50) SPINK2+ HSPCs and is stronger than any spatial association between SPINK2+ HSPCs and other structures. This finding is of particular importance as adipocytic density increases as humans age and local adipocytic density is markedly reduced in the context of [acute leukemia](https://www.sciencedirect.com/topics/medicine-and-dentistry/acute-leukemia "Learn more about acute leukemia from ScienceDirect's AI-generated Topic Pages"). Bone marrow [adipogenesis](https://www.sciencedirect.com/topics/medicine-and-dentistry/adipogenesis "Learn more about adipogenesis from ScienceDirect's AI-generated Topic Pages") has been proposed as a negative regulator of hematopoiesis in mice,[66](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib66) but a growing body of evidence has proposed adipocytes as both stemness-maintaining and supportive of stress hematopoiesis,[67](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib67),[68](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib68),[69](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib69),[70](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib70) and so whether this interaction represents a hematopoiesis-supportive or suppressive interaction in humans warrants further investigation. We surprisingly did not find an endosteal or perivascular preference for human HSPCs in [femoral head](https://www.sciencedirect.com/topics/medicine-and-dentistry/femoral-head "Learn more about femoral head from ScienceDirect's AI-generated Topic Pages") or [iliac crest](https://www.sciencedirect.com/topics/medicine-and-dentistry/iliac-crest "Learn more about iliac crest from ScienceDirect's AI-generated Topic Pages"). These data do not mean that sinusoid/osteoblast-HSC communication is unimportant, only that human HSPCs are not closer to sinusoids than expected by chance in older adult samples. Similar profiling efforts across the spectrum of aging will help reveal the relative contribution of these structures at different developmental stages.
Our data also have important implications for use and interpretation of mouse models. While mice and humans share mesenchymal subsets, humans have unique MSC populations like THY1+ and APOD+ MSCs that warrant further study. Fibro-MSCs in humans closely resemble early mesenchymal progenitors in mice, but [Podoplanin](https://www.sciencedirect.com/topics/medicine-and-dentistry/podoplanin "Learn more about Podoplanin from ScienceDirect's AI-generated Topic Pages") has not been reported as a key early MSC marker in mice. Spatially, we found significant divergence from mouse biology. For instance, [GMPs](https://www.sciencedirect.com/topics/immunology-and-microbiology/monocyte "Learn more about GMPs from ScienceDirect's AI-generated Topic Pages") have been reported to be randomly scattered in the bone marrow.[61](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib61) To our knowledge, the arterio-endosteal early myeloid niche has not been reported in mice. Perhaps most importantly, mouse bone marrow adiposity is several orders of magnitude below its human counterpart,[71](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib71) suggesting that murine samples may be representative of a much younger bone marrow microenvironment even in aged mice and therefore could explain why the HSPC-adipocyte spatial relationship is not observed in mice.[72](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib72)
We envision that this resource will have value to contextualize diseased bone marrow, as illustrated in our AML case study. While the small [sample size](https://www.sciencedirect.com/topics/immunology-and-microbiology/sample-size "Learn more about sample size from ScienceDirect's AI-generated Topic Pages") and specific use of one treatment approach in this demonstrative vignette precludes broad generalization of our findings, we show that it is both feasible and informative to use CODEX to map blast-stromal neighborhoods to study the bone marrow microenvironment. The preliminary findings that Adipo- and THY1+ MSCs were expanded in AML samples and enriched in leukemic neighborhoods provide further rationale to study blast-stromal interactions in AML. Furthermore, our ability to detect and map leukemic minimal residual disease (MRD) and rare PostTx cells showed an erythroid GATA1+ enrichment in resistant blasts, highlighting the translational potential of bone marrow CODEX to detect and find prognostic spatial biomarkers of residual disease.
### Limitations of the study
Our [study cohort](https://www.sciencedirect.com/topics/medicine-and-dentistry/cohort-analysis "Learn more about study cohort from ScienceDirect's AI-generated Topic Pages") comprises a representative sample of 12 individuals aged 52–80 years, approximating a relatively “old” microenvironment. Therefore, significant future work remains to determine the temporo-spatial landscape of hematopoiesis in humans. Additionally, adipocytes are a major component of the bone marrow microenvironment whose potential importance was highlighted in our spatial analysis. However, due to their size and fragility, we were not able to capture them in our scRNA-seq atlas. Future efforts to sequence bone marrow [adipose tissue](https://www.sciencedirect.com/topics/immunology-and-microbiology/adipose-tissue "Learn more about adipose tissue from ScienceDirect's AI-generated Topic Pages") would help resolve the mechanisms underlying HSPC-adipocyte co-localization.
Image analysis in the context of mapping complex cellular topologies, so-called “neighborhoods,” is in its infancy. Tools do not currently exist to consider both higher-order microenvironmental structures (e.g., [arterioles](https://www.sciencedirect.com/topics/immunology-and-microbiology/arteriole "Learn more about arterioles from ScienceDirect's AI-generated Topic Pages") or lymphoid follicles) and cell types for collective neighborhood analysis. Furthermore, 2D imaging is limited to distances that can be measured by observable analytes within a 2D plane within a complex 3D structure. This is particularly problematic for very rare cell types like HSCs, which could have supportive structures directly above or below the captured plane but would not have a spatial association captured. As 3D multiplexed imaging and corresponding analytical approaches develop, we will gain a more comprehensive quantitative view of the human bone marrow.
## STAR★Methods
### Key resources table
REAGENT or RESOURCE| SOURCE| IDENTIFIER  
---|---|---  
**Antibodies**  
Rabbit polyclonal anti-alpha smooth muscle Actin| Abcam| Cat# ab5694; RRID:  
Mouse monoclonal anti-CD19 (clone MRQ-36) custom carrier-free| Cell Marque| Cat# CD19;RRID:  
Anti-Hu CD4 (AKYP0048)-BX003-Alexa Fluor™ 647 for PhenoCycler| Akoya Biosciences| Cat# 4350018; RRID:  
Mouse monoclonal anti-CD71 (clone MRQ-48) custom carrier-free| Cell Marque| Cat# CD71; RRID:  
Anti-Hu CD44 (AKYP0073)-BX005-Atto 550 for PhenoCycler| Akoya Biosciences| Cat# 4450041; RRID:  
Rabbit recombinant monoclonal anti-CD271 (clone EP1039Y)| Abcam| Cat# ab256584;RRID:  
Mouse monoclonal anti-Vimentin (clone RV202)| Novus Biologicals| Cat# NBP1-97672;RRID:  
Goat polyclonal anti-CD117| R & D Systems| Cat# AF1356; RRID:  
Rabbit recombinant monoclonal anti-CD61| Abcam| Cat# ab240214; RRID:  
Rabbit polyclonal anti-CD90| Cell Signaling Technology| Cat# 16836; RRID:  
Anti-Hu CD68(AKYP0050)-BX015-Alexa Fluor™ 647 for PhenoCycler| Akoya Biosciences| Cat# 4550113; RRID:  
Mouse monoclonal anti-CD45RA (clone H100)| Biolegend| Cat# 304102; RRID:  
Rabbit polyclonal anti-GYPC| Thermo Fisher Scientific| Cat# PA5-80680; RRID:  
Rabbit recombinant monoclonal anti-CD49F| Abcam| Cat# ab240252; RRID:  
Rabbit recombinant monoclonal anti-CXCR4 (clone D4Z7W)| Cell Signaling Technology| Cat# 90927; RRID:  
Anti-Hu CD45 (AKYP0074)-BX021-Alexa Fluor™ 647 for PhenoCycler| Akoya Biosciences| Cat# 4450042; RRID:  
Mouse monoclonal anti-CD15 (clone SSEA-1)| Biolegend| Cat# 301902; RRID:  
Rabbit polyclonal anti-SPINK2| Novus Biologicals| Cat# NBP2-98568; RRID:  
Rabbit recombinant monoclonal anti-CD11C (clone EP1347Y)| Abcam| Cat# ab216655; RRID:  
Rabbit recombinant monoclonal anti-FOXC1 (clone EPR20678)| Abcam| Cat# ab230157; RRID:  
Anti-Hu CD8(AKYP0028)-BX026-Atto 550 for PhenoCycler| Akoya Biosciences| Cat# 4250012; RRID:  
Mouse monoclonal anti-Adiponectin (clone OTI1H1)| Novus Biologicals| Cat# NBP2-71422; RRID:  
Mouse monoclonal anti-VE-cadherin (clone 16B1)| Thermo Fisher Scientific| Cat# 14-1449-82; RRID:  
Rabbit recombinant monoclonal anti-CD123 (clone IL3RA/2947R)| Novus Biologicals| Cat# NBP2-79851; RRID:  
Rabbit recombinant monoclonal anti-CD14 (clone SP192)| Abcam| Cat# ab230903; RRID:  
Mouse monoclonal anti-OxPhos complex IV subunit IV (clone 10G8D12C12)| Thermo Fisher Scientific| Cat# A-21347; RRID:  
Rabbit recombinant monoclonal anti-PLP1 (clone EPR23504-106)| Abcam| Cat# ab275751; RRID:  
Anti-Hu HLA-DR(AKYP0063)-BX033-Alexa Fluor™ 647 for PhenoCycler| Akoya Biosciences| Cat# 4550118; RRID:  
Rabbit recombinant monoclonal anti- Mast Cell Tryptase (clone EPR8476)| Abcam| Cat# ab227572; RRID:  
Mouse monoclonal anti-CD34 (clone QBEnd/10)| Abcam| Cat# ab213054; RRID:  
Mouse monoclonal anti-CD10 (clone MME/1892)| Abcam| Cat# ab237866; RRID:  
Rabbit recombinant monoclonal anti-Pax5 (clone D7H5X)| Cell Signaling Technology| Cat# 93009; RRID:  
Rabbit recombinant monoclonal anti-CD11b (clone EPR1344)| Abcam| Cat# ab209970; RRID:  
Rabbit recombinant monoclonal anti-VCAM1 (clone EPR5047)| Abcam| Cat# ab271899; RRID:  
Rabbit recombinant monoclonal anti-TGF beta 1 (clone EPR21143)| Abcam| Cat# ab229856; RRID:  
Rabbit recombinant monoclonal anti-CD146 (clone EPR3208)| Abcam| Cat# ab210072; RRID:  
Anti-Hu CD3e(AKYP0062)-BX045-Alexa Fluor™ 647 for PhenoCycler| Akoya Biosciences| Cat# 4550119; RRID:  
Rabbit recombinant monoclonal anti-GATA1 (clone D52H6)| Cell Signaling Technology| Cat# 40977; RRID:  
Anti-Hu/Mu Ki67(AKYP0052)-BX047-Atto 550 for PhenoCycler| Akoya Biosciences| Cat# 4250019; RRID:)  
Rabbit recombinant monoclonal anti-CD138 (clone EPR6454)| Abcam| Cat# ab216458; RRID:  
Mouse monoclonal anti-CXCL12 (clone 79018)| R & D Systems| Cat# MAB350; RRID:  
Mouse monoclonal anti-CD33 (clone 996810)| R & D Systems| Cat# MAB11371; RRID:  
Rabbit monoclonal anti-CD56 (clone MRQ-42) custom carrier-free| Cell Marque| Cat# CD56; RRID:  
Rabbit recombinant monoclonal Anti-Sodium Potassium ATPase (clone EP1845Y)| Abcam| Cat# ab167390; RRID:  
Anti-Hu HIF1A(AKYP0100)-BX062-Alexa Fluor™ 647 for PhenoCycler| Akoya Biosciences| Cat# 4550069; RRID:  
Anti-Hu CD163(AKYP0114)-BX069-Atto 550 for PhenoCycler| Akoya Biosciences| Cat# 4250079; RRID:  
Anti-Hu Bcl-2(AKYP0120)-BX085-Atto 550 for PhenoCycler| Akoya Biosciences| Cat# 4250098; RRID:  
Anti-Hu CD141(AKYP0124)-BX087-Atto 550 for PhenoCycler| Akoya Biosciences| Cat# 4250097; RRID:  
Anti-Hu CD38(AKYP0110)-BX089-Atto 550 for PhenoCycler| Akoya Biosciences| Cat# 4250080; RRID:  
Anti-Hu CD79a(AKYP0109)-BX090-Alexa Fluor™ 750 for PhenoCycler| Akoya Biosciences| Cat# 4450078; RRID:  
Anti-Hu b-Catenin1(AKYP0068)-BX096-Atto 550 for PhenoCycler| Akoya Biosciences| Cat# 4250091; RRID:  
Anti-Hu MPO(AKYP0113)-BX098-Atto 550 for PhenoCycler| Akoya Biosciences| Cat# 4250083; RRID:  
Anti-Hu Podoplanin(AKYP0007)-BX121-Atto 550 for PhenoCycler| Akoya Biosciences| Cat# 4250094; RRID:  
Rabbit polyclonal anti-NPM1 (mutant)| Thermo Fisher Scientific| Cat# PA1-46356; RRID:  
Rabbit monoclonal anti-TDT (clone EP266) custom carrier-free| Cell Marque| Cat# TDT; RRID:  
Rabbit recombinant monoclonal Anti-COL1a (clone E8F4L)| Cell Signaling Technology| Cat# 81375; RRID:  
Rabbit monoclonal anti-JAG1 (clone D4Y1R) custom carrier-free| Cell Signaling Technology| Cat # Jagged1; RRID:  
Rabbit polyclonal anti-NOTCH3| Abcam| Cat# ab23426; RRID:  
Rat monoclonal anti-DLL4| R & D Systems| Cat# MAB1389; RRID:  
Rabbit monoclonal anti-FGF2 (clone E5Y6M) custom carrier-free| Cell Signaling Technology| Cat# Basic FGF; RRID:  
Mouse monoclonal anti-IL7 (clone IL7/4012)| Thermo Fisher Scientific| Cat# 3574-MSM2-P0; RRID:  
Rabbit recombinant monoclonal anti-IL7R (clone EPR23747-333)| Abcam| Cat# ab282011; RRID:  
Goat polyclonal anti-ANGPT1| Thermo Fisher Scientific| Cat# PA5-46968; RRID:  
Rabbit polyclonal anti-TIE2| Novus Biologicals| Cat# NBP1-69753; RRID:  
BD OptiBuild™ BUV395 Mouse Anti-Human CD61 (Clone RUU-PL7F12)| BD Biosciences| Cat# 745717; RRID:  
Brilliant Violet 510™ anti-human CD19 (Clone HIB19)| Biolegend| Cat# 302241; RRID:  
Brilliant Violet 605™ anti-human CD14 (Clone 63D3)| Biolegend| Cat# 367125; RRID:  
Brilliant Violet 650™ anti-human CD33 (Clone WM53)| Biolegend| Cat# 303429; RRID:  
Brilliant Violet 785™ anti-human CD38 (Clone HIT2)| Biolegend| Cat# 303530; RRID:  
FITC anti-human CD15 (Clone MMA)| Biolegend| Cat# 394705; RRID:  
PerCP/Cyanine5.5 anti-human CD235ab (Clone HIR2)| Biolegend| Cat# 306613; RRID:  
PE anti-human CD66a/c/e (Clone ASL-32)| Biolegend| Cat# 342303; RRID:  
APC/Cyanine7 anti-human CD45 (Clone 2D1)| Biolegend| Cat# 368516; RRID:  
Alexa Fluor® 647 anti-human CD163 (Clone GHI/61)| Biolegend| Cat# 333619; RRID:  
PE/Cyanine7 anti-human CD34 (Clone 561)| Biolegend| Cat# 343615; RRID:  
Alexa Fluor® 700 anti-human CD71 (Clone CY1G4)| Biolegend| Cat# 334130; RRID:  
FITC anti-human CD45 (Clone HI30)| Biolegend| Cat# 304038; RRID:  
PE/Dazzle anti-human Podoplanin (Clone NC-08)| Biolegend| Cat# 337028; RRID:  
PE/Cy7 anti-human CD56 (Clone 5.1H11)| Biolegend| Cat# 362510; RRID:  
Brilliant Violet 650™ anti-human CD90 (Clone 5E10)| Biolegend| Cat# 328143; RRID:  
APC/Fire anti-human CD235ab (Clone HIR2)| Biolegend| Cat# 306622; RRID:  
Alexa Fluor 647 anti-human VE-Cadherin (Clone BV9)| Biolegend| Cat# 348513; RRID:  
PE anti-human Leptin Receptor (LepR) (Clone 52263)| R&D Systems| Cat# FAB867P; RRID:  
**Biological samples**  
Human Femoral Head| This Manuscript| N/A  
Human Acute Myeloid Leukemia FFPE Blocks| This Manuscript| N/A  
Human Negative Staging Marrow FFPE Blocks| This Manuscript| N/A  
**Chemicals, peptides, and recombinant proteins**  
MesenCult Proliferation Kit| STEMCELL Technologies| Cat# 05411  
Dispase II| Sigma-Aldrich| Cat# D4693,CAS# 42613-33-2  
Collagenase, Type 1| Worthington Biochem| Cat# LS004196  
Indomethacin| Sigma-Aldrich| Cat# I7378,CAS# 53-86-1  
Isobutylmethylxanthin| Sigma-Aldrich| Cat# I5879,CAS# 28822-58-4  
Hydrocortisone| Sigma-Aldrich| Cat# H4001,CAS# 50-23-7  
Insulin| Gibco| Cat# 12585014  
Dexamethasone| Sigma-Aldrich| Cat# D8893,CAS# 50-02-2  
Potassium phosphate monobasic| Sigma-Aldrich| Cat# P5655,CAS# 7778-77-0  
TGF-beta1| Sigma-Aldrich| Cat# GF346  
Sodium Pyruvate| Gibco| Cat# 11360070  
L-Proline| Sigma-Aldrich| Cat# P5607,CAS# 147-85-3  
ITS+ Premix| Corning| Cat# 354352  
Alizarin Red S| Sigma-Aldrich| Cat# A5533,CAS# 130-22-3  
Alcian Blue| Sigma-Aldrich| Cat# A5268,CAS# 33864-99-2  
Crystal Violet| Sigma-Aldrich| Cat# C6158,CAS# 548-62-9  
Human TruStain FcX™ (Fc Receptor Blocking Solution)| Biolegend| Cat# 422302  
UltraComp eBeads Compensation Beads| Invitrogen| Cat# 01-2222-42  
Histochoice clearing agent| VWR| Cat# H103-4L  
Assay Reagent for PhenoCycler| Akoya Biosciences| Cat# 7000002  
10X Buffer Kit for PhenoCycler-Fusion| Akoya Biosciences| Cat# 7000019  
Nuclear Stain for PhenoCycler| Akoya Biosciences| Cat# 7000003  
Citrate Buffer, pH 6.0, 10×, Antigen Retriever| Sigma-Aldrich| C9999-1000ML  
EasySep™ Buffer| STEMCELL Technologies| Cat# 20144  
Paraformaldehyde 16% Aqueous Solution EM Grade| Electron Microcopy Sciences| Cat# 15710-S  
UltraPure 0.5M EDTA, pH8.0| Invitrogen| Cat# 15575-038  
Formical-4| StatLab| Cat# 1214-1  
**Critical commercial assays**  
SPRIselect Reagent Kit| Beckman Coulter| Cat# B23318  
Qubit dsDNA HS Assay Kit| Thermo Fisher Scientific| Cat# Q32854  
High Sensitivity DNA Kit| Agilent| Cat# 5067-4626  
KAPA Library Quantification Kit for Illumina Platforms| KAPA Biosystems| Cat# KK4824  
EasySep™ RBC Depletion Reagent| STEMCELL Technologies| Cat# 18170  
EasySep™ Human CD45 Depletion Kit II| STEMCELL Technologies| Cat# 17898  
EasySep™ Human CD34 Positive Selection Kit II| STEMCELL Technologies| Cat# 17856  
Chromium Next GEM Single Cell 3’ GEM, Library & Gel Bead Kit v3.1, 16 rxns| 10x Genomics| Cat# PN-1000121  
Chromium Next GEM Chip G Single Cell Kit, 48 rxns| 10x Genomics| Cat# PN-1000120  
Dual Index Kit TT Set A, 96 rxns| 10x Genomics| Cat# PN-1000215  
NovaSeq 6000 S1 Reagent Kit v1.5 (100 cycles)| Illumina| Cat# 20012850  
NovaSeq 6000 S2 Reagent Kit v1.5 (100 cycles)| Illumina| Cat# 20028316  
Amicon® Ultra Centrifugal Filter, 50 kDa MWCO| Millipore| Cat# UFC505096  
Oil Red O Staining Kit| LifeLine Cell Technology| Cat# LL-0052  
High-Capacity cDNA Reverse Transcription Kit| Applied Biosystems| Cat# 4368814  
Power SYBR Green PCR Master Mix| Applied Biosystems| Cat# 367659  
Sample Kit for PhenoCycler-Fusion| Akoya Biosciences| Cat# 7000017  
Antibody Conjugation Kit| Akoya Biosciences| Cat# 7000009  
**Deposited data**  
Single-cell RNA sequencing raw and process data| This paper| NCBI GEO:   
CODEX spatial proteomics raw and processed data| This paper  
Vitessce interactive data explorer| This paper  
**Oligonucleotides**  
CODEX oligonucleotides, see [Table S5](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#mmc5)| Akoya Biosciences| various  
qPCR oligonucleotides, see [Table S9](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#mmc9)| IDT| N/A  
**Software and algorithms**  
Seurat v4 (Main Dataset)Seurat v5 (L/R Analysis)| Hao and Hao et al., _Cell_ , 2021[73](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib87)Hao et al., _Nature Biotechnology_ , 2023[74](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib88)  
Cellranger v6.1.2| N/A  
PhenoCycler Experimental Designer v1.0.8| N/A  
Fusion v1.0.6| N/A  
Nolan Lab Neighborhood Analysis| Schurch et al., _Cell_ , 2020[52](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib52)  
AUCell| Aibar et al.[75](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib80)  
CellChat v1| Jin et al.[37](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib37)  
CytoTRACE v0.3.3| Gulati et al.[28](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib28)  
Spatstat| N/A  
Wsireg| N/A  
scRNA-seq/CODEX Analysis Pipeline and Code| This paper  
GraphPad Prism 10| N/A  
Napari v0.4.14| N/A  
QuPath v0.4.3| Bankhead et al.[76](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib86)  
Metascape v3.5| Zhou et al., Nature Communications, 2019[77](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib89)  
FlowJo 10.8.1| N/A  
Mesmer 0.12.1| Greenwald et al.[51](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib51)  
μCT Evaluation Program v6.6| Scanco Medical, AG| N/A  
**Other**  
Chromium Controller| 10x Genomics| Cat# PN 120270  
Phenocylcer-FUSION| Akoya Biosciences| Cat# AB-000011  
Eppendorf realplex2 Mastercylcer| Eppendorf| Cat# 5345 017187  
CFX Connect Real-Time PCR Detection System| BIO-RAD| Cat# 1855201  
C1000 Touch™ Thermal Cycler| BIO-RAD| Cat# 1851197  
NovaSeq 6000 System| Illumina| Cat# 20012850  
2100 Bioanalyzer| Agilent| Cat# G2939A  
10x Magnetic Separator| 10x Genomics| Cat# 230003  
EasySep™ Magnet| STEMCELL Technologies| Cat# 18000  
### Resource availability
#### Lead contact
Further information and requests for reagents and resources should be directed to and will be fulfilled by the lead contact, Kai Tan ().
#### Materials availability
All materials used in this study are commercially available, as specified in the [key resources table](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#sec4.1).
#### Data and code availability
Software and source code created for this study are publicly available and can be obtained at GitHub at . Single-cell RNA-seq data can be downloaded from NCBI Gene Expression Omnibus (GEO) database with the accession number . This includes the raw sequencing reads, count matrices, and annotated Seurat object.
Primary imaging data and processed CODEX data are publicly available and can be downloaded from FigShare at: .
This includes full multiplexed images for the normal bone marrow atlas, AML, NSM samples, corresponding cell segmentation masks and H&E images as well as annotated Seurat objects which contain unnormalized and normalized marker expression values for each cell.
Both scRNA-seq and CODEX data are publicly available to explore interactively through Vitessce widgets hosted at .
Any additional information required to reanalyze the data reported in this paper is available from the lead contact upon request.
### Experimental model and study participant details
#### Human clinical sample acquisition
Bone marrow samples were obtained from patients undergoing [total hip arthroplasty](https://www.sciencedirect.com/topics/medicine-and-dentistry/total-hip-arthroplasty "Learn more about total hip arthroplasty from ScienceDirect's AI-generated Topic Pages"). We reasoned that [osteoarthritis](https://www.sciencedirect.com/topics/immunology-and-microbiology/osteoarthritis "Learn more about osteoarthritis from ScienceDirect's AI-generated Topic Pages") patients undergoing total hip [arthroplasty](https://www.sciencedirect.com/topics/immunology-and-microbiology/arthroplasty "Learn more about arthroplasty from ScienceDirect's AI-generated Topic Pages") surgeries, where the femoral bone is removed and discarded as surgical waste, would be a good source of hematologically healthy bone marrow. We considered that these samples were representative of normal hematopoiesis due to two major factors: first, studies have reported that 75-90% of individuals over 75 years of age will develop some degree of [osteoarthritis](https://www.sciencedirect.com/topics/medicine-and-dentistry/osteoarthritis "Learn more about osteoarthritis from ScienceDirect's AI-generated Topic Pages"),[78](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib73) making the condition within the range of aging-associated changes; and second, osteoarthritis is a peri-articular disease not associated with hematologic complications.[79](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib74) We only sampled at the distal site to the [articular cartilage](https://www.sciencedirect.com/topics/medicine-and-dentistry/articular-cartilage "Learn more about articular cartilage from ScienceDirect's AI-generated Topic Pages") and only collected morphologically normal [trabecular bone](https://www.sciencedirect.com/topics/immunology-and-microbiology/trabecular-bone "Learn more about trabecular bone from ScienceDirect's AI-generated Topic Pages") and associated bone marrow. For femoral heads profiled via scRNA-seq or CODEX, the age range was 52-80 years, with a median of 65.5 (IQR: 58.5-71.25) years. Physicians provided common medical conditions information, which are tabulated in the clinical metadata table. All femoral head patient samples were deemed as medical waste and IRB exempt by the University of Pennsylvania Institutional Review Board. Only non-identifiable patient information is collected for each sample by the clinical staff. Researchers were never given access to [medical records](https://www.sciencedirect.com/topics/medicine-and-dentistry/medical-record "Learn more about medical records from ScienceDirect's AI-generated Topic Pages"). Acute myeloid leukemia (AML) and negative staging marrow (NSM) samples were obtained from the Hospital of the University of Pennsylvania’s Pathology sample repository. These formalin fixed paraffin embedded (FFPE) samples are collected via trephine bone marrow biopsy from iliac crest, fixed in 10% neutral buffered formalin (NBF) for a minimum of 1 hour, decalcified with Formical for 2 hours, and rinsed with water for 10 minutes before [paraffin embedding](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/paraffin-embedding "Learn more about paraffin embedding from ScienceDirect's AI-generated Topic Pages"). All AML and NSM samples were appropriately collected under University of Pennsylvania IRB #826755. All patient clinical information is shown in [Table S1](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#mmc1).
### Method details
#### Femoral head cell isolation
Femoral heads obtained from total hip arthroplasty surgeries were cut coronally in 4–6 mm thick slices by the [Anatomical Pathology](https://www.sciencedirect.com/topics/medicine-and-dentistry/anatomical-pathology "Learn more about Anatomical Pathology from ScienceDirect's AI-generated Topic Pages") Lab at Penn Presbyterian Medical Center. Tissue samples were transported on ice in AlphaMEM (Gibco) with 3% FBS (HyClone), 1% penicillin-streptomycin (Gibco), 0.02 mM L-ascorbic acid (Sigma) and 0.3 mM L-Glutamine (Gibco). Samples were placed in a petri dish and cut with surgical wire cutters into approximately 1 mm cubed pieces, weighed and transferred to a 15 mL centrifuge tube. For CODEX, a dental biopsy drill was used to harvest cylindrical samples, cut to about 4 mm thick, and fixed in 4% paraformaldehyde (PFA) (EMS). For scRNA-seq and cell sorting, 1 mm cubed of minced tissue was digested in filtered 4 mg/mL dispase II (Sigma), and 2 mg/mL collagenase I (Worthington Biochem) in PBS for one hour on a rocker at 37˚C. The cell suspension was then filtered through a 100 μm cell strainer (Fisher) and remaining tissue was washed 5 times in culture medium (AlphaMEM with 15% FBS, 1% P/S, 0.1 mM LAA, 1 mM L-glutamine) and filtered through a 70 μm cell strainer (Fisher). Total collected suspension was pelleted at 300 RCF for 10 minutes and resuspended in MesenCult Proliferation medium (STEMCELL Technologies, Cat# 05411), and immediately taken for immunomagnetic population isolation (see “[single-cell RNA sequencing](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#sec4.4.3)” and “[mesenchymal cell fluorescence assisted cell sorting](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#sec4.4.10)” sections).
#### Micro-CT BV/TV determination
[Bone tissue](https://www.sciencedirect.com/topics/medicine-and-dentistry/bone-tissue "Learn more about Bone tissue from ScienceDirect's AI-generated Topic Pages") cores were scanned with a micro–computed tomography scanner (Model MicroCT35, Scanco Medical AG, Switzerland) at a 6-μm isotropic voxel size. All images were analyzed with the Scano scanner software. A region of 4200 μm in diameter and 900 μm in height was selected and smoothed with a [Gaussian](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/gaussian-distribution "Learn more about Gaussian from ScienceDirect's AI-generated Topic Pages") filter (sigma = 1.2, support = 2.0) prior to binarization with a minimum threshold of 365 and bone volume to total volume (BV/TV) calculated.
#### Single-cell RNA sequencing
Digested tissue samples were subjected to immunomagnetic RBC depletion reagent (STEMCELL Technologies, Cat# 18170). 20 uL of RBC depleted cells were put on ice, while 67% of the remaining cells were used for immunomagnetic CD45 depletion using EasySep Human CD45 Depletion Kit II (STEMCELL Technologies, Cat# 17898) to isolate non-hematopoietic cells in an unbiased fashion, while 33% of cells were subjected to immunomagnetic [CD34](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/cd34 "Learn more about CD34 from ScienceDirect's AI-generated Topic Pages") enrichment using EasySep Human [CD34](https://www.sciencedirect.com/topics/immunology-and-microbiology/cd34 "Learn more about CD34 from ScienceDirect's AI-generated Topic Pages") Positive Selection Kit II (STEMCELL Technologies, Cat# 17856). Following enrichment, each of the three cell fractions (RBC depleted, CD45 depleted, CD34 enriched) were resuspended at a concentration of 1000 cells/uL in PBS + 0.04% [BSA](https://www.sciencedirect.com/topics/medicine-and-dentistry/bovine-serum-albumin "Learn more about BSA from ScienceDirect's AI-generated Topic Pages"). Cells were pooled and loaded into the 10x Genomics Chip G (Cat# 1000127). We used Chromium Next GEM Single Cell 3’ GEM, Library & Gel Bead Kit v3.1 (10x Genomics, Cat# 1000121) and followed the protocol as provided by 10x Genomics. Sequencing was performed at the [High Throughput Sequencing](https://www.sciencedirect.com/topics/immunology-and-microbiology/high-throughput-sequencing "Learn more about High Throughput Sequencing from ScienceDirect's AI-generated Topic Pages") Core at CHOP using a NovaSeq 6000 system. Sequencing parameters were set according to 10x Genomics recommendations.
#### Bone marrow decalcification
Femoral head samples were fixed for 24 hours in 4% paraformaldehyde (Electron Microscopy Sciences, Cat# 15710-S diluted to 4%) in PBS (Gibco, Cat# 14190-136) at 4°C. Samples were washed 3 times for 30 minutes in PBS at 4°C. Samples were decalcified in 0.5M [EDTA](https://www.sciencedirect.com/topics/medicine-and-dentistry/edetic-acid "Learn more about EDTA from ScienceDirect's AI-generated Topic Pages") at pH8 (Invitrogen, Cat# 15575-038) for 8 days. The EDTA was changed every 2 days. After [decalcification](https://www.sciencedirect.com/topics/medicine-and-dentistry/decalcification "Learn more about decalcification from ScienceDirect's AI-generated Topic Pages"), samples were washed 5 times for 1 hour in PBS and fixed for 24 hours in 4% paraformaldehyde at 4°C. Samples were processed and embedded in paraffin by the Children’s Hospital of Philadelphia’s pathology core. The AML and NSM samples were obtained from the University of Pennsylvania Department of Pathology sample repository. Bone marrow biopsies were fixed in buffered zinc formalin for a minimum of 1 hour. Bone marrow biopsies were then decalcified in Formical-4 (StatLab, SKU# 1214-1) for 2 hours. Samples were rinsed with running water for a minimum of 10 minutes. Samples were placed in neutral buffered formalin and submitted for [paraffin embedding](https://www.sciencedirect.com/topics/immunology-and-microbiology/paraffin-embedding "Learn more about paraffin embedding from ScienceDirect's AI-generated Topic Pages").
#### Antibody panel design and validation
Antibodies for CODEX were chosen based on a combination of literature (esp. (Kathryn Foucar’s Bone Marrow Pathology 4th edition textbook,[80](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib75) Bendall et al.,[81](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib76) STEMCELL Technologies’ recommendations ) and our scRNA-seq data. Mesenchymal markers were entirely derived from our scRNA-seq data and sorted assays. Antibodies were chosen for testing which had [FFPE](https://www.sciencedirect.com/topics/medicine-and-dentistry/formaldehyde "Learn more about FFPE from ScienceDirect's AI-generated Topic Pages") validation by the vendors and high signal-to-noise ratio on validation. Antibodies were further validated by co-labeling and cell-type appropriate expression, which is shown in the Supplemental Cell Type Annotation Guide ([Data S1](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#mmc10)).
#### CODEX antibody conjugation
Akoya antibodies were purchased pre-conjugated to their respective CODEX barcodes ([Table S5](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#mmc5)). All other antibodies were custom conjugated to their respective CODEX barcodes ([Table S5](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#mmc5)) according to Akoya’s PhenoCycler-Fusion user guide using the antibody conjugation kit (Akoya, Cat# 7000009). Briefly, 50 μg of carrier-free antibodies were concentrated by [centrifugation](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/centrifugation "Learn more about centrifugation from ScienceDirect's AI-generated Topic Pages") in 50kDa MWCO filters (EMD Millipore, Cat# UFC505096) and incubated in the antibody [disulfide](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/disulfide "Learn more about disulfide from ScienceDirect's AI-generated Topic Pages") reduction master mix for 30 minutes. After buffer exchange of the antibodies to conjugation solution by centrifugation, addition of conjugation solution and centrifugation, respective CODEX barcodes resuspended in conjugation solution were added to the concentrated antibody and incubated for 2 hours at room temperature. [Conjugated antibodies](https://www.sciencedirect.com/topics/immunology-and-microbiology/antibody-conjugate "Learn more about Conjugated antibodies from ScienceDirect's AI-generated Topic Pages") were purified by 3 buffer exchanges with [purification](https://www.sciencedirect.com/topics/medicine-and-dentistry/copurification "Learn more about purification from ScienceDirect's AI-generated Topic Pages") solution. 100 μl of antibody storage buffer was added to the concentrated purified antibodies.
#### CODEX staining
CODEX staining was done using the sample kit for PhenoCycler-Fusion (Akoya, Cat# 7000017) according to Akoya’s PhenoCycler-Fusion user guide with modifications to include a photobleaching step and overnight incubation in antibodies at 4°C. FFPE samples were sectioned at 5 μm thickness and mounted onto charged slides (Leica, Cat# 3800080). Sample slides were baked for 3 hours at 65°C. Sample slides were deparaffinized in Histochoice clearing agent (VWR, Cat# H103-4L) and rehydrated in a graded series of ethanol (2 times 100%, 90%, 70%, 50%, 30% and 2 times ddH2O). [Antigen retrieval](https://www.sciencedirect.com/topics/medicine-and-dentistry/antigen-retrieval "Learn more about Antigen retrieval from ScienceDirect's AI-generated Topic Pages") was performed in 1x citrate buffer pH6 (Sigma, Cat# C9999 diluted to 1x in ddH2O) with a pressure cooker for 20 minutes. After equilibrating to room temperature, sample slides were washed 2 times with ddH2O and submerged in a petri dish containing 4.5% H2O2 and 20 mM NaOH in PBS (bleaching solution) for photobleaching. The petri dish was sandwiched between two broad-spectrum LED [light sources](https://www.sciencedirect.com/topics/immunology-and-microbiology/light "Learn more about light sources from ScienceDirect's AI-generated Topic Pages") for 45 minutes at 4°C. After 45 minutes, sample slides were transferred to a new petri dish with fresh bleaching solution and photobleached for another 45 minutes at 4°C. Sample slides were washed 3 times in PBS and then 2 times in hydration buffer. Sample slides were equilibrated in staining buffer for 30 minutes and incubated in the antibodies diluted in staining buffer plus N Blocker, G Blocker, J Blocker, and S Blocker overnight at 4°C. After antibody incubation, sample slides were washed 2 times in Staining Buffer and fixed for 10 minutes in 1.6% paraformaldehyde in storage buffer. Sample slides were washed 3 times in PBS and incubated in ice cold methanol for 5 minutes. After incubation in methanol, sample slides were washed 3 times in PBS and incubated in final fixative solution for 20 minutes. The sample slides were then washed 3 times in PBS and stored in storage buffer for up to one week prior to imaging.
#### CODEX imaging
CODEX reporters were prepared according to Akoya’s PhenoCycler-Fusion user guide and added to a 96-well plate. The PhenoCycler-Fusion experimental [template](https://www.sciencedirect.com/topics/immunology-and-microbiology/dna-template "Learn more about template from ScienceDirect's AI-generated Topic Pages") was set up for a CODEX Run using Akoya’s PhenoCycler Experiment Designer software according to Akoya’s PhenoCycler-Fusion user guide. Details on the order of fluorescent CODEX barcodes/reporters and microscope exposure times, can be found on ([Table S6](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#mmc6)). The PhenoCycler-Fusion experimental run was performed using Akoya’s Fusion 1.0.6 software according to Akoya’s PhenoImager Fusion User Guide. Images were taken on the PhenoImager Fusion microscope using default settings and pre-processed (stitching, registration, background subtraction) by the Fusion software.
#### H&E staining and registration
Slides were removed from the microscope after CODEX imaging with the PhenoCycler-Fusion platform and kept in Histochoice for 3-5 days. The flow cell was gently removed and then the tissue slide was sent to CHOP’s pathology core for standard hematoxylin and eosin (H&E) staining. Imaging was performed on the PhenoCycler Fusion microscope using the brightfield mode. 40x H&Es were registered to the 20x CODEX imaging space using wsireg () with affine and rigid transformations, the default parameters, with the [DAPI](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/dapi "Learn more about DAPI from ScienceDirect's AI-generated Topic Pages") and NaK-ATPase channels as input from the CODEX image to label the nucleus and membrane (to match with hematoxylin and eosin) respectively.
#### Mesenchymal cell fluorescence assisted cell sorting
Femoral head samples from donors were digested as described above and subjected to RBC depletion and CD45 depletion using [MACS](https://www.sciencedirect.com/topics/medicine-and-dentistry/magnetic-activated-cell-sorting "Learn more about MACS from ScienceDirect's AI-generated Topic Pages") (STEMCELL Technologies, Cat# 18170 and 17898). Antibodies for [FACS](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/fluorescence-activated-cell-sorting "Learn more about FACS from ScienceDirect's AI-generated Topic Pages") were purchased from BioLegend unless otherwise specified. Isolated cells were stained for 15-30 minutes using anti-CD90-Bv650(Clone 5E10), anti-CD38-Bv785 (Clone HIT2), anti-CD45-FITC (Clone HI30), anti-LEPR-PE (R&D Systems, Clone 52263), anti-Podoplanin-PE/Dazzle (Clone NC-08), anti-CD56-PE/Cy7 (Clone 5.1H11), anti-VE-Cadherin-Alexa Fluor 647 (Clone BV9), and anti-CD235ab-APC/Fire (Clone HIR2). Cells were spun down and resuspended in PBS + 2% FBS containing [DAPI](https://www.sciencedirect.com/topics/medicine-and-dentistry/dapi "Learn more about DAPI from ScienceDirect's AI-generated Topic Pages") to exclude dead cells. Fluorescence minus one controls were used to confirm the gating strategy, and all mesenchymal cells were gated as CD45- DAPI- CD38lo CD235ab-, hereby referred to as hem-negative. Fibro-MSCs were defined as hem-negative PDPN+ VE-Cadherin-. Osteolineage cells were defined as heme-negative PDPN- VE-Cadherin- CD56+. THY1+ MSC were defined as hem-negative PDPN- VE-Cadherin- CD56- LEPR+ CD90+ and Adipo-MSC were defined as PDPN- VE-Cadherin- CD56- LEPR+ CD90-. Cells were directly sorted into MesenCult Proliferation Kit (STEMCELL Technologies, Cat# 05411) growth medium containing 25mM [HEPES](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/hepes "Learn more about HEPES from ScienceDirect's AI-generated Topic Pages") and cultured or used directly for downstream experiments.
#### Proliferation/Cell culturing
Sorted cells were seeded in MesenCult Proliferation medium (STEMCELL Technologies, Cat# 05411) for passaging. The population doubling was calculated as 3.32∗log((cell count at passage)/(cells seeded)). The cumulative population doubling is reported. The cumulative population doubling for the four cell types over eight passages was analyzed by a 2-way ANOVA followed by a Bonferroni post-hoc test comparing each cell type at each passage to Fibro-MSCs.
#### CFU-F assays
Cells were seeded immediately after fluorescence assisted cell sorting at 145 cells/cm2 in a 6-well or 12-well cell culture treated dish and maintained in MesenCult Proliferation Kit (STEMCELL Technologies, Cat# 05411), until colonies could be visualized under the microscope (8-12 days after beginning the assay). Cells were washed in PBS and stained with crystal violet (3%). Colonies were identified by morphology and manually counted as clusters with greater than 30 cells forming a tight grouping.
#### Differentiation potential assessment
Sorted Fibro-MSCs were expanded in culture before plating for differentiation assays to achieve the necessary input cell number per patient. At 90% confluence, growth medium was switched to osteogenic induction medium (AlphaMEM with 10% FBS, 1% P/S, 0.1 mM LAA, 1 mM L-glutamine, 10-2 μM [DMSO](https://www.sciencedirect.com/topics/medicine-and-dentistry/dimethyl-sulfoxide "Learn more about DMSO from ScienceDirect's AI-generated Topic Pages"), and 1.8 mM KH2PO4) or adipogenic induction medium (AlphaMEM with 10% FBS, 1% P/S, 0.1 mM LAA, 1 mM L-glutamine, 0.5 mM isobutylmethylxanthin, 60 μM indomethacin, 0.5 μM hydrocortisone, and 10 μg/mL insulin). Cells were harvested at day 14 for qRT-PCR analysis and at day 28 for [Alizarin](https://www.sciencedirect.com/topics/medicine-and-dentistry/alizarin "Learn more about Alizarin from ScienceDirect's AI-generated Topic Pages") Red (osteogenic), Oil Red O (adipogenic), or [Alcian Blue](https://www.sciencedirect.com/topics/medicine-and-dentistry/alcian-blue "Learn more about Alcian Blue from ScienceDirect's AI-generated Topic Pages") (chondrogenic) staining. For [qPCR](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/real-time-polymerase-chain-reaction "Learn more about qPCR from ScienceDirect's AI-generated Topic Pages") analysis ([Table S9](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#mmc9)), total [RNA](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/rna "Learn more about RNA from ScienceDirect's AI-generated Topic Pages") was isolated using a phenol-chloroform extraction. We were unable to extract RNA from chondrogenic pellets likely due to the 3D structure, and so focused on qPCR for adipogenic and osteogenic differentiations, relying on the Alcian blue staining alone for chondrogenic. 1 μg of RNA was used for reverse transcription (High-Capacity cDNA [Reverse Transcription](https://www.sciencedirect.com/topics/medicine-and-dentistry/reverse-transcription "Learn more about Reverse Transcription from ScienceDirect's AI-generated Topic Pages") Kit, Applied Biosystem). qPCR was performed using Power SYBR™ Green PCR Master Mix. Samples were normalized to endogenous control _GAPDH_ and fold change from the median control Ct value calculated using the ddCT method.[82](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib77)
#### Mesenchymal and hematopoietic cell co-culture assay
One day prior to hematopoietic progenitor cell isolation, sorted MSC cell subsets (Fibro-, THY1+, Osteo-, and Adipo-MSCs) were seeded at 10,000 cells/cm2 on a 6-well dish (Falcon, 353046). Hematopoietic progenitor cells were isolated from digested femoral head cell suspensions by performing RBC depletion (STEMCELL Technologies, Cat# 18170) followed by CD34+ cell selection (STEMCELL Technologies, Cat# 17856) with the StemCell EasySep system. The MSC cell subsets were washed twice with PBS and growth medium changed to DMEM with 15% FBS, 0.1 mM LAA and 1% penicillin-streptomyocin. CD34+ isolated cells were added to each well at 10,000 cells/cm2 and an additional well seeded with only CD34+ isolated cells. After three days, 1 mL medium was added. On day six, cells in suspension were combined with trypsinized [adherent cells](https://www.sciencedirect.com/topics/immunology-and-microbiology/adherent-cell "Learn more about adherent cells from ScienceDirect's AI-generated Topic Pages"), pelleted, and resuspended in PBS with 2% FBS. Cells were blocked with Fc [receptor blocker](https://www.sciencedirect.com/topics/medicine-and-dentistry/receptor-blocking-agent "Learn more about receptor blocker from ScienceDirect's AI-generated Topic Pages") (Biolegend, Cat# 422302) and then stained with Biolegend antibodies, unless otherwise stated, CD61-BUV395 (BD Biosciences, Cat# 745717), CD19-BV510 (Cat# 302241), CD14-BV605 (Cat# 367125), CD33-BV650 (Cat# 303429), CD38-BV785 (Cat#, 303530), CD15-FITC (Cat# 394705), CD235ab-PerCP/Cy5.5 (Cat# 306613), CD66a/c/e-PE (Cat# 342303), CD45-APC/Cy7 (Cat# 368516), CD163-AF647 (Cat# 333619), CD34-PE/Cy7 (Cat# 343615) and CD71-AF700 (Cat# 334130). DAPI was used as a live/dead stain. Flow cytometry was performed on an Aurora spectral analyzer with 5 lasers and 64 detectors (Cytek Biosciences) through the Children’s Hospital of Philadelphia Flow Cytometry Core. Spectral unmixing and compensation was accomplished with single stained compensation beads (Invitrogen). Differentiation potential was assessed as described previously[81](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib76) – that is, gating of lineage uncommitted CD34+ CD38- cells by excluding cells expressing markers of erythroid, B-cell, myeloid, and monocytic differentiation.
#### Figure making and plotting
Nearly all graphical figures were generated using R v4.0.5 and subsequently edited in Adobe Illustrator. Napari (doi:10.5281/zenodo.3555620) was used for generation of CODEX image figures, including fluorescent CODEX images, masks, and H&E images. Images in the manuscript figures are contrast-adjusted for illustrative purposes. For images, in certain cases Adobe Photoshop/Illustrator was used to globally adjust brightness, contrast, or threshold for visualization purposes. GraphPad Prism v9 was used to plot [Figure S3](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#figs3)D and perform statistical testing in [Figure 2](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#fig2)F.
### Quantification and statistical analysis
#### scRNA-seq data processing, annotation, and pathway analysis
scRNA-seq data were demultiplexed, aligned, and count matrices were generated using Cell Ranger v6.1.2 aligning to GRCh38 with intronic regions included. Cell Ranger matrices were directly imported into Seurat v4. Cells with 300–10,000 unique genes expressed, less than 10,000 unique UMIs, and less than 10% mitochondrial reads were retained in the final dataset. Individual samples were processed by log-normalization, the top 2,000 features were identified, the data was scaled, a [principal component analysis](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/principal-component-analysis "Learn more about principal component analysis from ScienceDirect's AI-generated Topic Pages") (PCA) was computed, and the data was clustered via Louvain clustering using the top 30 principal components (PCs). We used DoubletFinder[83](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib78) to remove doublets on a per-sample basis using the top 30 PCs, an expected doublet rate equal to the number of cells in the sample/125000, and other parameters remained as default. Doublet cells were removed, and samples were combined without computational batch correction. The full dataset was processed as above and visualized using UMAP with the top 30 PCs. Differentially expressed genes in each cluster were identified using the FindAllMarkers function in Seurat v4 with the Wilcoxon rank-sum test downsampled to a maximum of 1,000 cells per cluster, and these results wererecorded in [Table S2](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#mmc2). Cells were annotated by marker gene expression, and likely artifactual cells (only found in one sample or likely residual doublets) were removed from analysis using VisCello for interactive visualization and artifact selection and removal,[84](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib79) followed by dimensionality reduction and clustering in an iterative process to refine the dataset. Mesenchymal and endothelial cell clusters were computationally isolated for further analysis. Subset analysis was conducted similarly as above, first calculating the top 2,000 variable features, then scaling, recomputing a PCA, Louvain clustering using the top 20 PCs, and visualization using UMAP with the top 50 PCs. Likely artifactual cells were removed using VisCello and these subsets were iteratively reprocessed and visualized. AUCell[75](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib80) was used for pathway analysis with the human Hallmark gene set ().[85](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib81) One population, RNAlo MSCs, were excluded from downstream analysis because of their indeterminate QC profile which was consistent with highly differentiated cells like [neutrophils](https://www.sciencedirect.com/topics/immunology-and-microbiology/neutrophil-granulocyte "Learn more about neutrophils from ScienceDirect's AI-generated Topic Pages") or RBCs but did not match a known cell type and thus could also be artifact. Differentiation states of MSCs were inferred using CytoTRACE (v0.3.3) with default parameters using the log-normalized counts. Further details are provided in the accompanying source code.
#### External dataset mapping
External datasets were downloaded per the instructions on the respective manuscripts. Samples from De Jong et al. were downloaded from EBI (E-MTAB-9139), and [myeloma](https://www.sciencedirect.com/topics/medicine-and-dentistry/myeloma "Learn more about myeloma from ScienceDirect's AI-generated Topic Pages") samples were excluded (CBM1_1, CBM1_2, CBM2_1, CBM2_2, S9, S10, S11, S12, and S13). Count matrices were generated using Cell Ranger 4.0.0 with default parameters. Samples were integrated and CXCL12+ clusters were exported. Data from Li et al. were downloaded from NCBI [GEO](https://www.sciencedirect.com/topics/medicine-and-dentistry/gene-expression-omnibus "Learn more about GEO from ScienceDirect's AI-generated Topic Pages") (GSE190965) and directly read into Seurat and reprocessed as in our dataset described above. For Wang et al. data, we downloaded feature barcode matrices from NCBI GSE147287 and read into Seurat after concatenating the spliced and unspliced matrix files and reprocessed as above. We downloaded the Ennis et al. data from GSE227903 and used the SeuratDisk package to convert from the provided aggregated h5ad data to a Seurat object and reprocessed the data. For Triana et al. data, we retrieved the Seurat object directly from Figshare (). When reprocessing data was required for any dataset, we retained cells with 100–10,000 unique genes expressed, and less than 10% mitochondrial reads were retained. We identified the top 2,000 variable features, and recomputed a PCA before performing reference mapping.
For samples in public data in which annotations were available, mesenchymal cells were extracted and reference mapped to our MSC reference using reciprocal PCA (RPCA) in Seurat, using the FindTransferAnchors function with the top 50 dimensions followed by the MapQuery function to project the data onto our reference atlas. Default parameters were used generally, except for the Triana et al. (k.weight=30) and Ennis et al. data (k.filter=10, k.score=17, k.weight=30) where the very small cell numbers necessitated lowering the k parameters. In cases where annotations were not available, we mapped to our whole reference (hematopoietic + non-hematopoietic cells) and showed just the results for cells mapping to non-hematopoietic cells. The exception to this was the De Jong et al. data, which we filtered down to mesenchymal cells using marker gene expression (_VIM_ , _[PTPRC](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/protein-tyrosine-phosphatase-receptor-type-c "Learn more about PTPRC from ScienceDirect's AI-generated Topic Pages")_ , _CXCL12_ , [CD38](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/cd38 "Learn more about CD38 from ScienceDirect's AI-generated Topic Pages")) ourselves due to the size of the unfiltered dataset. More details for this analysis including preprocessing can be found in the accompanying source code.
#### Inflammation signature analysis
Hallmark [inflammatory response](https://www.sciencedirect.com/topics/immunology-and-microbiology/inflammation-response "Learn more about inflammatory response from ScienceDirect's AI-generated Topic Pages") scores were computed using AUCell[75](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib80) with 300 cells per cell type in [Figure S2](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#figs2)A. To compare [inflammatory cytokine](https://www.sciencedirect.com/topics/medicine-and-dentistry/inflammatory-cytokine "Learn more about inflammatory cytokine from ScienceDirect's AI-generated Topic Pages") expression across datasets, we used Chen et al. data[86](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib82) as a representative [pediatric](https://www.sciencedirect.com/topics/medicine-and-dentistry/pediatrics "Learn more about pediatric from ScienceDirect's AI-generated Topic Pages") atlas, the [Human Cell](https://www.sciencedirect.com/topics/immunology-and-microbiology/human-cell "Learn more about Human Cell from ScienceDirect's AI-generated Topic Pages") Atlas [Immune Cell](https://www.sciencedirect.com/topics/immunology-and-microbiology/immunocompetent-cell "Learn more about Immune Cell from ScienceDirect's AI-generated Topic Pages") Atlas Bone Marrow (“hcabm40k” installed using Seurat), and Li et al. data.[17](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib17) We filtered our dataset to just MSCs and re-calculated the Hallmark Inflammatory Response signature using AUCell with 5000 randomly chosen cells per dataset. We also included publicly available scRNA-seq data from cultured primary iliac-crest-aspirate-derived human MSCs treated with and without 15 ng/mL TNFα and 10 ng/mL IFNγ for 24 hours[87](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib83) (GSE115149). We called the TNFα/IFNγ treated MSC “Inflamed_MSC” and the untreated MSCs as “Control_MSC”. We also included Li et al. MSCs since they are representative bone marrow aspirate (and therefore can be considered “normal”) in this analysis. Li et al. MSCs were isolated as described in the External Dataset Mapping section. Statistical testing was performed using a two-sided Welch’s two sample t-test with each population tested against the inflamed MSCs.
#### Cellular interaction analysis
CellChat was used for cellular interaction analysis using the CellChatDB database. As described in the original manuscript,[37](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib37) scRNA-seq values were projected onto a protein-protein interaction network to account for typical dropout issues. The significant cell-cell interactions were determined using CellChat default parameters, with [population size](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/population-size "Learn more about population size from ScienceDirect's AI-generated Topic Pages") accounted for and “trimean” used for calculating mean gene expression. Signaling family (‘Pathways’ in the original manuscript) annotations were taken from the original manuscript and represent groups of signaling molecules that function in related ways, and are often in the same protein family. For the Non-negative Matrix Factorization (NMF) signaling pattern analysis, the value of K was selected when the Cophenetic and Silhouette scores began to drop. Our analysis was guided by () and further details can be found in our source code.
#### CODEX data processing
Whole-cell segmentation was performed using Mesmer[51](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib51) for each image. To generate the necessary input of a two-channel TIFF, we used DAPI for the nuclear channel and a composite channel of CD45, [Vimentin](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/vimentin "Learn more about Vimentin from ScienceDirect's AI-generated Topic Pages"), and NaK-ATPase for the [membrane channel](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/membrane-channel "Learn more about membrane channel from ScienceDirect's AI-generated Topic Pages") generated through manual rescaling of each individual channel followed by merging the channels (see source code for further details). Mean pixel intensity was extracted from each cell segmentation mask, yielding a cell x [protein matrix](https://www.sciencedirect.com/topics/medicine-and-dentistry/matrix-protein "Learn more about protein matrix from ScienceDirect's AI-generated Topic Pages") which was carried forward for analysis in Seurat v4. Cells with very low or high raw DAPI expression (<10 or >250 on a UINT8 scale) were removed. Images were cropped if there were instances of tissue detachment during the run, or if areas had noticeable artifactual staining. All such cropping is carefully documented in the source code. Finally, we computed the sum of all the marker expression in each cell and removed cells falling above the 99th percentile of total expression or below the 1st percentile of total expression to reduce the impact of reporter precipitates and similar artifacts. One Seurat object was created for each sample using this method. Each sample was then internally normalized using Seurat’s implementation of centered log-ratio (CLR) for each feature independently (margin=1) which is recommended for analysis of [protein expression](https://www.sciencedirect.com/topics/immunology-and-microbiology/protein-expression "Learn more about protein expression from ScienceDirect's AI-generated Topic Pages") as in CITE-Seq data. Samples were then combined into one object and integrated using reciprocal PCA (RPCA) in Seurat using the FindIntegrationAnchors and IntegrateData functions with default parameters. This combined object was then scaled, a PCA was computed, and Louvain clustering was performed with visualization using UMAP with the first 30 principal components (PCs).
#### CODEX cell type annotation
A flow chart depicting our annotation strategy is provided in [Figure S5](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#figs5)A. We employed Louvain clustering in Seurat with multilevel refinement (algorithm=2) to determine the cell types present in the CODEX data. Clusters in the integrated Seurat object containing all 12 normal bone marrow (NBM) samples were annotated based on the marker expression of each cluster. Clusters that could not be easily annotated were subjected to a second round of subclustering of just those ambiguous clusters and were resolved. To address the issue of lateral spillover (i.e., bright expression from one cell type may “spill” into its neighbor thereby causing that cell to be mislabeled), we subjected each cluster to an additional round of subclustering, after which we visualized the subclusters on the image itself using cell phenotype maps in Napari. Clusters which represented lateral spillover artifacts were corrected to the appropriate cell type. Lastly, to ensure maximum annotation accuracy, any cell labels which had been changed through this correction process were subjected to one final round of subclustering, where spillover artifacts, if present, were corrected by relabeling the mislabeled subcluster appropriately. In cases where we did not observe a clear cluster for a certain cell type we knew should be present, we used manual gating to distinguish them, such as immunophenotypic HSCs, CLPs, and GMPs. We identified cells which stained positive for most markers which we used mast cell tryptase (MCT) to visualize because the staining frequency is inconsistent with the relative scarcity of bone marrow mast cells of ∼0.01%,[88](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib84) and we thus determined these cells to have highly artifactual staining. We then removed cells positive for MCT and many incompatible markers simultaneously from subsequent analysis. Based on the H&E staining, it is possible these were [eosinophils](https://www.sciencedirect.com/topics/immunology-and-microbiology/eosinophil-granulocyte "Learn more about eosinophils from ScienceDirect's AI-generated Topic Pages"), but without a confirmatory marker we elected to remove them as they are still relatively rare. We also removed obvious artifacts which were not cells. Other likely artifacts were noted, including [CD117](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/cd117 "Learn more about CD117 from ScienceDirect's AI-generated Topic Pages") staining in [lymphoid cells](https://www.sciencedirect.com/topics/immunology-and-microbiology/innate-lymphoid-cell "Learn more about lymphoid cells from ScienceDirect's AI-generated Topic Pages"), [CD19](https://www.sciencedirect.com/topics/immunology-and-microbiology/cd19 "Learn more about CD19 from ScienceDirect's AI-generated Topic Pages") in [T cells](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/t-cell "Learn more about T cells from ScienceDirect's AI-generated Topic Pages"), MCT staining in [erythroid cells](https://www.sciencedirect.com/topics/immunology-and-microbiology/erythroid-cell "Learn more about erythroid cells from ScienceDirect's AI-generated Topic Pages"), CD10 staining in [erythroid cells](https://www.sciencedirect.com/topics/medicine-and-dentistry/erythroid-cell "Learn more about erythroid cells from ScienceDirect's AI-generated Topic Pages"), and CD138 staining in a number of cell types. Some antibodies bound RBCs, especially in AML/NSM samples, but this did not confound analysis as RBCs were annotated easily by GYPC/CD71. ADIPOQ was not specific and thus excluded from downstream analysis. In total, 91.6% of segmented objects were annotated as a specific bone marrow cell type. We detail the markers and strategy used for cell type annotation in [Table S4](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#mmc4) and the Supplemental Material file called “Cell Type Annotation Guide” ([Data S1](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#mmc10)). The source code documents the clustering/subclustering process in a step-by-step manner. Cells were assigned a coarse (level 1) and detailed (level 2) type annotation. This process is further detailed in the source code.
We considered how this CODEX cell phenotyping might compare to our scRNA-seq atlas. While differences in protein and RNA expression and their dynamic ranges make subtle cell type correlation challenging, we did find that when we harmonized the cell labels based on the two modalities, the CODEX-based cell annotations mapped reasonably to the scRNA-seq-based cell annotations. However, as it is difficult to directly link CODEX data to scRNA-seq data on a very fine-grained level (for instance, the difference between a GMP and an early myeloid progenitor), we proceeded by keeping the CODEX and scRNA-seq annotations separate for discussion purposes, while retaining the harmonized labels in the Seurat object metadata.
#### Analysis of distance between Osteo- and Fibro-MSCs and bone
Due to the bone detachment issues, we were not able to systematically identify these two MSC populations. However, manual inspection of the images revealed cells with the expected [immunophenotype](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/immunophenotyping "Learn more about immunophenotype from ScienceDirect's AI-generated Topic Pages") of Osteo-MSCs in rare instances of the bone staying on the slide, or in the case of Fibro-MSCs, we found very rare PDPN+ CXCL12+ MCT- cells exclusively in the middle of the region where the [trabecular bone](https://www.sciencedirect.com/topics/medicine-and-dentistry/cancellous-bone "Learn more about trabecular bone from ScienceDirect's AI-generated Topic Pages") used to be. Therefore, Osteo- and Fibro-MSCs were manually annotated in the images (64 Fibro-MSC and 422 Osteo-MSC across all 12 samples). Segmentation masks for each manually annotated binary mask were constructed using a custom Python script, and [protein expression](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/protein-expression "Learn more about protein expression from ScienceDirect's AI-generated Topic Pages") was quantified for each cell as described above yielding a cell x [protein matrix](https://www.sciencedirect.com/topics/immunology-and-microbiology/matrix-protein "Learn more about protein matrix from ScienceDirect's AI-generated Topic Pages") for each mask. These masks were concatenated and joined with the raw data in the full CODEX Seurat object. The same markers were carried forward with the addition of CD56, which was added back to the analysis for all cells. This merged data set was renormalized using centered log ratio (CLR) normalization for each feature independently (margin=1) as above.
Adipo-, THY1+, Osteo-, and Fibro-MSC centroids were extracted from the Seurat object. Then distance to bone was calculated as described above with some modifications. The bone contour points were used as described above, but without downsampling. Given that Osteo- and Fibro-MSCs were frequently observed to occur within regions that contained trabecular bone, Osteo- and Fibro-MSC centroids that were found to overlap the bone annotations were added to the bone contour point list in order to correctly calculate a distance of zero for those cells. Non-normalized distance of each cell to bone was subsequently plotted, and significance between groups was calculated using a Wilcoxon rank-sum test in R.
#### Correlating cell type annotations by scRNA-seq and CODEX data
To generate the scRNA-seq and CODEX correlative plot ([Figures S5](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#figs5)F and S5G) to check the relationship between the two sets of cell type annotations, we first generated harmonized cell type labels, representing a set of simplified and common cell type labels we were confident in. We then calculated the average normalized expression of harmonized cell types in both the scRNA-seq (log-normalized) and CODEX (CLR-normalized) data. We also averaged the mean expression across cell types in cases where multiple higher-level cell types mapped to a single coarser cell type. Gene names were mapped to corresponding protein names when possible, yielding 49 matching protein/gene pairs to do the analysis. We then computed the Pearson correlation of the min-max normalized (within cell type) expression profiles of a scRNA-seq and a CODEX-based cell type. The highest matching scRNA-seq cell types to each CODEX cell type were extracted. We kept the scRNA-seq profile for osteoblasts labeled as such to help compare to the CODEX “endosteal” label.
#### CODEX neighborhood analysis
Neighborhood analysis was performed as previously described[52](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib52) using the level 2 cell type annotations for NBM, AML, and NSM samples. Note that AML samples had the additional cell type label of [NPM1](https://www.sciencedirect.com/topics/medicine-and-dentistry/npm1 "Learn more about NPM1 from ScienceDirect's AI-generated Topic Pages") mutant blasts which was not represented in either NBM or NSM samples. The 10 nearest neighbors of each cell were grouped into windows, which were clustered using K-means clustering with K=15 to get neighborhood assignments. Statistical significance of cell type enrichment within each neighborhood was calculated using a hypergeometric test. The hypergeometric test _p_ -value was calculated based on the following four numbers: (1) the number of cells of a given type in the neighborhood; (2) the total number of cells in the neighborhood; (3) the number of cells of a given type in the CODEX dataset; and (4) the total number of cells in the CODEX dataset. _P_ -values were adjusted for multiple hypothesis testing using the Benjamini-Hochberg method and significance was defined as p-adjusted < 0.05.[52](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib52)
#### Integration of CellChat and CODEX neighborhood analyses
To integrate the CellChat and CODEX neighborhood analysis results, we needed a measure which reasonably captures the association of two cell types in a statistically consistent fashion. Given our neighborhood analysis already yielded these associations, we first calculated a neighborhood co-localization score (NCLS) for a given cell type pair i and j in neighborhood k, NCLSijk, as following:NCLSijk=eijk−ek¯σekwhere eijk is the joint fold enrichment of cell type i and cell type j in neighborhood k which is defined as the sum of the two individual fold enrichment values, calculated as previously described.[52](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib52) And ek¯ and σek are the mean and standard deviation of the joint fold enrichment of all cell type pairs in the given neighborhood. NCLSijk represents how enriched two cell types are in a given neighborhood relative to all other cell type pairs in that neighborhood. The maximum of all NCLSijk values (maxNCLSijk) indicates the maximum co-enrichment of the two cell types across all neighborhoods. Next, we devised a combined measure for prioritization of L-R predictions by simultaneously considering CODEX-derived cell type co-localization score with the CellChat L-R predictions, called CellChat-CODEX Effect Size, CCESij for cell type i and cell type j. CCESij is defined as following:CCESij=(sij−min(sij)max(sij)−min(sij))∗(maxNCLSijk−min(maxNCLSijk)max(maxNCLSijk)−min(maxNCLSijk))where sij is the aggregate L-R pair prediction weight by CellChat.[37](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib37) We reasoned that this approach gives users a helpful single metric by which to prioritize L-R predictions which are also associated with co-localization in neighborhoods. We tabulated NCLSijk and maxNCLSijk scores along with the CellChat L-R predictions in [Table S3](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#mmc3). In cases where a CODEX cell type mapped to multiple scRNA-seq cell types, we averaged the NCLSijk values.
In addition, we calculated the significance of pairwise cell-cell interactions in the CODEX data using a permutation test. Specifically, we used the testInteractions function of the imcRtools R package (v1.4.2)[89](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib85) with method=”classic” in which the total number of interactions between cell type i and cell type j (defined by a distance of ≤ 100 pixels (∼51 μm) between cell centroids) is divided by the total number of cells of type j. Cell labels were permutated 500 times to generate a null distribution of the interaction. Probabilities that the cells were co-localized greater than expected by chance were aggregated using Stouffer’s method and recorded in [Table S3](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#mmc3) for each cell type combination.
#### CODEX reference mapping and validation of predicted ligand-receptor interactions
CODEX data generated using the new antibody panel including antibodies against the predicted ligand and [receptor proteins](https://www.sciencedirect.com/topics/medicine-and-dentistry/receptor-protein "Learn more about receptor proteins from ScienceDirect's AI-generated Topic Pages") were computationally processed the same way as NBM samples until the cell type annotation step. Instead of using clustering and manual annotation, we used automated label transfer using RPCA in Seurat v5 with default parameters to map the L-R CODEX datasets onto our NBM atlas and predict their cell labels using the 21 overlapping features (n = 6 matching samples; H10, H14, H26, H32, H38, H41). Briefly, each dataset was individually normalized with CLR normalization for each feature independently (margin=1), scaled, and a PCA was computed. These objects were then computationally merged, and cells were automatically annotated using the FindTransferAnchors function with the top 30 dimensions and k.filter=NA, followed by the MapQuery function with default parameters.
Accuracy of reference matching for cell types of interests was evaluated by manually checking the annotation masks on the images. Distance analysis was conducted using the imcRtools R package[89](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib85) with the minDistToCells function to compute the minimum distance from each cell to the source cell type of interest. Due to their rarity and lack of the TDT antibody in the original CODEX dataset used for reference mapping, CLPs were manually annotated in QuPath using CD34+ and TDT+ double positivity and then matching cell IDs were found in the Seurat object and downstream calculations proceeded as with the other cell types. The [receptor expression](https://www.sciencedirect.com/topics/medicine-and-dentistry/receptor-expression "Learn more about receptor expression from ScienceDirect's AI-generated Topic Pages") on the receiving cell type was then stratified by quartile based on the CLR-normalized expression within that cell type (0-25%, 25-75%, 75-100%), and data above the 99th percentile of cell distances in each level were filtered prior to plotting. Distance data from the 6 samples were pooled and the minimum distances to the source cell type were compared using a one-sided Wilcoxon rank-sum test.
#### NPM1 mutant cell classification and bone distance calculation
NPM1 mutant cells were identified using a Random [Trees](https://www.sciencedirect.com/topics/immunology-and-microbiology/tree "Learn more about Trees from ScienceDirect's AI-generated Topic Pages") (RT) classifier in QuPath.[76](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib86) Mutant cells were defined as those which expressed NPM1C but not CD141, as CD141 mature [myeloid cells](https://www.sciencedirect.com/topics/medicine-and-dentistry/myeloid-cell "Learn more about myeloid cells from ScienceDirect's AI-generated Topic Pages") stained with the NPM1C mutant-specific antibody even in negative staging marrow controls. Wild type (WT) cell annotation in diagnostic AML samples was aided with visualization of lymphoid markers such as CD3E or mesenchymal markers such as FOXC1. Approximately 100 manually identified mutant and WT cells each were annotated using the brush tool. The cell segmentation mask defined by Mesmer was imported using custom scripts. Using QuPath, for each marker we calculated mean, standard deviation, min, max, Haralick Features (min-NaN, max-NaN, distance-1, bins-32), and the default QuPath morphologic features for each cell detection object. We then trained and applied an RT classifier to all cells from 5 samples and assigned a binary classification of NPM1_Mutant or NPM1_WT to each cell. Classification was manually evaluated in collaboration with clinical hematopathologists, and additional informative annotations were added in an iterative, human-in-the-loop fashion until the classification achieved optimal performance. Detection measurements were exported and merged with the default Seurat analysis by matching cells to their nearest spatial coordinates. We filtered out any leukemic cells which mapped to incompatible (e.g., lymphoid, mesenchymal) or very rare cell types. Spatial analysis with respect to bone was performed using the spatstat R package using the same approach as the healthy neighborhood distance to bone analysis.
#### Negative staging marrow and AML cell type classification
The same panel was used for AML/NSM CODEX as the femoral head samples, with the exception of [CD19](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/cd19 "Learn more about CD19 from ScienceDirect's AI-generated Topic Pages") being swapped for NPM1C. AML and NSM samples were computationally processed the same way as NBM samples until the cell type annotation step. Instead of using clustering and manual annotation, we used automated label transfer using RPCA in Seurat to map the NSM and AML datasets onto our NBM atlas and predict their cell labels as described above. Each sample was individually processed (CLR normalization, scaling, PCA) and then projected onto the NBM reference using the FindTransferAnchors function with the top 30 dimensions and k.filter=NA followed by the MapQuery function with default parameters. For comparison of NSM and AML myeloid classification proportions, we considered cells which mapped to GMP, GMP/Myeloblast, Early Myeloid Progenitor, Intermediate Myeloid, Monocyte, Non-Classical Monocyte, pDC, or Macrophages. NSM samples were chosen as the control to account for differences between sample site (femoral head versus iliac crest), sample processing, and generalized stress associated with having cancer.
#### Annotation of microanatomical structures
Microanatomical structures for proximity analysis were manually annotated. Bone was manually annotated based on morphology and a combination of CD56 expression, exclusion of vessels, and correlation with the corresponding H&E image. Adipocytes were annotated using [CD146](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/cd146 "Learn more about CD146 from ScienceDirect's AI-generated Topic Pages") and morphology, with VE-Cadherin-positive arterioles and sinusoids excluded. Structures with characteristic ASMA+ cells surrounding a layer of VECAD+ CXCL12+ endothelial cells were labeled as arterioles, while vessels with VECAD/CD34 expression but not ASMA or CXCL12 were labeled as sinusoids. CXCL12+ [stroma](https://www.sciencedirect.com/topics/medicine-and-dentistry/stroma "Learn more about stroma from ScienceDirect's AI-generated Topic Pages") and CD163+ macrophages were annotated using manual thresholding on a per-image basis in QuPath. Macrophages were considered a structure due to their considerable cytoplasmic projections which were not well captured by traditional cell/nuclear segmentation-based approaches. Stroma, macrophages, arterioles, and bone were annotated across each entire image. Adipocytes and sinusoids were prohibitively abundant to annotate across the entire image, and thus large regions of interest (ROIs) with hundreds of each structure were annotated per each image.
#### Structural proximity analysis
We measured the proximity to microanatomical structures (adipocyte, arteriolar, sinusoid, bone, macrophage, stroma) from different cell types, neighborhoods and structures using the median distance and applying a Poisson point process model. The key properties of the Poisson point process include the independence of events within non-overlapping regions and the probability of occurrence of events following a [Poisson distribution](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/poisson-distribution "Learn more about Poisson distribution from ScienceDirect's AI-generated Topic Pages"). This makes the Poisson point process particularly useful for modeling rare cells.λ(u)=exp(β0+β1S(u))
The model is applied to our data where u is the x-y coordinate of a cell, λ(u) is probability of a cell, β0 is the population of cells to optimize, β1 is decay rate of S(u) to optimize, S(u) is distance to the structure. The value of λ(u) is 0 when there are no cells at position u, and 1 if there is a cell. When fitting the data, λ(u) ranges from 0 to 1. β0 is highly correlated to the [population size](https://www.sciencedirect.com/topics/immunology-and-microbiology/population-size "Learn more about population size from ScienceDirect's AI-generated Topic Pages"), which can be counted directly. And β1 is what we are interested in, it represents the proximity. A small β1 value indicates that the distance is close. The Poisson point process model is implemented in the spatstat R package. We used the “ppm” function from that package to calculate the β0 and β1. We quantified spatial proximity using annotation mask images and a cells table that includes cell locations and labels (cell types and neighborhoods). To extract contour points for the structures, we used the cv2 package in Python. We then randomly sampled 10% of these points to create a spatial point pattern for the structures. To prevent overemphasis of outliers, we excluded rare cell types from our analysis. Specifically, if a cell type has a population of less than 10 in a given sample, it is excluded from that sample's rank calculation. Additionally, if there are fewer than 6 samples for a particular cell type after all calculations have been completed, that cell type is also excluded from our analysis. We utilized rank proximity as our final metric, rather than physical distance, because the sizes of different marrow cavities may differ, and an internally normalized metric was more appropriate. To determine the proximity of cell types, neighborhoods or structures to structures, we used a normalized rank calculation. This involved ranking the - β1 value for cells in comparison in every sample and scaling the ranking number to a range of 0-1 within each sample. We conducted a one-sided permutation test without replacement to determine if the cell types, neighborhoods, or other structures are significantly close to the target structure. We permuted the cell type labels 100 times and calculated their median distance across the 100 permutations. The proportion of distances smaller than that of the cell type is the p-value. We used Stouffer’s method for the calculation of p-values for cell types aggregated from the permutation test p-values.
## Acknowledgments
The authors thank the Children’s Hospital of Philadelphia Pathology Core for their considerable technical expertise and assistance in handling human tissue sections and the Research Information Services for providing computing support. We thank the many clinical contributors who assisted in procuring primary human samples through Penn Presbyterian Medical Center. Lastly, we also thank the University of Pennsylvania Cytomics and Cell Sorting Shared Resource Laboratory for their assistance in sorting primary MSC samples. This research was supported by the NIH Common Fund, through the Office of Strategic Coordination/Office of the NIH Director under awards [U54 HL156090](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#gs2) and [U54HL165442](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#gs2) (to K.T.). Additional support includes National Institutes of Health of United States of America grants [U2C CA233285](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#gs3) (to K.T.), [R01AG069401](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#gs3) (to L.Q.), and [P30AR069619](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#gs3) (to The University of Pennsylvania). M.P.D. was supported by NIH NIDDK [T32DK007314](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#gs4). S. Bandyopadhyay was supported by NIH [T32 GM007170](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#gs5), [T32 HL007439](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#gs5), and [F30CA277965](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#gs5).
### Author contributions
K.T., L.Q., and S. Bandyopadhyay conceptualized and designed the project. S. Bandyopadhyay, K.J.A., M.P.D., A.T., Chider Chen, and Y.L. performed the data generation and experiments. S. Bandyopadhyay, K.J.A., M.P.D., J.H.S., M.P., D.S., G.D., J.H., I.Z., Y.L., B.X., and T.I. performed analysis and wrote software and made figures. S. Bandyopadhyay, K.J.A., M.P.D., J.H.S., Changya Chen, J.X., M.R., Z.M., C.D., V.P., O.S., D.O., I.M., M.C., C.N., L.Q., and K.T. provided critical conceptual contributions and technical expertise. C.N., S. Bhattacharyya, O.S., and Changya Chen assisted with reagent validation and provided clinical samples. S. Bandyopadhyay, K.J.A., M.P.D., L.Q., and K.T. wrote the manuscript.
### Declaration of interests
Maillard: Garuda Therapeutics: Membership on an entity's Board of Directors or advisory committees; Regeneron: Research Funding; Genentech: Research Funding. Carroll: Cartography Bioscences: Membership on an entity's Board of Directors or advisory committees; Janssen Pharmaceuticals: Consultancy.
## Supplemental information
Download all supplementary files included with this article
Table S1. Table summarizing the known clinical information for NBM/AML/NSM samples, related to Figures 1, 2, 3, 4, 5, 6, and 7. 
Table S2. List of differentially expressed genes for all annotated cell types, related to Figures 1 and 2. A Wilcoxon rank-sum test was used comparing normalized gene expression in each cell type against all other cell types, and the default 0.25 threshold of log2-fold change was applied.
Table S3. Results of the CellChat and CODEX co-localization enrichment analyses, related to Figures 3, 4, and 5. For CellChat analysis, each significant ligand-receptor pair is included with information including the source, target, significance, and the pathway the interaction falls into. For CODEX co-localization enrichment analysis, this includes 1) columns per healthy bone marrow (NBM) neighborhood with the sum of the enrichment scores of the cell type pair scaled by neighborhood, 2) a column which has the maximum scaled co-localization score across any neighborhood, 3) a column with the pairwise cell-cell interaction significance computed via permutation testing aggregated using Stouffer’s method, and 4) CODEX/CellChat Effect Size (CCES) scores between cell type pairs.
Table S4. Summary of the cell types detected using CODEX multiplexed imaging and the markers used to annotate the different cell types, related to Figure 4. 
Table S5. List of antibodies used in the study including the vendors, clone information, and other relevant information about antibodies to allow for replication of our panel, related to Figures 4 and 7. 
Table S6. Imaging parameters including the exposure times and cycle/channel assignment for each antibody in the panel, related to Figures 4 and 7. 
Table S7. Table showing CODEX/CellChat CCES analysis on a cell-type-pair level, including intermediate metrics used to calculate the score including NCLS and CellChat network weights, related to Figures 3, 4, and 5. 
Table S8. Comprehensive metrics for the structural analysis, related to Figure 6. Coefficient 1 is β0 and correlated with population size, and coefficient 2 is β1 and corresponds to the distance to the structure. _P_ -value is derived from permutation testing on a per-sample basis. Both rank and normalized rank (0 is the most proximal, 1 is furthest) are included. We computed the variability in distance as the interquartile range (75th percentile – 25th percentile of distances across cells measured in microns).
Table S9. List of qPCR primers used in this study, related to Figure S3. 
Data S1. Cell-type annotation guide, related to Figure 4 and STAR Methods. 
Recommended articles
## References
  1. [1](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bbib1)
J. Fröbel, T. Landspersky, G. Percin, C. Schreck, S. Rahmig, A. Ori, D. Nowak, M. Essers, C. Waskow, R.A.J. Oostendorp
The hematopoietic bone marrow niche ecosystem
Front. Cell Dev. Biol., 9 (2021), Article 705410, 
  2. [2](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bbib2)
S. Pinho, P.S. Frenette
Haematopoietic stem cell activity and interactions with the niche
Nat. Rev. Mol. Cell Biol., 20 (2019), pp. 303-320, 
  3. [3](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bbib3)
L. Ding, S.J. Morrison
Haematopoietic stem cells and early lymphoid progenitors occupy distinct bone marrow niches
Nature, 495 (2013), pp. 231-235, 
  4. [4](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bbib4)
A. Greenbaum, Y.M. Hsu, R.B. Day, L.G. Schuettpelz, M.J. Christopher, J.N. Borgerding, T. Nagasawa, D.C. Link
CXCL12 in early mesenchymal progenitors is required for haematopoietic stem-cell maintenance
Nature, 495 (2013), pp. 227-230, 
  5. [5](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bbib5)
M. Zhao, F. Tao, A. Venkatraman, Z. Li, S.E. Smith, J. Unruh, S. Chen, C. Ward, P. Qian, J.M. Perry, _et al._
N-cadherin-expressing bone and marrow stromal progenitor cells maintain reserve hematopoietic stem cells
Cell Rep., 26 (2019), pp. 652-669.e6, 
[View PDF](https://www.sciencedirect.com/science/article/pii/S2211124718320539/pdfft?md5=cfdb63180c0ed5c6ca1f61ade4d5606f&pid=1-s2.0-S2211124718320539-main.pdf)[View article](https://www.sciencedirect.com/science/article/pii/S2211124718320539)
  6. [6](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bbib6)
C. Xu, X. Gao, Q. Wei, F. Nakahara, S.E. Zimmerman, J. Mar, P.S. Frenette
Stem cell factor is selectively secreted by arterial endothelial cells in bone marrow
Nat. Commun., 9 (2018), Article 2449, 
  7. [7](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bbib7)
A.N. Tikhonova, I. Dolgalev, H. Hu, K.K. Sivaraj, E. Hoxha, Á. Cuesta-Domínguez, S. Pinho, I. Akhmetzyanova, J. Gao, M. Witkowski, _et al._
The bone marrow microenvironment at single-cell resolution
Nature, 569 (2019), pp. 222-228, 
  8. [8](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bbib8)
L. Zhong, L. Yao, R.J. Tower, Y. Wei, Z. Miao, J. Park, R. Shrestha, L. Wang, W. Yu, N. Holdreith, _et al._
Single cell transcriptomics identifies a unique adipose lineage cell population that regulates bone marrow environment
eLife, 9 (2020), Article e54695, 
  9. [9](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bbib9)
C. Baccin, J. Al-Sabah, L. Velten, P.M. Helbling, F. Grünschläger, P. Hernández-Malmierca, C. Nombela-Arrieta, L.M. Steinmetz, A. Trumpp, S. Haas
Combined single-cell and spatial transcriptomics reveal the molecular, cellular and spatial bone marrow niche organization
Nat. Cell Biol., 22 (2020), pp. 38-48, 
  10. [10](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bbib10)
N. Baryawno, D. Przybylski, M.S. Kowalczyk, Y. Kfoury, N. Severe, K. Gustafsson, K.D. Kokkaliaris, F. Mercier, M. Tabaka, M. Hofree, _et al._
A cellular taxonomy of the bone marrow stroma in homeostasis and leukemia
Cell, 177 (2019), pp. 1915-1932.e16, 
[View PDF](https://www.sciencedirect.com/science/article/pii/S0092867419304593/pdfft?md5=f74aa9d453a152602323daae54d74abd&pid=1-s2.0-S0092867419304593-main.pdf)[View article](https://www.sciencedirect.com/science/article/pii/S0092867419304593)
  11. [11](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bbib11)
J.M. Granja, S. Klemm, L.M. McGinnis, A.S. Kathiria, A. Mezger, M.R. Corces, B. Parks, E. Gars, M. Liedtke, G.X.Y. Zheng, _et al._
Single-cell multiomic analysis identifies regulatory programs in mixed-phenotype acute leukemia
Nat. Biotechnol., 37 (2019), pp. 1458-1465, 
  12. [12](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bbib12)
K.A. Oetjen, K.E. Lindblad, M. Goswami, G. Gui, P.K. Dagur, C. Lai, L.W. Dillon, J.P. McCoy, C.S. Hourigan
Human bone marrow assessment by single-cell RNA sequencing, mass cytometry, and flow cytometry
JCI Insight, 3 (2018), Article e124928, 
  13. [13](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bbib13)
S. Triana, D. Vonficht, L. Jopp-Saile, S. Raffel, R. Lutz, D. Leonce, M. Antes, P. Hernández-Malmierca, D. Ordoñez-Rueda, B. Ramasz, _et al._
Single-cell proteo-genomic reference maps of the hematopoietic system enable the purification and massive profiling of precisely defined cell states
Nat. Immunol., 22 (2021), pp. 1577-1589, 
  14. [14](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bbib14)
P.E. Boulais, T. Mizoguchi, S. Zimmerman, F. Nakahara, J. Vivié, J.C. Mar, A. van Oudenaarden, P.S. Frenette
The majority of CD45– Ter119– CD31– bone marrow cell fraction is of hematopoietic origin and contains erythroid and lymphoid progenitors
Immunity, 49 (2018), pp. 627-639.e6, 
[View PDF](https://www.sciencedirect.com/science/article/pii/S1074761318303820/pdfft?md5=ab02cb64f475e81f1389fac2845fb1e6&pid=1-s2.0-S1074761318303820-main.pdf)[View article](https://www.sciencedirect.com/science/article/pii/S1074761318303820)
  15. [15](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bbib15)
F. Dazzi, R. Ramasamy, S. Glennie, S.P. Jones, I. Roberts
The role of mesenchymal stem cells in haemopoiesis
Blood Rev., 20 (2006), pp. 161-171, 
[View PDF](https://www.sciencedirect.com/science/article/pii/S0268960X05000597/pdfft?md5=065f7875a701e73fc2b30c7b89980214&pid=1-s2.0-S0268960X05000597-main.pdf)[View article](https://www.sciencedirect.com/science/article/pii/S0268960X05000597)
  16. [16](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bbib16)
M.M.E. de Jong, Z. Kellermayer, N. Papazian, S. Tahri, D. Hofste Op Bruinink, R. Hoogenboezem, M.A. Sanders, P.C. van de Woestijne, P.K. Bos, C. Khandanpour, _et al._
The multiple myeloma microenvironment is defined by an inflammatory stromal cell landscape
Nat. Immunol., 22 (2021), pp. 769-780, 
  17. [17](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bbib17)
H. Li, S. Bräunig, P. Dhapolar, G. Karlsson, S. Lang, S. Scheding
Identification of phenotypically, functionally, and anatomically distinct stromal niche populations in human bone marrow based on single-cell RNA sequencing
eLife, 12 (2023), Article e81656, 
  18. [18](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bbib18)
S. Ennis, A. Conforte, E. O’Reilly, J.S. Takanlu, T. Cichocka, S.P. Dhami, P. Nicholson, P. Krebs, P. Ó Broin, E. Szegezdi
Cell-cell interactome of the hematopoietic niche and its changes in acute myeloid leukemia
iScience, 26 (2023), Article 106943, 
[View PDF](https://www.sciencedirect.com/science/article/pii/S2589004223010209/pdfft?md5=72d8f7a926fe6615f64e95f1a03d59e7&pid=1-s2.0-S2589004223010209-main.pdf)[View article](https://www.sciencedirect.com/science/article/pii/S2589004223010209)
  19. [19](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bbib19)
Z. Xie, W. Yu, G. Ye, J. Li, G. Zheng, W. Liu, J. Lin, Z. Su, Y. Che, F. Ye, _et al._
Single-cell RNA sequencing analysis of human bone-marrow-derived mesenchymal stem cells and functional subpopulation identification
Exp. Mol. Med., 54 (2022), pp. 483-492, 
  20. [20](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bbib20)
Z. Wang, C. Chai, R. Wang, Y. Feng, L. Huang, Y. Zhang, X. Xiao, S. Yang, X. Zhang
Single-cell transcriptome atlas of human mesenchymal stem cells exploring cellular heterogeneity
Clin. Transl. Med., 11 (2021), Article e650, 
  21. [21](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bbib21)
Y. Goltsev, N. Samusik, J. Kennedy-Darling, S. Bhate, M. Hale, G. Vazquez, S. Black, G.P. Nolan
Deep profiling of mouse splenic architecture with CODEX multiplexed imaging
Cell, 174 (2018), pp. 968-981.e15, 
[View PDF](https://www.sciencedirect.com/science/article/pii/S0092867418309048/pdfft?md5=897230a8764b100ddf1e5d60dae6060c&pid=1-s2.0-S0092867418309048-main.pdf)[View article](https://www.sciencedirect.com/science/article/pii/S0092867418309048)
  22. [22](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bbib24)
I. Dolgalev, A.N. Tikhonova
Connecting the dots: resolving the bone marrow niche heterogeneity
Front. Cell Dev. Biol., 9 (2021), Article 622519, 
  23. [23](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bbib22)
M. Álvarez-Viejo, Y. Menéndez-Menéndez, J. Otero-Hernández
CD271 as a marker to identify mesenchymal stem cells from diverse sources before culture
World J. Stem Cells, 7 (2015), pp. 470-476, 
  24. [24](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bbib23)
L. Harkness, W. Zaher, N. Ditzel, A. Isa, M. Kassem
CD146/MCAM defines functionality of human bone marrow stromal stem cell populations
Stem Cell Res. Ther., 7 (2016), Article 4, 
  25. [25](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bbib25)
C.K.F. Chan, G.S. Gulati, R. Sinha, J.V. Tompkins, M. Lopez, A.C. Carter, R.C. Ransom, A. Reinisch, T. Wearda, M. Murphy, _et al._
Identification of the human skeletal stem cell
Cell, 175 (2018), pp. 43-56.e21, 
[View PDF](https://www.sciencedirect.com/science/article/pii/S0092867418309565/pdfft?md5=556ca71f83e4900df20b730f391f3396&pid=1-s2.0-S0092867418309565-main.pdf)[View article](https://www.sciencedirect.com/science/article/pii/S0092867418309565)
  26. [26](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bbib26)
M. Dominici, K. Le Blanc, I. Mueller, I. Slaper-Cortenbach, F. Marini, D. Krause, R. Deans, A. Keating, D. Prockop, E. Horwitz
Minimal criteria for defining multipotent mesenchymal stromal cells. The International Society for Cellular Therapy position statement
Cytotherapy, 8 (2006), pp. 315-317, 
[View PDF](https://www.sciencedirect.com/science/article/pii/S1465324906708817/pdfft?md5=c67082a52f98e93327a5247619682f56&pid=1-s2.0-S1465324906708817-main.pdf)[View article](https://www.sciencedirect.com/science/article/pii/S1465324906708817)
  27. [27](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bbib27)
S. Méndez-Ferrer, T.V. Michurina, F. Ferraro, A.R. Mazloom, B.D. Macarthur, S.A. Lira, D.T. Scadden, A. Ma'ayan, G.N. Enikolopov, P.S. Frenette
Mesenchymal and haematopoietic stem cells form a unique bone marrow niche
Nature, 466 (2010), pp. 829-834, 
  28. [28](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bbib28)
G.S. Gulati, S.S. Sikandar, D.J. Wesche, A. Manjunath, A. Bharadwaj, M.J. Berger, F. Ilagan, A.H. Kuo, R.W. Hsieh, S. Cai, _et al._
Single-cell transcriptional diversity is a hallmark of developmental potential
Science, 367 (2020), pp. 405-411, 
  29. [29](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bbib29)
Y. Peng, S. Wu, Y. Li, J.L. Crane
Type H blood vessels in bone modeling and remodeling
Theranostics, 10 (2020), pp. 426-436, 
  30. [30](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bbib30)
L. Biswas, J. Chen, J. De Angelis, A. Singh, C. Owen-Woods, Z. Ding, J.M. Pujol, N. Kumar, F. Zeng, S.K. Ramasamy, A.P. Kusumbe
Lymphatic vessels in bone support regeneration after injury
Cell, 186 (2023), pp. 382-397.e24, 
[View PDF](https://www.sciencedirect.com/science/article/pii/S0092867422015744/pdfft?md5=c11de72a558df34601a64bf8893c0d77&pid=1-s2.0-S0092867422015744-main.pdf)[View article](https://www.sciencedirect.com/science/article/pii/S0092867422015744)
  31. [31](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bbib31)
J.Y. Lee, C. Park, Y.P. Cho, E. Lee, H. Kim, P. Kim, S.H. Yun, Y.S. Yoon
Podoplanin-expressing cells derived from bone marrow play a crucial role in postnatal lymphatic neovascularization
Circulation, 122 (2010), pp. 1413-1425, 
  32. [32](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bbib32)
L. Jardine, S. Webb, I. Goh, M. Quiroga Londoño, G. Reynolds, M. Mather, B. Olabi, E. Stephenson, R.A. Botting, D. Horsfall, _et al._
Blood and immune development in human fetal bone marrow and Down syndrome
Nature, 598 (2021), pp. 327-331, 
  33. [33](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bbib33)
Z. Wang, X. Li, J. Yang, Y. Gong, H. Zhang, X. Qiu, Y. Liu, C. Zhou, Y. Chen, J. Greenbaum, _et al._
Single-cell RNA sequencing deconvolutes the in vivo heterogeneity of human bone marrow-derived mesenchymal stem cells
Int. J. Biol. Sci., 17 (2021), pp. 4192-4206, 
  34. [34](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bbib34)
J. Isern, A. García-García, A.M. Martín, L. Arranz, D. Martín-Pérez, C. Torroja, F. Sánchez-Cabo, S. Méndez-Ferrer
The neural crest is a source of mesenchymal stem cells with specialized hematopoietic stem cell niche function
eLife, 3 (2014), Article e03696, 
  35. [35](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bbib35)
S. Méndez-Ferrer, D. Bonnet, D.P. Steensma, R.P. Hasserjian, I.M. Ghobrial, J.G. Gribben, M. Andreeff, D.S. Krause
Bone marrow niches in haematological malignancies
Nat. Rev. Cancer, 20 (2020), pp. 285-298, 
  36. [36](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bbib36)
S. Comazzetto, B. Shen, S.J. Morrison
Niches that regulate stem cells and hematopoiesis in adult bone marrow
Dev. Cell, 56 (2021), pp. 1848-1860, 
[View PDF](https://www.sciencedirect.com/science/article/pii/S1534580721004780/pdfft?md5=e93dfb68b17d327d71bee6b89fc485d3&pid=1-s2.0-S1534580721004780-main.pdf)[View article](https://www.sciencedirect.com/science/article/pii/S1534580721004780)
  37. [37](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bbib37)
S. Jin, C.F. Guerrero-Juarez, L. Zhang, I. Chang, R. Ramos, C.-H. Kuan, P. Myung, M.V. Plikus, Q. Nie
Inference and analysis of cell-cell communication using CellChat
Nat. Commun., 12 (2021), Article 1088, 
  38. [38](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bbib38)
L.M. Calvi, G.B. Adams, K.W. Weibrecht, J.M. Weber, D.P. Olson, M.C. Knight, R.P. Martin, E. Schipani, P. Divieti, F.R. Bringhurst, _et al._
Osteoblastic cells regulate the haematopoietic stem cell niche
Nature, 425 (2003), pp. 841-846, 
  39. [39](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bbib39)
J. Zhang, C. Niu, L. Ye, H. Huang, X. He, W.G. Tong, J. Ross, J. Haug, T. Johnson, J.Q. Feng, _et al._
Identification of the haematopoietic stem cell niche and control of the niche size
Nature, 425 (2003), pp. 836-841, 
  40. [40](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bbib40)
T. Sugiyama, H. Kohara, M. Noda, T. Nagasawa
Maintenance of the hematopoietic stem cell pool by CXCL12-CXCR4 chemokine signaling in bone marrow stromal cell niches
Immunity, 25 (2006), pp. 977-988, 
[View PDF](https://www.sciencedirect.com/science/article/pii/S1074761306005152/pdfft?md5=bb018e9bdc9c1b073d2743c3474834e2&pid=1-s2.0-S1074761306005152-main.pdf)[View article](https://www.sciencedirect.com/science/article/pii/S1074761306005152)
  41. [41](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bbib41)
L. Ding, T.L. Saunders, G. Enikolopov, S.J. Morrison
Endothelial and perivascular cells maintain haematopoietic stem cells
Nature, 481 (2012), pp. 457-462, 
  42. [42](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bbib42)
U. Blank, S. Karlsson
TGF-β signaling in the control of hematopoietic stem cells
Blood, 125 (2015), pp. 3542-3550, 
[View PDF](https://www.sciencedirect.com/science/article/pii/S0006497120315731/pdfft?md5=da08744982e2f0fdfffcad4a91329ba6&pid=1-s2.0-S0006497120315731-main.pdf)[View article](https://www.sciencedirect.com/science/article/pii/S0006497120315731)
  43. [43](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bbib43)
P. Jiang, Y. Zhang, B. Ru, Y. Yang, T. Vu, R. Paul, A. Mirza, G. Altan-Bonnet, L. Liu, E. Ruppin, _et al._
Systematic investigation of cytokine signaling activity at the tissue and single-cell levels
Nat. Methods, 18 (2021), pp. 1181-1191, 
  44. [44](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bbib44)
C. Böiers, N. Buza-Vidas, C.T. Jensen, C.J.H. Pronk, S. Kharazi, L. Wittmann, E. Sitnicka, A. Hultquist, S.E.W. Jacobsen
Expression and role of FLT3 in regulation of the earliest stage of normal granulocyte-monocyte progenitor development
Blood, 115 (2010), pp. 5061-5068, 
[View PDF](https://www.sciencedirect.com/science/article/pii/S0006497120348643/pdfft?md5=511df13b4e8336235da285217f5d777d&pid=1-s2.0-S0006497120348643-main.pdf)[View article](https://www.sciencedirect.com/science/article/pii/S0006497120348643)
  45. [45](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bbib45)
E. Sitnicka, D. Bryder, K. Theilgaard-Mönch, N. Buza-Vidas, J. Adolfsson, S.E.W. Jacobsen
Key role of flt3 ligand in regulation of the common lymphoid progenitor but not in maintenance of the hematopoietic stem cell pool
Immunity, 17 (2002), pp. 463-472, 
[View PDF](https://www.sciencedirect.com/science/article/pii/S1074761302004193/pdfft?md5=f76a6f00d3201f3e5e61bf2028056d02&pid=1-s2.0-S1074761302004193-main.pdf)[View article](https://www.sciencedirect.com/science/article/pii/S1074761302004193)
  46. [46](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bbib46)
A. Cordeiro Gomes, T. Hara, V.Y. Lim, D. Herndler-Brandstetter, E. Nevius, T. Sugiyama, S. Tani-ichi, S. Schlenner, E. Richie, H.-R. Rodewald, _et al._
Hematopoietic stem cell niches produce lineage-instructive signals to control multipotent progenitor differentiation
Immunity, 45 (2016), pp. 1219-1231, 
[View PDF](https://www.sciencedirect.com/science/article/pii/S1074761316304769/pdfft?md5=e92bd728da98ad8714819a379c9941fd&pid=1-s2.0-S1074761316304769-main.pdf)[View article](https://www.sciencedirect.com/science/article/pii/S1074761316304769)
  47. [47](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bbib47)
A. Sambandam, I. Maillard, V.P. Zediak, L. Xu, R.M. Gerstein, J.C. Aster, W.S. Pear, A. Bhandoola
Notch signaling controls the generation and differentiation of early T lineage progenitors
Nat. Immunol., 6 (2005), pp. 663-670, 
  48. [48](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bbib48)
M. Corselli, C.J. Chin, C. Parekh, A. Sahaghian, W. Wang, S. Ge, D. Evseenko, X. Wang, E. Montelatici, L. Lazzari, _et al._
Perivascular support of human hematopoietic stem/progenitor cells
Blood, 121 (2013), pp. 2891-2901, 
[View PDF](https://www.sciencedirect.com/science/article/pii/S0006497120498027/pdfft?md5=714886b32ed0550f37c8c5586ae91037&pid=1-s2.0-S0006497120498027-main.pdf)[View article](https://www.sciencedirect.com/science/article/pii/S0006497120498027)
  49. [49](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bbib49)
A. Fidanza, P.S. Stumpf, P. Ramachandran, S. Tamagno, A. Babtie, M. Lopez-Yrigoyen, A.H. Taylor, J. Easterbrook, B.E.P. Henderson, R. Axton, _et al._
Single-cell analyses and machine learning define hematopoietic progenitor and HSC-like cells derived from human PSCs
Blood, 136 (2020), pp. 2893-2904, 
[View PDF](https://www.sciencedirect.com/science/article/pii/S0006497120778651/pdfft?md5=ce461926e156b8a8c07d35841c7fba26&pid=1-s2.0-S0006497120778651-main.pdf)[View article](https://www.sciencedirect.com/science/article/pii/S0006497120778651)
  50. [50](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bbib50)
V. Calvanese, S. Capellera-Garcia, F. Ma, I. Fares, S. Liebscher, E.S. Ng, S. Ekstrand, J. Aguadé-Gorgorió, A. Vavilina, D. Lefaudeux, _et al._
Mapping human haematopoietic stem cells from haemogenic endothelium to birth
Nature, 604 (2022), pp. 534-540, 
  51. [51](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bbib51)
N.F. Greenwald, G. Miller, E. Moen, A. Kong, A. Kagel, T. Dougherty, C.C. Fullaway, B.J. McIntosh, K.X. Leow, M.S. Schwartz, _et al._
Whole-cell segmentation of tissue images with human-level performance using large-scale data annotation and deep learning
Nat. Biotechnol., 40 (2022), pp. 555-565, 
  52. [52](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bbib52)
C.M. Schürch, S.S. Bhate, G.L. Barlow, D.J. Phillips, L. Noti, I. Zlobec, P. Chu, S. Black, J. Demeter, D.R. McIlwain, _et al._
Coordinated cellular neighborhoods orchestrate antitumoral immunity at the colorectal cancer invasive front
Cell, 182 (2020), pp. 1341-1359.e19, 
[View PDF](https://www.sciencedirect.com/science/article/pii/S0092867420308709/pdfft?md5=69caea8cbbbeb795b57ddbeb38465ae3&pid=1-s2.0-S0092867420308709-mainext.pdf)[View article](https://www.sciencedirect.com/science/article/pii/S0092867420308709)
  53. [53](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bbib53)
Q. Wu, J. Zhang, C.B. Johnson, A. Slaughter, B. Weinhaus, M. McKnight, B.E. Sherman, T.-Y. Shao, B. Song, M.D. Filippi, _et al._
Resilient anatomy and local microplasticity of naïve and stress hematopoiesis
Nature, 627 (2024), pp. 839-846, 
  54. [54](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bbib54)
F. Ishikawa, S. Yoshida, Y. Saito, A. Hijikata, H. Kitamura, S. Tanaka, R. Nakamura, T. Tanaka, H. Tomiyama, N. Saito, _et al._
Chemotherapy-resistant human AML stem cells home to and engraft within the bone-marrow endosteal region
Nat. Biotechnol., 25 (2007), pp. 1315-1321, 
  55. [55](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bbib55)
K.S. Stewart, Z. Zhou, P. Zweidler-McKay, E.S. Kleinerman
Delta-like ligand 4-Notch signaling regulates bone marrow-derived pericyte/vascular smooth muscle cell formation
Blood, 117 (2011), pp. 719-726, 
[View PDF](https://www.sciencedirect.com/science/article/pii/S0006497120592215/pdfft?md5=5217c7946ba090b895e644bf83c4c3c6&pid=1-s2.0-S0006497120592215-main.pdf)[View article](https://www.sciencedirect.com/science/article/pii/S0006497120592215)
  56. [56](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bbib56)
J.B. Tefft, J.L. Bays, A. Lammers, S. Kim, J. Eyckmans, C.S. Chen
Notch1 and notch3 coordinate for pericyte-induced stabilization of vasculature
Am. J. Physiol. Cell Physiol., 322 (2022), pp. C185-C196, 
  57. [57](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bbib57)
A. Baddeley, E. Rubak, R. Turner
Spatial point patterns: methodology and applications with R
(First edition), Routledge (2015)
  58. [58](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bbib58)
S.J. Morrison, D.T. Scadden
The bone marrow niche for haematopoietic stem cells
Nature, 505 (2014), pp. 327-334, 
  59. [59](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bbib59)
M. Huang, D. Thomas, M.X. Li, W. Feng, S.M. Chan, R. Majeti, B.S. Mitchell
Role of cysteine 288 in nucleophosmin cytoplasmic mutations: sensitization to toxicity induced by arsenic trioxide and bortezomib
Leukemia, 27 (2013), pp. 1970-1980, 
  60. [60](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bbib60)
H.J. Uckelmann, E.L. Haarer, R. Takeda, E.M. Wong, C. Hatton, C. Marinaccio, F. Perner, M. Rajput, N.J.C. Antonissen, Y. Wen, _et al._
Mutant NPM1 directly regulates oncogenic transcription in acute myeloid leukemia
Cancer Discov., 13 (2023), pp. 746-765, 
  61. [61](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bbib61)
A. Hérault, M. Binnewies, S. Leong, F.J. Calero-Nieto, S.Y. Zhang, Y.A. Kang, X. Wang, E.M. Pietras, S.H. Chu, K. Barry-Holson, _et al._
Myeloid progenitor cluster formation drives emergency and leukaemic myelopoiesis
Nature, 544 (2017), pp. 53-58, 
  62. [62](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bbib62)
A. Roca-Portoles, G. Rodriguez-Blanco, D. Sumpton, C. Cloix, M. Mullin, G.M. Mackay, K. O’Neill, L. Lemgruber, X. Luo, S.W.G. Tait
Venetoclax causes metabolic reprogramming independent of BCL-2 inhibition
Cell Death Dis., 11 (2020), Article 616, 
  63. [63](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bbib63)
C. Bosc, E. Saland, A. Bousard, N. Gadaud, M. Sabatier, G. Cognet, T. Farge, E. Boet, M. Gotanègre, N. Aroua, _et al._
Mitochondrial inhibitors circumvent adaptive resistance to venetoclax and cytarabine combination therapy in acute myeloid leukemia
Nat. Cancer, 2 (2021), pp. 1204-1223, 
  64. [64](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bbib64)
A. Sarachakov, A. Varlamova, V. Svekolkin, M. Polyakova, I. Valencia, C. Unkenholz, T. Pannellini, I. Galkin, P. Ovcharov, D. Tabakov, _et al._
Spatial mapping of human hematopoiesis at single-cell resolution reveals aging-associated topographic remodeling
Blood, 142 (2023), pp. 2282-2295, 
[View PDF](https://www.sciencedirect.com/science/article/pii/S0006497123024084/pdfft?md5=cd589a776cf5545268bc98bcd61d3f17&pid=1-s2.0-S0006497123024084-main.pdf)[View article](https://www.sciencedirect.com/science/article/pii/S0006497123024084)
  65. [65](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bbib65)
A.G. Aguilar-Navarro, B. Meza-León, D. Gratzinger, F.G. Juárez-Aguilar, Q. Chang, O. Ornatsky, H. Tsui, R. Esquivel-Gómez, A. Hernández-Ramírez, S.Z. Xie, _et al._
Human aging alters the spatial organization between CD34+ hematopoietic cells and adipocytes in bone marrow
Stem Cell Rep., 15 (2020), pp. 317-325, 
[View PDF](https://www.sciencedirect.com/science/article/pii/S2213671120302320/pdfft?md5=a9d81aa5da0ece5f122aad086df53af4&pid=1-s2.0-S2213671120302320-main.pdf)[View article](https://www.sciencedirect.com/science/article/pii/S2213671120302320)
  66. [66](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bbib66)
O. Naveiras, V. Nardi, P.L. Wenzel, P.V. Hauschka, F. Fahey, G.Q. Daley
Bone-marrow adipocytes as negative regulators of the haematopoietic microenvironment
Nature, 460 (2009), pp. 259-263, 
  67. [67](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bbib67)
L. DiMascio, C. Voermans, M. Uqoezwa, A. Duncan, D. Lu, J. Wu, U. Sankar, T. Reya
Identification of adiponectin as a novel hemopoietic stem cell growth factor
J. Immunol., 178 (2007), pp. 3511-3520, 
  68. [68](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bbib68)
B.O. Zhou, H. Yu, R. Yue, Z. Zhao, J.J. Rios, O. Naveiras, S.J. Morrison
Bone marrow adipocytes promote the regeneration of stem cells and haematopoiesis by secreting SCF
Nat. Cell Biol., 19 (2017), pp. 891-903, 
  69. [69](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bbib69)
Z. Zhang, Z. Huang, B. Ong, C. Sahu, H. Zeng, H.B. Ruan
Bone marrow adipose tissue-derived stem cell factor mediates metabolic regulation of hematopoiesis
Haematologica, 104 (2019), pp. 1731-1743, 
  70. [70](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bbib70)
S. Zhang, A. Paccalet, D. Rohde, S. Cremer, M. Hulsmans, I.H. Lee, K. Mentkowski, J. Grune, M.J. Schloss, L. Honold, _et al._
Bone marrow adipocytes fuel emergency hematopoiesis after myocardial infarction
Nat. Cardiovasc. Res., 2 (2023), pp. 1277-1290, 
  71. [71](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bbib71)
E.L. Scheller, W.P. Cawthorn, A.A. Burr, M.C. Horowitz, O.A. MacDougald
Marrow adipose tissue: trimming the fat
Trends Endocrinol. Metab., 27 (2016), pp. 392-403, 
[View PDF](https://www.sciencedirect.com/science/article/pii/S1043276016300029/pdfft?md5=3a2b75a8a9cdb4fc4fabd4193109a44b&pid=1-s2.0-S1043276016300029-main.pdf)[View article](https://www.sciencedirect.com/science/article/pii/S1043276016300029)
  72. [72](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bbib72)
K.D. Kokkaliaris, L. Kunz, N. Cabezas-Wallscheid, C. Christodoulou, S. Renders, F. Camargo, A. Trumpp, D.T. Scadden, T. Schroeder
Adult blood stem cell localization reflects the abundance of reported bone marrow niche cell types and their combinations
Blood, 136 (2020), pp. 2296-2307, 
[View PDF](https://www.sciencedirect.com/science/article/pii/S0006497120799301/pdfft?md5=78847ef24a93d426f4bf942f2da680b3&pid=1-s2.0-S0006497120799301-main.pdf)[View article](https://www.sciencedirect.com/science/article/pii/S0006497120799301)
  73. [73](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bbib87)
Y. Hao, S. Hao, E. Andersen-Nissen, W.M. Mauck 3rd, S. Zheng, A. Butler, M.J. Lee, A.J. Wilk, C. Darby, M. Zager, _et al._
Integrated analysis of multimodal single-cell data
Cell, 184 (2021), pp. 3573-3587, 
  74. [74](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bbib88)
Y. Hao, T. Stuart, M.H. Kowalski, S. Choudhary, P. Hoffman, A. Hartman, A. Srivastava, G. Molla, S. Madad, C. Fernandez-Granda, R. Satija
Dictionary learning for integrative, multimodal and scalable single-cell analysis
Nat Biotechnol, 42 (2024), pp. 293-304, 
  75. [75](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bbib80)
S. Aibar, C.B. González-Blas, T. Moerman, V.A. Huynh-Thu, H. Imrichova, G. Hulselmans, F. Rambow, J.C. Marine, P. Geurts, J. Aerts, _et al._
SCENIC: single-cell regulatory network inference and clustering
Nat. Methods, 14 (2017), pp. 1083-1086, 
  76. [76](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bbib86)
P. Bankhead, M.B. Loughrey, J.A. Fernández, Y. Dombrowski, D.G. McArt, P.D. Dunne, S. McQuaid, R.T. Gray, L.J. Murray, H.G. Coleman, _et al._
QuPath: open source software for digital pathology image analysis
Sci. Rep., 7 (2017), Article 16878, 
  77. [77](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bbib89)
Y. Zhou, B. Zhou, L. Pache, M. Chang, A.H. Khodabakhshi, O. Tanaseichuk, C. Benner, S.K. Chanda
Metascape provides a biologist-oriented resource for the analysis of systems-level datasets
Nat Commun, 10 (2019), p. 1523, 
  78. [78](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bbib73)
R. Hinton, R.L. Moody, A.W. Davis, S.F. Thomas
Osteoarthritis: diagnosis and therapeutic considerations
Am. Fam. Phys., 65 (2002), pp. 841-848
  79. [79](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bbib74)
A.A. Michael Doherty
Clinical manifestations and diagnosis of osteoarthritis
D.H. David Hunter, Philip Seo (Eds.), UpToDate (2022)
  80. [80](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bbib75)
F. Kathryn, K. Reichard, D. Czuchlewski
Bone Marrow Pathology
(Fourth Edition), American Society for Clinical Pathology Press (2019)
  81. [81](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bbib76)
S.C. Bendall, E.F. Simonds, P. Qiu, E.-a.D. Amir, P.O. Krutzik, R. Finck, R.V. Bruggner, R. Melamed, A. Trejo, O.I. Ornatsky, _et al._
Single-cell mass cytometry of differential immune and drug responses across a human hematopoietic continuum
Science, 332 (2011), pp. 687-696, 
  82. [82](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bbib77)
K.J. Livak, T.D. Schmittgen
Analysis of relative gene expression data using real-time quantitative PCR and the 2(-Delta Delta C(T)) Method
Methods, 25 (2001), pp. 402-408, 
[View PDF](https://www.sciencedirect.com/science/article/pii/S1046202301912629/pdf?md5=168f8e22f2c7d76f6021c5cbe37a3ba6&pid=1-s2.0-S1046202301912629-main.pdf)[View article](https://www.sciencedirect.com/science/article/pii/S1046202301912629)
  83. [83](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bbib78)
C.S. McGinnis, L.M. Murrow, Z.J. Gartner
DoubletFinder: doublet detection in single-cell RNA sequencing data using artificial nearest neighbors
Cell Syst., 8 (2019), pp. 329-337.e4, 
[View PDF](https://www.sciencedirect.com/science/article/pii/S2405471219300730/pdfft?md5=136d87ceb8b17cc7bfa1f8936a6e23b9&pid=1-s2.0-S2405471219300730-main.pdf)[View article](https://www.sciencedirect.com/science/article/pii/S2405471219300730)
  84. [84](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bbib79)
J.S. Packer, Q. Zhu, C. Huynh, P. Sivaramakrishnan, E. Preston, H. Dueck, D. Stefanik, K. Tan, C. Trapnell, J. Kim, _et al._
A lineage-resolved molecular atlas of C. elegans embryogenesis at single-cell resolution
Science, 365 (2019), Article eaax1971, 
  85. [85](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bbib81)
A. Liberzon, C. Birger, H. Thorvaldsdóttir, M. Ghandi, J.P. Mesirov, P. Tamayo
The Molecular Signatures Database (MSigDB) hallmark gene set collection
Cell Syst., 1 (2015), pp. 417-425, 
[View PDF](https://www.sciencedirect.com/science/article/pii/S2405471215002185/pdfft?md5=8f4aa9486fa28f0a824db6d5ef8e39b5&pid=1-s2.0-S2405471215002185-main.pdf)[View article](https://www.sciencedirect.com/science/article/pii/S2405471215002185)
  86. [86](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bbib82)
C. Chen, W. Yu, F. Alikarami, Q. Qiu, C.H. Chen, J. Flournoy, P. Gao, Y. Uzun, L. Fang, J.W. Davenport, _et al._
Single-cell multiomics reveals increased plasticity, resistant populations, and stem-cell-like blasts in KMT2A-rearranged leukemia
Blood, 139 (2022), pp. 2198-2211, 
[View PDF](https://www.sciencedirect.com/science/article/pii/S0006497121019996/pdfft?md5=5259397ad5210c67d19048dd8c8df143&pid=1-s2.0-S0006497121019996-main.pdf)[View article](https://www.sciencedirect.com/science/article/pii/S0006497121019996)
  87. [87](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bbib83)
R. Jitschin, M. Böttcher, D. Saul, S. Lukassen, H. Bruns, R. Loschinski, A.B. Ekici, A. Reis, A. Mackensen, D. Mougiakakos
Inflammation-induced glycolytic switch controls suppressivity of mesenchymal stem cells via STAT1 glycosylation
Leukemia, 33 (2019), pp. 1783-1796, 
  88. [88](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bbib84)
A. Orfao, L. Escribano, J. Villarrubia, J.L. Velasco, C. Cerveró, J. Ciudad, J.L. Navarro, J.F. San Miguel
Flow cytometric analysis of mast cells from normal and pathological human bone marrow samples: identification and enumeration
Am. J. Pathol., 149 (1996), pp. 1493-1499
  89. [89](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bbib85)
J. Windhager, V.R.T. Zanotelli, D. Schulz, L. Meyer, M. Daniel, B. Bodenmiller, N. Eling
An end-to-end workflow for multiplexed image processing and analysis
Nat. Protoc., 18 (2023), pp. 3565-3613, 

## Cited by (40)
  * ### [How age affects human hematopoietic stem and progenitor cells and the strategies to mitigate aging](https://www.sciencedirect.com/science/article/pii/S0301472X25000025)
2025, Experimental Hematology
Show abstract
Hematopoietic stem cells (HSCs) are central to blood formation and play a pivotal role in hematopoietic and systemic aging. With aging, HSCs undergo significant functional changes, such as an increased stem cell pool, declined homing and reconstitution capacity, and skewed differentiation toward myeloid and megakaryocyte/platelet progenitors. These phenotypic alterations are likely due to the expansion of certain clones, known as clonal hematopoiesis (CH), which leads to disrupted hematopoietic homeostasis, including anemia, impaired immunity, higher risks of hematological malignancies, and even associations with cardiovascular disease, highlighting the broader impact of HSC aging on overall health. HSC aging is driven by a range of mechanisms involving both intrinsic and extrinsic factors, such as DNA damage accumulation, epigenetic remodeling, inflammaging and metabolic regulation. In this review, we summarize the updated understanding of age-related changes in hematopoietic stem and progenitor cells (HSPCs) and the mechanisms underlying the aging process in mammalian models, especially in human study. Additionally, we provide insights into potential therapeutic strategies to counteract aging process and enhance HSC regenerative capacity, which will support therapeutic interventions and promote healthy aging.
  * ### [Reconsidering the usual suspects in age-related hematologic disorders: is stem cell dysfunction a root cause of aging?](https://www.sciencedirect.com/science/article/pii/S0301472X24005629)
2025, Experimental Hematology
Show abstract
Aging exerts a profound impact on the hematopoietic system, leading to increased susceptibility to infections, autoimmune diseases, chronic inflammation, anemia, thrombotic events, and hematologic malignancies. Within the field of experimental hematology, the functional decline of hematopoietic stem cells (HSCs) is often regarded as a primary driver of age-related hematologic conditions. However, aging is clearly a complex multifaceted process involving not only HSCs but also mature blood cells and their interactions with other tissues. This review reappraises an HSC-centric view of hematopoietic aging by exploring how the entire hematopoietic hierarchy, from stem cells to mature cells, contributes to age-related disorders. It highlights the decline of both innate and adaptive immunity, leading to increased susceptibility to infections and cancer, and the rise of autoimmunity as peripheral immune cells undergo aging-induced changes. It explores the concept of “inflammaging,” where persistent, low-grade inflammation driven by old immune cells creates a cycle of tissue damage and disease. Additionally, this review delves into the roles of inflammation and homeostatic regulation in age-related conditions such as thrombotic events and anemia, arguing that these issues arise from broader dysfunctions rather than stemming from HSC functional attrition alone. In summary, this review highlights the importance of taking a holistic approach to studying hematopoietic aging and its related pathologies. By looking beyond just stem cells and considering the full spectrum of age-associated changes, one can better capture the complexity of aging and attempt to develop preventative or rejuvenation strategies that target multiple facets of this process.
  * ### [Protocol for high-plex, whole-slide imaging of human formalin-fixed paraffin-embedded tissue using PhenoCycler-Fusion](https://www.sciencedirect.com/science/article/pii/S2666166724003915)
2024, STAR Protocols
Citation Excerpt :
A good starting point for troubleshooting this step is to check the literature for other immunohistochemistry or immunofluorescence studies that use the same tissue type. For example, Schürch et al. (2020) specified a 1 h baking time at 70°C for human colorectal tissue sections,15 while Bandyopadhyay et al. (2024) utilized a 3 h baking time at 65°C for human bone marrow biopsy sections.23 Presence of bubbles in the Flow Cell attached to sample slide after priming (Step 70).
Show abstract
Single-cell spatial analysis of proteins is rapidly becoming increasingly important in revealing biological insights. Here, we present a protocol for automated high-plex multi-slide immunofluorescence staining and imaging of human head and neck cancer formalin-fixed paraffin-embedded (FFPE) sections using PhenoCycler-Fusion 2.0 technology. We describe steps for preparing human head and neck cancer FFPE tissues, staining with a panel of immunophenotyping markers, and Flow Cell assembly. We then detail procedures for setting up for a PhenoCycler-Fusion run, post-run Flow Cell removal, and downstream analyses.
For complete details on the use and execution of this protocol, please refer to Jhaveri et al.[1](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bib1)
  * 2025, Nature Methods
  * 2025, Nature Reviews Molecular Cell Biology
  * 2024, International Journal of Molecular Sciences

[14](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bfn1)
    
These authors contributed equally

[15](https://www.sciencedirect.com/science/article/pii/S0092867424004082?via%3Dihub#bfn2)
    
Lead contact
© 2024 The Author(s). Published by Elsevier Inc.
## Recommended articles
  * ### [Global epistasis and the emergence of function in microbial consortia](https://www.sciencedirect.com/science/article/pii/S0092867424004112 "Global epistasis and the emergence of function in microbial consortia")
Cell, Volume 187, Issue 12, 2024, pp. 3108-3119.e30
Juan Diaz-Colunga, …, Alvaro Sanchez
[View PDF](https://www.sciencedirect.com/science/article/pii/S0092867424004112/pdfft?md5=87ae8ed3dd7370dfbcbdf430fb9e6fe4&pid=1-s2.0-S0092867424004112-main.pdf)
  * ### [Ligand binding initiates single-molecule integrin conformational activation](https://www.sciencedirect.com/science/article/pii/S0092867424004756 "Ligand binding initiates single-molecule integrin conformational activation")
Cell, Volume 187, Issue 12, 2024, pp. 2990-3005.e17
Jing Li, …, Timothy A. Springer
[View PDF](https://www.sciencedirect.com/science/article/pii/S0092867424004756/pdfft?md5=ea20586ec1004041fe8a90f75e37c303&pid=1-s2.0-S0092867424004756-main.pdf)
  * ### [A developmental constraint model of cancer cell states and tumor heterogeneity](https://www.sciencedirect.com/science/article/pii/S0092867424004586 "A developmental constraint model of cancer cell states and tumor heterogeneity")
Cell, Volume 187, Issue 12, 2024, pp. 2907-2918
Ayushi S. Patel, Itai Yanai
[View PDF](https://www.sciencedirect.com/science/article/pii/S0092867424004586/pdfft?md5=24afb12f69615f07ac1afe26871a7f34&pid=1-s2.0-S0092867424004586-main.pdf)
  * ### [Vertebrate centromeres in mitosis are functionally bipartite structures stabilized by cohesin](https://www.sciencedirect.com/science/article/pii/S0092867424004094 "Vertebrate centromeres in mitosis are functionally bipartite structures stabilized by cohesin")
Cell, Volume 187, Issue 12, 2024, pp. 3006-3023.e26
Carlos Sacristan, …, Geert J.P.L. Kops
[View PDF](https://www.sciencedirect.com/science/article/pii/S0092867424004094/pdfft?md5=480878966bacfd0086f0621b4f347426&pid=1-s2.0-S0092867424004094-main.pdf)
  * ### [Integrated OMICs unveil the bone-marrow microenvironment in human leukemia](https://www.sciencedirect.com/science/article/pii/S2211124721004587 "Integrated OMICs unveil the bone-marrow microenvironment in human leukemia")
Cell Reports, Volume 35, Issue 6, 2021, Article 109119
Diana Passaro, …, Dominique Bonnet
[View PDF](https://www.sciencedirect.com/science/article/pii/S2211124721004587/pdfft?md5=6c21c9bd68d5d9ba7924549aa5cb09cb&pid=1-s2.0-S2211124721004587-main.pdf)
  * ### [Deep Profiling of Mouse Splenic Architecture with CODEX Multiplexed Imaging](https://www.sciencedirect.com/science/article/pii/S0092867418309048 "Deep Profiling of Mouse Splenic Architecture with CODEX Multiplexed Imaging")
Cell, Volume 174, Issue 4, 2018, pp. 968-981.e15
Yury Goltsev, …, Garry P. Nolan
[View PDF](https://www.sciencedirect.com/science/article/pii/S0092867418309048/pdfft?md5=897230a8764b100ddf1e5d60dae6060c&pid=1-s2.0-S0092867418309048-main.pdf)

Show 3 more articles
  * [Remote access](https://www.sciencedirect.com/user/institution/login?targetURL=%2Fscience%2Farticle%2Fpii%2FS0092867424004082)

Cookies are used by this site. **Cookie Settings**
All content on this site: 
# 404 Not Found
nginx
# 404 Not Found
nginx
![Feedback](https://pendo-static-5661679399600128.storage.googleapis.com/D_T2uHq_M1r-XQq8htU6Z3GjHfE/guide-media-75af8ddc-3c43-49fd-8836-cfc7e2c3ea60)