Loading [MathJax]/jax/output/HTML-CSS/config.js
## Your privacy, your choice
We use essential cookies to make sure the site can function. We also use optional cookies for advertising, personalisation of content, usage analysis, and social media.
By accepting optional cookies, you consent to the processing of your personal data - including transfers to third parties. Some third parties are outside of the European Economic Area, with varying standards of data protection.
See our [privacy policy](https://www.biomedcentral.com/privacy-statement) for more information on the use of your personal data.
Manage preferences for further information and to change your choices.
Accept all cookies
[Skip to main content](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y#main-content)
Advertisement
[ ![BMC](https://cellandbioscience.biomedcentral.com/static/images/bmc/logos/logo-bmc-white-aj-be532aa3f0.svg) ![Part of Springer Nature](https://cellandbioscience.biomedcentral.com/static/images/bmc/logos/logo-bmc-white-strapline-sn-f224388d67.svg) ](https://www.biomedcentral.com)
Search
  * [ Explore journals ](https://www.biomedcentral.com/journals)
  * [ Get published ](https://www.biomedcentral.com/getpublished)
  * [ About BMC ](https://www.biomedcentral.com/about)
  * [Login](https://cellandbioscience.biomedcentral.com/login)

Menu
  * [ Explore journals ](https://www.biomedcentral.com/journals)
  * [ Get published ](https://www.biomedcentral.com/getpublished)
  * [ About BMC ](https://www.biomedcentral.com/about)
  * [Login](https://cellandbioscience.biomedcentral.com/login)

Search all BMC articles
Search
[ Cell & Bioscience ](https://cellandbioscience.biomedcentral.com/)
  * [Home](https://cellandbioscience.biomedcentral.com/)
  * [About](https://cellandbioscience.biomedcentral.com/about)
  * [Articles](https://cellandbioscience.biomedcentral.com/articles)
  * [Collections](https://cellandbioscience.biomedcentral.com/articles/collections)
  * [Submission Guidelines](https://cellandbioscience.biomedcentral.com/submission-guidelines)

The single-cell sequencing: new developments and medical applications 
[ Download PDF ](https://cellandbioscience.biomedcentral.com/counter/pdf/10.1186/s13578-019-0314-y.pdf)
[ Download PDF ](https://cellandbioscience.biomedcentral.com/counter/pdf/10.1186/s13578-019-0314-y.pdf)
  * Review
  * Published: 26 June 2019

# The single-cell sequencing: new developments and medical applications
  * [Xiaoning Tang](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y#auth-Xiaoning-Tang-Aff1-Aff2)[1](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y#Aff1),[2](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y#Aff2) [na1](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y#na1), 
  * [Yongmei Huang](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y#auth-Yongmei-Huang-Aff1-Aff2)[1](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y#Aff1),[2](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y#Aff2) [na1](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y#na1), 
  * [Jinli Lei](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y#auth-Jinli-Lei-Aff1-Aff2)[1](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y#Aff1),[2](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y#Aff2), 
  * [Hui Luo](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y#auth-Hui-Luo-Aff1-Aff2)[1](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y#Aff1),[2](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y#Aff2) &
  * …
  * [Xiao Zhu](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y#auth-Xiao-Zhu-Aff1-Aff2)[1](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y#Aff1),[2](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y#Aff2)

Show authors
[_Cell & Bioscience_](https://cellandbioscience.biomedcentral.com/) **volume 9**, Article number: 53 (2019) [Cite this article](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y#citeas)
  * 83k Accesses
  * 212 Citations
  * 149 Altmetric
  * [Metrics details](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y/metrics)

## Abstract
Single-cell sequencing technologies can be used to detect the genome, transcriptome and other multi-omics of single cells. They can show the differences and evolutionary relationships of various cells. This review introduces the latest advances in single-cell sequencing technologies and their applications in oncology, microbiology, neurology, reproduction, immunology, digestive and urinary systems, highlighting the important role that single-cell sequencing techniques play in these areas.
## Background
Single-cell sequencing technologies refer to the sequencing of a single-cell genome or transcriptome, so as to obtain genomic, transcriptome or other multi-omics information to reveal cell population differences and cellular evolutionary relationships. Traditional sequencing methods can only get the average of many cells, unable to analyze a small number of cells and lose cellular heterogeneity information. Compared with traditional sequencing technology, single-cell technologies have the advantages of detecting heterogeneity among individual cells [[1](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y#ref-CR1 "Wen L, Tang F. Boosting the power of single-cell analysis. Nat Biotechnol. 2018;36\(5\):408–9. 
         https://doi.org/10.1038/nbt.4131
         
        .")], distinguishing a small number of cells, and delineating cell maps. In 2013, it was named “Nature Methods” as the annual technology [[2](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y#ref-CR2 "Pennisi E. The biology of genomes. Single-cell sequencing tackles basic and biomedical questions. Science. 2012;336\(6084\):976–7. 
         https://doi.org/10.1126/science.336.6084.976
         
        .")]. However, early single-cell sequencing limited its widespread use due to its high cost. But as the research progressed, many new single-cell sequencing methods were developed that reduced the cost threshold for single-cell sequencing. Nowadays, single-cell sequencing technology is increasingly used in various fields. This review describes recent advances in single-cell sequencing methods and their applications in tumors, microbiology, neurology, reproduction, immunity, digestion, and urinary systems, and clarifies the important role of single-cell sequencing technologies in basic and clinical research.
## Single-cell sequencing methods and recent developments
### Development of single-cell sequencing methods
As research continues to deepen, the capabilities of single-cell sequencing methods (Fig. [1](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y#Fig1)) continue to increase and evolve toward lower detection costs, advancing scientists research on the molecular mechanisms at the single-cell level. Vitak et al. [[3](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y#ref-CR3 "Vitak SA, Torkenczy KA, Rosenkrantz JL, Fields AJ, Christiansen L, Wong MH, et al. Sequencing thousands of single-cell genomes with combinatorial indexing. Nat Methods. 2017;14\(3\):302–8. 
         https://doi.org/10.1038/nmeth.4154
         
        .")] proposed a single-cell combinatorial marker sequencing technique (SCI-seq) that can simultaneously construct thousands of single-cell libraries and detect variations in somatic cell copy number (Table [1](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y#Tab1)). This technique increases the number of cells detected and reduces the cost of library construction, and has important value in the study of somatic cell variation. Chen et al. [[4](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y#ref-CR4 "Chen C, Xing D, Tan L, Li H, Zhou G, Huang L, et al. Single-cell whole-genome analyses by Linear Amplification via Transposon Insertion \(LIANTI\). Science. 2017;356\(6334\):189–94. 
         https://doi.org/10.1126/science.aak9787
         
        .")] developed a novel single-cell whole-genome amplification method that can detect CNV at kilobase resolution and more effectively detect mutations in more diseases (Table [1](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y#Tab1)). Guo et al. [[5](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y#ref-CR5 "Guo F, Li L, Li J, Wu X, Hu B, Zhu P, et al. Single-cell multi-omics sequencing of mouse early embryos and embryonic stem cells. Cell Res. 2017;27\(8\):967–88. 
         https://doi.org/10.1038/cr.2017.82
         
        .")] developed a single-cell multiple sequencing technique (scCOOL-seq) that allows simultaneous analysis of single-cell chromatin state/nuclear niche localization, copy number variations, ploidy and DNA methylation, which can indicate different functions and patterns of chromatin state and DNA methylation (Table [1](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y#Tab1)). Casasent et al. [[6](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y#ref-CR6 "Casasent AK, Schalck A, Gao R, Sei E, Long A, Pangburn W, et al. Multiclonal invasion in breast tumors identified by topographic single cell sequencing. Cell. 2018;172\(1–2\):205–17. 
         https://doi.org/10.1016/j.cell.2017.12.007
         
        .")] invented a Topographic Single Cell Sequencing (TSCS) that provides accurate spatial location information for cells (Table [1](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y#Tab1)). This technique accurately measures and describes the specific characteristics of individual tumor cells spatially and helps to study the invasion and metastasis of tumor cells. Demaree et al. [[7](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y#ref-CR7 "Demaree B, Weisgerber D, Lan F, Abate AR. An ultrahigh-throughput microfluidic platform for single-cell genome sequencing. J Vis Exp. 2018;135:e57598. 
         https://doi.org/10.3791/57598
         
        .")] describe a high-throughput and low-deviation single-cell sequencing (SiC-seq) method that uses droplet microfluidics to separate, amplify, and barcode the genome of a single cell (Table [1](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y#Tab1)). This approach enables broader genomic studies for different cell populations. The Microwell-seq developed by Han et al. is a high-throughput and low-cost scRNA-seq platform [[8](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y#ref-CR8 "Han X, Wang R, Zhou Y, Fei L, Sun H, Lai S, et al. Mapping the mouse cell atlas by microwell-seq. Cell. 2018;172\(5\):1091–107. 
         https://doi.org/10.1016/j.cell.2018.02.001
         
        .")] (Table [1](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y#Tab1)). Not only does it improve the detection abundance of single-cell technologies, but it also reduces the cost of detection by an order of magnitude compared to single-cell sequencing techniques coated with oil droplets. The SPLit-seq technology from Rosenberg et al., based on the principle of a low-cost combined barcode, can reduce the cost of single-cell transcriptome sequencing to 1 cent. Once again broke the cost threshold for single cell detection [[9](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y#ref-CR9 "Rosenberg AB, Roco CM, Muscat RA, Kuchina A, Sample P, Yao Z, et al. Single-cell profiling of the developing mouse brain and spinal cord with split-pool barcoding. Science. 2018;360\(6385\):176–82. 
         https://doi.org/10.1126/science.aam8999
         
        .")] (Table [1](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y#Tab1)).
**Fig. 1**
[![figure 1](https://media.springernature.com/lw685/springer-static/image/art%3A10.1186%2Fs13578-019-0314-y/MediaObjects/13578_2019_314_Fig1_HTML.png)](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y/figures/1)
The principle of single-cell sequencing. It is a process of isolating a single cell for sequencing and studying cell heterogeneity, molecular mapping, immune infiltration and epigenetic changes
[Full size image](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y/figures/1)
**Table 1 Single-cell sequencing technologies**
[ Full size table](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y/tables/1)
### The joint use of single-cell sequencing technologies
The single-cell sequencing detection cost reduction is beneficial to the combination of other technologies and single-cell sequencing technologies, greatly improving the efficiency of single-cell detection. Datlinger et al. [[10](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y#ref-CR10 "Datlinger P, Rendeiro AF, Schmidl C, Krausgruber T, Traxler P, Klughammer J, et al. Pooled CRISPR screening with single-cell transcriptome readout. Nat Methods. 2017;14\(3\):297–301. 
         https://doi.org/10.1038/nmeth.4177
         
        .")] combined CRISPR screening with single-cell RNA sequencing to invent CROP-seq (Table [1](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y#Tab1)), which enables high-throughput functional analysis of complex regulatory mechanisms and heterogeneous cell populations. Gasperini et al. [[11](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y#ref-CR11 "Gasperini M, Hill AJ, McFaline-Figueroa JL, Martin B, Kim S, Zhang MD, et al. A Genome-wide framework for mapping gene regulation via cellular genetic screens. Cell. 2019;176\(1–2\):377–90. 
         https://doi.org/10.1016/j.cell.2018.11.029
         
        .")] combines CRISPRi and scRNA-seq to facilitate the study of the function of regulatory elements and the interrelationship between regulatory elements and genes (Table [1](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y#Tab1)). Habib et al. [[12](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y#ref-CR12 "Habib N, Avraham-Davidi I, Basu A, Burks T, Shekhar K, Hofree M, et al. Massively parallel single-nucleus RNA-seq with DroNc-seq. Nat Methods. 2017;14\(10\):955–8. 
         https://doi.org/10.1038/nmeth.4407
         
        .")] combined sNuc-Seq with microfluidic technology to introduce a single-cell nuclear RNA sequencing method with low cost, high-sensitive and high-efficiency cell classification, which is expected to be used in human cell mapping projects (Table [1](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y#Tab1)). Lake et al. [[13](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y#ref-CR13 "Lake BB, Chen S, Sos BC, Fan J, Kaeser GE, Yung YC, et al. Integrative single-cell analysis of transcriptional and epigenetic states in the human adult brain. Nat Biotechnol. 2018;36\(1\):70–80. 
         https://doi.org/10.1038/nbt.4038
         
        .")] combined single-core sequencing (snDrop-seq) and single-cell transposon hypersensitive site sequencing (scTHS-seq) to create a high-throughput sequencing platform for parallel detection of nuclear transcripts and epigenetic features (Table [1](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y#Tab1)). It provides a way to comprehensively analyze gene expression and regulation in cryopreserved samples of human tissues.
The above studies (Table [1](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y#Tab1)) show that single-cell sequencing technologies were constantly updated and developed, providing a technical basis for the construction of a complete cell map, greatly promoting the single-cell research process. In the future, single-cell sequencing technologies integrated multi-omics methods may become popular, playing an increasingly powerful role in single-cell research of complex organs and tissues, diagnosis and treatment of clinical diseases.
## Single-cell sequencing applications
### Applications in cancer
Studies have shown that genetic or genomic variation can lead to cells with different genetic and phenotypic characteristics within the tumor tissue, making the tumor tissue highly heterogeneous [[14](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y#ref-CR14 "Darmanis S, Sloan SA, Croote D, Mignardi M, Chernikova S, Samghababi P, et al. Single-cell RNA-seq analysis of infiltrating neoplastic cells at the migrating front of human glioblastoma. Cell Rep. 2017;21\(5\):1399–410. 
         https://doi.org/10.1016/j.celrep.2017.10.030
         
        ."), [15](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y#ref-CR15 "Xu Y, Zhou X. Applications of single-cell sequencing for multiomics. Methods Mol Biol. 2018;1754:327–74. 
         https://doi.org/10.1007/978-1-4939-7717-8_19
         
        .")]. This high degree of heterogeneity may be related to the mechanisms of tumorigenesis [[16](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y#ref-CR16 "Ley TJ, Mardis ER, Ding L, Fulton B, McLellan MD, Chen K, et al. DNA sequencing of a cytogenetically normal acute myeloid leukaemia genome. Nature. 2008;456\(7218\):66–72. 
         https://doi.org/10.1038/nature07485
         
        .")] and metastasis [[17](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y#ref-CR17 "Lawson DA, Kessenbrock K, Davis RT, Pervolarakis N, Werb Z. Tumour heterogeneity and metastasis at single-cell resolution. Nat Cell Biol. 2018;20\(12\):1349–60. 
         https://doi.org/10.1038/s41556-018-0236-7
         
        ."),[18](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y#ref-CR18 "Caswell DR, Swanton C. The role of tumour heterogeneity and clonal cooperativity in metastasis, immune evasion and clinical outcome. BMC Med. 2017;15\(1\):133. 
         https://doi.org/10.1186/s12916-017-0900-y
         
        ."),[19](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y#ref-CR19 "Alderton GK. Tumour evolution: epigenetic and genetic heterogeneity in metastasis. Nat Rev Cancer. 2017;17\(3\):141. 
         https://doi.org/10.1038/nrc.2017.11
         
        .")], so researchers need to perform more accurate analysis of tumor cells. Traditional sequencing methods can only detect cell populations, get the average of the signals in a group of cells, but the heterogeneity in tumor cells can be masked. Therefore, traditional sequencing methods do not study tumor cells well. Single-cell sequencing technologies can perfectly compensate for the shortcomings of traditional sequencing methods. The cell map of tumor cells and tumor microenvironment was drawn by detecting the heterogeneity of tumor cells. Further, clarify the cell group within the tumor tissue and find specific markers, as well as explain a series of problems such as tumorigenesis and metastasis. Therefore, single-cell sequencing technologies were widely used in the research of various tumors, and were of great significance for the development of new diagnostic and anti-tumor treatment methods (Fig. [2](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y#Fig2)).
**Fig. 2**
[![figure 2](https://media.springernature.com/lw685/springer-static/image/art%3A10.1186%2Fs13578-019-0314-y/MediaObjects/13578_2019_314_Fig2_HTML.png)](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y/figures/2)
The clinical applications of single-cell sequencing in different fields, like cancer, immune system, reproductive system, microorganisms, etc. These will help to clarify some of the problems and provide a basis for finding better treatments
[Full size image](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y/figures/2)
### Colorectal cancer
Zhang et al. [[20](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y#ref-CR20 "Zhang L, Yu X, Zheng L, Zhang Y, Li Y, Fang Q, et al. Lineage tracking reveals dynamic relationships of T cells in colorectal cancer. Nature. 2018;564\(7735\):268–72. 
         https://doi.org/10.1038/s41586-018-0694-x
         
        .")] mapped the T cell immunoreceptor of colorectal cancer by single-cell sequencing technology, and revealed the subgroup classification, tissue distribution characteristics, tumor heterogeneity and drug target gene expression of colorectal cancer T cells. A potential state transition relationship between T cell populations and subpopulations distributed across tissues was identified. In the same year, Bian et al. [[21](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y#ref-CR21 "Bian S, Hou Y, Zhou X, Li X, Yong J, Wang Y, et al. Single-cell multiomics sequencing and analyses of human colorectal cancer. Science. 2018;362\(6418\):1060–3. 
         https://doi.org/10.1126/science.aao3791
         
        .")] used this technique to analyze the characteristics and relationship of genomic copy number variation, DNA methylation abnormality and gene expression change during the occurrence and metastasis of human colorectal cancer from single cell resolution and multi-group level. The above studies are of great significance for revealing the new mechanism of human colon cancer and improving the diagnosis and treatment of colorectal cancer.
### Breast cancer
#### The origin and transformation of breast cancer
Breast cancer is a highly heterogeneous disease caused by genetically altered mammary epithelial cells. Nguyen et al. [[22](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y#ref-CR22 "Nguyen QH, Pervolarakis N, Nee K, Kessenbrock K. Experimental considerations for single-cell RNA sequencing approaches. Front Cell Dev Biol. 2018;6:108. 
         https://doi.org/10.3389/fcell.2018.00108
         
        ."),[23](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y#ref-CR23 "Nguyen A, Khoo WH, Moran I, Croucher PI, Phan TG. Single cell RNA sequencing of rare immune cell populations. Front Immunol. 2018;9:1553. 
         https://doi.org/10.3389/fimmu.2018.01553
         
        ."),[24](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y#ref-CR24 "Nguyen QH, Pervolarakis N, Blake K, Ma D, Davis RT, James N, et al. Profiling human breast epithelial cells using single cell RNA sequencing identifies cell diversity. Nat Commun. 2018;9\(1\):2028. 
         https://doi.org/10.1038/s41467-018-04334-1
         
        .")] mapped the single cell pattern of human mammary epithelial cells by single-cell sequencing technology and analyzed the diversity and status of cell types in breast cells. The findings help to understand the early origins of breast cancer and provide the basis for improving the early detection of cancer and preventing cancer progression. Casasent et al. [[6](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y#ref-CR6 "Casasent AK, Schalck A, Gao R, Sei E, Long A, Pangburn W, et al. Multiclonal invasion in breast tumors identified by topographic single cell sequencing. Cell. 2018;172\(1–2\):205–17. 
         https://doi.org/10.1016/j.cell.2017.12.007
         
        .")] reported the polyclonal origin of breast cancer and the mechanism of transformation between ductal carcinoma in situ and invasive carcinoma by topographic single-cell sequencing. Direct genomic lineages between in situ tumor subpopulations and invasive tumor subpopulations are revealed in this study. Furthermore, the time and reason for the occurrence of subcloning and the mechanisms of the formation of invasive carcinoma due to the intra-ductal cloning into adjacent tissues were further demonstrated. The results of this study provide a theoretical basis for preventing early breast cancer from turning into invasive cancer.
#### Breast cancer microenvironment
The single-cell technologies explored the immune microenvironment of breast cancer and finds immunizing changes. Azizi et al. [[25](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y#ref-CR25 "Azizi E, Carr AJ, Plitas G, Cornish AE, Konopacki C, Prabhakaran S, et al. Single-cell map of diverse immune phenotypes in the breast tumor microenvironment. Cell. 2018;174\(5\):1293–308. 
         https://doi.org/10.1016/j.cell.2018.05.060
         
        .")] mapped a single-cell map of various immune cells in the microenvironment of breast tumor. Through this map, the gene expression characteristics of human breast tumors and paired normal tissues, lymph node tissues and peripheral blood were analyzed. It was found that the heterogeneity of intratumoral lymphocytes and myeloid cells can induce combined gene expression through environmental stimulation. Based on T cell receptor (TCR) and T cell matched single-cell RNA sequencing data, the T cell continuous activation profile was inconsistent with the polarization model of macrophage in cancer. This study examined the microenvironment of breast cancer and helped to differentiate its immune cells and to identify the differences in gene expression of tumor-infiltrating immune cells and the causes of differences.
#### Breast cancer and estrogen
The researchers observed a close relationship between estrogen and breast cancer. Estrogen activates estrogen receptor alpha (ERα) to drive the progression of breast cancer. Zhu et al. [[26](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y#ref-CR26 "Zhu D, Zhao Z, Cui G, Chang S, Hu L, See YX, et al. Single-Cell Transcriptome Analysis Reveals Estrogen Signaling Coordinately Augments One-Carbon, Polyamine, and Purine Synthesis in Breast Cancer. Cell Rep. 2018;25\(8\):2285–98. 
         https://doi.org/10.1016/j.celrep.2018.10.093
         
        .")] studied the estrogen signaling pathway and the mechanism of ERα-regulated transcription by single-cell transcriptome sequencing, and found the dynamic transcriptional heterogeneity of ERα-positive breast cancer cells after estrogen stimulation. This study reveals the metabolic pathways involved in estrogen-mediated monocarbon, purine and polyamine synthesis in breast cancer, and the discovery that PPAT and AZIN1 were direct ERa targets for the cell survival and growth of breast cancer. It supplements the pathogenesis of breast cancer and provides a theoretical basis for the study of tumor-targeted drugs.
#### Brain tumor
It is generally believed that brain astrocytomas and oligodendrogliomas originate from different types of cells. However, Venteicher et al. [[27](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y#ref-CR27 "Venteicher AS, Tirosh I, Hebert C, Yizhak K, Neftel C, Filbin MG, et al. Decoupling genetics, lineages, and microenvironment in IDH-mutant gliomas by single-cell RNA-seq. Science. 2017. 
         https://doi.org/10.1126/science.aai8478
         
        .")] found that IDH mutant gliomas share a common origin through single-cell sequencing technology. The difference between them is mainly the difference in genetics and the composition of the tumor microenvironment. Two of the different subtypes of tumors have universally similar stem cell programs, which can be distinguished from similar neuroglial mass spectrometry systems. Tirosh et al. [[28](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y#ref-CR28 "Tirosh I, Venteicher AS, Hebert C, Escalante LE, Patel AP, Yizhak K, et al. Single-cell RNA-seq supports a developmental hierarchy in human oligodendroglioma. Nature. 2016;539\(7628\):309–13. 
         https://doi.org/10.1038/nature20123
         
        .")] identified cancer stem cells and their differentiated progeny through a single cell map in human oligodendroglioma. The findings support the cancer stem cell hypothesis and confirm that cancer stem cells were the main source of growth for oligodendroglioma. These studies suggest that these cells may become new therapeutic targets. Researcher Suvà suggested that immunotherapy can be used to attack specific cell types, thereby terminating tumor growth [[27](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y#ref-CR27 "Venteicher AS, Tirosh I, Hebert C, Yizhak K, Neftel C, Filbin MG, et al. Decoupling genetics, lineages, and microenvironment in IDH-mutant gliomas by single-cell RNA-seq. Science. 2017. 
         https://doi.org/10.1126/science.aai8478
         
        .")]. It is of great significance for the treatment of such diseases.
#### Head and neck cell carcinoma
Puram et al. [[29](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y#ref-CR29 "Puram SV, Tirosh I, Parikh AS, Patel AP, Yizhak K, Gillespie S, et al. Single-cell transcriptomic analysis of primary and metastatic tumor ecosystems in head and neck cancer. Cell. 2017;171\(7\):1611–24. 
         https://doi.org/10.1016/j.cell.2017.10.044
         
        .")] analyzed the ecosystem of head and neck squamous cell carcinoma (HNSCC) by single-cell sequencing technology, delineating the first cell map of head and neck tumors and revealing the types of head and neck tumors and their metastasis-related procedures. This study provides a new understanding of the causes and metastasis of tumor metastasis, and helps to find new targets for blocking tumor metastasis.
#### Hematological tumors
Single-cell sequencing technologies help to find classification markers and analyze the causes of disease occurrence in hematological tumors with high heterogeneity, providing a theoretical basis for clinical diagnosis and treatment of diseases.
Ledergor et al. [[30](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y#ref-CR30 "Ledergor G, Weiner A, Zada M, Wang SY, Cohen YC, Gatt ME, et al. Single cell dissection of plasma cell heterogeneity in symptomatic and asymptomatic myeloma. Nat Med. 2018;24\(12\):1867–76. 
         https://doi.org/10.1038/s41591-018-0269-2
         
        .")] found at the single-cell level that there were common expressions and differences in certain genes among different myeloma plasma cell samples. New markers can be sought by these common expressions. Differences in gene expression patterns and chromosome structure among the samples may be responsible for the high heterogeneity of myeloma plasma cells. This study summarizes the positive role of the single-cell approach in the diagnosis, treatment, and prognosis of myeloma. Peripheral blood is used to distinguish tumor cells and diagnose diseases. It can not only facilitate the collection of samples but also reduce the pain of patients during the collection of samples.
In the study of acute leukemia, Ley et al. [[16](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y#ref-CR16 "Ley TJ, Mardis ER, Ding L, Fulton B, McLellan MD, Chen K, et al. DNA sequencing of a cytogenetically normal acute myeloid leukaemia genome. Nature. 2008;456\(7218\):66–72. 
         https://doi.org/10.1038/nature07485
         
        .")] found genes associated with AML, and measured 9 mutations in each tumor sample cell. The results suggest that these heterogeneities caused by genetic mutations may be related to pathogenesis. Hou et al. [[31](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y#ref-CR31 "Hou Y, Song L, Zhu P, Zhang B, Tao Y, Xu X, et al. Single-cell exome sequencing and monoclonal evolution of a JAK2-negative myeloproliferative neoplasm. Cell. 2012;148\(5\):873–85. 
         https://doi.org/10.1016/j.cell.2012.02.028
         
        .")] performed a whole exome single-cell sequencing on cells from patients with JAK2-negative myeloproliferative tumors, and the results indicated that this tumor represents a monoclonal evolution. They further identified candidate mutations associated with essential thrombocythemia (ET) and speculated that these mutations may be involved in tumor progression. De Bie et al. [[32](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y#ref-CR32 "De Bie J, Demeyer S, Alberti-Servera L, Geerdens E, Segers H, Broux M, et al. Single-cell sequencing reveals the origin and the order of mutation acquisition in T-cell acute lymphoblastic leukemia. Leukemia. 2018;32\(6\):1358–69. 
         https://doi.org/10.1038/s41375-018-0127-8
         
        .")] demonstrated the sequence of mutations in T cell acute lymphoblastic leukemia (T-ALL) by targeted single-cell sequencing, and T-ALL development can begin in pluripotent progenitor cells.
#### Liver cancer and lung cancer
Chinese researchers at the single-cell level mapped the immunological map of the liver cancer microenvironment and the T cell immune map of lung cancer [[33](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y#ref-CR33 "Guo X, Zhang Y, Zheng L, Zheng C, Song J, Zhang Q, et al. Global characterization of T cells in non-small-cell lung cancer by single-cell sequencing. Nat Med. 2018;24\(7\):978–85. 
         https://doi.org/10.1038/s41591-018-0045-3
         
        ."), [34](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y#ref-CR34 "Zheng C, Zheng L, Yoo JK, Guo H, Zhang Y, Guo X, et al. Landscape of infiltrating T Cells in liver cancer revealed by single-cell sequencing. Cell. 2017;169\(7\):1342–56. 
         https://doi.org/10.1016/j.cell.2017.05.035
         
        .")]. The subgroup classification, tissue distribution characteristics, tumor heterogeneity and drug target gene expression of immune cells in liver cancer and lung cancer were revealed. The above studies are helpful to understand the immune microenvironment of liver cancer and lung cancer, to find effective biomarkers, new tumor immunotherapy, and drug targets. It is of great significance for the diagnosis and treatment of liver cancer and lung cancer.
Despite a large number of microorganisms in the world, researchers have only studied a small fraction of the microbial species. Due to the extremely small sample size of most microorganisms and the difficulty of culture, it is very difficult to study the microbial genome structure. However, the emergence of single-cell sequencing has led to important breakthroughs in the microbiology field. Single-cell sequencing technology provides cell-specific genetic information that is lacking in traditional genome-wide studies of bacteria. It also discovers genes associated with bacterial life activities and classifies microorganisms by genomic models [[35](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y#ref-CR35 "DeLorenzo S, Brauer SL, Edgmont CA, Herfort L, Tebo BM, Zuber P. Ubiquitous dissolved inorganic carbon assimilation by marine bacteria in the Pacific Northwest coastal ocean as determined by stable isotope probing. PLoS ONE. 2012;7\(10\):e46695. 
         https://doi.org/10.1371/journal.pone.0046695
         
        .")]. Lan et al. [[36](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y#ref-CR36 "Lan F, Demaree B, Ahmed N, Abate AR. Single-cell genome sequencing at ultra-high-throughput with microfluidic droplet barcoding. Nat Biotechnol. 2017;35\(7\):640–6. 
         https://doi.org/10.1038/nbt.3880
         
        .")] performed high-throughput single-cell genome sequencing (SiC-seq) on bacterial and fungal synthetic communities to analyze the distribution of antibiotic resistance genes, virulence factors and phage sequences in environmental microbial communities. This study enabled the microbial single-cell sequencing of prokaryotes. In the future, this technology may isolate low titer pathogenic bacterial samples and sequence them at the single-cell level to examine their virulence genotype sources [[7](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y#ref-CR7 "Demaree B, Weisgerber D, Lan F, Abate AR. An ultrahigh-throughput microfluidic platform for single-cell genome sequencing. J Vis Exp. 2018;135:e57598. 
         https://doi.org/10.3791/57598
         
        .")]. According to the above studies, single-cell sequencing technologies play important roles in the classification, evolution and drug resistance of microorganisms. It is good for mining meaningful environmental microbes and finding ways to resist antimicrobial resistance.
## Applications in the nervous system
### Nerve cell typing
In the nervous system, there is a difference among individual neurons because there are some unique copy number variations in nerve cells [[37](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y#ref-CR37 "McConnell MJ, Lindberg MR, Brennand KJ, Piper JC, Voet T, Cowing-Zitron C, et al. Mosaic copy number variation in human neurons. Science. 2013;342\(6158\):632–7. 
         https://doi.org/10.1126/science.1243472
         
        .")]. The heterogeneity among these neurons makes it difficult to study how brain circuits are formed and to resolve neuronal reconnections. However, single-cell sequencing technology can study many different periods of nerve cells, and draw a detailed single-cell map to understand and identify different types of neurons and their connecting molecules in the brain [[38](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y#ref-CR38 "Li H, Horns F, Wu B, Xie Q, Li J, Li T, et al. Classifying drosophila olfactory projection neuron subtypes by single-cell RNA sequencing. Cell. 2017;171\(5\):1206–20. 
         https://doi.org/10.1016/j.cell.2017.10.019
         
        .")]. Luo et al. [[39](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y#ref-CR39 "Luo C, Keown CL, Kurihara L, Zhou J, He Y, Li J, et al. Single-cell methylomes identify neuronal subtypes and regulatory elements in mammalian cortex. Science. 2017;357\(6351\):600–4. 
         https://doi.org/10.1126/science.aan3351
         
        .")] differentiated subtypes of mouse and human frontal cortical neuronal cells by high-throughput single-cell methylation sequencing. They identified a new set of neurons in the human frontal cortex and redefine the neuron type based on the methylation group of neurons. Lake et al. [[40](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y#ref-CR40 "Lake BB, Ai R, Kaeser GE, Salathia NS, Yung YC, Liu R, et al. Neuronal subtypes and diversity revealed by single-nucleus RNA sequencing of the human brain. Science. 2016;352\(6293\):1586–90. 
         https://doi.org/10.1126/science.aaf1204
         
        .")] used a new single-cell nuclear sequencing method to map the second generation single cell of an adult brain. A variety of different neurons, glial cell subtypes, and cell subtypes that are more susceptible to common risk factors for different brain diseases are found. It is expected to understand the pathogenesis of neurological diseases.
### Applications of brain nerve development and regeneration process
Single-cell sequencing technologies can study the types of brain cells and the relationships among cells during development (Fig. [2](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y#Fig2)). Fan et al. [[41](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y#ref-CR41 "Fan X, Dong J, Zhong S, Wei Y, Wu Q, Yan L, et al. Spatial transcriptomic survey of human embryonic cerebral cortex by single-cell RNA-seq analysis. Cell Res. 2018;28\(7\):730–45. 
         https://doi.org/10.1038/s41422-018-0053-3
         
        .")] identified multiple subpopulations of cells in each region in multiple regions of human metaphase embryos by single-cell sequencing techniques, and analyzed gene expression and neuronal maturation in these regions. Zhong et al. [[42](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y#ref-CR42 "Zhong S, Zhang S, Fan X, Wu Q, Yan L, Dong J, et al. A single-cell RNA-seq survey of the developmental landscape of the human prefrontal cortex. Nature. 2018;555\(7697\):524–8. 
         https://doi.org/10.1038/nature25980
         
        .")] mapped a single-cell transcriptome map of human prefrontal embryonic development through single-cell sequencing. From this map, the diversity of cell types in the prefrontal lobe of the human embryonic brain and the developmental relationship among different cell types were analyzed, and the molecular regulation mechanism of neuron production and loop formation was further revealed. It is important to study the function of key cell types and to map intact human brain cells. Carter et al. [[43](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y#ref-CR43 "Carter RA, Bihannic L, Rosencrance C, Hadley JL, Tong Y, Phoenix TN, et al. A single-cell transcriptional atlas of the developing murine cerebellum. Curr Biol. 2018;28\(18\):2910–20. 
         https://doi.org/10.1016/j.cub.2018.07.062
         
        .")] identified a major subset of cerebellar cells and a subpopulation that is beneficial to cerebellar development in a mouse cerebellar development map drawn by single-cell sequencing. These studies will facilitate future research on cerebellar development, neurobiology and diseases.
## Applications in the field of reproductive and embryonic medicine
The single-cell sequencing technologies can sequence and quantify the whole genome of germ cells and embryonic cells at the single-cell level (Fig. [2](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y#Fig2)). This will help to understand the occurrence of germ cells and the screening, diagnosis, and treatment of reproductive and genetic diseases.
### Germ cell
Chen et al. [[44](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y#ref-CR44 "Chen Y, Zheng Y, Gao Y, Lin Z, Yang S, Wang T, et al. Single-cell RNA-seq uncovers dynamic processes and critical regulators in mouse spermatogenesis. Cell Res. 2018;28\(9\):879–96. 
         https://doi.org/10.1038/s41422-018-0074-y
         
        .")] revealed the dynamic process and molecular characteristics of gene expression during spermatogenesis in mice and the specific patterns of alternative splicing through single-cell sequencing technology, as well as the discovery of key regulators for specific stages of male germ cell development. In the same year, the team performed single-cell RNA sequencing (scRNA-seq) analysis of human normal testicular cells and abnormal testicular cells. Based on the results, a hierarchical model of spermatogonial subtypes, spermatocyte subtypes and sperm cell subtypes was established, and specific markers for human germ cells were further discovered. In addition, changes of expression patterns in testicular somatic cells in an NOA patient were identified, which may be the pathogenesis of NOA [[45](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y#ref-CR45 "Wang M, Liu X, Chang G, Chen Y, An G, Yan L, et al. Single-cell RNA sequencing analysis reveals sequential cell fate transition during human spermatogenesis. Cell Stem Cell. 2018;23\(4\):599–614. 
         https://doi.org/10.1016/j.stem.2018.08.007
         
        .")]. The above studies provide valuable data sources for the development and maturation of mammalian sperm and the development of gametes. It also contributes to the understanding of male infertility and the treatment of such diseases.
### Reproductive support
Single cell sequencing technology can detect the advantages of a small number of cells, which can be applied to prenatal diagnosis and assisted reproduction. Detection of female egg cell polar cells or embryonic cells by single-cell sequencing to select healthy embryo transfer can reduce the birth rate of newborns with congenital genetic diseases and help prevent genetic diseases [[46](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y#ref-CR46 "Hou Y, Fan W, Yan L, Li R, Lian Y, Huang J, et al. Genome analyses of single human oocytes. Cell. 2013;155\(7\):1492–506. 
         https://doi.org/10.1016/j.cell.2013.11.040
         
        ."), [47](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y#ref-CR47 "Li W, Ma Y, Yu S, Sun N, Wang L, Chen D, et al. The mutation-free embryo for in vitro fertilization selected by MALBAC-PGD resulted in a healthy live birth from a family carrying PKD 1 mutation. J Assist Reprod Genet. 2017;34\(12\):1653–8. 
         https://doi.org/10.1007/s10815-017-1018-z
         
        .")].
### Embryonic cell
In terms of animals, single-cell transcriptome sequencing technology is used to map the cell development of zebrafish and African cockroach embryos, providing important clues for understanding developmental biology [[48](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y#ref-CR48 "Wagner DE, Weinreb C, Collins ZM, Briggs JA, Megason SG, Klein AM. Single-cell mapping of gene expression landscapes and lineage in the zebrafish embryo. Science. 2018;360\(6392\):981–7. 
         https://doi.org/10.1126/science.aar4362
         
        ."),[49](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y#ref-CR49 "Farrell JA, Wang Y, Riesenfeld SJ, Shekhar K, Regev A, Schier AF. Single-cell reconstruction of developmental trajectories during zebrafish embryogenesis. Science. 2018. 
         https://doi.org/10.1126/science.aar3131
         
        ."),[50](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y#ref-CR50 "Briggs JA, Weinreb C, Wagner DE, Megason S, Peshkin L, Kirschner MW, et al. The dynamics of gene expression in vertebrate embryogenesis at single-cell resolution. Science. 2018. 
         https://doi.org/10.1126/science.aar5780
         
        .")]. Li et al. [[51](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y#ref-CR51 "Li L, Guo F, Gao Y, Ren Y, Yuan P, Yan L, et al. Publisher Correction: Single-cell multi-omics sequencing of human early embryos. Nat Cell Biol. 2018;20\(10\):1227. 
         https://doi.org/10.1038/s41556-018-0166-4
         
        .")] mapped the genome-wide map of human embryos prior to implantation by single-cell multi-sequence sequencing. This study has implications for the analysis of complex and highly coordinated epigenetic processes in the pre-implantation development of human embryos. Vento-Tormo et al. [[52](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y#ref-CR52 "Vento-Tormo R, Efremova M, Botting RA, Turco MY, Vento-Tormo M, Meyer KB, et al. Single-cell reconstruction of the early maternal-fetal interface in humans. Nature. 2018;563\(7731\):347–53. 
         https://doi.org/10.1038/s41586-018-0698-6
         
        .")] performed a transcriptome analysis of placental cells in early pregnancy by single-cell sequencing technology and mapped placental cell maps. Through the cell map, three subpopulations of perivascular and stromal cells located in different decidual layers and dNK (decidual natural killer) were found. It has also identified regulatory responses that may minimize the immune response of harmful mothers, as well as revealing interactions that contribute to the success of placenta and reproduction. The findings are important for understanding early pregnancy processes and improving the diagnosis and treatment of pregnancy-related diseases.
## Applications in the field of immunology
As an important system for the body to perform immune response and immune function, the immune system has an important role in resisting the invasion of external pathogens. Many immune cells with unique functions in the immune system are a research hotspot.
### Application of natural killer (NK) cells, DC cells and lymphocytes
Single-cell sequencing technologies can detect individual immune cells, thereby distinguishing different groups of immune cells, as well as discovering new immune cell populations and their relationships (Fig. [2](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y#Fig2)). This helps to understand the complex immune system and propose new targets for disease treatment. Crinier et al. [[53](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y#ref-CR53 "Crinier A, Milpied P, Escaliere B, Piperoglou C, Galluso J, Balsamo A, et al. High-dimensional single-cell analysis identifies organ-specific signatures and conserved NK cell subsets in humans and mice. Immunity. 2018;49\(5\):971–86. 
         https://doi.org/10.1016/j.immuni.2018.09.009
         
        .")] identified subpopulations of the spleen and blood NK cells in mice and humans by single-cell RNA sequencing, revealing two distinct features that distinguish blood and spleen NK cells. And through the comparison of transcriptomics, the similarity between two major subgroups NK1 and NK2 in organs and species is highlighted. This study provides an in-depth understanding of the biology of NK cells and contributes to the translation of animal studies into human-related research. Villani et al. [[54](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y#ref-CR54 "Villani AC, Satija R, Reynolds G, Sarkizova S, Shekhar K, Fletcher J, et al. Single-cell RNA-seq reveals new types of human blood dendritic cells, monocytes, and progenitors. Science. 2017. 
         https://doi.org/10.1126/science.aah4573
         
        .")] identified multiple subtypes of DC cells and monocytes in human blood by single-cell RNA-seq and revealed a new subset of DC cells that have the properties of plasmacytoid DCs but are effective in activating T cells. In response to this finding, one of the researchers suggested “stimulating such cells or potentially enhancing the body’s immune system to fight cancer.” Such cells may become a new anti-cancer means to remove tumor cells through their own immune system, avoiding the damage of normal chemotherapy drugs to normal cells. The study redefines the relationship among DC cells, helps analyze the developmental processes and functions of the immune system, and completes immune surveillance under normal and disease states. Xin et al. [[55](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y#ref-CR55 "Xin G, Zander R, Schauder DM, Chen Y, Weinstein JS, Drobyski WR, et al. Single-cell RNA sequencing unveils an IL-10-producing helper subset that sustains humoral immunity during persistent infection. Nat Commun. 2018;9\(1\):5037. 
         https://doi.org/10.1038/s41467-018-07492-4
         
        .")] used single-cell RNA sequencing to study immune cells and cytokines during persistent infection. It was found that the heterogeneity of IL-10 expressing CD4 T cells and the production of IL-10 by a subset of helper cells during different infections play an important role in promoting humoral immunity.
### Causes of immune cell heterogeneity
Single-cell sequencing can study immune cells with high heterogeneity caused by pathogens, accurately detect the genetic material of individual immune cells, and help to understand the complex immune mechanism of the body [[56](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y#ref-CR56 "Papalexi E, Satija R. Single-cell RNA sequencing to explore immune cell heterogeneity. Nat Rev Immunol. 2018;18\(1\):35–45. 
         https://doi.org/10.1038/nri.2017.76
         
        .")]. Interestingly, in addition to pathogens, aging can also lead to increased cellular heterogeneity. Martinez-Jimenez et al. [[57](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y#ref-CR57 "Martinez-Jimenez CP, Eling N, Chen HC, Vallejos CA, Kolodziejczyk AA, Connor F, et al. Aging increases cell-to-cell transcriptional variability upon immune stimulation. Science. 2017;355\(6332\):1433–6. 
         https://doi.org/10.1126/science.aah4115
         
        .")] performed single-cell RNA sequencing on CD4+ T cells in different states of young and old mice, and found that aging affects cell transcriptional kinetics, resulting in increased heterogeneity of gene expression among immune cells. Further, the immune cells are unable to make an asynchronous reaction, and the immune performance is weakened. This study helps explain the weakening of the immune system with increasing age. It is suggested that the increase in intercellular transcriptional variation may represent the aging characteristics, laying the foundation for exploring the aging mechanism of cells. Recently, single-cell sequencing has found new discoveries in the immune cells of the central nervous system during the inflammatory phase. Jordão et al. [[58](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y#ref-CR58 "Jordao MJC, Sankowski R, Brendecke SM, Locatelli G, Tai YH, et al. Single-cell profiling identifies myeloid cell subsets with distinct fates during neuroinflammation. Science. 2019. 
         https://doi.org/10.1126/science.aat7554
         
        .")] found that in the development of neuroinflammatory pathology, the specific differentiation of myeloid cell subpopulations is obvious because the self-renewal, random proliferation and clonal expansion of CNS macrophage subpopulations. Moreover, the antigen presentation of CNS macrophages has no significant relationship with the pathological role of neuroinflammation. Studies have shown that dendritic cells and monocyte-derived cells are the main participants in antigen presentation during experimental autoimmune encephalomyelitis (EAE). This study maps the dynamic map of myeloid cell subpopulations in the central nervous system, reveals complex changes in the characteristic molecules of myeloid cell subsets during neuroinflammation, and provides a theoretical basis for improving the treatment of EAE.
According to the above research, the heterogeneity of immune cells will affect the immune status of the body. An increase in the heterogeneity of immune cells may cause immune cells to have specific functions that help fight against pathogen invasion. It may also cause the body’s immunity to decline, which is a feature of tissue aging. Therefore, the study of immune cell heterogeneity can help people understand the complex and delicate immune mechanisms of the body, and perhaps adjust some immune mechanisms to treat certain diseases.
## Application in the digestive system and urinary system
Haber et al. [[59](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y#ref-CR59 "Haber AL, Biton M, Rogel N, Herbst RH, Shekhar K, Smillie C, et al. A single-cell survey of the small intestinal epithelium. Nature. 2017;551\(7680\):333–9. 
         https://doi.org/10.1038/nature24489
         
        .")] found many new intestinal epithelial cell subtypes by single-cell transcriptome sequencing and mapped the expression of intestinal epithelial cells. From this map, the mechanism of intestinal cells maintaining homeostasis and responding to pathogenic microorganisms is explained. Gao et al. [[60](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y#ref-CR60 "Gao S, Yan L, Wang R, Li J, Yong J, Zhou X, et al. Tracing the temporal-spatial transcriptome landscapes of the human fetal digestive tract using single-cell RNA-sequencing. Nat Cell Biol. 2018;20\(6\):721–34. 
         https://doi.org/10.1038/s41556-018-0105-4
         
        .")] used high-precision single-cell transcriptome sequencing to analyze the four digestive organs of human embryonic stage and multiple cells of the adult large intestine, revealing the related mechanisms of gene regulation in the development of four human digestive organs (Fig. [2](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y#Fig2)). Wang et al. [[61](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y#ref-CR61 "Wang P, Chen Y, Yong J, Cui Y, Wang R, Wen L, et al. Dissecting the global dynamic molecular profiles of human fetal kidney development by single-cell RNA sequencing. Cell Rep. 2018;24\(13\):3554–67. 
         https://doi.org/10.1016/j.celrep.2018.08.056
         
        .")] demonstrated the heterogeneity of precursor cells forming the initial nephron in the human embryonic stage and the corresponding transcriptional regulatory events and signaling pathways in the process of differentiation of precursor cells into tubular epithelial cells by single-cell transcriptome sequencing (Fig. [2](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y#Fig2)). This study also demonstrated the expression characteristics of candidate pathogenic genes for congenital nephropathy and contributed to the treatment of congenital nephropathy.
## Conclusion and perspectives
After completing the Human Genome Project, scientists have proposed a human cell atlas program designed to complete the mapping of 37 trillion cells in the human body. Single-cell sequencing technologies can accurately study individual cells and will be an important driving force for this project. The cell maps depicted by single-cell sequencing technology help people distinguish among cell types and understand cell-to-cell relationships. One can further understand physiological processes and pathological mechanisms at the single-cell level to find new diagnostic markers or new therapeutic targets. This will provide a practical basis for improving the diagnosis and treatment of the diseases.
This review introduced single-cell sequencing methods and their applications in tumors, microorganism, nervous system, reproductive medicine and immunology, highlighting the great advantages of single-cell sequencing technologies in the study of highly heterogeneous single-cell. However, single-cell sequencing technologies still have problems such as cumbersome operation and high detection cost, which limits the promotion of technology. It is hoped that the single-cell sequencing technologies will be more simplified, more powerful, and the detection cost will be further reduced, so that the technologies can be applied to basic research and play an important role in clinical diagnosis and treatment.
## Availability of data and materials
Not applicable.
## Abbreviations

SCI-seq:
    
single-cell combinatorial marker sequencing technique 

scCOOL-seq:
    
single-cell chromatin overall omic-scale landscape sequencing 

TSCS:
    
topographic Single Cell Sequencing 

scRNA-seq:
    
single-cell RNA-seq 

CRISPR:
    
clustered regularly interspaced short palindromic repeats 

CROP-seq:
    
cRISPR droplet sequencing 

sNuc-Seq:
    
single-Nucleus RNA Sequencing 

snDrop-seq:
    
single-core sequencing 

scTHS-seq:
    
single-cell transposon hypersensitive site sequencing 

TCR:
    
T cell receptor 

ERα:
    
estrogen receptor alpha 

PPAT:
    
phosphoribosyl pyrophosphate amidotransferase 

AZIN1:
    
antizyme inhibitor 1 

IDH:
    
isocitrate dehydrogenase 

HNSCC:
    
head and neck squamous cell carcinoma 

AML:
    
acute myeloid leukemia 

ET:
    
essential thrombocythemia 

T-ALL:
    
T cell acute lymphoblastic leukemia 

NOA:
    
non obstructive azoospermic 

dNK cells:
    
decidual natural killer cells 

NK cells:
    
natural killer cell 

DC cells:
    
dendritic cells 

IL-10:
    
interleukin-10 

CD4:
    
cluster of differentiation 4 

CNS:
    
central nervous system 

EAE:
    
experimental autoimmune encephalomyelitis
## References
  1. Wen L, Tang F. Boosting the power of single-cell analysis. Nat Biotechnol. 2018;36(5):408–9. .
[CAS](https://cellandbioscience.biomedcentral.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXovFGntrw%3D)
  2. Pennisi E. The biology of genomes. Single-cell sequencing tackles basic and biomedical questions. Science. 2012;336(6084):976–7. .
[CAS](https://cellandbioscience.biomedcentral.com/articles/cas-redirect/1:CAS:528:DC%2BC38XptVCmu7c%3D)
  3. Vitak SA, Torkenczy KA, Rosenkrantz JL, Fields AJ, Christiansen L, Wong MH, et al. Sequencing thousands of single-cell genomes with combinatorial indexing. Nat Methods. 2017;14(3):302–8. .
[CAS](https://cellandbioscience.biomedcentral.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhvV2hsrc%3D)
  4. Chen C, Xing D, Tan L, Li H, Zhou G, Huang L, et al. Single-cell whole-genome analyses by Linear Amplification via Transposon Insertion (LIANTI). Science. 2017;356(6334):189–94. .
[CAS](https://cellandbioscience.biomedcentral.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXlvVems70%3D)
  5. Guo F, Li L, Li J, Wu X, Hu B, Zhu P, et al. Single-cell multi-omics sequencing of mouse early embryos and embryonic stem cells. Cell Res. 2017;27(8):967–88. .
[CAS](https://cellandbioscience.biomedcentral.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhtVCiurfF)
  6. Casasent AK, Schalck A, Gao R, Sei E, Long A, Pangburn W, et al. Multiclonal invasion in breast tumors identified by topographic single cell sequencing. Cell. 2018;172(1–2):205–17. .
[CAS](https://cellandbioscience.biomedcentral.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXkslehsA%3D%3D)
  7. Demaree B, Weisgerber D, Lan F, Abate AR. An ultrahigh-throughput microfluidic platform for single-cell genome sequencing. J Vis Exp. 2018;135:e57598. .
[CAS](https://cellandbioscience.biomedcentral.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXitlOmt73O)
  8. Han X, Wang R, Zhou Y, Fei L, Sun H, Lai S, et al. Mapping the mouse cell atlas by microwell-seq. Cell. 2018;172(5):1091–107. .
[CAS](https://cellandbioscience.biomedcentral.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXjt12ltr4%3D)
  9. Rosenberg AB, Roco CM, Muscat RA, Kuchina A, Sample P, Yao Z, et al. Single-cell profiling of the developing mouse brain and spinal cord with split-pool barcoding. Science. 2018;360(6385):176–82. .
[CAS](https://cellandbioscience.biomedcentral.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXnt1GhtbY%3D)
  10. Datlinger P, Rendeiro AF, Schmidl C, Krausgruber T, Traxler P, Klughammer J, et al. Pooled CRISPR screening with single-cell transcriptome readout. Nat Methods. 2017;14(3):297–301. .
[CAS](https://cellandbioscience.biomedcentral.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhtlKjsrk%3D)
  11. Gasperini M, Hill AJ, McFaline-Figueroa JL, Martin B, Kim S, Zhang MD, et al. A Genome-wide framework for mapping gene regulation via cellular genetic screens. Cell. 2019;176(1–2):377–90. .
[CAS](https://cellandbioscience.biomedcentral.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXjs1Squg%3D%3D)
  12. Habib N, Avraham-Davidi I, Basu A, Burks T, Shekhar K, Hofree M, et al. Massively parallel single-nucleus RNA-seq with DroNc-seq. Nat Methods. 2017;14(10):955–8. .
[CAS](https://cellandbioscience.biomedcentral.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhtl2gtL7L)
  13. Lake BB, Chen S, Sos BC, Fan J, Kaeser GE, Yung YC, et al. Integrative single-cell analysis of transcriptional and epigenetic states in the human adult brain. Nat Biotechnol. 2018;36(1):70–80. .
[CAS](https://cellandbioscience.biomedcentral.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhvFGmurrM)
  14. Darmanis S, Sloan SA, Croote D, Mignardi M, Chernikova S, Samghababi P, et al. Single-cell RNA-seq analysis of infiltrating neoplastic cells at the migrating front of human glioblastoma. Cell Rep. 2017;21(5):1399–410. .
[CAS](https://cellandbioscience.biomedcentral.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhslGit7zP)
  15. Xu Y, Zhou X. Applications of single-cell sequencing for multiomics. Methods Mol Biol. 2018;1754:327–74. .
[CAS](https://cellandbioscience.biomedcentral.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXitVOgt77L)
  16. Ley TJ, Mardis ER, Ding L, Fulton B, McLellan MD, Chen K, et al. DNA sequencing of a cytogenetically normal acute myeloid leukaemia genome. Nature. 2008;456(7218):66–72. .
[CAS](https://cellandbioscience.biomedcentral.com/articles/cas-redirect/1:CAS:528:DC%2BD1cXhtlCjtrfE)
  17. Lawson DA, Kessenbrock K, Davis RT, Pervolarakis N, Werb Z. Tumour heterogeneity and metastasis at single-cell resolution. Nat Cell Biol. 2018;20(12):1349–60. .
[CAS](https://cellandbioscience.biomedcentral.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXitlent7zN)
  18. Caswell DR, Swanton C. The role of tumour heterogeneity and clonal cooperativity in metastasis, immune evasion and clinical outcome. BMC Med. 2017;15(1):133. .
[CAS](https://cellandbioscience.biomedcentral.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXitVOrtr7P)
  19. Alderton GK. Tumour evolution: epigenetic and genetic heterogeneity in metastasis. Nat Rev Cancer. 2017;17(3):141. .
[CAS](https://cellandbioscience.biomedcentral.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXjt1ags7o%3D)
  20. Zhang L, Yu X, Zheng L, Zhang Y, Li Y, Fang Q, et al. Lineage tracking reveals dynamic relationships of T cells in colorectal cancer. Nature. 2018;564(7735):268–72. .
[CAS](https://cellandbioscience.biomedcentral.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXisVarsr7F)
  21. Bian S, Hou Y, Zhou X, Li X, Yong J, Wang Y, et al. Single-cell multiomics sequencing and analyses of human colorectal cancer. Science. 2018;362(6418):1060–3. .
[CAS](https://cellandbioscience.biomedcentral.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXitlWhtLnI)
  22. Nguyen QH, Pervolarakis N, Nee K, Kessenbrock K. Experimental considerations for single-cell RNA sequencing approaches. Front Cell Dev Biol. 2018;6:108. .
  23. Nguyen A, Khoo WH, Moran I, Croucher PI, Phan TG. Single cell RNA sequencing of rare immune cell populations. Front Immunol. 2018;9:1553. .
[CAS](https://cellandbioscience.biomedcentral.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXitlSisbnJ)
  24. Nguyen QH, Pervolarakis N, Blake K, Ma D, Davis RT, James N, et al. Profiling human breast epithelial cells using single cell RNA sequencing identifies cell diversity. Nat Commun. 2018;9(1):2028. .
[CAS](https://cellandbioscience.biomedcentral.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhtFelurvK)
  25. Azizi E, Carr AJ, Plitas G, Cornish AE, Konopacki C, Prabhakaran S, et al. Single-cell map of diverse immune phenotypes in the breast tumor microenvironment. Cell. 2018;174(5):1293–308. .
[CAS](https://cellandbioscience.biomedcentral.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXht1Cis7vN)
  26. Zhu D, Zhao Z, Cui G, Chang S, Hu L, See YX, et al. Single-Cell Transcriptome Analysis Reveals Estrogen Signaling Coordinately Augments One-Carbon, Polyamine, and Purine Synthesis in Breast Cancer. Cell Rep. 2018;25(8):2285–98. .
[CAS](https://cellandbioscience.biomedcentral.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXitlShtb3O)
  27. Venteicher AS, Tirosh I, Hebert C, Yizhak K, Neftel C, Filbin MG, et al. Decoupling genetics, lineages, and microenvironment in IDH-mutant gliomas by single-cell RNA-seq. Science. 2017. .
  28. Tirosh I, Venteicher AS, Hebert C, Escalante LE, Patel AP, Yizhak K, et al. Single-cell RNA-seq supports a developmental hierarchy in human oligodendroglioma. Nature. 2016;539(7628):309–13. .
[CAS](https://cellandbioscience.biomedcentral.com/articles/cas-redirect/1:CAS:528:DC%2BC28XhvVSiu73J)
  29. Puram SV, Tirosh I, Parikh AS, Patel AP, Yizhak K, Gillespie S, et al. Single-cell transcriptomic analysis of primary and metastatic tumor ecosystems in head and neck cancer. Cell. 2017;171(7):1611–24. .
[CAS](https://cellandbioscience.biomedcentral.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhvFWms7nE)
  30. Ledergor G, Weiner A, Zada M, Wang SY, Cohen YC, Gatt ME, et al. Single cell dissection of plasma cell heterogeneity in symptomatic and asymptomatic myeloma. Nat Med. 2018;24(12):1867–76. .
[CAS](https://cellandbioscience.biomedcentral.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXisVGmsLzO)
  31. Hou Y, Song L, Zhu P, Zhang B, Tao Y, Xu X, et al. Single-cell exome sequencing and monoclonal evolution of a JAK2-negative myeloproliferative neoplasm. Cell. 2012;148(5):873–85. .
[CAS](https://cellandbioscience.biomedcentral.com/articles/cas-redirect/1:CAS:528:DC%2BC38XjtlSltr0%3D)
  32. De Bie J, Demeyer S, Alberti-Servera L, Geerdens E, Segers H, Broux M, et al. Single-cell sequencing reveals the origin and the order of mutation acquisition in T-cell acute lymphoblastic leukemia. Leukemia. 2018;32(6):1358–69. .
[CAS](https://cellandbioscience.biomedcentral.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhtFOnu7fE)
  33. Guo X, Zhang Y, Zheng L, Zheng C, Song J, Zhang Q, et al. Global characterization of T cells in non-small-cell lung cancer by single-cell sequencing. Nat Med. 2018;24(7):978–85. .
[CAS](https://cellandbioscience.biomedcentral.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXht1WqsLjE)
  34. Zheng C, Zheng L, Yoo JK, Guo H, Zhang Y, Guo X, et al. Landscape of infiltrating T Cells in liver cancer revealed by single-cell sequencing. Cell. 2017;169(7):1342–56. .
[CAS](https://cellandbioscience.biomedcentral.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhtVWrtrvE)
  35. DeLorenzo S, Brauer SL, Edgmont CA, Herfort L, Tebo BM, Zuber P. Ubiquitous dissolved inorganic carbon assimilation by marine bacteria in the Pacific Northwest coastal ocean as determined by stable isotope probing. PLoS ONE. 2012;7(10):e46695. .
[CAS](https://cellandbioscience.biomedcentral.com/articles/cas-redirect/1:CAS:528:DC%2BC38XhsFWgsbfP)
  36. Lan F, Demaree B, Ahmed N, Abate AR. Single-cell genome sequencing at ultra-high-throughput with microfluidic droplet barcoding. Nat Biotechnol. 2017;35(7):640–6. .
[CAS](https://cellandbioscience.biomedcentral.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXosFSru7c%3D)
  37. McConnell MJ, Lindberg MR, Brennand KJ, Piper JC, Voet T, Cowing-Zitron C, et al. Mosaic copy number variation in human neurons. Science. 2013;342(6158):632–7. .
[CAS](https://cellandbioscience.biomedcentral.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXhs1yrsL7K)
  38. Li H, Horns F, Wu B, Xie Q, Li J, Li T, et al. Classifying drosophila olfactory projection neuron subtypes by single-cell RNA sequencing. Cell. 2017;171(5):1206–20. .
[CAS](https://cellandbioscience.biomedcentral.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhvFOhs7vJ)
  39. Luo C, Keown CL, Kurihara L, Zhou J, He Y, Li J, et al. Single-cell methylomes identify neuronal subtypes and regulatory elements in mammalian cortex. Science. 2017;357(6351):600–4. .
[CAS](https://cellandbioscience.biomedcentral.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhtlehur%2FE)
  40. Lake BB, Ai R, Kaeser GE, Salathia NS, Yung YC, Liu R, et al. Neuronal subtypes and diversity revealed by single-nucleus RNA sequencing of the human brain. Science. 2016;352(6293):1586–90. .
[CAS](https://cellandbioscience.biomedcentral.com/articles/cas-redirect/1:CAS:528:DC%2BC28XhtVaitLzF)
  41. Fan X, Dong J, Zhong S, Wei Y, Wu Q, Yan L, et al. Spatial transcriptomic survey of human embryonic cerebral cortex by single-cell RNA-seq analysis. Cell Res. 2018;28(7):730–45. .
[CAS](https://cellandbioscience.biomedcentral.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXht1ektLbL)
  42. Zhong S, Zhang S, Fan X, Wu Q, Yan L, Dong J, et al. A single-cell RNA-seq survey of the developmental landscape of the human prefrontal cortex. Nature. 2018;555(7697):524–8. .
[CAS](https://cellandbioscience.biomedcentral.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXksFegurc%3D)
  43. Carter RA, Bihannic L, Rosencrance C, Hadley JL, Tong Y, Phoenix TN, et al. A single-cell transcriptional atlas of the developing murine cerebellum. Curr Biol. 2018;28(18):2910–20. .
[CAS](https://cellandbioscience.biomedcentral.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhslahs73L)
  44. Chen Y, Zheng Y, Gao Y, Lin Z, Yang S, Wang T, et al. Single-cell RNA-seq uncovers dynamic processes and critical regulators in mouse spermatogenesis. Cell Res. 2018;28(9):879–96. .
[CAS](https://cellandbioscience.biomedcentral.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhsVWhtrfI)
  45. Wang M, Liu X, Chang G, Chen Y, An G, Yan L, et al. Single-cell RNA sequencing analysis reveals sequential cell fate transition during human spermatogenesis. Cell Stem Cell. 2018;23(4):599–614. .
[CAS](https://cellandbioscience.biomedcentral.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhs1eksLvK)
  46. Hou Y, Fan W, Yan L, Li R, Lian Y, Huang J, et al. Genome analyses of single human oocytes. Cell. 2013;155(7):1492–506. .
[CAS](https://cellandbioscience.biomedcentral.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXhvFylsrbE)
  47. Li W, Ma Y, Yu S, Sun N, Wang L, Chen D, et al. The mutation-free embryo for in vitro fertilization selected by MALBAC-PGD resulted in a healthy live birth from a family carrying PKD 1 mutation. J Assist Reprod Genet. 2017;34(12):1653–8. .
  48. Wagner DE, Weinreb C, Collins ZM, Briggs JA, Megason SG, Klein AM. Single-cell mapping of gene expression landscapes and lineage in the zebrafish embryo. Science. 2018;360(6392):981–7. .
[CAS](https://cellandbioscience.biomedcentral.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhtVags77J)
  49. Farrell JA, Wang Y, Riesenfeld SJ, Shekhar K, Regev A, Schier AF. Single-cell reconstruction of developmental trajectories during zebrafish embryogenesis. Science. 2018. .
  50. Briggs JA, Weinreb C, Wagner DE, Megason S, Peshkin L, Kirschner MW, et al. The dynamics of gene expression in vertebrate embryogenesis at single-cell resolution. Science. 2018. .
  51. Li L, Guo F, Gao Y, Ren Y, Yuan P, Yan L, et al. Publisher Correction: Single-cell multi-omics sequencing of human early embryos. Nat Cell Biol. 2018;20(10):1227. .
[CAS](https://cellandbioscience.biomedcentral.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhtlClsLrL)
  52. Vento-Tormo R, Efremova M, Botting RA, Turco MY, Vento-Tormo M, Meyer KB, et al. Single-cell reconstruction of the early maternal-fetal interface in humans. Nature. 2018;563(7731):347–53. .
[CAS](https://cellandbioscience.biomedcentral.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXit1SnsbzI)
  53. Crinier A, Milpied P, Escaliere B, Piperoglou C, Galluso J, Balsamo A, et al. High-dimensional single-cell analysis identifies organ-specific signatures and conserved NK cell subsets in humans and mice. Immunity. 2018;49(5):971–86. .
[CAS](https://cellandbioscience.biomedcentral.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXitFWjtrfE)
  54. Villani AC, Satija R, Reynolds G, Sarkizova S, Shekhar K, Fletcher J, et al. Single-cell RNA-seq reveals new types of human blood dendritic cells, monocytes, and progenitors. Science. 2017. .
  55. Xin G, Zander R, Schauder DM, Chen Y, Weinstein JS, Drobyski WR, et al. Single-cell RNA sequencing unveils an IL-10-producing helper subset that sustains humoral immunity during persistent infection. Nat Commun. 2018;9(1):5037. .
[CAS](https://cellandbioscience.biomedcentral.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXitlyhsr%2FI)
  56. Papalexi E, Satija R. Single-cell RNA sequencing to explore immune cell heterogeneity. Nat Rev Immunol. 2018;18(1):35–45. .
[CAS](https://cellandbioscience.biomedcentral.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXht1yiu7vP)
  57. Martinez-Jimenez CP, Eling N, Chen HC, Vallejos CA, Kolodziejczyk AA, Connor F, et al. Aging increases cell-to-cell transcriptional variability upon immune stimulation. Science. 2017;355(6332):1433–6. .
[CAS](https://cellandbioscience.biomedcentral.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXlsVWhu7Y%3D)
  58. Jordao MJC, Sankowski R, Brendecke SM, Locatelli G, Tai YH, et al. Single-cell profiling identifies myeloid cell subsets with distinct fates during neuroinflammation. Science. 2019. .
  59. Haber AL, Biton M, Rogel N, Herbst RH, Shekhar K, Smillie C, et al. A single-cell survey of the small intestinal epithelium. Nature. 2017;551(7680):333–9. .
[CAS](https://cellandbioscience.biomedcentral.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhslygtLjK)
  60. Gao S, Yan L, Wang R, Li J, Yong J, Zhou X, et al. Tracing the temporal-spatial transcriptome landscapes of the human fetal digestive tract using single-cell RNA-sequencing. Nat Cell Biol. 2018;20(6):721–34. .
[CAS](https://cellandbioscience.biomedcentral.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhtVaisLnO)
  61. Wang P, Chen Y, Yong J, Cui Y, Wang R, Wen L, et al. Dissecting the global dynamic molecular profiles of human fetal kidney development by single-cell RNA sequencing. Cell Rep. 2018;24(13):3554–67. .
[CAS](https://cellandbioscience.biomedcentral.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhslyqtr7O)

## Acknowledgements
The authors would like to thank Dr. Dongpei Li in Medical College of Georgia, Augusta University, GA USA for proofreading and constructive criticism of the manuscript.
## Funding
This work was supported partly by National Natural Science Foundation of China (81541153); Guangdong Provincial Science and Technology Programs (2016A050503046, 2015A050502048 and 2016B030309002); The Public Service Platform of South China Sea for R&D Marine Biomedicine Resources (GDMUK201808).
## Author information
Author notes
  1. Xiaoning Tang and Yongmei Huang contributed equally to this work

### Authors and Affiliations
  1. The Marine Biomedical Research Institute, Guangdong Medical University, Zhanjiang, 524023, China
Xiaoning Tang, Yongmei Huang, Jinli Lei, Hui Luo & Xiao Zhu
  2. Guangdong Provincial Zhanjiang Bay Key Laboratory, Zhanjiang, 524023, China
Xiaoning Tang, Yongmei Huang, Jinli Lei, Hui Luo & Xiao Zhu

Authors
  1. Xiaoning Tang
[View author publications](https://www.biomedcentral.com/search?query=author%23Xiaoning%20Tang)
You can also search for this author in
  2. Yongmei Huang
[View author publications](https://www.biomedcentral.com/search?query=author%23Yongmei%20Huang)
You can also search for this author in
  3. Jinli Lei
[View author publications](https://www.biomedcentral.com/search?query=author%23Jinli%20Lei)
You can also search for this author in
  4. Hui Luo
[View author publications](https://www.biomedcentral.com/search?query=author%23Hui%20Luo)
You can also search for this author in
  5. Xiao Zhu
[View author publications](https://www.biomedcentral.com/search?query=author%23Xiao%20Zhu)
You can also search for this author in

### Contributions
All authors read and approved the final manuscript.
### Corresponding authors
Correspondence to or .
## Ethics declarations
### Ethics approval and consent to participate
Not applicable.
### Consent for publication
All authors consent for publication.
### Competing interests
The authors declare that they have no competing interests.
## Additional information
### Publisher's Note
Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.
## Rights and permissions
**Open Access** This article is distributed under the terms of the Creative Commons Attribution 4.0 International License (), which permits unrestricted use, distribution, and reproduction in any medium, provided you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The Creative Commons Public Domain Dedication waiver () applies to the data made available in this article, unless otherwise stated.
## 
### Cite this article
Tang, X., Huang, Y., Lei, J. _et al._ The single-cell sequencing: new developments and medical applications. _Cell Biosci_ **9** , 53 (2019). https://doi.org/10.1186/s13578-019-0314-y
  * Received: 23 April 2019
  * Accepted: 18 June 2019
  * Published: 26 June 2019
  * DOI: https://doi.org/10.1186/s13578-019-0314-y

### 
Anyone you share the following link with will be able to read this content:
Get shareable link
Sorry, a shareable link is not currently available for this article.
Copy to clipboard
Provided by the Springer Nature SharedIt content-sharing initiative 
### Keywords
  * [Cancer](https://cellandbioscience.biomedcentral.com/search?query=Cancer&facet-discipline="Life%20Sciences")
  * [Clinical applications](https://cellandbioscience.biomedcentral.com/search?query=Clinical%20applications&facet-discipline="Life%20Sciences")
  * [Developments](https://cellandbioscience.biomedcentral.com/search?query=Developments&facet-discipline="Life%20Sciences")
  * [Immunology](https://cellandbioscience.biomedcentral.com/search?query=Immunology&facet-discipline="Life%20Sciences")
  * [Single-cell sequencing technologies](https://cellandbioscience.biomedcentral.com/search?query=Single-cell%20sequencing%20technologies&facet-discipline="Life%20Sciences")
  * [Transcriptome](https://cellandbioscience.biomedcentral.com/search?query=Transcriptome&facet-discipline="Life%20Sciences")

[ Download PDF ](https://cellandbioscience.biomedcentral.com/counter/pdf/10.1186/s13578-019-0314-y.pdf)
  * Sections
  * Figures
  * References

  * [Abstract](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y#Abs1)
  * [Background](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y#Sec1)
  * [Single-cell sequencing methods and recent developments](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y#Sec2)
  * [Single-cell sequencing applications](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y#Sec5)
  * [Applications in the nervous system](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y#Sec16)
  * [Applications in the field of reproductive and embryonic medicine](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y#Sec19)
  * [Applications in the field of immunology](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y#Sec23)
  * [Application in the digestive system and urinary system](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y#Sec26)
  * [Conclusion and perspectives](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y#Sec27)
  * [Availability of data and materials](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y#availability-of-data-and-materials)
  * [Abbreviations](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y#abbreviations)
  * [References](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y#Bib1)
  * [Acknowledgements](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y#Ack1)
  * [Funding](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y#Fun)
  * [Author information](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y#author-information)
  * [Ethics declarations](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y#ethics)
  * [Additional information](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y#additional-information)
  * [Rights and permissions](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y#rightslink)
  * [

Advertisement
  * **Fig. 1**
[View in article](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y#Fig1)[Full size image](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y/figures/1)
  * **Fig. 2**
[View in article](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y#Fig2)[Full size image](https://cellandbioscience.biomedcentral.com/articles/10.1186/s13578-019-0314-y/figures/2)

  1. Wen L, Tang F. Boosting the power of single-cell analysis. Nat Biotechnol. 2018;36(5):408–9. .
[CAS](https://cellandbioscience.biomedcentral.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXovFGntrw%3D)
  2. Pennisi E. The biology of genomes. Single-cell sequencing tackles basic and biomedical questions. Science. 2012;336(6084):976–7. .
[CAS](https://cellandbioscience.biomedcentral.com/articles/cas-redirect/1:CAS:528:DC%2BC38XptVCmu7c%3D)
  3. Vitak SA, Torkenczy KA, Rosenkrantz JL, Fields AJ, Christiansen L, Wong MH, et al. Sequencing thousands of single-cell genomes with combinatorial indexing. Nat Methods. 2017;14(3):302–8. .
[CAS](https://cellandbioscience.biomedcentral.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhvV2hsrc%3D)
  4. Chen C, Xing D, Tan L, Li H, Zhou G, Huang L, et al. Single-cell whole-genome analyses by Linear Amplification via Transposon Insertion (LIANTI). Science. 2017;356(6334):189–94. .
[CAS](https://cellandbioscience.biomedcentral.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXlvVems70%3D)
  5. Guo F, Li L, Li J, Wu X, Hu B, Zhu P, et al. Single-cell multi-omics sequencing of mouse early embryos and embryonic stem cells. Cell Res. 2017;27(8):967–88. .
[CAS](https://cellandbioscience.biomedcentral.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhtVCiurfF)
  6. Casasent AK, Schalck A, Gao R, Sei E, Long A, Pangburn W, et al. Multiclonal invasion in breast tumors identified by topographic single cell sequencing. Cell. 2018;172(1–2):205–17. .
[CAS](https://cellandbioscience.biomedcentral.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXkslehsA%3D%3D)
  7. Demaree B, Weisgerber D, Lan F, Abate AR. An ultrahigh-throughput microfluidic platform for single-cell genome sequencing. J Vis Exp. 2018;135:e57598. .
[CAS](https://cellandbioscience.biomedcentral.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXitlOmt73O)
  8. Han X, Wang R, Zhou Y, Fei L, Sun H, Lai S, et al. Mapping the mouse cell atlas by microwell-seq. Cell. 2018;172(5):1091–107. .
[CAS](https://cellandbioscience.biomedcentral.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXjt12ltr4%3D)
  9. Rosenberg AB, Roco CM, Muscat RA, Kuchina A, Sample P, Yao Z, et al. Single-cell profiling of the developing mouse brain and spinal cord with split-pool barcoding. Science. 2018;360(6385):176–82. .
[CAS](https://cellandbioscience.biomedcentral.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXnt1GhtbY%3D)
  10. Datlinger P, Rendeiro AF, Schmidl C, Krausgruber T, Traxler P, Klughammer J, et al. Pooled CRISPR screening with single-cell transcriptome readout. Nat Methods. 2017;14(3):297–301. .
[CAS](https://cellandbioscience.biomedcentral.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhtlKjsrk%3D)
  11. Gasperini M, Hill AJ, McFaline-Figueroa JL, Martin B, Kim S, Zhang MD, et al. A Genome-wide framework for mapping gene regulation via cellular genetic screens. Cell. 2019;176(1–2):377–90. .
[CAS](https://cellandbioscience.biomedcentral.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXjs1Squg%3D%3D)
  12. Habib N, Avraham-Davidi I, Basu A, Burks T, Shekhar K, Hofree M, et al. Massively parallel single-nucleus RNA-seq with DroNc-seq. Nat Methods. 2017;14(10):955–8. .
[CAS](https://cellandbioscience.biomedcentral.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhtl2gtL7L)
  13. Lake BB, Chen S, Sos BC, Fan J, Kaeser GE, Yung YC, et al. Integrative single-cell analysis of transcriptional and epigenetic states in the human adult brain. Nat Biotechnol. 2018;36(1):70–80. .
[CAS](https://cellandbioscience.biomedcentral.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhvFGmurrM)
  14. Darmanis S, Sloan SA, Croote D, Mignardi M, Chernikova S, Samghababi P, et al. Single-cell RNA-seq analysis of infiltrating neoplastic cells at the migrating front of human glioblastoma. Cell Rep. 2017;21(5):1399–410. .
[CAS](https://cellandbioscience.biomedcentral.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhslGit7zP)
  15. Xu Y, Zhou X. Applications of single-cell sequencing for multiomics. Methods Mol Biol. 2018;1754:327–74. .
[CAS](https://cellandbioscience.biomedcentral.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXitVOgt77L)
  16. Ley TJ, Mardis ER, Ding L, Fulton B, McLellan MD, Chen K, et al. DNA sequencing of a cytogenetically normal acute myeloid leukaemia genome. Nature. 2008;456(7218):66–72. .
[CAS](https://cellandbioscience.biomedcentral.com/articles/cas-redirect/1:CAS:528:DC%2BD1cXhtlCjtrfE)
  17. Lawson DA, Kessenbrock K, Davis RT, Pervolarakis N, Werb Z. Tumour heterogeneity and metastasis at single-cell resolution. Nat Cell Biol. 2018;20(12):1349–60. .
[CAS](https://cellandbioscience.biomedcentral.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXitlent7zN)
  18. Caswell DR, Swanton C. The role of tumour heterogeneity and clonal cooperativity in metastasis, immune evasion and clinical outcome. BMC Med. 2017;15(1):133. .
[CAS](https://cellandbioscience.biomedcentral.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXitVOrtr7P)
  19. Alderton GK. Tumour evolution: epigenetic and genetic heterogeneity in metastasis. Nat Rev Cancer. 2017;17(3):141. .
[CAS](https://cellandbioscience.biomedcentral.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXjt1ags7o%3D)
  20. Zhang L, Yu X, Zheng L, Zhang Y, Li Y, Fang Q, et al. Lineage tracking reveals dynamic relationships of T cells in colorectal cancer. Nature. 2018;564(7735):268–72. .
[CAS](https://cellandbioscience.biomedcentral.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXisVarsr7F)
  21. Bian S, Hou Y, Zhou X, Li X, Yong J, Wang Y, et al. Single-cell multiomics sequencing and analyses of human colorectal cancer. Science. 2018;362(6418):1060–3. .
[CAS](https://cellandbioscience.biomedcentral.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXitlWhtLnI)
  22. Nguyen QH, Pervolarakis N, Nee K, Kessenbrock K. Experimental considerations for single-cell RNA sequencing approaches. Front Cell Dev Biol. 2018;6:108. .
  23. Nguyen A, Khoo WH, Moran I, Croucher PI, Phan TG. Single cell RNA sequencing of rare immune cell populations. Front Immunol. 2018;9:1553. .
[CAS](https://cellandbioscience.biomedcentral.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXitlSisbnJ)
  24. Nguyen QH, Pervolarakis N, Blake K, Ma D, Davis RT, James N, et al. Profiling human breast epithelial cells using single cell RNA sequencing identifies cell diversity. Nat Commun. 2018;9(1):2028. .
[CAS](https://cellandbioscience.biomedcentral.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhtFelurvK)
  25. Azizi E, Carr AJ, Plitas G, Cornish AE, Konopacki C, Prabhakaran S, et al. Single-cell map of diverse immune phenotypes in the breast tumor microenvironment. Cell. 2018;174(5):1293–308. .
[CAS](https://cellandbioscience.biomedcentral.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXht1Cis7vN)
  26. Zhu D, Zhao Z, Cui G, Chang S, Hu L, See YX, et al. Single-Cell Transcriptome Analysis Reveals Estrogen Signaling Coordinately Augments One-Carbon, Polyamine, and Purine Synthesis in Breast Cancer. Cell Rep. 2018;25(8):2285–98. .
[CAS](https://cellandbioscience.biomedcentral.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXitlShtb3O)
  27. Venteicher AS, Tirosh I, Hebert C, Yizhak K, Neftel C, Filbin MG, et al. Decoupling genetics, lineages, and microenvironment in IDH-mutant gliomas by single-cell RNA-seq. Science. 2017. .
  28. Tirosh I, Venteicher AS, Hebert C, Escalante LE, Patel AP, Yizhak K, et al. Single-cell RNA-seq supports a developmental hierarchy in human oligodendroglioma. Nature. 2016;539(7628):309–13. .
[CAS](https://cellandbioscience.biomedcentral.com/articles/cas-redirect/1:CAS:528:DC%2BC28XhvVSiu73J)
  29. Puram SV, Tirosh I, Parikh AS, Patel AP, Yizhak K, Gillespie S, et al. Single-cell transcriptomic analysis of primary and metastatic tumor ecosystems in head and neck cancer. Cell. 2017;171(7):1611–24. .
[CAS](https://cellandbioscience.biomedcentral.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhvFWms7nE)
  30. Ledergor G, Weiner A, Zada M, Wang SY, Cohen YC, Gatt ME, et al. Single cell dissection of plasma cell heterogeneity in symptomatic and asymptomatic myeloma. Nat Med. 2018;24(12):1867–76. .
[CAS](https://cellandbioscience.biomedcentral.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXisVGmsLzO)
  31. Hou Y, Song L, Zhu P, Zhang B, Tao Y, Xu X, et al. Single-cell exome sequencing and monoclonal evolution of a JAK2-negative myeloproliferative neoplasm. Cell. 2012;148(5):873–85. .
[CAS](https://cellandbioscience.biomedcentral.com/articles/cas-redirect/1:CAS:528:DC%2BC38XjtlSltr0%3D)
  32. De Bie J, Demeyer S, Alberti-Servera L, Geerdens E, Segers H, Broux M, et al. Single-cell sequencing reveals the origin and the order of mutation acquisition in T-cell acute lymphoblastic leukemia. Leukemia. 2018;32(6):1358–69. .
[CAS](https://cellandbioscience.biomedcentral.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhtFOnu7fE)
  33. Guo X, Zhang Y, Zheng L, Zheng C, Song J, Zhang Q, et al. Global characterization of T cells in non-small-cell lung cancer by single-cell sequencing. Nat Med. 2018;24(7):978–85. .
[CAS](https://cellandbioscience.biomedcentral.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXht1WqsLjE)
  34. Zheng C, Zheng L, Yoo JK, Guo H, Zhang Y, Guo X, et al. Landscape of infiltrating T Cells in liver cancer revealed by single-cell sequencing. Cell. 2017;169(7):1342–56. .
[CAS](https://cellandbioscience.biomedcentral.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhtVWrtrvE)
  35. DeLorenzo S, Brauer SL, Edgmont CA, Herfort L, Tebo BM, Zuber P. Ubiquitous dissolved inorganic carbon assimilation by marine bacteria in the Pacific Northwest coastal ocean as determined by stable isotope probing. PLoS ONE. 2012;7(10):e46695. .
[CAS](https://cellandbioscience.biomedcentral.com/articles/cas-redirect/1:CAS:528:DC%2BC38XhsFWgsbfP)
  36. Lan F, Demaree B, Ahmed N, Abate AR. Single-cell genome sequencing at ultra-high-throughput with microfluidic droplet barcoding. Nat Biotechnol. 2017;35(7):640–6. .
[CAS](https://cellandbioscience.biomedcentral.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXosFSru7c%3D)
  37. McConnell MJ, Lindberg MR, Brennand KJ, Piper JC, Voet T, Cowing-Zitron C, et al. Mosaic copy number variation in human neurons. Science. 2013;342(6158):632–7. .
[CAS](https://cellandbioscience.biomedcentral.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXhs1yrsL7K)
  38. Li H, Horns F, Wu B, Xie Q, Li J, Li T, et al. Classifying drosophila olfactory projection neuron subtypes by single-cell RNA sequencing. Cell. 2017;171(5):1206–20. .
[CAS](https://cellandbioscience.biomedcentral.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhvFOhs7vJ)
  39. Luo C, Keown CL, Kurihara L, Zhou J, He Y, Li J, et al. Single-cell methylomes identify neuronal subtypes and regulatory elements in mammalian cortex. Science. 2017;357(6351):600–4. .
[CAS](https://cellandbioscience.biomedcentral.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhtlehur%2FE)
  40. Lake BB, Ai R, Kaeser GE, Salathia NS, Yung YC, Liu R, et al. Neuronal subtypes and diversity revealed by single-nucleus RNA sequencing of the human brain. Science. 2016;352(6293):1586–90. .
[CAS](https://cellandbioscience.biomedcentral.com/articles/cas-redirect/1:CAS:528:DC%2BC28XhtVaitLzF)
  41. Fan X, Dong J, Zhong S, Wei Y, Wu Q, Yan L, et al. Spatial transcriptomic survey of human embryonic cerebral cortex by single-cell RNA-seq analysis. Cell Res. 2018;28(7):730–45. .
[CAS](https://cellandbioscience.biomedcentral.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXht1ektLbL)
  42. Zhong S, Zhang S, Fan X, Wu Q, Yan L, Dong J, et al. A single-cell RNA-seq survey of the developmental landscape of the human prefrontal cortex. Nature. 2018;555(7697):524–8. .
[CAS](https://cellandbioscience.biomedcentral.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXksFegurc%3D)
  43. Carter RA, Bihannic L, Rosencrance C, Hadley JL, Tong Y, Phoenix TN, et al. A single-cell transcriptional atlas of the developing murine cerebellum. Curr Biol. 2018;28(18):2910–20. .
[CAS](https://cellandbioscience.biomedcentral.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhslahs73L)
  44. Chen Y, Zheng Y, Gao Y, Lin Z, Yang S, Wang T, et al. Single-cell RNA-seq uncovers dynamic processes and critical regulators in mouse spermatogenesis. Cell Res. 2018;28(9):879–96. .
[CAS](https://cellandbioscience.biomedcentral.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhsVWhtrfI)
  45. Wang M, Liu X, Chang G, Chen Y, An G, Yan L, et al. Single-cell RNA sequencing analysis reveals sequential cell fate transition during human spermatogenesis. Cell Stem Cell. 2018;23(4):599–614. .
[CAS](https://cellandbioscience.biomedcentral.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhs1eksLvK)
  46. Hou Y, Fan W, Yan L, Li R, Lian Y, Huang J, et al. Genome analyses of single human oocytes. Cell. 2013;155(7):1492–506. .
[CAS](https://cellandbioscience.biomedcentral.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXhvFylsrbE)
  47. Li W, Ma Y, Yu S, Sun N, Wang L, Chen D, et al. The mutation-free embryo for in vitro fertilization selected by MALBAC-PGD resulted in a healthy live birth from a family carrying PKD 1 mutation. J Assist Reprod Genet. 2017;34(12):1653–8. .
  48. Wagner DE, Weinreb C, Collins ZM, Briggs JA, Megason SG, Klein AM. Single-cell mapping of gene expression landscapes and lineage in the zebrafish embryo. Science. 2018;360(6392):981–7. .
[CAS](https://cellandbioscience.biomedcentral.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhtVags77J)
  49. Farrell JA, Wang Y, Riesenfeld SJ, Shekhar K, Regev A, Schier AF. Single-cell reconstruction of developmental trajectories during zebrafish embryogenesis. Science. 2018. .
  50. Briggs JA, Weinreb C, Wagner DE, Megason S, Peshkin L, Kirschner MW, et al. The dynamics of gene expression in vertebrate embryogenesis at single-cell resolution. Science. 2018. .
  51. Li L, Guo F, Gao Y, Ren Y, Yuan P, Yan L, et al. Publisher Correction: Single-cell multi-omics sequencing of human early embryos. Nat Cell Biol. 2018;20(10):1227. .
[CAS](https://cellandbioscience.biomedcentral.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhtlClsLrL)
  52. Vento-Tormo R, Efremova M, Botting RA, Turco MY, Vento-Tormo M, Meyer KB, et al. Single-cell reconstruction of the early maternal-fetal interface in humans. Nature. 2018;563(7731):347–53. .
[CAS](https://cellandbioscience.biomedcentral.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXit1SnsbzI)
  53. Crinier A, Milpied P, Escaliere B, Piperoglou C, Galluso J, Balsamo A, et al. High-dimensional single-cell analysis identifies organ-specific signatures and conserved NK cell subsets in humans and mice. Immunity. 2018;49(5):971–86. .
[CAS](https://cellandbioscience.biomedcentral.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXitFWjtrfE)
  54. Villani AC, Satija R, Reynolds G, Sarkizova S, Shekhar K, Fletcher J, et al. Single-cell RNA-seq reveals new types of human blood dendritic cells, monocytes, and progenitors. Science. 2017. .
  55. Xin G, Zander R, Schauder DM, Chen Y, Weinstein JS, Drobyski WR, et al. Single-cell RNA sequencing unveils an IL-10-producing helper subset that sustains humoral immunity during persistent infection. Nat Commun. 2018;9(1):5037. .
[CAS](https://cellandbioscience.biomedcentral.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXitlyhsr%2FI)
  56. Papalexi E, Satija R. Single-cell RNA sequencing to explore immune cell heterogeneity. Nat Rev Immunol. 2018;18(1):35–45. .
[CAS](https://cellandbioscience.biomedcentral.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXht1yiu7vP)
  57. Martinez-Jimenez CP, Eling N, Chen HC, Vallejos CA, Kolodziejczyk AA, Connor F, et al. Aging increases cell-to-cell transcriptional variability upon immune stimulation. Science. 2017;355(6332):1433–6. .
[CAS](https://cellandbioscience.biomedcentral.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXlsVWhu7Y%3D)
  58. Jordao MJC, Sankowski R, Brendecke SM, Locatelli G, Tai YH, et al. Single-cell profiling identifies myeloid cell subsets with distinct fates during neuroinflammation. Science. 2019. .
  59. Haber AL, Biton M, Rogel N, Herbst RH, Shekhar K, Smillie C, et al. A single-cell survey of the small intestinal epithelium. Nature. 2017;551(7680):333–9. .
[CAS](https://cellandbioscience.biomedcentral.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhslygtLjK)
  60. Gao S, Yan L, Wang R, Li J, Yong J, Zhou X, et al. Tracing the temporal-spatial transcriptome landscapes of the human fetal digestive tract using single-cell RNA-sequencing. Nat Cell Biol. 2018;20(6):721–34. .
[CAS](https://cellandbioscience.biomedcentral.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhtVaisLnO)
  61. Wang P, Chen Y, Yong J, Cui Y, Wang R, Wen L, et al. Dissecting the global dynamic molecular profiles of human fetal kidney development by single-cell RNA sequencing. Cell Rep. 2018;24(13):3554–67. .
[CAS](https://cellandbioscience.biomedcentral.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhslyqtr7O)

####  Cell & Bioscience
ISSN: 2045-3701
#### Contact us
  * Submission enquiries: 
  * General enquiries: 

![](https://cellandbioscience.biomedcentral.com/track/article/10.1186/s13578-019-0314-y)
  * [Read more on our blogs](https://blogs.biomedcentral.com/)
  * [Receive BMC newsletters](https://www.biomedcentral.com/login)
  * [Manage article alerts](https://www.biomedcentral.com/account)

  * [Policies](https://www.biomedcentral.com/about/policies)
  * [Accessibility](https://www.biomedcentral.com/accessibility)
  * [Press center](https://www.biomedcentral.com/about/press-centre)

  * [Support and Contact](https://support.biomedcentral.com/support/home)
  * [Careers](https://www.biomedcentral.com/about/jobs)

### Follow BMC
By using this website, you agree to our [Terms and Conditions](https://www.biomedcentral.com/terms-and-conditions), , [Privacy statement](https://www.biomedcentral.com/privacy-statement) and [Cookies](https://www.biomedcentral.com/cookies) policy. we use in the preference centre. 
![Springer Nature](https://cellandbioscience.biomedcentral.com/static/images/logo-springernature-acb40b85fb.svg)
© 2025 BioMed Central Ltd unless otherwise stated. Part of . 
  *[DOI]: Digital Object Identifier