[ Skip to main content ](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#main-content)
![](https://pmc.ncbi.nlm.nih.gov/static/img/us_flag.svg)
An official website of the United States government 
Here's how you know
Here's how you know
![](https://pmc.ncbi.nlm.nih.gov/static/img/icon-dot-gov.svg)
**Official websites use .gov** A **.gov** website belongs to an official government organization in the United States. 
![](https://pmc.ncbi.nlm.nih.gov/static/img/icon-https.svg)
**Secure .gov websites use HTTPS** A **lock** (  Lock Locked padlock icon  ) or **https://** means you've safely connected to the .gov website. Share sensitive information only on official, secure websites. 
[ ![
                 NCBI home page
               ](https://pmc.ncbi.nlm.nih.gov/static/img/ncbi-logos/nih-nlm-ncbi--white.svg) ](https://www.ncbi.nlm.nih.gov/)
Search 
Log in
  * [ Dashboard ](https://www.ncbi.nlm.nih.gov/myncbi/)
  * [ Publications ](https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/)
  * [ Account settings ](https://www.ncbi.nlm.nih.gov/account/settings/)
  * Log out 

Search…  Search NCBI 
Primary site navigation 
![Close](https://pmc.ncbi.nlm.nih.gov/static/img/usa-icons/close.svg) Search  ![Search](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582)
Logged in as: 
  * [ Dashboard ](https://www.ncbi.nlm.nih.gov/myncbi/)
  * [ Publications ](https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/)
  * [ Account settings ](https://www.ncbi.nlm.nih.gov/account/settings/)

Log in
[](https://pmc.ncbi.nlm.nih.gov/ "Home")
Search PMC Full-Text Archive Search in PMC ![Search](https://pmc.ncbi.nlm.nih.gov/static/img/usa-icons-bg/search--white.svg)
  * [ Advanced Search ](https://www.ncbi.nlm.nih.gov/pmc/advanced/)
  * [ Journal List ](https://pmc.ncbi.nlm.nih.gov/journals/)
  * [ User Guide ](https://pmc.ncbi.nlm.nih.gov/about/userguide/)

  * View on publisher site
  * [ ](https://pmc.ncbi.nlm.nih.gov/articles/pdf/ehp13855.pdf)Download PDF
  * Add to Collections
  * Cite
  * Permalink
## PERMALINK
Copy

As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with, the contents by NLM or the National Institutes of Health. Learn more: [PMC Disclaimer](https://pmc.ncbi.nlm.nih.gov/about/disclaimer/) | [ PMC 
![Environmental Health Perspectives logo](https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-envhper.png)
Environ Health Perspect
. 2024 May 21;132(5):057006. doi: 
  * [Search in PMC](https://www.ncbi.nlm.nih.gov/pmc/?term=%22Environ%20Health%20Perspect%22%5Bjour%5D)
  * [Search in PubMed](https://pubmed.ncbi.nlm.nih.gov/?term=%22Environ%20Health%20Perspect%22%5Bjour%5D)
  * [View in NLM Catalog](https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Environ%20Health%20Perspect%22%5BTitle%20Abbreviation%5D)
  * [Add to search](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582?term=%22Environ%20Health%20Perspect%22%5Bjour%5D)

# Single Cell Analysis of Human Colonoids Exposed to Uranium-Bearing Dust
[Roger Atanga](https://pubmed.ncbi.nlm.nih.gov/?term=%22Atanga%20R%22%5BAuthor%5D)
### Roger Atanga
1Division of Gastroenterology and Hepatology, Department of Internal Medicine, University of New Mexico Health Sciences Center, Albuquerque, New Mexico, USA
Find articles by [Roger Atanga](https://pubmed.ncbi.nlm.nih.gov/?term=%22Atanga%20R%22%5BAuthor%5D)
1, [Lidia L Appell](https://pubmed.ncbi.nlm.nih.gov/?term=%22Appell%20LL%22%5BAuthor%5D)
### Lidia L Appell
1Division of Gastroenterology and Hepatology, Department of Internal Medicine, University of New Mexico Health Sciences Center, Albuquerque, New Mexico, USA
Find articles by [Lidia L Appell](https://pubmed.ncbi.nlm.nih.gov/?term=%22Appell%20LL%22%5BAuthor%5D)
1, [Myranda N Thompson](https://pubmed.ncbi.nlm.nih.gov/?term=%22Thompson%20MN%22%5BAuthor%5D)
### Myranda N Thompson
1Division of Gastroenterology and Hepatology, Department of Internal Medicine, University of New Mexico Health Sciences Center, Albuquerque, New Mexico, USA
Find articles by [Myranda N Thompson](https://pubmed.ncbi.nlm.nih.gov/?term=%22Thompson%20MN%22%5BAuthor%5D)
1, [Fredine T Lauer](https://pubmed.ncbi.nlm.nih.gov/?term=%22Lauer%20FT%22%5BAuthor%5D)
### Fredine T Lauer
2Department of Pharmaceutical Sciences, College of Pharmacy, University of New Mexico Health Sciences Center, Albuquerque, New Mexico, USA
Find articles by [Fredine T Lauer](https://pubmed.ncbi.nlm.nih.gov/?term=%22Lauer%20FT%22%5BAuthor%5D)
2, [Adrian Brearley](https://pubmed.ncbi.nlm.nih.gov/?term=%22Brearley%20A%22%5BAuthor%5D)
### Adrian Brearley
3Department of Earth and Planetary Sciences, College of Arts and Sciences, University of New Mexico, Albuquerque, New Mexico, USA
Find articles by [Adrian Brearley](https://pubmed.ncbi.nlm.nih.gov/?term=%22Brearley%20A%22%5BAuthor%5D)
3, [Matthew J Campen](https://pubmed.ncbi.nlm.nih.gov/?term=%22Campen%20MJ%22%5BAuthor%5D)
### Matthew J Campen
2Department of Pharmaceutical Sciences, College of Pharmacy, University of New Mexico Health Sciences Center, Albuquerque, New Mexico, USA
Find articles by [Matthew J Campen](https://pubmed.ncbi.nlm.nih.gov/?term=%22Campen%20MJ%22%5BAuthor%5D)
2, [Eliseo F Castillo](https://pubmed.ncbi.nlm.nih.gov/?term=%22Castillo%20EF%22%5BAuthor%5D)
### Eliseo F Castillo
1Division of Gastroenterology and Hepatology, Department of Internal Medicine, University of New Mexico Health Sciences Center, Albuquerque, New Mexico, USA
4Autophagy, Inflammation and Metabolism Center of Biomedical Research Excellence, University of New Mexico Health Sciences Center, Albuquerque, New Mexico, USA
Find articles by [Eliseo F Castillo](https://pubmed.ncbi.nlm.nih.gov/?term=%22Castillo%20EF%22%5BAuthor%5D)
1,4, [Julie G In](https://pubmed.ncbi.nlm.nih.gov/?term=%22In%20JG%22%5BAuthor%5D)
### Julie G In
1Division of Gastroenterology and Hepatology, Department of Internal Medicine, University of New Mexico Health Sciences Center, Albuquerque, New Mexico, USA
4Autophagy, Inflammation and Metabolism Center of Biomedical Research Excellence, University of New Mexico Health Sciences Center, Albuquerque, New Mexico, USA
Find articles by [Julie G In](https://pubmed.ncbi.nlm.nih.gov/?term=%22In%20JG%22%5BAuthor%5D)
1,4,✉
  * Author information
  * Article notes
  * 

1Division of Gastroenterology and Hepatology, Department of Internal Medicine, University of New Mexico Health Sciences Center, Albuquerque, New Mexico, USA
2Department of Pharmaceutical Sciences, College of Pharmacy, University of New Mexico Health Sciences Center, Albuquerque, New Mexico, USA
3Department of Earth and Planetary Sciences, College of Arts and Sciences, University of New Mexico, Albuquerque, New Mexico, USA
4Autophagy, Inflammation and Metabolism Center of Biomedical Research Excellence, University of New Mexico Health Sciences Center, Albuquerque, New Mexico, USA
✉
Address correspondence to Julie G. In, Division of Gastroenterology and Hepatology, Department of Internal Medicine, University of New Mexico Health Sciences Center, Albuquerque, NM 87131 USA. Email: jgin@salud.unm.edu
✉
Corresponding author.
Supplemental Material is available online ().
The authors declare that there are no conflicts of interest with the contents of this manuscript.
**Note to readers with disabilities:** _EHP_ strives to ensure that all journal content is accessible to all readers. However, some figures and Supplemental Material published in _EHP_ articles may not conform to [508 standards](http://ehp.niehs.nih.gov/accessibility/) due to the complexity of the information being presented. If you need assistance accessing journal content, please contact ehpsubmissions@niehs.nih.gov. Our staff will work with you to assess and meet your accessibility needs within 3 working days.
Received 2023 Aug 21; Revised 2024 Apr 23; Accepted 2024 Apr 30; Collection date 2024 May.
_EHP_ is an open-access journal published with support from the National Institute of Environmental Health Sciences, National Institutes of Health. All content is public domain unless otherwise noted.
[PMC 
PMCID: PMC11108582 PMID: [38771937](https://pubmed.ncbi.nlm.nih.gov/38771937/)
## Abstract
### Background:
Uranium exposure remains an important environmental legacy and physiological health concern, with hundreds of abandoned uranium mines located in the Southwestern United States largely impacting underserved indigenous communities. The negative effects of heavy metals on barrier permeability and inhibition of intestinal epithelial healing have been described; however, transcriptomic changes within the intestinal epithelial cells and impacts on lineage differentiation are largely unknown.
### Objectives:
Herein, we sought to determine the molecular and cellular changes that occur in the colon in response to uranium bearing dust (UBD) exposure.
### Methods:
Human colonoids from three biologically distinct donors were acutely exposed to UBD then digested for single cell RNA sequencing to define the molecular changes that occur to specific identities of colonic epithelial cells. Validation in colonoids was assessed using morphological and imaging techniques.
### Results:
Human colonoids acutely exposed to UBD exhibited disrupted proliferation and hyperplastic differentiation of the secretory lineage cell, enteroendocrine cells (EEC). Single-cell RNA sequencing also showed more EEC subtypes present in UBD-exposed colonoids.
### Discussion:
These findings highlight the significance of crypt-based proliferative cells and secretory cell differentiation using human colonoids to model major colonic responses to uranium-bearing particulate dust exposure. 
## Introduction
Uranium mining in the United States arose in the 1940s, peaked in the 1970s, and subsided in the late 1980s, with extensive activity concentrated in Navajo, Puebloan, and other indigenous lands in the Southwest.[1](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#c1) To date, over 500 mines in this region are considered abandoned uranium mines (AUM).[2](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#c2) Heavy metals from AUMs transfer into the sediment and groundwater,[3](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#c3),[4](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#c4) with nonfissile uranium (238U) levels in surrounding communities exceeding maximum contamination levels () dictated by the US Environmental Protection Agency.[5](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#c5) For example, well water in Nambe Pueblo (Santa Fe, New Mexico) once contained 238U.[6](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#c6) Even partially remediated uranium mines degrade over time and cause heavy metal contamination of the surrounding sediment and groundwater.[7](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#c7) This has led to unacceptable 238U levels found in community water sources throughout the US, even in aquifers that are not near uranium mines.[8](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#c8),[9](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#c9)
Although 238U is weakly radioactive, it primarily negatively influences human health as a chemotoxicant and can accumulate in various organs.[10](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#c10)–[12](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#c12) The communities that reside near AUMs are chronically exposed to particulate dust with varying levels of uranium [referred to as uranium-bearing dust (UBD)] via inhalation and ingestion,[13](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#c13),[14](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#c14) increasing their overall health risks. Studies in rodents found that chronic environmental exposure to UBD was linked to impaired neurological function,[15](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#c15) pulmonary damage,[16](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#c16) hepatotoxicity,[17](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#c17) and reproductive defects.[18](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#c18) Epidemiological studies in humans have found that uranium exposure correlated with renal damage.[19](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#c19)
Specific to the gut, chronic exposure to environmental toxicants and heavy metals has been linked to intestinal inflammation,[20](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#c20) increased susceptibility to pathogen-induced diseases,[21](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#c21),[22](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#c22) circulating inflammatory potential,[23](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#c23) and colorectal cancer.[24](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#c24) Additionally, heavy metal exposure has been shown to negatively impact the microbiota,[25](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#c25) immune cells,[26](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#c26),[27](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#c27) and the intestinal epithelial barrier[28](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#c28) in both rodent and Caco-2 cell studies. Yet, the molecular changes that can lead to these pathologies are undefined. The 238U, via UBD, enters the intestine via ingestion or inhalation. Inhaled sedimentary particles are transported to the epiglottis by tracheobronchial cilia, then swallowed.[29](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#c29) Reported amounts and rate of uranium absorption in the digestive tract were fairly low, from 0.1 to 7%,[30](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#c30) bringing into question how systemic health impacts arise from direct toxicity to the gut. In mice, ingested uranium was primarily absorbed in the lower intestinal tract (colon),[26](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#c26) suggesting 238U-related cellular changes may originate in the colon.
The intestine is the largest immune organ in humans[31](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#c31) and directly communicates with the brain via hormone-secreting enteroendocrine cells (EECs) and the enteric nervous system.[32](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#c32),[33](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#c33) Thus, detrimental effects to the intestine can indirectly affect systemic health. The UBD utilized in this study was environmentally relevant, as it was obtained from a church attic in Paguate, NM, a village in the Laguna Pueblo located next to one of the largest open-pit uranium mines in the United States, the Jackpile Mine.[34](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#c34)
Human colonoid cultures are a tractable, epithelial-only model that can indefinitely proliferate due to the presence of adult intestinal stem cells,[35](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#c35) making them a valuable model to study how the epithelium responds to intestinal injury and the factors involved in epithelial regeneration. The current lack of understanding on how heavy metals, specifically in particulate form like UBD, affect intestinal epithelia suggest a novel approach is needed.
Enteroendocrine cells are rare hormone-producing cells in the gut.[36](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#c36),[37](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#c37) They are distributed throughout the intestinal epithelium, making up only of the epithelial cell population. Thus, EEC function is difficult to study in whole animals. Recent studies showed that intestinal organoids are a physiologically relevant model to study EEC differentiation and function.[38](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#c38),[39](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#c39) Intestinal organoids produce the various EEC subtypes, retain regional specificity to the gut hormones produced, and respond to a variety of physiological stimuli.[40](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#c40)
In this study, we used single-cell RNA sequencing (scRNA-seq) to characterize UBD-induced changes to intestinal epithelial cell lineages. We used human colonoids as a relevant model of human intestine to characterize UBD-induced transcriptomic changes and present them as a relevant pathophysiological model for studies in host–toxicant interactions.
## Methods
### Tissue Collection and Colonoid Generation
Human colonoid studies were reviewed and approved by the Johns Hopkins University School of Medicine institutional review board (IRB number NA_00038329) and University of New Mexico institutional review board (IRB number 18-171). Colonic biopsies were obtained from healthy individuals undergoing screening colonoscopies who had given informed written consent. All colonoids were generated from proximal colonic biopsies from patients undergoing routine, screening colonoscopy and with no pathophysiology. Colonic crypt isolation and colonoid generation were prepared as previously reported.[41](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#c41),[42](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#c42) Isolated crypts were resuspended in Matrigel (Corning) and droplets were plated in a 24-well plate (Corning). After polymerization at 37°C, of organoid expansion media was added for 2 d [Advanced Dulbecco’s modified Eagle medium (DMEM)/Ham’s F-12 (ThermoFisher), penicillin/streptomycin (Quality Biological), HEPES (ThermoFisher), and 1× GlutaMAX (ThermoFisher), with 50% vol/vol WNT3A conditioned medium (ATCC CRL-2647), 15% vol/vol R-spondin1 conditioned medium (cell line kindly provided by Calvin Kuo, Stanford University, currently available through Sigma-Aldrich), 10% vol/vol Noggin conditioned medium (cell line kindly provided by Gijs van den Brink, Tytgat Institute for Liver and Intestinal Research, and described in Heijmans et al.[43](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#c43)), 1× B27 supplement (ThermoFisher), _N_ -acetylcysteine (MilliporeSigma), human epidermal growth factor (ThermoFisher), (Leu-15) gastrin (AnaSpec), A83-01 (Tocris), SB202190 (MilliporeSigma), primocin (InvivoGen), CHIR99021 (Tocris), and Y-27632 (Tocris)]. All conditioned media were generated following, in precise detail, the protocol from the vendor or initial publication. Briefly, cells for conditioned media were grown to 80% confluency, upon which the media was replaced with Advanced DMEM with fetal bovine serum (FBS) (WNT3A) or without FBS (R-spondin1 and Noggin). Media were not changed until collection. Conditioned media from WNT3A cells were collected on days 4 and 7, combined, filtered through a filter, aliquoted, and stored at for up to 6 months. Conditioned media from R-spondin1 and Noggin cells were collected on day 7, filtered through a filter, aliquoted, and stored at for up to 6 months.
After 2 d, the organoid expansion media (without CHIR99021 and Y-27632) was replaced every other day. Colonoids were passaged every 10–14 d by harvesting in Cultrex Organoid Harvesting Solution (Bio-Techne, Minneapolis, MN) at 4°C with shaking for 45 min, as previously described.[44](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#c44) All colonoid cultures were maintained at 37°C and 5% . Unless noted, colonoid lines have been passaged times.
### Brightfield Imaging
Colonoids plated in Matrigel in 24-well plates were imaged during experiments on a Zeiss Axio Observer A1 inverted microscope (Zeiss) with images captured on CellSense imaging software (Olympus). As shown in previous intestinal organoid studies, propidium iodide (PI) was used to quantify changes in cell death.[44](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#c44)–[47](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#c47) PI and DAPI (ThermoFisher) were added at a final concentration of and , respectively. Colonoids were placed at 37°C for 1 h before imaging. Images were viewed and processed using OlyVia (Olympus). Total cells and PI+ cells were counted in ImageJ. independent experiments were performed, with at least 1,000 cells counted in each condition.
### Uranium-Bearing Dust Analysis
UBD was obtained from a church attic on indigenous lands that lie adjacent to the 3,000-acre Jackpile mine in New Mexico. Permission was granted by the Environmental and Natural Resources Department of the Laguna Pueblo with assistance in obtaining the sample from the Southwest Information and Research Center. Samples of the collected dust were sieved using a filter. Transmission electron microscopy (TEM) analyses of the dust were carried out on all particulates size fraction for an unbiased analysis. Particulates from this fraction, suspended in acetone, were deposited onto holey carbon film supported on Cu TEM grids using pipettes. For the TEM analysis, five TEM grid squares were studied in detail. The samples were air dried and then analyzed using a JEOL 2010F FEG TEM/STEM (JEOL USA) operating at . The samples were imaged using a variety of different TEM and scanning transmission electron microscopy (STEM) techniques, including bright-field TEM and high-angle annular dark-field STEM imaging. Uranium-bearing particulates were identified on the JEOL 2010F by energy dispersive X-ray spectroscopy (EDS) using an Oxford Instruments AZtecEnergy EDS system (Oxford Instruments) coupled to an Oxford X-Max silicon drift detector (SDD). The detailed distribution of U-bearing phases and their association with other minerals was obtained using STEM EDS spectral X-ray mapping. TEM and STEM images of two examples of these U-bearing grains are shown in Figure S1.
### Treatment and Preparation of Single Cells from 3D Colonoids for Single-Cell RNA Sequencing
Colonoids used for sequencing are described in Table S1. UBD was diluted in organoid expansion media, then colonoids in Matrigel were treated overnight (18 h) with UBD, with control colonoids receiving the same volume/volume of organoid expansion medium as the vehicle. Dosage was chosen based on prior studies[26](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#c26),[48](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#c48)–[51](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#c51) that showed molecular changes to cells or animals exposed to uranium, but with minimal cytotoxic damage, since acute uranium exposure is not linked to cytotoxicity. Next day, colonoids were harvested from Matrigel using Cultrex Organoid Harvesting Solution, then resuspended in TrypLE enzyme solution (ThermoFisher), transferred to a 24-well plate and spun at at 37°C for 45 min in a spinoculator (Mixer HC; USA Scientific) to digest into single cells. Single cells were washed in of Advanced DMEM/F12, pelleted again, resuspended in of organoid expansion media and filtered with Flowmi cell strainers (SP Bel-Art). Cells were transported to the UNM Analytical and Translational Genomics Core in the UNM Comprehensive Cancer Center for counting and processing. Average processing time from colonoids to library preparation was hours, and the cells were highly viable ().
### Single Cell and Library Preparation
Using the Cell Countess II FL (ThermoFisher), single cells and viability were counted by Trypan Blue exclusion. All reagents for single cell and library preparation were purchased from 10X Genomics and provided by the UNM Comprehensive Cancer Center. The Next GEM Chip G was loaded and put into the Chromium Controller for processing, creating the GEMs (gel beads-in-emulsion). cDNA synthesis of the GEMs was completed per the 10X Chromium Next GEM 3’ protocol. Synthesized cDNA was evaluated for quality and quantity on a DNA High-Sensitivity Chip on the Agilent BioAnalyzer machine. Next, cDNA was fragmented, adaptor ligated, and sample indexed (barcoded) with PCR. Once the libraries were completed, they were processed on a DNA High-Sensitivity Chip with the Agilent BioAnalyzer to ensure that the final libraries were bp in size. They were also processed with the KAPA Library quantification kit to verify the concentration of final libraries.
### Pooling and Sequencing Quality Control
Sample concentration was determined using Agilent Tape Station 4200 and Invitrogen Qubit 4.0. Samples were diluted with molecular grade water, normalized, and pooled at , according to Illumina protocol, prior to quantitative real-time polymerase chain reaction (qPCR) to determine cluster efficiency. Samples were loaded onto an Illumina NovaSEQ 6000 instrument and sequenced on S4 Flow cells at specified read length. Data were delivered in FASTQ file format.
### Processing FASTQ Reads into Gene Expression Matrices
Raw FASTQ reads from sequencing were processed using Cell Ranger (version 2.0.0) software from 10X Genomics. In brief, sequencing reads were demultiplexed, unique molecular identifiers (UMI) were collapsed, and reads aligned to the human transcriptome GRCh38-3.0.0 as the reference.[52](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#c52)
### scRNA-Seq Quality Control and Processing of 10X Sequencing Data
The raw data created by Cell Ranger was read into the Seurat R package for quality control and downstream processing. Low-quality cells expressing genes and genes expressed in fewer than three cells were filtered out. After looking at the violin plots of the number of counts, features (genes), and percent mitochondria per sample, we further filtered cells expressing genes in control samples and cells expressing genes in UBD-treated samples to avoid probable doublets. Cells expressing of mitochondrial genes were filtered out in control and UBD samples. To account for differences in sequencing depth across samples, the data was log-normalized and scaled.
### Clustering, Visualization, and Cell Annotation
For cell clustering, we sorted the top 2,000 highly variable genes that differed from cell to cell for downstream analysis. The dimensionality of the data was reduced by principal component analysis (PCA) on the highly variable genes. Since Seurat clusters cells based on their PCA scores, the elbow plot was used to determine the top principal components (PCs) for use in clustering the data. From the elbow plot, the top 15 PCs were used in clustering. Cells were divided into unsupervised clusters using a shared-nearest-neighbor modularity optimization-based clustering algorithm. Cells were clustered at a resolution of 0.4 and visualized using Uniform Manifold Approximation and Projection (UMAP) plots.[53](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#c53) Differentially expressed genes per cluster were calculated using the Seurat Wilcoxon signed-rank test. Cluster annotation was based off the top genes (log2 fold change and ) and marker genes described in literature. Control and UBD-treated samples were annotated together to draw direct comparisons. Marker gene expression was visualized using violin plots and dot-plots (size of the dot reflects the percentage of cells expressing the gene, and color indicates relative expression). R version 4.0.4 was used for all data plotting and analysis.
### Differences between Treatment
For comparisons between control and UBD-treated samples, the samples from the same subject were log normalized and scaled separately and were merged into one dataset for downstream analysis. Where necessary, all control samples were merged into one dataset, and all UBD-treated samples were merged into another dataset.
### Percentage of Cells and Statistical Analysis
The relative abundance of each cell type was calculated as percentage of cells per condition (control and UBD). We tested for statistical significance using two-tailed -test (GraphPad Prism 9.0).
### Pathway Analysis
Differentially expressed (DE) genes among clusters or samples were ranked according to their expression with the upregulated ones at the top of the list and the downregulated ones at the bottom. The ranked gene list was imported onto gProfiler (Ensembl 109). Pathway analysis was done using gene ontology (GO) biological process and Reactome (version 84). After running the analysis, the gProfiler data was downloaded as a GMT file, while the results file was downloaded in the Generic Enrichment Map format. Both types of files were then imported into Cytoscape (version 3.9.1) where the EnrichmentMap app was used to generate the enrichment map. We used a -value cutoff and _q_ -value cutoff of 1 and an Overlap Coefficient cutoff of 0.5. Using DE genes from Seurat as input, protein–protein interactions of upregulated genes in the UBD-treated enteroendocrine cluster were determined using the STRING app on Cytoscape.
### Immunofluorescence, RNAscope, and Confocal Imaging
Colonoids were harvested from Matrigel using Cultrex Organoid Harvesting Solution, then fixed for 40 min in 4% paraformaldehyde (Electron Microscopy Sciences). Immunostaining was performed as previously described.[44](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#c44) In brief, fixed colonoids were permeabilized and blocked and incubated with primary antibodies at 4 degrees overnight. After washing, organoids were incubated with fluorescent secondary antibodies and DAPI for 1 h at room temperature, washed, and mounted. Primary and secondary antibodies used are listed in Table S2.
Fixed colonoids were prepared for whole-mount staining with RNAscope Multiplex Fluorescent version 2 using probes Hs-CHGA and Hs-PROX1 (Table S3). Sample preparation and labeling were performed according to the manufacturer’s protocol (Advanced Cell Diagnostics) with slight modifications as previously described.[54](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#c54) Opal fluorophores (Perkin Elmer) and DAPI were used for visualization. Confocal imaging was carried out in the UNM Comprehensive Cancer Center Fluorescence Microscopy Shared Resource using an LSM 800 AiryScan confocal microscope (Zeiss) and UNM AIM Center using an LSM 900 AiryScan confocal microscope (Zeiss). Total cells and antibody- or oligo-positive cells were counted in ImageJ. Immunofluorescence and RNAscope were independent experiments, with at least 1,000 cells counted in each condition.
### Statistics
Data were represented as . Statistical significances were calculated using Student’s -test. Significance was represented as at least . All experiments were performed on a minimum of three different colonoid lines derived from separate normal human subjects, with a total of seven different colonoid lines used throughout these studies for validation. refers to numbers of independent replicates performed. All analyses were performed on GraphPad Prism 9.0.
### Additional Software
All graphical schematics within this manuscript were generated in BioRender.com under an academic plan and includes a publication license.
## Results
### Mineralogy Analysis of the UBD
Mineralogy of the size fraction in the UBD was based on an analysis of individual mineral grains or grain clusters and is shown in Figure S1. This fraction consisted of a variety of minerals that are commonly present in aeolian dust in arid environments including quartz, feldspar, micas/clay minerals, calcite, and gypsum.[55](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#c55) Rare minerals included baryte, rutile, and iron oxides such as hematite and magnetite. In addition to these minerals, extensive searching of the TEM grids revealed the presence of submicron grains of U-bearing minerals. Mineralogy of two of the five TEM grid square areas showed no U-bearing minerals, but in the three other grid squares, several grains were found, often in close spatial association. Of relevance, _a_) no U-bearing grains were found with grain sizes , _b_) the grains always occurred in clusters intimately mixed with other minerals—no individual isolated grains were found, and _c_) the individual U-bearing minerals were as small as in size or less. Estimating the abundance of these grains was very challenging from TEM data, but our estimate was . The mass fraction was significantly lower, because the grains were all submicron, a characteristic that is unique to the U-bearing minerals.[56](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#c56),[57](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#c57)
### Cytotoxicity Assay of Colonoids Post-Acute UBD Exposure
To determine if UBD induced cell death or morphological changes in human colonoids, three biologically distinct human colonoid cultures were treated with vehicle (control, organoid expansion media) or UBD (resuspended in organoid expansion media) overnight ( h). Brightfield imaging of both conditions showed no differences in gross morphology after acute UBD exposure ([Figure 1A](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#f1)). Propidium iodide (PI) staining was used to visualize dead cells with DAPI as the nuclear counterstain. No significant difference in PI-positive cells was observed between control ([Figure 1B](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#f1)) and UBD-treated colonoids ([Figure 1C](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#f1)) or found when PI-positive cells were quantified ([Figure 1D](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#f1)).
#### Figure 1.
[![Figure 1A is a stained colonoids and displays two columns, namely, control and uranium-bearing dust. Figures 1B and 1C are stained colonoids with two columns, namely, 4′,6-diamidino-2-phenylindole and propidium iodide. Figure 1D is a bar graph, plotting percent propidium iodide plus or per total cells, plotting 0.0 to 2.0 in increments of 0.5 \(y-axis\) across control and uranium-bearing dust \(x-axis\). Figure 1E is a set of two violin plots, plotting gene expression numbers per cell, ranging from 1,000 to 3,000 in increments of 1,000 and 2,500 to 7,500 in increments of 2,500 \(y-axis\) across control and uranium-bearing dust \(x-axis\) for number of genes detected per cell.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c5df/11108582/d7b2f7ec83ab/ehp13855_f1.jpg)](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=11108582_ehp13855_f1.jpg)
[Open in a new tab](https://pmc.ncbi.nlm.nih.gov/articles/figure/f1/)
Effect of acute uranium-bearing dust exposure on human colonoids. (A–D) Human colonoids were treated with vehicle (control) or uranium-bearing dust (UBD) overnight; . (A) Representative brightfield images of control and UBD-treated colonoids; . (B and C) Representative confocal images of control (B) and UBD (C) colonoids stained with propidium iodide (PI) (dead cells); . (D) Quantification of PI-positive cells relative to total cell populations in control and UBD-treated colonoids. (not significant) based on Student’s -test; . Source data can be found in Excel Table S1. (E) Representative violin plots for gene expression numbers per cell shows control colonoids express while some UBD-exposed express .
### scRNA-Seq Analysis of Colonoids Post-Acute UBD Exposure
To understand if UBD was changing specific cell lineages, we performed droplet-based single-cell RNA sequencing (scRNA-seq) on three biologically unique colonoid lines (Table S1) under control or UBD treated conditions. Despite the lack of changes in cell death and overall morphology, computationally there was a stark difference in cellular gene expression. Cells in control colonoids expressed genes while some UBD-treated cells expressed genes ([Figure 1E](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#f1)).
Graph-based clustering using the Louvain algorithm and marker-gene analysis revealed few clusters in control colonoids that primarily represented cells founds in the deep crypt, as shown in the representative control colonoid UMAP ([Figure 2A](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#f2)). Clusters were manually identified using marker genes and published datasets[58](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#c58)–[60](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#c60) as cycling transit amplifying (TA) cells, stem cells, and two clusters of secretory progenitor cells (Excel Tables S2–S4). The colonoids represented nondifferentiated epithelia since they were maintained in organoid expansion media that contained WNT3A/Rspo-1/Noggin.
#### Figure 2.
![Figure 2A is a graph titled control, plotting Uniform Manifold Approximation and Projection 2, ranging from negative 2 to 4 in increments of 2 \(y-axis\) across Uniform Manifold Approximation and Projection 1, ranging from negative 3 to 3 in increments of 3 \(x-axis\) for 0 implies cycling T A, 1 implies secretory progenitor, 2 implies secretory progenitor, and 3 implies stem cells. Figure 2B is a graph titled uranium-bearing dust, plotting Uniform Manifold Approximation and Projection 2, ranging from negative 5.0 to 2.5 in increments of 2.5 \(y-axis\) across Uniform Manifold Approximation and Projection 1, ranging from negative 5 to 5 in increments of 5 \(x-axis\) for 0 implies secretory progenitor, 1 implies goblet cells, 2 implies absorptive colonocytes, 3 implies cycling T A, 4 implies absorptive progenitor, 5 implies stem cells, 6 implies enteroendocrine-enterochromaffin, and 7 implies enteroendocrine-L. Figure 2C is a set of two graphs titled control and uranium-bearing dust, plotting Uniform Manifold Approximation and Projection 2, ranging from negative 5.0 to 2.5 in increments of 2.5 \(y-axis\) across Uniform Manifold Approximation and Projection 1, ranging from negative 3 to 3 in increments of 3 \(x-axis\) for 0 implies cycling T A, 1 implies stem cell, 2 implies secretory progenitor, 3 implies cycling T A, 4 implies absorptive progenitor, 5 implies absorptive colonocyte, 6 implies enteroendocrine, 7 implies goblet cell, respectively. Figure 2D is a clustered bar graph, plotting percent cells per total cells, ranging from 0 to 80 in increments of 20 \(y-axis\) across cluster number, ranging from 0 to 7 in unit increments \(x-axis\) for control and uranium-bearing dust.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c5df/11108582/eebaf9e6ecca/ehp13855_f2.jpg)
[Open in a new tab](https://pmc.ncbi.nlm.nih.gov/articles/figure/f2/)
Comparison of epithelial cell lineages in control and uranium-bearing dust (UBD)-treated colonoids using single-cell RNA sequencing. (A) Control and (B) UBD-treated colonoids were initially analyzed separately to determine differences between the biological controls. Putative cluster identification is listed below each representative Uniform Manifold Approximation and Projection (UMAP). Source data for the Seurat pipeline can be found in Excel Tables S2–S7. (C) Representative UMAP of merged control and UBD-treated colonoids to perform direct comparison analysis between each cell cluster. (D) Comparison of relative cell percentages for each cluster between control and UBD-treated colonoids. Proliferative clusters (0, 1, 3) are circled. Source data for merged analysis can be found in Excel Table S8, and source data for cluster percentages can be found in Excel Table S9. Note: TA, transit amplifying.
In contrast, UBD-treated colonoids maintained in identical media, had a higher degree of cell type variability, as shown in the representative UBD-treated colonoid UMAP. They grouped into clusters (Excel Tables S5–S7) that included differentiated cell types and were identified as secretory progenitors, goblet cells, absorptive colonocytes, cycling TA cells, absorptive progenitors, stem cells, and enteroendocrine (EEC)–enterochromaffin cells and EEC–L cells ([Figure 2B](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#f2)). Since the three colonoid lines were from biologically unique donors, differential gene and clustering analysis was performed on each control and UBD-treated colonoid set to first determine if there were major differences due to genetic variability. Similar clustering patterns and differentially expressed genes were seen in all three colonoid lines for control (mainly deep crypt cells) and UBD-treated (mainly differentiated cells) conditions. Thus, control and UBD-treated colonoid datasets were merged to directly compare cell proportions in each cluster ([Figure 2C](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#f2); Excel Table S8). There was an increase in all types of differentiated cells but a decline in proliferative (stem and cycling TA) cells in the UBD-treated compared to control colonoids ([Figure 2D](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#f2)).
To visualize differences within clusters between control and UBD-treated colonoids, the datasets were merged, clustered, and split according to treatment. UBD-treated cells (blue) did not entirely overlap with control cells (red) (Figure S2). Volcano plots of control and UBD-treated colonoids for all three colonoid lines showed significantly upregulated (red) and downregulated (blue) genes by treatment (Figure S3). In UBD-treated colonoids, genes associated with secretory lineage (such as _TFF3_ , _CHGA_ , _PCSK1N_ , _MUC2_ , _FCGBP_) were upregulated, while cell cycle and mitochondrial genes were downregulated (in relation to control). In contrast, mitochondrial genes were downregulated in UBD-treated colonoids (in relation to control colonoids).
### Measurement of EECs in UBD-Treated Human Colonoids
Disruption of EECs and hormones have been reported in various gut-related and systemic metabolic diseases.[61](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#c61)–[63](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#c63) Among the cell type differences observed in UBD-treated colonoids, there was an expansion in EECs ([Figure 2C](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#f2), cluster 6). Since EECs are rare and control colonoids did not show major biological variation when clustered independently, we merged the control and -treated colonoids for downstream analysis. We identified EECs by the expression of chromogranin A (_CHGA_) and chromogranin B (_CHGB_), both well characterized pan-EEC markers. Feature plots and dot plots were used to identify _CHGA_ - and _CHGB_ -expressing cells ([Figure 3A](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#f3)) and to quantify _CHGA_ and _CHGB_ expression ([Figure 3B](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#f3)), respectively. Total number of EECs were 3.6-fold higher, with higher transcript levels of _CHGA_ (4.4 fold) and _CHGB_ (6.3 fold) in UBD-treated colonoids compared to controls ([Figure 3C](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#f3)). These differences were quantified via immunostaining of CHGA-expressing cells ([Figure 3D,E](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#f3)). Additionally, detection of RNA oligos via RNAscope _in situ_ hybridization showed a significant difference in _CHGA_ + oligos per cell in UBD-treated colonoids compared to controls ([Figure 3F,G](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#f3)).
#### Figure 3.
![Figure 3A is a set of four Feature plots. On the left, the two Feature plots are titled control, plotting Uniform Manifold Approximation and Projection 2, ranging from negative 5 to 10 in increments of 5 \(left y-axis\) and Chromogranin A and Chromogranin B \(right y-axis\) across Uniform Manifold Approximation and Projection 1, ranging from negative 5 to 10 in increments of 5. On the right, the two Feature plots are titled uranium-bearing dust, plotting Uniform Manifold Approximation and Projection 2, ranging from negative 5 to 10 in increments of 5 \(left y-axis\) and Chromogranin A and Chromogranin B \(right y-axis\) across Uniform Manifold Approximation and Projection 1, ranging from negative 5 to 10 in increments of 5. Figure 3B is a dot plot, plotting Identity, ranging from control to uranium-bearing dust \(y-axis\) across Features, ranging from Chromogranin A to Chromogranin B \(x-axis\) for average expression, ranging from negative 0.4 to 0.4 in increments of 0.4 and percent expressed, ranging from 2.0 to 0.5 in decrements of 0.5. Figure 3C is a graph, plotting percentage of total cells, plotting from 0 to 5 in unit increments \(y-axis\) across total enteroendocrine, Chromogranin A, and Chromogranin B \(x-axis\) for control and uranium-bearing dust. Figure 3D is a stained colonoid displaying two columns, namely, control and uranium-bearing dust, and one row, namely, immunostaining. Figures 3E and 3G are bar graphs, plotting percent Chromogranin A plus or per total cells, ranging from 0 to 8 in increments of 2 and Chromogranin A ribonucleic acid scope dots per cell, ranging from 0 to 5 in unit increments \(y-axis\) across control and uranium-bearing dust \(x-axis\). Figure 3F is a stained colonoid displaying two columns, namely, control and uranium-bearing dust, and one row, namely, ribonucleic acid scope.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c5df/11108582/43393f9c6d2b/ehp13855_f3.jpg)
[Open in a new tab](https://pmc.ncbi.nlm.nih.gov/articles/figure/f3/)
Expression of enteroendocrine cell markers in control and uranium-bearing dust (UBD)-treated colonoids. (A) Feature plot of pooled control (left) and UBD-treated (right) colonoids () with cells differentially expressing Chromogranin A (_CHGA_) (top) and Chromogranin B (_CHGB_) (bottom) indicated by black arrows. (B) Dot plot of enteroendocrine cell (EEC) markers _CHGA_ and _CHGB_. (C) Quantification of total EECs and EEC markers _CHGA_ and _CHGB_ compared between the pooled control and UBD-exposed colonoids. At least 10,000 cells were analyzed for each condition. * (total EE), 0.07 (_CHGA_), and 0.03 (_CHGB_). Source data for EEC cluster quantification can be found in Excel Table S10. Source data for the Seurat pipeline for the merged analysis can be found in Excel Table S8. (D) Representative immunostaining for CHGA in control and UBD-treated colonoids. CHGA, green; phalloidin, white; nuclei, blue; . (E) Quantification of percent CHGA+ cells in total cell populations in control and UBD-treated colonoids. *; . Source data found in Excel Table S11. (F) Representative RNAscope fluorescence staining of _CHGA_ in colonoids to validate scRNA-seq. _CHGA_ , red; nuclei, blue; . (G) Quantification of RNAscope staining of _CHGA_ dots per cell in total cell populations in control and UBD-treated colonoids. *; . Data are presented as . Source data found in Excel Table S11. -Values based on Student’s -test. Note: SEM, standard error of the mean.
### Evaluation of Serotonin Producing-Enterochromaffin Cells and Peptide YY-Producing L Cells upon UBD Exposure
Enterochromaffin (EC) and L cells are the two most abundant EEC subtypes in the colon; however, there are at least 12 subtypes described in the small bowel and colon.[64](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#c64) We further analyzed these two major EEC subtypes in our scRNA-seq datasets. EC cells were marked by expression of tryptophan hydroxylase 1 (_TPH1_), the enzyme that catalyzes the rate-limiting step in synthesis of serotonin (5-HT) from tryptophan,[65](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#c65) while L cells were marked by expression of peptide tyrosine tyrosine (_PYY_).[66](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#c66) Feature plots of merged control and UBD-treated colonoids showed a 5.4-fold expansion of EC cells and a 3.4-fold expansion of L cells following UBD treatment ([Figure 4A–C](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#f4)). A scatter plot of average gene expression in EECs by treatment showed that _PYY_ was nearly absent in control colonoids but, like _CHGA_ , was highly expressed in UBD-treated colonoids (Figure S4).
#### Figure 4.
![Figure 4A is a set of four Feature plots. On the left, the two Feature plots are titled control, plotting Uniform Manifold Approximation and Projection 2, ranging from negative 5 to 10 in increments of 5 \(left y-axis\) and tryptophan hydroxylase 1 and peptide tyrosine tyrosine \(right y-axis\) across Uniform Manifold Approximation and Projection 1, ranging from negative 5 to 10 in increments of 5. On the right, the two Feature plots are titled uranium-bearing dust, plotting Uniform Manifold Approximation and Projection 2, ranging from negative 5 to 10 in increments of 5 \(left y-axis\) and tryptophan hydroxylase 1 and peptide tyrosine tyrosine \(right y-axis\) across Uniform Manifold Approximation and Projection 1, ranging from negative 5 to 10 in increments of 5. Figures 4B, 4C, 4E, and 4G are bar graphs, plotting percentage of total cells, plotting 0 to 4 in unit increments; percentage of total cells, ranging from 0.0 to 0.8 in increments of 0.2; percent 5-hydroxytryptamine receptors plus or per total cells, plotting 0 to 10 in increments of 2; and percent of peptide tyrosine tyrosine plus or per total cells, plotting 0 to 5 in unit increments \(y-axis\) across control and uranium-bearing dust \(x-axis\). Figure 4D is a stained colonoid displaying two columns, namely, control and uranium-bearing dust, and one row, namely, enterochromaffin 5-hydroxytryptamine. Figure 4F is a stained colonoid displaying two columns, namely, control and uranium-bearing dust, and one row, namely, L cell peptide tyrosine tyrosine.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c5df/11108582/9f723f972988/ehp13855_f4.jpg)
[Open in a new tab](https://pmc.ncbi.nlm.nih.gov/articles/figure/f4/)
Expression of enterochromaffin and L cell markers in control and uranium-bearing dust (UBD)-treated colonoids. (A) Feature plot of pooled control (left) and UBD-treated (right) colonoids () with cells differentially expressing tryptophan hydroxylase 1 (TPH1) (top) and peptide tyrosine tyrosine (PYY) (bottom) shown in orange and indicated with arrows. (B) Quantification of cells differentially expressing TPH1 compared between the pooled control and UBD-treated colonoids. *. (C) Quantification of cells differentially expressing PYY compared between the pooled control and UBD-treated colonoids. *. Source data for TPH1 and PYY quantification can be found in Excel Table S12. Source data for the Seurat pipeline for the merged analysis can be found in Excel Table S8. (D) Representative immunostaining for serotonin (5-HT) in control and UBD-treated colonoids. 5-HT, red; nuclei, blue; . (E) Quantification of percent 5-HT+ cells in total cell populations in control and UBD-treated colonoids. *; . Source data found in Excel Table S13. (F) Representative immunostaining for PYY in control and UBD-treated colonoids. White arrows indicate PYY+ basolateral extensions. PYY, red; nuclei, blue; . (G) Quantification of percent PYY+ cells in total cell populations in control and UBD-treated colonoids. *; . Data are presented as . Source data found in Excel Table S14. -Values are based on Student’s -test. Note: SEM, standard error of the mean.
We validated EC and L cell expansion upon acute UBD exposure by immunostaining for 5-HT ([Figure 4D](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#f4)) and PYY ([Figure 4F](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#f4)), respectively. Quantification of 5-HT showed that there were higher numbers of EC cells in UBD-treated colonoids compared to controls ([Figure 4E](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#f4)). Similarly, the proportion of PYY expressing L cells were also higher in UBD-treated colonoids when compared to controls ([Figure 4G](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#f4)). Interestingly, we observed PYY+ basal extensions ([Figure 4F](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#f4), arrows) from UBD-treated L cells that were absent in control colonoid L cells.
### Subclustering of EECs in Control and UBD-Treated Colonoids
To further determine the spectrum of EECs that results from acute UBD exposure, we reanalyzed the EEC clusters from the combined control and UBD-treated colonoids. The EEC clusters were subclustered at a higher resolution of 1 instead of 0.4, then identified by comparing the differentially expressed genes (Excel Tables S15 and S16) with published EEC datasets.[59](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#c59),[67](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#c67) This resulted in the control colonoid EECs separating into three clusters identified as EEC progenitors, X cells, and EC cells ([Figure 5A](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#f5), left UMAP). The UBD-treated EECs subclustered into eight distinct clusters identified as EEC progenitors, X cells, EC cells, N cells, K cells, L cells, and enterochromaffin-late cells ([Figure 5A](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#f5), right UMAP). Subclustering also identified two subtypes of L cells in the UBD-treated colonoids. A small group of L cells expressed both GCG (proglucagon) and PYY (circled cluster, solid arrow) while a larger group of L cells expressed only PYY (circled cluster, dashed arrow) ([Figure 5B](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#f5)).
#### Figure 5.
![Figure 5A is a set of two graphs title control and uranium-bearing dust, plotting Uniform Manifold Approximation and Projection 2, plotting negative 5 to 5 in increments of 5 and negative 5 to 10 in increments of 5 \(y-axis\) across Uniform Manifold Approximation and Projection 1, ranging from negative 4 to 6 in increments of 2 and negative 10 to 5 in increments of 5 \(x-axis\) for 0 implies enteroendocrine progenitor, 1 implies X cells, and 2 implies enterochromaffin; and 0 implies enterochromaffin, 1 implies X cells, 2 implies N cells, 3 implies K cells, 4 implies enterochromaffin late, 5 implies K cells, 6 implies enteroendocrine progenitor, and 7 implies L cells, respectively. Figure 5b is a set of two feature plots titled glucagon and peptide tyrosine tyrosine, plotting Uniform Manifold Approximation and Projection 2, plotting negative 5 to 10 in increments of 5 \(y-axis\) across Uniform Manifold Approximation and Projection 1, plotting negative 10 to 5 in increments of 5 \(x-axis\). A scale depicts the differential expression ranges from 0 to 3 in unit increments, respectively.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c5df/11108582/34ead566b5a6/ehp13855_f5.jpg)
[Open in a new tab](https://pmc.ncbi.nlm.nih.gov/articles/figure/f5/)
Higher resolution subclustering of enteroendocrine cells in control and uranium-bearing dust (UBD)-treated colonoids. (A) Representative Uniform Manifold Approximation and Projection (UMAP) of enteroendocrine cells (EECs) in control and UBD-treated colonoids subclustered at a higher resolution and identified. (B) Feature plots of UBD-treated EECs separated by glucagon (GCG) or peptide tyrosine tyrosine (PYY) differential expression. Some L cells express both GCG and PYY (solid arrow), but a subset of L cells express only PYY (dashed arrow). Source data for the Seurat pipeline can be found in Excel Tables S15–S16.
### Gene Expression Profile Analysis of EECs in Control and UBD-Treated Colonoids
We evaluated the transcriptome of the newly formed UBD-induced EECs compared to control colonoid EECs. We compared gene profiles of the three common EEC clusters between control and UBD-treated colonoids [progenitors (control, 0; UBD, 6), X cells (control, 1; UBD, 1), and EC cells (control, 2; UBD, 0)] ([Figure 6A](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#f6)). The most notable differences were observed in the EEC progenitor clusters. The topmost expressed genes in the EEC progenitor cluster in control ([Figure 6A](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#f6), cluster 0) were associated with secretory cells. In contrast, genes in EEC progenitors in UBD-treated cells ([Figure 6A](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#f6), cluster 6) were associated with the cell cycle.
#### Figure 6.
![Figure 6A is a set of two heatmaps. On the left, the heatmap titled control, plotting I E R 3, G P X 2, C H G B, M T-C Y B, P F N 1, M T-C O 2, K R T 8, R P S A, R P S 2, O L F M 4, X I S T, A Q P 5, F A M 3 B, C A L B 1, R P L 3, N E A T 1, T F F 1, T F F 3, and R P S 4 Y 1 \(y-axis\) across enteroendocrine cells progenitor, ranging from 0 to 2 in unit increments \(x-axis\). Expression, ranging from negative 2 to 2 in unit increments and identity, ranging from 2 to 0 in unit decrements. On the right, the heatmap titled uranium-bearing dust, plotting P T T G 1, C E N P F, I D 1, U B E 2 C, H I S T 1 H 4 C, P Y Y, C K S 2, U B E 2 S, A R H G D I B, C D K N 3, H M G B 2, C C N B 1, C A L B 1, A N X A1, C C N B 2, C D C 20, S C D, L D H A, K R T 19, M T 1 X, F A B P 1, S H 3 B G R L 3, P G K 1, T F F 3, P R S S 2, T F F 2, A L 16143.1, P C K 1, L Y P D 8, M A L A T 1, C H G B, C E S 1, R E G 4, E R H, T F F 1, D D I T 4, T U B B, L D H B, S 100 A 4, K L K 6, R P S 4 Y 1, A D H 1 C, C A P N 12, U R A D, A Q P 5, L E F T Y 1, F A M 3 B, A R E G, T P H 1, G D F 15, N E A T 1, and O L F M 4 \(y-axis\) across enterochromaffin cells, ranging from 0 to 7 in unit increments \(x-axis\). Expression, ranging from negative 2 to 2 in unit increments and identity, ranging from 7 to 0 in unit decrements, respectively. Figure 6B is a set of two heatmap. On the left, the heatmap titled control, plotting B M I 1, L G R 5, O L F M 4, P R O X 1, A T O H 1, M U C 2, O G F R L 1, I S L 1, A R X, G H R L, A M N, S C G 5, F A B P 1, F A B P 5, G C G, S C T, P Y Y, C L U, N T S, C C K, A R E G, V T N, T F F 1, A G R 2, F B N 1, R E G 4, T P H 1, and C H G A \(y-axis\) across enteroendocrine cells progenitor, ranging from 0 to 2 in unit increments \(x-axis\). Expression, ranging from 0 to 3 in unit increments and identity, ranging from 2 to 0 in unit decrements. On the right, the heatmap titled uranium-bearing dust, plotting B M I 1, L G R 5, O L F M 4, P R O X 1, A T O H 1, M U C 2, O G F R L 1, I S L1, A R X, G H R L, A M N, S C G 5, F A B P 1, F A B P 5, G C G, S C T, P Y Y, C L U, N T S, C C K, A R E G, V T N, T F F 1 A G R 2, F B N 1, R E G 4, T P H 1, and C H G A \(y-axis\) across enterochromaffin cells, ranging from 0 to 7 in unit increments \(x-axis\). Expression, ranging from 0 to 3 in unit increments and identity, ranging from 7 to 0 in unit decrements, respectively. Figure 6C is a Network analysis, including M T-CO, M T-C O 2, R P S 2, R P S 18, T P T 1, R P L 21, R P S 23, R P L 10, R P S 28, R P L 11, M T-A T P 6, R P S A, R P L 39, E N S P 00000449026, R P L 7 A, R P S 7, M T-C Y B, M T-C O 3. Figure 6D is an Enrichment map displaying the following information: the cell-cell signaling by Wnt, the cell surface receptor signaling pathway involved in cell-cell signaling, the canonical Wnt signaling pathway involved in negative regulation of the apoptotic process, and the Wnt signaling pathway that regulates cell proliferation. All of these pathways are interconnected.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c5df/11108582/17565dc43326/ehp13855_f6a.jpg)
![Figure 6A is a set of two heatmaps. On the left, the heatmap titled control, plotting I E R 3, G P X 2, C H G B, M T-C Y B, P F N 1, M T-C O 2, K R T 8, R P S A, R P S 2, O L F M 4, X I S T, A Q P 5, F A M 3 B, C A L B 1, R P L 3, N E A T 1, T F F 1, T F F 3, and R P S 4 Y 1 \(y-axis\) across enteroendocrine cells progenitor, ranging from 0 to 2 in unit increments \(x-axis\). Expression, ranging from negative 2 to 2 in unit increments and identity, ranging from 2 to 0 in unit decrements. On the right, the heatmap titled uranium-bearing dust, plotting P T T G 1, C E N P F, I D 1, U B E 2 C, H I S T 1 H 4 C, P Y Y, C K S 2, U B E 2 S, A R H G D I B, C D K N 3, H M G B 2, C C N B 1, C A L B 1, A N X A1, C C N B 2, C D C 20, S C D, L D H A, K R T 19, M T 1 X, F A B P 1, S H 3 B G R L 3, P G K 1, T F F 3, P R S S 2, T F F 2, A L 16143.1, P C K 1, L Y P D 8, M A L A T 1, C H G B, C E S 1, R E G 4, E R H, T F F 1, D D I T 4, T U B B, L D H B, S 100 A 4, K L K 6, R P S 4 Y 1, A D H 1 C, C A P N 12, U R A D, A Q P 5, L E F T Y 1, F A M 3 B, A R E G, T P H 1, G D F 15, N E A T 1, and O L F M 4 \(y-axis\) across enterochromaffin cells, ranging from 0 to 7 in unit increments \(x-axis\). Expression, ranging from negative 2 to 2 in unit increments and identity, ranging from 7 to 0 in unit decrements, respectively. Figure 6B is a set of two heatmap. On the left, the heatmap titled control, plotting B M I 1, L G R 5, O L F M 4, P R O X 1, A T O H 1, M U C 2, O G F R L 1, I S L 1, A R X, G H R L, A M N, S C G 5, F A B P 1, F A B P 5, G C G, S C T, P Y Y, C L U, N T S, C C K, A R E G, V T N, T F F 1, A G R 2, F B N 1, R E G 4, T P H 1, and C H G A \(y-axis\) across enteroendocrine cells progenitor, ranging from 0 to 2 in unit increments \(x-axis\). Expression, ranging from 0 to 3 in unit increments and identity, ranging from 2 to 0 in unit decrements. On the right, the heatmap titled uranium-bearing dust, plotting B M I 1, L G R 5, O L F M 4, P R O X 1, A T O H 1, M U C 2, O G F R L 1, I S L1, A R X, G H R L, A M N, S C G 5, F A B P 1, F A B P 5, G C G, S C T, P Y Y, C L U, N T S, C C K, A R E G, V T N, T F F 1 A G R 2, F B N 1, R E G 4, T P H 1, and C H G A \(y-axis\) across enterochromaffin cells, ranging from 0 to 7 in unit increments \(x-axis\). Expression, ranging from 0 to 3 in unit increments and identity, ranging from 7 to 0 in unit decrements, respectively. Figure 6C is a Network analysis, including M T-CO, M T-C O 2, R P S 2, R P S 18, T P T 1, R P L 21, R P S 23, R P L 10, R P S 28, R P L 11, M T-A T P 6, R P S A, R P L 39, E N S P 00000449026, R P L 7 A, R P S 7, M T-C Y B, M T-C O 3. Figure 6D is an Enrichment map displaying the following information: the cell-cell signaling by Wnt, the cell surface receptor signaling pathway involved in cell-cell signaling, the canonical Wnt signaling pathway involved in negative regulation of the apoptotic process, and the Wnt signaling pathway that regulates cell proliferation. All of these pathways are interconnected.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c5df/11108582/832c1ded6f99/ehp13855_f6b.jpg)
[Open in a new tab](https://pmc.ncbi.nlm.nih.gov/articles/figure/f6/)
Gene expression profile analysis of enteroendocrine cells. (A) Heatmap of pooled enteroendocrine cells (EECs) from control colonoids (left) and uranium-bearing dust (UBD)-treated colonoids (right) comparing the gene profiles of EEC progenitors, X cells, and enterochromaffin cells between the two conditions. (B) Heatmap of selected EEC genes [chromogranin A (CHGA), tryptophan hydroxylase 1 (TPH1), regenerating family member 4 (REG4), anterior gradient 2 (AGR2), peptide tyrosine tyrosine (PYY), and fatty acid binding protein 1 (FABP1)] are denoted with arrows in control (left) and UBD-treated (right) colonoids. (C) Network analysis of top upregulated genes in UBD-treated EECs. (D) Enrichment map of higher WNT signaling in UBD-treated EECs. Source data for the Seurat pipeline can be found in Excel Tables S15–S16.
Comparing relative expression amounts of select EEC-associated genes, we found that _CHGA, TPH1_ , _REG4_ , _AGR2_ , _PYY_ , and _FABP1_ transcripts had higher expression in the UBD-treated colonoids compared to control colonoids ([Figure 6B](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#f6), arrows). We performed differential gene analysis on EECs from UBD-treated colonoids to further characterize the transcriptomic changes. Over 700 genes were differentially expressed in EECs upon UBD treatment, of which 67% were upregulated compared to control. Network analysis of top upregulated genes in UBD-treated EECs showed interactions between mitochondrial and ribosomal genes ([Figure 6C](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#f6)). We performed pathway analysis in gProfiler using the genes that were upregulated in UBD-treated EECs. Upregulated pathways from gProfiler were imported into Cytoscape for enrichment analysis. The enrichment map of UBD treated EECs shows upregulation of multiple pathways related to WNT signaling ([Figure 6D](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#f6)).
### Analysis of EEC Differentiation Transcription Factors
Based on our result indicating that EEC hyperplasia in UBD-treated colonoids may be related to upregulated WNT signaling, we sought to understand the mechanism of EEC differentiation in the context of UBD treatment. _NEUROG3_ and _NEUROD1_ are the early and late transcription factors that drive EEC differentiation in mammals.[68](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#c68) Both transcription factors are considered WNT-independent, which is contrary to our network analysis of EECS in UBD-treated colonoids. Dot plot analysis showed that transcript levels of _NEUROG3_ and _NEUROD1_ were not very different between UBD treatment and control ([Figure 7A](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#f7)).
#### Figure 7.
![Figure 7A is a dot plot, plotting control, including 1, 0, 3, 2; and uranium-bearing dust, including 0, 4, 1, 6 under identity; 7, 3, 5 under cycling T A; 2 under secretory progenitor; and 8 under enteroendocrine cells \(y-axis\) across P R O X 1, A T O H 1, N E U R O D 1, and N E U R O G 3 \(x-axis\). Average expression, ranging from negative 2 to 2 in unit increments and percent expression, ranging from 16 to 0 in decrements of 4. Figure 7B is a schematic illustration of potential enteroendocrine cell differentiation regulation in Drosophila and the human colon. On the top, the midgut stem cell of the house fly regulated by escargot, scute, prospero leads to enteroendocrine cell. At the bottom, the colonic proliferative cell of a human regulated by Notch W N T and A T O H 1 in the secretory lineage leads to goblet cell. The colonic proliferative cell of a human regulated by canonical W N T signaling with T C F or L E F and P R O X 1 in the secretory lineage leads to enteroendocrine cell. Figure 7C is a set of two feature plots titled control and uranium-bearing dust, plotting Uniform Manifold Approximation and Projection 2, plotting negative 5 to 10 in increments of 5 \(y-axis\) across Uniform Manifold Approximation and Projection 1, plotting negative 5 to 10 in increments of 5 \(x-axis\) for P R O X 1. Figures 7D, 7F, 7H are bar graphs, plotting percentage of total cells, ranging from 0 to 5 in unit increments, percent P R O X 1 plus or per total cells, ranging from 0 to 15 in increments of 5, P R O X 1 ribonucleic acid scope dots per cell, ranging from 0.0 to 2.0 in increments of 0.5 \(y-axis\) across control and uranium-bearing dust \(x-axis\). Figure 7E is a stained colonoid displaying two columns, namely, control and uranium-bearing dust, and one row, namely, immunostaining. Figure 7G is a stained colonoid displaying two columns, namely, control and uranium-bearing dust, and one row, namely, ribonucleic acid scope.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c5df/11108582/22977d66efc6/ehp13855_f7.jpg)
[Open in a new tab](https://pmc.ncbi.nlm.nih.gov/articles/figure/f7/)
Analysis of transcription factor genes in control and uranium-bearing dust (UBD)-treated colonoids. (A) Dot plot of transcription factors prospero homeobox 1 (PROX1), atonal bHLH transcription factor 1 (ATOH1) (goblet cells), neuronal differentiation 1 (NEUROD1) (EECs), and neurogenin 3 (NEUROG3) (EECs) from control (blue line) and UBD-treated (orange line) colonoids. (B) Schematic of potential EEC differentiation regulation in _Drosophila_ (Prospero, top) and human colon (PROX1, bottom). Created with Biorender.com with an academic license. (C) Feature plot of cells differentially expressing PROX1 (arrows) in pooled control and UBD-treated colonoids (). (D) Quantification of cells differentially expressing PROX1 compared between the pooled control and UBD-treated colonoids. *. Source data for PROX1 quantification can be found in Excel Table S12. Source data for the Seurat pipeline for the merged analysis can be found in Excel Table S8. (E) Representative immunostaining for PROX1 in control and UBD-treated colonoids. PROX1, red; nuclei, blue; . (F) Quantification of percent PROX1+ cells in total cell populations in control and UBD-treated colonoids. *; . Source data found in Excel Table S15. (G) Representative RNAscope fluorescence staining of PROX1 in colonoids to validate scRNA-seq. PROX1, red; nuclei, blue; . (H) Quantification of RNAscope staining of PROX1 dots per cell in total cell populations in control and UBD-treated colonoids. *; . Data are presented as . Source data found in Excel Table S16. -Values are based on Student’s -test. Note: SEM, standard error of the mean.
Thus, we searched for a possible WNT-dependent transcription factor that may be driving UBD-induced EEC differentiation. In _Drosophila_ , the transcription factor Prospero drives differentiation of midgut stem cells into EECs.[69](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#c69) The mammalian homolog of Prospero is PROX1 and is WNT-dependent.[70](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#c70) Therefore, we hypothesized that UBD treatment leads to activation of WNT signaling and upregulates PROX1 expression, driving EEC differentiation from secretory progenitor cells ([Figure 7B](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#f7)). From our scRNA-seq analysis, _PROX1_ -expressing cells were higher in number in UBD-treated colonoids compared to control colonoids ([Figure 7C,D](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#f7)). Additionally, in UBD-treated colonoids, _PROX1_ transcripts were higher in cycling TA and secretory progenitor cells ([Figure 7A](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#f7)). This PROX1 difference was validated via immunostaining ([Figure 7E,F](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#f7)) and RNAscope ([Figure 7G,H](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#f7)).
## Discussion
Gut exposure to environmental toxins and heavy metals has been linked to intestinal and systemic diseases, as well as inhibition of intestinal epithelial healing.[21](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#c21),[22](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#c22),[24](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#c24) However, transcriptomic changes within intestinal epithelial cells due to these environmental toxicants are largely unknown. UBD was used to model environmental exposure of 238U to human colonoids (a simplified, yet physiologically relevant intestinal model). EEC numbers were higher and differentiated into _de novo_ EEC subtypes as determined by droplet based scRNA sequencing. We also observed higher expression of genes associated with WNT signaling but lower expression of genes associated with proliferative cells and identified a potential noncanonical mechanism of EEC differentiation upon UBD treatment.
The mammalian intestinal epithelium regenerates continuously by a proliferative pool of stem and transient amplifying cells.[71](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#c71) These proliferative cells give rise to various mature cell types required for an intact and properly functioning epithelial barrier. Thus, a decrease in proliferative cells, as seen with acute UBD exposure, can hamper the ability of the gut to repair its epithelium with potentially severe downstream consequences. Impairment of epithelium regeneration augments access of luminal toxins, antigens, and microbes to deeper layers of the mucosa. This can lead to recruitment of immune cells and subsequent inflammation.[72](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#c72) We used PI to determine if acute UBD exposure causes cell death and found no significant differences with control. However, PI staining does not mark cells initiating early stage apoptosis.[73](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#c73) Further studies are needed to determine if UBD exposure triggered apoptotic events.
Another major observation was the expansion of hormone-producing EECs, which are typically rare and make up of the total gut epithelium.[74](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#c74) The intestine is the largest endocrine organ in humans[75](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#c75) and communicates with the brain via EECs and the enteric nervous system.[32](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#c32),[33](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#c33) Gut–brain communication is partially attributed to EEC neuropods that directly synapse with enteric nerves.[33](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#c33),[76](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#c76) Therefore, modulation of the intestine via environmental toxicants can indirectly affect systemic health. An important caveat is that communities living near abandoned uranium mines are being chronically exposed to UBD. The long-term implications and EEC changes to chronic exposures are not yet known.
In this study, UBD-treated colonoids exhibited a 5-fold higher expression of genes known to be markers of EC cells, suggesting expansion of 5-HT-producing EC cells. In the gut, 5-HT stimulated motility and secretion,[77](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#c77),[78](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#c78) enhanced mucosal immunity, and was implicated in wound healing. Peripheral 5-HT can circulate systemically via platelets and acted as a mitogen in injured organs.[79](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#c79),[80](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#c80) In a murine model of liver injury, 5-HT promoted liver regeneration after partial hepatectomy, while mice lacking TPH1 could not fully regenerate their liver.[81](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#c81) These findings suggested that EECs can undergo adaptive reprogramming during injury and provided insights for potential new regenerative therapeutics in the gut. Despite the beneficial effects of 5-HT, high levels can be detrimental. As a vasoconstrictor, sustained 5-HT release elevated blood pressure with risk of severe fluctuations in hypertensive individuals,[82](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#c82) decreased bone mineral density,[83](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#c83) and dysregulated metabolic processes like gluconeogenesis and lipolysis which may predispose individuals to diabetes and obesity.[84](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#c84)
Acute UBD exposure also led to a 3-fold higher expression of L-cell-associated genes, suggesting an expansion in L cells, the second most abundant EEC subtype in the colon. The major peptides secreted by L cells [PYY, glucagon-like peptide 1 (GLP-1), and glucagon-like peptide 2 (GLP-2)] are associated with intestinal motility, protection against inflammation, and regeneration.[85](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#c85)–[87](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#c87) Based on the known function of L cell peptides, this suggested that UBD-induced L cell expansion may be beneficial by stimulating mitogenesis essential for regeneration.
We identified EEC subtypes (N and K cells) in the UBD-treated EEC subclusters that were not present in control colonoids. In homeostatic conditions, K cells are primarily found in the small bowel.[88](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#c88) Their appearance in the colon was unexpected. K cells produce glucose-dependent insulinotropic peptide (GIP), which inhibits gastric acid secretion and gastrointestinal motility.[89](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#c89) It is counter-intuitive to slow gut motility, as we would expect an increase in gut motility and secretion to flush away toxicants from the gut. Again, this suggested that we do not have a full understanding of EECs and the functional roles of their specific hormones and peptides. Neurotensin produced by N cells stimulates colonic regeneration, gut motility,[90](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#c90),[91](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#c91) and can facilitate exit of toxicants from the body. _De novo_ differentiation of N cells during UBD exposure may be beneficial.
There has been strong evidence that implicated EECs as facultative stem cells that promote plasticity and regeneration of the gut.[88](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#c88),[92](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#c92) However, EEC expansion can also contribute to the rise of small intestinal neuroendocrine tumors.[93](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#c93) EECs can revert to stem cells following injury with high intensity radiation (12 Gy) that causes massive cell death and ablation of crypt-based stem cells.[94](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#c94) On the contrary, UBD exposure in colonoids was not associated with massive cell death. Instead, UBD exposure resulted in differences in lineage differentiation, particularly expansion of _de novo_ and mature EECs. These opposing observations suggest EEC response to injury or toxicants might depend on severity of damage, further exemplifying the plasticity of these cells.
Downstream analysis of our scRNA-seq datasets found enrichment of genes involved in WNT signaling pathways in UBD-induced EECs. However, neither _NEUROG3_[67](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#c67),[68](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#c68) nor _NEUROD1_ ,[68](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#c68),[95](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#c95) the canonical transcription factors that drive EEC differentiation,[68](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#c68),[95](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#c95) are WNT dependent. Instead, our scRNA-seq data found _PROX1_ transcripts, a WNT-dependent transcription factor,[96](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#c96) were higher in cycling TA and secretory progenitor cells in UBD-treated colonoids. _PROX1_ is the mammalian homolog of Prospero, the _Drosophila_ transcription factor[69](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#c69),[96](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#c96) that drives differentiation of midgut stem cells into EECs.[69](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#c69) Since PROX1 is a WNT-dependent transcription factor and given that WNT signaling was enriched in UBD-induced EECs, we hypothesized that toxicant exposure led to increased WNT signaling, which upregulated PROX1 and drove secretory progenitor cells into noncanonical EEC differentiation.
The UBD used in this study was obtained from a church attic located on Laguna Pueblo, from the edge of a now dormant 3,000 acre open-pit uranium mine. A caveat to this study is the lack of knowledge on the causal factors in the UBD, such as physical and chemical interactions of uranium or other heavy metals and particulates. Since this study focused on assessing transcriptional and cell differentiation effects between control and UBD-exposed colonoids, extensive investigation into the physicochemical properties of UBD would not enable further insights into this putative toxicant. Another limitation to this study is the lack of immune cells in the culture. Coculturing intestinal enteroids with primary human macrophages has been shown to augment the intestinal barrier and promote maturation compared to enteroid mono-cultures.[97](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#c97) While this current study focused on the epithelial response to UBD, the colonoid model builds the foundation for future studies of UBD exposure on intestinal immune cells and colonoids-immune cocultures.
In summary, using a real-world environmental toxicant, we showed differences following UBD exposure that suggest human colonoids develop a pathophysiological response that shifts epithelial cell lineages. Proliferative cells were decreased while secretory lineage differentiation was increased, most notably EECs between UBD and control cells. Importantly, UBD exposure led to the development of EEC subtypes that were not present in control colonoids. Further investigation is required to determine if this expansion is mostly beneficial or detrimental. Understanding changes to EEC differentiation in toxicant exposure, injury, and regeneration will provide the basis for better understanding of systemic changes and dysfunction that can originate from enterohormones in the gut.
## 
[ehp13855.smcontents.508.pdf](https://pmc.ncbi.nlm.nih.gov/articles/instance/11108582/bin/ehp13855.smcontents.508.pdf) (114KB, pdf) 
[ehp13855.s001.acco.pdf](https://pmc.ncbi.nlm.nih.gov/articles/instance/11108582/bin/ehp13855.s001.acco.pdf) (794.5KB, pdf) 
[ehp13855.s002.codeanddata.acco.zip](https://pmc.ncbi.nlm.nih.gov/articles/instance/11108582/bin/ehp13855.s002.codeanddata.acco.zip) (587.4KB, zip) 
## Acknowledgments
M.J.C., E.F.C., and J.G.I. conceived of experiments. R.A., L.L.A., and J.G.I. designed experiments. A.B. characterized and analyzed uranium-bearing dust composition. R.A., L.L.A., M.N.T., and F.T.L. performed the experiments. R.A., L.L.A., A.B., E.F.C., M.J.C., and J.G.I. analyzed the data. R.A., L.L.A., and J.G.I. wrote the manuscript with critical edits from M.N.T., F.T.L., A.B., M.J.C., and E.F.C. All authors reviewed the final version.
We thank Michael L. Paffett, Jamie L. Padilla, and Kathryn J. Brayer (University of New Mexico Comprehensive Cancer Center, Albuquerque, NM, USA) for technical assistance with confocal microscopy, single cell sequencing, and bioinformatics, respectively. We also thank Jeff Gaco from the Environment and Natural Resources Department of the Laguna Pueblo and Chris Shuey from the Southwest Information and Research Center for their permission and assistance, respectively, in obtaining the UBD used in this study.
This work was supported by NIH grants K01DK106323 (J.G.I.), R56ES034400 (J.G.I.), P42ES025589 (E.F.C. and J.G.I.), P20GM121176 (E.F.C. and J.G.I.), P20GM130422 (M.J.C.), and P50MD15706 (J.G.I.), and American Gastroenterological Association-Aman Armaan Ahmed Family Summer Undergraduate Research Fellowship (L.L.A.). This research was partially supported by the UNM Comprehensive Cancer Center Support Grant NCI P30CA118100.
Further information and requests for resources and reagents should be directed to the Lead Contact, Julie G. In (jgin@salud.unm.edu).
All unique reagents generated in this study are available from the Lead Contact without restriction.
All scRNA-seq datasets are available in Gene Expression Omnibus (GEO [GSE239584](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE239584)).
Conclusions and opinions are those of the individual authors and do not necessarily reflect the policies or views of EHP Publishing or the National Institute of Environmental Health Sciences.
## References
  * 1.Brugge D, Goble R. 2002. The history of uranium mining and the Navajo people. Am J Public Health 92(9):1410–1419, PMID: 12197966, 10.2105/ajph.92.9.1410.  [] [[PMC free article](https://pmc.ncbi.nlm.nih.gov/articles/PMC3222290/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/12197966/)] []
  * 2.Hoover J, Gonzales M, Shuey C, Barney Y, Lewis J. 2017. Elevated arsenic and uranium concentrations in unregulated water sources on the Navajo nation, USA. Expo Health 9(2):113–124, PMID: 28553666, 10.1007/s12403-016-0226-6.  [] [[PMC free article](https://pmc.ncbi.nlm.nih.gov/articles/PMC5425493/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/28553666/)] []
  * 3.Eadie GG, Kaufmann RF. 1977. Radiological evaluation of the effects of uranium mining and milling operations on selected ground water supplies in the Grants Mineral Belt, New Mexico. Health Phys 32(4):231–241, PMID: 881310, 10.1097/00004032-197704000-00003.  [] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/881310/)] []
  * 4.Lin Y, Hoover J, Beene D, Erdei E, Liu Z. 2020. Environmental risk mapping of potential abandoned uranium mine contamination on the Navajo nation, USA, using a GIS-based multi-criteria decision analysis approach. Environ Sci Pollut Res Int 27(24):30542–30557, PMID: 32468361, 10.1007/s11356-020-09257-3.  [] [[PMC free article](https://pmc.ncbi.nlm.nih.gov/articles/PMC7387200/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/32468361/)] []
  * 5.EPA (US Environmental Protection Agency). 2006. Basic Information about Radionuclides in Drinking Water. Washington, DC: EPA. []
  * 6.Hakonson-Hayes AC, Fresquez PR, Whicker FW. 2002. Assessing potential risks from exposure to natural uranium in well water. J Environ Radioact 59(1):29–40, PMID: 11848150, 10.1016/s0265-931x(01)00034-0.  [] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/11848150/)] []
  * 7.Schaider LA, Senn DB, Estes ER, Brabander DJ, Shine JP. 2014. Sources and fates of heavy metals in a mining-impacted stream: temporal variability and the role of iron oxides. Sci Total Environ 490:456–466, PMID: 24867708, 10.1016/j.scitotenv.2014.04.126.  [] [[PMC free article](https://pmc.ncbi.nlm.nih.gov/articles/PMC4145074/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/24867708/)] []
  * 8.Ravalli F, Yu Y, Bostick BC, Chillrud SN, Schilling K, Basu A, et al. 2022. Sociodemographic inequalities in uranium and other metals in community water systems across the USA, 2006–11: a cross-sectional study. Lancet Planet Health 6(4):e320–e330, PMID: 35397220, 10.1016/S2542-5196(22)00043-2.  [] [[PMC free article](https://pmc.ncbi.nlm.nih.gov/articles/PMC9037820/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/35397220/)] []
  * 9.Nolan J, Weber KA. 2015. Natural uranium contamination in major U.S. Aquifers linked to nitrate. Environ Sci Technol Lett 2(8):215–220, 10.1021/acs.estlett.5b00174. [] []
  * 10.Rump A, Eder S, Lamkowski A, Hermann C, Abend M, Port M. 2019. A quantitative comparison of the chemo- and radiotoxicity of uranium at different enrichment grades. Toxicol Lett 313:159–168, PMID: 31276769, 10.1016/j.toxlet.2019.07.004.  [] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/31276769/)] []
  * 11.Erdei E, Shuey C, Pacheco B, Cajero M, Lewis J, Rubin RL. 2019. Elevated autoimmunity in residents living near abandoned uranium mine sites on the Navajo nation. J Autoimmun 99:15–23, PMID: 30878168, 10.1016/j.jaut.2019.01.006.  [] [[PMC free article](https://pmc.ncbi.nlm.nih.gov/articles/PMC6489502/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/30878168/)] []
  * 12.WHO (World Health Organization). 2001. Depleted Uranium: Sources, Exposure, and Health Effects. Geneva, Switzerland: WHO. []
  * 13.Ma M, Wang R, Xu L, Xu M, Liu S. 2020. Emerging health risks and underlying toxicological mechanisms of uranium contamination: lessons from the past two decades. Environ Int 145:106107, PMID: 32932066, 10.1016/j.envint.2020.106107.  [] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/32932066/)] []
  * 14.Zychowski KE, Kodali V, Harmon M, Tyler CR, Sanchez B, Ordonez Suarez Y, et al. 2018. Respirable uranyl-vanadate-containing particulate matter derived from a legacy uranium mine site exhibits potentiated cardiopulmonary toxicity. Toxicol Sci 164(1):101–114, PMID: 29660078, 10.1093/toxsci/kfy064.  [] [[PMC free article](https://pmc.ncbi.nlm.nih.gov/articles/PMC6016706/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/29660078/)] []
  * 15.Legrand M, Lam S, Anselme I, Gloaguen C, Ibanez C, Eriksson P, et al. 2016. Exposure to depleted uranium during development affects neuronal differentiation in the hippocampal dentate gyrus and induces depressive-like behavior in offspring. Neurotoxicology 57:153–162, PMID: 27639536, 10.1016/j.neuro.2016.09.006.  [] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/27639536/)] []
  * 16.Monleau M, Blanchardon E, Claraz M, Paquet F, Chazel V. 2006. The effect of repeated inhalation on the distribution of uranium in rats. J Toxicol Environ Health A 69(17):1629–1649, PMID: 16854790, 10.1080/15287390600629882.  [] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/16854790/)] []
  * 17.Souidi M, Gueguen Y, Linard C, Dudoignon N, Grison S, Baudelin C, et al. 2005. In vivo effects of chronic contamination with depleted uranium on CYP3A and associated nuclear receptors PXR and CAR in the rat. Toxicology 214(1–2):113–122, PMID: 16039771, 10.1016/j.tox.2005.06.006.  [] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/16039771/)] []
  * 18.Wang S, Ran Y, Lu B, Li J, Kuang H, Gong L, et al. 2020. A review of uranium-induced reproductive toxicity. Biol Trace Elem Res 196(1):204–213, PMID: 31621007, 10.1007/s12011-019-01920-2.  [] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/31621007/)] []
  * 19.Zamora ML, Zielinski JM, Moodie GB, Falcomer RA, Hunt WC, Capello K. 2009. Uranium in drinking water: renal effects of long-term ingestion by an aboriginal community. Arch Environ Occup Health 64(4):228–241, PMID: 20007119, 10.1080/19338240903241267.  [] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/20007119/)] []
  * 20.Legaki E, Gazouli M. 2016. Influence of environmental factors in the development of inflammatory bowel diseases. World J Gastrointest Pharmacol Ther 7(1):112–125, PMID: 26855817, 10.4292/wjgpt.v7.i1.112.  [] [[PMC free article](https://pmc.ncbi.nlm.nih.gov/articles/PMC4734944/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/26855817/)] []
  * 21.Wales AD, Davies RH. 2015. Co-selection of resistance to antibiotics, biocides and heavy metals, and its relevance to foodborne pathogens. Antibiotics (Basel) 4(4):567–604, PMID: 27025641, 10.3390/antibiotics4040567.  [] [[PMC free article](https://pmc.ncbi.nlm.nih.gov/articles/PMC4790313/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/27025641/)] []
  * 22.Chiu K, Warner G, Nowak RA, Flaws JA, Mei W. 2020. The impact of environmental chemicals on the gut microbiome. Toxicol Sci 176(2):253–284, PMID: 32392306, 10.1093/toxsci/kfaa065.  [] [[PMC free article](https://pmc.ncbi.nlm.nih.gov/articles/PMC7416318/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/32392306/)] []
  * 23.Harmon ME, Lewis J, Miller C, Hoover J, Ali A-MS, Shuey C, et al. 2017. Residential proximity to abandoned uranium mines and serum inflammatory potential in chronically exposed Navajo communities. J Expo Sci Environ Epidemiol 27(4):365–371, PMID: 28120833, 10.1038/jes.2016.79.  [] [[PMC free article](https://pmc.ncbi.nlm.nih.gov/articles/PMC5781233/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/28120833/)] []
  * 24.Wagner SE, Burch JB, Bottai M, Puett R, Porter D, Bolick-Aldrich S, et al. 2011. Groundwater uranium and cancer incidence in South Carolina. Cancer Causes Control 22(1):41–50, PMID: 21080052, 10.1007/s10552-010-9669-4.  [] [[PMC free article](https://pmc.ncbi.nlm.nih.gov/articles/PMC3383652/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/21080052/)] []
  * 25.Richardson JB, Dancy BCR, Horton CL, Lee YS, Madejczyk MS, Xu ZZ, et al. 2018. Exposure to toxic metals triggers unique responses from the rat gut microbiota. Sci Rep 8(1):6578, PMID: 29700420, 10.1038/s41598-018-24931-w.  [] [[PMC free article](https://pmc.ncbi.nlm.nih.gov/articles/PMC5919903/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/29700420/)] []
  * 26.Medina S, Lauer FT, Castillo EF, Bolt AM, Ali A-MS, Liu KJ, et al. 2020. Exposures to uranium and arsenic alter intraepithelial and innate immune cells in the small intestine of male and female mice. Toxicol Appl Pharmacol 403:115155, PMID: 32710956, 10.1016/j.taap.2020.115155.  [] [[PMC free article](https://pmc.ncbi.nlm.nih.gov/articles/PMC7490749/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/32710956/)] []
  * 27.Dublineau I, Grandcolas L, Grison S, Baudelin C, Paquet F, Voisin P, et al. 2007. Modifications of inflammatory pathways in rat intestine following chronic ingestion of depleted uranium. Toxicol Sci 98(2):458–468, PMID: 17566059, 10.1093/toxsci/kfm132.  [] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/17566059/)] []
  * 28.Bolan S, Seshadri B, Keely S, Kunhikrishnan A, Bruce J, Grainge I, et al. 2021. Bioavailability of arsenic, cadmium, lead and mercury as measured by intestinal permeability. Sci Rep 11(1):14675, PMID: 34282255, 10.1038/s41598-021-94174-9.  [] [[PMC free article](https://pmc.ncbi.nlm.nih.gov/articles/PMC8289861/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/34282255/)] []
  * 29.Entwistle JA, Hursthouse AS, Marinho Reis PA, Stewart AG. 2019. Metalliferous mine dust: human health impacts and the potential determinants of disease in mining communities. Curr Pollution Rep 5(3):67–83, 10.1007/s40726-019-00108-5. [] []
  * 30.Träber SC, Höllriegl V, Li WB, Czeslik U, Rühm W, Oeh U, et al. 2014. Estimating the absorption of soil-derived uranium in humans. Environ Sci Technol 48(24):14721–14727, PMID: 25417915, 10.1021/es504171r.  [] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/25417915/)] []
  * 31.Furness JB, Rivera LR, Cho HJ, Bravo DM, Callaghan B. 2013. The gut as a sensory organ. Nat Rev Gastroenterol Hepatol 10(12):729–740, PMID: 24061204, 10.1038/nrgastro.2013.180.  [] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/24061204/)] []
  * 32.Kuwahara A, Matsuda K, Kuwahara Y, Asano S, Inui T, Marunaka Y. 2020. Microbiota-gut-brain axis: enteroendocrine cells and the enteric nervous system form an interface between the microbiota and the central nervous system. Biomed Res 41(5):199–216, PMID: 33071256, 10.2220/biomedres.41.199.  [] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/33071256/)] []
  * 33.Latorre R, Sternini C, De Giorgio R, Greenwood-Van Meerveld B. 2016. Enteroendocrine cells: a review of their role in brain-gut communication. Neurogastroenterol Motil 28(5):620–630, PMID: 26691223, 10.1111/nmo.12754.  [] [[PMC free article](https://pmc.ncbi.nlm.nih.gov/articles/PMC4842178/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/26691223/)] []
  * 34.Moore-Nall A. 2015. The legacy of uranium development on or near Indian reservations and health implications rekindling public awareness. Geosciences 5(1):15–29, 10.3390/geosciences5010015. [] []
  * 35.Sato T, Stange DE, Ferrante M, Vries RGJ, Van Es JH, Van den Brink S, et al. 2011. Long-term expansion of epithelial organoids from human colon, adenoma, adenocarcinoma, and Barrett’s epithelium. Gastroenterology 141(5):1762–1772, PMID: 21889923, 10.1053/j.gastro.2011.07.050.  [] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/21889923/)] []
  * 36.Tsang DKL, Wang RJ, De Sa O, Ayyaz A, Foerster EG, Bayer G, et al. 2022. A single cell survey of the microbial impacts on the mouse small intestinal epithelium. Gut Microbes 14(1):2108281, PMID: 35939622, 10.1080/19490976.2022.2108281.  [] [[PMC free article](https://pmc.ncbi.nlm.nih.gov/articles/PMC9361762/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/35939622/)] []
  * 37.Osinski C, Moret D, Clement K, Serradas P, Ribeiro A. 2022. Enteroendocrine system and gut barrier in metabolic disorders. Int J Mol Sci 23(7):3732, 10.3390/ijms23073732.  [] [[PMC free article](https://pmc.ncbi.nlm.nih.gov/articles/PMC8998765/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/35409092/)] []
  * 38.Goldspink DA, Lu VB, Billing LJ, Larraufie P, Tolhurst G, Gribble FM, et al. 2018. Mechanistic insights into the detection of free fatty and bile acids by ileal glucagon-like peptide-1 secreting cells. Mol Metab 7:90–101, PMID: 29167062, 10.1016/j.molmet.2017.11.005.  [] [[PMC free article](https://pmc.ncbi.nlm.nih.gov/articles/PMC5784317/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/29167062/)] []
  * 39.Bellono NW, Bayrer JR, Leitch DB, Castro J, Zhang C, O’Donnell TA, et al. 2017. Enterochromaffin cells are gut chemosensors that couple to sensory neural pathways. Cell 170(1):185–198, PMID: 28648659, 10.1016/j.cell.2017.05.034.  [] [[PMC free article](https://pmc.ncbi.nlm.nih.gov/articles/PMC5839326/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/28648659/)] []
  * 40.Basak O, Beumer J, Wiebrands K, Seno H, van Oudenaarden A, Clevers H. 2017. Induced quiescence of Lgr5+ stem cells in intestinal organoids enables differentiation of hormone-producing enteroendocrine cells. Cell Stem Cell 20(2):177–190, PMID: 27939219, 10.1016/j.stem.2016.11.001.  [] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/27939219/)] []
  * 41.Jung P, Sato T, Merlos-Suárez A, Barriga FM, Iglesias M, Rossell D, et al. 2011. Isolation and in vitro expansion of human colonic stem cells. Nat Med 17(10):1225–1227, PMID: 21892181, 10.1038/nm.2470.  [] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/21892181/)] []
  * 42.In J, Foulke-Abel J, Zachos NC, Hansen A-M, Kaper JB, Bernstein HD, et al. 2016. Enterohemorrhagic Escherichia coli reduce mucus and intermicrovillar bridges in human stem cell-derived colonoids. Cell Mol Gastroenterol Hepatol 2(1):48–62, PMID: 26855967, 10.1016/j.jcmgh.2015.10.001.  [] [[PMC free article](https://pmc.ncbi.nlm.nih.gov/articles/PMC4740923/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/26855967/)] []
  * 43.Heijmans J, van Lidth de Jeude JF, Koo B-K, Rosekrans SL, Wielenga MCB, van de Wetering M, et al. 2013. ER stress causes rapid loss of intestinal epithelial stemness through activation of the unfolded protein response. Cell Rep 3(4):1128–1139, PMID: 23545496, 10.1016/j.celrep.2013.02.031.  [] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/23545496/)] []
  * 44.In JG, Yin J, Atanga R, Doucet M, Cole RN, DeVine L, et al. 2020. Epithelial WNT2B and desert hedgehog are necessary for human colonoid regeneration after bacterial cytotoxin injury. iScience 23(10):101618, PMID: 33089106, 10.1016/j.isci.2020.101618.  [] [[PMC free article](https://pmc.ncbi.nlm.nih.gov/articles/PMC7559866/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/33089106/)] []
  * 45.Bode KJ, Mueller S, Schweinlin M, Metzger M, Brunner T. 2019. A fast and simple fluorometric method to detect cell death in 3D intestinal organoids. Biotechniques 67(1):23–28, PMID: 31218886, 10.2144/btn-2019-0023.  [] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/31218886/)] []
  * 46.Gunasekara DB, DiSalvo M, Wang Y, Nguyen DL, Reed MI, Speer J, et al. 2018. Development of arrayed colonic organoids for screening of secretagogues associated with enterotoxins. Anal Chem 90(3):1941–1950, PMID: 29281259, 10.1021/acs.analchem.7b04032.  [] [[PMC free article](https://pmc.ncbi.nlm.nih.gov/articles/PMC6028038/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/29281259/)] []
  * 47.den Daas SA, Soffientini U, Chokshi S, Mehta G. 2022. A permeability assay for mouse intestinal organoids. STAR Protoc 3(2):101365, PMID: 35542176, 10.1016/j.xpro.2022.101365.  [] [[PMC free article](https://pmc.ncbi.nlm.nih.gov/articles/PMC9079111/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/35542176/)] []
  * 48.LaCerte C, Xie H, Aboueissa AM, Wise JP Sr.. 2010. Particulate depleted uranium is cytotoxic and clastogenic to human lung epithelial cells. Mutat Res 697(1–2):33–37, PMID: 20172046, 10.1016/j.mrgentox.2010.01.013.  [] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/20172046/)] []
  * 49.Hornung V, Bauernfeind F, Halle A, Samstad EO, Kono H, Rock KL, et al. 2008. Silica crystals and aluminum salts activate the NALP3 inflammasome through phagosomal destabilization. Nat Immunol 9(8):847–856, PMID: 18604214, 10.1038/ni.1631.  [] [[PMC free article](https://pmc.ncbi.nlm.nih.gov/articles/PMC2834784/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/18604214/)] []
  * 50.Ohshima S, Xu Y, Takahama M. 1998. Effects of uranium ore dust on cultured human lung cells. Environ Toxicol Pharmacol 5(4):267–271, PMID: 21781873, 10.1016/s1382-6689(98)00010-6.  [] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/21781873/)] []
  * 51.Valderrama A, Ortiz-Hernández P, Agraz-Cibrián JM, Tabares-Guevara JH, Gómez DM, Zambrano-Zaragoza JF, et al. 2022. Particulate matter (PM(10)) induces in vitro activation of human neutrophils, and lung histopathological alterations in a mouse model. Sci Rep 12(1):7581, PMID: 35534522, 10.1038/s41598-022-11553-6.  [] [[PMC free article](https://pmc.ncbi.nlm.nih.gov/articles/PMC9083477/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/35534522/)] []
  * 52.Rosenbloom KR, Armstrong J, Barber GP, Casper J, Clawson H, Diekhans M, et al. 2015. The UCSC genome browser database: 2015 update. Nucleic Acids Res 43(D1):D670–D681, PMID: 25428374, 10.1093/nar/gku1177.  [] [[PMC free article](https://pmc.ncbi.nlm.nih.gov/articles/PMC4383971/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/25428374/)] []
  * 53.Becht E, McInnes L, Healy J, Dutertre C-A, Kwok IWH, Ng LG, et al. 2018. Dimensionality reduction for visualizing single-cell data using UMAP. Nat Biotechnol 37:38–44, 10.1038/nbt.4314. [] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/30531897/)] []
  * 54.Atanga R, Parra AS, In JG. 2022. Efficient RNA and RNA-protein co-detection in 3D colonoids by whole-mount staining. STAR Protoc 3(4):101775, PMID: 36313534, 10.1016/j.xpro.2022.101775.  [] [[PMC free article](https://pmc.ncbi.nlm.nih.gov/articles/PMC9615320/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/36313534/)] []
  * 55.Muhs DR, Prospero JM, Baddock MC, Gill TE. 2014. Identifying sources of aeolian mineral dust: present and past. In: Mineral Dust: A Key Player in the Earth System. Knippertz P, Stuut J-BW, eds. Dordrecht, Netherlands: Springer, 51–74. []
  * 56.Min M, Chen J, Wang J, Wei G, Fayek M. 2005. Mineral paragenesis and textures associated with sandstone-hosted roll-front uranium deposits, NW China. Ore Geology Reviews 26(1–2):51–69, 10.1016/j.oregeorev.2004.10.001. [] []
  * 57.WoldeGabriel G, Boukhalfa H, Ware SD, Cheshire M, Reimus P, Heikoop J, et al. 2014. Characterization of cores from an in-situ recovery mined uranium deposit in Wyoming: implications for post-mining restoration. Chemical Geology 390:32–45, 10.1016/j.chemgeo.2014.10.009. [] []
  * 58.Burclaff J, Bliton RJ, Breau KA, Ok MT, Gomez-Martinez I, Ranek JS, et al. 2022. A proximal-to-distal survey of healthy adult human small intestine and colon epithelium by single-cell transcriptomics. Cell Mol Gastroenterol Hepatol 13(5):1554–1589, PMID: 35176508, 10.1016/j.jcmgh.2022.02.007.  [] [[PMC free article](https://pmc.ncbi.nlm.nih.gov/articles/PMC9043569/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/35176508/)] []
  * 59.Beumer J, Puschhof J, Bauzá-Martinez J, Martínez-Silgado A, Elmentaite R, James KR, et al. 2020. High-resolution mRNA and secretome atlas of human enteroendocrine cells. Cell 181(6):1291–1306, PMID: 32407674, 10.1016/j.cell.2020.04.036.  [] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/32407674/)] []
  * 60.Wang Y, Song W, Wang J, Wang T, Xiong X, Qi Z, et al. 2020. Single-cell transcriptome analysis reveals differential nutrient absorption functions in human intestine. J Exp Med 217(2):e20191130, 10.1084/jem.20191130.  [] [[PMC free article](https://pmc.ncbi.nlm.nih.gov/articles/PMC7041720/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/31753849/)] []
  * 61.Keller J, Beglinger C, Holst JJ, Andresen V, Layer P. 2009. Mechanisms of gastric emptying disturbances in chronic and acute inflammation of the distal gastrointestinal tract. Am J Physiol Gastrointest Liver Physiol 297(5):G861–G868, PMID: 20501434, 10.1152/ajpgi.00145.2009.  [] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/20501434/)] []
  * 62.Jorsal T, Rhee NA, Pedersen J, Wahlgren CD, Mortensen B, Jepsen SL, et al. 2018. Enteroendocrine K and L cells in healthy and type 2 diabetic individuals. Diabetologia 61(2):284–294, PMID: 28956082, 10.1007/s00125-017-4450-9.  [] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/28956082/)] []
  * 63.French SJ, Murray B, Rumsey RD, Sepple CP, Read NW. 1993. Preliminary studies on the gastrointestinal responses to fatty meals in obese people. Int J Obes Relat Metab Disord 17(5):295–300, PMID: 8389339.  [[PubMed](https://pubmed.ncbi.nlm.nih.gov/8389339/)] []
  * 64.Goldspink DA, Reimann F, Gribble FM. 2018. Models and tools for studying enteroendocrine cells. Endocrinology 159(12):3874–3884, PMID: 30239642, 10.1210/en.2018-00672.  [] [[PMC free article](https://pmc.ncbi.nlm.nih.gov/articles/PMC6215081/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/30239642/)] []
  * 65.Liu N, Sun S, Wang P, Sun Y, Hu Q, Wang X. 2021. The mechanism of secretion and metabolism of gut-derived 5-hydroxytryptamine. Int J Mol Sci 22(15):7931, 10.3390/ijms22157931.  [] [[PMC free article](https://pmc.ncbi.nlm.nih.gov/articles/PMC8347425/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/34360695/)] []
  * 66.Spreckley E, Murphy KG. 2015. The L-cell in nutritional sensing and the regulation of appetite. Front Nutr 2:23, PMID: 26258126, 10.3389/fnut.2015.00023.  [] [[PMC free article](https://pmc.ncbi.nlm.nih.gov/articles/PMC4507148/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/26258126/)] []
  * 67.Gehart H, van Es JH, Hamer K, Beumer J, Kretzschmar K, Dekkers JF, et al. 2019. Identification of enteroendocrine regulators by real-time single-cell differentiation mapping. Cell 176(5):1158–1173, PMID: 30712869, 10.1016/j.cell.2018.12.029.  [] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/30712869/)] []
  * 68.Li HJ, Ray SK, Singh NK, Johnston B, Leiter AB. 2011. Basic helix-loop-helix transcription factors and enteroendocrine cell differentiation. Diabetes Obes Metab 13(1):5–12, PMID: 21824251, 10.1111/j.1463-1326.2011.01438.x.  [] [[PMC free article](https://pmc.ncbi.nlm.nih.gov/articles/PMC3467197/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/21824251/)] []
  * 69.Zeng X, Hou SX. 2015. Enteroendocrine cells are generated from stem cells through a distinct progenitor in the adult drosophila posterior midgut. Development 142(4):644–653, PMID: 25670791, 10.1242/dev.113357.  [] [[PMC free article](https://pmc.ncbi.nlm.nih.gov/articles/PMC4325374/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/25670791/)] []
  * 70.Cha B, Geng X, Mahamud MR, Zhang JY, Chen L, Kim W, et al. 2018. Complementary Wnt sources regulate lymphatic vascular development via PROX1-Dependent Wnt/beta-Catenin signaling. Cell Rep 25(3):571–584, PMID: 30332639, 10.1016/j.celrep.2018.09.049.  [] [[PMC free article](https://pmc.ncbi.nlm.nih.gov/articles/PMC6264919/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/30332639/)] []
  * 71.Rees WD, Tandun R, Yau E, Zachos NC, Steiner TS. 2020. Regenerative intestinal stem cells induced by acute and chronic injury: the saving grace of the epithelium? Front Cell Dev Biol 8:583919, PMID: 33282867, 10.3389/fcell.2020.583919.  [] [[PMC free article](https://pmc.ncbi.nlm.nih.gov/articles/PMC7688923/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/33282867/)] []
  * 72.Zundler S, Becker E, Schulze LL, Neurath MF. 2019. Immune cell trafficking and retention in inflammatory bowel disease: mechanistic insights and therapeutic advances. Gut 68(9):1688–1700, PMID: 31127023, 10.1136/gutjnl-2018-317977.  [] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/31127023/)] []
  * 73.Darzynkiewicz Z, Bruno S, Del Bino G, Gorczyca W, Hotz MA, Lassota P, et al. 1992. Features of apoptotic cells measured by flow cytometry. Cytometry 13(8):795–808, PMID: 1333943, 10.1002/cyto.990130802.  [] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/1333943/)] []
  * 74.Sternini C, Anselmi L, Rozengurt E. 2008. Enteroendocrine cells: a site of ‘taste’ in gastrointestinal chemosensing. Curr Opin Endocrinol Diabetes Obes 15(1):73–78, PMID: 18185066, 10.1097/MED.0b013e3282f43a73.  [] [[PMC free article](https://pmc.ncbi.nlm.nih.gov/articles/PMC2943060/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/18185066/)] []
  * 75.Buchan AM. 1999. Nutrient tasting and signaling mechanisms in the gut III. Endocrine cell recognition of luminal nutrients. Am J Physiol 277(6):G1103–G1107, PMID: 10600808, 10.1152/ajpgi.1999.277.6.G1103.  [] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/10600808/)] []
  * 76.Bohorquez DV, Chandra R, Samsa LA, Vigna SR, Liddle RA. 2011. Characterization of basal pseudopod-like processes in ileal and colonic PYY cells. J Mol Histol 42(1):3–13, PMID: 21061049, 10.1007/s10735-010-9302-6.  [] [[PMC free article](https://pmc.ncbi.nlm.nih.gov/articles/PMC4281170/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/21061049/)] []
  * 77.Gershon MD, Tack J. 2007. The serotonin signaling system: from basic understanding to drug development for functional GI disorders. Gastroenterology 132(1):397–414, PMID: 17241888, 10.1053/j.gastro.2006.11.002.  [] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/17241888/)] []
  * 78.Pithadia AB, Jain SM. 2009. 5-Hydroxytryptamine receptor subtypes and their modulators with therapeutic potentials. J Clin Med Res 1(2):72–80, PMID: 22505971, 10.4021/jocmr2009.05.1237.  [] [[PMC free article](https://pmc.ncbi.nlm.nih.gov/articles/PMC3318857/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/22505971/)] []
  * 79.Shah PA, Park CJ, Shaughnessy MP, Cowles RA. 2021. Serotonin as a mitogen in the gastrointestinal tract: revisiting a familiar molecule in a new role. Cell Mol Gastroenterol Hepatol 12(3):1093–1104, PMID: 34022423, 10.1016/j.jcmgh.2021.05.008.  [] [[PMC free article](https://pmc.ncbi.nlm.nih.gov/articles/PMC8350061/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/34022423/)] []
  * 80.Atanga R, Singh V, In JG. 2023. Intestinal enteroendocrine cells: present and future druggable targets. Int J Mol Sci 24(10):8836, 10.3390/ijms24108836.  [] [[PMC free article](https://pmc.ncbi.nlm.nih.gov/articles/PMC10218851/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/37240181/)] []
  * 81.Lesurtel M, Graf R, Aleil B, Walther DJ, Tian Y, Jochum W, et al. 2006. Platelet-derived serotonin mediates liver regeneration. Science 312(5770):104–107, PMID: 16601191, 10.1126/science.1123842.  [] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/16601191/)] []
  * 82.Amstein R, Fetkovska N, Luscher TF, Kiowski W, Buhler FR. 1988. Age and the platelet serotonin vasoconstrictor axis in essential hypertension. J Cardiovasc Pharmacol 11(suppl 1):S35–S40, PMID: 2459512.  [[PubMed](https://pubmed.ncbi.nlm.nih.gov/2459512/)] []
  * 83.Bliziotes M. 2010. Update in serotonin and bone. J Clin Endocrinol Metab 95(9):4124–4132, PMID: 20823465, 10.1210/jc.2010-0861.  [] [[PMC free article](https://pmc.ncbi.nlm.nih.gov/articles/PMC2936065/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/20823465/)] []
  * 84.Yabut JM, Crane JD, Green AE, Keating DJ, Khan WI, Steinberg GR. 2019. Emerging roles for serotonin in regulating metabolism: new implications for an ancient molecule. Endocr Rev 40(4):1092–1107, PMID: 30901029, 10.1210/er.2018-00283.  [] [[PMC free article](https://pmc.ncbi.nlm.nih.gov/articles/PMC6624793/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/30901029/)] []
  * 85.Drucker DJ. 2018. Mechanisms of action and therapeutic application of glucagon-like peptide-1. Cell Metab 27(4):740–756, PMID: 29617641, 10.1016/j.cmet.2018.03.001.  [] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/29617641/)] []
  * 86.Lebrun LJ, Lenaerts K, Kiers D, Pais de Barros J-P, Le Guern N, Plesnik J, et al. 2017. Enteroendocrine L cells sense LPS after gut barrier injury to enhance GLP-1 secretion. Cell Rep 21(5):1160–1168, PMID: 29091756, 10.1016/j.celrep.2017.10.008.  [] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/29091756/)] []
  * 87.Drucker DJ, Erlich P, Asa SL, Brubaker PL. 1996. Induction of intestinal epithelial proliferation by glucagon-like peptide 2. Proc Natl Acad Sci USA 93(15):7911–7916, PMID: 8755576, 10.1073/pnas.93.15.7911.  [] [[PMC free article](https://pmc.ncbi.nlm.nih.gov/articles/PMC38848/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/8755576/)] []
  * 88.Rindi G, Leiter AB, Kopin AS, Bordi C, Solcia E. 2004. The “normal” endocrine cell of the gut: changing concepts and new evidences. Ann NY Acad Sci 1014:1–12, PMID: 15153415, 10.1196/annals.1294.001.  [] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/15153415/)] []
  * 89.Cho YM, Kieffer TJ. 2010. K-cells and glucose-dependent insulinotropic polypeptide in health and disease. Vitam Horm 84:111–150, PMID: 21094898, 10.1016/B978-0-12-381517-0.00004-7.  [] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/21094898/)] []
  * 90.Evers BM, Izukura M, Chung DH, Parekh D, Yoshinaga K, Greeley GH, et al. 1992. Neurotensin stimulates growth of colonic mucosa in young and aged rats. Gastroenterology 103(1):86–91, PMID: 1612361, 10.1016/0016-5085(92)91099-p.  [] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/1612361/)] []
  * 91.Rock SA, Jiang K, Wu Y, Liu Y, Li J, Weiss HL, et al. 2022. Neurotensin regulates proliferation and stem cell function in the small intestine in a nutrient-dependent manner. Cell Mol Gastroenterol Hepatol 13(2):501–516, PMID: 34560309, 10.1016/j.jcmgh.2021.09.006.  [] [[PMC free article](https://pmc.ncbi.nlm.nih.gov/articles/PMC8688554/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/34560309/)] []
  * 92.Johnson NM, Parham LR, Na J, Monaghan KE, Kolev HM, Klochkova A, et al. 2022. Autophagic state prospectively identifies facultative stem cells in the intestinal epithelium. EMBO Rep 23(11):e55209, PMID: 36120829, 10.15252/embr.202255209.  [] [[PMC free article](https://pmc.ncbi.nlm.nih.gov/articles/PMC9638868/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/36120829/)] []
  * 93.Sei Y, Feng J, Zhao X, Wank SA. 2020. Role of an active reserve stem cell subset of enteroendocrine cells in intestinal stem cell dynamics and the genesis of small intestinal neuroendocrine tumors. Am J Physiol Gastrointest Liver Physiol 319(4):G494–G501, PMID: 32845170, 10.1152/ajpgi.00278.2020.  [] [[PMC free article](https://pmc.ncbi.nlm.nih.gov/articles/PMC7654644/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/32845170/)] []
  * 94.Yan KS, Gevaert O, Zheng GXY, Anchang B, Probert CS, Larkin KA, et al. 2017. Intestinal enteroendocrine lineage cells possess homeostatic and injury-inducible stem cell activity. Cell Stem Cell 21(1):78–90, PMID: 28686870, 10.1016/j.stem.2017.06.014.  [] [[PMC free article](https://pmc.ncbi.nlm.nih.gov/articles/PMC5642297/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/28686870/)] []
  * 95.Schonhoff SE, Giel-Moloney M, Leiter AB. 2004. Minireview: development and differentiation of gut endocrine cells. Endocrinology 145(6):2639–2644, PMID: 15044355, 10.1210/en.2004-0051.  [] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/15044355/)] []
  * 96.Karalay O, Doberauer K, Vadodaria KC, Knobloch M, Berti L, Miquelajauregui A, et al. 2011. Prospero-related homeobox 1 gene (Prox1) is regulated by canonical Wnt signaling and has a stage-specific role in adult hippocampal neurogenesis. Proc Natl Acad Sci USA 108(14):5807–5812, PMID: 21436036, 10.1073/pnas.1013456108.  [] [[PMC free article](https://pmc.ncbi.nlm.nih.gov/articles/PMC3078392/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/21436036/)] []
  * 97.Noel G, Baetz NW, Staab JF, Donowitz M, Kovbasnjuk O, Pasetti MF, et al. 2017. A primary human macrophage-enteroid co-culture model to investigate mucosal gut physiology and host-pathogen interactions. Sci Rep 7:45270, PMID: 28345602, 10.1038/srep45270.  [] [[PMC free article](https://pmc.ncbi.nlm.nih.gov/articles/PMC5366908/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/28345602/)] []

## Associated Data
_This section collects any data citations, data availability statements, or 
### 
[ehp13855.smcontents.508.pdf](https://pmc.ncbi.nlm.nih.gov/articles/instance/11108582/bin/ehp13855.smcontents.508.pdf) (114KB, pdf) 
[ehp13855.s001.acco.pdf](https://pmc.ncbi.nlm.nih.gov/articles/instance/11108582/bin/ehp13855.s001.acco.pdf) (794.5KB, pdf) 
[ehp13855.s002.codeanddata.acco.zip](https://pmc.ncbi.nlm.nih.gov/articles/instance/11108582/bin/ehp13855.s002.codeanddata.acco.zip) (587.4KB, zip) 
Articles from Environmental Health Perspectives are provided here courtesy of **National Institute of Environmental Health Sciences**
![Close](https://pmc.ncbi.nlm.nih.gov/static/img/usa-icons/close.svg)
## ACTIONS
  * [ PDF (6.4 MB) ](https://pmc.ncbi.nlm.nih.gov/articles/pdf/ehp13855.pdf)
  * Cite
  * Collections
  * Permalink
## PERMALINK
Copy

## RESOURCES
###  Similar articles 
###  Cited by other articles 
###  Links to NCBI Databases 
## On this page
  * [Abstract](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#abstract1)
  * [Introduction](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#s1)
  * [Methods](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#s2)
  * [Results](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#s3)
  * [Discussion](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#s4)
  * [
  * [Acknowledgments](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#ack1)
  * [References](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#ref-list1)
  * [Associated Data](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582#_ad93_)

## Cite
  * Copy
  * [ Download .nbib .nbib ](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582 "Download a file for external citation management software")
  * Format: AMA  APA  MLA  NLM 

## Add to Collections
Create a new collection
Add to an existing collection
Name your collection *
Choose a collection 
Unable to load your collection due to an error [Please try again](https://pmc.ncbi.nlm.nih.gov/articles/PMC11108582)
Add  Cancel 
Follow NCBI 
[ NCBI RSS feed ](https://ncbiinsights.ncbi.nlm.nih.gov/)
Connect with NLM 
  * [ Web Policies ](https://www.nlm.nih.gov/web_policies.html)
  * [ FOIA ](https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office)

  * [ Help ](https://support.nlm.nih.gov/?pagename=cloudpmc-viewer%3Apmc%3Aarticle%3Ajournal)
  * [ Accessibility ](https://www.nlm.nih.gov/accessibility.html)
  * [ Careers ](https://www.nlm.nih.gov/careers/careers.html)

  * [ NLM ](https://www.nlm.nih.gov/)
  * [ NIH ](https://www.nih.gov/)

Back to Top
  *[*]: required